Index: test/dom/button.test.js
===================================================================
--- test/dom/button.test.js
+++ test/dom/button.test.js
@@ -3,11 +3,11 @@
 import { sendKeys } from '@web/test-runner-commands';
 import '../../src/vaadin-button.js';
 
-describe('vaadin-button', () => {
+describe('vaadin24-button', () => {
   let button;
 
   beforeEach(() => {
-    button = fixtureSync('<vaadin-button>Confirm</vaadin-button>');
+    button = fixtureSync('<vaadin24-button>Confirm</vaadin24-button>');
   });
 
   describe('host', () => {
