Index: iconset.js
===================================================================
--- iconset.js
+++ iconset.js
@@ -6,7 +6,7 @@
 import '@polymer/iron-iconset-svg/iron-iconset-svg.js';
 
 console.warn(
-  `WARNING: Since Vaadin 23.1, using <iron-icon> is deprecated. Please use <vaadin-icon> and '@vaadin/icons/vaadin-iconset.js' instead.`,
+  `WARNING: Since Vaadin 23.1, using <iron-icon> is deprecated. Please use <vaadin23-icon> and '@vaadin/icons/vaadin-iconset.js' instead.`,
 );
 
 const template = document.createElement('template');
