Index: scroller/src/vaadin-scroller.d.ts
===================================================================
--- scroller/src/vaadin-scroller.d.ts	null
+++ scroller/src/vaadin-scroller.d.ts	null
@@ -3,18 +3,18 @@
  * Copyright (c) 2020 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * `<vaadin-scroller>` provides a simple way to enable scrolling when its content is overflowing.
+ * `<vaadin24-scroller>` provides a simple way to enable scrolling when its content is overflowing.
  *
  * ```
- * <vaadin-scroller>
+ * <vaadin24-scroller>
  *   <div>Content</div>
- * </vaadin-scroller>
+ * </vaadin24-scroller>
  * ```
  * The following attributes are exposed for styling:
  *
@@ -40,7 +40,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-scroller': Scroller;
+    'vaadin24-scroller': Scroller;
   }
 }
 
