Index: src/vaadin-scroller.js
===================================================================
--- src/vaadin-scroller.js
+++ src/vaadin-scroller.js
@@ -4,20 +4,20 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { OverflowController } from '@vaadin/component-base/src/overflow-controller.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { OverflowController } from '@scoped-vaadin/component-base/src/overflow-controller.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * `<vaadin-scroller>` provides a simple way to enable scrolling when its content is overflowing.
+ * `<vaadin24-scroller>` provides a simple way to enable scrolling when its content is overflowing.
  *
  * ```
- * <vaadin-scroller>
+ * <vaadin24-scroller>
  *   <div>Content</div>
- * </vaadin-scroller>
+ * </vaadin24-scroller>
  * ```
  * The following attributes are exposed for styling:
  *
@@ -65,7 +65,7 @@
   }
 
   static get is() {
-    return 'vaadin-scroller';
+    return 'vaadin24-scroller';
   }
 
   static get properties() {
