Index: test/visual/vaadin-iconset.test.js
===================================================================
--- test/visual/vaadin-iconset.test.js
+++ test/visual/vaadin-iconset.test.js
@@ -1,9 +1,9 @@
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/icon/vaadin-icon.js';
+import '@scoped-vaadin/icon/vaadin-icon.js';
 import '../../vaadin-iconset.js';
 
-describe('vaadin-iconset', () => {
+describe('vaadin24-iconset', () => {
   let wrapper;
 
   before(async () => {
@@ -13,13 +13,13 @@
     const icons = await data.json();
     wrapper = fixtureSync(`
       <div style="display: flex; flex-wrap: wrap; width: 600px">
-        ${icons.map((icon) => `<vaadin-icon icon="lumo:${icon}"></vaadin-icon>`).join('\n')}
+        ${icons.map((icon) => `<vaadin24-icon icon="lumo:${icon}"></vaadin24-icon>`).join('\n')}
       </div>
     `);
     await nextFrame();
   });
 
   it('basic', async () => {
-    await visualDiff(wrapper, 'vaadin-iconset');
+    await visualDiff(wrapper, 'vaadin24-iconset');
   });
 });
