Index: test/dom/list-box.test.js
===================================================================
--- test/dom/list-box.test.js
+++ test/dom/list-box.test.js
@@ -2,11 +2,11 @@
 import { fixtureSync } from '@vaadin/testing-helpers';
 import '../../src/vaadin-list-box.js';
 
-describe('vaadin-list-box', () => {
+describe('vaadin23-list-box', () => {
   let listBox;
 
   beforeEach(() => {
-    listBox = fixtureSync('<vaadin-list-box></vaadin-list-box>');
+    listBox = fixtureSync('<vaadin23-list-box></vaadin23-list-box>');
   });
 
   describe('host', () => {
