Index: theme/lumo/vaadin-lit-grid-filter.d.ts
===================================================================
--- theme/lumo/vaadin-lit-grid-filter.d.ts
+++ theme/lumo/vaadin-lit-grid-filter.d.ts
@@ -1,2 +1,2 @@
-import '@vaadin/text-field/theme/lumo/vaadin-lit-text-field.js';
+import '@scoped-vaadin/text-field/theme/lumo/vaadin-lit-text-field.js';
 import '../../src/vaadin-lit-grid-filter.js';
