Index: grid/theme/lumo/vaadin-grid-tree-toggle-styles.js
===================================================================
--- grid/theme/lumo/vaadin-grid-tree-toggle-styles.js	null
+++ grid/theme/lumo/vaadin-grid-tree-toggle-styles.js	null
@@ -1,10 +1,10 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-grid-tree-toggle',
+  'vaadin24-grid-tree-toggle',
   css`
     :host {
       --vaadin-grid-tree-toggle-level-offset: 2em;
