Index: src/vaadin-text-area.js
===================================================================
--- src/vaadin-text-area.js
+++ src/vaadin-text-area.js
@@ -1,41 +1,42 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/src/vaadin-input-container.js';
+import '@scoped-vaadin/input-container/src/vaadin-input-container.js';
 import { html, PolymerElement } from '@polymer/polymer';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ResizeMixin } from '@vaadin/component-base/src/resize-mixin.js';
-import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
-import { InputFieldMixin } from '@vaadin/field-base/src/input-field-mixin.js';
-import { LabelledInputController } from '@vaadin/field-base/src/labelled-input-controller.js';
-import { PatternMixin } from '@vaadin/field-base/src/pattern-mixin.js';
-import { inputFieldShared } from '@vaadin/field-base/src/styles/input-field-shared-styles.js';
-import { TextAreaController } from '@vaadin/field-base/src/text-area-controller.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ResizeMixin } from '@scoped-vaadin/component-base/src/resize-mixin.js';
+import { TooltipController } from '@scoped-vaadin/component-base/src/tooltip-controller.js';
+import { InputFieldMixin } from '@scoped-vaadin/field-base/src/input-field-mixin.js';
+import { LabelledInputController } from '@scoped-vaadin/field-base/src/labelled-input-controller.js';
+import { PatternMixin } from '@scoped-vaadin/field-base/src/pattern-mixin.js';
+import { inputFieldShared } from '@scoped-vaadin/field-base/src/styles/input-field-shared-styles.js';
+import { TextAreaController } from '@scoped-vaadin/field-base/src/text-area-controller.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
-registerStyles('vaadin-text-area', inputFieldShared, { moduleId: 'vaadin-text-area-styles' });
+registerStyles('vaadin23-text-area', inputFieldShared, { moduleId: 'vaadin-text-area-styles' });
 
 /**
- * `<vaadin-text-area>` is a web component for multi-line text input.
+ * `<vaadin23-text-area>` is a web component for multi-line text input.
  *
  * ```html
- * <vaadin-text-area label="Comment"></vaadin-text-area>
+ * <vaadin23-text-area label="Comment"></vaadin23-text-area>
  * ```
  *
  * ### Prefixes and suffixes
  *
- * These are child elements of a `<vaadin-text-area>` that are displayed
+ * These are child elements of a `<vaadin23-text-area>` that are displayed
  * inline with the input, before or after.
  * In order for an element to be considered as a prefix, it must have the slot
  * attribute set to `prefix` (and similarly for `suffix`).
  *
  * ```html
- * <vaadin-text-area label="Description">
+ * <vaadin23-text-area label="Description">
  *   <div slot="prefix">Details:</div>
  *   <div slot="suffix">The end!</div>
- * </vaadin-text-area>
+ * </vaadin23-text-area>
  * ```
  *
  * ### Styling
@@ -46,8 +47,8 @@
  * -------------------------------|----------------------------|---------
  * `--vaadin-field-default-width` | Default width of the field | `12em`
  *
- * `<vaadin-text-area>` provides the same set of shadow DOM parts and state attributes as `<vaadin-text-field>`.
- * See [`<vaadin-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
+ * `<vaadin23-text-area>` provides the same set of shadow DOM parts and state attributes as `<vaadin23-text-field>`.
+ * See [`<vaadin23-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/custom-theme/styling-components) documentation.
  *
@@ -66,7 +67,7 @@
  */
 export class TextArea extends ResizeMixin(PatternMixin(InputFieldMixin(ThemableMixin(ElementMixin(PolymerElement))))) {
   static get is() {
-    return 'vaadin-text-area';
+    return 'vaadin23-text-area';
   }
 
   static get template() {
@@ -116,7 +117,7 @@
           box-shadow: none;
         }
 
-        /* Override styles from <vaadin-input-container> */
+        /* Override styles from <vaadin23-input-container> */
         [part='input-field'] ::slotted(textarea) {
           align-self: stretch;
           white-space: pre-wrap;
@@ -144,7 +145,7 @@
           <span part="required-indicator" aria-hidden="true"></span>
         </div>
 
-        <vaadin-input-container
+        <vaadin23-input-container
           part="input-field"
           readonly="[[readonly]]"
           disabled="[[disabled]]"
@@ -156,7 +157,7 @@
           <slot name="textarea"></slot>
           <slot name="suffix" slot="suffix"></slot>
           <div id="clearButton" part="clear-button" slot="suffix" aria-hidden="true"></div>
-        </vaadin-input-container>
+        </vaadin23-input-container>
 
         <div part="helper-text">
           <slot name="helper"></slot>
@@ -351,4 +352,4 @@
   }
 }
 
-customElements.define(TextArea.is, TextArea);
+internalCustomElements.define(TextArea.is, TextArea);
