Index: src/vaadin-text-area-mixin.js
===================================================================
--- src/vaadin-text-area-mixin.js
+++ src/vaadin-text-area-mixin.js
@@ -3,10 +3,10 @@
  * Copyright (c) 2021 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ResizeMixin } from '@vaadin/component-base/src/resize-mixin.js';
-import { InputFieldMixin } from '@vaadin/field-base/src/input-field-mixin.js';
-import { LabelledInputController } from '@vaadin/field-base/src/labelled-input-controller.js';
-import { TextAreaController } from '@vaadin/field-base/src/text-area-controller.js';
+import { ResizeMixin } from '@scoped-vaadin/component-base/src/resize-mixin.js';
+import { InputFieldMixin } from '@scoped-vaadin/field-base/src/input-field-mixin.js';
+import { LabelledInputController } from '@scoped-vaadin/field-base/src/labelled-input-controller.js';
+import { TextAreaController } from '@scoped-vaadin/field-base/src/text-area-controller.js';
 
 /**
  * A mixin providing common text area functionality.
@@ -258,7 +258,7 @@
      */
     __minRowsChanged(minRows) {
       if (minRows < 2) {
-        console.warn('<vaadin-text-area> minRows must be at least 2.');
+        console.warn('<vaadin24-text-area> minRows must be at least 2.');
       }
     }
 
