Index: test/visual/lumo/overlay.test.js
===================================================================
--- test/visual/lumo/overlay.test.js
+++ test/visual/lumo/overlay.test.js
@@ -9,7 +9,7 @@
     div = document.createElement('div');
     div.style.height = '100%';
 
-    element = fixtureSync('<vaadin-overlay></vaadin-overlay>', div);
+    element = fixtureSync('<vaadin24-overlay></vaadin24-overlay>', div);
     element.renderer = (root) => {
       root.textContent = 'Simple overlay with text';
     };
