Index: notification/src/vaadin-notification.d.ts
===================================================================
--- notification/src/vaadin-notification.d.ts
+++ notification/src/vaadin-notification.d.ts
@@ -4,10 +4,10 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { TemplateResult } from 'lit';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { OverlayClassMixin } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
-import { ThemePropertyMixin } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { OverlayClassMixin } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ThemePropertyMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 
 export type NotificationPosition =
   | 'bottom-center'
@@ -40,7 +40,7 @@
 }
 
 /**
- * An element used internally by `<vaadin-notification>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-notification>`. Not intended to be used separately.
  */
 declare class NotificationContainer extends ElementMixin(ThemableMixin(HTMLElement)) {
   /**
@@ -50,12 +50,12 @@
 }
 
 /**
- * An element used internally by `<vaadin-notification>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-notification>`. Not intended to be used separately.
  */
 declare class NotificationCard extends ThemableMixin(HTMLElement) {}
 
 /**
- * `<vaadin-notification>` is a Web Component providing accessible and customizable notifications (toasts).
+ * `<vaadin24-notification>` is a Web Component providing accessible and customizable notifications (toasts).
  *
  * ### Rendering
  *
@@ -67,7 +67,7 @@
  * users are able to check if there is already content in `root` for reusing it.
  *
  * ```html
- * <vaadin-notification id="notification"></vaadin-notification>
+ * <vaadin24-notification id="notification"></vaadin24-notification>
  * ```
  * ```js
  * const notification = document.querySelector('#notification');
@@ -83,10 +83,10 @@
  *
  * ### Styling
  *
- * `<vaadin-notification>` uses `<vaadin-notification-card>` internal
+ * `<vaadin24-notification>` uses `<vaadin24-notification-card>` internal
  * themable component as the actual visible notification cards.
  *
- * The following shadow DOM parts of the `<vaadin-notification-card>` are available for styling:
+ * The following shadow DOM parts of the `<vaadin24-notification-card>` are available for styling:
  *
  * Part name | Description
  * ----------------|----------------
@@ -95,8 +95,8 @@
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
- * Note: the `theme` attribute value set on `<vaadin-notification>` is
- * propagated to the internal `<vaadin-notification-card>`.
+ * Note: the `theme` attribute value set on `<vaadin24-notification>` is
+ * propagated to the internal `<vaadin24-notification-card>`.
  *
  * @fires {CustomEvent} opened-changed - Fired when the `opened` property changes.
  */
@@ -145,9 +145,9 @@
    * Custom function for rendering the content of the notification.
    * Receives two arguments:
    *
-   * - `root` The `<vaadin-notification-card>` DOM element. Append
+   * - `root` The `<vaadin24-notification-card>` DOM element. Append
    *   your content to it.
-   * - `notification` The reference to the `<vaadin-notification>` element.
+   * - `notification` The reference to the `<vaadin24-notification>` element.
    */
   renderer: NotificationRenderer | undefined;
 
@@ -184,9 +184,9 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-notification-container': NotificationContainer;
-    'vaadin-notification-card': NotificationCard;
-    'vaadin-notification': Notification;
+    'vaadin24-notification-container': NotificationContainer;
+    'vaadin24-notification-card': NotificationCard;
+    'vaadin24-notification': Notification;
   }
 }
 
