Index: src/vaadin-notification-mixin.d.ts
===================================================================
--- src/vaadin-notification-mixin.d.ts
+++ src/vaadin-notification-mixin.d.ts
@@ -1,7 +1,7 @@
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { OverlayClassMixinClass } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import type { OverlayStackMixinClass } from '@vaadin/overlay/src/vaadin-overlay-stack-mixin.js';
-import type { ThemePropertyMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import type { OverlayClassMixinClass } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import type { OverlayStackMixinClass } from '@scoped-vaadin/overlay/src/vaadin-overlay-stack-mixin.js';
+import type { ThemePropertyMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 import type { Notification } from './vaadin-notification.js';
 
 export type NotificationPosition =
@@ -67,9 +67,9 @@
    * Custom function for rendering the content of the notification.
    * Receives two arguments:
    *
-   * - `root` The `<vaadin-notification-card>` DOM element. Append
+   * - `root` The `<vaadin24-notification-card>` DOM element. Append
    *   your content to it.
-   * - `notification` The reference to the `<vaadin-notification>` element.
+   * - `notification` The reference to the `<vaadin24-notification>` element.
    */
   renderer: NotificationRenderer | undefined;
 
