Index: src/vaadin-notification.js
===================================================================
--- src/vaadin-notification.js
+++ src/vaadin-notification.js
@@ -4,22 +4,22 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { NotificationContainerMixin, NotificationMixin } from './vaadin-notification-mixin.js';
 import { notificationCardStyles, notificationContainerStyles } from './vaadin-notification-styles.js';
 
-registerStyles('vaadin-notification-container', notificationContainerStyles, {
+registerStyles('vaadin24-notification-container', notificationContainerStyles, {
   moduleId: 'vaadin-notification-container-styles',
 });
 
-registerStyles('vaadin-notification-card', notificationCardStyles, {
+registerStyles('vaadin24-notification-card', notificationCardStyles, {
   moduleId: 'vaadin-notification-card-styles',
 });
 
 /**
- * An element used internally by `<vaadin-notification>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-notification>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -48,12 +48,12 @@
   }
 
   static get is() {
-    return 'vaadin-notification-container';
+    return 'vaadin24-notification-container';
   }
 }
 
 /**
- * An element used internally by `<vaadin-notification>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-notification>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -72,7 +72,7 @@
   }
 
   static get is() {
-    return 'vaadin-notification-card';
+    return 'vaadin24-notification-card';
   }
 
   /** @protected */
@@ -83,7 +83,7 @@
 }
 
 /**
- * `<vaadin-notification>` is a Web Component providing accessible and customizable notifications (toasts).
+ * `<vaadin24-notification>` is a Web Component providing accessible and customizable notifications (toasts).
  *
  * ### Rendering
  *
@@ -95,7 +95,7 @@
  * users are able to check if there is already content in `root` for reusing it.
  *
  * ```html
- * <vaadin-notification id="notification"></vaadin-notification>
+ * <vaadin24-notification id="notification"></vaadin24-notification>
  * ```
  * ```js
  * const notification = document.querySelector('#notification');
@@ -111,10 +111,10 @@
  *
  * ### Styling
  *
- * `<vaadin-notification>` uses `<vaadin-notification-card>` internal
+ * `<vaadin24-notification>` uses `<vaadin24-notification-card>` internal
  * themable component as the actual visible notification cards.
  *
- * The following shadow DOM parts of the `<vaadin-notification-card>` are available for styling:
+ * The following shadow DOM parts of the `<vaadin24-notification-card>` are available for styling:
  *
  * Part name | Description
  * ----------------|----------------
@@ -123,8 +123,8 @@
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
- * Note: the `theme` attribute value set on `<vaadin-notification>` is
- * propagated to the internal `<vaadin-notification-card>`.
+ * Note: the `theme` attribute value set on `<vaadin24-notification>` is
+ * propagated to the internal `<vaadin24-notification-card>`.
  *
  * @fires {CustomEvent} opened-changed - Fired when the `opened` property changes.
  * @fires {CustomEvent} closed - Fired when the notification is closed.
@@ -142,15 +142,15 @@
           display: none !important;
         }
       </style>
-      <vaadin-notification-card
+      <vaadin24-notification-card
         theme$="[[_theme]]"
         aria-live$="[[__computeAriaLive(assertive)]]"
-      ></vaadin-notification-card>
+      ></vaadin24-notification-card>
     `;
   }
 
   static get is() {
-    return 'vaadin-notification';
+    return 'vaadin24-notification';
   }
 }
 
