Index: theme/lumo/vaadin-vertical-layout-styles.js
===================================================================
--- theme/lumo/vaadin-vertical-layout-styles.js
+++ theme/lumo/vaadin-vertical-layout-styles.js
@@ -1,5 +1,5 @@
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const verticalLayout = css`
   :host([theme~='margin']) {
@@ -31,4 +31,4 @@
   }
 `;
 
-registerStyles('vaadin-vertical-layout', verticalLayout, { moduleId: 'lumo-vertical-layout' });
+registerStyles('vaadin23-vertical-layout', verticalLayout, { moduleId: 'lumo-vertical-layout' });
