Index: theme/material/vaadin-side-nav-styles.js
===================================================================
--- theme/material/vaadin-side-nav-styles.js
+++ theme/material/vaadin-side-nav-styles.js
@@ -1,7 +1,7 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 export const sideNavStyles = css`
   :host {
@@ -65,4 +65,4 @@
   }
 `;
 
-registerStyles('vaadin-side-nav', sideNavStyles, { moduleId: 'material-side-nav' });
+registerStyles('vaadin24-side-nav', sideNavStyles, { moduleId: 'material-side-nav' });
