Index: theme/material/vaadin-side-nav-item-styles.js
===================================================================
--- theme/material/vaadin-side-nav-item-styles.js
+++ theme/material/vaadin-side-nav-item-styles.js
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { fieldButton } from '@vaadin/vaadin-material-styles/mixins/field-button.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { fieldButton } from '@scoped-vaadin/vaadin-material-styles/mixins/field-button.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 export const sideNavItemStyles = css`
   [part='content'] {
@@ -146,4 +146,4 @@
   }
 `;
 
-registerStyles('vaadin-side-nav-item', [fieldButton, sideNavItemStyles], { moduleId: 'material-side-nav-item' });
+registerStyles('vaadin24-side-nav-item', [fieldButton, sideNavItemStyles], { moduleId: 'material-side-nav-item' });
