Index: virtual-list/src/lit/renderer-directives.d.ts
===================================================================
--- virtual-list/src/lit/renderer-directives.d.ts	null
+++ virtual-list/src/lit/renderer-directives.d.ts	null
@@ -4,8 +4,8 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { DirectiveResult } from 'lit/directive.js';
-import type { LitRendererResult } from '@vaadin/lit-renderer';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import type { LitRendererResult } from '@scoped-vaadin/lit-renderer';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 import type { VirtualList, VirtualListItemModel } from '../vaadin-virtual-list.js';
 
 export type VirtualListLitRenderer<TItem> = (
@@ -35,7 +35,7 @@
 }
 
 /**
- * A Lit directive for rendering the content of the `<vaadin-virtual-list>` item elements.
+ * A Lit directive for rendering the content of the `<vaadin24-virtual-list>` item elements.
  *
  * The directive accepts a renderer callback returning a Lit template and assigns it to the virtual
  * list via the `renderer` property. The renderer is called once to populate the content when
@@ -51,9 +51,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-virtual-list
+ * `<vaadin24-virtual-list
  *   ${virtualListRenderer((item, model, virtualList) => html`...`)}
- * ></vaadin-virtual-list>`
+ * ></vaadin24-virtual-list>`
  * ```
  *
  * @param renderer the renderer callback that returns a Lit template.
