Index: src/lit/renderer-directives.js
===================================================================
--- src/lit/renderer-directives.js
+++ src/lit/renderer-directives.js
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { directive } from 'lit/directive.js';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 
 export class VirtualListRendererDirective extends LitRendererDirective {
   /**
@@ -32,7 +32,7 @@
 }
 
 /**
- * A Lit directive for rendering the content of the `<vaadin-virtual-list>` item elements.
+ * A Lit directive for rendering the content of the `<vaadin23-virtual-list>` item elements.
  *
  * The directive accepts a renderer callback returning a Lit template and assigns it to the virtual
  * list via the `renderer` property. The renderer is called once to populate the content when
@@ -48,9 +48,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-virtual-list
+ * `<vaadin23-virtual-list
  *   ${virtualListRenderer((item, model, virtualList) => html`...`)}
- * ></vaadin-virtual-list>`
+ * ></vaadin23-virtual-list>`
  * ```
  *
  * @param renderer the renderer callback that returns a Lit template.
