Index: theme/material/vaadin-multi-select-combo-box-chip-styles.d.ts
===================================================================
--- theme/material/vaadin-multi-select-combo-box-chip-styles.d.ts
+++ theme/material/vaadin-multi-select-combo-box-chip-styles.d.ts
@@ -3,6 +3,6 @@
  * Copyright (c) 2021 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/typography.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
