Index: theme/material/vaadin-multi-select-combo-box-styles.js
===================================================================
--- theme/material/vaadin-multi-select-combo-box-styles.js
+++ theme/material/vaadin-multi-select-combo-box-styles.js
@@ -3,14 +3,14 @@
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { inputFieldShared } from '@vaadin/vaadin-material-styles/mixins/input-field-shared.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { inputFieldShared } from '@scoped-vaadin/vaadin-material-styles/mixins/input-field-shared.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-multi-select-combo-box-item',
+  'vaadin23-multi-select-combo-box-item',
   css`
     @media (any-hover: hover) {
       :host(:hover[readonly]) {
@@ -56,6 +56,6 @@
   }
 `;
 
-registerStyles('vaadin-multi-select-combo-box', [inputFieldShared, multiSelectComboBox], {
+registerStyles('vaadin23-multi-select-combo-box', [inputFieldShared, multiSelectComboBox], {
   moduleId: 'material-multi-select-combo-box',
 });
