Index: multi-select-combo-box/theme/material/vaadin-multi-select-combo-box-styles.js
===================================================================
--- multi-select-combo-box/theme/material/vaadin-multi-select-combo-box-styles.js	null
+++ multi-select-combo-box/theme/material/vaadin-multi-select-combo-box-styles.js	null
@@ -3,17 +3,17 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/theme/material/vaadin-input-container-styles.js';
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { comboBoxItem } from '@vaadin/combo-box/theme/material/vaadin-combo-box-item-styles.js';
-import { comboBoxLoader, comboBoxOverlay } from '@vaadin/combo-box/theme/material/vaadin-combo-box-overlay-styles.js';
-import { item } from '@vaadin/item/theme/material/vaadin-item-styles.js';
-import { inputFieldShared } from '@vaadin/vaadin-material-styles/mixins/input-field-shared.js';
-import { loader } from '@vaadin/vaadin-material-styles/mixins/loader.js';
-import { menuOverlay } from '@vaadin/vaadin-material-styles/mixins/menu-overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/input-container/theme/material/vaadin-input-container-styles.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { comboBoxItem } from '@scoped-vaadin/combo-box/theme/material/vaadin-combo-box-item-styles.js';
+import { comboBoxLoader, comboBoxOverlay } from '@scoped-vaadin/combo-box/theme/material/vaadin-combo-box-overlay-styles.js';
+import { item } from '@scoped-vaadin/item/theme/material/vaadin-item-styles.js';
+import { inputFieldShared } from '@scoped-vaadin/vaadin-material-styles/mixins/input-field-shared.js';
+import { loader } from '@scoped-vaadin/vaadin-material-styles/mixins/loader.js';
+import { menuOverlay } from '@scoped-vaadin/vaadin-material-styles/mixins/menu-overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const multiSelectComboBoxItem = css`
   @media (any-hover: hover) {
@@ -24,12 +24,12 @@
   }
 `;
 
-registerStyles('vaadin-multi-select-combo-box-item', [item, comboBoxItem, multiSelectComboBoxItem], {
+registerStyles('vaadin24-multi-select-combo-box-item', [item, comboBoxItem, multiSelectComboBoxItem], {
   moduleId: 'material-multi-select-combo-box-item',
 });
 
 registerStyles(
-  'vaadin-multi-select-combo-box-overlay',
+  'vaadin24-multi-select-combo-box-overlay',
   [
     menuOverlay,
     comboBoxOverlay,
@@ -87,6 +87,6 @@
   }
 `;
 
-registerStyles('vaadin-multi-select-combo-box', [inputFieldShared, multiSelectComboBox], {
+registerStyles('vaadin24-multi-select-combo-box', [inputFieldShared, multiSelectComboBox], {
   moduleId: 'material-multi-select-combo-box',
 });
