Index: test/visual/lumo/custom-field.test.js
===================================================================
--- test/visual/lumo/custom-field.test.js
+++ test/visual/lumo/custom-field.test.js
@@ -1,16 +1,16 @@
 import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/combo-box/theme/lumo/vaadin-combo-box.js';
-import '@vaadin/date-picker/theme/lumo/vaadin-date-picker.js';
-import '@vaadin/email-field/theme/lumo/vaadin-email-field.js';
-import '@vaadin/form-layout/theme/lumo/vaadin-form-item.js';
-import '@vaadin/form-layout/theme/lumo/vaadin-form-layout.js';
-import '@vaadin/number-field/theme/lumo/vaadin-number-field.js';
-import '@vaadin/password-field/theme/lumo/vaadin-password-field.js';
-import '@vaadin/select/theme/lumo/vaadin-select.js';
-import '@vaadin/text-area/theme/lumo/vaadin-text-area.js';
-import '@vaadin/text-field/theme/lumo/vaadin-text-field.js';
-import '@vaadin/time-picker/theme/lumo/vaadin-time-picker.js';
+import '@scoped-vaadin/combo-box/theme/lumo/vaadin-combo-box.js';
+import '@scoped-vaadin/date-picker/theme/lumo/vaadin-date-picker.js';
+import '@scoped-vaadin/email-field/theme/lumo/vaadin-email-field.js';
+import '@scoped-vaadin/form-layout/theme/lumo/vaadin-form-item.js';
+import '@scoped-vaadin/form-layout/theme/lumo/vaadin-form-layout.js';
+import '@scoped-vaadin/number-field/theme/lumo/vaadin-number-field.js';
+import '@scoped-vaadin/password-field/theme/lumo/vaadin-password-field.js';
+import '@scoped-vaadin/select/theme/lumo/vaadin-select.js';
+import '@scoped-vaadin/text-area/theme/lumo/vaadin-text-area.js';
+import '@scoped-vaadin/text-field/theme/lumo/vaadin-text-field.js';
+import '@scoped-vaadin/time-picker/theme/lumo/vaadin-time-picker.js';
 import '../../../theme/lumo/vaadin-custom-field.js';
 
 describe('custom-field', () => {
@@ -19,10 +19,10 @@
 
     beforeEach(() => {
       element = fixtureSync(`
-        <vaadin-custom-field>
+        <vaadin23-custom-field>
           <input type="text" />
           <input type="number" />
-        </vaadin-custom-field>
+        </vaadin23-custom-field>
       `);
       inputs = element.querySelectorAll('input');
     });
@@ -111,12 +111,12 @@
       beforeEach(() => {
         wrapper = fixtureSync(`
           <div>
-            <vaadin-custom-field invalid error-message="Invalid">
-              <vaadin-number-field value="1"></vaadin-number-field>
-              <vaadin-password-field value="password"></vaadin-password-field>
-            </vaadin-custom-field>
-            <vaadin-text-field invalid error-message="Invalid"></vaadin-text-field>
-            <vaadin-time-picker value="09:00"></vaadin-time-picker>
+            <vaadin23-custom-field invalid error-message="Invalid">
+              <vaadin23-number-field value="1"></vaadin23-number-field>
+              <vaadin23-password-field value="password"></vaadin23-password-field>
+            </vaadin23-custom-field>
+            <vaadin23-text-field invalid error-message="Invalid"></vaadin23-text-field>
+            <vaadin23-time-picker value="09:00"></vaadin23-time-picker>
           </div>
         `);
       });
@@ -130,19 +130,19 @@
       beforeEach(() => {
         wrapper = fixtureSync(`
           <div>
-            <vaadin-custom-field label="Custom field">
-              <vaadin-select
+            <vaadin23-custom-field label="Custom field">
+              <vaadin23-select
                 value="+358"
                 items='[{ "label": "+358" }, { "label": "+46" }, { "label": "+34" }]'
-              ></vaadin-select>
-              <vaadin-text-field></vaadin-text-field>
-            </vaadin-custom-field>
-            <vaadin-text-field label="Text field" value="Text"></vaadin-text-field>
-            <vaadin-select
+              ></vaadin23-select>
+              <vaadin23-text-field></vaadin23-text-field>
+            </vaadin23-custom-field>
+            <vaadin23-text-field label="Text field" value="Text"></vaadin23-text-field>
+            <vaadin23-select
               label="Plain select"
               value="Option one"
               items='[{ "label": "Option one" }, { "label": "Option two" }]'
-            ></vaadin-select>
+            ></vaadin23-select>
           </div>
         `);
       });
@@ -156,12 +156,12 @@
       beforeEach(() => {
         wrapper = fixtureSync(`
           <div>
-            <vaadin-custom-field label="Custom field" invalid error-message="Invalid">
-              <vaadin-email-field value="user@example.com"></vaadin-email-field>
-              <vaadin-date-picker value="1980-08-14" clear-button-visible></vaadin-date-picker>
-            </vaadin-custom-field>
-            <vaadin-number-field label="Number" value="2" invalid error-message="Invalid"></vaadin-number-field>
-            <vaadin-date-picker label="Date" invalid error-message="Invalid"></vaadin-date-picker>
+            <vaadin23-custom-field label="Custom field" invalid error-message="Invalid">
+              <vaadin23-email-field value="user@example.com"></vaadin23-email-field>
+              <vaadin23-date-picker value="1980-08-14" clear-button-visible></vaadin23-date-picker>
+            </vaadin23-custom-field>
+            <vaadin23-number-field label="Number" value="2" invalid error-message="Invalid"></vaadin23-number-field>
+            <vaadin23-date-picker label="Date" invalid error-message="Invalid"></vaadin23-date-picker>
           </div>
         `);
       });
@@ -175,12 +175,12 @@
       beforeEach(() => {
         wrapper = fixtureSync(`
           <div>
-            <vaadin-custom-field label="Custom field" helper-text="Helper">
-              <vaadin-combo-box value="Combo item" allow-custom-value clear-button-visible></vaadin-combo-box>
-              <vaadin-time-picker value="09:00"></vaadin-time-picker>
-            </vaadin-custom-field>
-            <vaadin-combo-box label="Combo" value="Combo item" allow-custom-value helper-text="Helper"></vaadin-combo-box>
-            <vaadin-password-field value="password"></vaadin-password-field>
+            <vaadin23-custom-field label="Custom field" helper-text="Helper">
+              <vaadin23-combo-box value="Combo item" allow-custom-value clear-button-visible></vaadin23-combo-box>
+              <vaadin23-time-picker value="09:00"></vaadin23-time-picker>
+            </vaadin23-custom-field>
+            <vaadin23-combo-box label="Combo" value="Combo item" allow-custom-value helper-text="Helper"></vaadin23-combo-box>
+            <vaadin23-password-field value="password"></vaadin23-password-field>
           </div>
         `);
       });
@@ -197,12 +197,12 @@
     describe('label + error message', () => {
       beforeEach(() => {
         layout = fixtureSync(`
-          <vaadin-form-layout style="width: 60em">
-            <vaadin-text-field label="Text field" invalid error-message="Error"></vaadin-text-field>
-            <vaadin-custom-field label="Custom field" invalid error-message="Error">
-              <vaadin-text-field></vaadin-text-field>
-            </vaadin-custom-field>
-          </vaadin-form-layout>
+          <vaadin23-form-layout style="width: 60em">
+            <vaadin23-text-field label="Text field" invalid error-message="Error"></vaadin23-text-field>
+            <vaadin23-custom-field label="Custom field" invalid error-message="Error">
+              <vaadin23-text-field></vaadin23-text-field>
+            </vaadin23-custom-field>
+          </vaadin23-form-layout>
         `);
       });
 
@@ -214,20 +214,20 @@
     describe('form-item', () => {
       beforeEach(() => {
         layout = fixtureSync(`
-          <vaadin-form-layout style="width: 60em">
-            <vaadin-form-item>
+          <vaadin23-form-layout style="width: 60em">
+            <vaadin23-form-item>
               <label slot="label">Custom field with text area</label>
-              <vaadin-custom-field>
-                <vaadin-text-area
+              <vaadin23-custom-field>
+                <vaadin23-text-area
                   value="Sed libero enim, sed faucibus turpis in eu? Euismod lacinia at quis risus sed vulputate odio ut enim blandit volutpat maecenas volutpat blandit aliquam etiam erat velit, scelerisque in dictum!"
-                ></vaadin-text-area>
-              </vaadin-custom-field>
-            </vaadin-form-item>
-            <vaadin-form-item>
+                ></vaadin23-text-area>
+              </vaadin23-custom-field>
+            </vaadin23-form-item>
+            <vaadin23-form-item>
               <label slot="label">Text field</label>
-              <vaadin-text-field></vaadin-text-field>
-            </vaadin-form-item>
-          </vaadin-form-layout>
+              <vaadin23-text-field></vaadin23-text-field>
+            </vaadin23-form-item>
+          </vaadin23-form-layout>
         `);
       });
 
