Index: src/vaadin-tooltip-overlay.js
===================================================================
--- src/vaadin-tooltip-overlay.js
+++ src/vaadin-tooltip-overlay.js
@@ -1,14 +1,15 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { Overlay } from '@vaadin/overlay/src/vaadin-overlay.js';
-import { PositionMixin } from '@vaadin/overlay/src/vaadin-overlay-position-mixin.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { Overlay } from '@scoped-vaadin/overlay/src/vaadin-overlay.js';
+import { PositionMixin } from '@scoped-vaadin/overlay/src/vaadin-overlay-position-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-tooltip-overlay',
+  'vaadin23-tooltip-overlay',
   css`
     [part='overlay'] {
       max-width: 40ch;
@@ -40,14 +41,14 @@
 let memoizedTemplate;
 
 /**
- * An element used internally by `<vaadin-tooltip>`. Not intended to be used separately.
+ * An element used internally by `<vaadin23-tooltip>`. Not intended to be used separately.
  *
  * @extends Overlay
  * @private
  */
 class TooltipOverlay extends PositionMixin(Overlay) {
   static get is() {
-    return 'vaadin-tooltip-overlay';
+    return 'vaadin23-tooltip-overlay';
   }
 
   static get template() {
@@ -76,7 +77,7 @@
     super.ready();
 
     // When setting `manual` and `opened` attributes, the overlay is already moved to body
-    // by the time when `ready()` callback of the `vaadin-tooltip` is executed by Polymer,
+    // by the time when `ready()` callback of the `vaadin23-tooltip` is executed by Polymer,
     // so querySelector() would return null. So we use this workaround to set properties.
     this.owner = this.__dataHost;
     this.owner._overlayElement = this;
@@ -143,4 +144,4 @@
   }
 }
 
-customElements.define(TooltipOverlay.is, TooltipOverlay);
+internalCustomElements.define(TooltipOverlay.is, TooltipOverlay);
