Index: tooltip/src/vaadin-tooltip-overlay-styles.js
===================================================================
--- tooltip/src/vaadin-tooltip-overlay-styles.js	null
+++ tooltip/src/vaadin-tooltip-overlay-styles.js	null
@@ -3,7 +3,7 @@
  * Copyright (c) 2022 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 export const tooltipOverlayStyles = css`
   :host {
