Index: src/vaadin-tooltip-mixin.d.ts
===================================================================
--- src/vaadin-tooltip-mixin.d.ts
+++ src/vaadin-tooltip-mixin.d.ts
@@ -4,9 +4,9 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { OverlayClassMixinClass } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import type { PopoverPositionMixinClass } from '@vaadin/popover/src/vaadin-popover-position-mixin.js';
-import type { PopoverTargetMixinClass } from '@vaadin/popover/src/vaadin-popover-target-mixin.js';
+import type { OverlayClassMixinClass } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import type { PopoverPositionMixinClass } from '@scoped-vaadin/popover/src/vaadin-popover-position-mixin.js';
+import type { PopoverTargetMixinClass } from '@scoped-vaadin/popover/src/vaadin-popover-target-mixin.js';
 
 export type { PopoverPosition as TooltipPosition } from '@vaadin/popover/src/vaadin-popover-position-mixin.js';
 
