Index: src/vaadin-tooltip.d.ts
===================================================================
--- src/vaadin-tooltip.d.ts
+++ src/vaadin-tooltip.d.ts
@@ -3,8 +3,8 @@
  * Copyright (c) 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemePropertyMixin } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemePropertyMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 
 export type TooltipPosition =
   | 'bottom-end'
@@ -21,20 +21,20 @@
   | 'top';
 
 /**
- * `<vaadin-tooltip>` is a Web Component for creating tooltips.
+ * `<vaadin23-tooltip>` is a Web Component for creating tooltips.
  *
  * ```html
  * <button id="confirm">Confirm</button>
- * <vaadin-tooltip text="Click to save changes" for="confirm"></vaadin-tooltip>
+ * <vaadin23-tooltip text="Click to save changes" for="confirm"></vaadin23-tooltip>
  * ```
  *
  * ### Styling
  *
- * `<vaadin-tooltip>` uses `<vaadin-tooltip-overlay>` internal
+ * `<vaadin23-tooltip>` uses `<vaadin23-tooltip-overlay>` internal
  * themable component as the actual visible overlay.
  *
- * See [`<vaadin-overlay>`](#/elements/vaadin-overlay) documentation
- * for `<vaadin-tooltip-overlay>` parts.
+ * See [`<vaadin23-overlay>`](#/elements/vaadin-overlay) documentation
+ * for `<vaadin23-tooltip-overlay>` parts.
  *
  * The following state attributes are available for styling:
  *
@@ -42,12 +42,12 @@
  * -----------------|----------------------------------------
  * `position`       | Reflects the `position` property value.
  *
- * Note: the `theme` attribute value set on `<vaadin-tooltip>` is
- * propagated to the internal `<vaadin-tooltip-overlay>` component.
+ * Note: the `theme` attribute value set on `<vaadin23-tooltip>` is
+ * propagated to the internal `<vaadin23-tooltip-overlay>` component.
  *
  * ### Custom CSS Properties
  *
- * The following custom CSS properties are available on the `<vaadin-tooltip>` element:
+ * The following custom CSS properties are available on the `<vaadin23-tooltip>` element:
  *
  * Custom CSS property              | Description
  * ---------------------------------|-------------
@@ -165,7 +165,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-tooltip': Tooltip;
+    'vaadin23-tooltip': Tooltip;
   }
 }
 
