Index: test/visual/common.js
===================================================================
--- test/visual/common.js
+++ test/visual/common.js
@@ -1,8 +1,8 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /* Hide caret */
 registerStyles(
-  'vaadin-combo-box',
+  'vaadin23-combo-box',
   css`
     :host([focus-ring]) ::slotted(input),
     :host([opened]) ::slotted(input) {
