Index: src/vaadin-radio-group.d.ts
===================================================================
--- src/vaadin-radio-group.d.ts
+++ src/vaadin-radio-group.d.ts
@@ -3,21 +3,21 @@
  * Copyright (c) 2017 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { type RadioGroupEventMap, RadioGroupMixin } from './vaadin-radio-group-mixin.js';
 
 export * from './vaadin-radio-group-mixin.js';
 
 /**
- * `<vaadin-radio-group>` is a web component that allows the user to choose one item from a group of choices.
+ * `<vaadin24-radio-group>` is a web component that allows the user to choose one item from a group of choices.
  *
  * ```html
- * <vaadin-radio-group label="Travel class">
- *   <vaadin-radio-button value="economy" label="Economy"></vaadin-radio-button>
- *   <vaadin-radio-button value="business" label="Business"></vaadin-radio-button>
- *   <vaadin-radio-button value="firstClass" label="First Class"></vaadin-radio-button>
- * </vaadin-radio-group>
+ * <vaadin24-radio-group label="Travel class">
+ *   <vaadin24-radio-button value="economy" label="Economy"></vaadin24-radio-button>
+ *   <vaadin24-radio-button value="business" label="Business"></vaadin24-radio-button>
+ *   <vaadin24-radio-button value="firstClass" label="First Class"></vaadin24-radio-button>
+ * </vaadin24-radio-group>
  * ```
  *
  * ### Styling
@@ -67,7 +67,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-radio-group': RadioGroup;
+    'vaadin24-radio-group': RadioGroup;
   }
 }
 
