Index: src/vaadin-radio-button-mixin.js
===================================================================
--- src/vaadin-radio-button-mixin.js
+++ src/vaadin-radio-button-mixin.js
@@ -3,12 +3,12 @@
  * Copyright (c) 2017 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ActiveMixin } from '@vaadin/a11y-base/src/active-mixin.js';
-import { DelegateFocusMixin } from '@vaadin/a11y-base/src/delegate-focus-mixin.js';
-import { CheckedMixin } from '@vaadin/field-base/src/checked-mixin.js';
-import { InputController } from '@vaadin/field-base/src/input-controller.js';
-import { LabelMixin } from '@vaadin/field-base/src/label-mixin.js';
-import { LabelledInputController } from '@vaadin/field-base/src/labelled-input-controller.js';
+import { ActiveMixin } from '@scoped-vaadin/a11y-base/src/active-mixin.js';
+import { DelegateFocusMixin } from '@scoped-vaadin/a11y-base/src/delegate-focus-mixin.js';
+import { CheckedMixin } from '@scoped-vaadin/field-base/src/checked-mixin.js';
+import { InputController } from '@scoped-vaadin/field-base/src/input-controller.js';
+import { LabelMixin } from '@scoped-vaadin/field-base/src/label-mixin.js';
+import { LabelledInputController } from '@scoped-vaadin/field-base/src/labelled-input-controller.js';
 
 /**
  * A mixin providing common radio-button functionality.
