Index: radio-group/src/vaadin-radio-group-mixin.d.ts
===================================================================
--- radio-group/src/vaadin-radio-group-mixin.d.ts	null
+++ radio-group/src/vaadin-radio-group-mixin.d.ts	null
@@ -4,13 +4,13 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { DisabledMixinClass } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin.js';
-import type { KeyboardMixinClass } from '@vaadin/a11y-base/src/keyboard-mixin.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { FieldMixinClass } from '@vaadin/field-base/src/field-mixin.js';
-import type { LabelMixinClass } from '@vaadin/field-base/src/label-mixin.js';
-import type { ValidateMixinClass } from '@vaadin/field-base/src/validate-mixin.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import type { KeyboardMixinClass } from '@scoped-vaadin/a11y-base/src/keyboard-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { FieldMixinClass } from '@scoped-vaadin/field-base/src/field-mixin.js';
+import type { LabelMixinClass } from '@scoped-vaadin/field-base/src/label-mixin.js';
+import type { ValidateMixinClass } from '@scoped-vaadin/field-base/src/validate-mixin.js';
 
 /**
  * Fired when the `invalid` property changes.
