Index: radio-group/src/vaadin-radio-group-mixin.js
===================================================================
--- radio-group/src/vaadin-radio-group-mixin.js	null
+++ radio-group/src/vaadin-radio-group-mixin.js	null
@@ -3,13 +3,13 @@
  * Copyright (c) 2017 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { DisabledMixin } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { KeyboardMixin } from '@vaadin/a11y-base/src/keyboard-mixin.js';
-import { SlotObserver } from '@vaadin/component-base/src/slot-observer.js';
-import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
-import { generateUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
-import { FieldMixin } from '@vaadin/field-base/src/field-mixin.js';
+import { DisabledMixin } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { KeyboardMixin } from '@scoped-vaadin/a11y-base/src/keyboard-mixin.js';
+import { SlotObserver } from '@scoped-vaadin/component-base/src/slot-observer.js';
+import { TooltipController } from '@scoped-vaadin/component-base/src/tooltip-controller.js';
+import { generateUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
+import { FieldMixin } from '@scoped-vaadin/field-base/src/field-mixin.js';
 
 /**
  * A mixin providing common radio-group functionality.
@@ -143,7 +143,7 @@
      * @private
      */
     __filterRadioButtons(nodes) {
-      return nodes.filter((node) => node.nodeType === Node.ELEMENT_NODE && node.localName === 'vaadin-radio-button');
+      return nodes.filter((node) => node.nodeType === Node.ELEMENT_NODE && node.localName === 'vaadin24-radio-button');
     }
 
     /**
@@ -160,7 +160,7 @@
 
       const radioButton = event
         .composedPath()
-        .find((node) => node.nodeType === Node.ELEMENT_NODE && node.localName === 'vaadin-radio-button');
+        .find((node) => node.nodeType === Node.ELEMENT_NODE && node.localName === 'vaadin24-radio-button');
 
       if (['ArrowLeft', 'ArrowUp'].includes(event.key)) {
         event.preventDefault();
