Index: src/vaadin-radio-button.js
===================================================================
--- src/vaadin-radio-button.js
+++ src/vaadin-radio-button.js
@@ -1,30 +1,31 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ActiveMixin } from '@vaadin/component-base/src/active-mixin.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { CheckedMixin } from '@vaadin/field-base/src/checked-mixin.js';
-import { DelegateFocusMixin } from '@vaadin/field-base/src/delegate-focus-mixin.js';
-import { InputController } from '@vaadin/field-base/src/input-controller.js';
-import { LabelMixin } from '@vaadin/field-base/src/label-mixin.js';
-import { LabelledInputController } from '@vaadin/field-base/src/labelled-input-controller.js';
-import { SlotTargetController } from '@vaadin/field-base/src/slot-target-controller.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ActiveMixin } from '@scoped-vaadin/component-base/src/active-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { CheckedMixin } from '@scoped-vaadin/field-base/src/checked-mixin.js';
+import { DelegateFocusMixin } from '@scoped-vaadin/field-base/src/delegate-focus-mixin.js';
+import { InputController } from '@scoped-vaadin/field-base/src/input-controller.js';
+import { LabelMixin } from '@scoped-vaadin/field-base/src/label-mixin.js';
+import { LabelledInputController } from '@scoped-vaadin/field-base/src/labelled-input-controller.js';
+import { SlotTargetController } from '@scoped-vaadin/field-base/src/slot-target-controller.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * `<vaadin-radio-button>` is a web component representing a choice in a radio group.
+ * `<vaadin23-radio-button>` is a web component representing a choice in a radio group.
  * Only one radio button in the group can be selected at the same time.
  *
  * ```html
- * <vaadin-radio-group label="Travel class">
- *   <vaadin-radio-button value="economy" label="Economy"></vaadin-radio-button>
- *   <vaadin-radio-button value="business" label="Business"></vaadin-radio-button>
- *   <vaadin-radio-button value="firstClass" label="First Class"></vaadin-radio-button>
- * </vaadin-radio-group>
+ * <vaadin23-radio-group label="Travel class">
+ *   <vaadin23-radio-button value="economy" label="Economy"></vaadin23-radio-button>
+ *   <vaadin23-radio-button value="business" label="Business"></vaadin23-radio-button>
+ *   <vaadin23-radio-button value="firstClass" label="First Class"></vaadin23-radio-button>
+ * </vaadin23-radio-group>
  * ```
  *
  * ### Styling
@@ -62,7 +63,7 @@
   CheckedMixin(DelegateFocusMixin(ActiveMixin(ElementMixin(ThemableMixin(ControllerMixin(PolymerElement)))))),
 ) {
   static get is() {
-    return 'vaadin-radio-button';
+    return 'vaadin23-radio-button';
   }
 
   static get template() {
@@ -184,12 +185,12 @@
   /** @private */
   __warnDeprecated() {
     console.warn(
-      `WARNING: Since Vaadin 22, placing the label as a direct child of a <vaadin-radio-button> is deprecated.
+      `WARNING: Since Vaadin 22, placing the label as a direct child of a <vaadin23-radio-button> is deprecated.
   Please use <label slot="label"> wrapper or the label property instead.`,
     );
   }
 }
 
-customElements.define(RadioButton.is, RadioButton);
+internalCustomElements.define(RadioButton.is, RadioButton);
 
 export { RadioButton };
