Index: src/vaadin-radio-group.d.ts
===================================================================
--- src/vaadin-radio-group.d.ts
+++ src/vaadin-radio-group.d.ts
@@ -3,12 +3,12 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { DisabledMixin } from '@vaadin/component-base/src/disabled-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { FocusMixin } from '@vaadin/component-base/src/focus-mixin.js';
-import { KeyboardMixin } from '@vaadin/component-base/src/keyboard-mixin.js';
-import { FieldMixin } from '@vaadin/field-base/src/field-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DisabledMixin } from '@scoped-vaadin/component-base/src/disabled-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/component-base/src/focus-mixin.js';
+import { KeyboardMixin } from '@scoped-vaadin/component-base/src/keyboard-mixin.js';
+import { FieldMixin } from '@scoped-vaadin/field-base/src/field-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
  * Fired when the `invalid` property changes.
@@ -36,14 +36,14 @@
 export interface RadioGroupEventMap extends HTMLElementEventMap, RadioGroupCustomEventMap {}
 
 /**
- * `<vaadin-radio-group>` is a web component that allows the user to choose one item from a group of choices.
+ * `<vaadin23-radio-group>` is a web component that allows the user to choose one item from a group of choices.
  *
  * ```html
- * <vaadin-radio-group label="Travel class">
- *   <vaadin-radio-button value="economy" label="Economy"></vaadin-radio-button>
- *   <vaadin-radio-button value="business" label="Business"></vaadin-radio-button>
- *   <vaadin-radio-button value="firstClass" label="First Class"></vaadin-radio-button>
- * </vaadin-radio-group>
+ * <vaadin23-radio-group label="Travel class">
+ *   <vaadin23-radio-button value="economy" label="Economy"></vaadin23-radio-button>
+ *   <vaadin23-radio-button value="business" label="Business"></vaadin23-radio-button>
+ *   <vaadin23-radio-button value="firstClass" label="First Class"></vaadin23-radio-button>
+ * </vaadin23-radio-group>
  * ```
  *
  * ### Styling
@@ -108,7 +108,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-radio-group': RadioGroup;
+    'vaadin23-radio-group': RadioGroup;
   }
 }
 
