Index: test/dom/__snapshots__/confirm-dialog.test.snap.js
===================================================================
--- test/dom/__snapshots__/confirm-dialog.test.snap.js
+++ test/dom/__snapshots__/confirm-dialog.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-confirm-dialog overlay"] = 
-`<vaadin-confirm-dialog-overlay
+`<vaadin24-confirm-dialog-overlay
   aria-describedby="confirm-dialog-message-0"
   aria-label="Unsaved changes"
   focus-trap=""
@@ -20,7 +20,7 @@
   <div id="confirm-dialog-message-0">
     Do you want to save or discard the changes?
   </div>
-  <vaadin-button
+  <vaadin24-button
     hidden=""
     role="button"
     slot="cancel-button"
@@ -28,8 +28,8 @@
     theme="tertiary"
   >
     Cancel
-  </vaadin-button>
-  <vaadin-button
+  </vaadin24-button>
+  <vaadin24-button
     focused=""
     role="button"
     slot="confirm-button"
@@ -37,8 +37,8 @@
     theme="primary"
   >
     Confirm
-  </vaadin-button>
-  <vaadin-button
+  </vaadin24-button>
+  <vaadin24-button
     hidden=""
     role="button"
     slot="reject-button"
@@ -46,13 +46,13 @@
     theme="error tertiary"
   >
     Reject
-  </vaadin-button>
-</vaadin-confirm-dialog-overlay>
+  </vaadin24-button>
+</vaadin24-confirm-dialog-overlay>
 `;
 /* end snapshot vaadin-confirm-dialog overlay */
 
 snapshots["vaadin-confirm-dialog overlay theme"] = 
-`<vaadin-confirm-dialog-overlay
+`<vaadin24-confirm-dialog-overlay
   aria-describedby="confirm-dialog-message-1"
   aria-label="Unsaved changes"
   focus-trap=""
@@ -71,7 +71,7 @@
   <div id="confirm-dialog-message-1">
     Do you want to save or discard the changes?
   </div>
-  <vaadin-button
+  <vaadin24-button
     hidden=""
     role="button"
     slot="cancel-button"
@@ -79,8 +79,8 @@
     theme="tertiary"
   >
     Cancel
-  </vaadin-button>
-  <vaadin-button
+  </vaadin24-button>
+  <vaadin24-button
     focused=""
     role="button"
     slot="confirm-button"
@@ -88,8 +88,8 @@
     theme="primary"
   >
     Confirm
-  </vaadin-button>
-  <vaadin-button
+  </vaadin24-button>
+  <vaadin24-button
     hidden=""
     role="button"
     slot="reject-button"
@@ -97,13 +97,13 @@
     theme="error tertiary"
   >
     Reject
-  </vaadin-button>
-</vaadin-confirm-dialog-overlay>
+  </vaadin24-button>
+</vaadin24-confirm-dialog-overlay>
 `;
 /* end snapshot vaadin-confirm-dialog overlay theme */
 
 snapshots["vaadin-confirm-dialog overlay class"] = 
-`<vaadin-confirm-dialog-overlay
+`<vaadin24-confirm-dialog-overlay
   aria-describedby="confirm-dialog-message-2"
   aria-label="Unsaved changes"
   class="confirm-dialog-overlay custom"
@@ -122,7 +122,7 @@
   <div id="confirm-dialog-message-2">
     Do you want to save or discard the changes?
   </div>
-  <vaadin-button
+  <vaadin24-button
     hidden=""
     role="button"
     slot="cancel-button"
@@ -130,8 +130,8 @@
     theme="tertiary"
   >
     Cancel
-  </vaadin-button>
-  <vaadin-button
+  </vaadin24-button>
+  <vaadin24-button
     focused=""
     role="button"
     slot="confirm-button"
@@ -139,8 +139,8 @@
     theme="primary"
   >
     Confirm
-  </vaadin-button>
-  <vaadin-button
+  </vaadin24-button>
+  <vaadin24-button
     hidden=""
     role="button"
     slot="reject-button"
@@ -148,8 +148,8 @@
     theme="error tertiary"
   >
     Reject
-  </vaadin-button>
-</vaadin-confirm-dialog-overlay>
+  </vaadin24-button>
+</vaadin24-confirm-dialog-overlay>
 `;
 /* end snapshot vaadin-confirm-dialog overlay class */
 
