Index: test/typings/message-input.types.ts
===================================================================
--- test/typings/message-input.types.ts
+++ test/typings/message-input.types.ts
@@ -1,10 +1,10 @@
 import '../../vaadin-message-input.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
 import type { MessageInputSubmitEvent } from '../../vaadin-message-input.js';
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const input = document.createElement('vaadin-message-input');
+const input = document.createElement('vaadin24-message-input');
 
 assertType<ControllerMixinClass>(input);
 
