Index: theme/material/vaadin-tabs-styles.js
===================================================================
--- theme/material/vaadin-tabs-styles.js
+++ theme/material/vaadin-tabs-styles.js
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/color.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-tabs',
+  'vaadin24-tabs',
   css`
     :host {
       -webkit-tap-highlight-color: transparent;
@@ -66,7 +66,7 @@
     }
 
     /* Fixed width tabs */
-    :host([theme~='fixed']) [part='tabs'] ::slotted(vaadin-tab) {
+    :host([theme~='fixed']) [part='tabs'] ::slotted(vaadin24-tab) {
       flex-basis: 0.0001px;
     }
 
