Index: theme/material/vaadin-tab-styles.js
===================================================================
--- theme/material/vaadin-tab-styles.js
+++ theme/material/vaadin-tab-styles.js
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-tab',
+  'vaadin23-tab',
   css`
     :host {
       display: flex;
@@ -128,7 +128,7 @@
     }
 
     /* Small space between icon and label */
-    ::slotted(vaadin-icon:not(:only-child)),
+    ::slotted(vaadin23-icon:not(:only-child)),
     ::slotted(iron-icon:not(:only-child)) {
       margin-bottom: 8px;
     }
