Index: src/vaadin-number-field.d.ts
===================================================================
--- src/vaadin-number-field.d.ts
+++ src/vaadin-number-field.d.ts
@@ -3,8 +3,8 @@
  * Copyright (c) 2021 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { NumberFieldMixin } from './vaadin-number-field-mixin.js';
 
 /**
@@ -49,18 +49,18 @@
 }
 
 /**
- * `<vaadin-number-field>` is an input field web component that only accepts numeric input.
+ * `<vaadin24-number-field>` is an input field web component that only accepts numeric input.
  *
  * ```html
- * <vaadin-number-field label="Balance"></vaadin-number-field>
+ * <vaadin24-number-field label="Balance"></vaadin24-number-field>
  * ```
  *
  * ### Styling
  *
- * `<vaadin-number-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin-text-field>`.
- * See [`<vaadin-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
+ * `<vaadin24-number-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin24-text-field>`.
+ * See [`<vaadin24-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
  *
- * In addition to `<vaadin-text-field>` parts, the following parts are available for theming:
+ * In addition to `<vaadin24-text-field>` parts, the following parts are available for theming:
  *
  * Part name         | Description
  * ------------------|-------------------------
@@ -111,7 +111,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-number-field': NumberField;
+    'vaadin24-number-field': NumberField;
   }
 }
 
