Index: src/vaadin-lit-number-field.js
===================================================================
--- src/vaadin-lit-number-field.js
+++ src/vaadin-lit-number-field.js
@@ -3,19 +3,19 @@
  * Copyright (c) 2021 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/src/vaadin-lit-input-container.js';
+import '@scoped-vaadin/input-container/src/vaadin-lit-input-container.js';
 import { html, LitElement } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { inputFieldShared } from '@vaadin/field-base/src/styles/input-field-shared-styles.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { inputFieldShared } from '@scoped-vaadin/field-base/src/styles/input-field-shared-styles.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { numberFieldStyles } from './vaadin-number-field-core-styles.js';
 import { NumberFieldMixin } from './vaadin-number-field-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-number-field>` web component.
+ * LitElement based version of `<vaadin24-number-field>` web component.
  *
  * ## Disclaimer
  *
@@ -25,7 +25,7 @@
  */
 class NumberField extends NumberFieldMixin(ThemableMixin(ElementMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-number-field';
+    return 'vaadin24-number-field';
   }
 
   static get styles() {
@@ -41,7 +41,7 @@
           <span part="required-indicator" aria-hidden="true" @click="${this.focus}"></span>
         </div>
 
-        <vaadin-input-container
+        <vaadin24-input-container
           part="input-field"
           .readonly="${this.readonly}"
           .disabled="${this.disabled}"
@@ -70,7 +70,7 @@
             aria-hidden="true"
             slot="suffix"
           ></div>
-        </vaadin-input-container>
+        </vaadin24-input-container>
 
         <div part="helper-text">
           <slot name="helper"></slot>
