Index: tabsheet/theme/material/vaadin-tabsheet-styles.js
===================================================================
--- tabsheet/theme/material/vaadin-tabsheet-styles.js	null
+++ tabsheet/theme/material/vaadin-tabsheet-styles.js	null
@@ -1,7 +1,7 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { loader } from '@vaadin/vaadin-material-styles/mixins/loader.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { loader } from '@scoped-vaadin/vaadin-material-styles/mixins/loader.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const tabsheet = css`
   :host {
@@ -43,4 +43,4 @@
   }
 `;
 
-registerStyles('vaadin-tabsheet', [tabsheet, loader], { moduleId: 'material-tabsheet' });
+registerStyles('vaadin24-tabsheet', [tabsheet, loader], { moduleId: 'material-tabsheet' });
