Index: test/keyboard-desktop.test.js
===================================================================
--- test/keyboard-desktop.test.js
+++ test/keyboard-desktop.test.js
@@ -15,15 +15,15 @@
   beforeEach(async () => {
     const wrapper = fixtureSync(`
       <div>
-        <vaadin-app-layout style="--vaadin-app-layout-transition: none;">
-          <vaadin-drawer-toggle slot="navbar"></vaadin-drawer-toggle>
+        <vaadin24-app-layout style="--vaadin-app-layout-transition: none;">
+          <vaadin24-drawer-toggle slot="navbar"></vaadin24-drawer-toggle>
           <section slot="drawer">
             <input placeholder="Drawer input" />
           </section>
           <main>
             <input placeholder="Content input" />
           </main>
-        </vaadin-app-layout>
+        </vaadin24-app-layout>
         <input placeholder="Outer input" />
       </div>
     `);
