Index: src/vaadin-multi-select-combo-box-overlay.js
===================================================================
--- src/vaadin-multi-select-combo-box-overlay.js
+++ src/vaadin-multi-select-combo-box-overlay.js
@@ -3,13 +3,13 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ComboBoxOverlayMixin } from '@vaadin/combo-box/src/vaadin-combo-box-overlay-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { Overlay } from '@vaadin/overlay/src/vaadin-overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ComboBoxOverlayMixin } from '@scoped-vaadin/combo-box/src/vaadin-combo-box-overlay-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { Overlay } from '@scoped-vaadin/overlay/src/vaadin-overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-multi-select-combo-box-overlay',
+  'vaadin24-multi-select-combo-box-overlay',
   css`
     #overlay {
       width: var(
@@ -30,7 +30,7 @@
 let memoizedTemplate;
 
 /**
- * An element used internally by `<vaadin-multi-select-combo-box>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-multi-select-combo-box>`. Not intended to be used separately.
  *
  * @customElement
  * @extends ComboBoxOverlay
@@ -38,7 +38,7 @@
  */
 class MultiSelectComboBoxOverlay extends ComboBoxOverlayMixin(Overlay) {
   static get is() {
-    return 'vaadin-multi-select-combo-box-overlay';
+    return 'vaadin24-multi-select-combo-box-overlay';
   }
 
   static get template() {
