Index: src/vaadin-lit-multi-select-combo-box-internal.js
===================================================================
--- src/vaadin-lit-multi-select-combo-box-internal.js
+++ src/vaadin-lit-multi-select-combo-box-internal.js
@@ -8,13 +8,13 @@
 import './vaadin-lit-multi-select-combo-box-scroller.js';
 import { css, html, LitElement } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { MultiSelectComboBoxInternalMixin } from './vaadin-multi-select-combo-box-internal-mixin.js';
 
 /**
- * An element used internally by `<vaadin-multi-select-combo-box>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-multi-select-combo-box>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -24,7 +24,7 @@
  */
 class MultiSelectComboBoxInternal extends MultiSelectComboBoxInternalMixin(ThemableMixin(PolylitMixin(LitElement))) {
   static get is() {
-    return 'vaadin-multi-select-combo-box-internal';
+    return 'vaadin24-multi-select-combo-box-internal';
   }
 
   static get styles() {
@@ -40,7 +40,7 @@
     return html`
       <slot></slot>
 
-      <vaadin-multi-select-combo-box-overlay
+      <vaadin24-multi-select-combo-box-overlay
         id="overlay"
         .opened="${this._overlayOpened}"
         ?loading="${this.loading}"
@@ -48,7 +48,7 @@
         .positionTarget="${this._target}"
         no-vertical-overlap
         .restoreFocusNode="${this.inputElement}"
-      ></vaadin-multi-select-combo-box-overlay>
+      ></vaadin24-multi-select-combo-box-overlay>
     `;
   }
 }
