Index: src/vaadin-multi-select-combo-box.js
===================================================================
--- src/vaadin-multi-select-combo-box.js
+++ src/vaadin-multi-select-combo-box.js
@@ -7,23 +7,23 @@
 import './vaadin-multi-select-combo-box-container.js';
 import './vaadin-multi-select-combo-box-internal.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { inputFieldShared } from '@vaadin/field-base/src/styles/input-field-shared-styles.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { inputFieldShared } from '@scoped-vaadin/field-base/src/styles/input-field-shared-styles.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { MultiSelectComboBoxMixin } from './vaadin-multi-select-combo-box-mixin.js';
 import { multiSelectComboBox } from './vaadin-multi-select-combo-box-styles.js';
 
-registerStyles('vaadin-multi-select-combo-box', [inputFieldShared, multiSelectComboBox], {
+registerStyles('vaadin24-multi-select-combo-box', [inputFieldShared, multiSelectComboBox], {
   moduleId: 'vaadin-multi-select-combo-box-styles',
 });
 
 /**
- * `<vaadin-multi-select-combo-box>` is a web component that wraps `<vaadin-combo-box>` and extends
+ * `<vaadin24-multi-select-combo-box>` is a web component that wraps `<vaadin24-combo-box>` and extends
  * its functionality to allow selecting multiple items, in addition to basic features.
  *
  * ```html
- * <vaadin-multi-select-combo-box id="comboBox"></vaadin-multi-select-combo-box>
+ * <vaadin24-multi-select-combo-box id="comboBox"></vaadin24-multi-select-combo-box>
  * ```
  *
  * ```js
@@ -75,14 +75,14 @@
  *
  * ### Internal components
  *
- * In addition to `<vaadin-multi-select-combo-box>` itself, the following internal
+ * In addition to `<vaadin24-multi-select-combo-box>` itself, the following internal
  * components are themable:
  *
- * - `<vaadin-multi-select-combo-box-overlay>` - has the same API as `<vaadin-overlay>`.
- * - `<vaadin-multi-select-combo-box-item>` - has the same API as `<vaadin-item>`.
- * - `<vaadin-multi-select-combo-box-container>` - has the same API as `<vaadin-input-container>`.
+ * - `<vaadin24-multi-select-combo-box-overlay>` - has the same API as `<vaadin24-overlay>`.
+ * - `<vaadin24-multi-select-combo-box-item>` - has the same API as `<vaadin24-item>`.
+ * - `<vaadin24-multi-select-combo-box-container>` - has the same API as `<vaadin24-input-container>`.
  *
- * Note: the `theme` attribute value set on `<vaadin-multi-select-combo-box>` is
+ * Note: the `theme` attribute value set on `<vaadin24-multi-select-combo-box>` is
  * propagated to these components.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
@@ -103,18 +103,18 @@
  */
 class MultiSelectComboBox extends MultiSelectComboBoxMixin(ThemableMixin(ElementMixin(PolymerElement))) {
   static get is() {
-    return 'vaadin-multi-select-combo-box';
+    return 'vaadin24-multi-select-combo-box';
   }
 
   static get template() {
     return html`
-      <div class="vaadin-multi-select-combo-box-container">
+      <div class="vaadin24-multi-select-combo-box-container">
         <div part="label">
           <slot name="label"></slot>
           <span part="required-indicator" aria-hidden="true" on-click="focus"></span>
         </div>
 
-        <vaadin-multi-select-combo-box-internal
+        <vaadin24-multi-select-combo-box-internal
           id="comboBox"
           items="[[items]]"
           item-id-path="[[itemIdPath]]"
@@ -144,7 +144,7 @@
           on-custom-value-set="_onCustomValueSet"
           on-filtered-items-changed="_onFilteredItemsChanged"
         >
-          <vaadin-multi-select-combo-box-container
+          <vaadin24-multi-select-combo-box-container
             part="input-field"
             auto-expand-vertically="[[autoExpandVertically]]"
             readonly="[[readonly]]"
@@ -165,8 +165,8 @@
               aria-hidden="true"
             ></div>
             <div id="toggleButton" class="toggle-button" part="toggle-button" slot="suffix" aria-hidden="true"></div>
-          </vaadin-multi-select-combo-box-container>
-        </vaadin-multi-select-combo-box-internal>
+          </vaadin24-multi-select-combo-box-container>
+        </vaadin24-multi-select-combo-box-internal>
 
         <div part="helper-text">
           <slot name="helper"></slot>
