Index: src/vaadin-multi-select-combo-box-scroller.js
===================================================================
--- src/vaadin-multi-select-combo-box-scroller.js
+++ src/vaadin-multi-select-combo-box-scroller.js
@@ -4,12 +4,12 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ComboBoxPlaceholder } from '@vaadin/combo-box/src/vaadin-combo-box-placeholder.js';
-import { ComboBoxScrollerMixin } from '@vaadin/combo-box/src/vaadin-combo-box-scroller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import { ComboBoxPlaceholder } from '@scoped-vaadin/combo-box/src/vaadin-combo-box-placeholder.js';
+import { ComboBoxScrollerMixin } from '@scoped-vaadin/combo-box/src/vaadin-combo-box-scroller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
 
 /**
- * An element used internally by `<vaadin-multi-select-combo-box>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-multi-select-combo-box>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -18,7 +18,7 @@
  */
 export class MultiSelectComboBoxScroller extends ComboBoxScrollerMixin(PolymerElement) {
   static get is() {
-    return 'vaadin-multi-select-combo-box-scroller';
+    return 'vaadin24-multi-select-combo-box-scroller';
   }
 
   static get template() {
