Index: src/vaadin-multi-select-combo-box-internal.js
===================================================================
--- src/vaadin-multi-select-combo-box-internal.js
+++ src/vaadin-multi-select-combo-box-internal.js
@@ -7,12 +7,12 @@
 import './vaadin-multi-select-combo-box-overlay.js';
 import './vaadin-multi-select-combo-box-scroller.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { MultiSelectComboBoxInternalMixin } from './vaadin-multi-select-combo-box-internal-mixin.js';
 
 /**
- * An element used internally by `<vaadin-multi-select-combo-box>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-multi-select-combo-box>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -22,7 +22,7 @@
  */
 class MultiSelectComboBoxInternal extends MultiSelectComboBoxInternalMixin(ThemableMixin(PolymerElement)) {
   static get is() {
-    return 'vaadin-multi-select-combo-box-internal';
+    return 'vaadin24-multi-select-combo-box-internal';
   }
 
   static get template() {
@@ -35,7 +35,7 @@
 
       <slot></slot>
 
-      <vaadin-multi-select-combo-box-overlay
+      <vaadin24-multi-select-combo-box-overlay
         id="overlay"
         opened="[[_overlayOpened]]"
         loading$="[[loading]]"
@@ -43,7 +43,7 @@
         position-target="[[_target]]"
         no-vertical-overlap
         restore-focus-node="[[inputElement]]"
-      ></vaadin-multi-select-combo-box-overlay>
+      ></vaadin24-multi-select-combo-box-overlay>
     `;
   }
 }
