Index: src/vaadin-multi-select-combo-box-internal-mixin.js
===================================================================
--- src/vaadin-multi-select-combo-box-internal-mixin.js
+++ src/vaadin-multi-select-combo-box-internal-mixin.js
@@ -3,9 +3,9 @@
  * Copyright (c) 2021 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ComboBoxDataProviderMixin } from '@vaadin/combo-box/src/vaadin-combo-box-data-provider-mixin.js';
-import { ComboBoxMixin } from '@vaadin/combo-box/src/vaadin-combo-box-mixin.js';
-import { ComboBoxPlaceholder } from '@vaadin/combo-box/src/vaadin-combo-box-placeholder.js';
+import { ComboBoxDataProviderMixin } from '@scoped-vaadin/combo-box/src/vaadin-combo-box-data-provider-mixin.js';
+import { ComboBoxMixin } from '@scoped-vaadin/combo-box/src/vaadin-combo-box-mixin.js';
+import { ComboBoxPlaceholder } from '@scoped-vaadin/combo-box/src/vaadin-combo-box-placeholder.js';
 
 /**
  * @polymerMixin
@@ -118,7 +118,7 @@
      * @return {string}
      */
     get _tagNamePrefix() {
-      return 'vaadin-multi-select-combo-box';
+      return 'vaadin24-multi-select-combo-box';
     }
 
     constructor() {
