Index: src/vaadin-lit-multi-select-combo-box-container.js
===================================================================
--- src/vaadin-lit-multi-select-combo-box-container.js
+++ src/vaadin-lit-multi-select-combo-box-container.js
@@ -4,11 +4,11 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { css, html } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { InputContainer } from '@vaadin/input-container/src/vaadin-lit-input-container.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { InputContainer } from '@scoped-vaadin/input-container/src/vaadin-lit-input-container.js';
 
 /**
- * An element used internally by `<vaadin-multi-select-combo-box>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-multi-select-combo-box>`. Not intended to be used separately.
  *
  * @customElement
  * @extends InputContainer
@@ -16,7 +16,7 @@
  */
 class MultiSelectComboBoxContainer extends InputContainer {
   static get is() {
-    return 'vaadin-multi-select-combo-box-container';
+    return 'vaadin24-multi-select-combo-box-container';
   }
 
   static get styles() {
