Index: src/vaadin-email-field.js
===================================================================
--- src/vaadin-email-field.js
+++ src/vaadin-email-field.js
@@ -3,24 +3,24 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { TextField } from '@vaadin/text-field/src/vaadin-text-field.js';
-import { registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { TextField } from '@scoped-vaadin/text-field/src/vaadin-text-field.js';
+import { registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { emailFieldStyles } from './vaadin-email-field-styles.js';
 
-registerStyles('vaadin-email-field', emailFieldStyles, { moduleId: 'vaadin-email-field-styles' });
+registerStyles('vaadin24-email-field', emailFieldStyles, { moduleId: 'vaadin-email-field-styles' });
 
 /**
- * `<vaadin-email-field>` is a Web Component for email field control in forms.
+ * `<vaadin24-email-field>` is a Web Component for email field control in forms.
  *
  * ```html
- * <vaadin-email-field label="Email"></vaadin-email-field>
+ * <vaadin24-email-field label="Email"></vaadin24-email-field>
  * ```
  *
  * ### Styling
  *
- * `<vaadin-email-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin-text-field>`.
- * See [`<vaadin-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
+ * `<vaadin24-email-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin24-text-field>`.
+ * See [`<vaadin24-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
@@ -35,7 +35,7 @@
  */
 export class EmailField extends TextField {
   static get is() {
-    return 'vaadin-email-field';
+    return 'vaadin24-email-field';
   }
 
   constructor() {
