Index: src/vaadin-email-field.js
===================================================================
--- src/vaadin-email-field.js
+++ src/vaadin-email-field.js
@@ -1,14 +1,15 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { TextField } from '@vaadin/text-field/src/vaadin-text-field.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { TextField } from '@scoped-vaadin/text-field/src/vaadin-text-field.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 // See https://github.com/vaadin/vaadin-text-field/issues/466
 registerStyles(
-  'vaadin-email-field',
+  'vaadin23-email-field',
   css`
     :host([dir='rtl']) [part='input-field'] {
       direction: ltr;
@@ -23,16 +24,16 @@
 );
 
 /**
- * `<vaadin-email-field>` is a Web Component for email field control in forms.
+ * `<vaadin23-email-field>` is a Web Component for email field control in forms.
  *
  * ```html
- * <vaadin-email-field label="Email"></vaadin-email-field>
+ * <vaadin23-email-field label="Email"></vaadin23-email-field>
  * ```
  *
  * ### Styling
  *
- * `<vaadin-email-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin-text-field>`.
- * See [`<vaadin-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
+ * `<vaadin23-email-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin23-text-field>`.
+ * See [`<vaadin23-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/custom-theme/styling-components) documentation.
  *
@@ -46,7 +47,7 @@
  */
 export class EmailField extends TextField {
   static get is() {
-    return 'vaadin-email-field';
+    return 'vaadin23-email-field';
   }
 
   constructor() {
@@ -65,4 +66,4 @@
   }
 }
 
-customElements.define('vaadin-email-field', EmailField);
+internalCustomElements.define('vaadin23-email-field', EmailField);
