Index: test/integer-field.common.js
===================================================================
--- test/integer-field.common.js
+++ test/integer-field.common.js
@@ -6,7 +6,7 @@
   let integerField, input;
 
   beforeEach(async () => {
-    integerField = fixtureSync('<vaadin-integer-field></vaadin-integer-field>');
+    integerField = fixtureSync('<vaadin24-integer-field></vaadin24-integer-field>');
     await nextRender();
     input = integerField.inputElement;
   });
@@ -292,9 +292,9 @@
     expect(() =>
       fixtureSync(`
         <div>
-          <vaadin-integer-field label="integer fld"></vaadin-integer-field>
-          <vaadin-text-field label="text fld"></vaadin-text-field>
-          <vaadin-number-field label="number fld"></vaadin-number-field>
+          <vaadin24-integer-field label="integer fld"></vaadin24-integer-field>
+          <vaadin24-text-field label="text fld"></vaadin24-text-field>
+          <vaadin24-number-field label="number fld"></vaadin24-number-field>
         </div>
       `),
     ).to.not.throw(Error);
