Index: test/value-commit.common.js
===================================================================
--- test/value-commit.common.js
+++ test/value-commit.common.js
@@ -38,7 +38,7 @@
   }
 
   beforeEach(async () => {
-    integerField = fixtureSync(`<vaadin-integer-field></vaadin-integer-field>`);
+    integerField = fixtureSync(`<vaadin24-integer-field></vaadin24-integer-field>`);
     await nextRender();
     validateSpy = sinon.spy(integerField, 'validate').named('validateSpy');
 
