Index: src/vaadin-lit-custom-field.js
===================================================================
--- src/vaadin-lit-custom-field.js
+++ src/vaadin-lit-custom-field.js
@@ -4,15 +4,15 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { CustomFieldMixin } from './vaadin-custom-field-mixin.js';
 import { customFieldStyles } from './vaadin-custom-field-styles.js';
 
 /**
- * LitElement based version of `<vaadin-custom-field>` web component.
+ * LitElement based version of `<vaadin24-custom-field>` web component.
  *
  * ## Disclaimer
  *
@@ -22,7 +22,7 @@
  */
 class CustomField extends CustomFieldMixin(ThemableMixin(ElementMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-custom-field';
+    return 'vaadin24-custom-field';
   }
 
   static get styles() {
