Index: custom-field/src/vaadin-custom-field.d.ts
===================================================================
--- custom-field/src/vaadin-custom-field.d.ts	null
+++ custom-field/src/vaadin-custom-field.d.ts	null
@@ -3,8 +3,8 @@
  * Copyright (c) 2019 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { CustomFieldMixin } from './vaadin-custom-field-mixin.js';
 
 export { CustomFieldFormatValueFn, CustomFieldParseValueFn } from './vaadin-custom-field-mixin.js';
@@ -53,13 +53,13 @@
 }
 
 /**
- * `<vaadin-custom-field>` is a web component for wrapping multiple components as a single field.
+ * `<vaadin24-custom-field>` is a web component for wrapping multiple components as a single field.
  *
  * ```
- * <vaadin-custom-field label="Appointment time">
- *   <vaadin-date-picker></vaadin-date-picker>
- *   <vaadin-time-picker></vaadin-time-picker>
- * </vaadin-custom-field>
+ * <vaadin24-custom-field label="Appointment time">
+ *   <vaadin24-date-picker></vaadin24-date-picker>
+ *   <vaadin24-time-picker></vaadin24-time-picker>
+ * </vaadin24-custom-field>
  * ```
  *
  * ### Styling
@@ -86,7 +86,7 @@
  * `has-error-message` | Set when the element has an error message | :host
  *
  * You may also manually set `disabled` or `readonly` attribute on this component to make the label
- * part look visually the same as on a `<vaadin-text-field>` when it is disabled or readonly.
+ * part look visually the same as on a `<vaadin24-text-field>` when it is disabled or readonly.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
@@ -112,7 +112,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-custom-field': CustomField;
+    'vaadin24-custom-field': CustomField;
   }
 }
 
