Index: src/vaadin-custom-field-mixin.js
===================================================================
--- src/vaadin-custom-field-mixin.js
+++ src/vaadin-custom-field-mixin.js
@@ -3,11 +3,11 @@
  * Copyright (c) 2019 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { KeyboardMixin } from '@vaadin/a11y-base/src/keyboard-mixin.js';
-import { getFlattenedElements } from '@vaadin/component-base/src/dom-utils.js';
-import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
-import { FieldMixin } from '@vaadin/field-base/src/field-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { KeyboardMixin } from '@scoped-vaadin/a11y-base/src/keyboard-mixin.js';
+import { getFlattenedElements } from '@scoped-vaadin/component-base/src/dom-utils.js';
+import { TooltipController } from '@scoped-vaadin/component-base/src/tooltip-controller.js';
+import { FieldMixin } from '@scoped-vaadin/field-base/src/field-mixin.js';
 
 /**
  * Default implementation of the parse function that creates individual field
