Index: test/aria.common.js
===================================================================
--- test/aria.common.js
+++ test/aria.common.js
@@ -6,7 +6,7 @@
 
   beforeEach(async () => {
     dateTimePicker = fixtureSync(
-      `<vaadin-date-time-picker helper-text="Helper text" label="Date and time"></vaadin-date-time-picker>`,
+      `<vaadin24-date-time-picker helper-text="Helper text" label="Date and time"></vaadin24-date-time-picker>`,
     );
     await nextRender();
     label = dateTimePicker.querySelector(':scope > [slot=label]');
