Index: test/timezone.test.js
===================================================================
--- test/timezone.test.js
+++ test/timezone.test.js
@@ -8,7 +8,7 @@
   let timePicker;
 
   beforeEach(async () => {
-    dateTimePicker = fixtureSync('<vaadin-date-time-picker></vaadin-date-time-picker>');
+    dateTimePicker = fixtureSync('<vaadin24-date-time-picker></vaadin24-date-time-picker>');
     await nextRender();
     datePicker = dateTimePicker.querySelector('[slot="date-picker"]');
     timePicker = dateTimePicker.querySelector('[slot="time-picker"]');
