Index: test/validation.test.js
===================================================================
--- test/validation.test.js
+++ test/validation.test.js
@@ -1,25 +1,26 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { aTimeout, fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
 import '../vaadin-date-time-picker.js';
-import { waitForOverlayRender } from '@vaadin/date-picker/test/helpers.js';
+import { waitForOverlayRender } from '@scoped-vaadin/date-picker/test/helpers.js';
 
-class DateTimePicker2020Element extends customElements.get('vaadin-date-time-picker') {
+class DateTimePicker2020Element extends internalCustomElements.get('vaadin24-date-time-picker') {
   checkValidity() {
     return this.value === '2020-02-02T20:20';
   }
 }
 
-customElements.define('vaadin-date-time-picker-2020', DateTimePicker2020Element);
+internalCustomElements.define('vaadin24-date-time-picker-2020', DateTimePicker2020Element);
 
 const fixtures = {
-  default: '<vaadin-date-time-picker></vaadin-date-time-picker>',
+  default: '<vaadin24-date-time-picker></vaadin24-date-time-picker>',
   slotted: `
-    <vaadin-date-time-picker>
-      <vaadin-date-picker slot="date-picker"></vaadin-date-picker>
-      <vaadin-time-picker slot="time-picker"></vaadin-time-picker>
-    </vaadin-date-time-picker>
+    <vaadin24-date-time-picker>
+      <vaadin24-date-picker slot="date-picker"></vaadin24-date-picker>
+      <vaadin24-time-picker slot="time-picker"></vaadin24-time-picker>
+    </vaadin24-date-time-picker>
   `,
 };
 
@@ -230,7 +231,7 @@
   let validateSpy, dateTimePicker;
 
   beforeEach(() => {
-    dateTimePicker = document.createElement('vaadin-date-time-picker');
+    dateTimePicker = document.createElement('vaadin24-date-time-picker');
     validateSpy = sinon.spy(dateTimePicker, 'validate');
   });
 
@@ -282,7 +283,7 @@
   let dateTimePicker;
 
   beforeEach(() => {
-    dateTimePicker = fixtureSync('<vaadin-date-time-picker-2020></vaadin-date-time-picker-2020>');
+    dateTimePicker = fixtureSync('<vaadin24-date-time-picker-2020></vaadin24-date-time-picker-2020>');
   });
 
   it('should validate correctly with custom validator', () => {
