Index: test/value-commit.test.js
===================================================================
--- test/value-commit.test.js
+++ test/value-commit.test.js
@@ -48,7 +48,7 @@
     dateTimePicker = fixtureSync(`
       <div>
         <input/>
-        <vaadin-date-time-picker></vaadin-date-time-picker>
+        <vaadin24-date-time-picker></vaadin24-date-time-picker>
         <input/>
       </div>
     `).children[1];
