Index: test/grid-editor.test.js
===================================================================
--- test/grid-editor.test.js
+++ test/grid-editor.test.js
@@ -1,7 +1,7 @@
 import { expect } from '@esm-bundle/chai';
 import { enter, fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import '@vaadin/grid-pro';
+import '@scoped-vaadin/grid-pro';
 import '../template-renderer.js';
 import './fixtures/mock-grid-pro-host.js';
 import { GridTemplatizer } from '../src/template-renderer-grid-templatizer.js';
@@ -9,11 +9,11 @@
 describe('grid editor template', () => {
   function fixtureGrid() {
     return fixtureSync(`
-      <vaadin-grid-pro>
-        <vaadin-grid-pro-edit-column>
+      <vaadin24-grid-pro>
+        <vaadin24-grid-pro-edit-column>
           <template class="editor"></template>
-        </vaadin-grid-pro-edit-column>
-      </vaadin-grid-pro>
+        </vaadin24-grid-pro-edit-column>
+      </vaadin24-grid-pro>
     `);
   }
 
@@ -33,7 +33,7 @@
     column.editModeRenderer = () => {};
 
     expect(() => window.Vaadin.templateRendererCallback(column)).to.throw(
-      /^Cannot use both a template and a renderer for <vaadin-grid-pro-edit-column \/>\.$/u,
+      /^Cannot use both a template and a renderer for <vaadin24-grid-pro-edit-column \/>\.$/u,
     );
   });
 });
