Index: test/grid-footer.test.js
===================================================================
--- test/grid-footer.test.js
+++ test/grid-footer.test.js
@@ -1,7 +1,7 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import '@vaadin/grid';
+import '@scoped-vaadin/grid';
 import '../template-renderer.js';
 import './fixtures/mock-grid-host.js';
 import { Templatizer } from '../src/template-renderer-templatizer.js';
@@ -9,11 +9,11 @@
 describe('grid footer template', () => {
   function fixtureGrid() {
     return fixtureSync(`
-      <vaadin-grid>
-        <vaadin-grid-column>
+      <vaadin23-grid>
+        <vaadin23-grid-column>
           <template class="footer">footer</template>
-        </vaadin-grid-column>
-      </vaadin-grid>
+        </vaadin23-grid-column>
+      </vaadin23-grid>
     `);
   }
 
@@ -42,7 +42,7 @@
     column.footerRenderer = () => {};
 
     expect(() => window.Vaadin.templateRendererCallback(column)).to.throw(
-      /^Cannot use both a template and a renderer for <vaadin-grid-column \/>\.$/,
+      /^Cannot use both a template and a renderer for <vaadin23-grid-column \/>\.$/,
     );
   });
 });
