Index: test/style-modules-lit.test.js
===================================================================
--- test/style-modules-lit.test.js
+++ test/style-modules-lit.test.js
@@ -2,12 +2,12 @@
 // Import the adapter
 import '../style-modules.js';
 // Use LitElement based custom elements in the tests
-import '@vaadin/vaadin-themable-mixin/test/lit-setup.js';
+import '@scoped-vaadin/vaadin-themable-mixin/test/lit-setup.js';
 // Use <dom-module theme-for="..."> styling (via the adapter) in the tests
 import './setup.js';
 // Import tests from themable-mixin package to be run with the adapter in place
-import '@vaadin/vaadin-themable-mixin/test/register-styles.test.js';
-import '@vaadin/vaadin-themable-mixin/test/themable-mixin.test.js';
+import '@scoped-vaadin/vaadin-themable-mixin/test/register-styles.test.js';
+import '@scoped-vaadin/vaadin-themable-mixin/test/themable-mixin.test.js';
 import { LitElement } from 'lit';
 
 describe('style-modules-lit', () => {
