Index: test/style-modules.test.js
===================================================================
--- test/style-modules.test.js
+++ test/style-modules.test.js
@@ -3,8 +3,8 @@
 import '../style-modules.js';
 import './setup.js';
 // Import tests from themable-mixin package to be run with the adapter in place
-import '@vaadin/vaadin-themable-mixin/test/register-styles.test';
-import '@vaadin/vaadin-themable-mixin/test/themable-mixin.test';
+import '@scoped-vaadin/vaadin-themable-mixin/test/register-styles.test';
+import '@scoped-vaadin/vaadin-themable-mixin/test/themable-mixin.test';
 
 describe('style-modules', () => {
   it('should have created styles as dom-modules', () => {
