Index: src/layouts/responsive-steps-layout.js
===================================================================
--- src/layouts/responsive-steps-layout.js
+++ src/layouts/responsive-steps-layout.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2021 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { isElementHidden } from '@vaadin/a11y-base/src/focus-utils.js';
+import { isElementHidden } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 import { AbstractLayout } from './abstract-layout.js';
 
 function isValidCSSLength(value) {
@@ -184,7 +184,7 @@
         // Move the column counter
         col = (col + colspan) % this.__columnCount;
 
-        if (child.localName === 'vaadin-form-item') {
+        if (child.localName === 'vaadin24-form-item') {
           if (this.__labelsOnTop) {
             if (child.getAttribute('label-position') !== 'top') {
               child.__useLayoutLabelPosition = true;
