Index: test/dom/context-menu.test.js
===================================================================
--- test/dom/context-menu.test.js
+++ test/dom/context-menu.test.js
@@ -59,9 +59,9 @@
 
   beforeEach(async () => {
     menu = fixtureSync(`
-      <vaadin-context-menu>
+      <vaadin24-context-menu>
         <div style="padding: 10px">Target</div>
-      </vaadin-context-menu>
+      </vaadin24-context-menu>
     `);
     await nextRender();
     overlay = menu._overlayElement;
@@ -83,7 +83,7 @@
     await openSubMenus(menu);
     await nextRender();
 
-    const subMenu = overlay.querySelector('vaadin-context-menu');
+    const subMenu = overlay.querySelector('vaadin24-context-menu');
     await expect(subMenu._overlayElement).dom.to.equalSnapshot(SNAPSHOT_CONFIG);
   });
 
@@ -105,7 +105,7 @@
     await openSubMenus(menu);
     await nextRender();
 
-    const subMenu = overlay.querySelector('vaadin-context-menu');
+    const subMenu = overlay.querySelector('vaadin24-context-menu');
     await expect(subMenu._overlayElement).dom.to.equalSnapshot(SNAPSHOT_CONFIG);
   });
 });
