Index: theme/lumo/vaadin-lit-message-input.d.ts
===================================================================
--- theme/lumo/vaadin-lit-message-input.d.ts
+++ theme/lumo/vaadin-lit-message-input.d.ts
@@ -1,4 +1,4 @@
-import '@vaadin/button/theme/lumo/vaadin-lit-button.js';
-import '@vaadin/text-area/theme/lumo/vaadin-lit-text-area.js';
+import '@scoped-vaadin/button/theme/lumo/vaadin-lit-button.js';
+import '@scoped-vaadin/text-area/theme/lumo/vaadin-lit-text-area.js';
 import './vaadin-message-input-styles.js';
 import '../../src/vaadin-lit-message-input.js';
