Index: src/vaadin-lit-input-container.js
===================================================================
--- src/vaadin-lit-input-container.js
+++ src/vaadin-lit-input-container.js
@@ -4,15 +4,15 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { inputContainerStyles } from './vaadin-input-container-core-styles.js';
 import { InputContainerMixin } from './vaadin-input-container-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-input-container>` web component.
+ * LitElement based version of `<vaadin24-input-container>` web component.
  *
  * ## Disclaimer
  *
@@ -27,7 +27,7 @@
  */
 export class InputContainer extends InputContainerMixin(ThemableMixin(DirMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-input-container';
+    return 'vaadin24-input-container';
   }
 
   static get styles() {
