Index: src/vaadin-input-container.js
===================================================================
--- src/vaadin-input-container.js
+++ src/vaadin-input-container.js
@@ -1,15 +1,16 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 export class InputContainer extends ThemableMixin(DirMixin(PolymerElement)) {
   static get is() {
-    return 'vaadin-input-container';
+    return 'vaadin23-input-container';
   }
 
   static get template() {
@@ -119,4 +120,4 @@
   }
 }
 
-customElements.define(InputContainer.is, InputContainer);
+internalCustomElements.define(InputContainer.is, InputContainer);
