Index: test/dom/__snapshots__/tooltip.test.snap.js
===================================================================
--- test/dom/__snapshots__/tooltip.test.snap.js
+++ test/dom/__snapshots__/tooltip.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-tooltip default"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -10,12 +10,12 @@
   position="bottom"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip default */
 
 snapshots["vaadin-tooltip top-start"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -23,12 +23,12 @@
   position="top-start"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip top-start */
 
 snapshots["vaadin-tooltip top"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -36,12 +36,12 @@
   position="top"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip top */
 
 snapshots["vaadin-tooltip top-end"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -49,12 +49,12 @@
   position="top-end"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip top-end */
 
 snapshots["vaadin-tooltip bottom-start"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -62,12 +62,12 @@
   position="bottom-start"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip bottom-start */
 
 snapshots["vaadin-tooltip bottom"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -75,12 +75,12 @@
   position="bottom"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip bottom */
 
 snapshots["vaadin-tooltip bottom-end"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -88,12 +88,12 @@
   position="bottom-end"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip bottom-end */
 
 snapshots["vaadin-tooltip start-top"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -101,12 +101,12 @@
   position="start-top"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip start-top */
 
 snapshots["vaadin-tooltip start"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -114,12 +114,12 @@
   position="start"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip start */
 
 snapshots["vaadin-tooltip start-bottom"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -127,12 +127,12 @@
   position="start-bottom"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip start-bottom */
 
 snapshots["vaadin-tooltip end-top"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -140,12 +140,12 @@
   position="end-top"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip end-top */
 
 snapshots["vaadin-tooltip end"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -153,12 +153,12 @@
   position="end"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip end */
 
 snapshots["vaadin-tooltip end-bottom"] = 
-`<vaadin-tooltip-overlay
+`<vaadin23-tooltip-overlay
   hidden=""
   id="vaadin-tooltip-0"
   modeless=""
@@ -166,7 +166,7 @@
   position="end-bottom"
   role="tooltip"
 >
-</vaadin-tooltip-overlay>
+</vaadin23-tooltip-overlay>
 `;
 /* end snapshot vaadin-tooltip end-bottom */
 
