Index: test/visual/common.js
===================================================================
--- test/visual/common.js
+++ test/visual/common.js
@@ -1,4 +1,4 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /* Hide caret */
 registerStyles(
@@ -11,7 +11,7 @@
 );
 
 registerStyles(
-  'vaadin-text-area',
+  'vaadin24-text-area',
   css`
     :host([focus-ring]) ::slotted(textarea) {
       caret-color: transparent;
@@ -20,7 +20,7 @@
 );
 
 registerStyles(
-  'vaadin-user-tags-overlay',
+  'vaadin24-user-tags-overlay',
   css`
     :host([opening]),
     :host([closing]),
