Index: theme/lumo/vaadin-list-box-styles.js
===================================================================
--- theme/lumo/vaadin-list-box-styles.js
+++ theme/lumo/vaadin-list-box-styles.js
@@ -1,7 +1,7 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const listBox = css`
   :host {
@@ -19,6 +19,6 @@
   }
 `;
 
-registerStyles('vaadin-list-box', listBox, { moduleId: 'lumo-list-box' });
+registerStyles('vaadin24-list-box', listBox, { moduleId: 'lumo-list-box' });
 
 export { listBox };
