Index: test/dom/__snapshots__/details.test.snap.js
===================================================================
--- test/dom/__snapshots__/details.test.snap.js
+++ test/dom/__snapshots__/details.test.snap.js
@@ -2,8 +2,8 @@
 export const snapshots = {};
 
 snapshots["vaadin-details host default"] = 
-`<vaadin-details>
-  <vaadin-details-summary
+`<vaadin24-details>
+  <vaadin24-details-summary
     aria-controls="content-vaadin-details-0"
     aria-expanded="false"
     id="summary-vaadin-details-1"
@@ -12,20 +12,20 @@
     tabindex="0"
   >
     Summary
-  </vaadin-details-summary>
+  </vaadin24-details-summary>
   <div
     aria-hidden="true"
     id="content-vaadin-details-0"
   >
     Content
   </div>
-</vaadin-details>
+</vaadin24-details>
 `;
 /* end snapshot vaadin-details host default */
 
 snapshots["vaadin-details host opened"] = 
-`<vaadin-details opened="">
-  <vaadin-details-summary
+`<vaadin24-details opened="">
+  <vaadin24-details-summary
     aria-controls="content-vaadin-details-0"
     aria-expanded="true"
     id="summary-vaadin-details-1"
@@ -35,20 +35,20 @@
     tabindex="0"
   >
     Summary
-  </vaadin-details-summary>
+  </vaadin24-details-summary>
   <div
     aria-hidden="false"
     id="content-vaadin-details-0"
   >
     Content
   </div>
-</vaadin-details>
+</vaadin24-details>
 `;
 /* end snapshot vaadin-details host opened */
 
 snapshots["vaadin-details host disabled"] = 
-`<vaadin-details disabled="">
-  <vaadin-details-summary
+`<vaadin24-details disabled="">
+  <vaadin24-details-summary
     aria-controls="content-vaadin-details-0"
     aria-disabled="true"
     aria-expanded="false"
@@ -59,20 +59,20 @@
     tabindex="-1"
   >
     Summary
-  </vaadin-details-summary>
+  </vaadin24-details-summary>
   <div
     aria-hidden="true"
     id="content-vaadin-details-0"
   >
     Content
   </div>
-</vaadin-details>
+</vaadin24-details>
 `;
 /* end snapshot vaadin-details host disabled */
 
 snapshots["vaadin-details host theme"] = 
-`<vaadin-details theme="filled">
-  <vaadin-details-summary
+`<vaadin24-details theme="filled">
+  <vaadin24-details-summary
     aria-controls="content-vaadin-details-0"
     aria-expanded="false"
     id="summary-vaadin-details-1"
@@ -82,14 +82,14 @@
     theme="filled"
   >
     Summary
-  </vaadin-details-summary>
+  </vaadin24-details-summary>
   <div
     aria-hidden="true"
     id="content-vaadin-details-0"
   >
     Content
   </div>
-</vaadin-details>
+</vaadin24-details>
 `;
 /* end snapshot vaadin-details host theme */
 
