Index: test/dom/__snapshots__/details.test.snap.js
===================================================================
--- test/dom/__snapshots__/details.test.snap.js
+++ test/dom/__snapshots__/details.test.snap.js
@@ -2,12 +2,12 @@
 export const snapshots = {};
 
 snapshots["vaadin-crud host default"] = 
-`<vaadin-details tabindex="0">
+`<vaadin23-details tabindex="0">
   <div slot="summary">
     Summary
   </div>
   <input>
-</vaadin-details>
+</vaadin23-details>
 `;
 /* end snapshot vaadin-crud host default */
 
