Index: test/dom/__snapshots__/custom-field.test.snap.js
===================================================================
--- test/dom/__snapshots__/custom-field.test.snap.js
+++ test/dom/__snapshots__/custom-field.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-custom-field host default"] = 
-`<vaadin-custom-field role="group">
+`<vaadin24-custom-field role="group">
   <label
     id="label-vaadin-custom-field-0"
     slot="label"
@@ -14,12 +14,12 @@
     slot="error-message"
   >
   </div>
-</vaadin-custom-field>
+</vaadin24-custom-field>
 `;
 /* end snapshot vaadin-custom-field host default */
 
 snapshots["vaadin-custom-field host label"] = 
-`<vaadin-custom-field
+`<vaadin24-custom-field
   aria-labelledby="label-vaadin-custom-field-0"
   has-label=""
   role="group"
@@ -36,12 +36,12 @@
     slot="error-message"
   >
   </div>
-</vaadin-custom-field>
+</vaadin24-custom-field>
 `;
 /* end snapshot vaadin-custom-field host label */
 
 snapshots["vaadin-custom-field host required"] = 
-`<vaadin-custom-field
+`<vaadin24-custom-field
   aria-required="true"
   required=""
   role="group"
@@ -57,12 +57,12 @@
     slot="error-message"
   >
   </div>
-</vaadin-custom-field>
+</vaadin24-custom-field>
 `;
 /* end snapshot vaadin-custom-field host required */
 
 snapshots["vaadin-custom-field host helper"] = 
-`<vaadin-custom-field
+`<vaadin24-custom-field
   aria-describedby="helper-vaadin-custom-field-1"
   has-helper=""
   role="group"
@@ -84,12 +84,12 @@
   >
     Helper
   </div>
-</vaadin-custom-field>
+</vaadin24-custom-field>
 `;
 /* end snapshot vaadin-custom-field host helper */
 
 snapshots["vaadin-custom-field host error"] = 
-`<vaadin-custom-field
+`<vaadin24-custom-field
   aria-describedby="error-message-vaadin-custom-field-2"
   has-error-message=""
   invalid=""
@@ -107,7 +107,7 @@
   >
     Error
   </div>
-</vaadin-custom-field>
+</vaadin24-custom-field>
 `;
 /* end snapshot vaadin-custom-field host error */
 
