Index: test/typings/password-field.types.ts
===================================================================
--- test/typings/password-field.types.ts
+++ test/typings/password-field.types.ts
@@ -9,7 +9,7 @@
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const field = document.createElement('vaadin-password-field');
+const field = document.createElement('vaadin23-password-field');
 
 field.addEventListener('change', (event) => {
   assertType<PasswordFieldChangeEvent>(event);
