Index: details/src/collapsible-mixin.js
===================================================================
--- details/src/collapsible-mixin.js
+++ details/src/collapsible-mixin.js
@@ -7,7 +7,7 @@
 
 /**
  * A mixin providing common functionality for making content collapsible,
- * used by `<vaadin-details>` and `<vaadin-accordion-panel>` elements.
+ * used by `<vaadin24-details>` and `<vaadin24-accordion-panel>` elements.
  *
  * @polymerMixin
  */
@@ -61,8 +61,8 @@
 
       this.addController(this._contentController);
 
-      // Only handle click and not keydown, because `vaadin-details-summary` uses `ButtonMixin`
-      // that already covers this logic, and `vaadin-accordion-heading` uses native `<button>`.
+      // Only handle click and not keydown, because `vaadin24-details-summary` uses `ButtonMixin`
+      // that already covers this logic, and `vaadin24-accordion-heading` uses native `<button>`.
       this.addEventListener('click', ({ target }) => {
         if (this.disabled) {
           return;
