Index: details/src/summary-controller.js
===================================================================
--- details/src/summary-controller.js
+++ details/src/summary-controller.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2019 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { SlotChildObserveController } from '@vaadin/component-base/src/slot-child-observe-controller.js';
+import { SlotChildObserveController } from '@scoped-vaadin/component-base/src/slot-child-observe-controller.js';
 
 /**
  * A controller to manage the summary element.
