Index: src/vaadin-details.d.ts
===================================================================
--- src/vaadin-details.d.ts
+++ src/vaadin-details.d.ts
@@ -3,9 +3,9 @@
  * Copyright (c) 2019 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { DetailsBaseMixin } from './vaadin-details-base-mixin.js';
 
 /**
@@ -20,16 +20,16 @@
 export type DetailsEventMap = DetailsCustomEventMap & HTMLElementEventMap;
 
 /**
- * `<vaadin-details>` is a Web Component which the creates an
+ * `<vaadin24-details>` is a Web Component which the creates an
  * expandable panel similar to `<details>` HTML element.
  *
  * ```
- * <vaadin-details>
- *   <vaadin-details-summary slot="summary">Expandable Details</vaadin-details-summary>
+ * <vaadin24-details>
+ *   <vaadin24-details-summary slot="summary">Expandable Details</vaadin24-details-summary>
  *   <div>
  *     Toggle using mouse, Enter and Space keys.
  *   </div>
- * </vaadin-details>
+ * </vaadin24-details>
  * ```
  *
  * ### Styling
@@ -69,7 +69,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-details': Details;
+    'vaadin24-details': Details;
   }
 }
 
