Index: src/vaadin-lit-details.js
===================================================================
--- src/vaadin-lit-details.js
+++ src/vaadin-lit-details.js
@@ -5,14 +5,14 @@
  */
 import './vaadin-lit-details-summary.js';
 import { css, html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { DetailsBaseMixin } from './vaadin-details-base-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-details>` web component.
+ * LitElement based version of `<vaadin24-details>` web component.
  *
  * ## Disclaimer
  *
@@ -22,7 +22,7 @@
  */
 class Details extends DetailsBaseMixin(ElementMixin(ThemableMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-details';
+    return 'vaadin24-details';
   }
 
   static get styles() {
