Index: test/text-area.test.js
===================================================================
--- test/text-area.test.js
+++ test/text-area.test.js
@@ -7,7 +7,7 @@
   let textArea;
 
   beforeEach(async () => {
-    textArea = fixtureSync('<vaadin-text-area></vaadin-text-area>');
+    textArea = fixtureSync('<vaadin24-text-area></vaadin24-text-area>');
     await nextRender();
   });
 
@@ -280,10 +280,10 @@
                 box-sizing: border-box;
               }
             </style>
-            <vaadin-text-area></vaadin-text-area>
+            <vaadin24-text-area></vaadin24-text-area>
           </div>
         `);
-        textArea = fixture.querySelector('vaadin-text-area');
+        textArea = fixture.querySelector('vaadin24-text-area');
         await nextUpdate(textArea);
         native = textArea.querySelector('textarea');
 
@@ -316,7 +316,7 @@
         textArea.minRows = 0;
         await nextUpdate(textArea);
 
-        expect(console.warn).to.be.calledWith('<vaadin-text-area> minRows must be at least 1.');
+        expect(console.warn).to.be.calledWith('<vaadin24-text-area> minRows must be at least 1.');
       });
 
       it('should not log warning when setting minRows to two rows or more', async () => {
