Index: test/validation.test.js
===================================================================
--- test/validation.test.js
+++ test/validation.test.js
@@ -8,7 +8,7 @@
 
   describe('initial', () => {
     beforeEach(() => {
-      textArea = document.createElement('vaadin-text-area');
+      textArea = document.createElement('vaadin24-text-area');
       validateSpy = sinon.spy(textArea, 'validate');
     });
 
@@ -60,7 +60,7 @@
     let validatedSpy;
 
     beforeEach(async () => {
-      textArea = fixtureSync('<vaadin-text-area></vaadin-text-area>');
+      textArea = fixtureSync('<vaadin24-text-area></vaadin24-text-area>');
       await nextRender();
       validatedSpy = sinon.spy();
       textArea.addEventListener('validated', validatedSpy);
@@ -87,7 +87,7 @@
 
   describe('minlength', () => {
     beforeEach(async () => {
-      textArea = fixtureSync('<vaadin-text-area></vaadin-text-area>');
+      textArea = fixtureSync('<vaadin24-text-area></vaadin24-text-area>');
       await nextRender();
     });
 
@@ -109,7 +109,7 @@
 
   describe('maxlength', () => {
     beforeEach(async () => {
-      textArea = fixtureSync('<vaadin-text-area></vaadin-text-area>');
+      textArea = fixtureSync('<vaadin24-text-area></vaadin24-text-area>');
       await nextRender();
     });
 
