Index: src/vaadin-lit-grid-tree-column.js
===================================================================
--- src/vaadin-lit-grid-tree-column.js
+++ src/vaadin-lit-grid-tree-column.js
@@ -4,12 +4,12 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import './vaadin-lit-grid-tree-toggle.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
 import { GridTreeColumnMixin } from './vaadin-grid-tree-column-mixin.js';
 import { GridColumn } from './vaadin-lit-grid-column.js';
 
 /**
- * LitElement based version of `<vaadin-grid-tree-column>` web component.
+ * LitElement based version of `<vaadin24-grid-tree-column>` web component.
  *
  * ## Disclaimer
  *
@@ -19,7 +19,7 @@
  */
 class GridTreeColumn extends GridTreeColumnMixin(GridColumn) {
   static get is() {
-    return 'vaadin-grid-tree-column';
+    return 'vaadin24-grid-tree-column';
   }
 }
 
