Index: grid/src/vaadin-lit-grid-column.js
===================================================================
--- grid/src/vaadin-lit-grid-column.js	null
+++ grid/src/vaadin-lit-grid-column.js	null
@@ -4,12 +4,12 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
 import { GridColumnMixin } from './vaadin-grid-column-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-grid-column>` web component.
+ * LitElement based version of `<vaadin24-grid-column>` web component.
  *
  * ## Disclaimer
  *
@@ -19,7 +19,7 @@
  */
 class GridColumn extends GridColumnMixin(PolylitMixin(LitElement)) {
   static get is() {
-    return 'vaadin-grid-column';
+    return 'vaadin24-grid-column';
   }
 }
 
