Index: src/vaadin-grid-active-item-mixin.js
===================================================================
--- src/vaadin-grid-active-item-mixin.js
+++ src/vaadin-grid-active-item-mixin.js
@@ -77,7 +77,7 @@
      */
     _onClick(e) {
       if (e.defaultPrevented) {
-        // Something has handled this click already, e. g., <vaadin-grid-sorter>
+        // Something has handled this click already, e. g., <vaadin24-grid-sorter>
         return;
       }
 
