Index: src/vaadin-lit-grid-selection-column.js
===================================================================
--- src/vaadin-lit-grid-selection-column.js
+++ src/vaadin-lit-grid-selection-column.js
@@ -3,13 +3,13 @@
  * Copyright (c) 2016 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/checkbox/src/vaadin-lit-checkbox.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import '@scoped-vaadin/checkbox/src/vaadin-lit-checkbox.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
 import { GridSelectionColumnMixin } from './vaadin-grid-selection-column-mixin.js';
 import { GridColumn } from './vaadin-lit-grid-column.js';
 
 /**
- * LitElement based version of `<vaadin-grid-selection-column>` web component.
+ * LitElement based version of `<vaadin24-grid-selection-column>` web component.
  *
  * ## Disclaimer
  *
@@ -19,7 +19,7 @@
  */
 class GridSelectionColumn extends GridSelectionColumnMixin(GridColumn) {
   static get is() {
-    return 'vaadin-grid-selection-column';
+    return 'vaadin24-grid-selection-column';
   }
 }
 
