Index: src/vaadin-lit-grid-sorter.js
===================================================================
--- src/vaadin-lit-grid-sorter.js
+++ src/vaadin-lit-grid-sorter.js
@@ -4,14 +4,14 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin';
 import { GridSorterMixin } from './vaadin-grid-sorter-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-grid-sorter>` web component.
+ * LitElement based version of `<vaadin24-grid-sorter>` web component.
  *
  * ## Disclaimer
  *
@@ -21,7 +21,7 @@
  */
 class GridSorter extends GridSorterMixin(ThemableMixin(DirMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-grid-sorter';
+    return 'vaadin24-grid-sorter';
   }
 
   /** @protected */
