Index: grid/src/vaadin-grid-sort-mixin.d.ts
===================================================================
--- grid/src/vaadin-grid-sort-mixin.d.ts
+++ grid/src/vaadin-grid-sort-mixin.d.ts
@@ -16,7 +16,7 @@
   static setDefaultMultiSortPriority(priority: 'append' | 'prepend'): void;
 
   /**
-   * When `true`, all `<vaadin-grid-sorter>` are applied for sorting.
+   * When `true`, all `<vaadin24-grid-sorter>` are applied for sorting.
    * @attr {boolean} multi-sort
    */
   multiSort: boolean;
