Index: grid/src/vaadin-grid-styling-mixin.js
===================================================================
--- grid/src/vaadin-grid-styling-mixin.js
+++ grid/src/vaadin-grid-styling-mixin.js
@@ -19,7 +19,7 @@
          * characters.
          *
          * Receives two arguments:
-         * - `column` The `<vaadin-grid-column>` element (`undefined` for details-cell).
+         * - `column` The `<vaadin24-grid-column>` element (`undefined` for details-cell).
          * - `model` The object with the properties related with
          *   the rendered item, contains:
          *   - `model.index` The index of the item.
@@ -44,7 +44,7 @@
          * separated by whitespace characters.
          *
          * Receives two arguments:
-         * - `column` The `<vaadin-grid-column>` element (`undefined` for details-cell).
+         * - `column` The `<vaadin24-grid-column>` element (`undefined` for details-cell).
          * - `model` The object with the properties related with
          *   the rendered item, contains:
          *   - `model.index` The index of the item.
