Index: src/vaadin-grid-tree-column.d.ts
===================================================================
--- src/vaadin-grid-tree-column.d.ts
+++ src/vaadin-grid-tree-column.d.ts
@@ -8,15 +8,15 @@
 import type { GridTreeColumnMixinClass } from './vaadin-grid-tree-column-mixin.js';
 
 /**
- * `<vaadin-grid-tree-column>` is a helper element for the `<vaadin-grid>`
+ * `<vaadin24-grid-tree-column>` is a helper element for the `<vaadin24-grid>`
  * that provides default renderer and functionality for toggling tree/hierarchical items.
  *
  * #### Example:
  * ```html
- * <vaadin-grid>
- *  <vaadin-grid-tree-column path="name.first"></vaadin-grid-tree-column>
+ * <vaadin24-grid>
+ *  <vaadin24-grid-tree-column path="name.first"></vaadin24-grid-tree-column>
  *
- *  <vaadin-grid-column>
+ *  <vaadin24-grid-column>
  *    ...
  * ```
  */
@@ -29,7 +29,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-grid-tree-column': GridTreeColumn<GridDefaultItem>;
+    'vaadin24-grid-tree-column': GridTreeColumn<GridDefaultItem>;
   }
 }
 
