Index: src/vaadin-grid-active-item-mixin.js
===================================================================
--- src/vaadin-grid-active-item-mixin.js
+++ src/vaadin-grid-active-item-mixin.js
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 
-import { isElementFocusable } from '@vaadin/a11y-base/src/focus-utils.js';
+import { isElementFocusable } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 
 /**
  * @param {!Element} target
@@ -68,7 +68,7 @@
     _shouldPreventCellActivationOnClick(e) {
       const { cell } = this._getGridEventLocation(e);
       return (
-        // Something has handled this click already, e. g., <vaadin-grid-sorter>
+        // Something has handled this click already, e. g., <vaadin24-grid-sorter>
         e.defaultPrevented ||
         // No clicked cell available
         !cell ||
