Index: src/vaadin-grid-filter-column-mixin.js
===================================================================
--- src/vaadin-grid-filter-column-mixin.js
+++ src/vaadin-grid-filter-column-mixin.js
@@ -49,8 +49,8 @@
       let textField = filter ? filter.firstElementChild : undefined;
 
       if (!filter) {
-        filter = document.createElement('vaadin-grid-filter');
-        textField = document.createElement('vaadin-text-field');
+        filter = document.createElement('vaadin24-grid-filter');
+        textField = document.createElement('vaadin24-text-field');
         textField.setAttribute('theme', 'small');
         textField.setAttribute('style', 'max-width: 100%;');
         textField.setAttribute('focus-target', '');
