Index: src/vaadin-grid-active-item-mixin.js
===================================================================
--- src/vaadin-grid-active-item-mixin.js
+++ src/vaadin-grid-active-item-mixin.js
@@ -76,7 +76,7 @@
     _shouldPreventCellActivationOnClick(e) {
       const { cell } = this._getGridEventLocation(e);
       return (
-        // Something has handled this click already, e. g., <vaadin-grid-sorter>
+        // Something has handled this click already, e. g., <vaadin24-grid-sorter>
         e.defaultPrevented ||
         // No clicked cell available
         !cell ||
