Index: src/vaadin-grid-a11y-mixin.js
===================================================================
--- src/vaadin-grid-a11y-mixin.js
+++ src/vaadin-grid-a11y-mixin.js
@@ -148,7 +148,7 @@
 
     /** @protected */
     _a11yUpdateSorters() {
-      Array.from(this.querySelectorAll('vaadin-grid-sorter')).forEach((sorter) => {
+      Array.from(this.querySelectorAll('vaadin24-grid-sorter')).forEach((sorter) => {
         let cellContent = sorter.parentNode;
         while (cellContent && cellContent.localName !== 'vaadin-grid-cell-content') {
           cellContent = cellContent.parentNode;
