Index: src/vaadin-grid-selection-column.d.ts
===================================================================
--- src/vaadin-grid-selection-column.d.ts
+++ src/vaadin-grid-selection-column.d.ts
@@ -14,19 +14,19 @@
 export * from './vaadin-grid-selection-column-mixin.js';
 
 /**
- * `<vaadin-grid-selection-column>` is a helper element for the `<vaadin-grid>`
+ * `<vaadin24-grid-selection-column>` is a helper element for the `<vaadin24-grid>`
  * that provides default renderers and functionality for item selection.
  *
  * #### Example:
  * ```html
- * <vaadin-grid>
- *  <vaadin-grid-selection-column frozen auto-select></vaadin-grid-selection-column>
+ * <vaadin24-grid>
+ *  <vaadin24-grid-selection-column frozen auto-select></vaadin24-grid-selection-column>
  *
- *  <vaadin-grid-column>
+ *  <vaadin24-grid-column>
  *    ...
  * ```
  *
- * By default the selection column displays `<vaadin-checkbox>` elements in the
+ * By default the selection column displays `<vaadin24-checkbox>` elements in the
  * column cells. The checkboxes in the body rows toggle selection of the corresponding row items.
  *
  * When the grid data is provided as an array of [`items`](#/elements/vaadin-grid#property-items),
@@ -56,7 +56,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-grid-selection-column': GridSelectionColumn<GridDefaultItem>;
+    'vaadin24-grid-selection-column': GridSelectionColumn<GridDefaultItem>;
   }
 }
 
