Index: src/vaadin-grid-styles.js
===================================================================
--- src/vaadin-grid-styles.js
+++ src/vaadin-grid-styles.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 export const gridStyles = css`
   @keyframes vaadin-grid-appear {
