Index: src/vaadin-grid-column.d.ts
===================================================================
--- src/vaadin-grid-column.d.ts
+++ src/vaadin-grid-column.d.ts
@@ -65,7 +65,7 @@
    * Receives two arguments:
    *
    * - `root` The header cell content DOM element. Append your content to it.
-   * - `column` The `<vaadin-grid-column>` element.
+   * - `column` The `<vaadin23-grid-column>` element.
    */
   headerRenderer: GridHeaderFooterRenderer<TItem> | null | undefined;
 
@@ -74,17 +74,17 @@
    * Receives two arguments:
    *
    * - `root` The footer cell content DOM element. Append your content to it.
-   * - `column` The `<vaadin-grid-column>` element.
+   * - `column` The `<vaadin23-grid-column>` element.
    */
   footerRenderer: GridHeaderFooterRenderer<TItem> | null | undefined;
 }
 
 /**
- * A `<vaadin-grid-column>` is used to configure how a column in `<vaadin-grid>`
+ * A `<vaadin23-grid-column>` is used to configure how a column in `<vaadin23-grid>`
  * should look like.
  *
- * See [`<vaadin-grid>`](#/elements/vaadin-grid) documentation for instructions on how
- * to configure the `<vaadin-grid-column>`.
+ * See [`<vaadin23-grid>`](#/elements/vaadin-grid) documentation for instructions on how
+ * to configure the `<vaadin23-grid-column>`.
  */
 declare class GridColumn<TItem = GridDefaultItem> extends HTMLElement {
   /**
@@ -103,7 +103,7 @@
    * Receives three arguments:
    *
    * - `root` The cell content DOM element. Append your content to it.
-   * - `column` The `<vaadin-grid-column>` element.
+   * - `column` The `<vaadin23-grid-column>` element.
    * - `model` The object with the properties related with
    *   the rendered item, contains:
    *   - `model.index` The index of the item.
@@ -144,7 +144,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-grid-column': GridColumn<GridDefaultItem>;
+    'vaadin23-grid-column': GridColumn<GridDefaultItem>;
   }
 }
 
