Index: src/vaadin-grid-sort-column.d.ts
===================================================================
--- src/vaadin-grid-sort-column.d.ts
+++ src/vaadin-grid-sort-column.d.ts
@@ -19,15 +19,15 @@
 export interface GridSortColumnEventMap extends HTMLElementEventMap, GridSortColumnCustomEventMap {}
 
 /**
- * `<vaadin-grid-sort-column>` is a helper element for the `<vaadin-grid>`
+ * `<vaadin23-grid-sort-column>` is a helper element for the `<vaadin23-grid>`
  * that provides default header renderer and functionality for sorting.
  *
  * #### Example:
  * ```html
- * <vaadin-grid items="[[items]]">
- *  <vaadin-grid-sort-column path="name.first" direction="asc"></vaadin-grid-sort-column>
+ * <vaadin23-grid items="[[items]]">
+ *  <vaadin23-grid-sort-column path="name.first" direction="asc"></vaadin23-grid-sort-column>
  *
- *  <vaadin-grid-column>
+ *  <vaadin23-grid-column>
  *    ...
  * ```
  *
@@ -61,7 +61,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-grid-sort-column': GridSortColumn<GridDefaultItem>;
+    'vaadin23-grid-sort-column': GridSortColumn<GridDefaultItem>;
   }
 }
 
