Index: grid/src/vaadin-grid-column-mixin.js
===================================================================
--- grid/src/vaadin-grid-column-mixin.js	null
+++ grid/src/vaadin-grid-column-mixin.js	null
@@ -3,11 +3,11 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { animationFrame } from '@vaadin/component-base/src/async.js';
-import { Debouncer } from '@vaadin/component-base/src/debounce.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { get } from '@vaadin/component-base/src/path-utils.js';
-import { processTemplates } from '@vaadin/component-base/src/templates.js';
+import { animationFrame } from '@scoped-vaadin/component-base/src/async.js';
+import { Debouncer } from '@scoped-vaadin/component-base/src/debounce.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { get } from '@scoped-vaadin/component-base/src/path-utils.js';
+import { processTemplates } from '@scoped-vaadin/component-base/src/templates.js';
 import { updateCellState } from './vaadin-grid-helpers.js';
 
 /**
@@ -25,12 +25,12 @@
           type: Boolean,
           sync: true,
           value() {
-            if (this.localName === 'vaadin-grid-column-group') {
+            if (this.localName === 'vaadin24-grid-column-group') {
               return;
             }
 
             const parent = this.parentNode;
-            if (parent && parent.localName === 'vaadin-grid-column-group') {
+            if (parent && parent.localName === 'vaadin24-grid-column-group') {
               return parent.resizable || false;
             }
             return false;
@@ -203,7 +203,7 @@
          * Receives two arguments:
          *
          * - `root` The header cell content DOM element. Append your content to it.
-         * - `column` The `<vaadin-grid-column>` element.
+         * - `column` The `<vaadin24-grid-column>` element.
          *
          * @type {GridHeaderFooterRenderer | null | undefined}
          */
@@ -230,7 +230,7 @@
          * Receives two arguments:
          *
          * - `root` The footer cell content DOM element. Append your content to it.
-         * - `column` The `<vaadin-grid-column>` element.
+         * - `column` The `<vaadin24-grid-column>` element.
          *
          * @type {GridHeaderFooterRenderer | null | undefined}
          */
@@ -875,7 +875,7 @@
          * Receives three arguments:
          *
          * - `root` The cell content DOM element. Append your content to it.
-         * - `column` The `<vaadin-grid-column>` element.
+         * - `column` The `<vaadin24-grid-column>` element.
          * - `model` The object with the properties related with
          *   the rendered item, contains:
          *   - `model.index` The index of the item.
