Index: src/vaadin-grid-sorter.d.ts
===================================================================
--- src/vaadin-grid-sorter.d.ts
+++ src/vaadin-grid-sorter.d.ts
@@ -3,26 +3,26 @@
  * Copyright (c) 2016 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { type GridSorterEventMap, GridSorterMixin } from './vaadin-grid-sorter-mixin.js';
 
 export * from './vaadin-grid-sorter-mixin.js';
 
 /**
- * `<vaadin-grid-sorter>` is a helper element for the `<vaadin-grid>` that provides out-of-the-box UI controls,
+ * `<vaadin24-grid-sorter>` is a helper element for the `<vaadin24-grid>` that provides out-of-the-box UI controls,
  * visual feedback, and handlers for sorting the grid data.
  *
  * #### Example:
  * ```html
- * <vaadin-grid-column id="column"></vaadin-grid-column>
+ * <vaadin24-grid-column id="column"></vaadin24-grid-column>
  * ```
  * ```js
  * const column = document.querySelector('#column');
  * column.renderer = (root, column, model) => {
  *   let sorter = root.firstElementChild;
  *   if (!sorter) {
- *     sorter = document.createElement('vaadin-grid-sorter');
+ *     sorter = document.createElement('vaadin24-grid-sorter');
  *     root.appendChild(sorter);
  *   }
  *   sorter.path = 'name.first';
@@ -64,7 +64,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-grid-sorter': GridSorter;
+    'vaadin24-grid-sorter': GridSorter;
   }
 }
 
