Index: src/vaadin-grid-filter.d.ts
===================================================================
--- src/vaadin-grid-filter.d.ts
+++ src/vaadin-grid-filter.d.ts
@@ -16,19 +16,19 @@
 export interface GridFilterEventMap extends HTMLElementEventMap, GridFilterCustomEventMap {}
 
 /**
- * `<vaadin-grid-filter>` is a helper element for the `<vaadin-grid>` that provides out-of-the-box UI controls,
+ * `<vaadin23-grid-filter>` is a helper element for the `<vaadin23-grid>` that provides out-of-the-box UI controls,
  * and handlers for filtering the grid data.
  *
  * #### Example:
  * ```html
- * <vaadin-grid-column id="column"></vaadin-grid-column>
+ * <vaadin23-grid-column id="column"></vaadin23-grid-column>
  * ```
  * ```js
  * const column = document.querySelector('#column');
  * column.headerRenderer = (root, column) => {
  *   let filter = root.firstElementChild;
  *   if (!filter) {
- *     filter = document.createElement('vaadin-grid-filter');
+ *     filter = document.createElement('vaadin23-grid-filter');
  *     root.appendChild(filter);
  *   }
  *   filter.path = 'name.first';
@@ -66,7 +66,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-grid-filter': GridFilter;
+    'vaadin23-grid-filter': GridFilter;
   }
 }
 
