Index: src/vaadin-grid-column-resizing-mixin.js
===================================================================
--- src/vaadin-grid-column-resizing-mixin.js
+++ src/vaadin-grid-column-resizing-mixin.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2016 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { addListener } from '@vaadin/component-base/src/gestures.js';
+import { addListener } from '@scoped-vaadin/component-base/src/gestures.js';
 
 /**
  * @polymerMixin
@@ -42,7 +42,7 @@
         this.$.scroller.toggleAttribute('column-resizing', true);
 
         // Get the target column to resize
-        while (column.localName === 'vaadin-grid-column-group') {
+        while (column.localName === 'vaadin24-grid-column-group') {
           column = column._childColumns
             .slice(0)
             .sort((a, b) => a._order - b._order)
