Index: src/vaadin-grid-filter-column.d.ts
===================================================================
--- src/vaadin-grid-filter-column.d.ts
+++ src/vaadin-grid-filter-column.d.ts
@@ -7,15 +7,15 @@
 import { GridColumn } from './vaadin-grid-column.js';
 
 /**
- * `<vaadin-grid-filter-column>` is a helper element for the `<vaadin-grid>`
+ * `<vaadin23-grid-filter-column>` is a helper element for the `<vaadin23-grid>`
  * that provides default header renderer and functionality for filtering.
  *
  * #### Example:
  * ```html
- * <vaadin-grid items="[[items]]">
- *  <vaadin-grid-filter-column path="name.first"></vaadin-grid-filter-column>
+ * <vaadin23-grid items="[[items]]">
+ *  <vaadin23-grid-filter-column path="name.first"></vaadin23-grid-filter-column>
  *
- *  <vaadin-grid-column>
+ *  <vaadin23-grid-column>
  *    ...
  * ```
  */
@@ -33,7 +33,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-grid-filter-column': GridFilterColumn<GridDefaultItem>;
+    'vaadin23-grid-filter-column': GridFilterColumn<GridDefaultItem>;
   }
 }
 
