Index: grid/src/vaadin-grid-event-context-mixin.js
===================================================================
--- grid/src/vaadin-grid-event-context-mixin.js	null
+++ grid/src/vaadin-grid-event-context-mixin.js	null
@@ -24,7 +24,7 @@
      *
      * The returned object is populated only when a grid cell, header, footer or row details is found in `event.composedPath()`.
      * This means mostly mouse and keyboard events. If such a grid part is not found in the path, an empty object is returned.
-     * This may be the case eg. if the event is fired on the `<vaadin-grid>` element and not any deeper in the DOM, or if
+     * This may be the case eg. if the event is fired on the `<vaadin24-grid>` element and not any deeper in the DOM, or if
      * the event targets the empty part of the grid body.
      *
      * @param {!Event} event
