Index: src/vaadin-lit-grid-column-group.js
===================================================================
--- src/vaadin-lit-grid-column-group.js
+++ src/vaadin-lit-grid-column-group.js
@@ -4,12 +4,12 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
 import { GridColumnGroupMixin } from './vaadin-grid-column-group-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-grid-column-group>` web component.
+ * LitElement based version of `<vaadin24-grid-column-group>` web component.
  *
  * ## Disclaimer
  *
@@ -19,7 +19,7 @@
  */
 class GridColumnGroup extends GridColumnGroupMixin(PolylitMixin(LitElement)) {
   static get is() {
-    return 'vaadin-grid-column-group';
+    return 'vaadin24-grid-column-group';
   }
 }
 
