Index: grid/src/vaadin-grid-column-reordering-mixin.js
===================================================================
--- grid/src/vaadin-grid-column-reordering-mixin.js
+++ grid/src/vaadin-grid-column-reordering-mixin.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
-import { addListener } from '@vaadin/component-base/src/gestures.js';
+import { isTouch } from '@scoped-vaadin/component-base/src/browser-utils.js';
+import { addListener } from '@scoped-vaadin/component-base/src/gestures.js';
 import { iterateChildren, updateColumnOrders } from './vaadin-grid-helpers.js';
 
 /**
