Index: src/vaadin-grid-tree-column.js
===================================================================
--- src/vaadin-grid-tree-column.js
+++ src/vaadin-grid-tree-column.js
@@ -4,20 +4,20 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import './vaadin-grid-tree-toggle.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
 import { GridColumn } from './vaadin-grid-column.js';
 import { GridTreeColumnMixin } from './vaadin-grid-tree-column-mixin.js';
 
 /**
- * `<vaadin-grid-tree-column>` is a helper element for the `<vaadin-grid>`
+ * `<vaadin24-grid-tree-column>` is a helper element for the `<vaadin24-grid>`
  * that provides default renderer and functionality for toggling tree/hierarchical items.
  *
  * #### Example:
  * ```html
- * <vaadin-grid items="[[items]]">
- *  <vaadin-grid-tree-column path="name.first"></vaadin-grid-tree-column>
+ * <vaadin24-grid items="[[items]]">
+ *  <vaadin24-grid-tree-column path="name.first"></vaadin24-grid-tree-column>
  *
- *  <vaadin-grid-column>
+ *  <vaadin24-grid-column>
  *    ...
  * ```
  * @customElement
@@ -26,7 +26,7 @@
  */
 class GridTreeColumn extends GridTreeColumnMixin(GridColumn) {
   static get is() {
-    return 'vaadin-grid-tree-column';
+    return 'vaadin24-grid-tree-column';
   }
 }
 
