Index: src/vaadin-grid-sort-column-mixin.js
===================================================================
--- src/vaadin-grid-sort-column-mixin.js
+++ src/vaadin-grid-sort-column-mixin.js
@@ -51,7 +51,7 @@
     _defaultHeaderRenderer(root, _column) {
       let sorter = root.firstElementChild;
       if (!sorter) {
-        sorter = document.createElement('vaadin-grid-sorter');
+        sorter = document.createElement('vaadin24-grid-sorter');
         sorter.addEventListener('direction-changed', this.__boundOnDirectionChanged);
         root.appendChild(sorter);
       }
