Index: theme/lumo/vaadin-checkbox-group-styles.js
===================================================================
--- theme/lumo/vaadin-checkbox-group-styles.js
+++ theme/lumo/vaadin-checkbox-group-styles.js
@@ -1,11 +1,11 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { helper } from '@vaadin/vaadin-lumo-styles/mixins/helper.js';
-import { requiredField } from '@vaadin/vaadin-lumo-styles/mixins/required-field.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { helper } from '@scoped-vaadin/vaadin-lumo-styles/mixins/helper.js';
+import { requiredField } from '@scoped-vaadin/vaadin-lumo-styles/mixins/required-field.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const checkboxGroup = css`
   :host {
@@ -53,6 +53,6 @@
   }
 `;
 
-registerStyles('vaadin-checkbox-group', [requiredField, helper, checkboxGroup], {
+registerStyles('vaadin23-checkbox-group', [requiredField, helper, checkboxGroup], {
   moduleId: 'lumo-checkbox-group',
 });
