Index: test/visual/lumo/menu-bar.test.js
===================================================================
--- test/visual/lumo/menu-bar.test.js
+++ test/visual/lumo/menu-bar.test.js
@@ -1,7 +1,7 @@
 import { arrowDown, fixtureSync, nextRender, oneEvent } from '@vaadin/testing-helpers';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/icon/theme/lumo/vaadin-icon.js';
-import '@vaadin/vaadin-lumo-styles/vaadin-iconset.js';
+import '@scoped-vaadin/icon/theme/lumo/vaadin-icon.js';
+import '@scoped-vaadin/vaadin-lumo-styles/vaadin-iconset.js';
 import '../../../theme/lumo/vaadin-menu-bar.js';
 import '../../not-animated-styles.js';
 
@@ -23,7 +23,7 @@
           div = document.createElement('div');
           div.style.padding = '10px';
 
-          element = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>', div);
+          element = fixtureSync('<vaadin24-menu-bar></vaadin24-menu-bar>', div);
           element.items = [
             { text: 'Home' },
             {
@@ -51,7 +51,7 @@
           div = document.createElement('div');
           div.style.padding = '10px';
 
-          element = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>', div);
+          element = fixtureSync('<vaadin24-menu-bar></vaadin24-menu-bar>', div);
           element.items = [{ text: 'Actions' }];
         });
 
@@ -70,8 +70,8 @@
         let overlay;
 
         function makeIcon(img) {
-          const item = document.createElement('vaadin-menu-bar-item');
-          const icon = document.createElement('vaadin-icon');
+          const item = document.createElement('vaadin24-menu-bar-item');
+          const icon = document.createElement('vaadin24-icon');
           icon.setAttribute('icon', img);
           item.appendChild(icon);
           return item;
@@ -81,7 +81,7 @@
           div = document.createElement('div');
           div.style.padding = '10px';
 
-          element = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>', div);
+          element = fixtureSync('<vaadin24-menu-bar></vaadin24-menu-bar>', div);
           element.items = [
             { component: 'u', text: 'Home' },
             {
