Index: test/a11y.test.js
===================================================================
--- test/a11y.test.js
+++ test/a11y.test.js
@@ -1,14 +1,14 @@
 import { expect } from '@esm-bundle/chai';
 import { arrowDown, arrowRight, enter, fixtureSync, nextRender, outsideClick } from '@vaadin/testing-helpers';
 import '../src/vaadin-menu-bar.js';
-import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
+import { getDeepActiveElement } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 
 describe('a11y', () => {
   describe('focus restoration', () => {
     let menuBar, overlay, buttons;
 
     beforeEach(() => {
-      menuBar = fixtureSync(`<vaadin-menu-bar></vaadin-menu-bar>`);
+      menuBar = fixtureSync(`<vaadin24-menu-bar></vaadin24-menu-bar>`);
       menuBar.items = [
         {
           text: 'Item 0',
@@ -22,7 +22,7 @@
         },
       ];
       overlay = menuBar._subMenu._overlayElement;
-      buttons = menuBar.querySelectorAll('vaadin-menu-bar-button');
+      buttons = menuBar.querySelectorAll('vaadin24-menu-bar-button');
       buttons[0].focus();
     });
 
