Index: test/a11y.common.js
===================================================================
--- test/a11y.common.js
+++ test/a11y.common.js
@@ -1,13 +1,13 @@
 import { expect } from '@vaadin/chai-plugins';
 import { arrowDown, arrowRight, enter, fixtureSync, nextRender, outsideClick } from '@vaadin/testing-helpers';
-import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
+import { getDeepActiveElement } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 
 describe('a11y', () => {
   describe('ARIA attributes', () => {
     let menu, buttons, subMenu, overflow;
 
     beforeEach(async () => {
-      menu = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>');
+      menu = fixtureSync('<vaadin24-menu-bar></vaadin24-menu-bar>');
       menu.items = [
         {
           text: 'Menu Item 1',
@@ -100,7 +100,7 @@
     let menuBar, overlay, buttons;
 
     beforeEach(async () => {
-      menuBar = fixtureSync(`<vaadin-menu-bar></vaadin-menu-bar>`);
+      menuBar = fixtureSync(`<vaadin24-menu-bar></vaadin24-menu-bar>`);
       menuBar.items = [
         {
           text: 'Item 0',
@@ -115,7 +115,7 @@
       ];
       await nextRender();
       overlay = menuBar._subMenu._overlayElement;
-      buttons = menuBar.querySelectorAll('vaadin-menu-bar-button');
+      buttons = menuBar.querySelectorAll('vaadin24-menu-bar-button');
       buttons[0].focus();
     });
 
