Index: test/keyboard-navigation.test.js
===================================================================
--- test/keyboard-navigation.test.js
+++ test/keyboard-navigation.test.js
@@ -16,7 +16,7 @@
     const wrapper = fixtureSync(`
       <div>
         <input id="first-global-focusable" />
-        <vaadin-menu-bar></vaadin-menu-bar>
+        <vaadin24-menu-bar></vaadin24-menu-bar>
         <input id="last-global-focusable" />
       </div>
     `);
@@ -221,7 +221,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Move to previous button with submenu
         await sendKeys({ press: 'ArrowLeft' });
@@ -237,7 +237,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Move to previous button without submenu
         await sendKeys({ press: 'ArrowLeft' });
@@ -253,7 +253,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Move to next button with submenu
         await sendKeys({ press: 'ArrowRight' });
@@ -269,7 +269,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Move to next button without submenu
         await sendKeys({ press: 'ArrowRight' });
@@ -285,7 +285,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         await sendKeys({ press: 'ArrowLeft' });
         await nextRender();
@@ -303,7 +303,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         await sendKeys({ press: 'ArrowRight' });
         await nextRender();
@@ -327,7 +327,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Tab to next button with submenu
         await sendKeys({ press: 'Tab' });
@@ -343,7 +343,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Shift Tab to previous button with submenu
         await sendKeys({ press: 'Shift+Tab' });
@@ -359,7 +359,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Tab to next button without submenu
         await sendKeys({ press: 'Tab' });
@@ -375,7 +375,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Tab to next button without submenu
         await sendKeys({ press: 'Shift+Tab' });
@@ -391,7 +391,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Tab outside the menu bar
         await sendKeys({ press: 'Tab' });
@@ -407,7 +407,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Shift + Tab outside the menu bar
         await sendKeys({ press: 'Shift+Tab' });
@@ -425,7 +425,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Tab outside the menu bar (since next button is disabled)
         await sendKeys({ press: 'Tab' });
@@ -444,7 +444,7 @@
         await sendKeys({ press: 'ArrowDown' });
         await nextRender();
         expect(subMenu.opened).to.be.true;
-        expect(document.activeElement).to.equal(overlay.querySelector('vaadin-menu-bar-item'));
+        expect(document.activeElement).to.equal(overlay.querySelector('vaadin24-menu-bar-item'));
 
         // Tab outside the menu bar (since previous buttons are disabled)
         await sendKeys({ press: 'Shift+Tab' });
