Index: test/menu-bar-custom-styles.js
===================================================================
--- test/menu-bar-custom-styles.js
+++ test/menu-bar-custom-styles.js
@@ -1,9 +1,9 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-menu-bar',
+  'vaadin24-menu-bar',
   css`
-    :host([theme='big']) ::slotted(vaadin-menu-bar-button) {
+    :host([theme='big']) ::slotted(vaadin24-menu-bar-button) {
       width: 100px;
     }
   `,
