Index: test/menu-bar-test-styles.js
===================================================================
--- test/menu-bar-test-styles.js
+++ test/menu-bar-test-styles.js
@@ -1,11 +1,11 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 // TODO: subset of Lumo needed for unit tests to pass.
 // These should be eventually covered by base styles.
 registerStyles(
-  'vaadin-menu-bar',
+  'vaadin24-menu-bar',
   css`
-    ::slotted(vaadin-menu-bar-button) {
+    ::slotted(vaadin24-menu-bar-button) {
       box-sizing: border-box;
       min-width: 4.5rem;
       margin: 2px;
@@ -21,7 +21,7 @@
     }
 
     /* Used for testing overflow detection on theme change */
-    :host([theme='big']) ::slotted(vaadin-menu-bar-button) {
+    :host([theme='big']) ::slotted(vaadin24-menu-bar-button) {
       width: 100px;
     }
   `,
@@ -29,7 +29,7 @@
 );
 
 registerStyles(
-  'vaadin-menu-bar-item',
+  'vaadin24-menu-bar-item',
   css`
     :host {
       display: flex;
