Index: test/overflow.common.js
===================================================================
--- test/overflow.common.js
+++ test/overflow.common.js
@@ -23,10 +23,10 @@
       wrapper = fixtureSync(`
         <div style="display: flex">
           <div style="width: 100px;"></div>
-          <vaadin-menu-bar style="width: 200px"></vaadin-menu-bar>
+          <vaadin24-menu-bar style="width: 200px"></vaadin24-menu-bar>
         </div>
       `);
-      menu = wrapper.querySelector('vaadin-menu-bar');
+      menu = wrapper.querySelector('vaadin24-menu-bar');
       await nextRender(menu);
       menu.items = [
         { text: 'Item 1' },
@@ -241,7 +241,7 @@
     let menu, buttons, overflow;
 
     beforeEach(async () => {
-      menu = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>');
+      menu = fixtureSync('<vaadin24-menu-bar></vaadin24-menu-bar>');
       menu.items = [{ text: 'Item 1' }, { text: 'Item 2' }, { text: 'Item 3' }, { text: 'Item 4' }];
       await nextRender(menu);
       buttons = menu._buttons;
@@ -313,7 +313,7 @@
 
     beforeEach(async () => {
       container = fixtureSync(
-        '<div style="display: flex;"><vaadin-menu-bar style="min-width: 100%"></vaadin-menu-bar></div>',
+        '<div style="display: flex;"><vaadin24-menu-bar style="min-width: 100%"></vaadin24-menu-bar></div>',
       );
       menu = container.firstChild;
 
@@ -373,7 +373,7 @@
       container = fixtureSync('<div style="display: flex; max-width: 300px"></div>');
       text = document.createElement('div');
       text.textContent = 'Sibling';
-      menu = document.createElement('vaadin-menu-bar');
+      menu = document.createElement('vaadin24-menu-bar');
       menu.items = [{ text: 'Item 1' }, { text: 'Item 2' }, { text: 'Item 3' }, { text: 'Item 4' }];
       menu.style.minWidth = '100px';
     });
@@ -443,13 +443,13 @@
     }
 
     beforeEach(async () => {
-      menu = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>');
+      menu = fixtureSync('<vaadin24-menu-bar></vaadin24-menu-bar>');
       menu.items = [
         { text: 'Item 1' },
         { text: 'Item 2' },
         { component: makeComponent('3') },
         { text: 'Item 4 text', component: makeComponent('4') },
-        { text: 'Item 5', component: document.createElement('vaadin-menu-bar-item') },
+        { text: 'Item 5', component: document.createElement('vaadin24-menu-bar-item') },
         { component: makeComponent('6'), children: [{ text: 'SubItem6.1' }, { text: 'SubItem6.2' }] },
       ];
       await nextRender(menu);
@@ -481,7 +481,7 @@
     it('should teleport the same component to overflow sub-menu and back', async () => {
       overflow.click();
       await nextRender(subMenu);
-      const listBox = subMenu._overlayElement.querySelector('vaadin-menu-bar-list-box');
+      const listBox = subMenu._overlayElement.querySelector('vaadin24-menu-bar-list-box');
       expect(listBox.items[0]).to.equal(buttons[2].item.component);
       expect(listBox.items[0].firstChild).to.equal(menu.items[2].component);
       expect(listBox.items[0].firstChild.localName).to.equal('div');
