Index: test/dom/grid.test.js
===================================================================
--- test/dom/grid.test.js
+++ test/dom/grid.test.js
@@ -3,15 +3,15 @@
 import '../../vaadin-grid.js';
 import { users } from '../visual/users.js';
 
-describe('vaadin-grid', () => {
+describe('vaadin24-grid', () => {
   let grid;
 
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid>
-        <vaadin-grid-column path="name.first"></vaadin-grid-column>
-        <vaadin-grid-column path="name.last"></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid>
+        <vaadin24-grid-column path="name.first"></vaadin24-grid-column>
+        <vaadin24-grid-column path="name.last"></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
     grid.items = users.slice(0, 2);
     await nextFrame();
@@ -39,14 +39,14 @@
     });
 
     it('hidden column', async () => {
-      grid.querySelector('vaadin-grid-column').hidden = true;
+      grid.querySelector('vaadin24-grid-column').hidden = true;
       await nextFrame();
       await expect(grid).shadowDom.to.equalSnapshot();
     });
 
     it('hidden column selected', async () => {
       grid.selectedItems = [grid.items[0]];
-      grid.querySelector('vaadin-grid-column').hidden = true;
+      grid.querySelector('vaadin24-grid-column').hidden = true;
       await nextFrame();
       await expect(grid).shadowDom.to.equalSnapshot();
     });
