Index: test/visual/material/upload.test.js
===================================================================
--- test/visual/material/upload.test.js
+++ test/visual/material/upload.test.js
@@ -1,8 +1,8 @@
 import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
 import { sendKeys } from '@web/test-runner-commands';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/form-layout/theme/material/vaadin-form-layout.js';
-import '@vaadin/form-layout/theme/material/vaadin-form-item.js';
+import '@scoped-vaadin/form-layout/theme/material/vaadin-form-layout.js';
+import '@scoped-vaadin/form-layout/theme/material/vaadin-form-item.js';
 import '../../../theme/material/vaadin-upload.js';
 
 describe('upload', () => {
@@ -20,7 +20,7 @@
       });
 
       beforeEach(() => {
-        element = fixtureSync('<vaadin-upload></vaadin-upload>', div);
+        element = fixtureSync('<vaadin23-upload></vaadin23-upload>', div);
       });
 
       after(() => {
@@ -48,7 +48,7 @@
 
   describe('focus', () => {
     beforeEach(() => {
-      element = fixtureSync('<vaadin-upload></vaadin-upload>', div);
+      element = fixtureSync('<vaadin23-upload></vaadin23-upload>', div);
       element.files = [{ name: 'Don Quixote.pdf' }, { name: 'Hamlet.pdf', progress: 100, complete: true }];
       // To show the start button
       element.files[0].held = true;
@@ -104,7 +104,7 @@
 
   describe('states', () => {
     beforeEach(() => {
-      element = fixtureSync('<vaadin-upload></vaadin-upload>', div);
+      element = fixtureSync('<vaadin23-upload></vaadin23-upload>', div);
     });
 
     it('max files reached', async () => {
@@ -118,21 +118,21 @@
   describe('form-item', () => {
     beforeEach(() => {
       element = fixtureSync(
-        `<vaadin-form-layout>
-          <vaadin-upload></vaadin-upload>
-          <vaadin-form-item>
+        `<vaadin23-form-layout>
+          <vaadin23-upload></vaadin23-upload>
+          <vaadin23-form-item>
             <label slot="label">Description</label>
             <textarea></textarea>
-          </vaadin-form-item>
-          <vaadin-form-item>
+          </vaadin23-form-item>
+          <vaadin23-form-item>
             <label slot="label">File</label>
-            <vaadin-upload></vaadin-upload>
-          </vaadin-form-item>
-          <vaadin-form-item>
+            <vaadin23-upload></vaadin23-upload>
+          </vaadin23-form-item>
+          <vaadin23-form-item>
             <label slot="label">Description</label>
             <textarea></textarea>
-          </vaadin-form-item>
-        </vaadin-form-layout>`,
+          </vaadin23-form-item>
+        </vaadin23-form-layout>`,
         div,
       );
     });
