Index: test/dom/__snapshots__/dialog.test.snap.js
===================================================================
--- test/dom/__snapshots__/dialog.test.snap.js
+++ test/dom/__snapshots__/dialog.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-dialog overlay"] = 
-`<vaadin-dialog-overlay
+`<vaadin24-dialog-overlay
   focus-trap=""
   id="overlay"
   opened=""
@@ -11,12 +11,12 @@
   with-backdrop=""
 >
   content
-</vaadin-dialog-overlay>
+</vaadin24-dialog-overlay>
 `;
 /* end snapshot vaadin-dialog overlay */
 
 snapshots["vaadin-dialog overlay modeless"] = 
-`<vaadin-dialog-overlay
+`<vaadin24-dialog-overlay
   focus-trap=""
   id="overlay"
   modeless=""
@@ -25,12 +25,12 @@
   role="dialog"
 >
   content
-</vaadin-dialog-overlay>
+</vaadin24-dialog-overlay>
 `;
 /* end snapshot vaadin-dialog overlay modeless */
 
 snapshots["vaadin-dialog overlay theme"] = 
-`<vaadin-dialog-overlay
+`<vaadin24-dialog-overlay
   focus-trap=""
   id="overlay"
   opened=""
@@ -40,12 +40,12 @@
   with-backdrop=""
 >
   content
-</vaadin-dialog-overlay>
+</vaadin24-dialog-overlay>
 `;
 /* end snapshot vaadin-dialog overlay theme */
 
 snapshots["vaadin-dialog overlay class"] = 
-`<vaadin-dialog-overlay
+`<vaadin24-dialog-overlay
   class="custom dialog-overlay"
   focus-trap=""
   id="overlay"
@@ -55,7 +55,7 @@
   with-backdrop=""
 >
   content
-</vaadin-dialog-overlay>
+</vaadin24-dialog-overlay>
 `;
 /* end snapshot vaadin-dialog overlay class */
 
