Index: theme/material/vaadin-login-form-wrapper.js
===================================================================
--- theme/material/vaadin-login-form-wrapper.js
+++ theme/material/vaadin-login-form-wrapper.js
@@ -1,3 +1,3 @@
-import '@vaadin/button/theme/material/vaadin-button.js';
+import '@scoped-vaadin/button/theme/material/vaadin-button.js';
 import './vaadin-login-form-wrapper-styles.js';
 import '../../src/vaadin-login-form-wrapper.js';
