Index: overlay/theme/material/vaadin-overlay-styles.js
===================================================================
--- overlay/theme/material/vaadin-overlay-styles.js	null
+++ overlay/theme/material/vaadin-overlay-styles.js	null
@@ -1,4 +1,4 @@
-import { overlay } from '@vaadin/vaadin-material-styles/mixins/overlay.js';
-import { registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { overlay } from '@scoped-vaadin/vaadin-material-styles/mixins/overlay.js';
+import { registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
-registerStyles('vaadin-overlay', overlay, { moduleId: 'material-vaadin-overlay' });
+registerStyles('vaadin24-overlay', overlay, { moduleId: 'material-vaadin-overlay' });
