Index: test/dom/__snapshots__/login-overlay.test.snap.js
===================================================================
--- test/dom/__snapshots__/login-overlay.test.snap.js
+++ test/dom/__snapshots__/login-overlay.test.snap.js
@@ -2,17 +2,17 @@
 export const snapshots = {};
 
 snapshots["vaadin-login-overlay host default"] = 
-`<vaadin-login-overlay-wrapper
+`<vaadin24-login-overlay-wrapper
   focus-trap=""
   id="vaadinLoginOverlayWrapper"
   opened=""
   with-backdrop=""
 >
-  <vaadin-login-form
+  <vaadin24-login-form
     id="vaadinLoginForm"
     theme="with-overlay"
   >
-    <vaadin-login-form-wrapper
+    <vaadin24-login-form-wrapper
       id="vaadinLoginFormWrapper"
       theme="with-overlay"
     >
@@ -24,7 +24,7 @@
           id="csrf"
           type="hidden"
         >
-        <vaadin-text-field
+        <vaadin24-text-field
           autocapitalize="none"
           autocomplete="username"
           autocorrect="off"
@@ -59,8 +59,8 @@
             slot="error-message"
           >
           </div>
-        </vaadin-text-field>
-        <vaadin-password-field
+        </vaadin24-text-field>
+        <vaadin24-password-field
           autocomplete="current-password"
           has-label=""
           id="vaadinLoginPassword"
@@ -91,50 +91,50 @@
             slot="error-message"
           >
           </div>
-          <vaadin-password-field-button
+          <vaadin24-password-field-button
             aria-label="Show password"
             aria-pressed="false"
             role="button"
             slot="reveal"
             tabindex="0"
           >
-          </vaadin-password-field-button>
-        </vaadin-password-field>
+          </vaadin24-password-field-button>
+        </vaadin24-password-field>
       </form>
-      <vaadin-button
+      <vaadin24-button
         role="button"
         slot="submit"
         tabindex="0"
         theme="primary contained submit"
       >
         Log in
-      </vaadin-button>
-      <vaadin-button
+      </vaadin24-button>
+      <vaadin24-button
         role="button"
         slot="forgot-password"
         tabindex="0"
         theme="tertiary small"
       >
         Forgot password
-      </vaadin-button>
-    </vaadin-login-form-wrapper>
-  </vaadin-login-form>
-</vaadin-login-overlay-wrapper>
+      </vaadin24-button>
+    </vaadin24-login-form-wrapper>
+  </vaadin24-login-form>
+</vaadin24-login-overlay-wrapper>
 `;
 /* end snapshot vaadin-login-overlay host default */
 
 snapshots["vaadin-login-overlay host i18n"] = 
-`<vaadin-login-overlay-wrapper
+`<vaadin24-login-overlay-wrapper
   focus-trap=""
   id="vaadinLoginOverlayWrapper"
   opened=""
   with-backdrop=""
 >
-  <vaadin-login-form
+  <vaadin24-login-form
     id="vaadinLoginForm"
     theme="with-overlay"
   >
-    <vaadin-login-form-wrapper
+    <vaadin24-login-form-wrapper
       id="vaadinLoginFormWrapper"
       theme="with-overlay"
     >
@@ -146,7 +146,7 @@
           id="csrf"
           type="hidden"
         >
-        <vaadin-text-field
+        <vaadin24-text-field
           autocapitalize="none"
           autocomplete="username"
           autocorrect="off"
@@ -181,8 +181,8 @@
             slot="error-message"
           >
           </div>
-        </vaadin-text-field>
-        <vaadin-password-field
+        </vaadin24-text-field>
+        <vaadin24-password-field
           autocomplete="current-password"
           has-label=""
           id="vaadinLoginPassword"
@@ -213,51 +213,51 @@
             slot="error-message"
           >
           </div>
-          <vaadin-password-field-button
+          <vaadin24-password-field-button
             aria-label="Show password"
             aria-pressed="false"
             role="button"
             slot="reveal"
             tabindex="0"
           >
-          </vaadin-password-field-button>
-        </vaadin-password-field>
+          </vaadin24-password-field-button>
+        </vaadin24-password-field>
       </form>
-      <vaadin-button
+      <vaadin24-button
         role="button"
         slot="submit"
         tabindex="0"
         theme="primary contained submit"
       >
         Kirjaudu sisään
-      </vaadin-button>
-      <vaadin-button
+      </vaadin24-button>
+      <vaadin24-button
         role="button"
         slot="forgot-password"
         tabindex="0"
         theme="tertiary small"
       >
         Unohtuiko salasana?
-      </vaadin-button>
-    </vaadin-login-form-wrapper>
-  </vaadin-login-form>
-</vaadin-login-overlay-wrapper>
+      </vaadin24-button>
+    </vaadin24-login-form-wrapper>
+  </vaadin24-login-form>
+</vaadin24-login-overlay-wrapper>
 `;
 /* end snapshot vaadin-login-overlay host i18n */
 
 snapshots["vaadin-login-overlay host overlay class"] = 
-`<vaadin-login-overlay-wrapper
+`<vaadin24-login-overlay-wrapper
   class="custom login-overlay"
   focus-trap=""
   id="vaadinLoginOverlayWrapper"
   opened=""
   with-backdrop=""
 >
-  <vaadin-login-form
+  <vaadin24-login-form
     id="vaadinLoginForm"
     theme="with-overlay"
   >
-    <vaadin-login-form-wrapper
+    <vaadin24-login-form-wrapper
       id="vaadinLoginFormWrapper"
       theme="with-overlay"
     >
@@ -269,7 +269,7 @@
           id="csrf"
           type="hidden"
         >
-        <vaadin-text-field
+        <vaadin24-text-field
           autocapitalize="none"
           autocomplete="username"
           autocorrect="off"
@@ -304,8 +304,8 @@
             slot="error-message"
           >
           </div>
-        </vaadin-text-field>
-        <vaadin-password-field
+        </vaadin24-text-field>
+        <vaadin24-password-field
           autocomplete="current-password"
           has-label=""
           id="vaadinLoginPassword"
@@ -336,35 +336,35 @@
             slot="error-message"
           >
           </div>
-          <vaadin-password-field-button
+          <vaadin24-password-field-button
             aria-label="Show password"
             aria-pressed="false"
             role="button"
             slot="reveal"
             tabindex="0"
           >
-          </vaadin-password-field-button>
-        </vaadin-password-field>
+          </vaadin24-password-field-button>
+        </vaadin24-password-field>
       </form>
-      <vaadin-button
+      <vaadin24-button
         role="button"
         slot="submit"
         tabindex="0"
         theme="primary contained submit"
       >
         Log in
-      </vaadin-button>
-      <vaadin-button
+      </vaadin24-button>
+      <vaadin24-button
         role="button"
         slot="forgot-password"
         tabindex="0"
         theme="tertiary small"
       >
         Forgot password
-      </vaadin-button>
-    </vaadin-login-form-wrapper>
-  </vaadin-login-form>
-</vaadin-login-overlay-wrapper>
+      </vaadin24-button>
+    </vaadin24-login-form-wrapper>
+  </vaadin24-login-form>
+</vaadin24-login-overlay-wrapper>
 `;
 /* end snapshot vaadin-login-overlay host overlay class */
 
