Index: test/dom/__snapshots__/login-overlay.test.snap.js
===================================================================
--- test/dom/__snapshots__/login-overlay.test.snap.js
+++ test/dom/__snapshots__/login-overlay.test.snap.js
@@ -2,17 +2,17 @@
 export const snapshots = {};
 
 snapshots["vaadin-login-overlay host default"] = 
-`<vaadin-login-overlay-wrapper
+`<vaadin24-login-overlay-wrapper
   focus-trap=""
   id="vaadinLoginOverlayWrapper"
   opened=""
   with-backdrop=""
 >
-  <vaadin-login-form
+  <vaadin24-login-form
     id="vaadinLoginForm"
     theme="with-overlay"
   >
-    <vaadin-login-form-wrapper
+    <vaadin24-login-form-wrapper
       part="vaadin-login-native-form-wrapper"
       theme="with-overlay"
     >
@@ -25,7 +25,7 @@
           id="csrf"
           type="hidden"
         >
-        <vaadin-text-field
+        <vaadin24-text-field
           autocapitalize="none"
           autocomplete="username"
           autocorrect="off"
@@ -60,8 +60,8 @@
             slot="error-message"
           >
           </div>
-        </vaadin-text-field>
-        <vaadin-password-field
+        </vaadin24-text-field>
+        <vaadin24-password-field
           autocomplete="current-password"
           has-label=""
           id="vaadinLoginPassword"
@@ -92,42 +92,42 @@
             slot="error-message"
           >
           </div>
-          <vaadin-password-field-button
+          <vaadin24-password-field-button
             aria-label="Show password"
             aria-pressed="false"
             role="button"
             slot="reveal"
             tabindex="0"
           >
-          </vaadin-password-field-button>
-        </vaadin-password-field>
-        <vaadin-button
+          </vaadin24-password-field-button>
+        </vaadin24-password-field>
+        <vaadin24-button
           part="vaadin-login-submit"
           role="button"
           tabindex="0"
           theme="primary contained"
         >
           Log in
-        </vaadin-button>
+        </vaadin24-button>
       </form>
-    </vaadin-login-form-wrapper>
-  </vaadin-login-form>
-</vaadin-login-overlay-wrapper>
+    </vaadin24-login-form-wrapper>
+  </vaadin24-login-form>
+</vaadin24-login-overlay-wrapper>
 `;
 /* end snapshot vaadin-login-overlay host default */
 
 snapshots["vaadin-login-overlay host i18n"] = 
-`<vaadin-login-overlay-wrapper
+`<vaadin24-login-overlay-wrapper
   focus-trap=""
   id="vaadinLoginOverlayWrapper"
   opened=""
   with-backdrop=""
 >
-  <vaadin-login-form
+  <vaadin24-login-form
     id="vaadinLoginForm"
     theme="with-overlay"
   >
-    <vaadin-login-form-wrapper
+    <vaadin24-login-form-wrapper
       part="vaadin-login-native-form-wrapper"
       theme="with-overlay"
     >
@@ -140,7 +140,7 @@
           id="csrf"
           type="hidden"
         >
-        <vaadin-text-field
+        <vaadin24-text-field
           autocapitalize="none"
           autocomplete="username"
           autocorrect="off"
@@ -175,8 +175,8 @@
             slot="error-message"
           >
           </div>
-        </vaadin-text-field>
-        <vaadin-password-field
+        </vaadin24-text-field>
+        <vaadin24-password-field
           autocomplete="current-password"
           has-label=""
           id="vaadinLoginPassword"
@@ -207,27 +207,27 @@
             slot="error-message"
           >
           </div>
-          <vaadin-password-field-button
+          <vaadin24-password-field-button
             aria-label="Show password"
             aria-pressed="false"
             role="button"
             slot="reveal"
             tabindex="0"
           >
-          </vaadin-password-field-button>
-        </vaadin-password-field>
-        <vaadin-button
+          </vaadin24-password-field-button>
+        </vaadin24-password-field>
+        <vaadin24-button
           part="vaadin-login-submit"
           role="button"
           tabindex="0"
           theme="primary contained"
         >
           Kirjaudu sisään
-        </vaadin-button>
+        </vaadin24-button>
       </form>
-    </vaadin-login-form-wrapper>
-  </vaadin-login-form>
-</vaadin-login-overlay-wrapper>
+    </vaadin24-login-form-wrapper>
+  </vaadin24-login-form>
+</vaadin24-login-overlay-wrapper>
 `;
 /* end snapshot vaadin-login-overlay host i18n */
 
