Index: test/dom/__snapshots__/login-form.test.snap.js
===================================================================
--- test/dom/__snapshots__/login-form.test.snap.js
+++ test/dom/__snapshots__/login-form.test.snap.js
@@ -2,8 +2,8 @@
 export const snapshots = {};
 
 snapshots["vaadin-login-form host default"] = 
-`<vaadin-login-form>
-  <vaadin-login-form-wrapper id="vaadinLoginFormWrapper">
+`<vaadin24-login-form>
+  <vaadin24-login-form-wrapper id="vaadinLoginFormWrapper">
     <form
       method="POST"
       slot="form"
@@ -12,7 +12,7 @@
         id="csrf"
         type="hidden"
       >
-      <vaadin-text-field
+      <vaadin24-text-field
         autocapitalize="none"
         autocomplete="username"
         autocorrect="off"
@@ -47,8 +47,8 @@
           slot="error-message"
         >
         </div>
-      </vaadin-text-field>
-      <vaadin-password-field
+      </vaadin24-text-field>
+      <vaadin24-password-field
         autocomplete="current-password"
         has-label=""
         id="vaadinLoginPassword"
@@ -79,40 +79,40 @@
           slot="error-message"
         >
         </div>
-        <vaadin-password-field-button
+        <vaadin24-password-field-button
           aria-label="Show password"
           aria-pressed="false"
           role="button"
           slot="reveal"
           tabindex="0"
         >
-        </vaadin-password-field-button>
-      </vaadin-password-field>
+        </vaadin24-password-field-button>
+      </vaadin24-password-field>
     </form>
-    <vaadin-button
+    <vaadin24-button
       role="button"
       slot="submit"
       tabindex="0"
       theme="primary contained submit"
     >
       Log in
-    </vaadin-button>
-    <vaadin-button
+    </vaadin24-button>
+    <vaadin24-button
       role="button"
       slot="forgot-password"
       tabindex="0"
       theme="tertiary small"
     >
       Forgot password
-    </vaadin-button>
-  </vaadin-login-form-wrapper>
-</vaadin-login-form>
+    </vaadin24-button>
+  </vaadin24-login-form-wrapper>
+</vaadin24-login-form>
 `;
 /* end snapshot vaadin-login-form host default */
 
 snapshots["vaadin-login-form host required"] = 
-`<vaadin-login-form>
-  <vaadin-login-form-wrapper id="vaadinLoginFormWrapper">
+`<vaadin24-login-form>
+  <vaadin24-login-form-wrapper id="vaadinLoginFormWrapper">
     <form
       method="POST"
       slot="form"
@@ -121,7 +121,7 @@
         id="csrf"
         type="hidden"
       >
-      <vaadin-text-field
+      <vaadin24-text-field
         autocapitalize="none"
         autocomplete="username"
         autocorrect="off"
@@ -161,8 +161,8 @@
         >
           Username is required
         </div>
-      </vaadin-text-field>
-      <vaadin-password-field
+      </vaadin24-text-field>
+      <vaadin24-password-field
         autocomplete="current-password"
         has-error-message=""
         has-label=""
@@ -197,40 +197,40 @@
         >
           Password is required
         </div>
-        <vaadin-password-field-button
+        <vaadin24-password-field-button
           aria-label="Show password"
           aria-pressed="false"
           role="button"
           slot="reveal"
           tabindex="0"
         >
-        </vaadin-password-field-button>
-      </vaadin-password-field>
+        </vaadin24-password-field-button>
+      </vaadin24-password-field>
     </form>
-    <vaadin-button
+    <vaadin24-button
       role="button"
       slot="submit"
       tabindex="0"
       theme="primary contained submit"
     >
       Log in
-    </vaadin-button>
-    <vaadin-button
+    </vaadin24-button>
+    <vaadin24-button
       role="button"
       slot="forgot-password"
       tabindex="0"
       theme="tertiary small"
     >
       Forgot password
-    </vaadin-button>
-  </vaadin-login-form-wrapper>
-</vaadin-login-form>
+    </vaadin24-button>
+  </vaadin24-login-form-wrapper>
+</vaadin24-login-form>
 `;
 /* end snapshot vaadin-login-form host required */
 
 snapshots["vaadin-login-form host i18n"] = 
-`<vaadin-login-form>
-  <vaadin-login-form-wrapper id="vaadinLoginFormWrapper">
+`<vaadin24-login-form>
+  <vaadin24-login-form-wrapper id="vaadinLoginFormWrapper">
     <form
       method="POST"
       slot="form"
@@ -239,7 +239,7 @@
         id="csrf"
         type="hidden"
       >
-      <vaadin-text-field
+      <vaadin24-text-field
         autocapitalize="none"
         autocomplete="username"
         autocorrect="off"
@@ -274,8 +274,8 @@
           slot="error-message"
         >
         </div>
-      </vaadin-text-field>
-      <vaadin-password-field
+      </vaadin24-text-field>
+      <vaadin24-password-field
         autocomplete="current-password"
         has-label=""
         id="vaadinLoginPassword"
@@ -306,40 +306,40 @@
           slot="error-message"
         >
         </div>
-        <vaadin-password-field-button
+        <vaadin24-password-field-button
           aria-label="Show password"
           aria-pressed="false"
           role="button"
           slot="reveal"
           tabindex="0"
         >
-        </vaadin-password-field-button>
-      </vaadin-password-field>
+        </vaadin24-password-field-button>
+      </vaadin24-password-field>
     </form>
-    <vaadin-button
+    <vaadin24-button
       role="button"
       slot="submit"
       tabindex="0"
       theme="primary contained submit"
     >
       Kirjaudu sisään
-    </vaadin-button>
-    <vaadin-button
+    </vaadin24-button>
+    <vaadin24-button
       role="button"
       slot="forgot-password"
       tabindex="0"
       theme="tertiary small"
     >
       Unohtuiko salasana?
-    </vaadin-button>
-  </vaadin-login-form-wrapper>
-</vaadin-login-form>
+    </vaadin24-button>
+  </vaadin24-login-form-wrapper>
+</vaadin24-login-form>
 `;
 /* end snapshot vaadin-login-form host i18n */
 
 snapshots["vaadin-login-form host i18n-required"] = 
-`<vaadin-login-form>
-  <vaadin-login-form-wrapper id="vaadinLoginFormWrapper">
+`<vaadin24-login-form>
+  <vaadin24-login-form-wrapper id="vaadinLoginFormWrapper">
     <form
       method="POST"
       slot="form"
@@ -348,7 +348,7 @@
         id="csrf"
         type="hidden"
       >
-      <vaadin-text-field
+      <vaadin24-text-field
         autocapitalize="none"
         autocomplete="username"
         autocorrect="off"
@@ -388,8 +388,8 @@
         >
           Käyttäjätunnus vaaditaan
         </div>
-      </vaadin-text-field>
-      <vaadin-password-field
+      </vaadin24-text-field>
+      <vaadin24-password-field
         autocomplete="current-password"
         has-error-message=""
         has-label=""
@@ -424,40 +424,40 @@
         >
           Salasana vaaditaan
         </div>
-        <vaadin-password-field-button
+        <vaadin24-password-field-button
           aria-label="Show password"
           aria-pressed="false"
           role="button"
           slot="reveal"
           tabindex="0"
         >
-        </vaadin-password-field-button>
-      </vaadin-password-field>
+        </vaadin24-password-field-button>
+      </vaadin24-password-field>
     </form>
-    <vaadin-button
+    <vaadin24-button
       role="button"
       slot="submit"
       tabindex="0"
       theme="primary contained submit"
     >
       Kirjaudu sisään
-    </vaadin-button>
-    <vaadin-button
+    </vaadin24-button>
+    <vaadin24-button
       role="button"
       slot="forgot-password"
       tabindex="0"
       theme="tertiary small"
     >
       Unohtuiko salasana?
-    </vaadin-button>
-  </vaadin-login-form-wrapper>
-</vaadin-login-form>
+    </vaadin24-button>
+  </vaadin24-login-form-wrapper>
+</vaadin24-login-form>
 `;
 /* end snapshot vaadin-login-form host i18n-required */
 
 snapshots["vaadin-login-form host noForgotPassword"] = 
-`<vaadin-login-form>
-  <vaadin-login-form-wrapper id="vaadinLoginFormWrapper">
+`<vaadin24-login-form>
+  <vaadin24-login-form-wrapper id="vaadinLoginFormWrapper">
     <form
       method="POST"
       slot="form"
@@ -466,7 +466,7 @@
         id="csrf"
         type="hidden"
       >
-      <vaadin-text-field
+      <vaadin24-text-field
         autocapitalize="none"
         autocomplete="username"
         autocorrect="off"
@@ -501,8 +501,8 @@
           slot="error-message"
         >
         </div>
-      </vaadin-text-field>
-      <vaadin-password-field
+      </vaadin24-text-field>
+      <vaadin24-password-field
         autocomplete="current-password"
         has-label=""
         id="vaadinLoginPassword"
@@ -533,25 +533,25 @@
           slot="error-message"
         >
         </div>
-        <vaadin-password-field-button
+        <vaadin24-password-field-button
           aria-label="Show password"
           aria-pressed="false"
           role="button"
           slot="reveal"
           tabindex="0"
         >
-        </vaadin-password-field-button>
-      </vaadin-password-field>
+        </vaadin24-password-field-button>
+      </vaadin24-password-field>
     </form>
-    <vaadin-button
+    <vaadin24-button
       role="button"
       slot="submit"
       tabindex="0"
       theme="primary contained submit"
     >
       Log in
-    </vaadin-button>
-    <vaadin-button
+    </vaadin24-button>
+    <vaadin24-button
       hidden=""
       role="button"
       slot="forgot-password"
@@ -559,9 +559,9 @@
       theme="tertiary small"
     >
       Forgot password
-    </vaadin-button>
-  </vaadin-login-form-wrapper>
-</vaadin-login-form>
+    </vaadin24-button>
+  </vaadin24-login-form-wrapper>
+</vaadin24-login-form>
 `;
 /* end snapshot vaadin-login-form host noForgotPassword */
 
