Index: test/fixtures/mock-combo-box-light-template-wrapper.js
===================================================================
--- test/fixtures/mock-combo-box-light-template-wrapper.js
+++ test/fixtures/mock-combo-box-light-template-wrapper.js
@@ -1,11 +1,12 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import '../../vaadin-combo-box-light.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
 
 class MockComboBoxLightTemplateWrapper extends PolymerElement {
   static get template() {
     return html`
-      <vaadin-combo-box-light id="comboBox" items="[[items]]">
-        <vaadin-text-field></vaadin-text-field>
+      <vaadin24-combo-box-light id="comboBox" items="[[items]]">
+        <vaadin24-text-field></vaadin24-text-field>
 
         <template>
           <div>index: [[index]]</div>
@@ -17,7 +18,7 @@
           <div>parentMethod: [[parentMethod()]]</div>
           <button on-click="parentEventHandler"></button>
         </template>
-      </vaadin-combo-box-light>
+      </vaadin24-combo-box-light>
     `;
   }
 
@@ -36,4 +37,4 @@
   }
 }
 
-customElements.define('mock-combo-box-light-template-wrapper', MockComboBoxLightTemplateWrapper);
+internalCustomElements.define('mock-combo-box-light-template-wrapper', MockComboBoxLightTemplateWrapper);
