Index: test/visual/material/date-picker.test.js
===================================================================
--- test/visual/material/date-picker.test.js
+++ test/visual/material/date-picker.test.js
@@ -12,7 +12,7 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-date-picker></vaadin-date-picker>', div);
+    element = fixtureSync('<vaadin23-date-picker></vaadin23-date-picker>', div);
   });
 
   it('basic', async () => {
