Index: test/dom/__snapshots__/radio-button.test.snap.js
===================================================================
--- test/dom/__snapshots__/radio-button.test.snap.js
+++ test/dom/__snapshots__/radio-button.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-radio-button host default"] = 
-`<vaadin-radio-button
+`<vaadin23-radio-button
   has-label=""
   has-value=""
   label="Radio button"
@@ -20,12 +20,12 @@
     type="radio"
     value="on"
   >
-</vaadin-radio-button>
+</vaadin23-radio-button>
 `;
 /* end snapshot vaadin-radio-button host default */
 
 snapshots["vaadin-radio-button host disabled"] = 
-`<vaadin-radio-button
+`<vaadin23-radio-button
   aria-disabled="true"
   disabled=""
   has-label=""
@@ -47,7 +47,7 @@
     type="radio"
     value="on"
   >
-</vaadin-radio-button>
+</vaadin23-radio-button>
 `;
 /* end snapshot vaadin-radio-button host disabled */
 
