Index: test/dom/confirm-dialog.test.js
===================================================================
--- test/dom/confirm-dialog.test.js
+++ test/dom/confirm-dialog.test.js
@@ -2,7 +2,7 @@
 import { fixtureSync, nextRender, oneEvent } from '@vaadin/testing-helpers';
 import '../../src/vaadin-confirm-dialog.js';
 
-describe('vaadin-confirm-dialog', () => {
+describe('vaadin24-confirm-dialog', () => {
   let dialog, overlay;
 
   const SNAPSHOT_CONFIG = {
@@ -13,9 +13,9 @@
 
   beforeEach(async () => {
     dialog = fixtureSync(`
-      <vaadin-confirm-dialog header="Unsaved changes">
+      <vaadin24-confirm-dialog header="Unsaved changes">
         Do you want to save or discard the changes?
-      </vaadin-confirm-dialog>
+      </vaadin24-confirm-dialog>
     `);
     await nextRender();
     overlay = dialog.$.dialog._overlayElement;
