Index: utility.js
===================================================================
--- utility.js
+++ utility.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/register-styles.js';
 import { accessibility } from './utilities/accessibility.js';
 import { background } from './utilities/background.js';
 import { border } from './utilities/border.js';
