Index: vaadin-lumo-styles/vaadin-iconset.js
===================================================================
--- vaadin-lumo-styles/vaadin-iconset.js	null
+++ vaadin-lumo-styles/vaadin-iconset.js	null
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import './version.js';
-import { Iconset } from '@vaadin/icon/vaadin-iconset.js';
+import { Iconset } from '@scoped-vaadin/icon/vaadin-iconset.js';
 
 const template = document.createElement('template');
 
