Index: vaadin-iconset.js
===================================================================
--- vaadin-iconset.js
+++ vaadin-iconset.js
@@ -3,12 +3,12 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/icon/vaadin-iconset.js';
+import '@scoped-vaadin/icon/vaadin-iconset.js';
 import './version.js';
 
 const template = document.createElement('template');
 
-template.innerHTML = `<vaadin-iconset name="lumo" size="1000">
+template.innerHTML = `<vaadin23-iconset name="lumo" size="1000">
 <svg xmlns="http://www.w3.org/2000/svg">
 <defs>
 <g id="lumo:align-center"><path d="M167 217c0-18 17-33 38-34H795c21 0 38 15 38 34 0 18-17 33-38 33H205C184 250 167 235 167 217z m83 191c0-18 13-33 29-33H721c16 0 29 15 29 33 0 18-13 33-29 34H279C263 442 250 427 250 408zM250 792c0-18 13-33 29-34H721c16 0 29 15 29 34s-13 33-29 33H279C263 825 250 810 250 792z m-83-192c0-18 17-33 38-33H795c21 0 38 15 38 33s-17 33-38 33H205C184 633 167 618 167 600z" fill-rule="evenodd" clip-rule="evenodd"></path></g>
@@ -55,6 +55,6 @@
 <g id="lumo:user"><path d="M500 500c-69 0-125-56-125-125s56-125 125-125 125 56 125 125-56 125-125 125z m-292 292c0-115 131-208 292-209s292 93 292 209H208z"></path></g>
 </defs>
 </svg>
-</vaadin-iconset>`;
+</vaadin23-iconset>`;
 
 document.head.appendChild(template.content);
