Index: multi-select-combo-box/src/lit/renderer-directives.js
===================================================================
--- multi-select-combo-box/src/lit/renderer-directives.js
+++ multi-select-combo-box/src/lit/renderer-directives.js
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { directive } from 'lit/directive.js';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 
 export class MultiSelectComboBoxRendererDirective extends LitRendererDirective {
   /**
@@ -32,7 +32,7 @@
 }
 
 /**
- * A Lit directive for rendering the content of the `<vaadin-multi-select-combo-box-item>` elements.
+ * A Lit directive for rendering the content of the `<vaadin24-multi-select-combo-box-item>` elements.
  *
  * The directive accepts a renderer callback returning a Lit template and assigns it to the combo-box
  * via the `renderer` property. The renderer is called for each combo-box item when assigned
@@ -48,9 +48,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-multi-select-combo-box
+ * `<vaadin24-multi-select-combo-box
  *   ${multiSelectComboBoxRenderer((item, model, comboBox) => html`...`)}
- * ></vaadin-multi-select-combo-box>`
+ * ></vaadin24-multi-select-combo-box>`
  * ```
  *
  * @param renderer the renderer callback that returns a Lit template.
