Index: src/lit/renderer-directives.d.ts
===================================================================
--- src/lit/renderer-directives.d.ts
+++ src/lit/renderer-directives.d.ts
@@ -5,8 +5,8 @@
  */
 import type { TemplateResult } from 'lit';
 import type { DirectiveResult } from 'lit/directive.js';
-import type { ComboBoxItemModel } from '@vaadin/combo-box/src/vaadin-combo-box.js';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import type { ComboBoxItemModel } from '@scoped-vaadin/combo-box/src/vaadin-combo-box.js';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 import type { MultiSelectComboBox } from '../vaadin-multi-select-combo-box.js';
 
 export type MultiSelectComboBoxLitRenderer<TItem> = (
@@ -36,7 +36,7 @@
 }
 
 /**
- * A Lit directive for rendering the content of the `<vaadin-multi-select-combo-box-item>` elements.
+ * A Lit directive for rendering the content of the `<vaadin23-multi-select-combo-box-item>` elements.
  *
  * The directive accepts a renderer callback returning a Lit template and assigns it to the combo-box
  * via the `renderer` property. The renderer is called for each combo-box item when assigned
@@ -52,9 +52,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-multi-select-combo-box
+ * `<vaadin23-multi-select-combo-box
  *   ${multiSelectComboBoxRenderer((item, model, comboBox) => html`...`)}
- * ></vaadin-multi-select-combo-box>`
+ * ></vaadin23-multi-select-combo-box>`
  * ```
  *
  * @param renderer the renderer callback that returns a Lit template.
