Index: multi-select-combo-box/src/lit/renderer-directives.d.ts
===================================================================
--- multi-select-combo-box/src/lit/renderer-directives.d.ts
+++ multi-select-combo-box/src/lit/renderer-directives.d.ts
@@ -4,9 +4,9 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { DirectiveResult } from 'lit/directive.js';
-import type { ComboBoxItemModel } from '@vaadin/combo-box/src/vaadin-combo-box.js';
-import type { LitRendererResult } from '@vaadin/lit-renderer';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import type { ComboBoxItemModel } from '@scoped-vaadin/combo-box/src/vaadin-combo-box.js';
+import type { LitRendererResult } from '@scoped-vaadin/lit-renderer';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 import type { MultiSelectComboBox } from '../vaadin-multi-select-combo-box.js';
 
 export type MultiSelectComboBoxLitRenderer<TItem> = (
@@ -36,7 +36,7 @@
 }
 
 /**
- * A Lit directive for rendering the content of the `<vaadin-multi-select-combo-box-item>` elements.
+ * A Lit directive for rendering the content of the `<vaadin24-multi-select-combo-box-item>` elements.
  *
  * The directive accepts a renderer callback returning a Lit template and assigns it to the combo-box
  * via the `renderer` property. The renderer is called for each combo-box item when assigned
@@ -52,9 +52,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-multi-select-combo-box
+ * `<vaadin24-multi-select-combo-box
  *   ${multiSelectComboBoxRenderer((item, model, comboBox) => html`...`)}
- * ></vaadin-multi-select-combo-box>`
+ * ></vaadin24-multi-select-combo-box>`
  * ```
  *
  * @param renderer the renderer callback that returns a Lit template.
