Index: test/visual/lumo/notification.test.js
===================================================================
--- test/visual/lumo/notification.test.js
+++ test/visual/lumo/notification.test.js
@@ -7,7 +7,7 @@
   let element;
 
   beforeEach(() => {
-    element = fixtureSync('<vaadin-notification duration="0"></vaadin-notification>');
+    element = fixtureSync('<vaadin24-notification duration="0"></vaadin24-notification>');
     element.renderer = (root) => {
       root.textContent = element.position;
     };
