Index: test/basic.common.js
===================================================================
--- test/basic.common.js
+++ test/basic.common.js
@@ -9,7 +9,7 @@
   let datePicker, input, overlay;
 
   beforeEach(async () => {
-    datePicker = fixtureSync(`<vaadin-date-picker></vaadin-date-picker>`);
+    datePicker = fixtureSync(`<vaadin24-date-picker></vaadin24-date-picker>`);
     await nextRender();
     input = datePicker.inputElement;
     overlay = datePicker.$.overlay;
@@ -217,7 +217,7 @@
     });
 
     it('should notify i18n mutation to children', () => {
-      const monthCalendar = overlayContent.querySelector('vaadin-month-calendar');
+      const monthCalendar = overlayContent.querySelector('vaadin24-month-calendar');
       const weekdays = monthCalendar.$.monthGrid.querySelectorAll('[part="weekday"]:not(:empty)');
       const weekdayTitles = Array.prototype.map.call(weekdays, (weekday) => weekday.textContent.trim());
       expect(weekdayTitles).to.eql(['ma', 'ti', 'ke', 'to', 'pe', 'la', 'su']);
@@ -260,10 +260,10 @@
   it('date-picker should stretch to fit the column flex container', () => {
     const container = fixtureSync(`
     <div style="display: flex; flex-direction: column; width: 500px;">
-      <vaadin-date-picker></vaadin-date-picker>
+      <vaadin24-date-picker></vaadin24-date-picker>
     </div>
     `);
-    const flexDatePicker = container.querySelector('vaadin-date-picker');
+    const flexDatePicker = container.querySelector('vaadin24-date-picker');
     expect(window.getComputedStyle(container).width).to.eql('500px');
     expect(window.getComputedStyle(flexDatePicker).width).to.eql('500px');
   });
@@ -273,7 +273,7 @@
   let datePicker, clearButton;
 
   beforeEach(async () => {
-    datePicker = fixtureSync('<vaadin-date-picker clear-button-visible></vaadin-date-picker>');
+    datePicker = fixtureSync('<vaadin24-date-picker clear-button-visible></vaadin24-date-picker>');
     await nextRender();
     clearButton = datePicker.shadowRoot.querySelector('[part="clear-button"]');
   });
@@ -376,11 +376,11 @@
     container = fixtureSync(`
       <div style="height: 100px; overflow: scroll;">
         <div style="height: 1000px;">
-          <vaadin-date-picker></vaadin-date-picker>
+          <vaadin24-date-picker></vaadin24-date-picker>
         </div>
       </div>
     `);
-    datePicker = container.querySelector('vaadin-date-picker');
+    datePicker = container.querySelector('vaadin24-date-picker');
   });
 
   it('should match the parent width', () => {
@@ -394,7 +394,7 @@
   let datePicker, input;
 
   beforeEach(async () => {
-    datePicker = fixtureSync('<vaadin-date-picker value="2000-01-01"></vaadin-date-picker>');
+    datePicker = fixtureSync('<vaadin24-date-picker value="2000-01-01"></vaadin24-date-picker>');
     await nextRender();
     input = datePicker.inputElement;
   });
@@ -408,7 +408,7 @@
   let datePicker, input, toggleButton;
 
   beforeEach(async () => {
-    datePicker = fixtureSync('<vaadin-date-picker value="2000-01-01"></vaadin-date-picker>');
+    datePicker = fixtureSync('<vaadin24-date-picker value="2000-01-01"></vaadin24-date-picker>');
     await nextRender();
     input = datePicker.inputElement;
     toggleButton = datePicker.shadowRoot.querySelector('[part="toggle-button"]');
@@ -441,7 +441,7 @@
   let datePicker, input;
 
   beforeEach(async () => {
-    datePicker = fixtureSync('<vaadin-date-picker value="2000-01-01"></vaadin-date-picker>');
+    datePicker = fixtureSync('<vaadin24-date-picker value="2000-01-01"></vaadin24-date-picker>');
     await nextRender();
     input = datePicker.inputElement;
     datePicker._ios = true;
@@ -485,7 +485,7 @@
   let datePicker;
 
   beforeEach(async () => {
-    datePicker = fixtureSync(`<vaadin-date-picker required></vaadin-date-picker>`);
+    datePicker = fixtureSync(`<vaadin24-date-picker required></vaadin24-date-picker>`);
     await nextRender();
   });
 
