Index: test/custom-input.test.js
===================================================================
--- test/custom-input.test.js
+++ test/custom-input.test.js
@@ -10,9 +10,9 @@
 
   beforeEach(() => {
     datePicker = fixtureSync(`
-      <vaadin-date-picker-light attr-for-value="value">
+      <vaadin24-date-picker-light attr-for-value="value">
         <input class="input">
-      </vaadin-date-picker-light>
+      </vaadin24-date-picker-light>
     `);
     overlay = datePicker.$.overlay;
   });
@@ -69,7 +69,7 @@
 
       it('should propagate theme attribute to month calendar', async () => {
         await waitForScrollToFinish(datePicker);
-        const monthCalendar = datePicker._overlayContent.querySelector('vaadin-month-calendar');
+        const monthCalendar = datePicker._overlayContent.querySelector('vaadin24-month-calendar');
         expect(monthCalendar.getAttribute('theme')).to.equal('foo');
       });
     });
