Index: test/dropdown.common.js
===================================================================
--- test/dropdown.common.js
+++ test/dropdown.common.js
@@ -10,7 +10,7 @@
   beforeEach(async () => {
     datePicker = fixtureSync(
       `<div>
-        <vaadin-date-picker></vaadin-date-picker>
+        <vaadin24-date-picker></vaadin24-date-picker>
         <input id="last-global-focusable" />
       </div>`,
     ).firstElementChild;
