Index: test/dropdown.common.js
===================================================================
--- test/dropdown.common.js
+++ test/dropdown.common.js
@@ -18,7 +18,7 @@
   let datePicker, input, overlay;
 
   beforeEach(async () => {
-    datePicker = fixtureSync(`<vaadin-date-picker></vaadin-date-picker>`);
+    datePicker = fixtureSync(`<vaadin24-date-picker></vaadin24-date-picker>`);
     await nextRender();
     input = datePicker.inputElement;
     overlay = datePicker.$.overlay;
