Index: test/helpers.js
===================================================================
--- test/helpers.js
+++ test/helpers.js
@@ -1,7 +1,7 @@
 import { fire, makeSoloTouchEvent, nextRender } from '@vaadin/testing-helpers';
 import { flush } from '@polymer/polymer/lib/utils/flush.js';
 import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
-import { isElementFocused } from '@vaadin/a11y-base/src/focus-utils.js';
+import { isElementFocused } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 
 export function activateScroller(scroller) {
   scroller.active = true;
@@ -107,7 +107,7 @@
  */
 export function getFocusableCell(root) {
   const overlayContent = root._overlayContent ?? root;
-  const focusableMonth = [...overlayContent.querySelectorAll('vaadin-month-calendar')].find((month) => {
+  const focusableMonth = [...overlayContent.querySelectorAll('vaadin24-month-calendar')].find((month) => {
     return !!month.shadowRoot.querySelector('[tabindex="0"]');
   });
 
