Index: test/keyboard-input.common.js
===================================================================
--- test/keyboard-input.common.js
+++ test/keyboard-input.common.js
@@ -14,7 +14,7 @@
   }
 
   beforeEach(async () => {
-    datePicker = fixtureSync('<vaadin-date-picker></vaadin-date-picker>');
+    datePicker = fixtureSync('<vaadin24-date-picker></vaadin24-date-picker>');
     await nextRender();
     input = datePicker.inputElement;
     input.focus();
