Index: test/keyboard-input.test.js
===================================================================
--- test/keyboard-input.test.js
+++ test/keyboard-input.test.js
@@ -22,7 +22,7 @@
   }
 
   beforeEach(async () => {
-    datePicker = fixtureSync('<vaadin-date-picker></vaadin-date-picker>');
+    datePicker = fixtureSync('<vaadin24-date-picker></vaadin24-date-picker>');
     await nextRender();
     input = datePicker.inputElement;
     input.focus();
