Index: test/keyboard-input.test.js
===================================================================
--- test/keyboard-input.test.js
+++ test/keyboard-input.test.js
@@ -24,7 +24,7 @@
   }
 
   beforeEach(() => {
-    datePicker = fixtureSync('<vaadin-date-picker></vaadin-date-picker>');
+    datePicker = fixtureSync('<vaadin24-date-picker></vaadin24-date-picker>');
     input = datePicker.inputElement;
     input.focus();
   });
