Index: test/keyboard-navigation.test.js
===================================================================
--- test/keyboard-navigation.test.js
+++ test/keyboard-navigation.test.js
@@ -21,7 +21,7 @@
 
     describe('default', () => {
       beforeEach(() => {
-        datePicker = fixtureSync('<vaadin-date-picker></vaadin-date-picker>');
+        datePicker = fixtureSync('<vaadin23-date-picker></vaadin23-date-picker>');
         input = datePicker.inputElement;
         input.focus();
       });
@@ -57,7 +57,7 @@
 
     describe('value', () => {
       beforeEach(() => {
-        datePicker = fixtureSync('<vaadin-date-picker value="2001-01-01"></vaadin-date-picker>');
+        datePicker = fixtureSync('<vaadin23-date-picker value="2001-01-01"></vaadin23-date-picker>');
         input = datePicker.inputElement;
         input.focus();
       });
@@ -91,7 +91,7 @@
 
     describe('initial position', () => {
       beforeEach(() => {
-        datePicker = fixtureSync('<vaadin-date-picker initial-position="2001-01-01"></vaadin-date-picker>');
+        datePicker = fixtureSync('<vaadin23-date-picker initial-position="2001-01-01"></vaadin23-date-picker>');
         input = datePicker.inputElement;
         input.focus();
       });
@@ -128,10 +128,10 @@
 
     beforeEach(async () => {
       overlay = fixtureSync(`
-        <vaadin-date-picker-overlay-content
+        <vaadin23-date-picker-overlay-content
           style="position: absolute; top: 0; width: 400px"
           scroll-duration="0"
-        ></vaadin-date-picker-overlay-content>
+        ></vaadin23-date-picker-overlay-content>
       `);
       overlay.i18n = getDefaultI18n();
 
