Index: test/keyboard-navigation.common.js
===================================================================
--- test/keyboard-navigation.common.js
+++ test/keyboard-navigation.common.js
@@ -18,7 +18,7 @@
 
     describe('default', () => {
       beforeEach(async () => {
-        datePicker = fixtureSync('<vaadin-date-picker></vaadin-date-picker>');
+        datePicker = fixtureSync('<vaadin24-date-picker></vaadin24-date-picker>');
         await nextRender();
         input = datePicker.inputElement;
         input.focus();
@@ -54,7 +54,7 @@
 
     describe('value', () => {
       beforeEach(async () => {
-        datePicker = fixtureSync('<vaadin-date-picker value="2001-01-01"></vaadin-date-picker>');
+        datePicker = fixtureSync('<vaadin24-date-picker value="2001-01-01"></vaadin24-date-picker>');
         await nextRender();
         input = datePicker.inputElement;
         input.focus();
@@ -89,7 +89,7 @@
 
     describe('initial position', () => {
       beforeEach(async () => {
-        datePicker = fixtureSync('<vaadin-date-picker initial-position="2001-01-01"></vaadin-date-picker>');
+        datePicker = fixtureSync('<vaadin24-date-picker initial-position="2001-01-01"></vaadin24-date-picker>');
         await nextRender();
         input = datePicker.inputElement;
         input.focus();
@@ -126,10 +126,10 @@
 
     beforeEach(async () => {
       overlay = fixtureSync(`
-        <vaadin-date-picker-overlay-content
+        <vaadin24-date-picker-overlay-content
           style="position: absolute; top: 0; width: 400px"
           scroll-duration="0"
-        ></vaadin-date-picker-overlay-content>
+        ></vaadin24-date-picker-overlay-content>
       `);
       overlay.i18n = getDefaultI18n();
 
