Index: test/month-calendar.common.js
===================================================================
--- test/month-calendar.common.js
+++ test/month-calendar.common.js
@@ -3,7 +3,7 @@
 import sinon from 'sinon';
 import { getDefaultI18n } from './helpers.js';
 
-describe('vaadin-month-calendar', () => {
+describe('vaadin24-month-calendar', () => {
   let monthCalendar, valueChangedSpy;
 
   function getDateCells(calendar) {
@@ -15,7 +15,7 @@
   }
 
   beforeEach(async () => {
-    monthCalendar = fixtureSync('<vaadin-month-calendar></vaadin-month-calendar>');
+    monthCalendar = fixtureSync('<vaadin24-month-calendar></vaadin24-month-calendar>');
     monthCalendar.i18n = getDefaultI18n();
     monthCalendar.month = new Date(2016, 1, 1);
     valueChangedSpy = sinon.spy();
