Index: test/month-calendar.test.js
===================================================================
--- test/month-calendar.test.js
+++ test/month-calendar.test.js
@@ -4,14 +4,14 @@
 import '../src/vaadin-month-calendar.js';
 import { getDefaultI18n } from './helpers.js';
 
-describe('vaadin-month-calendar', () => {
+describe('vaadin23-month-calendar', () => {
   let monthCalendar, valueChangedSpy;
 
   beforeEach(async () => {
     monthCalendar = fixtureSync(`
-    <vaadin-month-calendar
+    <vaadin23-month-calendar
       style="position: absolute; top: 0"
-    ></vaadin-month-calendar>`);
+    ></vaadin23-month-calendar>`);
     monthCalendar.i18n = getDefaultI18n();
     monthCalendar.month = new Date(2016, 1, 1);
     valueChangedSpy = sinon.spy();
