Index: test/overlay.test.js
===================================================================
--- test/overlay.test.js
+++ test/overlay.test.js
@@ -18,7 +18,7 @@
 }
 
 async function customizeFixture({ initialPosition, monthScrollerItems, monthScrollerOffset }) {
-  const overlay = fixtureSync(`<vaadin-date-picker-overlay-content></vaadin-date-picker-overlay-content>`);
+  const overlay = fixtureSync(`<vaadin23-date-picker-overlay-content></vaadin23-date-picker-overlay-content>`);
   const monthScroller = overlay.$.monthScroller;
   monthScroller.style.setProperty('--vaadin-infinite-scroller-buffer-offset', monthScrollerOffset);
   monthScroller.style.height = `${270 * monthScrollerItems}px`;
@@ -37,9 +37,9 @@
   describe('basic', () => {
     beforeEach((done) => {
       overlay = fixtureSync(`
-        <vaadin-date-picker-overlay-content
+        <vaadin23-date-picker-overlay-content
           style="position: absolute; top: 0"
-        ></vaadin-date-picker-overlay-content>`);
+        ></vaadin23-date-picker-overlay-content>`);
       overlay.i18n = getDefaultI18n();
       overlay.$.monthScroller.bufferSize = 1;
       overlay.$.yearScroller.bufferSize = 1;
@@ -87,7 +87,7 @@
 
       it('should set ignoreTaps to calendar on scroll', (done) => {
         listenOnce(overlay.$.monthScroller.$.scroller, 'scroll', () => {
-          expect(overlay.$.monthScroller.querySelector('vaadin-month-calendar').ignoreTaps).to.be.true;
+          expect(overlay.$.monthScroller.querySelector('vaadin23-month-calendar').ignoreTaps).to.be.true;
           done();
         });
 
@@ -296,9 +296,9 @@
   describe('scrollToDate', () => {
     beforeEach(async () => {
       overlay = fixtureSync(`
-        <vaadin-date-picker-overlay-content
+        <vaadin23-date-picker-overlay-content
           style="position: absolute; top: 0;"
-        ></vaadin-date-picker-overlay-content>
+        ></vaadin23-date-picker-overlay-content>
       `);
       overlay.i18n = getDefaultI18n();
       overlay.$.monthScroller.bufferSize = 1;
