Index: test/validation.common.js
===================================================================
--- test/validation.common.js
+++ test/validation.common.js
@@ -1,16 +1,17 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { enter, fixtureSync, nextRender, nextUpdate, outsideClick } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
 import { close, open, setInputValue, waitForOverlayRender, waitForScrollToFinish } from './helpers.js';
 
-class DatePicker2016 extends customElements.get('vaadin-date-picker') {
+class DatePicker2016 extends internalCustomElements.get('vaadin24-date-picker') {
   checkValidity() {
     return new Date(this.value).getFullYear() === 2016;
   }
 }
 
-customElements.define('vaadin-date-picker-2016', DatePicker2016);
+internalCustomElements.define('vaadin24-date-picker-2016', DatePicker2016);
 
 function waitForValueChange(datePicker, callback) {
   let stub;
@@ -32,7 +33,7 @@
     let validateSpy;
 
     beforeEach(() => {
-      datePicker = document.createElement('vaadin-date-picker');
+      datePicker = document.createElement('vaadin24-date-picker');
       validateSpy = sinon.spy(datePicker, 'validate');
     });
 
@@ -84,7 +85,7 @@
     let valueChangedSpy, validateSpy, changeSpy, input;
 
     beforeEach(async () => {
-      datePicker = fixtureSync(`<vaadin-date-picker></vaadin-date-picker>`);
+      datePicker = fixtureSync(`<vaadin24-date-picker></vaadin24-date-picker>`);
       await nextRender();
       input = datePicker.inputElement;
       validateSpy = sinon.spy(datePicker, 'validate');
@@ -230,7 +231,7 @@
     let input;
 
     beforeEach(async () => {
-      datePicker = fixtureSync(`<vaadin-date-picker></vaadin-date-picker>`);
+      datePicker = fixtureSync(`<vaadin24-date-picker></vaadin24-date-picker>`);
       await nextRender();
       input = datePicker.inputElement;
       input.focus();
@@ -275,7 +276,7 @@
 
   describe('required', () => {
     beforeEach(async () => {
-      datePicker = fixtureSync(`<vaadin-date-picker required></vaadin-date-picker>`);
+      datePicker = fixtureSync(`<vaadin24-date-picker required></vaadin24-date-picker>`);
       await nextRender();
     });
 
@@ -291,7 +292,7 @@
 
   describe('min', () => {
     beforeEach(async () => {
-      datePicker = fixtureSync(`<vaadin-date-picker min="2010-01-01"></vaadin-date-picker>`);
+      datePicker = fixtureSync(`<vaadin24-date-picker min="2010-01-01"></vaadin24-date-picker>`);
       await nextRender();
     });
 
@@ -317,7 +318,7 @@
 
   describe('max', () => {
     beforeEach(async () => {
-      datePicker = fixtureSync(`<vaadin-date-picker max="2010-01-01"></vaadin-date-picker>`);
+      datePicker = fixtureSync(`<vaadin24-date-picker max="2010-01-01"></vaadin24-date-picker>`);
       await nextRender();
     });
 
@@ -343,7 +344,7 @@
 
   describe('custom validator', () => {
     beforeEach(() => {
-      datePicker = fixtureSync(`<vaadin-date-picker-2016></vaadin-date-picker-2016>`);
+      datePicker = fixtureSync(`<vaadin24-date-picker-2016></vaadin24-date-picker-2016>`);
     });
 
     it('should validate correctly with custom validator', () => {
