Index: test/value-commit.common.js
===================================================================
--- test/value-commit.common.js
+++ test/value-commit.common.js
@@ -2,7 +2,7 @@
 import { aTimeout, fixtureSync, nextRender, outsideClick, tap } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
-import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
+import { getDeepActiveElement } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 import { waitForOverlayRender, waitForScrollToFinish } from './helpers.js';
 
 function formatDateISO(date) {
@@ -49,7 +49,7 @@
   }
 
   beforeEach(async () => {
-    datePicker = fixtureSync(`<vaadin-date-picker></vaadin-date-picker>`);
+    datePicker = fixtureSync(`<vaadin24-date-picker></vaadin24-date-picker>`);
     await nextRender();
     validateSpy = sinon.spy(datePicker, 'validate').named('validateSpy');
 
