Index: test/value-commit.test.js
===================================================================
--- test/value-commit.test.js
+++ test/value-commit.test.js
@@ -3,7 +3,7 @@
 import { fixtureSync, nextRender, outsideClick, tap } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import '../src/vaadin-date-picker.js';
-import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
+import { getDeepActiveElement } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 import { untilOverlayRendered, untilOverlayScrolled } from './helpers.js';
 
 describe('value commit', () => {
@@ -43,7 +43,7 @@
   }
 
   beforeEach(async () => {
-    datePicker = fixtureSync(`<vaadin-date-picker></vaadin-date-picker>`);
+    datePicker = fixtureSync(`<vaadin24-date-picker></vaadin24-date-picker>`);
     await nextRender();
     validateSpy = sinon.spy(datePicker, 'validate').named('validateSpy');
 
