Index: test/wai-aria.test.js
===================================================================
--- test/wai-aria.test.js
+++ test/wai-aria.test.js
@@ -9,7 +9,7 @@
     let datePicker, input;
 
     beforeEach(() => {
-      datePicker = fixtureSync(`<vaadin-date-picker></vaadin-date-picker>`);
+      datePicker = fixtureSync(`<vaadin24-date-picker></vaadin24-date-picker>`);
       input = datePicker.inputElement;
     });
 
@@ -23,7 +23,7 @@
     it('should set aria-hidden on all calendars except focused one', async () => {
       await open(datePicker);
       await nextRender(datePicker);
-      const calendars = datePicker._overlayContent.querySelectorAll('vaadin-month-calendar');
+      const calendars = datePicker._overlayContent.querySelectorAll('vaadin24-month-calendar');
       calendars.forEach((calendar) => {
         const focused = calendar.shadowRoot.querySelector('[part="date"][focused]');
         expect(calendar.getAttribute('aria-hidden')).to.equal(focused ? null : 'true');
@@ -35,7 +35,7 @@
     let overlay;
 
     beforeEach(async () => {
-      overlay = fixtureSync(`<vaadin-date-picker-overlay-content></vaadin-date-picker-overlay-content>`);
+      overlay = fixtureSync(`<vaadin24-date-picker-overlay-content></vaadin24-date-picker-overlay-content>`);
       overlay.$.monthScroller.bufferSize = 0;
       overlay.$.yearScroller.bufferSize = 1;
       overlay.i18n = getDefaultI18n();
@@ -71,7 +71,7 @@
     let monthCalendar;
 
     beforeEach(async () => {
-      monthCalendar = fixtureSync(`<vaadin-month-calendar></vaadin-month-calendar>`);
+      monthCalendar = fixtureSync(`<vaadin24-month-calendar></vaadin24-month-calendar>`);
       monthCalendar.i18n = getDefaultI18n();
       monthCalendar.month = new Date(2016, 1, 1);
       await nextFrame();
