Index: test/number-field.common.js
===================================================================
--- test/number-field.common.js
+++ test/number-field.common.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { arrowDown, arrowUp, fixtureSync, nextRender, nextUpdate } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
@@ -10,16 +11,16 @@
     let tagName;
 
     beforeEach(() => {
-      numberField = fixtureSync('<vaadin-number-field></vaadin-number-field>');
+      numberField = fixtureSync('<vaadin24-number-field></vaadin24-number-field>');
       tagName = numberField.tagName.toLowerCase();
     });
 
     it('should be defined in custom element registry', () => {
-      expect(customElements.get(tagName)).to.be.ok;
+      expect(internalCustomElements.get(tagName)).to.be.ok;
     });
 
     it('should have a valid static "is" getter', () => {
-      expect(customElements.get(tagName).is).to.equal(tagName);
+      expect(internalCustomElements.get(tagName).is).to.equal(tagName);
     });
   });
 
@@ -27,7 +28,7 @@
     let input;
 
     beforeEach(async () => {
-      numberField = fixtureSync('<vaadin-number-field></vaadin-number-field>');
+      numberField = fixtureSync('<vaadin24-number-field></vaadin24-number-field>');
       await nextRender();
       input = numberField.inputElement;
     });
@@ -109,7 +110,7 @@
     let hasInputValueChangedSpy;
 
     beforeEach(async () => {
-      numberField = fixtureSync('<vaadin-number-field></vaadin-number-field>');
+      numberField = fixtureSync('<vaadin24-number-field></vaadin24-number-field>');
       await nextRender();
       hasInputValueChangedSpy = sinon.spy();
       numberField.addEventListener('has-input-value-changed', hasInputValueChangedSpy);
@@ -141,7 +142,7 @@
 
   describe('required', () => {
     beforeEach(async () => {
-      numberField = fixtureSync('<vaadin-number-field required></vaadin-number-field>');
+      numberField = fixtureSync('<vaadin24-number-field required></vaadin24-number-field>');
       await nextRender();
     });
 
