Index: theme/lumo/vaadin-time-picker-styles.js
===================================================================
--- theme/lumo/vaadin-time-picker-styles.js
+++ theme/lumo/vaadin-time-picker-styles.js
@@ -3,22 +3,22 @@
  * Copyright (c) 2018 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/theme/lumo/vaadin-input-container-styles.js';
-import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import { comboBoxItem } from '@vaadin/combo-box/theme/lumo/vaadin-combo-box-item-styles.js';
-import { comboBoxOverlay } from '@vaadin/combo-box/theme/lumo/vaadin-combo-box-overlay-styles.js';
-import { item } from '@vaadin/item/theme/lumo/vaadin-item-styles.js';
-import { inputFieldShared } from '@vaadin/vaadin-lumo-styles/mixins/input-field-shared.js';
-import { menuOverlayCore } from '@vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
-import { overlay } from '@vaadin/vaadin-lumo-styles/mixins/overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/input-container/theme/lumo/vaadin-input-container-styles.js';
+import '@scoped-vaadin/vaadin-lumo-styles/font-icons.js';
+import { comboBoxItem } from '@scoped-vaadin/combo-box/theme/lumo/vaadin-combo-box-item-styles.js';
+import { comboBoxOverlay } from '@scoped-vaadin/combo-box/theme/lumo/vaadin-combo-box-overlay-styles.js';
+import { item } from '@scoped-vaadin/item/theme/lumo/vaadin-item-styles.js';
+import { inputFieldShared } from '@scoped-vaadin/vaadin-lumo-styles/mixins/input-field-shared.js';
+import { menuOverlayCore } from '@scoped-vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
+import { overlay } from '@scoped-vaadin/vaadin-lumo-styles/mixins/overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
-registerStyles('vaadin-time-picker-item', [item, comboBoxItem], {
+registerStyles('vaadin24-time-picker-item', [item, comboBoxItem], {
   moduleId: 'lumo-time-picker-item',
 });
 
 registerStyles(
-  'vaadin-time-picker-overlay',
+  'vaadin24-time-picker-overlay',
   [
     overlay,
     menuOverlayCore,
@@ -49,4 +49,4 @@
   }
 `;
 
-registerStyles('vaadin-time-picker', [inputFieldShared, timePicker], { moduleId: 'lumo-time-picker' });
+registerStyles('vaadin24-time-picker', [inputFieldShared, timePicker], { moduleId: 'lumo-time-picker' });
