Index: form-layout/theme/lumo/vaadin-form-layout-styles.js
===================================================================
--- form-layout/theme/lumo/vaadin-form-layout-styles.js	null
+++ form-layout/theme/lumo/vaadin-form-layout-styles.js	null
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-form-layout',
+  'vaadin24-form-layout',
   css`
     :host {
       --vaadin-form-layout-column-spacing: var(--lumo-space-l);
