Index: theme/lumo/vaadin-split-layout-styles.d.ts
===================================================================
--- theme/lumo/vaadin-split-layout-styles.d.ts
+++ theme/lumo/vaadin-split-layout-styles.d.ts
@@ -1,4 +1,4 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
