Index: src/vaadin-popover-target-mixin.js
===================================================================
--- src/vaadin-popover-target-mixin.js
+++ src/vaadin-popover-target-mixin.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2024 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { microTask } from '@vaadin/component-base/src/async.js';
-import { Debouncer } from '@vaadin/component-base/src/debounce.js';
+import { microTask } from '@scoped-vaadin/component-base/src/async.js';
+import { Debouncer } from '@scoped-vaadin/component-base/src/debounce.js';
 
 /**
  * A mixin providing popover target functionality.
