Index: src/vaadin-popover-overlay-mixin.js
===================================================================
--- src/vaadin-popover-overlay-mixin.js
+++ src/vaadin-popover-overlay-mixin.js
@@ -3,9 +3,9 @@
  * Copyright (c) 2022 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { OverlayMixin } from '@vaadin/overlay/src/vaadin-overlay-mixin.js';
-import { PositionMixin } from '@vaadin/overlay/src/vaadin-overlay-position-mixin.js';
-import { setNestedOverlay } from '@vaadin/overlay/src/vaadin-overlay-stack-mixin.js';
+import { OverlayMixin } from '@scoped-vaadin/overlay/src/vaadin-overlay-mixin.js';
+import { PositionMixin } from '@scoped-vaadin/overlay/src/vaadin-overlay-position-mixin.js';
+import { setNestedOverlay } from '@scoped-vaadin/overlay/src/vaadin-overlay-stack-mixin.js';
 
 /**
  * Returns the closest parent overlay for given node, if any.
@@ -54,7 +54,7 @@
      * @return {string}
      */
     get _tagNamePrefix() {
-      return 'vaadin-popover';
+      return 'vaadin24-popover';
     }
 
     /**
@@ -72,7 +72,7 @@
       if (this.owner) {
         const style = getComputedStyle(this.owner);
         ['top', 'bottom', 'start', 'end'].forEach((prop) => {
-          const propertyName = `--${this._tagNamePrefix}-offset-${prop}`;
+          const propertyName = `--${this._tagNamePrefix.replace('vaadin24', 'vaadin')}-offset-${prop}`;
           this.style.setProperty(propertyName, style.getPropertyValue(propertyName));
         });
       }
