Index: vaadin-material-styles/mixins/input-field-shared.js
===================================================================
--- vaadin-material-styles/mixins/input-field-shared.js	null
+++ vaadin-material-styles/mixins/input-field-shared.js	null
@@ -6,7 +6,7 @@
 import '../color.js';
 import '../font-icons.js';
 import '../typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { fieldButton } from './field-button.js';
 import { helper } from './helper.js';
 import { requiredField } from './required-field.js';
