Index: vaadin-material-styles/mixins/required-field.js
===================================================================
--- vaadin-material-styles/mixins/required-field.js
+++ vaadin-material-styles/mixins/required-field.js
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import '../color.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const requiredField = css`
   [part='label'] {
