Index: theme/material/vaadin-context-menu-styles.js
===================================================================
--- theme/material/vaadin-context-menu-styles.js
+++ theme/material/vaadin-context-menu-styles.js
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { menuOverlay } from '@vaadin/vaadin-material-styles/mixins/menu-overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { menuOverlay } from '@scoped-vaadin/vaadin-material-styles/mixins/menu-overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const contextMenuOverlay = css`
   [part='overlay'] {
@@ -11,12 +11,12 @@
   }
 `;
 
-registerStyles('vaadin-context-menu-overlay', [menuOverlay, contextMenuOverlay], {
+registerStyles('vaadin23-context-menu-overlay', [menuOverlay, contextMenuOverlay], {
   moduleId: 'material-context-menu-overlay',
 });
 
 registerStyles(
-  'vaadin-context-menu-list-box',
+  'vaadin23-context-menu-list-box',
   css`
     [part='items'] ::slotted(.vaadin-menu-item:not(hr)) {
       min-height: 36px;
@@ -48,7 +48,7 @@
 );
 
 registerStyles(
-  'vaadin-context-menu-item',
+  'vaadin23-context-menu-item',
   css`
     :host(.vaadin-menu-item.vaadin-context-menu-parent-item)::after {
       font-family: material-icons;
