Index: context-menu/theme/material/vaadin-context-menu-overlay-styles.js
===================================================================
--- context-menu/theme/material/vaadin-context-menu-overlay-styles.js	null
+++ context-menu/theme/material/vaadin-context-menu-overlay-styles.js	null
@@ -1,5 +1,5 @@
-import { menuOverlay } from '@vaadin/vaadin-material-styles/mixins/menu-overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { menuOverlay } from '@scoped-vaadin/vaadin-material-styles/mixins/menu-overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const contextMenuOverlay = css`
   [part='overlay'] {
@@ -8,7 +8,7 @@
   }
 `;
 
-registerStyles('vaadin-context-menu-overlay', [menuOverlay, contextMenuOverlay], {
+registerStyles('vaadin24-context-menu-overlay', [menuOverlay, contextMenuOverlay], {
   moduleId: 'material-context-menu-overlay',
 });
 
