Index: context-menu/theme/material/vaadin-context-menu-item-styles.js
===================================================================
--- context-menu/theme/material/vaadin-context-menu-item-styles.js
+++ context-menu/theme/material/vaadin-context-menu-item-styles.js
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { item } from '@vaadin/item/theme/material/vaadin-item-styles.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { item } from '@scoped-vaadin/item/theme/material/vaadin-item-styles.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const contextMenuItem = css`
   :host([aria-haspopup='true'])::after {
@@ -31,6 +31,6 @@
   }
 `;
 
-registerStyles('vaadin-context-menu-item', [item, contextMenuItem], { moduleId: 'material-context-menu-item' });
+registerStyles('vaadin24-context-menu-item', [item, contextMenuItem], { moduleId: 'material-context-menu-item' });
 
 export { contextMenuItem };
