Index: theme/material/vaadin-avatar-group.d.ts
===================================================================
--- theme/material/vaadin-avatar-group.d.ts
+++ theme/material/vaadin-avatar-group.d.ts
@@ -1,3 +1,3 @@
-import '@vaadin/avatar/theme/material/vaadin-avatar.js';
+import '@scoped-vaadin/avatar/theme/material/vaadin-avatar.js';
 import './vaadin-avatar-group-styles.js';
 import '../../src/vaadin-avatar-group.js';
