Index: test/dir-mixin.test.js
===================================================================
--- test/dir-mixin.test.js
+++ test/dir-mixin.test.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { nextRender, nextUpdate } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
@@ -12,7 +13,7 @@
   }
 }
 
-customElements.define(DirMixinPolymerElement.is, DirMixinPolymerElement);
+internalCustomElements.define(DirMixinPolymerElement.is, DirMixinPolymerElement);
 
 class DirMixinLitElement extends DirMixin(PolylitMixin(LitElement)) {
   static get is() {
@@ -20,7 +21,7 @@
   }
 }
 
-customElements.define(DirMixinLitElement.is, DirMixinLitElement);
+internalCustomElements.define(DirMixinLitElement.is, DirMixinLitElement);
 
 const runTests = (baseClass) => {
   const tag = `dir-mixin-${baseClass}-element`;
