Index: test/focus-mixin.test.js
===================================================================
--- test/focus-mixin.test.js
+++ test/focus-mixin.test.js
@@ -1,9 +1,10 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, focusin, focusout, keyDownOn, mousedown } from '@vaadin/testing-helpers';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
 import { FocusMixin } from '../src/focus-mixin.js';
 
-customElements.define(
+internalCustomElements.define(
   'focus-mixin-element',
   class extends FocusMixin(PolymerElement) {
     static get template() {
