Index: test/overflow-controller.test.js
===================================================================
--- test/overflow-controller.test.js
+++ test/overflow-controller.test.js
@@ -1,10 +1,11 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextFrame, nextRender, nextResize } from '@vaadin/testing-helpers';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
 import { ControllerMixin } from '../src/controller-mixin.js';
 import { OverflowController } from '../src/overflow-controller.js';
 
-customElements.define(
+internalCustomElements.define(
   'overflow-element',
   class OverflowElement extends ControllerMixin(PolymerElement) {
     static get template() {
@@ -30,7 +31,7 @@
   },
 );
 
-customElements.define(
+internalCustomElements.define(
   'overflow-wrapper-element',
   class OverflowWrapperElement extends ControllerMixin(PolymerElement) {
     static get template() {
