Index: test/overflow-controller.test.js
===================================================================
--- test/overflow-controller.test.js
+++ test/overflow-controller.test.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextFrame, nextRender } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
@@ -25,7 +26,7 @@
   await onceInvoked(controller, '__updateOverflow');
 }
 
-customElements.define(
+internalCustomElements.define(
   'overflow-element',
   class OverflowElement extends ControllerMixin(PolymerElement) {
     static get template() {
@@ -51,7 +52,7 @@
   },
 );
 
-customElements.define(
+internalCustomElements.define(
   'overflow-wrapper-element',
   class OverflowWrapperElement extends ControllerMixin(PolymerElement) {
     static get template() {
