Index: test/slot-controller.test.js
===================================================================
--- test/slot-controller.test.js
+++ test/slot-controller.test.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
@@ -15,7 +16,7 @@
   }
 }
 
-customElements.define('slot-controller-element', SlotControllerHost);
+internalCustomElements.define('slot-controller-element', SlotControllerHost);
 
 describe('slot-controller', () => {
   let element, child, controller;
@@ -128,7 +129,7 @@
 
       const initializeSpy = sinon.spy();
 
-      customElements.define(
+      internalCustomElements.define(
         'default-slot-element',
         class extends SlotControllerHost {
           ready() {
@@ -140,7 +141,7 @@
         },
       );
 
-      customElements.define(
+      internalCustomElements.define(
         'custom-default-wrapper',
         class extends LitElement {
           render() {
