Index: test/tooltip-controller.test.js
===================================================================
--- test/tooltip-controller.test.js
+++ test/tooltip-controller.test.js
@@ -20,10 +20,10 @@
       host = fixtureSync(`
         <${tag}>
           <div>Target</div>
-          <vaadin-tooltip slot="tooltip"></vaadin-tooltip>
+          <vaadin24-tooltip slot="tooltip"></vaadin24-tooltip>
         </${tag}>
       `);
-      tooltip = host.querySelector('vaadin-tooltip');
+      tooltip = host.querySelector('vaadin24-tooltip');
       controller = new TooltipController(host);
       host.addController(controller);
     });
@@ -94,7 +94,7 @@
       controller = new TooltipController(host);
       host.addController(controller);
 
-      tooltip = document.createElement('vaadin-tooltip');
+      tooltip = document.createElement('vaadin24-tooltip');
       tooltip.setAttribute('slot', 'tooltip');
     });
 
