Index: test/dom/vaadin-upload.test.js
===================================================================
--- test/dom/vaadin-upload.test.js
+++ test/dom/vaadin-upload.test.js
@@ -2,11 +2,11 @@
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import '../../src/vaadin-upload.js';
 
-describe('vaadin-upload', () => {
+describe('vaadin24-upload', () => {
   let upload;
 
   beforeEach(async () => {
-    upload = fixtureSync('<vaadin-upload></vaadin-upload>');
+    upload = fixtureSync('<vaadin24-upload></vaadin24-upload>');
     upload.files = [
       { name: 'Annual Report.docx', complete: true },
       {
@@ -22,7 +22,7 @@
   describe('host', () => {
     const SNAPSHOT_CONFIG = {
       // Exclude inline style as we are not testing
-      // the `vaadin-progress-bar` internal logic.
+      // the `vaadin24-progress-bar` internal logic.
       ignoreAttributes: ['style'],
     };
 
