Index: test/dom/tab.test.js
===================================================================
--- test/dom/tab.test.js
+++ test/dom/tab.test.js
@@ -2,11 +2,11 @@
 import { fixtureSync } from '@vaadin/testing-helpers';
 import '../../src/vaadin-tab.js';
 
-describe('vaadin-tab', () => {
+describe('vaadin24-tab', () => {
   let tab;
 
   beforeEach(() => {
-    tab = fixtureSync('<vaadin-tab></vaadin-tab>');
+    tab = fixtureSync('<vaadin24-tab></vaadin24-tab>');
   });
 
   it('default', async () => {
