Index: test/avatar-group-test-styles.js
===================================================================
--- test/avatar-group-test-styles.js
+++ test/avatar-group-test-styles.js
@@ -1,7 +1,7 @@
-import { addGlobalThemeStyles, css, registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { addGlobalThemeStyles, css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/register-styles.js';
 
 registerStyles(
-  'vaadin-avatar-group',
+  'vaadin24-avatar-group',
   css`
     :host {
       --vaadin-avatar-size: 2.25rem;
