Index: test/visual/lumo/hide-scrollbar.js
===================================================================
--- test/visual/lumo/hide-scrollbar.js
+++ test/visual/lumo/hide-scrollbar.js
@@ -1,8 +1,8 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /* Hide scrollbars */
 registerStyles(
-  'vaadin-scroller',
+  'vaadin24-scroller',
   css`
     :host([theme~='overflow-indicators'])::-webkit-scrollbar {
       display: none;
