Index: test/dom/month-calendar.test.js
===================================================================
--- test/dom/month-calendar.test.js
+++ test/dom/month-calendar.test.js
@@ -1,15 +1,15 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import '../../src/vaadin-month-calendar.js';
-import { resetUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
+import { resetUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
 import { getDefaultI18n } from '../helpers.js';
 
-describe('vaadin-month-calendar', () => {
+describe('vaadin23-month-calendar', () => {
   let monthCalendar;
 
   beforeEach(async () => {
     resetUniqueId();
-    monthCalendar = fixtureSync('<vaadin-month-calendar></vaadin-month-calendar>');
+    monthCalendar = fixtureSync('<vaadin23-month-calendar></vaadin23-month-calendar>');
     monthCalendar.i18n = getDefaultI18n();
     monthCalendar.month = new Date(2016, 1, 1);
     await nextFrame();
