Index: accordion/theme/material/vaadin-accordion-heading-styles.js
===================================================================
--- accordion/theme/material/vaadin-accordion-heading-styles.js	null
+++ accordion/theme/material/vaadin-accordion-heading-styles.js	null
@@ -1,6 +1,6 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import { detailsSummary } from '@vaadin/details/theme/material/vaadin-details-summary-styles.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import { detailsSummary } from '@scoped-vaadin/details/theme/material/vaadin-details-summary-styles.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const accordionHeading = css`
   :host(:not([opened]))::after {
@@ -39,6 +39,6 @@
   }
 `;
 
-registerStyles('vaadin-accordion-heading', [detailsSummary, accordionHeading], {
+registerStyles('vaadin24-accordion-heading', [detailsSummary, accordionHeading], {
   moduleId: 'material-accordion-heading',
 });
