Index: accordion/theme/material/vaadin-accordion-panel-styles.js
===================================================================
--- accordion/theme/material/vaadin-accordion-panel-styles.js
+++ accordion/theme/material/vaadin-accordion-panel-styles.js
@@ -1,6 +1,6 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import { details } from '@vaadin/details/theme/material/vaadin-details-styles.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import { details } from '@scoped-vaadin/details/theme/material/vaadin-details-styles.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const accordionPanel = css`
   :host([opened]:not(:first-child)) {
@@ -12,6 +12,6 @@
   }
 `;
 
-registerStyles('vaadin-accordion-panel', [details, accordionPanel], {
+registerStyles('vaadin24-accordion-panel', [details, accordionPanel], {
   moduleId: 'material-accordion-panel',
 });
