Index: theme/material/vaadin-lit-integer-field.d.ts
===================================================================
--- theme/material/vaadin-lit-integer-field.d.ts
+++ theme/material/vaadin-lit-integer-field.d.ts
@@ -1,2 +1,2 @@
-import '@vaadin/number-field/theme/material/vaadin-lit-number-field.js';
+import '@scoped-vaadin/number-field/theme/material/vaadin-lit-number-field.js';
 import '../../src/vaadin-lit-integer-field.js';
