Index: test/dom/message-list.test.js
===================================================================
--- test/dom/message-list.test.js
+++ test/dom/message-list.test.js
@@ -2,11 +2,11 @@
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import '../../src/vaadin-message-list.js';
 
-describe('vaadin-message-list', () => {
+describe('vaadin23-message-list', () => {
   let list;
 
   beforeEach(() => {
-    list = fixtureSync('<vaadin-message-list></vaadin-message-list>');
+    list = fixtureSync('<vaadin23-message-list></vaadin23-message-list>');
   });
 
   it('default', async () => {
