Index: test/dom/message.test.js
===================================================================
--- test/dom/message.test.js
+++ test/dom/message.test.js
@@ -2,7 +2,7 @@
 import { fixtureSync } from '@vaadin/testing-helpers';
 import '../../src/vaadin-message.js';
 
-describe('vaadin-message', () => {
+describe('vaadin24-message', () => {
   const SNAPSHOT_CONFIG = {
     // Exclude custom CSS property set as inline style,
     // as corresponding logic is covered by unit tests.
@@ -12,7 +12,7 @@
   let message;
 
   beforeEach(() => {
-    message = fixtureSync('<vaadin-message></vaadin-message>');
+    message = fixtureSync('<vaadin24-message></vaadin24-message>');
   });
 
   it('default', async () => {
