Index: test/dom/message.test.js
===================================================================
--- test/dom/message.test.js
+++ test/dom/message.test.js
@@ -2,11 +2,11 @@
 import { fixtureSync } from '@vaadin/testing-helpers';
 import '../../src/vaadin-message.js';
 
-describe('vaadin-message', () => {
+describe('vaadin24-message', () => {
   let message;
 
   beforeEach(() => {
-    message = fixtureSync('<vaadin-message></vaadin-message>');
+    message = fixtureSync('<vaadin24-message></vaadin24-message>');
   });
 
   it('default', async () => {
