Index: src/vaadin-split-layout-mixin.js
===================================================================
--- src/vaadin-split-layout-mixin.js
+++ src/vaadin-split-layout-mixin.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2016 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { addListener } from '@vaadin/component-base/src/gestures.js';
+import { addListener } from '@scoped-vaadin/component-base/src/gestures.js';
 
 /**
  * @polymerMixin
