Index: a11y-base/src/active-mixin.js
===================================================================
--- a11y-base/src/active-mixin.js
+++ a11y-base/src/active-mixin.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { addListener } from '@vaadin/component-base/src/gestures.js';
+import { addListener } from '@scoped-vaadin/component-base/src/gestures.js';
 import { DisabledMixin } from './disabled-mixin.js';
 import { KeyboardMixin } from './keyboard-mixin.js';
 
@@ -57,7 +57,7 @@
       // When the element is disconnecting from the DOM at the moment being active,
       // the `active` attribute needs to be manually removed from the element.
       // Otherwise, it will preserve on the element until the element is activated once again.
-      // The case reproduces for `<vaadin-date-picker>` when closing on `Cancel` or `Today` click.
+      // The case reproduces for `<vaadin24-date-picker>` when closing on `Cancel` or `Today` click.
       this._setActive(false);
     }
 
