Index: a11y-base/src/list-mixin.js
===================================================================
--- a11y-base/src/list-mixin.js	null
+++ a11y-base/src/list-mixin.js	null
@@ -3,11 +3,11 @@
  * Copyright (c) 2017 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { timeOut } from '@vaadin/component-base/src/async.js';
-import { Debouncer } from '@vaadin/component-base/src/debounce.js';
-import { getNormalizedScrollLeft, setNormalizedScrollLeft } from '@vaadin/component-base/src/dir-utils.js';
-import { getFlattenedElements } from '@vaadin/component-base/src/dom-utils.js';
-import { SlotObserver } from '@vaadin/component-base/src/slot-observer.js';
+import { timeOut } from '@scoped-vaadin/component-base/src/async.js';
+import { Debouncer } from '@scoped-vaadin/component-base/src/debounce.js';
+import { getNormalizedScrollLeft, setNormalizedScrollLeft } from '@scoped-vaadin/component-base/src/dir-utils.js';
+import { getFlattenedElements } from '@scoped-vaadin/component-base/src/dom-utils.js';
+import { SlotObserver } from '@scoped-vaadin/component-base/src/slot-observer.js';
 import { KeyboardDirectionMixin } from './keyboard-direction-mixin.js';
 
 /**
@@ -60,7 +60,7 @@
          *
          * The item elements must implement `Vaadin.ItemMixin`.
          *
-         * Note: unlike `<vaadin-combo-box>`, this property is read-only,
+         * Note: unlike `<vaadin24-combo-box>`, this property is read-only,
          * so if you want to provide items by iterating array of data,
          * you have to use `dom-repeat` and place it to the light DOM.
          * @type {!Array<!Element> | undefined}
