Index: test/markdown.test.js
===================================================================
--- test/markdown.test.js
+++ test/markdown.test.js
@@ -1,18 +1,19 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextUpdate } from '@vaadin/testing-helpers';
 import '../src/vaadin-markdown.js';
 
-describe('vaadin-markdown', () => {
+describe('vaadin24-markdown', () => {
   let element;
 
   beforeEach(async () => {
-    element = fixtureSync('<vaadin-markdown></vaadin-markdown>');
+    element = fixtureSync('<vaadin24-markdown></vaadin24-markdown>');
     await nextUpdate(element);
   });
 
   it('should be defined', () => {
-    expect(element.localName).to.equal('vaadin-markdown');
-    expect(window.customElements.get('vaadin-markdown')).to.be.ok;
+    expect(element.localName).to.equal('vaadin24-markdown');
+    expect(window.internalCustomElements.get('vaadin24-markdown')).to.be.ok;
   });
 
   it('should be hidden', () => {
