Index: test/dom/text-area.test.js
===================================================================
--- test/dom/text-area.test.js
+++ test/dom/text-area.test.js
@@ -1,14 +1,14 @@
 import { expect } from '@esm-bundle/chai';
 import { aTimeout, fixtureSync } from '@vaadin/testing-helpers';
 import '../../src/vaadin-text-area.js';
-import { resetUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
+import { resetUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
 
-describe('vaadin-text-area', () => {
+describe('vaadin24-text-area', () => {
   let textArea;
 
   beforeEach(() => {
     resetUniqueId();
-    textArea = fixtureSync('<vaadin-text-area></vaadin-text-area>');
+    textArea = fixtureSync('<vaadin24-text-area></vaadin24-text-area>');
   });
 
   describe('host', () => {
