Index: test/item.test.js
===================================================================
--- test/item.test.js
+++ test/item.test.js
@@ -1,21 +1,22 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync } from '@vaadin/testing-helpers';
 import '../vaadin-item.js';
 
-describe('vaadin-item', () => {
+describe('vaadin23-item', () => {
   let item, tagName;
 
   beforeEach(() => {
-    item = fixtureSync('<vaadin-item>label</vaadin-item>');
+    item = fixtureSync('<vaadin23-item>label</vaadin23-item>');
     tagName = item.tagName.toLowerCase();
   });
 
   it('should be defined in custom element registry', () => {
-    expect(customElements.get(tagName)).to.be.ok;
+    expect(internalCustomElements.get(tagName)).to.be.ok;
   });
 
   it('should have a valid static "is" getter', () => {
-    expect(customElements.get(tagName).is).to.equal(tagName);
+    expect(internalCustomElements.get(tagName).is).to.equal(tagName);
   });
 
   it('should extend item-mixin', () => {
