Index: test/item-mixin.test.js
===================================================================
--- test/item-mixin.test.js
+++ test/item-mixin.test.js
@@ -16,8 +16,8 @@
   tabKeyDown,
 } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
 import { ItemMixin } from '../src/vaadin-item-mixin.js';
 
 const runTests = (defineHelper, baseMixin) => {
