Index: test/visual/material/overlay.test.js
===================================================================
--- test/visual/material/overlay.test.js
+++ test/visual/material/overlay.test.js
@@ -11,11 +11,11 @@
 
     element = fixtureSync(
       `
-        <vaadin-overlay>
+        <vaadin23-overlay>
           <template>
             <div>Simple overlay with text</div>
           </template>
-        </vaadin-overlay>
+        </vaadin23-overlay>
       `,
       div,
     );
