Index: theme/lumo/vaadin-date-picker-year-styles.js
===================================================================
--- theme/lumo/vaadin-date-picker-year-styles.js
+++ theme/lumo/vaadin-date-picker-year-styles.js
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-date-picker-year',
+  'vaadin24-date-picker-year',
   css`
     :host([current]) [part='year-number'] {
       color: var(--lumo-primary-text-color);
