Index: date-picker/theme/lumo/vaadin-date-picker-overlay-styles.js
===================================================================
--- date-picker/theme/lumo/vaadin-date-picker-overlay-styles.js	null
+++ date-picker/theme/lumo/vaadin-date-picker-overlay-styles.js	null
@@ -1,7 +1,7 @@
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { menuOverlay } from '@vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { menuOverlay } from '@scoped-vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const datePickerOverlay = css`
   [part='overlay'] {
@@ -53,6 +53,6 @@
   }
 `;
 
-registerStyles('vaadin-date-picker-overlay', [menuOverlay, datePickerOverlay], {
+registerStyles('vaadin24-date-picker-overlay', [menuOverlay, datePickerOverlay], {
   moduleId: 'lumo-date-picker-overlay',
 });
