Index: theme/lumo/vaadin-date-picker-styles.js
===================================================================
--- theme/lumo/vaadin-date-picker-styles.js
+++ theme/lumo/vaadin-date-picker-styles.js
@@ -1,7 +1,7 @@
-import '@vaadin/input-container/theme/lumo/vaadin-input-container-styles.js';
-import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import { inputFieldShared } from '@vaadin/vaadin-lumo-styles/mixins/input-field-shared.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/input-container/theme/lumo/vaadin-input-container-styles.js';
+import '@scoped-vaadin/vaadin-lumo-styles/font-icons.js';
+import { inputFieldShared } from '@scoped-vaadin/vaadin-lumo-styles/mixins/input-field-shared.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const datePicker = css`
   [part='toggle-button']::before {
@@ -27,4 +27,4 @@
   }
 `;
 
-registerStyles('vaadin-date-picker', [inputFieldShared, datePicker], { moduleId: 'lumo-date-picker' });
+registerStyles('vaadin24-date-picker', [inputFieldShared, datePicker], { moduleId: 'lumo-date-picker' });
