Index: test/visual/material/tabsheet.test.js
===================================================================
--- test/visual/material/tabsheet.test.js
+++ test/visual/material/tabsheet.test.js
@@ -1,6 +1,6 @@
 import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/tabs/test/visual/not-animated-styles.js';
+import '@scoped-vaadin/tabs/test/visual/not-animated-styles.js';
 import '../../../theme/material/vaadin-tabsheet.js';
 
 describe('tabsheet', () => {
@@ -25,15 +25,15 @@
         beforeEach(() => {
           element = fixtureSync(
             `
-            <vaadin-tabsheet>
+            <vaadin24-tabsheet>
               <div slot="prefix">Prefix</div>
               <div slot="suffix">Suffix</div>
 
-              <vaadin-tabs slot="tabs">
-                <vaadin-tab id="tab-1">Tab 1</vaadin-tab>
-                <vaadin-tab id="tab-2">Tab 2</vaadin-tab>
-                <vaadin-tab id="tab-3">Tab 3</vaadin-tab>
-              </vaadin-tabs>
+              <vaadin24-tabs slot="tabs">
+                <vaadin24-tab id="tab-1">Tab 1</vaadin24-tab>
+                <vaadin24-tab id="tab-2">Tab 2</vaadin24-tab>
+                <vaadin24-tab id="tab-3">Tab 3</vaadin24-tab>
+              </vaadin24-tabs>
 
               <div tab="tab-1">
                 Odio quis mi. Aliquam massa pede, pharetra quis, tincidunt quis, fringilla at, mauris. Vestibulum a massa.
@@ -45,7 +45,7 @@
               </div>
               <div tab="tab-2">Panel 2</div>
               <div tab="tab-3">Panel 3</div>
-            </vaadin-tabsheet>
+            </vaadin24-tabsheet>
             `,
             div,
           );
