Index: src/vaadin-lit-confirm-dialog.js
===================================================================
--- src/vaadin-lit-confirm-dialog.js
+++ src/vaadin-lit-confirm-dialog.js
@@ -3,18 +3,18 @@
  * Copyright (c) 2018 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/button/src/vaadin-lit-button.js';
+import '@scoped-vaadin/button/src/vaadin-lit-button.js';
 import './vaadin-lit-confirm-dialog-overlay.js';
 import { css, html, LitElement } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemePropertyMixin } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemePropertyMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 import { ConfirmDialogMixin } from './vaadin-confirm-dialog-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-confirm-dialog>` web component.
+ * LitElement based version of `<vaadin24-confirm-dialog>` web component.
  *
  * ## Disclaimer
  *
@@ -29,7 +29,7 @@
  */
 class ConfirmDialog extends ConfirmDialogMixin(ElementMixin(ThemePropertyMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-confirm-dialog';
+    return 'vaadin24-confirm-dialog';
   }
 
   static get styles() {
@@ -44,7 +44,7 @@
   /** @protected */
   render() {
     return html`
-      <vaadin-confirm-dialog-dialog
+      <vaadin24-confirm-dialog-dialog
         id="dialog"
         .opened="${this.opened}"
         .overlayClass="${this.overlayClass}"
@@ -55,7 +55,7 @@
         .contentHeight="${this._contentHeight}"
         .contentWidth="${this._contentWidth}"
         @opened-changed="${this._onOpenedChanged}"
-      ></vaadin-confirm-dialog-dialog>
+      ></vaadin24-confirm-dialog-dialog>
 
       <div hidden>
         <slot name="header"></slot>
