Index: src/vaadin-confirm-dialog-overlay.js
===================================================================
--- src/vaadin-confirm-dialog-overlay.js
+++ src/vaadin-confirm-dialog-overlay.js
@@ -4,24 +4,24 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { OverlayClassMixin } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import { DialogBaseMixin } from '@vaadin/dialog/src/vaadin-dialog-base-mixin.js';
-import { dialogOverlay } from '@vaadin/dialog/src/vaadin-dialog-styles.js';
-import { OverlayMixin } from '@vaadin/overlay/src/vaadin-overlay-mixin.js';
-import { overlayStyles } from '@vaadin/overlay/src/vaadin-overlay-styles.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
-import { ThemePropertyMixin } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { OverlayClassMixin } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import { DialogBaseMixin } from '@scoped-vaadin/dialog/src/vaadin-dialog-base-mixin.js';
+import { dialogOverlay } from '@scoped-vaadin/dialog/src/vaadin-dialog-styles.js';
+import { OverlayMixin } from '@scoped-vaadin/overlay/src/vaadin-overlay-mixin.js';
+import { overlayStyles } from '@scoped-vaadin/overlay/src/vaadin-overlay-styles.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ThemePropertyMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 import { ConfirmDialogBaseMixin } from './vaadin-confirm-dialog-base-mixin.js';
 import { confirmDialogOverlay } from './vaadin-confirm-dialog-overlay-styles.js';
 
-registerStyles('vaadin-confirm-dialog-overlay', [overlayStyles, dialogOverlay, confirmDialogOverlay], {
+registerStyles('vaadin24-confirm-dialog-overlay', [overlayStyles, dialogOverlay, confirmDialogOverlay], {
   moduleId: 'vaadin-confirm-dialog-overlay-styles',
 });
 
 /**
- * An element used internally by `<vaadin-confirm-dialog>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-confirm-dialog>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -32,7 +32,7 @@
  */
 class ConfirmDialogOverlay extends OverlayMixin(DirMixin(ThemableMixin(PolymerElement))) {
   static get is() {
-    return 'vaadin-confirm-dialog-overlay';
+    return 'vaadin24-confirm-dialog-overlay';
   }
 
   static get template() {
@@ -76,14 +76,14 @@
 defineCustomElement(ConfirmDialogOverlay);
 
 /**
- * An element used internally by `<vaadin-confirm-dialog>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-confirm-dialog>`. Not intended to be used separately.
  * @private
  */
 class ConfirmDialogDialog extends ConfirmDialogBaseMixin(
   DialogBaseMixin(OverlayClassMixin(ThemePropertyMixin(PolymerElement))),
 ) {
   static get is() {
-    return 'vaadin-confirm-dialog-dialog';
+    return 'vaadin24-confirm-dialog-dialog';
   }
 
   static get template() {
@@ -94,7 +94,7 @@
         }
       </style>
 
-      <vaadin-confirm-dialog-overlay
+      <vaadin24-confirm-dialog-overlay
         id="overlay"
         opened="[[opened]]"
         on-opened-changed="_onOverlayOpened"
@@ -107,7 +107,7 @@
         aria-label$="[[ariaLabel]]"
         restore-focus-on-close
         focus-trap
-      ></vaadin-confirm-dialog-overlay>
+      ></vaadin24-confirm-dialog-overlay>
     `;
   }
 }
