Index: src/vaadin-confirm-dialog.js
===================================================================
--- src/vaadin-confirm-dialog.js
+++ src/vaadin-confirm-dialog.js
@@ -3,31 +3,31 @@
  * Copyright (c) 2018 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/button/src/vaadin-button.js';
+import '@scoped-vaadin/button/src/vaadin-button.js';
 import './vaadin-confirm-dialog-overlay.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemePropertyMixin } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemePropertyMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 import { ConfirmDialogMixin } from './vaadin-confirm-dialog-mixin.js';
 
 /**
- * `<vaadin-confirm-dialog>` is a Web Component for showing alerts and asking for user confirmation.
+ * `<vaadin24-confirm-dialog>` is a Web Component for showing alerts and asking for user confirmation.
  *
  * ```
- * <vaadin-confirm-dialog cancel-button-visible>
+ * <vaadin24-confirm-dialog cancel-button-visible>
  *   There are unsaved changes. Do you really want to leave?
- * </vaadin-confirm-dialog>
+ * </vaadin24-confirm-dialog>
  * ```
  *
  * ### Styling
  *
- * The `<vaadin-confirm-dialog>` is not themable. Apply styles to `<vaadin-confirm-dialog-overlay>`
+ * The `<vaadin24-confirm-dialog>` is not themable. Apply styles to `<vaadin24-confirm-dialog-overlay>`
  * component and use its shadow parts for styling.
- * See [`<vaadin-overlay>`](#/elements/vaadin-overlay) for the overlay styling documentation.
+ * See [`<vaadin24-overlay>`](#/elements/vaadin-overlay) for the overlay styling documentation.
  *
- * In addition to `<vaadin-overlay>` parts, the following parts are available for theming:
+ * In addition to `<vaadin24-overlay>` parts, the following parts are available for theming:
  *
  * Part name        | Description
  * -----------------|-------------------------------------------
@@ -39,8 +39,8 @@
  * `reject-button`  | The "Reject" button wrapper
  *
  * Use `confirmTheme`, `cancelTheme` and `rejectTheme` properties to customize buttons theme.
- * Also, the `theme` attribute value set on `<vaadin-confirm-dialog>` is propagated to the
- * `<vaadin-confirm-dialog-overlay>` component.
+ * Also, the `theme` attribute value set on `<vaadin24-confirm-dialog>` is propagated to the
+ * `<vaadin24-confirm-dialog-overlay>` component.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
@@ -77,7 +77,7 @@
         }
       </style>
 
-      <vaadin-confirm-dialog-dialog
+      <vaadin24-confirm-dialog-dialog
         id="dialog"
         opened="{{opened}}"
         overlay-class="[[overlayClass]]"
@@ -87,7 +87,7 @@
         no-close-on-esc="[[noCloseOnEsc]]"
         content-height="[[_contentHeight]]"
         content-width="[[_contentWidth]]"
-      ></vaadin-confirm-dialog-dialog>
+      ></vaadin24-confirm-dialog-dialog>
 
       <div hidden>
         <slot name="header"></slot>
@@ -100,7 +100,7 @@
   }
 
   static get is() {
-    return 'vaadin-confirm-dialog';
+    return 'vaadin24-confirm-dialog';
   }
 
   /** @protected */
