Index: polymer-legacy-adapter/src/template-renderer-templatizer.js
===================================================================
--- polymer-legacy-adapter/src/template-renderer-templatizer.js	null
+++ polymer-legacy-adapter/src/template-renderer-templatizer.js	null
@@ -5,11 +5,11 @@
  */
 import { PolymerElement } from '@polymer/polymer';
 import { templatize } from '@polymer/polymer/lib/utils/templatize.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
 
 export class Templatizer extends PolymerElement {
   static get is() {
-    return 'vaadin-template-renderer-templatizer';
+    return 'vaadin24-template-renderer-templatizer';
   }
 
   static create(component, template) {
