Index: src/vaadin-combo-box-mixin.js
===================================================================
--- src/vaadin-combo-box-mixin.js
+++ src/vaadin-combo-box-mixin.js
@@ -3,14 +3,14 @@
  * Copyright (c) 2015 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { DisabledMixin } from '@vaadin/component-base/src/disabled-mixin.js';
-import { isElementFocused } from '@vaadin/component-base/src/focus-utils.js';
-import { KeyboardMixin } from '@vaadin/component-base/src/keyboard-mixin.js';
-import { processTemplates } from '@vaadin/component-base/src/templates.js';
-import { InputMixin } from '@vaadin/field-base/src/input-mixin.js';
-import { VirtualKeyboardController } from '@vaadin/field-base/src/virtual-keyboard-controller.js';
+import { isTouch } from '@scoped-vaadin/component-base/src/browser-utils.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { DisabledMixin } from '@scoped-vaadin/component-base/src/disabled-mixin.js';
+import { isElementFocused } from '@scoped-vaadin/component-base/src/focus-utils.js';
+import { KeyboardMixin } from '@scoped-vaadin/component-base/src/keyboard-mixin.js';
+import { processTemplates } from '@scoped-vaadin/component-base/src/templates.js';
+import { InputMixin } from '@scoped-vaadin/field-base/src/input-mixin.js';
+import { VirtualKeyboardController } from '@scoped-vaadin/field-base/src/virtual-keyboard-controller.js';
 import { ComboBoxPlaceholder } from './vaadin-combo-box-placeholder.js';
 
 /**
@@ -83,8 +83,8 @@
          * Custom function for rendering the content of every item.
          * Receives three arguments:
          *
-         * - `root` The `<vaadin-combo-box-item>` internal container DOM element.
-         * - `comboBox` The reference to the `<vaadin-combo-box>` element.
+         * - `root` The `<vaadin23-combo-box-item>` internal container DOM element.
+         * - `comboBox` The reference to the `<vaadin23-combo-box>` element.
          * - `model` The object with the properties related with the rendered
          *   item, contains:
          *   - `model.index` The index of the rendered item.
@@ -267,7 +267,7 @@
      * @return {string}
      */
     get _tagNamePrefix() {
-      return 'vaadin-combo-box';
+      return 'vaadin23-combo-box';
     }
 
     /**
@@ -610,7 +610,7 @@
      * @private
      */
     _onToggleButtonClick(event) {
-      // Prevent parent components such as `vaadin-grid`
+      // Prevent parent components such as `vaadin23-grid`
       // from handling the click event after it bubbles.
       event.preventDefault();
 
@@ -1338,13 +1338,13 @@
      */
 
     /**
-     * Fired after the `vaadin-combo-box-overlay` opens.
+     * Fired after the `vaadin23-combo-box-overlay` opens.
      *
      * @event vaadin-combo-box-dropdown-opened
      */
 
     /**
-     * Fired after the `vaadin-combo-box-overlay` closes.
+     * Fired after the `vaadin23-combo-box-overlay` closes.
      *
      * @event vaadin-combo-box-dropdown-closed
      */
