Index: src/vaadin-combo-box.d.ts
===================================================================
--- src/vaadin-combo-box.d.ts
+++ src/vaadin-combo-box.d.ts
@@ -3,22 +3,22 @@
  * Copyright (c) 2015 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { DisabledMixinClass } from '@vaadin/component-base/src/disabled-mixin.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { FocusMixinClass } from '@vaadin/component-base/src/focus-mixin.js';
-import type { KeyboardMixinClass } from '@vaadin/component-base/src/keyboard-mixin.js';
-import type { DelegateFocusMixinClass } from '@vaadin/field-base/src/delegate-focus-mixin.js';
-import type { DelegateStateMixinClass } from '@vaadin/field-base/src/delegate-state-mixin.js';
-import type { FieldMixinClass } from '@vaadin/field-base/src/field-mixin.js';
-import type { InputConstraintsMixinClass } from '@vaadin/field-base/src/input-constraints-mixin.js';
-import type { InputControlMixinClass } from '@vaadin/field-base/src/input-control-mixin.js';
-import type { InputMixinClass } from '@vaadin/field-base/src/input-mixin.js';
-import type { LabelMixinClass } from '@vaadin/field-base/src/label-mixin.js';
-import type { PatternMixinClass } from '@vaadin/field-base/src/pattern-mixin.js';
-import type { ValidateMixinClass } from '@vaadin/field-base/src/validate-mixin.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
-import type { ThemePropertyMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/component-base/src/disabled-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/component-base/src/focus-mixin.js';
+import type { KeyboardMixinClass } from '@scoped-vaadin/component-base/src/keyboard-mixin.js';
+import type { DelegateFocusMixinClass } from '@scoped-vaadin/field-base/src/delegate-focus-mixin.js';
+import type { DelegateStateMixinClass } from '@scoped-vaadin/field-base/src/delegate-state-mixin.js';
+import type { FieldMixinClass } from '@scoped-vaadin/field-base/src/field-mixin.js';
+import type { InputConstraintsMixinClass } from '@scoped-vaadin/field-base/src/input-constraints-mixin.js';
+import type { InputControlMixinClass } from '@scoped-vaadin/field-base/src/input-control-mixin.js';
+import type { InputMixinClass } from '@scoped-vaadin/field-base/src/input-mixin.js';
+import type { LabelMixinClass } from '@scoped-vaadin/field-base/src/label-mixin.js';
+import type { PatternMixinClass } from '@scoped-vaadin/field-base/src/pattern-mixin.js';
+import type { ValidateMixinClass } from '@scoped-vaadin/field-base/src/validate-mixin.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import type { ThemePropertyMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 import type { ComboBoxDataProviderMixinClass } from './vaadin-combo-box-data-provider-mixin.js';
 import type { ComboBoxMixinClass } from './vaadin-combo-box-mixin.js';
 import type { ComboBoxDefaultItem } from './vaadin-combo-box-mixin.js';
@@ -90,12 +90,12 @@
 }
 
 /**
- * `<vaadin-combo-box>` is a web component for choosing a value from a filterable list of options
+ * `<vaadin23-combo-box>` is a web component for choosing a value from a filterable list of options
  * presented in a dropdown overlay. The options can be provided as a list of strings or objects
  * by setting [`items`](#/elements/vaadin-combo-box#property-items) property on the element.
  *
  * ```html
- * <vaadin-combo-box id="combo-box"></vaadin-combo-box>
+ * <vaadin23-combo-box id="combo-box"></vaadin23-combo-box>
  * ```
  *
  * ```js
@@ -106,7 +106,7 @@
  *
  * ### Item rendering
  *
- * To customize the content of the `<vaadin-combo-box-item>` elements placed in the dropdown, use
+ * To customize the content of the `<vaadin23-combo-box-item>` elements placed in the dropdown, use
  * [`renderer`](#/elements/vaadin-combo-box#property-renderer) property which accepts a function.
  * The renderer function is called with `root`, `comboBox`, and `model` as arguments.
  *
@@ -141,7 +141,7 @@
  *
  * In addition to assigning an array to the items property, you can alternatively use the
  * [`dataProvider`](#/elements/vaadin-combo-box#property-dataProvider) function property.
- * The `<vaadin-combo-box>` calls this function lazily, only when it needs more data
+ * The `<vaadin23-combo-box>` calls this function lazily, only when it needs more data
  * to be displayed.
  *
  * __Note that when using function data providers, the total number of items
@@ -172,16 +172,16 @@
  * `--vaadin-combo-box-overlay-width`      | Width of the overlay       | `auto`
  * `--vaadin-combo-box-overlay-max-height` | Max height of the overlay  | `65vh`
  *
- * `<vaadin-combo-box>` provides the same set of shadow DOM parts and state attributes as `<vaadin-text-field>`.
- * See [`<vaadin-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
+ * `<vaadin23-combo-box>` provides the same set of shadow DOM parts and state attributes as `<vaadin23-text-field>`.
+ * See [`<vaadin23-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
  *
- * In addition to `<vaadin-text-field>` parts, the following parts are available for theming:
+ * In addition to `<vaadin23-text-field>` parts, the following parts are available for theming:
  *
  * Part name       | Description
  * ----------------|----------------
  * `toggle-button` | The toggle button
  *
- * In addition to `<vaadin-text-field>` state attributes, the following state attributes are available for theming:
+ * In addition to `<vaadin23-text-field>` state attributes, the following state attributes are available for theming:
  *
  * Attribute | Description | Part name
  * ----------|-------------|------------
@@ -189,18 +189,18 @@
  * `loading` | Set when new items are expected | :host
  *
  * If you want to replace the default `<input>` and its container with a custom implementation to get full control
- * over the input field, consider using the [`<vaadin-combo-box-light>`](#/elements/vaadin-combo-box-light) element.
+ * over the input field, consider using the [`<vaadin23-combo-box-light>`](#/elements/vaadin-combo-box-light) element.
  *
  * ### Internal components
  *
- * In addition to `<vaadin-combo-box>` itself, the following internal
+ * In addition to `<vaadin23-combo-box>` itself, the following internal
  * components are themable:
  *
- * - `<vaadin-combo-box-overlay>` - has the same API as [`<vaadin-overlay>`](#/elements/vaadin-overlay).
- * - `<vaadin-combo-box-item>` - has the same API as [`<vaadin-item>`](#/elements/vaadin-item).
- * - [`<vaadin-input-container>`](#/elements/vaadin-input-container) - an internal element wrapping the input.
+ * - `<vaadin23-combo-box-overlay>` - has the same API as [`<vaadin23-overlay>`](#/elements/vaadin-overlay).
+ * - `<vaadin23-combo-box-item>` - has the same API as [`<vaadin23-item>`](#/elements/vaadin-item).
+ * - [`<vaadin23-input-container>`](#/elements/vaadin-input-container) - an internal element wrapping the input.
  *
- * Note: the `theme` attribute value set on `<vaadin-combo-box>` is
+ * Note: the `theme` attribute value set on `<vaadin23-combo-box>` is
  * propagated to the internal components listed above.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/custom-theme/styling-components) documentation.
@@ -250,7 +250,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-combo-box': ComboBox;
+    'vaadin23-combo-box': ComboBox;
   }
 }
 
