Index: combo-box/src/vaadin-combo-box-overlay.js
===================================================================
--- combo-box/src/vaadin-combo-box-overlay.js	null
+++ combo-box/src/vaadin-combo-box-overlay.js	null
@@ -4,11 +4,11 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { OverlayMixin } from '@vaadin/overlay/src/vaadin-overlay-mixin.js';
-import { overlayStyles } from '@vaadin/overlay/src/vaadin-overlay-styles.js';
-import { css, registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { OverlayMixin } from '@scoped-vaadin/overlay/src/vaadin-overlay-mixin.js';
+import { overlayStyles } from '@scoped-vaadin/overlay/src/vaadin-overlay-styles.js';
+import { css, registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ComboBoxOverlayMixin } from './vaadin-combo-box-overlay-mixin.js';
 
 const comboBoxOverlayStyles = css`
@@ -23,12 +23,12 @@
   }
 `;
 
-registerStyles('vaadin-combo-box-overlay', [overlayStyles, comboBoxOverlayStyles], {
+registerStyles('vaadin24-combo-box-overlay', [overlayStyles, comboBoxOverlayStyles], {
   moduleId: 'vaadin-combo-box-overlay-styles',
 });
 
 /**
- * An element used internally by `<vaadin-combo-box>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-combo-box>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -40,7 +40,7 @@
  */
 export class ComboBoxOverlay extends ComboBoxOverlayMixin(OverlayMixin(DirMixin(ThemableMixin(PolymerElement)))) {
   static get is() {
-    return 'vaadin-combo-box-overlay';
+    return 'vaadin24-combo-box-overlay';
   }
 
   static get template() {
