Index: combo-box/src/vaadin-combo-box-scroller-mixin.js
===================================================================
--- combo-box/src/vaadin-combo-box-scroller-mixin.js
+++ combo-box/src/vaadin-combo-box-scroller-mixin.js
@@ -3,9 +3,9 @@
  * Copyright (c) 2015 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { get } from '@vaadin/component-base/src/path-utils.js';
-import { generateUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
-import { Virtualizer } from '@vaadin/component-base/src/virtualizer.js';
+import { get } from '@scoped-vaadin/component-base/src/path-utils.js';
+import { generateUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
+import { Virtualizer } from '@scoped-vaadin/component-base/src/virtualizer.js';
 import { ComboBoxPlaceholder } from './vaadin-combo-box-placeholder.js';
 
 /**
