Index: src/vaadin-combo-box-light-mixin.js
===================================================================
--- src/vaadin-combo-box-light-mixin.js
+++ src/vaadin-combo-box-light-mixin.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2015 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ValidateMixin } from '@vaadin/field-base/src/validate-mixin.js';
+import { ValidateMixin } from '@scoped-vaadin/field-base/src/validate-mixin.js';
 import { ComboBoxDataProviderMixin } from './vaadin-combo-box-data-provider-mixin.js';
 import { ComboBoxMixin } from './vaadin-combo-box-mixin.js';
 
@@ -70,7 +70,7 @@
           return input;
         }
 
-        // Support `<input>` in light DOM (e.g. `vaadin-text-field`)
+        // Support `<input>` in light DOM (e.g. `vaadin24-text-field`)
         const slottedInput = input.querySelector('input');
         if (slottedInput) {
           return slottedInput;
@@ -96,7 +96,7 @@
 
       // Wait until the slotted input DOM is ready
       queueMicrotask(() => {
-        this._setInputElement(this.querySelector('vaadin-text-field,.input'));
+        this._setInputElement(this.querySelector('vaadin24-text-field,.input'));
         this._revertInputValue();
       });
     }
