Index: test/dom/__snapshots__/menu-bar.test.snap.js
===================================================================
--- test/dom/__snapshots__/menu-bar.test.snap.js
+++ test/dom/__snapshots__/menu-bar.test.snap.js
@@ -2,22 +2,22 @@
 export const snapshots = {};
 
 snapshots["menu-bar host"] = 
-`<vaadin-menu-bar role="menubar">
-</vaadin-menu-bar>
+`<vaadin24-menu-bar role="menubar">
+</vaadin24-menu-bar>
 `;
 /* end snapshot menu-bar host */
 
 snapshots["menu-bar shadow"] = 
 `<div part="container">
-  <vaadin-menu-bar-button
+  <vaadin24-menu-bar-button
     first-visible=""
     part="menu-bar-button"
     role="menuitem"
     tabindex="0"
   >
     Home
-  </vaadin-menu-bar-button>
-  <vaadin-menu-bar-button
+  </vaadin24-menu-bar-button>
+  <vaadin24-menu-bar-button
     aria-expanded="false"
     aria-haspopup="true"
     part="menu-bar-button"
@@ -25,8 +25,8 @@
     tabindex="0"
   >
     Reports
-  </vaadin-menu-bar-button>
-  <vaadin-menu-bar-button
+  </vaadin24-menu-bar-button>
+  <vaadin24-menu-bar-button
     aria-disabled="true"
     disabled=""
     part="menu-bar-button"
@@ -34,14 +34,14 @@
     tabindex="-1"
   >
     Dashboard
-  </vaadin-menu-bar-button>
-  <vaadin-menu-bar-button
+  </vaadin24-menu-bar-button>
+  <vaadin24-menu-bar-button
     last-visible=""
     part="menu-bar-button"
     role="menuitem"
     tabindex="0"
   >
-    <vaadin-context-menu-item
+    <vaadin24-context-menu-item
       aria-selected="false"
       role="option"
       theme="menu-bar-item"
@@ -49,9 +49,9 @@
       <strong>
         Help
       </strong>
-    </vaadin-context-menu-item>
-  </vaadin-menu-bar-button>
-  <vaadin-menu-bar-button
+    </vaadin24-context-menu-item>
+  </vaadin24-menu-bar-button>
+  <vaadin24-menu-bar-button
     aria-expanded="false"
     aria-haspopup="true"
     aria-label="More options"
@@ -62,17 +62,17 @@
   >
     <div class="dots">
     </div>
-  </vaadin-menu-bar-button>
+  </vaadin24-menu-bar-button>
 </div>
-<vaadin-menu-bar-submenu is-root="">
-</vaadin-menu-bar-submenu>
+<vaadin24-menu-bar-submenu is-root="">
+</vaadin24-menu-bar-submenu>
 <slot name="tooltip">
 </slot>
 `;
 /* end snapshot menu-bar shadow */
 
 snapshots["menu-bar overlay"] = 
-`<vaadin-context-menu-overlay
+`<vaadin24-context-menu-overlay
   dir="ltr"
   id="overlay"
   opened=""
@@ -80,12 +80,12 @@
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-context-menu-list-box
+  <vaadin24-context-menu-list-box
     aria-orientation="vertical"
     class="vaadin-menu-list-box"
     role="listbox"
   >
-    <vaadin-context-menu-item
+    <vaadin24-context-menu-item
       aria-haspopup="false"
       aria-selected="false"
       class="vaadin-menu-item"
@@ -93,8 +93,8 @@
       tabindex="0"
     >
       View Reports
-    </vaadin-context-menu-item>
-    <vaadin-context-menu-item
+    </vaadin24-context-menu-item>
+    <vaadin24-context-menu-item
       aria-haspopup="false"
       aria-selected="false"
       class="vaadin-menu-item"
@@ -102,11 +102,11 @@
       tabindex="-1"
     >
       Generate Report
-    </vaadin-context-menu-item>
-  </vaadin-context-menu-list-box>
-  <vaadin-menu-bar-submenu hidden="">
-  </vaadin-menu-bar-submenu>
-</vaadin-context-menu-overlay>
+    </vaadin24-context-menu-item>
+  </vaadin24-context-menu-list-box>
+  <vaadin24-menu-bar-submenu hidden="">
+  </vaadin24-menu-bar-submenu>
+</vaadin24-context-menu-overlay>
 `;
 /* end snapshot menu-bar overlay */
 
