Index: test/typings/radio-button.types.ts
===================================================================
--- test/typings/radio-button.types.ts
+++ test/typings/radio-button.types.ts
@@ -1,16 +1,16 @@
 import '../../vaadin-radio-button.js';
 import '../../vaadin-radio-group.js';
-import type { ActiveMixinClass } from '@vaadin/a11y-base/src/active-mixin.js';
-import type { DelegateFocusMixinClass } from '@vaadin/a11y-base/src/delegate-focus-mixin.js';
-import type { DisabledMixinClass } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin.js';
-import type { KeyboardMixinClass } from '@vaadin/a11y-base/src/keyboard-mixin.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { CheckedMixinClass } from '@vaadin/field-base/src/checked-mixin.js';
-import type { FieldMixinClass } from '@vaadin/field-base/src/field-mixin.js';
-import type { LabelMixinClass } from '@vaadin/field-base/src/label-mixin.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import type { ActiveMixinClass } from '@scoped-vaadin/a11y-base/src/active-mixin.js';
+import type { DelegateFocusMixinClass } from '@scoped-vaadin/a11y-base/src/delegate-focus-mixin.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import type { KeyboardMixinClass } from '@scoped-vaadin/a11y-base/src/keyboard-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { CheckedMixinClass } from '@scoped-vaadin/field-base/src/checked-mixin.js';
+import type { FieldMixinClass } from '@scoped-vaadin/field-base/src/field-mixin.js';
+import type { LabelMixinClass } from '@scoped-vaadin/field-base/src/label-mixin.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import type { RadioGroupMixinClass } from '../../src/vaadin-radio-group-mixin.js';
 import type { RadioButtonCheckedChangedEvent } from '../../vaadin-radio-button.js';
 import type {
@@ -21,7 +21,7 @@
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const radio = document.createElement('vaadin-radio-button');
+const radio = document.createElement('vaadin24-radio-button');
 
 // Radio properties
 assertType<boolean>(radio.autofocus);
@@ -49,7 +49,7 @@
   assertType<boolean>(event.detail.value);
 });
 
-const group = document.createElement('vaadin-radio-group');
+const group = document.createElement('vaadin24-radio-group');
 
 // Group properties
 assertType<string | null | undefined>(group.name);
