Index: test/a11y.test.js
===================================================================
--- test/a11y.test.js
+++ test/a11y.test.js
@@ -7,11 +7,11 @@
 const FAKE_FILE = createFile(100000, 'application/uknown');
 
 describe('a11y', () => {
-  describe('<vaadin-upload-file>', () => {
+  describe('<vaadin23-upload-file>', () => {
     let uploadFileElement, i18n, button, name;
 
     beforeEach(() => {
-      uploadFileElement = fixtureSync(`<vaadin-upload-file></vaadin-upload-file>`);
+      uploadFileElement = fixtureSync(`<vaadin23-upload-file></vaadin23-upload-file>`);
       i18n = {
         file: {
           start: 'Start',
@@ -76,7 +76,7 @@
     });
 
     beforeEach(() => {
-      upload = fixtureSync(`<vaadin-upload></vaadin-upload>`);
+      upload = fixtureSync(`<vaadin23-upload></vaadin23-upload>`);
       clock = sinon.useFakeTimers();
     });
 
