Index: test/a11y.common.js
===================================================================
--- test/a11y.common.js
+++ test/a11y.common.js
@@ -6,11 +6,11 @@
 const FAKE_FILE = createFile(100000, 'application/uknown');
 
 describe('a11y', () => {
-  describe('<vaadin-upload-file>', () => {
+  describe('<vaadin24-upload-file>', () => {
     let uploadFileElement, i18n, button, name;
 
     beforeEach(async () => {
-      uploadFileElement = fixtureSync(`<vaadin-upload-file></vaadin-upload-file>`);
+      uploadFileElement = fixtureSync(`<vaadin24-upload-file></vaadin24-upload-file>`);
       i18n = {
         file: {
           start: 'Start',
@@ -75,7 +75,7 @@
     });
 
     beforeEach(async () => {
-      upload = fixtureSync(`<vaadin-upload></vaadin-upload>`);
+      upload = fixtureSync(`<vaadin24-upload></vaadin24-upload>`);
       await nextRender();
       clock = sinon.useFakeTimers();
     });
