Index: test/adding-files.test.js
===================================================================
--- test/adding-files.test.js
+++ test/adding-files.test.js
@@ -18,7 +18,7 @@
   const testFileSize = 512;
 
   beforeEach(async () => {
-    upload = fixtureSync(`<vaadin-upload></vaadin-upload>`);
+    upload = fixtureSync(`<vaadin24-upload></vaadin24-upload>`);
     upload.target = 'http://foo.com/bar';
     upload._createXhr = xhrCreator({ size: testFileSize, uploadTime: 200, stepTime: 50 });
     await nextRender();
