Index: test/file.test.js
===================================================================
--- test/file.test.js
+++ test/file.test.js
@@ -4,11 +4,11 @@
 import '../vaadin-upload.js';
 import { createFile } from './helpers.js';
 
-describe('<vaadin-upload-file> element', () => {
+describe('<vaadin24-upload-file> element', () => {
   let fileElement, fileObject;
 
   beforeEach(() => {
-    fileElement = fixtureSync(`<vaadin-upload-file></vaadin-upload-file>`);
+    fileElement = fixtureSync(`<vaadin24-upload-file></vaadin24-upload-file>`);
     fileObject = createFile(100000, 'application/unknown');
     fileElement.file = fileObject;
   });
