Index: test/helpers.js
===================================================================
--- test/helpers.js
+++ test/helpers.js
@@ -179,7 +179,7 @@
  * Removes a file at given index.
  */
 export function removeFile(upload, idx = 0) {
-  const files = upload.querySelectorAll('vaadin-upload-file');
+  const files = upload.querySelectorAll('vaadin24-upload-file');
   const file = files[idx];
   file.shadowRoot.querySelector('[part="remove-button"]').click();
 }
