Index: test/keyboard-navigation.test.js
===================================================================
--- test/keyboard-navigation.test.js
+++ test/keyboard-navigation.test.js
@@ -28,12 +28,12 @@
   });
 
   beforeEach(async () => {
-    uploadElement = fixtureSync(`<vaadin-upload></vaadin-upload>`);
+    uploadElement = fixtureSync(`<vaadin24-upload></vaadin24-upload>`);
     uploadElement.files = [FAKE_FILE];
 
     await nextRender();
-    uploadButton = uploadElement.querySelector('vaadin-button[slot=add-button]');
-    fileElements = uploadElement.querySelectorAll('vaadin-upload-file');
+    uploadButton = uploadElement.querySelector('vaadin24-button[slot=add-button]');
+    fileElements = uploadElement.querySelectorAll('vaadin24-upload-file');
   });
 
   afterEach(() => {
@@ -65,7 +65,7 @@
       uploadElement.files = fileList;
 
       await nextRender();
-      fileElements = uploadElement.querySelectorAll('vaadin-upload-file');
+      fileElements = uploadElement.querySelectorAll('vaadin24-upload-file');
     });
 
     it('should focus on the start button', async () => {
