Index: test/keyboard-navigation.test.js
===================================================================
--- test/keyboard-navigation.test.js
+++ test/keyboard-navigation.test.js
@@ -28,12 +28,12 @@
   });
 
   beforeEach(async () => {
-    uploadElement = fixtureSync(`<vaadin-upload></vaadin-upload>`);
+    uploadElement = fixtureSync(`<vaadin24-upload></vaadin24-upload>`);
     uploadElement.files = [FAKE_FILE];
 
     await nextRender();
 
-    fileElement = uploadElement.shadowRoot.querySelector('vaadin-upload-file');
+    fileElement = uploadElement.shadowRoot.querySelector('vaadin24-upload-file');
   });
 
   afterEach(() => {
