Index: test/upload.test.js
===================================================================
--- test/upload.test.js
+++ test/upload.test.js
@@ -9,7 +9,7 @@
   let upload, file;
 
   beforeEach(() => {
-    upload = fixtureSync(`<vaadin-upload></vaadin-upload>`);
+    upload = fixtureSync(`<vaadin23-upload></vaadin23-upload>`);
     upload.target = 'http://foo.com/bar';
     file = createFile(100000, 'application/unknown');
   });
