Index: src/vaadin-form-layout.js
===================================================================
--- src/vaadin-form-layout.js
+++ src/vaadin-form-layout.js
@@ -4,37 +4,37 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { FormLayoutMixin } from './vaadin-form-layout-mixin.js';
 import { formLayoutStyles } from './vaadin-form-layout-styles.js';
 
-registerStyles('vaadin-form-layout', formLayoutStyles, { moduleId: 'vaadin-form-layout-styles' });
+registerStyles('vaadin24-form-layout', formLayoutStyles, { moduleId: 'vaadin-form-layout-styles' });
 
 /**
- * `<vaadin-form-layout>` is a Web Component providing configurable responsive
+ * `<vaadin24-form-layout>` is a Web Component providing configurable responsive
  * layout for form elements.
  *
  * ```html
- * <vaadin-form-layout>
+ * <vaadin24-form-layout>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">First Name</label>
  *     <input class="full-width" value="Jane">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">Last Name</label>
  *     <input class="full-width" value="Doe">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">Email</label>
  *     <input class="full-width" value="jane.doe@example.com">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- * </vaadin-form-layout>
+ * </vaadin24-form-layout>
  * ```
  *
  * It supports any child elements as layout items.
@@ -51,24 +51,24 @@
  * In the example below, the first text field spans on two columns:
  *
  * ```html
- * <vaadin-form-layout>
+ * <vaadin24-form-layout>
  *
- *   <vaadin-form-item colspan="2">
+ *   <vaadin24-form-item colspan="2">
  *     <label slot="label">Address</label>
  *     <input class="full-width">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">First Name</label>
  *     <input class="full-width" value="Jane">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">Last Name</label>
  *     <input class="full-width" value="Doe">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- * </vaadin-form-layout>
+ * </vaadin24-form-layout>
  * ```
  *
  * ### Explicit New Row
@@ -76,21 +76,21 @@
  * Use the `<br>` line break element to wrap the items on a new row:
  *
  * ```html
- * <vaadin-form-layout>
+ * <vaadin24-form-layout>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">Email</label>
  *     <input class="full-width">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
  *   <br>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">Confirm Email</label>
  *     <input class="full-width">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- * </vaadin-form-layout>
+ * </vaadin24-form-layout>
  * ```
  *
  * ### Auto Responsive Mode
@@ -108,15 +108,15 @@
  * `auto-responsive` attribute:
  *
  * ```html
- * <vaadin-form-layout auto-responsive>
- *   <vaadin-text-field label="First Name"></vaadin-text-field>
- *   <vaadin-text-field label="Last Name"></vaadin-text-field>
- *   <vaadin-text-area label="Address" colspan="2"></vaadin-text-area>
- * </vaadin-form-layout>
+ * <vaadin24-form-layout auto-responsive>
+ *   <vaadin24-text-field label="First Name"></vaadin24-text-field>
+ *   <vaadin24-text-field label="Last Name"></vaadin24-text-field>
+ *   <vaadin24-text-area label="Address" colspan="2"></vaadin24-text-area>
+ * </vaadin24-form-layout>
  * ```
  *
  * You can also enable it for all instances by enabling the following feature flag
- * before `<vaadin-form-layout>` elements are added to the DOM:
+ * before `<vaadin24-form-layout>` elements are added to the DOM:
  *
  * ```js
  * window.Vaadin.featureFlags.defaultAutoResponsiveFormLayout = true;
@@ -126,24 +126,24 @@
  *
  * By default, each field is placed on a new row. To organize fields into rows, you can either:
  *
- * 1. Manually wrap fields into [`<vaadin-form-row>`](#/elements/vaadin-form-row) elements.
+ * 1. Manually wrap fields into [`<vaadin24-form-row>`](#/elements/vaadin-form-row) elements.
  *
  * 2. Enable the [`autoRows`](#/elements/vaadin-form-layout#property-autoRows) property to
  *    let Form Layout automatically arrange fields in available columns, wrapping to a new
  *    row when necessary. `<br>` elements can be used to force a new row.
  *
- * Here is an example of using `<vaadin-form-row>`:
+ * Here is an example of using `<vaadin24-form-row>`:
  *
  * ```html
- * <vaadin-form-layout auto-responsive>
- *   <vaadin-form-row>
- *     <vaadin-text-field label="First Name"></vaadin-text-field>
- *     <vaadin-text-field label="Last Name"></vaadin-text-field>
- *   </vaadin-form-row>
- *   <vaadin-form-row>
- *     <vaadin-text-area label="Address" colspan="2"></vaadin-text-area>
- *   </vaadin-form-row>
- * </vaadin-form-layout>
+ * <vaadin24-form-layout auto-responsive>
+ *   <vaadin24-form-row>
+ *     <vaadin24-text-field label="First Name"></vaadin24-text-field>
+ *     <vaadin24-text-field label="Last Name"></vaadin24-text-field>
+ *   </vaadin24-form-row>
+ *   <vaadin24-form-row>
+ *     <vaadin24-text-area label="Address" colspan="2"></vaadin24-text-area>
+ *   </vaadin24-form-row>
+ * </vaadin24-form-layout>
  * ```
  *
  * #### Expanding Columns and Fields
@@ -160,29 +160,29 @@
  * #### Customizing Label Position
  *
  * By default, Form Layout displays labels above the fields. To position labels beside fields, you
- * need to wrap each field in a `<vaadin-form-item>` element and define its labels on the wrapper.
+ * need to wrap each field in a `<vaadin24-form-item>` element and define its labels on the wrapper.
  * Then, you can enable the [`labelsAside`](#/elements/vaadin-form-layout#property-labelsAside)
  * property:
  *
  * ```html
- * <vaadin-form-layout auto-responsive labels-aside>
- *   <vaadin-form-row>
- *     <vaadin-form-item>
+ * <vaadin24-form-layout auto-responsive labels-aside>
+ *   <vaadin24-form-row>
+ *     <vaadin24-form-item>
  *       <label slot="label">First Name</label>
- *       <vaadin-text-field></vaadin-text-field>
- *    </vaadin-form-item>
- *    <vaadin-form-item>
+ *       <vaadin24-text-field></vaadin24-text-field>
+ *    </vaadin24-form-item>
+ *    <vaadin24-form-item>
  *      <label slot="label">Last Name</label>
- *       <vaadin-text-field></vaadin-text-field>
- *     </vaadin-form-item>
- *   </vaadin-form-row>
- *   <vaadin-form-row>
- *     <vaadin-form-item colspan="2">
+ *       <vaadin24-text-field></vaadin24-text-field>
+ *     </vaadin24-form-item>
+ *   </vaadin24-form-row>
+ *   <vaadin24-form-row>
+ *     <vaadin24-form-item colspan="2">
  *       <label slot="label">Address</label>
- *       <vaadin-text-area></vaadin-text-area>
- *     </vaadin-form-item>
- *   </vaadin-form-row>
- * </vaadin-form-layout>
+ *       <vaadin24-text-area></vaadin24-text-area>
+ *     </vaadin24-form-item>
+ *   </vaadin24-form-row>
+ * </vaadin24-form-layout>
  * ```
  *
  * With this, FormLayout will display labels beside fields, falling back to
@@ -190,7 +190,7 @@
  *
  * ### CSS Properties Reference
  *
- * The following custom CSS properties are available on the `<vaadin-form-layout>`
+ * The following custom CSS properties are available on the `<vaadin24-form-layout>`
  * element:
  *
  * Custom CSS property | Description | Default
@@ -208,7 +208,7 @@
  */
 class FormLayout extends FormLayoutMixin(ThemableMixin(ElementMixin(PolymerElement))) {
   static get is() {
-    return 'vaadin-form-layout';
+    return 'vaadin24-form-layout';
   }
 
   static get template() {
