Index: src/vaadin-lit-form-row.js
===================================================================
--- src/vaadin-lit-form-row.js
+++ src/vaadin-lit-form-row.js
@@ -4,13 +4,13 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { formRowStyles } from './vaadin-form-layout-styles.js';
 
 /**
- * LitElement based version of `<vaadin-form-row>` web component.
+ * LitElement based version of `<vaadin24-form-row>` web component.
  *
  * ## Disclaimer
  *
@@ -20,7 +20,7 @@
  */
 class FormRow extends ThemableMixin(PolylitMixin(LitElement)) {
   static get is() {
-    return 'vaadin-form-row';
+    return 'vaadin24-form-row';
   }
 
   static get styles() {
