Index: src/vaadin-form-layout.d.ts
===================================================================
--- src/vaadin-form-layout.d.ts
+++ src/vaadin-form-layout.d.ts
@@ -3,9 +3,9 @@
  * Copyright (c) 2017 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ResizeMixin } from '@vaadin/component-base/src/resize-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ResizeMixin } from '@scoped-vaadin/component-base/src/resize-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 export type FormLayoutLabelsPosition = 'aside' | 'top';
 
@@ -16,28 +16,28 @@
 };
 
 /**
- * `<vaadin-form-layout>` is a Web Component providing configurable responsive
+ * `<vaadin24-form-layout>` is a Web Component providing configurable responsive
  * layout for form elements.
  *
  * ```html
- * <vaadin-form-layout>
+ * <vaadin24-form-layout>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">First Name</label>
  *     <input class="full-width" value="Jane">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">Last Name</label>
  *     <input class="full-width" value="Doe">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">Email</label>
  *     <input class="full-width" value="jane.doe@example.com">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- * </vaadin-form-layout>
+ * </vaadin24-form-layout>
  * ```
  *
  * It supports any child elements as layout items.
@@ -54,24 +54,24 @@
  * In the example below, the first text field spans on two columns:
  *
  * ```html
- * <vaadin-form-layout>
+ * <vaadin24-form-layout>
  *
- *   <vaadin-form-item colspan="2">
+ *   <vaadin24-form-item colspan="2">
  *     <label slot="label">Address</label>
  *     <input class="full-width">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">First Name</label>
  *     <input class="full-width" value="Jane">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">Last Name</label>
  *     <input class="full-width" value="Doe">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- * </vaadin-form-layout>
+ * </vaadin24-form-layout>
  * ```
  *
  * ### Explicit New Row
@@ -79,26 +79,26 @@
  * Use the `<br>` line break element to wrap the items on a new row:
  *
  * ```html
- * <vaadin-form-layout>
+ * <vaadin24-form-layout>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">Email</label>
  *     <input class="full-width">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
  *   <br>
  *
- *   <vaadin-form-item>
+ *   <vaadin24-form-item>
  *     <label slot="label">Confirm Email</label>
  *     <input class="full-width">
- *   </vaadin-form-item>
+ *   </vaadin24-form-item>
  *
- * </vaadin-form-layout>
+ * </vaadin24-form-layout>
  * ```
  *
  * ### CSS Properties Reference
  *
- * The following custom CSS properties are available on the `<vaadin-form-layout>`
+ * The following custom CSS properties are available on the `<vaadin24-form-layout>`
  * element:
  *
  * Custom CSS property | Description | Default
@@ -153,7 +153,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-form-layout': FormLayout;
+    'vaadin24-form-layout': FormLayout;
   }
 }
 
