Index: src/vaadin-form-item-mixin.js
===================================================================
--- src/vaadin-form-item-mixin.js
+++ src/vaadin-form-item-mixin.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2017 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { addValueToAttribute, removeValueFromAttribute } from '@vaadin/component-base/src/dom-utils.js';
-import { generateUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
+import { addValueToAttribute, removeValueFromAttribute } from '@scoped-vaadin/component-base/src/dom-utils.js';
+import { generateUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
 
 let spacingDeprecationNotified = false;
 let labelWidthDeprecationNotified = false;
@@ -60,21 +60,21 @@
 
       if (!spacingDeprecationNotified && spacing !== '' && parseInt(spacing) !== 0) {
         console.warn(
-          '`--vaadin-form-item-row-spacing` is deprecated since 24.7. Use `--vaadin-form-layout-row-spacing` on <vaadin-form-layout> instead.',
+          '`--vaadin-form-item-row-spacing` is deprecated since 24.7. Use `--vaadin-form-layout-row-spacing` on <vaadin24-form-layout> instead.',
         );
         spacingDeprecationNotified = true;
       }
 
       if (!labelWidthDeprecationNotified && labelWidth !== '' && parseInt(labelWidth) !== 0) {
         console.warn(
-          '`--vaadin-form-item-label-width` is deprecated since 24.7. Use `--vaadin-form-layout-label-width` on <vaadin-form-layout> instead.',
+          '`--vaadin-form-item-label-width` is deprecated since 24.7. Use `--vaadin-form-layout-label-width` on <vaadin24-form-layout> instead.',
         );
         labelWidthDeprecationNotified = true;
       }
 
       if (!labelSpacingDeprecationNotified && labelSpacing !== '' && parseInt(labelSpacing) !== 0) {
         console.warn(
-          '`--vaadin-form-item-label-spacing` is deprecated since 24.7. Use `--vaadin-form-layout-label-spacing` on <vaadin-form-layout> instead.',
+          '`--vaadin-form-item-label-spacing` is deprecated since 24.7. Use `--vaadin-form-layout-label-spacing` on <vaadin24-form-layout> instead.',
         );
         labelSpacingDeprecationNotified = true;
       }
@@ -186,8 +186,8 @@
       const fieldNodes = this.$.contentSlot.assignedElements();
       if (fieldNodes.length > 1) {
         console.warn(
-          `WARNING: Since Vaadin 23, placing multiple fields directly to a <vaadin-form-item> is deprecated.
-Please wrap fields with a <vaadin-custom-field> instead.`,
+          `WARNING: Since Vaadin 23, placing multiple fields directly to a <vaadin24-form-item> is deprecated.
+Please wrap fields with a <vaadin24-custom-field> instead.`,
         );
       }
 
