Index: theme/lumo/vaadin-tooltip-styles.js
===================================================================
--- theme/lumo/vaadin-tooltip-styles.js
+++ theme/lumo/vaadin-tooltip-styles.js
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { overlay } from '@vaadin/vaadin-lumo-styles/mixins/overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { overlay } from '@scoped-vaadin/vaadin-lumo-styles/mixins/overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const tooltipOverlay = css`
   :host {
@@ -24,4 +24,4 @@
   }
 `;
 
-registerStyles('vaadin-tooltip-overlay', [overlay, tooltipOverlay], { moduleId: 'lumo-tooltip-overlay' });
+registerStyles('vaadin23-tooltip-overlay', [overlay, tooltipOverlay], { moduleId: 'lumo-tooltip-overlay' });
