Index: test/typings/button.types.ts
===================================================================
--- test/typings/button.types.ts
+++ test/typings/button.types.ts
@@ -1,16 +1,16 @@
 import '../../vaadin-button.js';
-import type { ActiveMixinClass } from '@vaadin/a11y-base/src/active-mixin.js';
-import type { DisabledMixinClass } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin.js';
-import type { KeyboardMixinClass } from '@vaadin/a11y-base/src/keyboard-mixin.js';
-import type { TabindexMixinClass } from '@vaadin/a11y-base/src/tabindex-mixin.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import type { ActiveMixinClass } from '@scoped-vaadin/a11y-base/src/active-mixin.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import type { KeyboardMixinClass } from '@scoped-vaadin/a11y-base/src/keyboard-mixin.js';
+import type { TabindexMixinClass } from '@scoped-vaadin/a11y-base/src/tabindex-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const assertType = <TExpected>(value: TExpected) => value;
 
-const button = document.createElement('vaadin-button');
+const button = document.createElement('vaadin24-button');
 
 // Properties
 assertType<boolean>(button.autofocus);
