Index: theme/material/vaadin-button-styles.js
===================================================================
--- theme/material/vaadin-button-styles.js
+++ theme/material/vaadin-button-styles.js
@@ -1,7 +1,7 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/shadow.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/shadow.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const button = css`
   :host {
@@ -116,18 +116,18 @@
 
   /* Icon alignment */
 
-  [part] ::slotted(vaadin-icon) {
+  [part] ::slotted(vaadin24-icon) {
     display: block;
     width: 18px;
     height: 18px;
   }
 
-  [part='prefix'] ::slotted(vaadin-icon) {
+  [part='prefix'] ::slotted(vaadin24-icon) {
     margin-right: 8px;
     margin-left: -4px;
   }
 
-  [part='suffix'] ::slotted(vaadin-icon) {
+  [part='suffix'] ::slotted(vaadin24-icon) {
     margin-left: 8px;
     margin-right: -4px;
   }
@@ -153,17 +153,17 @@
     transform: translate(50%, -50%) scale(1);
   }
 
-  :host([dir='rtl']) [part='prefix'] ::slotted(vaadin-icon) {
+  :host([dir='rtl']) [part='prefix'] ::slotted(vaadin24-icon) {
     margin-right: -4px;
     margin-left: 8px;
   }
 
-  :host([dir='rtl']) [part='suffix'] ::slotted(vaadin-icon) {
+  :host([dir='rtl']) [part='suffix'] ::slotted(vaadin24-icon) {
     margin-left: -4px;
     margin-right: 8px;
   }
 `;
 
-registerStyles('vaadin-button', button, { moduleId: 'material-button' });
+registerStyles('vaadin24-button', button, { moduleId: 'material-button' });
 
 export { button };
