Index: test/typings/login.types.ts
===================================================================
--- test/typings/login.types.ts
+++ test/typings/login.types.ts
@@ -1,6 +1,6 @@
 import '../../vaadin-login-form.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { OverlayClassMixinClass } from '@vaadin/component-base/src/overlay-class-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { OverlayClassMixinClass } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
 import type {
   LoginFormDisabledChangedEvent,
   LoginFormErrorChangedEvent,
@@ -16,7 +16,7 @@
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const overlay = document.createElement('vaadin-login-overlay');
+const overlay = document.createElement('vaadin24-login-overlay');
 
 assertType<ElementMixinClass>(overlay);
 assertType<OverlayClassMixinClass>(overlay);
@@ -46,7 +46,7 @@
   assertType<string>(event.detail.value);
 });
 
-const form = document.createElement('vaadin-login-form');
+const form = document.createElement('vaadin24-login-form');
 
 assertType<number>(form.headingLevel);
 assertType<LoginI18n>(form.i18n);
