Index: src/vaadin-icon.d.ts
===================================================================
--- src/vaadin-icon.d.ts
+++ src/vaadin-icon.d.ts
@@ -3,27 +3,27 @@
  * Copyright (c) 2021 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { IconMixin } from './vaadin-icon-mixin.js';
 
 /**
- * `<vaadin-icon>` is a Web Component for displaying SVG icons.
+ * `<vaadin24-icon>` is a Web Component for displaying SVG icons.
  *
  * ### Icon property
  *
- * The `<vaadin-icon>` component is designed to be used as a drop-in replacement for `<iron-icon>`.
+ * The `<vaadin24-icon>` component is designed to be used as a drop-in replacement for `<iron-icon>`.
  * For example, you can use it with `vaadin-icons` like this:
  *
  * ```html
- * <vaadin-icon icon="vaadin:angle-down"></vaadin-icon>
+ * <vaadin24-icon icon="vaadin:angle-down"></vaadin24-icon>
  * ```
  *
  * Alternatively, you can also pick one of the Lumo icons:
  *
  * ```html
- * <vaadin-icon icon="lumo:user"></vaadin-icon>
+ * <vaadin24-icon icon="lumo:user"></vaadin24-icon>
  * ```
  *
  * ### Custom SVG icon
@@ -39,10 +39,10 @@
  * render() {
  *   const svgIcon = svg`<path d="M13 4v2l-5 5-5-5v-2l5 5z"></path>`;
  *   return html`
- *     <vaadin-icon
+ *     <vaadin24-icon
  *       .svg="${svgIcon}"
  *       size="16"
- *     ></vaadin-icon>
+ *     ></vaadin24-icon>
  *   `;
  * }
  * ```
@@ -51,7 +51,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-icon': Icon;
+    'vaadin24-icon': Icon;
   }
 }
 
