Index: src/vaadin-lit-icon.js
===================================================================
--- src/vaadin-lit-icon.js
+++ src/vaadin-lit-icon.js
@@ -6,15 +6,15 @@
 import './vaadin-iconset.js';
 import { html, LitElement } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { IconMixin } from './vaadin-icon-mixin.js';
 import { iconStyles } from './vaadin-icon-styles.js';
 
 /**
- * LitElement based version of `<vaadin-icon>` web component.
+ * LitElement based version of `<vaadin24-icon>` web component.
  *
  * ## Disclaimer
  *
@@ -52,7 +52,7 @@
   }
 
   static get is() {
-    return 'vaadin-icon';
+    return 'vaadin24-icon';
   }
 }
 
