Index: src/vaadin-icon.js
===================================================================
--- src/vaadin-icon.js
+++ src/vaadin-icon.js
@@ -5,31 +5,31 @@
  */
 import './vaadin-iconset.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { IconMixin } from './vaadin-icon-mixin.js';
 import { iconStyles } from './vaadin-icon-styles.js';
 
-registerStyles('vaadin-icon', iconStyles, { moduleId: 'vaadin-icon-styles' });
+registerStyles('vaadin24-icon', iconStyles, { moduleId: 'vaadin-icon-styles' });
 
 /**
- * `<vaadin-icon>` is a Web Component for displaying SVG icons.
+ * `<vaadin24-icon>` is a Web Component for displaying SVG icons.
  *
  * ### Icon property
  *
- * The `<vaadin-icon>` component is designed to be used as a drop-in replacement for `<iron-icon>`.
+ * The `<vaadin24-icon>` component is designed to be used as a drop-in replacement for `<iron-icon>`.
  * For example, you can use it with `vaadin-icons` like this:
  *
  * ```html
- * <vaadin-icon icon="vaadin:angle-down"></vaadin-icon>
+ * <vaadin24-icon icon="vaadin:angle-down"></vaadin24-icon>
  * ```
  *
  * Alternatively, you can also pick one of the Lumo icons:
  *
  * ```html
- * <vaadin-icon icon="lumo:user"></vaadin-icon>
+ * <vaadin24-icon icon="lumo:user"></vaadin24-icon>
  * ```
  *
  * ### Custom SVG icon
@@ -45,10 +45,10 @@
  * render() {
  *   const svgIcon = svg`<path d="M13 4v2l-5 5-5-5v-2l5 5z"></path>`;
  *   return html`
- *     <vaadin-icon
+ *     <vaadin24-icon
  *       .svg="${svgIcon}"
  *       size="16"
- *     ></vaadin-icon>
+ *     ></vaadin24-icon>
  *   `;
  * }
  * ```
@@ -87,7 +87,7 @@
   }
 
   static get is() {
-    return 'vaadin-icon';
+    return 'vaadin24-icon';
   }
 }
 
