Index: src/vaadin-iconset.js
===================================================================
--- src/vaadin-iconset.js
+++ src/vaadin-iconset.js
@@ -3,11 +3,11 @@
  * Copyright (c) 2021 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
 import { IconsetMixin } from './vaadin-iconset-mixin.js';
 
 /**
- * `<vaadin-iconset>` is a Web Component for creating SVG icon collections.
+ * `<vaadin24-iconset>` is a Web Component for creating SVG icon collections.
  *
  * @customElement
  * @extends HTMLElement
@@ -15,7 +15,7 @@
  */
 class Iconset extends IconsetMixin(HTMLElement) {
   static get is() {
-    return 'vaadin-iconset';
+    return 'vaadin24-iconset';
   }
 }
 
