Index: src/vaadin-iconset.js
===================================================================
--- src/vaadin-iconset.js
+++ src/vaadin-iconset.js
@@ -4,12 +4,12 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
 import { IconsetMixin } from './vaadin-iconset-mixin.js';
 
 /**
- * `<vaadin-iconset>` is a Web Component for creating SVG icon collections.
+ * `<vaadin24-iconset>` is a Web Component for creating SVG icon collections.
  *
  * @customElement
  * @extends HTMLElement
@@ -22,7 +22,7 @@
   }
 
   static get is() {
-    return 'vaadin-iconset';
+    return 'vaadin24-iconset';
   }
 }
 
