Index: src/vaadin-iconset.d.ts
===================================================================
--- src/vaadin-iconset.d.ts
+++ src/vaadin-iconset.d.ts
@@ -3,11 +3,11 @@
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
 import type { IconSvgLiteral } from './vaadin-icon-svg.js';
 
 /**
- * `<vaadin-iconset>` is a Web Component for creating SVG icon collections.
+ * `<vaadin23-iconset>` is a Web Component for creating SVG icon collections.
  */
 declare class Iconset extends ElementMixin(HTMLElement) {
   /**
@@ -19,14 +19,14 @@
    * The name of the iconset. Every iconset is required to have its own unique name.
    * All the SVG icons in the iconset must have IDs conforming to its name.
    *
-   * See also [`name`](#/elements/vaadin-icon#property-name) property of `vaadin-icon`.
+   * See also [`name`](#/elements/vaadin-icon#property-name) property of `vaadin23-icon`.
    */
   name: string;
 
   /**
    * The size of an individual icon. Note that icons must be square.
    *
-   * When using `vaadin-icon`, the size of the iconset will take precedence
+   * When using `vaadin23-icon`, the size of the iconset will take precedence
    * over the size defined by the user to ensure correct appearance.
    */
   size: number;
@@ -40,7 +40,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-iconset': Iconset;
+    'vaadin23-iconset': Iconset;
   }
 }
 
