Index: src/vaadin-iconset-mixin.d.ts
===================================================================
--- src/vaadin-iconset-mixin.d.ts
+++ src/vaadin-iconset-mixin.d.ts
@@ -15,14 +15,14 @@
    * The name of the iconset. Every iconset is required to have its own unique name.
    * All the SVG icons in the iconset must have IDs conforming to its name.
    *
-   * See also [`name`](#/elements/vaadin-icon#property-name) property of `vaadin-icon`.
+   * See also [`name`](#/elements/vaadin-icon#property-name) property of `vaadin24-icon`.
    */
   name: string;
 
   /**
    * The size of an individual icon. Note that icons must be square.
    *
-   * When using `vaadin-icon`, the size of the iconset will take precedence
+   * When using `vaadin24-icon`, the size of the iconset will take precedence
    * over the size defined by the user to ensure correct appearance.
    */
   size: number;
