Index: src/vaadin-icon-mixin.d.ts
===================================================================
--- src/vaadin-icon-mixin.d.ts
+++ src/vaadin-icon-mixin.d.ts
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { SlotStylesMixinClass } from '@vaadin/component-base/src/slot-styles-mixin.js';
+import type { SlotStylesMixinClass } from '@scoped-vaadin/component-base/src/slot-styles-mixin.js';
 import type { IconFontSizeMixinClass } from './vaadin-icon-font-size-mixin.js';
 import type { IconSvgLiteral } from './vaadin-icon-svg.js';
 
@@ -22,9 +22,9 @@
    * to omit the first part and only use `icon_name` as a value.
    *
    * Setting the `icon` property updates the `svg` and `size` based on the
-   * values provided by the corresponding `vaadin-iconset` element.
+   * values provided by the corresponding `vaadin24-iconset` element.
    *
-   * See also [`name`](#/elements/vaadin-iconset#property-name) property of `vaadin-iconset`.
+   * See also [`name`](#/elements/vaadin-iconset#property-name) property of `vaadin24-iconset`.
    *
    * @attr {string} icon
    */
