Index: src/vaadin-icon-font-size-mixin.js
===================================================================
--- src/vaadin-icon-font-size-mixin.js
+++ src/vaadin-icon-font-size-mixin.js
@@ -3,15 +3,15 @@
  * Copyright (c) 2021 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ResizeMixin } from '@vaadin/component-base/src/resize-mixin.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ResizeMixin } from '@scoped-vaadin/component-base/src/resize-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { needsFontIconSizingFallback } from './vaadin-icon-helpers.js';
 
 const usesFontIconSizingFallback = needsFontIconSizingFallback();
 
 if (usesFontIconSizingFallback) {
   registerStyles(
-    'vaadin-icon',
+    'vaadin24-icon',
     css`
       :host::after,
       :host::before {
