Index: theme/lumo/vaadin-login-form-wrapper-styles.js
===================================================================
--- theme/lumo/vaadin-login-form-wrapper-styles.js
+++ theme/lumo/vaadin-login-form-wrapper-styles.js
@@ -1,7 +1,7 @@
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { color } from '@vaadin/vaadin-lumo-styles/color.js';
-import { typography } from '@vaadin/vaadin-lumo-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { color } from '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import { typography } from '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const loginFormWrapper = css`
   :host {
@@ -83,6 +83,6 @@
   }
 `;
 
-registerStyles('vaadin-login-form-wrapper', [color, typography, loginFormWrapper], {
+registerStyles('vaadin24-login-form-wrapper', [color, typography, loginFormWrapper], {
   moduleId: 'lumo-login-form-wrapper',
 });
