Index: src/vaadin-lit-checkbox-group.js
===================================================================
--- src/vaadin-lit-checkbox-group.js
+++ src/vaadin-lit-checkbox-group.js
@@ -3,17 +3,17 @@
  * Copyright (c) 2018 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/checkbox/src/vaadin-lit-checkbox.js';
+import '@scoped-vaadin/checkbox/src/vaadin-lit-checkbox.js';
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { CheckboxGroupMixin } from './vaadin-checkbox-group-mixin.js';
 import { checkboxGroupStyles } from './vaadin-checkbox-group-styles.js';
 
 /**
- * LitElement based version of `<vaadin-checkbox-group>` web component.
+ * LitElement based version of `<vaadin24-checkbox-group>` web component.
  *
  * ## Disclaimer
  *
@@ -23,7 +23,7 @@
  */
 class CheckboxGroup extends CheckboxGroupMixin(ElementMixin(ThemableMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-checkbox-group';
+    return 'vaadin24-checkbox-group';
   }
 
   static get styles() {
