Index: checkbox-group/src/vaadin-checkbox-group.js
===================================================================
--- checkbox-group/src/vaadin-checkbox-group.js	null
+++ checkbox-group/src/vaadin-checkbox-group.js	null
@@ -4,26 +4,26 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { DisabledMixin } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { Checkbox } from '@vaadin/checkbox/src/vaadin-checkbox.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { SlotObserver } from '@vaadin/component-base/src/slot-observer.js';
-import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
-import { FieldMixin } from '@vaadin/field-base/src/field-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DisabledMixin } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { Checkbox } from '@scoped-vaadin/checkbox/src/vaadin-checkbox.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { SlotObserver } from '@scoped-vaadin/component-base/src/slot-observer.js';
+import { TooltipController } from '@scoped-vaadin/component-base/src/tooltip-controller.js';
+import { FieldMixin } from '@scoped-vaadin/field-base/src/field-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * `<vaadin-checkbox-group>` is a web component that allows the user to choose several items from a group of binary choices.
+ * `<vaadin24-checkbox-group>` is a web component that allows the user to choose several items from a group of binary choices.
  *
  * ```html
- * <vaadin-checkbox-group label="Export data">
- *   <vaadin-checkbox value="0" label="Order ID"></vaadin-checkbox>
- *   <vaadin-checkbox value="1" label="Product name"></vaadin-checkbox>
- *   <vaadin-checkbox value="2" label="Customer"></vaadin-checkbox>
- *   <vaadin-checkbox value="3" label="Status"></vaadin-checkbox>
- * </vaadin-checkbox-group>
+ * <vaadin24-checkbox-group label="Export data">
+ *   <vaadin24-checkbox value="0" label="Order ID"></vaadin24-checkbox>
+ *   <vaadin24-checkbox value="1" label="Product name"></vaadin24-checkbox>
+ *   <vaadin24-checkbox value="2" label="Customer"></vaadin24-checkbox>
+ *   <vaadin24-checkbox value="3" label="Status"></vaadin24-checkbox>
+ * </vaadin24-checkbox-group>
  * ```
  *
  * ### Styling
@@ -66,7 +66,7 @@
  */
 class CheckboxGroup extends FieldMixin(FocusMixin(DisabledMixin(ElementMixin(ThemableMixin(PolymerElement))))) {
   static get is() {
-    return 'vaadin-checkbox-group';
+    return 'vaadin24-checkbox-group';
   }
 
   static get template() {
