Index: password-field/theme/lumo/vaadin-password-field-button-styles.js
===================================================================
--- password-field/theme/lumo/vaadin-password-field-button-styles.js
+++ password-field/theme/lumo/vaadin-password-field-button-styles.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { button } from '@vaadin/button/theme/lumo/vaadin-button-styles.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { button } from '@scoped-vaadin/button/theme/lumo/vaadin-button-styles.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const passwordFieldButton = css`
   :host {
@@ -21,6 +21,6 @@
   }
 `;
 
-registerStyles('vaadin-password-field-button', [button, passwordFieldButton], {
+registerStyles('vaadin24-password-field-button', [button, passwordFieldButton], {
   moduleId: 'lumo-password-field-button',
 });
