Index: test/dom/__snapshots__/combo-box.test.snap.js
===================================================================
--- test/dom/__snapshots__/combo-box.test.snap.js
+++ test/dom/__snapshots__/combo-box.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-combo-box host default"] = 
-`<vaadin-combo-box>
+`<vaadin23-combo-box>
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -26,12 +26,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin23-combo-box>
 `;
 /* end snapshot vaadin-combo-box host default */
 
 snapshots["vaadin-combo-box host disabled"] = 
-`<vaadin-combo-box
+`<vaadin23-combo-box
   aria-disabled="true"
   disabled=""
 >
@@ -60,12 +60,12 @@
     spellcheck="false"
     tabindex="-1"
   >
-</vaadin-combo-box>
+</vaadin23-combo-box>
 `;
 /* end snapshot vaadin-combo-box host disabled */
 
 snapshots["vaadin-combo-box host placeholder"] = 
-`<vaadin-combo-box placeholder="Placeholder">
+`<vaadin23-combo-box placeholder="Placeholder">
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -90,12 +90,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin23-combo-box>
 `;
 /* end snapshot vaadin-combo-box host placeholder */
 
 snapshots["vaadin-combo-box host readonly"] = 
-`<vaadin-combo-box readonly="">
+`<vaadin23-combo-box readonly="">
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -120,12 +120,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin23-combo-box>
 `;
 /* end snapshot vaadin-combo-box host readonly */
 
 snapshots["vaadin-combo-box host required"] = 
-`<vaadin-combo-box required="">
+`<vaadin23-combo-box required="">
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -150,12 +150,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin23-combo-box>
 `;
 /* end snapshot vaadin-combo-box host required */
 
 snapshots["vaadin-combo-box host pattern"] = 
-`<vaadin-combo-box>
+`<vaadin23-combo-box>
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -180,12 +180,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin23-combo-box>
 `;
 /* end snapshot vaadin-combo-box host pattern */
 
 snapshots["vaadin-combo-box host label"] = 
-`<vaadin-combo-box has-label="">
+`<vaadin23-combo-box has-label="">
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -211,12 +211,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin23-combo-box>
 `;
 /* end snapshot vaadin-combo-box host label */
 
 snapshots["vaadin-combo-box host helper"] = 
-`<vaadin-combo-box has-helper="">
+`<vaadin23-combo-box has-helper="">
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -247,12 +247,12 @@
   >
     Helper
   </div>
-</vaadin-combo-box>
+</vaadin23-combo-box>
 `;
 /* end snapshot vaadin-combo-box host helper */
 
 snapshots["vaadin-combo-box host error"] = 
-`<vaadin-combo-box
+`<vaadin23-combo-box
   has-error-message=""
   invalid=""
 >
@@ -283,12 +283,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin23-combo-box>
 `;
 /* end snapshot vaadin-combo-box host error */
 
 snapshots["vaadin-combo-box host opened default"] = 
-`<vaadin-combo-box
+`<vaadin23-combo-box
   focused=""
   opened=""
 >
@@ -316,12 +316,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin23-combo-box>
 `;
 /* end snapshot vaadin-combo-box host opened default */
 
 snapshots["vaadin-combo-box host opened overlay"] = 
-`<vaadin-combo-box-overlay
+`<vaadin23-combo-box-overlay
   dir="ltr"
   id="overlay"
   no-vertical-overlap=""
@@ -329,11 +329,11 @@
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin23-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-    <vaadin-combo-box-item
+    <vaadin23-combo-box-item
       aria-posinset="1"
       aria-selected="false"
       aria-setsize="2"
@@ -342,8 +342,8 @@
       tabindex="-1"
     >
       Item 1
-    </vaadin-combo-box-item>
-    <vaadin-combo-box-item
+    </vaadin23-combo-box-item>
+    <vaadin23-combo-box-item
       aria-posinset="2"
       aria-selected="false"
       aria-setsize="2"
@@ -352,14 +352,14 @@
       tabindex="-1"
     >
       Item 2
-    </vaadin-combo-box-item>
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+    </vaadin23-combo-box-item>
+  </vaadin23-combo-box-scroller>
+</vaadin23-combo-box-overlay>
 `;
 /* end snapshot vaadin-combo-box host opened overlay */
 
 snapshots["vaadin-combo-box host opened theme overlay"] = 
-`<vaadin-combo-box-overlay
+`<vaadin23-combo-box-overlay
   dir="ltr"
   id="overlay"
   no-vertical-overlap=""
@@ -368,11 +368,11 @@
   theme="align-right"
   top-aligned=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin23-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-    <vaadin-combo-box-item
+    <vaadin23-combo-box-item
       aria-posinset="1"
       aria-selected="false"
       aria-setsize="2"
@@ -381,8 +381,8 @@
       tabindex="-1"
     >
       Item 1
-    </vaadin-combo-box-item>
-    <vaadin-combo-box-item
+    </vaadin23-combo-box-item>
+    <vaadin23-combo-box-item
       aria-posinset="2"
       aria-selected="false"
       aria-setsize="2"
@@ -391,9 +391,9 @@
       tabindex="-1"
     >
       Item 2
-    </vaadin-combo-box-item>
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+    </vaadin23-combo-box-item>
+  </vaadin23-combo-box-scroller>
+</vaadin23-combo-box-overlay>
 `;
 /* end snapshot vaadin-combo-box host opened theme overlay */
 
@@ -408,7 +408,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container part="input-field">
+  <vaadin23-input-container part="input-field">
     <slot
       name="prefix"
       slot="prefix"
@@ -430,7 +430,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin23-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -440,16 +440,16 @@
     </slot>
   </div>
 </div>
-<vaadin-combo-box-overlay
+<vaadin23-combo-box-overlay
   id="overlay"
   no-vertical-overlap=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin23-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+  </vaadin23-combo-box-scroller>
+</vaadin23-combo-box-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -466,7 +466,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin23-input-container
     disabled=""
     part="input-field"
   >
@@ -491,7 +491,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin23-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -501,16 +501,16 @@
     </slot>
   </div>
 </div>
-<vaadin-combo-box-overlay
+<vaadin23-combo-box-overlay
   id="overlay"
   no-vertical-overlap=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin23-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+  </vaadin23-combo-box-scroller>
+</vaadin23-combo-box-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -527,7 +527,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin23-input-container
     part="input-field"
     readonly=""
   >
@@ -552,7 +552,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin23-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -562,16 +562,16 @@
     </slot>
   </div>
 </div>
-<vaadin-combo-box-overlay
+<vaadin23-combo-box-overlay
   id="overlay"
   no-vertical-overlap=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin23-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+  </vaadin23-combo-box-scroller>
+</vaadin23-combo-box-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -588,7 +588,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin23-input-container
     invalid=""
     part="input-field"
   >
@@ -613,7 +613,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin23-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -623,16 +623,16 @@
     </slot>
   </div>
 </div>
-<vaadin-combo-box-overlay
+<vaadin23-combo-box-overlay
   id="overlay"
   no-vertical-overlap=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin23-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+  </vaadin23-combo-box-scroller>
+</vaadin23-combo-box-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -649,7 +649,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin23-input-container
     part="input-field"
     theme="align-right"
   >
@@ -674,7 +674,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin23-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -684,17 +684,17 @@
     </slot>
   </div>
 </div>
-<vaadin-combo-box-overlay
+<vaadin23-combo-box-overlay
   id="overlay"
   no-vertical-overlap=""
   theme="align-right"
 >
-  <vaadin-combo-box-scroller
+  <vaadin23-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+  </vaadin23-combo-box-scroller>
+</vaadin23-combo-box-overlay>
 <slot name="tooltip">
 </slot>
 `;
