Index: test/dom/__snapshots__/combo-box.test.snap.js
===================================================================
--- test/dom/__snapshots__/combo-box.test.snap.js
+++ test/dom/__snapshots__/combo-box.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-combo-box host default"] = 
-`<vaadin-combo-box>
+`<vaadin24-combo-box>
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -26,12 +26,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin24-combo-box>
 `;
 /* end snapshot vaadin-combo-box host default */
 
 snapshots["vaadin-combo-box host disabled"] = 
-`<vaadin-combo-box
+`<vaadin24-combo-box
   aria-disabled="true"
   disabled=""
 >
@@ -60,12 +60,12 @@
     spellcheck="false"
     tabindex="-1"
   >
-</vaadin-combo-box>
+</vaadin24-combo-box>
 `;
 /* end snapshot vaadin-combo-box host disabled */
 
 snapshots["vaadin-combo-box host placeholder"] = 
-`<vaadin-combo-box placeholder="Placeholder">
+`<vaadin24-combo-box placeholder="Placeholder">
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -90,12 +90,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin24-combo-box>
 `;
 /* end snapshot vaadin-combo-box host placeholder */
 
 snapshots["vaadin-combo-box host readonly"] = 
-`<vaadin-combo-box readonly="">
+`<vaadin24-combo-box readonly="">
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -120,12 +120,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin24-combo-box>
 `;
 /* end snapshot vaadin-combo-box host readonly */
 
 snapshots["vaadin-combo-box host required"] = 
-`<vaadin-combo-box required="">
+`<vaadin24-combo-box required="">
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -150,12 +150,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin24-combo-box>
 `;
 /* end snapshot vaadin-combo-box host required */
 
 snapshots["vaadin-combo-box host pattern"] = 
-`<vaadin-combo-box>
+`<vaadin24-combo-box>
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -180,12 +180,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin24-combo-box>
 `;
 /* end snapshot vaadin-combo-box host pattern */
 
 snapshots["vaadin-combo-box host label"] = 
-`<vaadin-combo-box has-label="">
+`<vaadin24-combo-box has-label="">
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -211,12 +211,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin24-combo-box>
 `;
 /* end snapshot vaadin-combo-box host label */
 
 snapshots["vaadin-combo-box host helper"] = 
-`<vaadin-combo-box has-helper="">
+`<vaadin24-combo-box has-helper="">
   <label
     for="input-vaadin-combo-box-4"
     id="label-vaadin-combo-box-0"
@@ -247,12 +247,12 @@
   >
     Helper
   </div>
-</vaadin-combo-box>
+</vaadin24-combo-box>
 `;
 /* end snapshot vaadin-combo-box host helper */
 
 snapshots["vaadin-combo-box host error"] = 
-`<vaadin-combo-box
+`<vaadin24-combo-box
   has-error-message=""
   invalid=""
 >
@@ -283,12 +283,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin24-combo-box>
 `;
 /* end snapshot vaadin-combo-box host error */
 
 snapshots["vaadin-combo-box host opened default"] = 
-`<vaadin-combo-box
+`<vaadin24-combo-box
   focused=""
   opened=""
 >
@@ -316,12 +316,12 @@
     slot="input"
     spellcheck="false"
   >
-</vaadin-combo-box>
+</vaadin24-combo-box>
 `;
 /* end snapshot vaadin-combo-box host opened default */
 
 snapshots["vaadin-combo-box host opened overlay"] = 
-`<vaadin-combo-box-overlay
+`<vaadin24-combo-box-overlay
   dir="ltr"
   id="overlay"
   no-vertical-overlap=""
@@ -329,11 +329,11 @@
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin24-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-    <vaadin-combo-box-item
+    <vaadin24-combo-box-item
       aria-posinset="1"
       aria-selected="false"
       aria-setsize="2"
@@ -342,8 +342,8 @@
       tabindex="-1"
     >
       Item 1
-    </vaadin-combo-box-item>
-    <vaadin-combo-box-item
+    </vaadin24-combo-box-item>
+    <vaadin24-combo-box-item
       aria-posinset="2"
       aria-selected="false"
       aria-setsize="2"
@@ -352,14 +352,14 @@
       tabindex="-1"
     >
       Item 2
-    </vaadin-combo-box-item>
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+    </vaadin24-combo-box-item>
+  </vaadin24-combo-box-scroller>
+</vaadin24-combo-box-overlay>
 `;
 /* end snapshot vaadin-combo-box host opened overlay */
 
 snapshots["vaadin-combo-box host opened overlay class"] = 
-`<vaadin-combo-box-overlay
+`<vaadin24-combo-box-overlay
   class="combo-box-overlay custom"
   dir="ltr"
   id="overlay"
@@ -368,11 +368,11 @@
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin24-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-    <vaadin-combo-box-item
+    <vaadin24-combo-box-item
       aria-posinset="1"
       aria-selected="false"
       aria-setsize="2"
@@ -381,8 +381,8 @@
       tabindex="-1"
     >
       Item 1
-    </vaadin-combo-box-item>
-    <vaadin-combo-box-item
+    </vaadin24-combo-box-item>
+    <vaadin24-combo-box-item
       aria-posinset="2"
       aria-selected="false"
       aria-setsize="2"
@@ -391,14 +391,14 @@
       tabindex="-1"
     >
       Item 2
-    </vaadin-combo-box-item>
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+    </vaadin24-combo-box-item>
+  </vaadin24-combo-box-scroller>
+</vaadin24-combo-box-overlay>
 `;
 /* end snapshot vaadin-combo-box host opened overlay class */
 
 snapshots["vaadin-combo-box host opened theme overlay"] = 
-`<vaadin-combo-box-overlay
+`<vaadin24-combo-box-overlay
   dir="ltr"
   id="overlay"
   no-vertical-overlap=""
@@ -407,11 +407,11 @@
   theme="align-right"
   top-aligned=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin24-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-    <vaadin-combo-box-item
+    <vaadin24-combo-box-item
       aria-posinset="1"
       aria-selected="false"
       aria-setsize="2"
@@ -420,8 +420,8 @@
       tabindex="-1"
     >
       Item 1
-    </vaadin-combo-box-item>
-    <vaadin-combo-box-item
+    </vaadin24-combo-box-item>
+    <vaadin24-combo-box-item
       aria-posinset="2"
       aria-selected="false"
       aria-setsize="2"
@@ -430,9 +430,9 @@
       tabindex="-1"
     >
       Item 2
-    </vaadin-combo-box-item>
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+    </vaadin24-combo-box-item>
+  </vaadin24-combo-box-scroller>
+</vaadin24-combo-box-overlay>
 `;
 /* end snapshot vaadin-combo-box host opened theme overlay */
 
@@ -447,7 +447,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container part="input-field">
+  <vaadin24-input-container part="input-field">
     <slot
       name="prefix"
       slot="prefix"
@@ -469,7 +469,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -479,16 +479,16 @@
     </slot>
   </div>
 </div>
-<vaadin-combo-box-overlay
+<vaadin24-combo-box-overlay
   id="overlay"
   no-vertical-overlap=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin24-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+  </vaadin24-combo-box-scroller>
+</vaadin24-combo-box-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -505,7 +505,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     disabled=""
     part="input-field"
   >
@@ -530,7 +530,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -540,16 +540,16 @@
     </slot>
   </div>
 </div>
-<vaadin-combo-box-overlay
+<vaadin24-combo-box-overlay
   id="overlay"
   no-vertical-overlap=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin24-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+  </vaadin24-combo-box-scroller>
+</vaadin24-combo-box-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -566,7 +566,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     part="input-field"
     readonly=""
   >
@@ -591,7 +591,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -601,16 +601,16 @@
     </slot>
   </div>
 </div>
-<vaadin-combo-box-overlay
+<vaadin24-combo-box-overlay
   id="overlay"
   no-vertical-overlap=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin24-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+  </vaadin24-combo-box-scroller>
+</vaadin24-combo-box-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -627,7 +627,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     invalid=""
     part="input-field"
   >
@@ -652,7 +652,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -662,16 +662,16 @@
     </slot>
   </div>
 </div>
-<vaadin-combo-box-overlay
+<vaadin24-combo-box-overlay
   id="overlay"
   no-vertical-overlap=""
 >
-  <vaadin-combo-box-scroller
+  <vaadin24-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+  </vaadin24-combo-box-scroller>
+</vaadin24-combo-box-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -688,7 +688,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     part="input-field"
     theme="align-right"
   >
@@ -713,7 +713,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -723,17 +723,17 @@
     </slot>
   </div>
 </div>
-<vaadin-combo-box-overlay
+<vaadin24-combo-box-overlay
   id="overlay"
   no-vertical-overlap=""
   theme="align-right"
 >
-  <vaadin-combo-box-scroller
+  <vaadin24-combo-box-scroller
     id="vaadin-combo-box-scroller-3"
     role="listbox"
   >
-  </vaadin-combo-box-scroller>
-</vaadin-combo-box-overlay>
+  </vaadin24-combo-box-scroller>
+</vaadin24-combo-box-overlay>
 <slot name="tooltip">
 </slot>
 `;
