Index: src/checked-mixin.js
===================================================================
--- src/checked-mixin.js
+++ src/checked-mixin.js
@@ -4,8 +4,8 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { dedupingMixin } from '@polymer/polymer/lib/utils/mixin.js';
-import { DisabledMixin } from '@vaadin/component-base/src/disabled-mixin.js';
-import { isElementFocused } from '@vaadin/component-base/src/focus-utils.js';
+import { DisabledMixin } from '@scoped-vaadin/component-base/src/disabled-mixin.js';
+import { isElementFocused } from '@scoped-vaadin/component-base/src/focus-utils.js';
 import { DelegateStateMixin } from './delegate-state-mixin.js';
 import { InputMixin } from './input-mixin.js';
 
