Index: field-base/src/input-mixin.js
===================================================================
--- field-base/src/input-mixin.js	null
+++ field-base/src/input-mixin.js	null
@@ -23,7 +23,7 @@
            *
            * A typical case is using `InputController` that does this automatically.
            * However, the input element does not have to always be native <input>:
-           * as an example, <vaadin-combo-box-light> accepts other components.
+           * as an example, <vaadin24-combo-box-light> accepts other components.
            *
            * @protected
            * @type {!HTMLElement}
