Index: field-base/src/input-mixin.d.ts
===================================================================
--- field-base/src/input-mixin.d.ts	null
+++ field-base/src/input-mixin.d.ts	null
@@ -19,7 +19,7 @@
    *
    * A typical case is using `InputController` that does this automatically.
    * However, the input element does not have to always be native <input>:
-   * as an example, <vaadin-combo-box-light> accepts other components.
+   * as an example, <vaadin24-combo-box-light> accepts other components.
    */
   readonly inputElement: HTMLElement;
 
