Index: src/input-constraints-mixin.js
===================================================================
--- src/input-constraints-mixin.js
+++ src/input-constraints-mixin.js
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { dedupeMixin } from '@open-wc/dedupe-mixin';
-import { DelegateStateMixin } from '@vaadin/component-base/src/delegate-state-mixin.js';
+import { DelegateStateMixin } from '@scoped-vaadin/component-base/src/delegate-state-mixin.js';
 import { InputMixin } from './input-mixin.js';
 import { ValidateMixin } from './validate-mixin.js';
 
