Index: src/shadow-focus-mixin.d.ts
===================================================================
--- src/shadow-focus-mixin.d.ts
+++ src/shadow-focus-mixin.d.ts
@@ -4,10 +4,10 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { DisabledMixinClass } from '@vaadin/component-base/src/disabled-mixin.js';
-import type { FocusMixinClass } from '@vaadin/component-base/src/focus-mixin.js';
-import type { KeyboardMixinClass } from '@vaadin/component-base/src/keyboard-mixin.js';
-import type { TabindexMixinClass } from '@vaadin/component-base/src/tabindex-mixin.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/component-base/src/disabled-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/component-base/src/focus-mixin.js';
+import type { KeyboardMixinClass } from '@scoped-vaadin/component-base/src/keyboard-mixin.js';
+import type { TabindexMixinClass } from '@scoped-vaadin/component-base/src/tabindex-mixin.js';
 import type { DelegateFocusMixinClass } from './delegate-focus-mixin.js';
 
 /**
