Index: test/events.test.js
===================================================================
--- test/events.test.js
+++ test/events.test.js
@@ -16,10 +16,10 @@
   describe('default', () => {
     beforeEach(async () => {
       layout = fixtureSync(`
-        <vaadin-master-detail-layout>
+        <vaadin24-master-detail-layout>
           <master-content></master-content>
           <detail-content slot="detail"></detail-content>
-        </vaadin-master-detail-layout>
+        </vaadin24-master-detail-layout>
       `);
       await nextRender();
     });
@@ -103,15 +103,15 @@
 
     beforeEach(async () => {
       layout = fixtureSync(`
-        <vaadin-master-detail-layout>
+        <vaadin24-master-detail-layout>
           <div>Master</div>
-          <vaadin-master-detail-layout slot="detail">
+          <vaadin24-master-detail-layout slot="detail">
             <div>Nested master</div>
             <div slot="detail">
               <input />
             </div>
-          </vaadin-master-detail-layout>
-        </vaadin-master-detail-layout>
+          </vaadin24-master-detail-layout>
+        </vaadin24-master-detail-layout>
       `);
       await nextRender();
       nested = layout.querySelector('[slot="detail"]');
