Index: test/stack-mode.test.js
===================================================================
--- test/stack-mode.test.js
+++ test/stack-mode.test.js
@@ -26,10 +26,10 @@
   describe('default', () => {
     beforeEach(async () => {
       layout = fixtureSync(`
-        <vaadin-master-detail-layout>
+        <vaadin24-master-detail-layout>
           <master-content></master-content>
           <detail-content slot="detail"></detail-content>
-        </vaadin-master-detail-layout>
+        </vaadin24-master-detail-layout>
       `);
       await nextRender();
       master = layout.shadowRoot.querySelector('[part="master"]');
@@ -271,9 +271,9 @@
 
     beforeEach(async () => {
       layout = fixtureSync(`
-        <vaadin-master-detail-layout master-size="200px">
+        <vaadin24-master-detail-layout master-size="200px">
           <div>Master</div>
-          <vaadin-master-detail-layout
+          <vaadin24-master-detail-layout
             slot="detail"
             master-min-size="300px"
             detail-min-size="200px"
@@ -282,8 +282,8 @@
           >
             <div>Nested master</div>
             <div slot="detail">Nested detail</div>
-          </vaadin-master-detail-layout>
-        </vaadin-master-detail-layout>
+          </vaadin24-master-detail-layout>
+        </vaadin24-master-detail-layout>
       `);
       await nextRender();
       nested = layout.querySelector('[slot="detail"]');
