Index: test/dom/__snapshots__/accordion-panel.test.snap.js
===================================================================
--- test/dom/__snapshots__/accordion-panel.test.snap.js
+++ test/dom/__snapshots__/accordion-panel.test.snap.js
@@ -2,15 +2,15 @@
 export const snapshots = {};
 
 snapshots["vaadin-accordion-panel host default"] = 
-`<vaadin-accordion-panel>
-  <vaadin-accordion-heading
+`<vaadin24-accordion-panel>
+  <vaadin24-accordion-heading
     aria-controls="content-vaadin-accordion-panel-0"
     id="summary-vaadin-accordion-panel-1"
     role="heading"
     slot="summary"
   >
     Summary
-  </vaadin-accordion-heading>
+  </vaadin24-accordion-heading>
   <div
     aria-hidden="true"
     aria-labelledby="summary-vaadin-accordion-panel-1"
@@ -19,13 +19,13 @@
   >
     Content
   </div>
-</vaadin-accordion-panel>
+</vaadin24-accordion-panel>
 `;
 /* end snapshot vaadin-accordion-panel host default */
 
 snapshots["vaadin-accordion-panel host opened"] = 
-`<vaadin-accordion-panel opened="">
-  <vaadin-accordion-heading
+`<vaadin24-accordion-panel opened="">
+  <vaadin24-accordion-heading
     aria-controls="content-vaadin-accordion-panel-0"
     id="summary-vaadin-accordion-panel-1"
     opened=""
@@ -33,7 +33,7 @@
     slot="summary"
   >
     Summary
-  </vaadin-accordion-heading>
+  </vaadin24-accordion-heading>
   <div
     aria-hidden="false"
     aria-labelledby="summary-vaadin-accordion-panel-1"
@@ -42,13 +42,13 @@
   >
     Content
   </div>
-</vaadin-accordion-panel>
+</vaadin24-accordion-panel>
 `;
 /* end snapshot vaadin-accordion-panel host opened */
 
 snapshots["vaadin-accordion-panel host disabled"] = 
-`<vaadin-accordion-panel disabled="">
-  <vaadin-accordion-heading
+`<vaadin24-accordion-panel disabled="">
+  <vaadin24-accordion-heading
     aria-controls="content-vaadin-accordion-panel-0"
     aria-disabled="true"
     disabled=""
@@ -58,7 +58,7 @@
     tabindex="-1"
   >
     Summary
-  </vaadin-accordion-heading>
+  </vaadin24-accordion-heading>
   <div
     aria-hidden="true"
     aria-labelledby="summary-vaadin-accordion-panel-1"
@@ -67,13 +67,13 @@
   >
     Content
   </div>
-</vaadin-accordion-panel>
+</vaadin24-accordion-panel>
 `;
 /* end snapshot vaadin-accordion-panel host disabled */
 
 snapshots["vaadin-accordion-panel host theme"] = 
-`<vaadin-accordion-panel theme="filled">
-  <vaadin-accordion-heading
+`<vaadin24-accordion-panel theme="filled">
+  <vaadin24-accordion-heading
     aria-controls="content-vaadin-accordion-panel-0"
     id="summary-vaadin-accordion-panel-1"
     role="heading"
@@ -81,7 +81,7 @@
     theme="filled"
   >
     Summary
-  </vaadin-accordion-heading>
+  </vaadin24-accordion-heading>
   <div
     aria-hidden="true"
     aria-labelledby="summary-vaadin-accordion-panel-1"
@@ -90,7 +90,7 @@
   >
     Content
   </div>
-</vaadin-accordion-panel>
+</vaadin24-accordion-panel>
 `;
 /* end snapshot vaadin-accordion-panel host theme */
 
