Index: src/lit/renderer-directives.js
===================================================================
--- src/lit/renderer-directives.js
+++ src/lit/renderer-directives.js
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { directive } from 'lit/directive.js';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 
 export class PopoverRendererDirective extends LitRendererDirective {
   /**
@@ -32,7 +32,7 @@
 }
 
 /**
- * A Lit directive for populating the content of the `<vaadin-popover-overlay>` element.
+ * A Lit directive for populating the content of the `<vaadin24-popover-overlay>` element.
  *
  * The directive accepts a renderer callback returning a Lit template and assigns it to the popover
  * via the `renderer` property. The renderer is called once to populate the content when assigned
@@ -48,9 +48,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-popover
+ * `<vaadin24-popover
  *   ${popoverRenderer((popover) => html`...`)}
- * ></vaadin-popover>`
+ * ></vaadin24-popover>`
  * ```
  *
  * @param renderer the renderer callback that returns a Lit template.
