Index: avatar/theme/lumo/vaadin-avatar.js
===================================================================
--- avatar/theme/lumo/vaadin-avatar.js	null
+++ avatar/theme/lumo/vaadin-avatar.js	null
@@ -1,3 +1,3 @@
-import '@vaadin/tooltip/theme/lumo/vaadin-tooltip.js';
+import '@scoped-vaadin/tooltip/theme/lumo/vaadin-tooltip.js';
 import './vaadin-avatar-styles.js';
 import '../../src/vaadin-avatar.js';
