Index: src/vaadin-time-picker-scroller.js
===================================================================
--- src/vaadin-time-picker-scroller.js
+++ src/vaadin-time-picker-scroller.js
@@ -4,11 +4,11 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ComboBoxScrollerMixin } from '@vaadin/combo-box/src/vaadin-combo-box-scroller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import { ComboBoxScrollerMixin } from '@scoped-vaadin/combo-box/src/vaadin-combo-box-scroller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
 
 /**
- * An element used internally by `<vaadin-time-picker>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-time-picker>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -17,7 +17,7 @@
  */
 export class TimePickerScroller extends ComboBoxScrollerMixin(PolymerElement) {
   static get is() {
-    return 'vaadin-time-picker-scroller';
+    return 'vaadin24-time-picker-scroller';
   }
 
   static get template() {
