Index: src/vaadin-time-picker-overlay.js
===================================================================
--- src/vaadin-time-picker-overlay.js
+++ src/vaadin-time-picker-overlay.js
@@ -4,12 +4,12 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ComboBoxOverlayMixin } from '@vaadin/combo-box/src/vaadin-combo-box-overlay-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { OverlayMixin } from '@vaadin/overlay/src/vaadin-overlay-mixin.js';
-import { overlayStyles } from '@vaadin/overlay/src/vaadin-overlay-styles.js';
-import { css, registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ComboBoxOverlayMixin } from '@scoped-vaadin/combo-box/src/vaadin-combo-box-overlay-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { OverlayMixin } from '@scoped-vaadin/overlay/src/vaadin-overlay-mixin.js';
+import { overlayStyles } from '@scoped-vaadin/overlay/src/vaadin-overlay-styles.js';
+import { css, registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const timePickerOverlayStyles = css`
   #overlay {
@@ -23,12 +23,12 @@
   }
 `;
 
-registerStyles('vaadin-time-picker-overlay', [overlayStyles, timePickerOverlayStyles], {
+registerStyles('vaadin24-time-picker-overlay', [overlayStyles, timePickerOverlayStyles], {
   moduleId: 'vaadin-time-picker-overlay-styles',
 });
 
 /**
- * An element used internally by `<vaadin-time-picker>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-time-picker>`. Not intended to be used separately.
  *
  * @extends HTMLElement
  * @mixes ComboBoxOverlayMixin
@@ -39,7 +39,7 @@
  */
 export class TimePickerOverlay extends ComboBoxOverlayMixin(OverlayMixin(DirMixin(ThemableMixin(PolymerElement)))) {
   static get is() {
-    return 'vaadin-time-picker-overlay';
+    return 'vaadin24-time-picker-overlay';
   }
 
   static get template() {
