Index: src/vaadin-lit-time-picker-combo-box.js
===================================================================
--- src/vaadin-lit-time-picker-combo-box.js
+++ src/vaadin-lit-time-picker-combo-box.js
@@ -8,13 +8,13 @@
 import './vaadin-lit-time-picker-scroller.js';
 import { css, html, LitElement } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { ComboBoxMixin } from '@vaadin/combo-box/src/vaadin-combo-box-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ComboBoxMixin } from '@scoped-vaadin/combo-box/src/vaadin-combo-box-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * An element used internally by `<vaadin-time-picker>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-time-picker>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -24,7 +24,7 @@
  */
 class TimePickerComboBox extends ComboBoxMixin(ThemableMixin(PolylitMixin(LitElement))) {
   static get is() {
-    return 'vaadin-time-picker-combo-box';
+    return 'vaadin24-time-picker-combo-box';
   }
 
   static get styles() {
@@ -49,7 +49,7 @@
    * @return {string}
    */
   get _tagNamePrefix() {
-    return 'vaadin-time-picker';
+    return 'vaadin24-time-picker';
   }
 
   /**
@@ -66,7 +66,7 @@
     return html`
       <slot></slot>
 
-      <vaadin-time-picker-overlay
+      <vaadin24-time-picker-overlay
         id="overlay"
         .opened="${this._overlayOpened}"
         ?loading="${this.loading}"
@@ -74,7 +74,7 @@
         .positionTarget="${this.positionTarget}"
         .restoreFocusNode="${this.inputElement}"
         no-vertical-overlap
-      ></vaadin-time-picker-overlay>
+      ></vaadin24-time-picker-overlay>
     `;
   }
   /** @protected */
