Index: src/vaadin-lit-time-picker-scroller.js
===================================================================
--- src/vaadin-lit-time-picker-scroller.js
+++ src/vaadin-lit-time-picker-scroller.js
@@ -4,12 +4,12 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { css, html, LitElement } from 'lit';
-import { ComboBoxScrollerMixin } from '@vaadin/combo-box/src/vaadin-combo-box-scroller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { ComboBoxScrollerMixin } from '@scoped-vaadin/combo-box/src/vaadin-combo-box-scroller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
 
 /**
- * An element used internally by `<vaadin-time-picker>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-time-picker>`. Not intended to be used separately.
  *
  * @extends HTMLElement
  * @mixes ComboBoxScrollerMixin
@@ -17,7 +17,7 @@
  */
 export class TimePickerScroller extends ComboBoxScrollerMixin(PolylitMixin(LitElement)) {
   static get is() {
-    return 'vaadin-time-picker-scroller';
+    return 'vaadin24-time-picker-scroller';
   }
 
   static get styles() {
