Index: test/tab.test.js
===================================================================
--- test/tab.test.js
+++ test/tab.test.js
@@ -6,12 +6,12 @@
   let tab;
 
   beforeEach(async () => {
-    tab = fixtureSync('<vaadin-tab>text-content</vaadin-tab>');
+    tab = fixtureSync('<vaadin24-tab>text-content</vaadin24-tab>');
     await nextRender();
   });
 
   it('should have a correct localName', () => {
-    expect(tab.localName).to.be.equal('vaadin-tab');
+    expect(tab.localName).to.be.equal('vaadin24-tab');
   });
 
   it('should have a correct role', () => {
