Index: test/tabs-test-styles.js
===================================================================
--- test/tabs-test-styles.js
+++ test/tabs-test-styles.js
@@ -1,11 +1,11 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 // TODO: subset of Lumo needed for unit tests to pass.
 // These should be eventually covered by base styles.
 registerStyles(
-  'vaadin-tabs',
+  'vaadin24-tabs',
   css`
-    ::slotted(vaadin-tab) {
+    ::slotted(vaadin24-tab) {
       display: flex;
       box-sizing: border-box;
       padding: 0.5rem 0.75rem;
@@ -14,7 +14,7 @@
       font-weight: 500;
     }
 
-    ::slotted(vaadin-tab[orientation='vertical']) {
+    ::slotted(vaadin24-tab[orientation='vertical']) {
       padding: 0.25rem 1rem;
     }
 
@@ -58,7 +58,7 @@
       flex: auto;
     }
 
-    :host([theme~='equal-width-tabs']) [part='tabs'] ::slotted(vaadin-tab) {
+    :host([theme~='equal-width-tabs']) [part='tabs'] ::slotted(vaadin24-tab) {
       flex: 1 0 0%;
     }
   `,
