Index: src/vaadin-password-field-mixin.js
===================================================================
--- src/vaadin-password-field-mixin.js
+++ src/vaadin-password-field-mixin.js
@@ -3,11 +3,11 @@
  * Copyright (c) 2021 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { DisabledMixin } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
-import { SlotStylesMixin } from '@vaadin/component-base/src/slot-styles-mixin.js';
-import { InputMixin } from '@vaadin/field-base/src/input-mixin.js';
+import { DisabledMixin } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { SlotController } from '@scoped-vaadin/component-base/src/slot-controller.js';
+import { SlotStylesMixin } from '@scoped-vaadin/component-base/src/slot-styles-mixin.js';
+import { InputMixin } from '@scoped-vaadin/field-base/src/input-mixin.js';
 
 /**
  * @polymerMixin
@@ -100,7 +100,7 @@
 
       this._revealPart = this.shadowRoot.querySelector('[part="reveal-button"]');
 
-      this._revealButtonController = new SlotController(this, 'reveal', 'vaadin-password-field-button', {
+      this._revealButtonController = new SlotController(this, 'reveal', 'vaadin24-password-field-button', {
         initializer: (btn) => {
           btn.disabled = this.disabled;
 
