Index: src/vaadin-password-field-button.js
===================================================================
--- src/vaadin-password-field-button.js
+++ src/vaadin-password-field-button.js
@@ -4,11 +4,11 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html } from '@polymer/polymer/polymer-element.js';
-import { Button } from '@vaadin/button/src/vaadin-button.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import { Button } from '@scoped-vaadin/button/src/vaadin-button.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
 
 /**
- * An element used internally by `<vaadin-password-field>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-password-field>`. Not intended to be used separately.
  *
  * @customElement
  * @extends Button
@@ -16,7 +16,7 @@
  */
 class PasswordFieldButton extends Button {
   static get is() {
-    return 'vaadin-password-field-button';
+    return 'vaadin24-password-field-button';
   }
 
   static get template() {
