Index: src/vaadin-lit-password-field.js
===================================================================
--- src/vaadin-lit-password-field.js
+++ src/vaadin-lit-password-field.js
@@ -5,12 +5,12 @@
  */
 import './vaadin-lit-password-field-button.js';
 import { html } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { TextField } from '@vaadin/text-field/src/vaadin-lit-text-field.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { TextField } from '@scoped-vaadin/text-field/src/vaadin-lit-text-field.js';
 import { PasswordFieldMixin } from './vaadin-password-field-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-password-field>` web component.
+ * LitElement based version of `<vaadin24-password-field>` web component.
  *
  * ## Disclaimer
  *
@@ -20,7 +20,7 @@
  */
 export class PasswordField extends PasswordFieldMixin(TextField) {
   static get is() {
-    return 'vaadin-password-field';
+    return 'vaadin24-password-field';
   }
 
   /**
