Index: src/vaadin-password-field-button.js
===================================================================
--- src/vaadin-password-field-button.js
+++ src/vaadin-password-field-button.js
@@ -1,20 +1,21 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html } from '@polymer/polymer/polymer-element.js';
-import { Button } from '@vaadin/button/src/vaadin-button.js';
+import { Button } from '@scoped-vaadin/button/src/vaadin-button.js';
 
 /**
- * An element used internally by `<vaadin-password-field>`. Not intended to be used separately.
+ * An element used internally by `<vaadin23-password-field>`. Not intended to be used separately.
  *
  * @extends Button
  * @private
  */
 class PasswordFieldButton extends Button {
   static get is() {
-    return 'vaadin-password-field-button';
+    return 'vaadin23-password-field-button';
   }
 
   static get template() {
@@ -33,4 +34,4 @@
   }
 }
 
-customElements.define(PasswordFieldButton.is, PasswordFieldButton);
+internalCustomElements.define(PasswordFieldButton.is, PasswordFieldButton);
