Index: src/vaadin-password-field.js
===================================================================
--- src/vaadin-password-field.js
+++ src/vaadin-password-field.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
@@ -5,8 +6,8 @@
  */
 import './vaadin-password-field-button.js';
 import { html } from '@polymer/polymer/lib/utils/html-tag.js';
-import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
-import { TextField } from '@vaadin/text-field/src/vaadin-text-field.js';
+import { SlotController } from '@scoped-vaadin/component-base/src/slot-controller.js';
+import { TextField } from '@scoped-vaadin/text-field/src/vaadin-text-field.js';
 
 const ownTemplate = html`
   <div part="reveal-button" slot="suffix">
@@ -17,24 +18,24 @@
 let memoizedTemplate;
 
 /**
- * `<vaadin-password-field>` is an extension of `<vaadin-text-field>` component for entering passwords.
+ * `<vaadin23-password-field>` is an extension of `<vaadin23-text-field>` component for entering passwords.
  *
  * ```html
- * <vaadin-password-field label="Password"></vaadin-password-field>
+ * <vaadin23-password-field label="Password"></vaadin23-password-field>
  * ```
  *
  * ### Styling
  *
- * `<vaadin-password-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin-text-field>`.
- * See [`<vaadin-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
+ * `<vaadin23-password-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin23-text-field>`.
+ * See [`<vaadin23-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
  *
- * In addition to `<vaadin-text-field>` parts, the following parts are available for theming:
+ * In addition to `<vaadin23-text-field>` parts, the following parts are available for theming:
  *
  * Part name       | Description
  * ----------------|----------------------------------------------------
  * `reveal-button` | The eye icon which toggles the password visibility
  *
- * In addition to `<vaadin-text-field>` state attributes, the following state attributes are available for theming:
+ * In addition to `<vaadin23-text-field>` state attributes, the following state attributes are available for theming:
  *
  * Attribute          | Description
  * -------------------|---------------------------------
@@ -52,7 +53,7 @@
  */
 export class PasswordField extends TextField {
   static get is() {
-    return 'vaadin-password-field';
+    return 'vaadin23-password-field';
   }
 
   static get template() {
@@ -156,7 +157,7 @@
     this._revealButtonController = new SlotController(
       this,
       'reveal',
-      () => document.createElement('vaadin-password-field-button'),
+      () => document.createElement('vaadin23-password-field-button'),
       (host, btn) => {
         btn.disabled = host.disabled;
 
@@ -320,4 +321,4 @@
   }
 }
 
-customElements.define(PasswordField.is, PasswordField);
+internalCustomElements.define(PasswordField.is, PasswordField);
