Index: password-field/src/vaadin-password-field.js
===================================================================
--- password-field/src/vaadin-password-field.js
+++ password-field/src/vaadin-password-field.js
@@ -5,9 +5,9 @@
  */
 import './vaadin-password-field-button.js';
 import { html } from '@polymer/polymer/lib/utils/html-tag.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
-import { TextField } from '@vaadin/text-field/src/vaadin-text-field.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { SlotController } from '@scoped-vaadin/component-base/src/slot-controller.js';
+import { TextField } from '@scoped-vaadin/text-field/src/vaadin-text-field.js';
 
 const ownTemplate = html`
   <div part="reveal-button" slot="suffix">
@@ -18,24 +18,24 @@
 let memoizedTemplate;
 
 /**
- * `<vaadin-password-field>` is an extension of `<vaadin-text-field>` component for entering passwords.
+ * `<vaadin24-password-field>` is an extension of `<vaadin24-text-field>` component for entering passwords.
  *
  * ```html
- * <vaadin-password-field label="Password"></vaadin-password-field>
+ * <vaadin24-password-field label="Password"></vaadin24-password-field>
  * ```
  *
  * ### Styling
  *
- * `<vaadin-password-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin-text-field>`.
- * See [`<vaadin-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
+ * `<vaadin24-password-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin24-text-field>`.
+ * See [`<vaadin24-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
  *
- * In addition to `<vaadin-text-field>` parts, the following parts are available for theming:
+ * In addition to `<vaadin24-text-field>` parts, the following parts are available for theming:
  *
  * Part name       | Description
  * ----------------|----------------------------------------------------
  * `reveal-button` | The eye icon which toggles the password visibility
  *
- * In addition to `<vaadin-text-field>` state attributes, the following state attributes are available for theming:
+ * In addition to `<vaadin24-text-field>` state attributes, the following state attributes are available for theming:
  *
  * Attribute          | Description
  * -------------------|---------------------------------
@@ -54,7 +54,7 @@
  */
 export class PasswordField extends TextField {
   static get is() {
-    return 'vaadin-password-field';
+    return 'vaadin24-password-field';
   }
 
   static get template() {
@@ -155,7 +155,7 @@
 
     this._revealPart = this.shadowRoot.querySelector('[part="reveal-button"]');
 
-    this._revealButtonController = new SlotController(this, 'reveal', 'vaadin-password-field-button', {
+    this._revealButtonController = new SlotController(this, 'reveal', 'vaadin24-password-field-button', {
       initializer: (btn) => {
         btn.disabled = this.disabled;
 
