Index: src/vaadin-lit-password-field-button.js
===================================================================
--- src/vaadin-lit-password-field-button.js
+++ src/vaadin-lit-password-field-button.js
@@ -4,15 +4,15 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { buttonStyles } from '@vaadin/button/src/vaadin-button-base.js';
-import { ButtonMixin } from '@vaadin/button/src/vaadin-button-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { buttonStyles } from '@scoped-vaadin/button/src/vaadin-button-base.js';
+import { ButtonMixin } from '@scoped-vaadin/button/src/vaadin-button-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * An element used internally by `<vaadin-password-field>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-password-field>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -23,7 +23,7 @@
  */
 class PasswordFieldButton extends ButtonMixin(DirMixin(ThemableMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-password-field-button';
+    return 'vaadin24-password-field-button';
   }
 
   static get styles() {
