Index: password-field/src/vaadin-password-field.d.ts
===================================================================
--- password-field/src/vaadin-password-field.d.ts	null
+++ password-field/src/vaadin-password-field.d.ts	null
@@ -3,7 +3,7 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { TextField } from '@vaadin/text-field/src/vaadin-text-field.js';
+import { TextField } from '@scoped-vaadin/text-field/src/vaadin-text-field.js';
 
 /**
  * Fired when the user commits a value change.
@@ -40,24 +40,24 @@
 }
 
 /**
- * `<vaadin-password-field>` is an extension of `<vaadin-text-field>` component for entering passwords.
+ * `<vaadin24-password-field>` is an extension of `<vaadin24-text-field>` component for entering passwords.
  *
  * ```html
- * <vaadin-password-field label="Password"></vaadin-password-field>
+ * <vaadin24-password-field label="Password"></vaadin24-password-field>
  * ```
  *
  * ### Styling
  *
- * `<vaadin-password-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin-text-field>`.
- * See [`<vaadin-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
+ * `<vaadin24-password-field>` provides the same set of shadow DOM parts and state attributes as `<vaadin24-text-field>`.
+ * See [`<vaadin24-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
  *
- * In addition to `<vaadin-text-field>` parts, the following parts are available for theming:
+ * In addition to `<vaadin24-text-field>` parts, the following parts are available for theming:
  *
  * Part name       | Description
  * ----------------|----------------------------------------------------
  * `reveal-button` | The eye icon which toggles the password visibility
  *
- * In addition to `<vaadin-text-field>` state attributes, the following state attributes are available for theming:
+ * In addition to `<vaadin24-text-field>` state attributes, the following state attributes are available for theming:
  *
  * Attribute          | Description
  * -------------------|---------------------------------
@@ -112,7 +112,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-password-field': PasswordField;
+    'vaadin24-password-field': PasswordField;
   }
 }
 
