Index: test/dom/__snapshots__/text-area.test.snap.js
===================================================================
--- test/dom/__snapshots__/text-area.test.snap.js
+++ test/dom/__snapshots__/text-area.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-text-area host default"] = 
-`<vaadin-text-area>
+`<vaadin24-text-area>
   <label
     for="textarea-vaadin-text-area-3"
     id="label-vaadin-text-area-0"
@@ -20,12 +20,12 @@
     slot="textarea"
   >
   </textarea>
-</vaadin-text-area>
+</vaadin24-text-area>
 `;
 /* end snapshot vaadin-text-area host default */
 
 snapshots["vaadin-text-area host helper"] = 
-`<vaadin-text-area has-helper="">
+`<vaadin24-text-area has-helper="">
   <label
     for="textarea-vaadin-text-area-3"
     id="label-vaadin-text-area-0"
@@ -50,12 +50,12 @@
   >
     Helper
   </div>
-</vaadin-text-area>
+</vaadin24-text-area>
 `;
 /* end snapshot vaadin-text-area host helper */
 
 snapshots["vaadin-text-area host error"] = 
-`<vaadin-text-area
+`<vaadin24-text-area
   has-error-message=""
   invalid=""
 >
@@ -80,7 +80,7 @@
     slot="textarea"
   >
   </textarea>
-</vaadin-text-area>
+</vaadin24-text-area>
 `;
 /* end snapshot vaadin-text-area host error */
 
@@ -95,7 +95,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container part="input-field">
+  <vaadin24-input-container part="input-field">
     <slot
       name="prefix"
       slot="prefix"
@@ -115,7 +115,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -141,7 +141,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     disabled=""
     part="input-field"
   >
@@ -164,7 +164,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -190,7 +190,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     part="input-field"
     readonly=""
   >
@@ -213,7 +213,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -239,7 +239,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     invalid=""
     part="input-field"
   >
@@ -262,7 +262,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -288,7 +288,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     part="input-field"
     theme="align-right"
   >
@@ -311,7 +311,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
