Index: test/helpers/detail-content.js
===================================================================
--- test/helpers/detail-content.js
+++ test/helpers/detail-content.js
@@ -1,6 +1,7 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { css, html, LitElement } from 'lit';
 
-customElements.define(
+internalCustomElements.define(
   'detail-content',
   class extends LitElement {
     static get styles() {
