Index: src/vaadin-horizontal-layout-styles.js
===================================================================
--- src/vaadin-horizontal-layout-styles.js
+++ src/vaadin-horizontal-layout-styles.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 export const baseStyles = css`
   :host {
@@ -55,8 +55,8 @@
     flex: 1;
   }
 
-  ::slotted(vaadin-horizontal-layout[data-width-full]),
-  ::slotted(vaadin-vertical-layout[data-width-full]) {
+  ::slotted(vaadin24-horizontal-layout[data-width-full]),
+  ::slotted(vaadin24-vertical-layout[data-width-full]) {
     min-width: 0;
   }
 `;
