Index: theme/material/vaadin-avatar-styles.d.ts
===================================================================
--- theme/material/vaadin-avatar-styles.d.ts
+++ theme/material/vaadin-avatar-styles.d.ts
@@ -1,3 +1,3 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import '@vaadin/vaadin-material-styles/user-colors.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import '@scoped-vaadin/vaadin-material-styles/user-colors.js';
