Index: src/lit/renderer-directives.d.ts
===================================================================
--- src/lit/renderer-directives.d.ts
+++ src/lit/renderer-directives.d.ts
@@ -5,7 +5,7 @@
  */
 import type { TemplateResult } from 'lit';
 import type { DirectiveResult } from 'lit/directive.js';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 import type { Select } from '../vaadin-select.js';
 
 export type SelectLitRenderer = (select: Select) => TemplateResult;
@@ -28,7 +28,7 @@
 }
 
 /**
- * A Lit directive for populating the content of the `<vaadin-select-overlay>` element.
+ * A Lit directive for populating the content of the `<vaadin23-select-overlay>` element.
  *
  * The directive accepts a renderer callback returning a Lit template and assigns it to the select
  * via the `renderer` property. The renderer is called once to populate the content when assigned
@@ -44,9 +44,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-select
+ * `<vaadin23-select
  *   ${selectRenderer((select) => html`...`)}
- * ></vaadin-select>`
+ * ></vaadin23-select>`
  * ```
  *
  * @param renderer the renderer callback that returns a Lit template.
