Index: test/visual/icon.common.js
===================================================================
--- test/visual/icon.common.js
+++ test/visual/icon.common.js
@@ -28,18 +28,18 @@
       fixtureSync(
         `
         <!-- Font icons using icon class -->
-        <vaadin-icon icon-class="my-icon-font icon-before"></vaadin-icon>
-        <vaadin-icon icon-class="my-icon-font icon-after"></vaadin-icon>
-        <vaadin-icon icon-class="my-icon-font icon-ligature"></vaadin-icon>
+        <vaadin24-icon icon-class="my-icon-font icon-before"></vaadin24-icon>
+        <vaadin24-icon icon-class="my-icon-font icon-after"></vaadin24-icon>
+        <vaadin24-icon icon-class="my-icon-font icon-ligature"></vaadin24-icon>
 
         <!-- Font icons using char and ligature -->
-        <vaadin-icon icon-class="my-icon-font" char="e900"></vaadin-icon>
-        <vaadin-icon icon-class="my-icon-font" ligature="figma"></vaadin-icon>
-        <vaadin-icon style="font-family: 'My icons'" char="e900"></vaadin-icon>
+        <vaadin24-icon icon-class="my-icon-font" char="e900"></vaadin24-icon>
+        <vaadin24-icon icon-class="my-icon-font" ligature="figma"></vaadin24-icon>
+        <vaadin24-icon style="font-family: 'My icons'" char="e900"></vaadin24-icon>
 
         <!-- Font icons using fontFamily -->
-        <vaadin-icon font-family="My icons" char="\ue900"></vaadin-icon>
-        <vaadin-icon font-family="My icons" ligature="figma"></vaadin-icon>
+        <vaadin24-icon font-family="My icons" char="\ue900"></vaadin24-icon>
+        <vaadin24-icon font-family="My icons" ligature="figma"></vaadin24-icon>
         `,
         div,
       );
