Index: theme/lumo/vaadin-tabsheet-styles.d.ts
===================================================================
--- theme/lumo/vaadin-tabsheet-styles.d.ts
+++ theme/lumo/vaadin-tabsheet-styles.d.ts
@@ -1,5 +1,5 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
 declare const tabsheet: import("lit").CSSResult;
 export { tabsheet };
