Index: theme/lumo/vaadin-tabsheet.d.ts
===================================================================
--- theme/lumo/vaadin-tabsheet.d.ts
+++ theme/lumo/vaadin-tabsheet.d.ts
@@ -1,4 +1,4 @@
-import '@vaadin/tabs/theme/lumo/vaadin-tabs.js';
-import '@vaadin/scroller/theme/lumo/vaadin-scroller.js';
+import '@scoped-vaadin/tabs/theme/lumo/vaadin-tabs.js';
+import '@scoped-vaadin/scroller/theme/lumo/vaadin-scroller.js';
 import './vaadin-tabsheet-styles.js';
 import '../../src/vaadin-tabsheet.js';
