Index: theme/lumo/vaadin-tabsheet-styles.js
===================================================================
--- theme/lumo/vaadin-tabsheet-styles.js
+++ theme/lumo/vaadin-tabsheet-styles.js
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { loader } from '@vaadin/vaadin-lumo-styles/mixins/loader.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { loader } from '@scoped-vaadin/vaadin-lumo-styles/mixins/loader.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const tabsheet = css`
   :host {
@@ -44,6 +44,6 @@
   }
 `;
 
-registerStyles('vaadin-tabsheet', [tabsheet, loader], { moduleId: 'lumo-tabsheet' });
+registerStyles('vaadin24-tabsheet', [tabsheet, loader], { moduleId: 'lumo-tabsheet' });
 
 export { tabsheet };
