Index: test/typings/checkbox-group.types.ts
===================================================================
--- test/typings/checkbox-group.types.ts
+++ test/typings/checkbox-group.types.ts
@@ -1,11 +1,11 @@
 import '../../vaadin-checkbox-group.js';
-import type { DisabledMixinClass } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { FieldMixinClass } from '@vaadin/field-base/src/field-mixin.js';
-import type { LabelMixinClass } from '@vaadin/field-base/src/label-mixin.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { FieldMixinClass } from '@scoped-vaadin/field-base/src/field-mixin.js';
+import type { LabelMixinClass } from '@scoped-vaadin/field-base/src/label-mixin.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import type { CheckboxGroupMixinClass } from '../../src/vaadin-checkbox-group-mixin.js';
 import type {
   CheckboxGroupInvalidChangedEvent,
@@ -15,7 +15,7 @@
 
 const assertType = <TExpected>(value: TExpected) => value;
 
-const group = document.createElement('vaadin-checkbox-group');
+const group = document.createElement('vaadin24-checkbox-group');
 
 group.addEventListener('invalid-changed', (event) => {
   assertType<CheckboxGroupInvalidChangedEvent>(event);
