Index: scroller/theme/lumo/vaadin-scroller-styles.js
===================================================================
--- scroller/theme/lumo/vaadin-scroller-styles.js	null
+++ scroller/theme/lumo/vaadin-scroller-styles.js	null
@@ -1,6 +1,6 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const scroller = css`
   :host {
@@ -38,6 +38,6 @@
   }
 `;
 
-registerStyles('vaadin-scroller', scroller, { moduleId: 'lumo-scroller' });
+registerStyles('vaadin24-scroller', scroller, { moduleId: 'lumo-scroller' });
 
 export { scroller };
