Index: theme/lumo/vaadin-scroller-styles.js
===================================================================
--- theme/lumo/vaadin-scroller-styles.js
+++ theme/lumo/vaadin-scroller-styles.js
@@ -1,5 +1,5 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const scroller = css`
   :host {
@@ -11,6 +11,6 @@
   }
 `;
 
-registerStyles('vaadin-scroller', scroller, { moduleId: 'lumo-scroller' });
+registerStyles('vaadin23-scroller', scroller, { moduleId: 'lumo-scroller' });
 
 export { scroller };
