Index: button/theme/lumo/vaadin-button-styles.js
===================================================================
--- button/theme/lumo/vaadin-button-styles.js
+++ button/theme/lumo/vaadin-button-styles.js
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const button = css`
   :host {
@@ -270,6 +270,6 @@
   }
 `;
 
-registerStyles('vaadin-button', button, { moduleId: 'lumo-button' });
+registerStyles('vaadin24-button', button, { moduleId: 'lumo-button' });
 
 export { button };
