Index: theme/lumo/vaadin-button-styles.js
===================================================================
--- theme/lumo/vaadin-button-styles.js
+++ theme/lumo/vaadin-button-styles.js
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const button = css`
   :host {
@@ -227,14 +227,14 @@
 
   /* Icons */
 
-  [part] ::slotted(vaadin-icon) {
+  [part] ::slotted(vaadin24-icon) {
     display: inline-block;
     width: var(--lumo-icon-size-m);
     height: var(--lumo-icon-size-m);
   }
 
   /* Vaadin icons are based on a 16x16 grid (unlike Lumo and Material icons with 24x24), so they look too big by default */
-  [part] ::slotted(vaadin-icon[icon^='vaadin:']) {
+  [part] ::slotted(vaadin24-icon[icon^='vaadin:']) {
     padding: 0.25em;
     box-sizing: border-box !important;
   }
@@ -282,6 +282,6 @@
   }
 `;
 
-registerStyles('vaadin-button', button, { moduleId: 'lumo-button' });
+registerStyles('vaadin24-button', button, { moduleId: 'lumo-button' });
 
 export { button };
