Index: src/vaadin-list-box.js
===================================================================
--- src/vaadin-list-box.js
+++ src/vaadin-list-box.js
@@ -1,25 +1,26 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
-import { MultiSelectListMixin } from '@vaadin/vaadin-list-mixin/vaadin-multi-select-list-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { TooltipController } from '@scoped-vaadin/component-base/src/tooltip-controller.js';
+import { MultiSelectListMixin } from '@scoped-vaadin/vaadin-list-mixin/vaadin-multi-select-list-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * `<vaadin-list-box>` is a Web Component for creating menus.
+ * `<vaadin23-list-box>` is a Web Component for creating menus.
  *
  * ```
- *   <vaadin-list-box selected="2">
- *     <vaadin-item>Item 1</vaadin-item>
- *     <vaadin-item>Item 2</vaadin-item>
- *     <vaadin-item>Item 3</vaadin-item>
- *     <vaadin-item>Item 4</vaadin-item>
- *   </vaadin-list-box>
+ *   <vaadin23-list-box selected="2">
+ *     <vaadin23-item>Item 1</vaadin23-item>
+ *     <vaadin23-item>Item 2</vaadin23-item>
+ *     <vaadin23-item>Item 3</vaadin23-item>
+ *     <vaadin23-item>Item 4</vaadin23-item>
+ *   </vaadin23-list-box>
  * ```
  *
  * ### Styling
@@ -70,7 +71,7 @@
   }
 
   static get is() {
-    return 'vaadin-list-box';
+    return 'vaadin23-list-box';
   }
 
   static get properties() {
@@ -117,13 +118,13 @@
 
   /** @private */
   _checkImport() {
-    const item = this.querySelector('vaadin-item');
+    const item = this.querySelector('vaadin23-item');
     if (item && !(item instanceof PolymerElement)) {
       console.warn(`Make sure you have imported the vaadin-item element.`);
     }
   }
 }
 
-customElements.define(ListBox.is, ListBox);
+internalCustomElements.define(ListBox.is, ListBox);
 
 export { ListBox };
