Index: src/vaadin-multi-select-list-mixin.js
===================================================================
--- src/vaadin-multi-select-list-mixin.js
+++ src/vaadin-multi-select-list-mixin.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ListMixin } from '@vaadin/a11y-base/src/list-mixin.js';
+import { ListMixin } from '@scoped-vaadin/a11y-base/src/list-mixin.js';
 
 /**
  * A mixin for `nav` elements, facilitating multiple selection of childNodes.
