Index: theme/lumo/vaadin-notification-styles.js
===================================================================
--- theme/lumo/vaadin-notification-styles.js
+++ theme/lumo/vaadin-notification-styles.js
@@ -1,11 +1,11 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-notification-card',
+  'vaadin24-notification-card',
   css`
     :host {
       position: relative;
@@ -34,7 +34,7 @@
       justify-content: space-between;
     }
 
-    [part='content'] ::slotted(vaadin-button) {
+    [part='content'] ::slotted(vaadin24-button) {
       flex: none;
       margin: 0 calc(var(--lumo-space-s) * -1) 0 var(--lumo-space-m);
     }
