Index: test/visual/lumo/select.test.js
===================================================================
--- test/visual/lumo/select.test.js
+++ test/visual/lumo/select.test.js
@@ -1,8 +1,8 @@
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/item/theme/lumo/vaadin-item.js';
-import '@vaadin/list-box/theme/lumo/vaadin-list-box.js';
+import '@scoped-vaadin/item/theme/lumo/vaadin-item.js';
+import '@scoped-vaadin/list-box/theme/lumo/vaadin-list-box.js';
 import '../../not-animated-styles.js';
 import '../../../theme/lumo/vaadin-select.js';
 
@@ -13,14 +13,14 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-select></vaadin-select>', div);
+    element = fixtureSync('<vaadin24-select></vaadin24-select>', div);
     element.renderer = (root) => {
       root.innerHTML = `
-        <vaadin-list-box>
-          <vaadin-item>item 1</vaadin-item>
-          <vaadin-item>item 2</vaadin-item>
-          <vaadin-item>item 3</vaadin-item>
-        </vaadin-list-box>
+        <vaadin24-list-box>
+          <vaadin24-item>item 1</vaadin24-item>
+          <vaadin24-item>item 2</vaadin24-item>
+          <vaadin24-item>item 3</vaadin24-item>
+        </vaadin24-list-box>
       `;
     };
     await nextFrame();
@@ -89,9 +89,9 @@
   it('value-overflow', async () => {
     element.renderer = (root) => {
       root.innerHTML = `
-        <vaadin-list-box>
-          <vaadin-item value="long">Very long item text content</vaadin-item>
-        </vaadin-list-box>
+        <vaadin24-list-box>
+          <vaadin24-item value="long">Very long item text content</vaadin24-item>
+        </vaadin24-list-box>
       `;
     };
     element.value = 'long';
@@ -101,15 +101,15 @@
   it('value-custom', async () => {
     element.renderer = (root) => {
       root.innerHTML = `
-        <vaadin-list-box>
-          <vaadin-item value="custom">
+        <vaadin24-list-box>
+          <vaadin24-item value="custom">
             <div>
               Line 1
               <br>
               Line 2
             </div>
-          </vaadin-item>
-        </vaadin-list-box>
+          </vaadin24-item>
+        </vaadin24-list-box>
       `;
     };
     element.value = 'custom';
@@ -191,9 +191,9 @@
     element.placeholder = 'Placeholder';
     element.renderer = (root) => {
       root.innerHTML = `
-        <vaadin-list-box>
-          <vaadin-item value="">Empty</vaadin-item>
-        </vaadin-list-box>
+        <vaadin24-list-box>
+          <vaadin24-item value="">Empty</vaadin24-item>
+        </vaadin24-list-box>
         `;
     };
     await visualDiff(div, 'empty-value');
