Index: test/typings/time-picker.types.ts
===================================================================
--- test/typings/time-picker.types.ts
+++ test/typings/time-picker.types.ts
@@ -2,15 +2,15 @@
 import type {
   ComboBoxItemMixinClass,
   ComboBoxItemRenderer,
-} from '@vaadin/combo-box/src/vaadin-combo-box-item-mixin.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { DirMixinClass } from '@vaadin/component-base/src/dir-mixin.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { ClearButtonMixinClass } from '@vaadin/field-base/src/clear-button-mixin.js';
-import type { InputControlMixinClass } from '@vaadin/field-base/src/input-control-mixin.js';
-import type { PatternMixinClass } from '@vaadin/field-base/src/pattern-mixin.js';
-import type { ValidateMixinClass } from '@vaadin/field-base/src/validate-mixin.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+} from '@scoped-vaadin/combo-box/src/vaadin-combo-box-item-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { DirMixinClass } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { ClearButtonMixinClass } from '@scoped-vaadin/field-base/src/clear-button-mixin.js';
+import type { InputControlMixinClass } from '@scoped-vaadin/field-base/src/input-control-mixin.js';
+import type { PatternMixinClass } from '@scoped-vaadin/field-base/src/pattern-mixin.js';
+import type { ValidateMixinClass } from '@scoped-vaadin/field-base/src/validate-mixin.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import type { TimePickerItem } from '../../src/vaadin-time-picker-item.js';
 import type {
   TimePicker,
@@ -23,7 +23,7 @@
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const timePicker = document.createElement('vaadin-time-picker');
+const timePicker = document.createElement('vaadin24-time-picker');
 
 // Mixins
 assertType<ControllerMixinClass>(timePicker);
@@ -70,7 +70,7 @@
 });
 
 // Item
-const item = document.createElement('vaadin-time-picker-item');
+const item = document.createElement('vaadin24-time-picker-item');
 assertType<TimePickerItem>(item);
 
 // Item properties
