Index: src/vaadin-tabsheet.js
===================================================================
--- src/vaadin-tabsheet.js
+++ src/vaadin-tabsheet.js
@@ -5,20 +5,20 @@
  */
 import './vaadin-tabsheet-scroller.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DelegateStateMixin } from '@vaadin/component-base/src/delegate-state-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { OverflowController } from '@vaadin/component-base/src/overflow-controller.js';
-import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
-import { SlotObserver } from '@vaadin/component-base/src/slot-observer.js';
-import { generateUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
-import { Tabs } from '@vaadin/tabs/src/vaadin-tabs.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DelegateStateMixin } from '@scoped-vaadin/component-base/src/delegate-state-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { OverflowController } from '@scoped-vaadin/component-base/src/overflow-controller.js';
+import { SlotController } from '@scoped-vaadin/component-base/src/slot-controller.js';
+import { SlotObserver } from '@scoped-vaadin/component-base/src/slot-observer.js';
+import { generateUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
+import { Tabs } from '@scoped-vaadin/tabs/src/vaadin-tabs.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
  * @private
- * A controller which observes the <vaadin-tabs> slotted to the tabs slot.
+ * A controller which observes the <vaadin24-tabs> slotted to the tabs slot.
  */
 class TabsSlotController extends SlotController {
   constructor(host) {
@@ -39,7 +39,7 @@
 
   initCustomNode(tabs) {
     if (!(tabs instanceof Tabs)) {
-      throw Error('The "tabs" slot of a <vaadin-tabsheet> must only contain a <vaadin-tabs> element!');
+      throw Error('The "tabs" slot of a <vaadin24-tabsheet> must only contain a <vaadin24-tabs> element!');
     }
     this.tabs = tabs;
     tabs.addEventListener('items-changed', this.__tabsItemsChangedListener);
@@ -60,24 +60,24 @@
 }
 
 /**
- * `<vaadin-tabsheet>` is a Web Component for organizing and grouping content
+ * `<vaadin24-tabsheet>` is a Web Component for organizing and grouping content
  * into scrollable panels. The panels can be switched between by using tabs.
  *
  * ```
- *  <vaadin-tabsheet>
+ *  <vaadin24-tabsheet>
  *    <div slot="prefix">Prefix</div>
  *    <div slot="suffix">Suffix</div>
  *
- *    <vaadin-tabs slot="tabs">
- *      <vaadin-tab id="tab-1">Tab 1</vaadin-tab>
- *      <vaadin-tab id="tab-2">Tab 2</vaadin-tab>
- *      <vaadin-tab id="tab-3">Tab 3</vaadin-tab>
- *    </vaadin-tabs>
+ *    <vaadin24-tabs slot="tabs">
+ *      <vaadin24-tab id="tab-1">Tab 1</vaadin24-tab>
+ *      <vaadin24-tab id="tab-2">Tab 2</vaadin24-tab>
+ *      <vaadin24-tab id="tab-3">Tab 3</vaadin24-tab>
+ *    </vaadin24-tabs>
  *
  *    <div tab="tab-1">Panel 1</div>
  *    <div tab="tab-2">Panel 2</div>
  *    <div tab="tab-3">Panel 3</div>
- *  </vaadin-tabsheet>
+ *  </vaadin24-tabsheet>
  * ```
  *
  * ### Styling
@@ -146,25 +146,25 @@
         <slot name="suffix"></slot>
       </div>
 
-      <vaadin-tabsheet-scroller part="content">
+      <vaadin24-tabsheet-scroller part="content">
         <div part="loader"></div>
         <slot id="panel-slot"></slot>
-      </vaadin-tabsheet-scroller>
+      </vaadin24-tabsheet-scroller>
     `;
   }
 
   static get is() {
-    return 'vaadin-tabsheet';
+    return 'vaadin24-tabsheet';
   }
 
   static get properties() {
     return {
       /**
-       * The list of `<vaadin-tab>`s from which a selection can be made.
+       * The list of `<vaadin24-tab>`s from which a selection can be made.
        * It is populated from the elements passed inside the slotted
-       * `<vaadin-tabs>`, and updated dynamically when adding or removing items.
+       * `<vaadin24-tabs>`, and updated dynamically when adding or removing items.
        *
-       * Note: unlike `<vaadin-combo-box>`, this property is read-only.
+       * Note: unlike `<vaadin24-combo-box>`, this property is read-only.
        * @type {!Array<!Tab> | undefined}
        */
       items: {
@@ -183,7 +183,7 @@
       },
 
       /**
-       * The slotted <vaadin-tabs> element.
+       * The slotted <vaadin24-tabs> element.
        */
       __tabs: {
         type: Object,
