Index: tabsheet/src/vaadin-tabsheet-scroller.js
===================================================================
--- tabsheet/src/vaadin-tabsheet-scroller.js
+++ tabsheet/src/vaadin-tabsheet-scroller.js
@@ -3,11 +3,11 @@
  * Copyright (c) 2022 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { Scroller } from '@vaadin/scroller/src/vaadin-scroller.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { Scroller } from '@scoped-vaadin/scroller/src/vaadin-scroller.js';
 
 /**
- * An element used internally by `<vaadin-tabsheet>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-tabsheet>`. Not intended to be used separately.
  *
  * @customElement
  * @extends Scroller
@@ -15,7 +15,7 @@
  */
 class TabsheetScroller extends Scroller {
   static get is() {
-    return 'vaadin-tabsheet-scroller';
+    return 'vaadin24-tabsheet-scroller';
   }
 }
 
