Index: src/vaadin-tabsheet-mixin.js
===================================================================
--- src/vaadin-tabsheet-mixin.js
+++ src/vaadin-tabsheet-mixin.js
@@ -1,17 +1,18 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2022 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { DelegateStateMixin } from '@vaadin/component-base/src/delegate-state-mixin.js';
-import { OverflowController } from '@vaadin/component-base/src/overflow-controller.js';
-import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
-import { SlotObserver } from '@vaadin/component-base/src/slot-observer.js';
-import { generateUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
+import { DelegateStateMixin } from '@scoped-vaadin/component-base/src/delegate-state-mixin.js';
+import { OverflowController } from '@scoped-vaadin/component-base/src/overflow-controller.js';
+import { SlotController } from '@scoped-vaadin/component-base/src/slot-controller.js';
+import { SlotObserver } from '@scoped-vaadin/component-base/src/slot-observer.js';
+import { generateUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
 
 /**
  * @private
- * A controller which observes the <vaadin-tabs> slotted to the tabs slot.
+ * A controller which observes the <vaadin24-tabs> slotted to the tabs slot.
  */
 class TabsSlotController extends SlotController {
   constructor(host) {
@@ -49,8 +50,8 @@
   }
 
   initCustomNode(tabs) {
-    if (!(tabs instanceof customElements.get('vaadin-tabs'))) {
-      throw Error('The "tabs" slot of a <vaadin-tabsheet> must only contain a <vaadin-tabs> element!');
+    if (!(tabs instanceof internalCustomElements.get('vaadin24-tabs'))) {
+      throw Error('The "tabs" slot of a <vaadin24-tabsheet> must only contain a <vaadin24-tabs> element!');
     }
     this.tabs = tabs;
     tabs.addEventListener('items-changed', this.__tabsItemsChangedListener);
@@ -79,11 +80,11 @@
     static get properties() {
       return {
         /**
-         * The list of `<vaadin-tab>`s from which a selection can be made.
+         * The list of `<vaadin24-tab>`s from which a selection can be made.
          * It is populated from the elements passed inside the slotted
-         * `<vaadin-tabs>`, and updated dynamically when adding or removing items.
+         * `<vaadin24-tabs>`, and updated dynamically when adding or removing items.
          *
-         * Note: unlike `<vaadin-combo-box>`, this property is read-only.
+         * Note: unlike `<vaadin24-combo-box>`, this property is read-only.
          * @type {!Array<!Tab> | undefined}
          */
         items: {
@@ -102,7 +103,7 @@
         },
 
         /**
-         * The slotted <vaadin-tabs> element.
+         * The slotted <vaadin24-tabs> element.
          */
         __tabs: {
           type: Object,
