Index: src/vaadin-lit-tabsheet.js
===================================================================
--- src/vaadin-lit-tabsheet.js
+++ src/vaadin-lit-tabsheet.js
@@ -3,17 +3,17 @@
  * Copyright (c) 2022 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/tabs/src/vaadin-lit-tabs.js';
+import '@scoped-vaadin/tabs/src/vaadin-lit-tabs.js';
 import './vaadin-lit-tabsheet-scroller.js';
 import { css, html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { TabSheetMixin } from './vaadin-tabsheet-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-tabsheet>` web component.
+ * LitElement based version of `<vaadin24-tabsheet>` web component.
  *
  * ## Disclaimer
  *
@@ -23,7 +23,7 @@
  */
 class TabSheet extends TabSheetMixin(ThemableMixin(ElementMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-tabsheet';
+    return 'vaadin24-tabsheet';
   }
 
   static get styles() {
@@ -70,10 +70,10 @@
         <slot name="suffix"></slot>
       </div>
 
-      <vaadin-tabsheet-scroller part="content">
+      <vaadin24-tabsheet-scroller part="content">
         <div part="loader"></div>
         <slot id="panel-slot"></slot>
-      </vaadin-tabsheet-scroller>
+      </vaadin24-tabsheet-scroller>
     `;
   }
 }
