Index: src/vaadin-tabsheet-mixin.d.ts
===================================================================
--- src/vaadin-tabsheet-mixin.d.ts
+++ src/vaadin-tabsheet-mixin.d.ts
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { DelegateStateMixinClass } from '@vaadin/component-base/src/delegate-state-mixin.js';
+import type { DelegateStateMixinClass } from '@scoped-vaadin/component-base/src/delegate-state-mixin.js';
 
 /**
  * A mixin providing common tab-sheet functionality.
@@ -21,11 +21,11 @@
   selected: number | null | undefined;
 
   /**
-   * The list of `<vaadin-tab>`s from which a selection can be made.
+   * The list of `<vaadin24-tab>`s from which a selection can be made.
    * It is populated from the elements passed inside the slotted
-   * `<vaadin-tabs>`, and updated dynamically when adding or removing items.
+   * `<vaadin24-tabs>`, and updated dynamically when adding or removing items.
    *
-   * Note: unlike `<vaadin-combo-box>`, this property is read-only.
+   * Note: unlike `<vaadin24-combo-box>`, this property is read-only.
    */
   readonly items: Tab[] | undefined;
 }
