Index: field-highlighter/theme/lumo/vaadin-field-highlighter.js
===================================================================
--- field-highlighter/theme/lumo/vaadin-field-highlighter.js	null
+++ field-highlighter/theme/lumo/vaadin-field-highlighter.js	null
@@ -3,7 +3,7 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/vaadin-lumo-styles/user-colors.js';
+import '@scoped-vaadin/vaadin-lumo-styles/user-colors.js';
 import './vaadin-field-outline-styles.js';
 import './vaadin-user-tags.js';
 import '../../src/vaadin-field-highlighter.js';
