Index: theme/lumo/vaadin-field-outline-styles.d.ts
===================================================================
--- theme/lumo/vaadin-field-outline-styles.d.ts
+++ theme/lumo/vaadin-field-outline-styles.d.ts
@@ -3,5 +3,5 @@
  * Copyright (c) 2021 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
