Index: theme/lumo/vaadin-user-tags-styles.js
===================================================================
--- theme/lumo/vaadin-user-tags-styles.js
+++ theme/lumo/vaadin-user-tags-styles.js
@@ -3,14 +3,14 @@
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-user-tags-overlay',
+  'vaadin23-user-tags-overlay',
   css`
     [part='overlay'] {
       will-change: opacity, transform;
@@ -42,7 +42,7 @@
 );
 
 registerStyles(
-  'vaadin-user-tag',
+  'vaadin23-user-tag',
   css`
     :host {
       font-family: var(--lumo-font-family);
