Index: context-menu/src/vaadin-lit-context-menu-overlay.js
===================================================================
--- context-menu/src/vaadin-lit-context-menu-overlay.js	null
+++ context-menu/src/vaadin-lit-context-menu-overlay.js	null
@@ -4,17 +4,17 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { OverlayMixin } from '@vaadin/overlay/src/vaadin-overlay-mixin.js';
-import { overlayStyles } from '@vaadin/overlay/src/vaadin-overlay-styles.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { OverlayMixin } from '@scoped-vaadin/overlay/src/vaadin-overlay-mixin.js';
+import { overlayStyles } from '@scoped-vaadin/overlay/src/vaadin-overlay-styles.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { MenuOverlayMixin } from './vaadin-menu-overlay-mixin.js';
 import { styles } from './vaadin-menu-overlay-styles.js';
 
 /**
- * An element used internally by `<vaadin-context-menu>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-context-menu>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -28,7 +28,7 @@
   OverlayMixin(DirMixin(ThemableMixin(PolylitMixin(LitElement)))),
 ) {
   static get is() {
-    return 'vaadin-context-menu-overlay';
+    return 'vaadin24-context-menu-overlay';
   }
 
   static get properties() {
