Index: context-menu/src/vaadin-lit-context-menu-item.js
===================================================================
--- context-menu/src/vaadin-lit-context-menu-item.js	null
+++ context-menu/src/vaadin-lit-context-menu-item.js	null
@@ -4,14 +4,14 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { css, html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ItemMixin } from '@vaadin/item/src/vaadin-item-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ItemMixin } from '@scoped-vaadin/item/src/vaadin-item-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * An element used internally by `<vaadin-context-menu>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-context-menu>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -22,7 +22,7 @@
  */
 class ContextMenuItem extends ItemMixin(ThemableMixin(DirMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-context-menu-item';
+    return 'vaadin24-context-menu-item';
   }
 
   static get styles() {
