Index: src/vaadin-contextmenu-event.js
===================================================================
--- src/vaadin-contextmenu-event.js
+++ src/vaadin-contextmenu-event.js
@@ -3,9 +3,9 @@
  * Copyright (c) 2016 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { isKeyboardActive } from '@vaadin/a11y-base/src/focus-utils.js';
-import { isFirefox, isIOS } from '@vaadin/component-base/src/browser-utils.js';
-import { prevent, register } from '@vaadin/component-base/src/gestures.js';
+import { isKeyboardActive } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
+import { isFirefox, isIOS } from '@scoped-vaadin/component-base/src/browser-utils.js';
+import { prevent, register } from '@scoped-vaadin/component-base/src/gestures.js';
 
 register({
   name: 'vaadin-contextmenu',
