Index: test/visual/common.js
===================================================================
--- test/visual/common.js
+++ test/visual/common.js
@@ -1,8 +1,8 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /* Hide caret */
 registerStyles(
-  'vaadin-date-picker',
+  'vaadin24-date-picker',
   css`
     :host([focus-ring]) ::slotted(input),
     :host([opened]) ::slotted(input) {
@@ -13,7 +13,7 @@
 
 /* Stop focused day animation */
 registerStyles(
-  'vaadin-month-calendar',
+  'vaadin24-month-calendar',
   css`
     [part='date'][focused]::before {
       animation: none !important;
