Index: test/dom/notification.test.js
===================================================================
--- test/dom/notification.test.js
+++ test/dom/notification.test.js
@@ -2,11 +2,11 @@
 import { fixtureSync, nextRender, oneEvent } from '@vaadin/testing-helpers';
 import '../../vaadin-notification.js';
 
-describe('vaadin-notification', () => {
+describe('vaadin24-notification', () => {
   let notification, card;
 
   beforeEach(async () => {
-    notification = fixtureSync('<vaadin-notification></vaadin-notification>');
+    notification = fixtureSync('<vaadin24-notification></vaadin24-notification>');
     await nextRender();
     card = notification._card;
     notification.renderer = (root) => {
