Index: theme/lumo/vaadin-master-detail-layout-styles.d.ts
===================================================================
--- theme/lumo/vaadin-master-detail-layout-styles.d.ts
+++ theme/lumo/vaadin-master-detail-layout-styles.d.ts
@@ -1,2 +1,2 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
