Index: theme/lumo/vaadin-master-detail-layout-styles.js
===================================================================
--- theme/lumo/vaadin-master-detail-layout-styles.js
+++ theme/lumo/vaadin-master-detail-layout-styles.js
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-master-detail-layout',
+  'vaadin24-master-detail-layout',
   css`
     :host(:is([drawer], [stack])) [part='detail'] {
       background-color: var(--lumo-base-color);
