Index: src/vaadin-date-time-picker-mixin.d.ts
===================================================================
--- src/vaadin-date-time-picker-mixin.d.ts
+++ src/vaadin-date-time-picker-mixin.d.ts
@@ -4,14 +4,14 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { DisabledMixinClass } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { DatePickerI18n } from '@vaadin/date-picker/src/vaadin-date-picker.js';
-import type { FieldMixinClass } from '@vaadin/field-base/src/field-mixin.js';
-import type { LabelMixinClass } from '@vaadin/field-base/src/label-mixin.js';
-import type { ValidateMixinClass } from '@vaadin/field-base/src/validate-mixin.js';
-import type { TimePickerI18n } from '@vaadin/time-picker/src/vaadin-time-picker.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { DatePickerI18n } from '@scoped-vaadin/date-picker/src/vaadin-date-picker.js';
+import type { FieldMixinClass } from '@scoped-vaadin/field-base/src/field-mixin.js';
+import type { LabelMixinClass } from '@scoped-vaadin/field-base/src/label-mixin.js';
+import type { ValidateMixinClass } from '@scoped-vaadin/field-base/src/validate-mixin.js';
+import type { TimePickerI18n } from '@scoped-vaadin/time-picker/src/vaadin-time-picker.js';
 
 export interface DateTimePickerI18n extends DatePickerI18n, TimePickerI18n {
   /**
@@ -149,17 +149,17 @@
    * `i18n` object or just the properties you want to modify.
    *
    * The object is a combination of the i18n properties supported by
-   * [`<vaadin-date-picker>`](#/elements/vaadin-date-picker) and
-   * [`<vaadin-time-picker>`](#/elements/vaadin-time-picker).
+   * [`<vaadin24-date-picker>`](#/elements/vaadin-date-picker) and
+   * [`<vaadin24-time-picker>`](#/elements/vaadin-time-picker).
    */
   i18n: DateTimePickerI18n;
 
   /**
    * A space-delimited list of CSS class names to set on the overlay elements
-   * of the internal components controlled by the `<vaadin-date-time-picker>`:
+   * of the internal components controlled by the `<vaadin24-date-time-picker>`:
    *
-   * - [`<vaadin-date-picker>`](#/elements/vaadin-date-picker#property-overlayClass)
-   * - [`<vaadin-time-picker>`](#/elements/vaadin-time-picker#property-overlayClass)
+   * - [`<vaadin24-date-picker>`](#/elements/vaadin-date-picker#property-overlayClass)
+   * - [`<vaadin24-time-picker>`](#/elements/vaadin-time-picker#property-overlayClass)
    *
    * @attr {string} overlay-class
    */
