Index: src/vaadin-date-time-picker.d.ts
===================================================================
--- src/vaadin-date-time-picker.d.ts
+++ src/vaadin-date-time-picker.d.ts
@@ -3,13 +3,13 @@
  * Copyright (c) 2019 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { DisabledMixin } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import type { DatePickerI18n } from '@vaadin/date-picker/src/vaadin-date-picker.js';
-import { FieldMixin } from '@vaadin/field-base/src/field-mixin.js';
-import type { TimePickerI18n } from '@vaadin/time-picker/src/vaadin-time-picker.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DisabledMixin } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { DatePickerI18n } from '@scoped-vaadin/date-picker/src/vaadin-date-picker.js';
+import { FieldMixin } from '@scoped-vaadin/field-base/src/field-mixin.js';
+import type { TimePickerI18n } from '@scoped-vaadin/time-picker/src/vaadin-time-picker.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 export interface DateTimePickerI18n extends DatePickerI18n, TimePickerI18n {
   /**
@@ -62,10 +62,10 @@
 }
 
 /**
- * `<vaadin-date-time-picker>` is a Web Component providing a date time selection field.
+ * `<vaadin24-date-time-picker>` is a Web Component providing a date time selection field.
  *
  * ```html
- * <vaadin-date-time-picker value="2019-09-16T15:00"></vaadin-date-time-picker>
+ * <vaadin24-date-time-picker value="2019-09-16T15:00"></vaadin24-date-time-picker>
  * ```
  *
  * ```js
@@ -101,12 +101,12 @@
  *
  * ### Internal components
  *
- * The following components are created by `<vaadin-date-time-picker>` and placed in light DOM:
+ * The following components are created by `<vaadin24-date-time-picker>` and placed in light DOM:
  *
- * - [`<vaadin-date-picker>`](#/elements/vaadin-date-picker).
- * - [`<vaadin-time-picker>`](#/elements/vaadin-time-picker).
+ * - [`<vaadin24-date-picker>`](#/elements/vaadin-date-picker).
+ * - [`<vaadin24-time-picker>`](#/elements/vaadin-time-picker).
  *
- * Note: the `theme` attribute value set on `<vaadin-date-time-picker>` is
+ * Note: the `theme` attribute value set on `<vaadin24-date-time-picker>` is
  * propagated to these components.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
@@ -220,17 +220,17 @@
    * `i18n` object or just the properties you want to modify.
    *
    * The object is a combination of the i18n properties supported by
-   * [`<vaadin-date-picker>`](#/elements/vaadin-date-picker) and
-   * [`<vaadin-time-picker>`](#/elements/vaadin-time-picker).
+   * [`<vaadin24-date-picker>`](#/elements/vaadin-date-picker) and
+   * [`<vaadin24-time-picker>`](#/elements/vaadin-time-picker).
    */
   i18n: DateTimePickerI18n;
 
   /**
    * A space-delimited list of CSS class names to set on the overlay elements
-   * of the internal components controlled by the `<vaadin-date-time-picker>`:
+   * of the internal components controlled by the `<vaadin24-date-time-picker>`:
    *
-   * - [`<vaadin-date-picker>`](#/elements/vaadin-date-picker#property-overlayClass)
-   * - [`<vaadin-time-picker>`](#/elements/vaadin-time-picker#property-overlayClass)
+   * - [`<vaadin24-date-picker>`](#/elements/vaadin-date-picker#property-overlayClass)
+   * - [`<vaadin24-time-picker>`](#/elements/vaadin-time-picker#property-overlayClass)
    *
    * @attr {string} overlay-class
    */
@@ -251,7 +251,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-date-time-picker': DateTimePicker;
+    'vaadin24-date-time-picker': DateTimePicker;
   }
 }
 
