Index: src/vaadin-date-time-picker.js
===================================================================
--- src/vaadin-date-time-picker.js
+++ src/vaadin-date-time-picker.js
@@ -3,22 +3,22 @@
  * Copyright (c) 2019 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/date-picker/src/vaadin-date-picker.js';
-import '@vaadin/time-picker/src/vaadin-time-picker.js';
+import '@scoped-vaadin/date-picker/src/vaadin-date-picker.js';
+import '@scoped-vaadin/time-picker/src/vaadin-time-picker.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { inputFieldShared } from '@vaadin/field-base/src/styles/input-field-shared-styles.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { inputFieldShared } from '@scoped-vaadin/field-base/src/styles/input-field-shared-styles.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { DateTimePickerMixin } from './vaadin-date-time-picker-mixin.js';
 
-registerStyles('vaadin-date-time-picker', inputFieldShared, { moduleId: 'vaadin-date-time-picker' });
+registerStyles('vaadin24-date-time-picker', inputFieldShared, { moduleId: 'vaadin24-date-time-picker' });
 
 /**
- * `<vaadin-date-time-picker>` is a Web Component providing a date time selection field.
+ * `<vaadin24-date-time-picker>` is a Web Component providing a date time selection field.
  *
  * ```html
- * <vaadin-date-time-picker value="2019-09-16T15:00"></vaadin-date-time-picker>
+ * <vaadin24-date-time-picker value="2019-09-16T15:00"></vaadin24-date-time-picker>
  * ```
  *
  * ```js
@@ -54,12 +54,12 @@
  *
  * ### Internal components
  *
- * The following components are created by `<vaadin-date-time-picker>` and placed in light DOM:
+ * The following components are created by `<vaadin24-date-time-picker>` and placed in light DOM:
  *
- * - [`<vaadin-date-picker>`](#/elements/vaadin-date-picker).
- * - [`<vaadin-time-picker>`](#/elements/vaadin-time-picker).
+ * - [`<vaadin24-date-picker>`](#/elements/vaadin-date-picker).
+ * - [`<vaadin24-time-picker>`](#/elements/vaadin-time-picker).
  *
- * Note: the `theme` attribute value set on `<vaadin-date-time-picker>` is
+ * Note: the `theme` attribute value set on `<vaadin24-date-time-picker>` is
  * propagated to these components.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
@@ -124,7 +124,7 @@
   }
 
   static get is() {
-    return 'vaadin-date-time-picker';
+    return 'vaadin24-date-time-picker';
   }
 }
 
