Index: src/vaadin-date-time-picker.d.ts
===================================================================
--- src/vaadin-date-time-picker.d.ts
+++ src/vaadin-date-time-picker.d.ts
@@ -3,8 +3,8 @@
  * Copyright (c) 2019 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { DateTimePickerMixin } from './vaadin-date-time-picker-mixin.js';
 
 export { DateTimePickerI18n } from './vaadin-date-time-picker-mixin.js';
@@ -44,10 +44,10 @@
 }
 
 /**
- * `<vaadin-date-time-picker>` is a Web Component providing a date time selection field.
+ * `<vaadin24-date-time-picker>` is a Web Component providing a date time selection field.
  *
  * ```html
- * <vaadin-date-time-picker value="2019-09-16T15:00"></vaadin-date-time-picker>
+ * <vaadin24-date-time-picker value="2019-09-16T15:00"></vaadin24-date-time-picker>
  * ```
  *
  * ```js
@@ -83,12 +83,12 @@
  *
  * ### Internal components
  *
- * The following components are created by `<vaadin-date-time-picker>` and placed in light DOM:
+ * The following components are created by `<vaadin24-date-time-picker>` and placed in light DOM:
  *
- * - [`<vaadin-date-picker>`](#/elements/vaadin-date-picker).
- * - [`<vaadin-time-picker>`](#/elements/vaadin-time-picker).
+ * - [`<vaadin24-date-picker>`](#/elements/vaadin-date-picker).
+ * - [`<vaadin24-time-picker>`](#/elements/vaadin-time-picker).
  *
- * Note: the `theme` attribute value set on `<vaadin-date-time-picker>` is
+ * Note: the `theme` attribute value set on `<vaadin24-date-time-picker>` is
  * propagated to these components.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
@@ -137,7 +137,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-date-time-picker': DateTimePicker;
+    'vaadin24-date-time-picker': DateTimePicker;
   }
 }
 
