Index: src/vaadin-lit-date-time-picker.js
===================================================================
--- src/vaadin-lit-date-time-picker.js
+++ src/vaadin-lit-date-time-picker.js
@@ -3,18 +3,18 @@
  * Copyright (c) 2019 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/date-picker/src/vaadin-lit-date-picker.js';
-import '@vaadin/time-picker/src/vaadin-lit-time-picker.js';
+import '@scoped-vaadin/date-picker/src/vaadin-lit-date-picker.js';
+import '@scoped-vaadin/time-picker/src/vaadin-lit-time-picker.js';
 import { css, html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { inputFieldShared } from '@vaadin/field-base/src/styles/input-field-shared-styles.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { inputFieldShared } from '@scoped-vaadin/field-base/src/styles/input-field-shared-styles.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { DateTimePickerMixin } from './vaadin-date-time-picker-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-date-time-picker>` web component.
+ * LitElement based version of `<vaadin24-date-time-picker>` web component.
  *
  * ## Disclaimer
  *
@@ -24,7 +24,7 @@
  */
 class DateTimePicker extends DateTimePickerMixin(ThemableMixin(ElementMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-date-time-picker';
+    return 'vaadin24-date-time-picker';
   }
 
   static get styles() {
