Index: src/vaadin-date-time-picker.js
===================================================================
--- src/vaadin-date-time-picker.js
+++ src/vaadin-date-time-picker.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2019 - 2022 Vaadin Ltd.
@@ -7,17 +8,17 @@
 import './vaadin-date-time-picker-time-picker.js';
 import { FlattenedNodesObserver } from '@polymer/polymer/lib/utils/flattened-nodes-observer.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { DisabledMixin } from '@vaadin/component-base/src/disabled-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { FocusMixin } from '@vaadin/component-base/src/focus-mixin.js';
-import { SlotMixin } from '@vaadin/component-base/src/slot-mixin.js';
-import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
-import { dateEquals, parseDate } from '@vaadin/date-picker/src/vaadin-date-picker-helper.js';
-import { FieldMixin } from '@vaadin/field-base/src/field-mixin.js';
-import { inputFieldShared } from '@vaadin/field-base/src/styles/input-field-shared-styles.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DisabledMixin } from '@scoped-vaadin/component-base/src/disabled-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/component-base/src/focus-mixin.js';
+import { SlotMixin } from '@scoped-vaadin/component-base/src/slot-mixin.js';
+import { TooltipController } from '@scoped-vaadin/component-base/src/tooltip-controller.js';
+import { dateEquals, parseDate } from '@scoped-vaadin/date-picker/src/vaadin-date-picker-helper.js';
+import { FieldMixin } from '@scoped-vaadin/field-base/src/field-mixin.js';
+import { inputFieldShared } from '@scoped-vaadin/field-base/src/styles/input-field-shared-styles.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
-registerStyles('vaadin-date-time-picker', inputFieldShared, { moduleId: 'vaadin-date-time-picker' });
+registerStyles('vaadin23-date-time-picker', inputFieldShared, { moduleId: 'vaadin23-date-time-picker' });
 
 /**
  * @typedef {object} TimePickerTime
@@ -37,18 +38,18 @@
   }
 }
 
-const datePickerClass = customElements.get('vaadin-date-time-picker-date-picker');
-const timePickerClass = customElements.get('vaadin-date-time-picker-time-picker');
+const datePickerClass = internalCustomElements.get('vaadin23-date-time-picker-date-picker');
+const timePickerClass = internalCustomElements.get('vaadin23-date-time-picker-time-picker');
 const datePickerI18nDefaults = getPropertyFromPrototype(datePickerClass, 'i18n').value();
 const timePickerI18nDefaults = getPropertyFromPrototype(timePickerClass, 'i18n').value();
 const datePickerI18nProps = Object.keys(datePickerI18nDefaults);
 const timePickerI18nProps = Object.keys(timePickerI18nDefaults);
 
 /**
- * `<vaadin-date-time-picker>` is a Web Component providing a date time selection field.
+ * `<vaadin23-date-time-picker>` is a Web Component providing a date time selection field.
  *
  * ```html
- * <vaadin-date-time-picker value="2019-09-16T15:00"></vaadin-date-time-picker>
+ * <vaadin23-date-time-picker value="2019-09-16T15:00"></vaadin23-date-time-picker>
  * ```
  *
  * ```js
@@ -84,13 +85,13 @@
  *
  * ### Internal components
  *
- * In addition to `<vaadin-date-time-picker>` itself, the following internal
+ * In addition to `<vaadin23-date-time-picker>` itself, the following internal
  * components are themable:
  *
- * - `<vaadin-date-time-picker-date-picker>` - has the same API as [`<vaadin-date-picker>`](#/elements/vaadin-date-picker).
- * - `<vaadin-date-time-picker-time-picker>` - has the same API as [`<vaadin-time-picker>`](#/elements/vaadin-time-picker).
+ * - `<vaadin23-date-time-picker-date-picker>` - has the same API as [`<vaadin23-date-picker>`](#/elements/vaadin-date-picker).
+ * - `<vaadin23-date-time-picker-time-picker>` - has the same API as [`<vaadin23-time-picker>`](#/elements/vaadin-time-picker).
  *
- * Note: the `theme` attribute value set on `<vaadin-date-time-picker>` is
+ * Note: the `theme` attribute value set on `<vaadin23-date-time-picker>` is
  * propagated to the internal components listed above.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/custom-theme/styling-components) documentation.
@@ -159,7 +160,7 @@
   }
 
   static get is() {
-    return 'vaadin-date-time-picker';
+    return 'vaadin23-date-time-picker';
   }
 
   static get properties() {
@@ -327,8 +328,8 @@
        * `i18n` object or just the properties you want to modify.
        *
        * The object is a combination of the i18n properties supported by
-       * [`<vaadin-date-picker>`](#/elements/vaadin-date-picker) and
-       * [`<vaadin-time-picker>`](#/elements/vaadin-time-picker).
+       * [`<vaadin23-date-picker>`](#/elements/vaadin-date-picker) and
+       * [`<vaadin23-time-picker>`](#/elements/vaadin-time-picker).
        * @type {!DateTimePickerI18n}
        */
       i18n: {
@@ -380,12 +381,12 @@
     return {
       ...super.slots,
       'date-picker': () => {
-        const element = document.createElement('vaadin-date-time-picker-date-picker');
+        const element = document.createElement('vaadin23-date-time-picker-date-picker');
         element.__defaultPicker = true;
         return element;
       },
       'time-picker': () => {
-        const element = document.createElement('vaadin-date-time-picker-time-picker');
+        const element = document.createElement('vaadin23-date-time-picker-time-picker');
         element.__defaultPicker = true;
         return element;
       },
@@ -1063,6 +1064,6 @@
    */
 }
 
-customElements.define(DateTimePicker.is, DateTimePicker);
+internalCustomElements.define(DateTimePicker.is, DateTimePicker);
 
 export { DateTimePicker };
