Index: test/position-styles.js
===================================================================
--- test/position-styles.js
+++ test/position-styles.js
@@ -1,7 +1,7 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-tooltip-overlay',
+  'vaadin24-tooltip-overlay',
   css`
     [part='overlay'] {
       width: 50px;
