Index: test/tooltip-position.test.js
===================================================================
--- test/tooltip-position.test.js
+++ test/tooltip-position.test.js
@@ -1,10 +1,10 @@
 import { expect } from '@esm-bundle/chai';
 import { fire, fixtureSync, oneEvent } from '@vaadin/testing-helpers';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { Tooltip } from '../src/vaadin-tooltip.js';
 
 registerStyles(
-  'vaadin-tooltip-overlay',
+  'vaadin24-tooltip-overlay',
   css`
     [part='overlay'] {
       width: 50px;
@@ -23,10 +23,10 @@
   });
 
   beforeEach(() => {
-    tooltip = fixtureSync('<vaadin-tooltip text="tooltip"></vaadin-tooltip>');
+    tooltip = fixtureSync('<vaadin24-tooltip text="tooltip"></vaadin24-tooltip>');
     target = fixtureSync('<div style="width: 100px; height: 100px; margin: 100px; outline: 1px solid red;"></div>');
     tooltip.target = target;
-    overlay = tooltip.shadowRoot.querySelector('vaadin-tooltip-overlay');
+    overlay = tooltip.shadowRoot.querySelector('vaadin24-tooltip-overlay');
   });
 
   async function openAndMeasure() {
