Index: theme/lumo/vaadin-popover-styles.js
===================================================================
--- theme/lumo/vaadin-popover-styles.js
+++ theme/lumo/vaadin-popover-styles.js
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { overlay } from '@vaadin/vaadin-lumo-styles/mixins/overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { overlay } from '@scoped-vaadin/vaadin-lumo-styles/mixins/overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const popoverOverlay = css`
   :host {
@@ -108,4 +108,4 @@
   }
 `;
 
-registerStyles('vaadin-popover-overlay', [overlay, popoverOverlay], { moduleId: 'lumo-popover-overlay' });
+registerStyles('vaadin24-popover-overlay', [overlay, popoverOverlay], { moduleId: 'lumo-popover-overlay' });
