Index: vaadin-list-mixin.js
===================================================================
--- vaadin-list-mixin.js
+++ vaadin-list-mixin.js
@@ -4,10 +4,10 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { FlattenedNodesObserver } from '@polymer/polymer/lib/utils/flattened-nodes-observer.js';
-import { timeOut } from '@vaadin/component-base/src/async.js';
-import { Debouncer } from '@vaadin/component-base/src/debounce.js';
-import { DirHelper } from '@vaadin/component-base/src/dir-helper.js';
-import { KeyboardDirectionMixin } from '@vaadin/component-base/src/keyboard-direction-mixin.js';
+import { timeOut } from '@scoped-vaadin/component-base/src/async.js';
+import { Debouncer } from '@scoped-vaadin/component-base/src/debounce.js';
+import { DirHelper } from '@scoped-vaadin/component-base/src/dir-helper.js';
+import { KeyboardDirectionMixin } from '@scoped-vaadin/component-base/src/keyboard-direction-mixin.js';
 
 /**
  * A mixin for `nav` elements, facilitating navigation and selection of childNodes.
@@ -56,7 +56,7 @@
          *
          * The item elements must implement `Vaadin.ItemMixin`.
          *
-         * Note: unlike `<vaadin-combo-box>`, this property is read-only,
+         * Note: unlike `<vaadin23-combo-box>`, this property is read-only,
          * so if you want to provide items by iterating array of data,
          * you have to use `dom-repeat` and place it to the light DOM.
          * @type {!Array<!Element> | undefined}
