Index: test/dom/accordion-heading.test.js
===================================================================
--- test/dom/accordion-heading.test.js
+++ test/dom/accordion-heading.test.js
@@ -2,11 +2,11 @@
 import { fixtureSync } from '@vaadin/testing-helpers';
 import '../../src/vaadin-accordion-heading.js';
 
-describe('vaadin-accordion-heading', () => {
+describe('vaadin24-accordion-heading', () => {
   let heading;
 
   beforeEach(() => {
-    heading = fixtureSync('<vaadin-accordion-heading>Heading</vaadin-accordion-heading>');
+    heading = fixtureSync('<vaadin24-accordion-heading>Heading</vaadin24-accordion-heading>');
   });
 
   describe('host', () => {
