Index: checkbox-group/theme/material/vaadin-checkbox-group.js
===================================================================
--- checkbox-group/theme/material/vaadin-checkbox-group.js
+++ checkbox-group/theme/material/vaadin-checkbox-group.js
@@ -1,3 +1,3 @@
-import '@vaadin/checkbox/theme/material/vaadin-checkbox.js';
+import '@scoped-vaadin/checkbox/theme/material/vaadin-checkbox.js';
 import './vaadin-checkbox-group-styles.js';
 import '../../src/vaadin-checkbox-group.js';
