Index: src/vaadin-lit-menu-bar-overlay.js
===================================================================
--- src/vaadin-lit-menu-bar-overlay.js
+++ src/vaadin-lit-menu-bar-overlay.js
@@ -4,17 +4,17 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { MenuOverlayMixin } from '@vaadin/context-menu/src/vaadin-menu-overlay-mixin.js';
-import { styles } from '@vaadin/context-menu/src/vaadin-menu-overlay-styles.js';
-import { OverlayMixin } from '@vaadin/overlay/src/vaadin-overlay-mixin.js';
-import { overlayStyles } from '@vaadin/overlay/src/vaadin-overlay-styles.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { MenuOverlayMixin } from '@scoped-vaadin/context-menu/src/vaadin-menu-overlay-mixin.js';
+import { styles } from '@scoped-vaadin/context-menu/src/vaadin-menu-overlay-styles.js';
+import { OverlayMixin } from '@scoped-vaadin/overlay/src/vaadin-overlay-mixin.js';
+import { overlayStyles } from '@scoped-vaadin/overlay/src/vaadin-overlay-styles.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * An element used internally by `<vaadin-menu-bar>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-menu-bar>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -26,7 +26,7 @@
  */
 export class MenuBarOverlay extends MenuOverlayMixin(OverlayMixin(DirMixin(ThemableMixin(PolylitMixin(LitElement))))) {
   static get is() {
-    return 'vaadin-menu-bar-overlay';
+    return 'vaadin24-menu-bar-overlay';
   }
 
   static get styles() {
