Index: src/vaadin-menu-bar-submenu.js
===================================================================
--- src/vaadin-menu-bar-submenu.js
+++ src/vaadin-menu-bar-submenu.js
@@ -1,19 +1,20 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2019 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ContextMenu } from '@vaadin/context-menu/src/vaadin-context-menu.js';
+import { ContextMenu } from '@scoped-vaadin/context-menu/src/vaadin-context-menu.js';
 
 /**
- * An element used internally by `<vaadin-menu-bar>`. Not intended to be used separately.
+ * An element used internally by `<vaadin23-menu-bar>`. Not intended to be used separately.
  *
  * @extends ContextMenu
  * @protected
  */
 class MenuBarSubmenu extends ContextMenu {
   static get is() {
-    return 'vaadin-menu-bar-submenu';
+    return 'vaadin23-menu-bar-submenu';
   }
 
   constructor() {
@@ -42,4 +43,4 @@
   }
 }
 
-customElements.define(MenuBarSubmenu.is, MenuBarSubmenu);
+internalCustomElements.define(MenuBarSubmenu.is, MenuBarSubmenu);
