Index: menu-bar/src/vaadin-menu-bar-mixin.d.ts
===================================================================
--- menu-bar/src/vaadin-menu-bar-mixin.d.ts	null
+++ menu-bar/src/vaadin-menu-bar-mixin.d.ts	null
@@ -4,12 +4,12 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { DisabledMixinClass } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin.js';
-import type { KeyboardDirectionMixinClass } from '@vaadin/a11y-base/src/keyboard-direction-mixin.js';
-import type { KeyboardMixinClass } from '@vaadin/a11y-base/src/keyboard-mixin.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { ResizeMixinClass } from '@vaadin/component-base/src/resize-mixin.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import type { KeyboardDirectionMixinClass } from '@scoped-vaadin/a11y-base/src/keyboard-direction-mixin.js';
+import type { KeyboardMixinClass } from '@scoped-vaadin/a11y-base/src/keyboard-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { ResizeMixinClass } from '@scoped-vaadin/component-base/src/resize-mixin.js';
 
 export interface MenuBarItem {
   /**
@@ -18,12 +18,12 @@
   text?: string;
   /**
    * Text to be set as the menu button's tooltip.
-   * Requires a `<vaadin-tooltip slot="tooltip">` element to be added inside the `<vaadin-menu-bar>`.
+   * Requires a `<vaadin24-tooltip slot="tooltip">` element to be added inside the `<vaadin24-menu-bar>`.
    */
   tooltip?: string;
   /**
    * The component to represent the button content.
-   * Either a tagName or an element instance. Defaults to "vaadin-menu-bar-item".
+   * Either a tagName or an element instance. Defaults to "vaadin24-menu-bar-item".
    */
   component?: HTMLElement | string;
   /**
