Index: src/vaadin-menu-bar-submenu-mixin.js
===================================================================
--- src/vaadin-menu-bar-submenu-mixin.js
+++ src/vaadin-menu-bar-submenu-mixin.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2019 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { OverlayClassMixin } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import { ContextMenuMixin } from '@vaadin/context-menu/src/vaadin-context-menu-mixin.js';
+import { OverlayClassMixin } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import { ContextMenuMixin } from '@scoped-vaadin/context-menu/src/vaadin-context-menu-mixin.js';
 
 /**
  * @polymerMixin
@@ -25,7 +25,7 @@
      * @return {string}
      */
     get _tagNamePrefix() {
-      return 'vaadin-menu-bar';
+      return 'vaadin24-menu-bar';
     }
 
     /**
