Index: src/vaadin-lit-menu-bar-button.js
===================================================================
--- src/vaadin-lit-menu-bar-button.js
+++ src/vaadin-lit-menu-bar-button.js
@@ -3,12 +3,12 @@
  * Copyright (c) 2019 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { Button } from '@vaadin/button/src/vaadin-lit-button.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { Button } from '@scoped-vaadin/button/src/vaadin-lit-button.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { css } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * An element used internally by `<vaadin-menu-bar>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-menu-bar>`. Not intended to be used separately.
  *
  * @customElement
  * @extends Button
@@ -16,7 +16,7 @@
  */
 class MenuBarButton extends Button {
   static get is() {
-    return 'vaadin-menu-bar-button';
+    return 'vaadin24-menu-bar-button';
   }
 
   static get styles() {
