Index: src/vaadin-menu-bar.d.ts
===================================================================
--- src/vaadin-menu-bar.d.ts
+++ src/vaadin-menu-bar.d.ts
@@ -3,9 +3,9 @@
  * Copyright (c) 2019 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { DisabledMixin } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DisabledMixin } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { type MenuBarItem, MenuBarMixin } from './vaadin-menu-bar-mixin.js';
 
 export { MenuBarItem, MenuBarI18n, SubMenuItem } from './vaadin-menu-bar-mixin.js';
@@ -22,20 +22,20 @@
 export interface MenuBarEventMap extends HTMLElementEventMap, MenuBarCustomEventMap {}
 
 /**
- * `<vaadin-menu-bar>` is a Web Component providing a set of horizontally stacked buttons offering
+ * `<vaadin24-menu-bar>` is a Web Component providing a set of horizontally stacked buttons offering
  * the user quick access to a consistent set of commands. Each button can toggle a submenu with
  * support for additional levels of nested menus.
  *
  * To create the menu bar, first add the component to the page:
  *
  * ```
- * <vaadin-menu-bar></vaadin-menu-bar>
+ * <vaadin24-menu-bar></vaadin24-menu-bar>
  * ```
  *
  * And then use [`items`](#/elements/vaadin-menu-bar#property-items) property to initialize the structure:
  *
  * ```
- * document.querySelector('vaadin-menu-bar').items = [{text: 'File'}, {text: 'Edit'}];
+ * document.querySelector('vaadin24-menu-bar').items = [{text: 'File'}, {text: 'Edit'}];
  * ```
  *
  * ### Styling
@@ -57,13 +57,13 @@
  *
  * ### Internal components
  *
- * In addition to `<vaadin-menu-bar>` itself, the following internal
+ * In addition to `<vaadin24-menu-bar>` itself, the following internal
  * components are themable:
  *
- * - `<vaadin-menu-bar-button>` - has the same API as [`<vaadin-button>`](#/elements/vaadin-button).
- * - `<vaadin-menu-bar-item>` - has the same API as [`<vaadin-item>`](#/elements/vaadin-item).
- * - `<vaadin-menu-bar-list-box>` - has the same API as [`<vaadin-list-box>`](#/elements/vaadin-list-box).
- * - `<vaadin-menu-bar-overlay>` - has the same API as [`<vaadin-overlay>`](#/elements/vaadin-overlay).
+ * - `<vaadin24-menu-bar-button>` - has the same API as [`<vaadin24-button>`](#/elements/vaadin-button).
+ * - `<vaadin24-menu-bar-item>` - has the same API as [`<vaadin24-item>`](#/elements/vaadin-item).
+ * - `<vaadin24-menu-bar-list-box>` - has the same API as [`<vaadin24-list-box>`](#/elements/vaadin-list-box).
+ * - `<vaadin24-menu-bar-overlay>` - has the same API as [`<vaadin24-overlay>`](#/elements/vaadin-overlay).
  *
  * @fires {CustomEvent} item-selected - Fired when a submenu item or menu bar button without children is clicked.
  */
@@ -83,7 +83,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-menu-bar': MenuBar;
+    'vaadin24-menu-bar': MenuBar;
   }
 }
 
