Index: src/vaadin-menu-bar-interactions-mixin.d.ts
===================================================================
--- src/vaadin-menu-bar-interactions-mixin.d.ts
+++ src/vaadin-menu-bar-interactions-mixin.d.ts
@@ -4,9 +4,9 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { FocusMixinClass } from '@vaadin/component-base/src/focus-mixin.js';
-import type { KeyboardDirectionMixinClass } from '@vaadin/component-base/src/keyboard-direction-mixin.js';
-import type { KeyboardMixinClass } from '@vaadin/component-base/src/keyboard-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/component-base/src/focus-mixin.js';
+import type { KeyboardDirectionMixinClass } from '@scoped-vaadin/component-base/src/keyboard-direction-mixin.js';
+import type { KeyboardMixinClass } from '@scoped-vaadin/component-base/src/keyboard-mixin.js';
 
 export declare function InteractionsMixin<T extends Constructor<HTMLElement>>(
   base: T,
