Index: src/vaadin-menu-bar-buttons-mixin.d.ts
===================================================================
--- src/vaadin-menu-bar-buttons-mixin.d.ts
+++ src/vaadin-menu-bar-buttons-mixin.d.ts
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { ResizeMixinClass } from '@vaadin/component-base/src/resize-mixin.js';
+import type { ResizeMixinClass } from '@scoped-vaadin/component-base/src/resize-mixin.js';
 
 export declare function ButtonsMixin<T extends Constructor<HTMLElement>>(
   base: T,
