Index: src/vaadin-menu-bar-button.js
===================================================================
--- src/vaadin-menu-bar-button.js
+++ src/vaadin-menu-bar-button.js
@@ -3,12 +3,12 @@
  * Copyright (c) 2019 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { Button } from '@vaadin/button/src/vaadin-button.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { Button } from '@scoped-vaadin/button/src/vaadin-button.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-menu-bar-button',
+  'vaadin24-menu-bar-button',
   css`
     :host {
       flex-shrink: 0;
@@ -22,7 +22,7 @@
 );
 
 /**
- * An element used internally by `<vaadin-menu-bar>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-menu-bar>`. Not intended to be used separately.
  *
  * @customElement
  * @extends Button
@@ -30,7 +30,7 @@
  */
 class MenuBarButton extends Button {
   static get is() {
-    return 'vaadin-menu-bar-button';
+    return 'vaadin24-menu-bar-button';
   }
 
   /**
