Index: overlay/src/vaadin-overlay-focus-mixin.d.ts
===================================================================
--- overlay/src/vaadin-overlay-focus-mixin.d.ts	null
+++ overlay/src/vaadin-overlay-focus-mixin.d.ts	null
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
 
 export declare function OverlayFocusMixin<T extends Constructor<HTMLElement>>(
   base: T,
