Index: src/vaadin-lit-overlay.js
===================================================================
--- src/vaadin-lit-overlay.js
+++ src/vaadin-lit-overlay.js
@@ -4,15 +4,15 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { OverlayMixin } from './vaadin-overlay-mixin.js';
 import { overlayStyles } from './vaadin-overlay-styles.js';
 
 /**
- * LitElement based version of `<vaadin-overlay>` web component.
+ * LitElement based version of `<vaadin24-overlay>` web component.
  *
  * ## Disclaimer
  *
@@ -22,7 +22,7 @@
  */
 class Overlay extends OverlayMixin(DirMixin(ThemableMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-overlay';
+    return 'vaadin24-overlay';
   }
 
   static get styles() {
