Index: dialog/theme/lumo/vaadin-dialog-styles.js
===================================================================
--- dialog/theme/lumo/vaadin-dialog-styles.js
+++ dialog/theme/lumo/vaadin-dialog-styles.js
@@ -1,6 +1,6 @@
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { overlay } from '@vaadin/vaadin-lumo-styles/mixins/overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { overlay } from '@scoped-vaadin/vaadin-lumo-styles/mixins/overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const dialogOverlay = css`
   /* Optical centering */
@@ -102,6 +102,6 @@
   }
 `;
 
-registerStyles('vaadin-dialog-overlay', [overlay, dialogOverlay], { moduleId: 'lumo-dialog' });
+registerStyles('vaadin24-dialog-overlay', [overlay, dialogOverlay], { moduleId: 'lumo-dialog' });
 
 export { dialogOverlay };
