Index: src/vaadin-lit-vertical-layout.js
===================================================================
--- src/vaadin-lit-vertical-layout.js
+++ src/vaadin-lit-vertical-layout.js
@@ -4,13 +4,13 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { css, html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-vertical-layout>` web component.
+ * LitElement based version of `<vaadin24-vertical-layout>` web component.
  *
  * ## Disclaimer
  *
@@ -48,7 +48,7 @@
   }
 
   static get is() {
-    return 'vaadin-vertical-layout';
+    return 'vaadin24-vertical-layout';
   }
 
   render() {
