Index: test/visual/material/context-menu.test.js
===================================================================
--- test/visual/material/context-menu.test.js
+++ test/visual/material/context-menu.test.js
@@ -33,16 +33,16 @@
       describe('basic', () => {
         beforeEach(() => {
           element = fixtureSync(`
-            <vaadin-context-menu>
+            <vaadin24-context-menu>
               <div style="padding: 10px">Target</div>
-            </vaadin-context-menu>
+            </vaadin24-context-menu>
           `);
           element.renderer = (root) => {
             root.innerHTML = `
-              <vaadin-list-box>
-                <vaadin-item>Item 1</vaadin-item>
-                <vaadin-item>Item 2</vaadin-item>
-              </vaadin-list-box>
+              <vaadin24-list-box>
+                <vaadin24-item>Item 1</vaadin24-item>
+                <vaadin24-item>Item 2</vaadin24-item>
+              </vaadin24-list-box>
             `;
           };
         });
@@ -56,18 +56,18 @@
       describe('long', () => {
         beforeEach(() => {
           element = fixtureSync(`
-            <vaadin-context-menu>
+            <vaadin24-context-menu>
               <div style="padding: 10px">Target</div>
-            </vaadin-context-menu>
+            </vaadin24-context-menu>
           `);
           element.renderer = (root) => {
             root.innerHTML = `
-              <vaadin-list-box>
+              <vaadin24-list-box>
                 ${new Array(30)
                   .fill(0)
-                  .map((_, idx) => `<vaadin-item>Item ${idx}</vaadin-item>`)
+                  .map((_, idx) => `<vaadin24-item>Item ${idx}</vaadin24-item>`)
                   .join('')}
-              </vaadin-list-box>
+              </vaadin24-list-box>
             `;
           };
         });
@@ -89,9 +89,9 @@
       describe('items', () => {
         beforeEach(() => {
           element = fixtureSync(`
-            <vaadin-context-menu>
+            <vaadin24-context-menu>
               <div style="padding: 10px">Target</div>
-            </vaadin-context-menu>
+            </vaadin24-context-menu>
           `);
         });
 
