Index: theme/material/vaadin-field-outline-styles.js
===================================================================
--- theme/material/vaadin-field-outline-styles.js
+++ theme/material/vaadin-field-outline-styles.js
@@ -3,10 +3,10 @@
  * Copyright (c) 2021 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-field-outline',
+  'vaadin24-field-outline',
   css`
     :host {
       transition: opacity 0.3s;
