Index: test/visual/lumo/vertical-layout.test.js
===================================================================
--- test/visual/lumo/vertical-layout.test.js
+++ test/visual/lumo/vertical-layout.test.js
@@ -10,10 +10,10 @@
     div.style.display = 'flex';
     element = fixtureSync(
       `
-        <vaadin-vertical-layout style="border: solid 2px blue">
+        <vaadin23-vertical-layout style="border: solid 2px blue">
           <div style="background: #e2e2e2; padding: 20px;">Item 1</div>
           <div style="background: #f3f3f3; padding: 20px;">Item 2</div>
-        </vaadin-vertical-layout>
+        </vaadin23-vertical-layout>
       `,
       div,
     );
