Index: theme/lumo/vaadin-multi-select-combo-box-styles.js
===================================================================
--- theme/lumo/vaadin-multi-select-combo-box-styles.js
+++ theme/lumo/vaadin-multi-select-combo-box-styles.js
@@ -3,19 +3,19 @@
  * Copyright (c) 2021 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/theme/lumo/vaadin-input-container-styles.js';
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { comboBoxItem } from '@vaadin/combo-box/theme/lumo/vaadin-combo-box-item-styles.js';
-import { comboBoxLoader, comboBoxOverlay } from '@vaadin/combo-box/theme/lumo/vaadin-combo-box-overlay-styles.js';
-import { item } from '@vaadin/item/theme/lumo/vaadin-item-styles.js';
-import { inputFieldShared } from '@vaadin/vaadin-lumo-styles/mixins/input-field-shared.js';
-import { loader } from '@vaadin/vaadin-lumo-styles/mixins/loader.js';
-import { menuOverlayCore } from '@vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
-import { overlay } from '@vaadin/vaadin-lumo-styles/mixins/overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/input-container/theme/lumo/vaadin-input-container-styles.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { comboBoxItem } from '@scoped-vaadin/combo-box/theme/lumo/vaadin-combo-box-item-styles.js';
+import { comboBoxLoader, comboBoxOverlay } from '@scoped-vaadin/combo-box/theme/lumo/vaadin-combo-box-overlay-styles.js';
+import { item } from '@scoped-vaadin/item/theme/lumo/vaadin-item-styles.js';
+import { inputFieldShared } from '@scoped-vaadin/vaadin-lumo-styles/mixins/input-field-shared.js';
+import { loader } from '@scoped-vaadin/vaadin-lumo-styles/mixins/loader.js';
+import { menuOverlayCore } from '@scoped-vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
+import { overlay } from '@scoped-vaadin/vaadin-lumo-styles/mixins/overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const multiSelectComboBoxItem = css`
   @media (any-hover: hover) {
@@ -26,12 +26,12 @@
   }
 `;
 
-registerStyles('vaadin-multi-select-combo-box-item', [item, comboBoxItem, multiSelectComboBoxItem], {
+registerStyles('vaadin24-multi-select-combo-box-item', [item, comboBoxItem, multiSelectComboBoxItem], {
   moduleId: 'lumo-multi-select-combo-box-item',
 });
 
 registerStyles(
-  'vaadin-multi-select-combo-box-overlay',
+  'vaadin24-multi-select-combo-box-overlay',
   [
     overlay,
     menuOverlayCore,
@@ -49,7 +49,7 @@
 );
 
 registerStyles(
-  'vaadin-multi-select-combo-box-container',
+  'vaadin24-multi-select-combo-box-container',
   css`
     :host([auto-expand-vertically]) {
       padding-block: var(--lumo-space-xs);
@@ -113,6 +113,6 @@
   }
 `;
 
-registerStyles('vaadin-multi-select-combo-box', [inputFieldShared, multiSelectComboBox], {
+registerStyles('vaadin24-multi-select-combo-box', [inputFieldShared, multiSelectComboBox], {
   moduleId: 'lumo-multi-select-combo-box',
 });
