Index: src/vaadin-message-input.d.ts
===================================================================
--- src/vaadin-message-input.d.ts
+++ src/vaadin-message-input.d.ts
@@ -3,14 +3,14 @@
  * Copyright (c) 2021 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { MessageInputMixin } from './vaadin-message-input-mixin.js';
 
 export { MessageInputI18n } from './vaadin-message-input-mixin.js';
 
 /**
- * Fired when a new message is submitted with `<vaadin-message-input>`, either
+ * Fired when a new message is submitted with `<vaadin24-message-input>`, either
  * by clicking the "send" button, or pressing the Enter key.
  */
 export type MessageInputSubmitEvent = CustomEvent<{ value: string }>;
@@ -22,7 +22,7 @@
 export type MessageInputEventMap = HTMLElementEventMap & MessageInputCustomEventMap;
 
 /**
- * `<vaadin-message-input>` is a Web Component for sending messages.
+ * `<vaadin24-message-input>` is a Web Component for sending messages.
  * It consists of a text area that grows on along with the content, and a send button to send message.
  *
  * The message can be sent by one of the following actions:
@@ -30,7 +30,7 @@
  * - by clicking `submit` button.
  *
  * ```html
- * <vaadin-message-input></vaadin-message-input>
+ * <vaadin24-message-input></vaadin24-message-input>
  * ```
  */
 declare class MessageInput extends MessageInputMixin(ThemableMixin(ElementMixin(HTMLElement))) {
@@ -49,7 +49,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-message-input': MessageInput;
+    'vaadin24-message-input': MessageInput;
   }
 }
 
