Index: src/vaadin-message-input.js
===================================================================
--- src/vaadin-message-input.js
+++ src/vaadin-message-input.js
@@ -3,16 +3,16 @@
  * Copyright (c) 2021 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/button/src/vaadin-button.js';
-import '@vaadin/text-area/src/vaadin-text-area.js';
+import '@scoped-vaadin/button/src/vaadin-button.js';
+import '@scoped-vaadin/text-area/src/vaadin-text-area.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { MessageInputMixin } from './vaadin-message-input-mixin.js';
 
 /**
- * `<vaadin-message-input>` is a Web Component for sending messages.
+ * `<vaadin24-message-input>` is a Web Component for sending messages.
  * It consists of a text area that grows on along with the content, and a send button to send message.
  *
  * The message can be sent by one of the following actions:
@@ -20,7 +20,7 @@
  * - by clicking `submit` button.
  *
  * ```html
- * <vaadin-message-input></vaadin-message-input>
+ * <vaadin24-message-input></vaadin24-message-input>
  * ```
  *
  * @customElement
@@ -64,7 +64,7 @@
   }
 
   static get is() {
-    return 'vaadin-message-input';
+    return 'vaadin24-message-input';
   }
 }
 
