Index: src/vaadin-message-input-mixin.js
===================================================================
--- src/vaadin-message-input-mixin.js
+++ src/vaadin-message-input-mixin.js
@@ -3,10 +3,10 @@
  * Copyright (c) 2021 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { I18nMixin } from '@vaadin/component-base/src/i18n-mixin.js';
-import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
-import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { I18nMixin } from '@scoped-vaadin/component-base/src/i18n-mixin.js';
+import { SlotController } from '@scoped-vaadin/component-base/src/slot-controller.js';
+import { TooltipController } from '@scoped-vaadin/component-base/src/tooltip-controller.js';
 
 const DEFAULT_I18N = {
   send: 'Send',
@@ -91,7 +91,7 @@
     ready() {
       super.ready();
 
-      this._buttonController = new SlotController(this, 'button', 'vaadin-button', {
+      this._buttonController = new SlotController(this, 'button', 'vaadin24-button', {
         initializer: (btn) => {
           btn.setAttribute('theme', 'primary contained');
 
@@ -104,7 +104,7 @@
       });
       this.addController(this._buttonController);
 
-      this._textAreaController = new SlotController(this, 'textarea', 'vaadin-text-area', {
+      this._textAreaController = new SlotController(this, 'textarea', 'vaadin24-text-area', {
         initializer: (textarea) => {
           textarea.addEventListener('value-changed', (event) => {
             this.value = event.detail.value;
@@ -170,7 +170,7 @@
     }
 
     /**
-     * Fired when a new message is submitted with `<vaadin-message-input>`, either
+     * Fired when a new message is submitted with `<vaadin24-message-input>`, either
      * by clicking the "send" button, or pressing the Enter key.
      * @event submit
      */
