Index: test/accordion-panel.common.js
===================================================================
--- test/accordion-panel.common.js
+++ test/accordion-panel.common.js
@@ -3,7 +3,7 @@
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
 
-describe('vaadin-accordion-panel', () => {
+describe('vaadin24-accordion-panel', () => {
   let panel;
 
   describe('opened', () => {
@@ -11,9 +11,9 @@
 
     beforeEach(async () => {
       panel = fixtureSync(`
-        <vaadin-accordion-panel>
+        <vaadin24-accordion-panel>
           <div>Content</div>
-        </vaadin-accordion-panel>
+        </vaadin24-accordion-panel>
       `);
       await nextRender();
       contentPart = panel.shadowRoot.querySelector('[part="content"]');
@@ -54,15 +54,15 @@
   ['default', 'custom'].forEach((type) => {
     const fixtures = {
       default: `
-        <vaadin-accordion-panel summary="Summary">
+        <vaadin24-accordion-panel summary="Summary">
           <div>Content</div>
-        </vaadin-accordion-panel>
+        </vaadin24-accordion-panel>
       `,
       custom: `
-        <vaadin-accordion-panel>
-          <vaadin-accordion-heading slot="summary">Summary</vaadin-accordion-heading>
+        <vaadin24-accordion-panel>
+          <vaadin24-accordion-heading slot="summary">Summary</vaadin24-accordion-heading>
           <div>Content</div>
-        </vaadin-accordion-panel>
+        </vaadin24-accordion-panel>
       `,
     };
 
@@ -166,13 +166,13 @@
 
     beforeEach(async () => {
       panel = fixtureSync(`
-        <vaadin-accordion-panel>
-          <vaadin-accordion-heading slot="summary">
+        <vaadin24-accordion-panel>
+          <vaadin24-accordion-heading slot="summary">
             Toggle
             <a href="#">Link</a>
-          </vaadin-accordion-heading>
+          </vaadin24-accordion-heading>
           <div>Content</div>
-        </vaadin-accordion-panel>
+        </vaadin24-accordion-panel>
       `);
       await nextRender();
       link = panel.querySelector('a');
