Index: utilities/background.js
===================================================================
--- utilities/background.js
+++ utilities/background.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { css } from '@scoped-vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const background = css`
   /* === Background color === */
