Index: test/visual/lumo/number-field.test.js
===================================================================
--- test/visual/lumo/number-field.test.js
+++ test/visual/lumo/number-field.test.js
@@ -11,7 +11,7 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-number-field></vaadin-number-field>', div);
+    element = fixtureSync('<vaadin24-number-field></vaadin24-number-field>', div);
   });
 
   it('basic', async () => {
