Index: src/fields/vaadin-component-observer.js
===================================================================
--- src/fields/vaadin-component-observer.js
+++ src/fields/vaadin-component-observer.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { timeOut } from '@vaadin/component-base/src/async.js';
-import { Debouncer } from '@vaadin/component-base/src/debounce.js';
+import { timeOut } from '@scoped-vaadin/component-base/src/async.js';
+import { Debouncer } from '@scoped-vaadin/component-base/src/debounce.js';
 import { initOutline } from './outline.js';
 
 export class ComponentObserver {
@@ -26,7 +26,7 @@
   }
 
   initTags(component) {
-    const tags = document.createElement('vaadin-user-tags');
+    const tags = document.createElement('vaadin24-user-tags');
     component.shadowRoot.appendChild(tags);
     tags.target = component;
     this._tags = tags;
