Index: src/lit/renderer-directives.d.ts
===================================================================
--- src/lit/renderer-directives.d.ts
+++ src/lit/renderer-directives.d.ts
@@ -5,7 +5,7 @@
  */
 import type { TemplateResult } from 'lit';
 import type { DirectiveResult } from 'lit/directive.js';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 import type { ComboBox, ComboBoxItemModel } from '../vaadin-combo-box.js';
 
 export type ComboBoxLitRenderer<TItem> = (
@@ -32,7 +32,7 @@
 }
 
 /**
- * A Lit directive for rendering the content of the `<vaadin-combo-box-item>` elements.
+ * A Lit directive for rendering the content of the `<vaadin23-combo-box-item>` elements.
  *
  * The directive accepts a renderer callback returning a Lit template and assigns it to the combo-box
  * via the `renderer` property. The renderer is called for each combo-box item when assigned
@@ -48,9 +48,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-combo-box
+ * `<vaadin23-combo-box
  *   ${comboBoxRenderer((item, model, comboBox) => html`...`)}
- * ></vaadin-combo-box>`
+ * ></vaadin23-combo-box>`
  * ```
  *
  * @param renderer the renderer callback that returns a Lit template.
