Index: custom-field/theme/material/vaadin-custom-field-styles.js
===================================================================
--- custom-field/theme/material/vaadin-custom-field-styles.js
+++ custom-field/theme/material/vaadin-custom-field-styles.js
@@ -3,11 +3,11 @@
  * Copyright (c) 2019 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { helper } from '@vaadin/vaadin-material-styles/mixins/helper.js';
-import { requiredField } from '@vaadin/vaadin-material-styles/mixins/required-field.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { helper } from '@scoped-vaadin/vaadin-material-styles/mixins/helper.js';
+import { requiredField } from '@scoped-vaadin/vaadin-material-styles/mixins/required-field.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const customField = css`
   :host {
@@ -44,7 +44,7 @@
   }
 `;
 
-registerStyles('vaadin-custom-field', [requiredField, helper, customField], {
+registerStyles('vaadin24-custom-field', [requiredField, helper, customField], {
   moduleId: 'material-custom-field',
 });
 
