Index: src/vaadin-field-outline.js
===================================================================
--- src/vaadin-field-outline.js
+++ src/vaadin-field-outline.js
@@ -4,12 +4,12 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * An element used internally by `<vaadin-field-highlighter>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-field-highlighter>`. Not intended to be used separately.
  *
  * @extends HTMLElement
  * @mixes DirMixin
@@ -18,7 +18,7 @@
  */
 export class FieldOutline extends ThemableMixin(DirMixin(PolymerElement)) {
   static get is() {
-    return 'vaadin-field-outline';
+    return 'vaadin24-field-outline';
   }
 
   static get template() {
