Index: src/vaadin-field-outline.js
===================================================================
--- src/vaadin-field-outline.js
+++ src/vaadin-field-outline.js
@@ -4,16 +4,16 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { fieldOutlineStyles } from './vaadin-field-highlighter-styles.js';
 
-registerStyles('vaadin-field-outline', fieldOutlineStyles, { moduleId: 'vaadin-field-outline-styles' });
+registerStyles('vaadin24-field-outline', fieldOutlineStyles, { moduleId: 'vaadin-field-outline-styles' });
 
 /**
- * An element used internally by `<vaadin-field-highlighter>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-field-highlighter>`. Not intended to be used separately.
  *
  * @extends HTMLElement
  * @mixes DirMixin
@@ -22,7 +22,7 @@
  */
 export class FieldOutline extends ThemableMixin(DirMixin(PolylitMixin(LitElement))) {
   static get is() {
-    return 'vaadin-field-outline';
+    return 'vaadin24-field-outline';
   }
 
   static get properties() {
