Index: src/vaadin-user-tag.js
===================================================================
--- src/vaadin-user-tag.js
+++ src/vaadin-user-tag.js
@@ -4,14 +4,14 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { userTagStyles } from './vaadin-field-highlighter-styles.js';
 
 /**
- * An element used internally by `<vaadin-field-highlighter>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-field-highlighter>`. Not intended to be used separately.
  *
  * @customElement
  * @extends HTMLElement
@@ -21,7 +21,7 @@
  */
 export class UserTag extends ThemableMixin(DirMixin(PolylitMixin(LitElement))) {
   static get is() {
-    return 'vaadin-user-tag';
+    return 'vaadin24-user-tag';
   }
 
   static get styles() {
