Index: src/vaadin-field-outline.js
===================================================================
--- src/vaadin-field-outline.js
+++ src/vaadin-field-outline.js
@@ -1,14 +1,15 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * An element used internally by `<vaadin-field-highlighter>`. Not intended to be used separately.
+ * An element used internally by `<vaadin23-field-highlighter>`. Not intended to be used separately.
  *
  * @extends HTMLElement
  * @mixes DirMixin
@@ -17,7 +18,7 @@
  */
 export class FieldOutline extends ThemableMixin(DirMixin(PolymerElement)) {
   static get is() {
-    return 'vaadin-field-outline';
+    return 'vaadin23-field-outline';
   }
 
   static get template() {
@@ -85,4 +86,4 @@
   }
 }
 
-customElements.define(FieldOutline.is, FieldOutline);
+internalCustomElements.define(FieldOutline.is, FieldOutline);
