Index: item/src/vaadin-item.d.ts
===================================================================
--- item/src/vaadin-item.d.ts
+++ item/src/vaadin-item.d.ts
@@ -3,25 +3,25 @@
  * Copyright (c) 2017 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ItemMixin } from './vaadin-item-mixin.js';
 
 /**
- * `<vaadin-item>` is a Web Component providing layout for items in tabs and menus.
+ * `<vaadin24-item>` is a Web Component providing layout for items in tabs and menus.
  *
  * ```
- *   <vaadin-item>
+ *   <vaadin24-item>
  *     Item content
- *   </vaadin-item>
+ *   </vaadin24-item>
  * ```
  *
  * ### Selectable
  *
- * `<vaadin-item>` has the `selected` property and the corresponding state attribute.
+ * `<vaadin24-item>` has the `selected` property and the corresponding state attribute.
  * Currently, the component sets the `selected` to false, when `disabled` property is set to true.
- * But other than that, the `<vaadin-item>` does not switch selection by itself.
- * In general, it is the wrapper component, like `<vaadin-list-box>`, which should update
+ * But other than that, the `<vaadin24-item>` does not switch selection by itself.
+ * In general, it is the wrapper component, like `<vaadin24-list-box>`, which should update
  * the `selected` property on the items, e. g. on mousedown or when Enter / Spacebar is pressed.
  *
  * ### Styling
@@ -52,14 +52,14 @@
   value: string;
 
   /**
-   * String that can be set to visually represent the selected item in `vaadin-select`.
+   * String that can be set to visually represent the selected item in `vaadin24-select`.
    */
   label: string | undefined;
 }
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-item': Item;
+    'vaadin24-item': Item;
   }
 }
 
