Index: src/vaadin-lit-item.js
===================================================================
--- src/vaadin-lit-item.js
+++ src/vaadin-lit-item.js
@@ -4,14 +4,14 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { css, html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ItemMixin } from './vaadin-item-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-item>` web component.
+ * LitElement based version of `<vaadin24-item>` web component.
  *
  * ## Disclaimer
  *
@@ -21,7 +21,7 @@
  */
 class Item extends ItemMixin(ThemableMixin(DirMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-item';
+    return 'vaadin24-item';
   }
 
   static get styles() {
