Index: src/vaadin-item.js
===================================================================
--- src/vaadin-item.js
+++ src/vaadin-item.js
@@ -1,28 +1,29 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ItemMixin } from './vaadin-item-mixin.js';
 
 /**
- * `<vaadin-item>` is a Web Component providing layout for items in tabs and menus.
+ * `<vaadin23-item>` is a Web Component providing layout for items in tabs and menus.
  *
  * ```
- *   <vaadin-item>
+ *   <vaadin23-item>
  *     Item content
- *   </vaadin-item>
+ *   </vaadin23-item>
  * ```
  *
  * ### Selectable
  *
- * `<vaadin-item>` has the `selected` property and the corresponding state attribute.
+ * `<vaadin23-item>` has the `selected` property and the corresponding state attribute.
  * Currently, the component sets the `selected` to false, when `disabled` property is set to true.
- * But other than that, the `<vaadin-item>` does not switch selection by itself.
- * In general, it is the wrapper component, like `<vaadin-list-box>`, which should update
+ * But other than that, the `<vaadin23-item>` does not switch selection by itself.
+ * In general, it is the wrapper component, like `<vaadin23-list-box>`, which should update
  * the `selected` property on the items, e. g. on mousedown or when Enter / Spacebar is pressed.
  *
  * ### Styling
@@ -71,7 +72,7 @@
   }
 
   static get is() {
-    return 'vaadin-item';
+    return 'vaadin23-item';
   }
 
   constructor() {
@@ -84,7 +85,7 @@
     this.value; // eslint-disable-line no-unused-expressions
 
     /**
-     * String that can be set to visually represent the selected item in `vaadin-select`.
+     * String that can be set to visually represent the selected item in `vaadin23-select`.
      * @type {string}
      */
     this.label; // eslint-disable-line no-unused-expressions
@@ -98,6 +99,6 @@
   }
 }
 
-customElements.define(Item.is, Item);
+internalCustomElements.define(Item.is, Item);
 
 export { Item };
