Index: src/vaadin-item.js
===================================================================
--- src/vaadin-item.js
+++ src/vaadin-item.js
@@ -4,29 +4,29 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { itemStyles } from './vaadin-item-core-styles.js';
 import { ItemMixin } from './vaadin-item-mixin.js';
 
-registerStyles('vaadin-item', itemStyles, { moduleId: 'vaadin-item-styles' });
+registerStyles('vaadin24-item', itemStyles, { moduleId: 'vaadin-item-styles' });
 
 /**
- * `<vaadin-item>` is a Web Component providing layout for items in tabs and menus.
+ * `<vaadin24-item>` is a Web Component providing layout for items in tabs and menus.
  *
  * ```
- *   <vaadin-item>
+ *   <vaadin24-item>
  *     Item content
- *   </vaadin-item>
+ *   </vaadin24-item>
  * ```
  *
  * ### Selectable
  *
- * `<vaadin-item>` has the `selected` property and the corresponding state attribute.
+ * `<vaadin24-item>` has the `selected` property and the corresponding state attribute.
  * Currently, the component sets the `selected` to false, when `disabled` property is set to true.
- * But other than that, the `<vaadin-item>` does not switch selection by itself.
- * In general, it is the wrapper component, like `<vaadin-list-box>`, which should update
+ * But other than that, the `<vaadin24-item>` does not switch selection by itself.
+ * In general, it is the wrapper component, like `<vaadin24-list-box>`, which should update
  * the `selected` property on the items, e. g. on mousedown or when Enter / Spacebar is pressed.
  *
  * ### Styling
@@ -67,7 +67,7 @@
   }
 
   static get is() {
-    return 'vaadin-item';
+    return 'vaadin24-item';
   }
 
   constructor() {
@@ -80,7 +80,7 @@
     this.value;
 
     /**
-     * String that can be set to visually represent the selected item in `vaadin-select`.
+     * String that can be set to visually represent the selected item in `vaadin24-select`.
      * @type {string}
      */
     this.label;
