Index: test/radio-group-keyboard-navigation.common.js
===================================================================
--- test/radio-group-keyboard-navigation.common.js
+++ test/radio-group-keyboard-navigation.common.js
@@ -7,16 +7,16 @@
   beforeEach(async () => {
     group = fixtureSync(
       `<div>
-        <vaadin-radio-group>
-          <vaadin-radio-button label="Button 1" value="1"></vaadin-radio-button>
-          <vaadin-radio-button label="Button 2" value="2"></vaadin-radio-button>
-          <vaadin-radio-button label="Button 3" value="3"></vaadin-radio-button>
-        </vaadin-radio-group>
+        <vaadin24-radio-group>
+          <vaadin24-radio-button label="Button 1" value="1"></vaadin24-radio-button>
+          <vaadin24-radio-button label="Button 2" value="2"></vaadin24-radio-button>
+          <vaadin24-radio-button label="Button 3" value="3"></vaadin24-radio-button>
+        </vaadin24-radio-group>
         <input id="last-global-focusable" />
       </div>`,
     ).firstElementChild;
     await nextFrame();
-    buttons = [...group.querySelectorAll('vaadin-radio-button')];
+    buttons = [...group.querySelectorAll('vaadin24-radio-button')];
   });
 
   describe('Tab', () => {
