Index: test/radio-group-keyboard-navigation.common.js
===================================================================
--- test/radio-group-keyboard-navigation.common.js
+++ test/radio-group-keyboard-navigation.common.js
@@ -6,14 +6,14 @@
 
   beforeEach(async () => {
     group = fixtureSync(`
-      <vaadin-radio-group>
-        <vaadin-radio-button label="Button 1" value="1"></vaadin-radio-button>
-        <vaadin-radio-button label="Button 2" value="2"></vaadin-radio-button>
-        <vaadin-radio-button label="Button 3" value="3"></vaadin-radio-button>
-      </vaadin-radio-group>
+      <vaadin24-radio-group>
+        <vaadin24-radio-button label="Button 1" value="1"></vaadin24-radio-button>
+        <vaadin24-radio-button label="Button 2" value="2"></vaadin24-radio-button>
+        <vaadin24-radio-button label="Button 3" value="3"></vaadin24-radio-button>
+      </vaadin24-radio-group>
     `);
     await nextFrame();
-    buttons = [...group.querySelectorAll('vaadin-radio-button')];
+    buttons = [...group.querySelectorAll('vaadin24-radio-button')];
   });
 
   describe('Tab', () => {
