Index: test/validation.test.js
===================================================================
--- test/validation.test.js
+++ test/validation.test.js
@@ -9,9 +9,9 @@
 
   describe('initial', () => {
     beforeEach(() => {
-      group = document.createElement('vaadin-radio-group');
+      group = document.createElement('vaadin24-radio-group');
 
-      const radio = document.createElement('vaadin-radio-button');
+      const radio = document.createElement('vaadin24-radio-button');
       radio.value = '1';
       group.appendChild(radio);
 
@@ -51,10 +51,10 @@
       [firstGlobalFocusable, group] = fixtureSync(
         `<div>
           <input id="first-global-focusable" />
-          <vaadin-radio-group>
-            <vaadin-radio-button label="Button 1" value="1"></vaadin-radio-button>
-            <vaadin-radio-button label="Button 2" value="2"></vaadin-radio-button>
-          </vaadin-radio-group>
+          <vaadin24-radio-group>
+            <vaadin24-radio-button label="Button 1" value="1"></vaadin24-radio-button>
+            <vaadin24-radio-button label="Button 2" value="2"></vaadin24-radio-button>
+          </vaadin24-radio-group>
         </div>`,
       ).children;
       firstGlobalFocusable.focus();
@@ -67,7 +67,7 @@
     });
 
     it('should validate on radio button click', async () => {
-      group.querySelector('vaadin-radio-button').click();
+      group.querySelector('vaadin24-radio-button').click();
       await nextUpdate(group);
       expect(validateSpy.calledOnce).to.be.true;
     });
@@ -133,10 +133,10 @@
   describe('required', () => {
     beforeEach(async () => {
       group = fixtureSync(`
-        <vaadin-radio-group required>
-          <vaadin-radio-button label="Button 1" value="1"></vaadin-radio-button>
-          <vaadin-radio-button label="Button 2" value="2"></vaadin-radio-button>
-        </vaadin-radio-group>
+        <vaadin24-radio-group required>
+          <vaadin24-radio-button label="Button 1" value="1"></vaadin24-radio-button>
+          <vaadin24-radio-button label="Button 2" value="2"></vaadin24-radio-button>
+        </vaadin24-radio-group>
       `);
       await nextFrame();
     });
@@ -155,7 +155,7 @@
     });
 
     it('should be valid after selecting a radio button', () => {
-      group.querySelector('vaadin-radio-button').click();
+      group.querySelector('vaadin24-radio-button').click();
       expect(group.invalid).to.be.false;
     });
   });
