Index: test/validation.test.js
===================================================================
--- test/validation.test.js
+++ test/validation.test.js
@@ -9,9 +9,9 @@
 
   describe('initial', () => {
     beforeEach(() => {
-      group = document.createElement('vaadin-radio-group');
+      group = document.createElement('vaadin24-radio-group');
 
-      const radio = document.createElement('vaadin-radio-button');
+      const radio = document.createElement('vaadin24-radio-button');
       radio.value = '1';
       group.appendChild(radio);
 
@@ -47,10 +47,10 @@
   describe('basic', () => {
     beforeEach(async () => {
       group = fixtureSync(`
-        <vaadin-radio-group>
-          <vaadin-radio-button label="Button 1" value="1"></vaadin-radio-button>
-          <vaadin-radio-button label="Button 2" value="2"></vaadin-radio-button>
-        </vaadin-radio-group>
+        <vaadin24-radio-group>
+          <vaadin24-radio-button label="Button 1" value="1"></vaadin24-radio-button>
+          <vaadin24-radio-button label="Button 2" value="2"></vaadin24-radio-button>
+        </vaadin24-radio-group>
       `);
       await nextFrame();
       validateSpy = sinon.spy(group, 'validate');
@@ -61,7 +61,7 @@
     });
 
     it('should validate on radio button click', () => {
-      group.querySelector('vaadin-radio-button').click();
+      group.querySelector('vaadin24-radio-button').click();
       expect(validateSpy.calledOnce).to.be.true;
     });
 
@@ -107,10 +107,10 @@
   describe('required', () => {
     beforeEach(async () => {
       group = fixtureSync(`
-        <vaadin-radio-group required>
-          <vaadin-radio-button label="Button 1" value="1"></vaadin-radio-button>
-          <vaadin-radio-button label="Button 2" value="2"></vaadin-radio-button>
-        </vaadin-radio-group>
+        <vaadin24-radio-group required>
+          <vaadin24-radio-button label="Button 1" value="1"></vaadin24-radio-button>
+          <vaadin24-radio-button label="Button 2" value="2"></vaadin24-radio-button>
+        </vaadin24-radio-group>
       `);
       await nextFrame();
     });
@@ -129,7 +129,7 @@
     });
 
     it('should be valid after selecting a radio button', () => {
-      group.querySelector('vaadin-radio-button').click();
+      group.querySelector('vaadin24-radio-button').click();
       expect(group.invalid).to.be.false;
     });
   });
