Index: test/css-injection-mixin-lit.test.js
===================================================================
--- test/css-injection-mixin-lit.test.js
+++ test/css-injection-mixin-lit.test.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextRender, oneEvent } from '@vaadin/testing-helpers';
 import { css, html, LitElement } from 'lit';
@@ -27,7 +28,7 @@
   }
 }
 
-customElements.define(TestFoo.is, TestFoo);
+internalCustomElements.define(TestFoo.is, TestFoo);
 
 class TestBar extends CSSInjectionMixin(LitElement) {
   static get is() {
@@ -39,7 +40,7 @@
   }
 }
 
-customElements.define(TestBar.is, TestBar);
+internalCustomElements.define(TestBar.is, TestBar);
 
 class TestBaz extends TestFoo {
   static get is() {
@@ -51,7 +52,7 @@
   }
 }
 
-customElements.define(TestBaz.is, TestBaz);
+internalCustomElements.define(TestBaz.is, TestBaz);
 
 const TEST_FOO_STYLES = `
   html, :host {
