Index: test/post-finalize-styles-lit.test.ts
===================================================================
--- test/post-finalize-styles-lit.test.ts
+++ test/post-finalize-styles-lit.test.ts
@@ -1,7 +1,8 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import './post-finalize-styles.common.ts';
 import { css, LitElement } from 'lit';
 
-customElements.define(
+internalCustomElements.define(
   'test-element',
   class extends LitElement {
     static get styles() {
