Index: test/themable-mixin.test.js
===================================================================
--- test/themable-mixin.test.js
+++ test/themable-mixin.test.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import { PolymerElement } from '@polymer/polymer/polymer-element.js';
@@ -12,7 +13,7 @@
 const defineCustomElement =
   window.defineCustomElementFunction ||
   ((name, parentName, content, styles, noIs) => {
-    const parentElement = parentName ? customElements.get(parentName) : PolymerElement;
+    const parentElement = parentName ? internalCustomElements.get(parentName) : PolymerElement;
     class CustomElement extends ThemableMixin(parentElement) {}
 
     if (content) {
@@ -38,7 +39,7 @@
       });
     }
 
-    customElements.define(name, CustomElement);
+    internalCustomElements.define(name, CustomElement);
   });
 
 createStyles(
