Index: test/accessibility.test.js
===================================================================
--- test/accessibility.test.js
+++ test/accessibility.test.js
@@ -9,7 +9,7 @@
     let sideNav;
 
     beforeEach(async () => {
-      sideNav = fixtureSync('<vaadin-side-nav></vaadin-side-nav>');
+      sideNav = fixtureSync('<vaadin24-side-nav></vaadin24-side-nav>');
       await nextRender(sideNav);
     });
 
@@ -18,7 +18,7 @@
     });
 
     it('should have custom role effective', async () => {
-      const sideNavWithCustomRole = fixtureSync('<vaadin-side-nav role="custom role"></vaadin-side-nav>');
+      const sideNavWithCustomRole = fixtureSync('<vaadin24-side-nav role="custom role"></vaadin24-side-nav>');
       await nextRender(sideNavWithCustomRole);
       expect(sideNavWithCustomRole.getAttribute('role')).to.equal('custom role');
     });
@@ -28,7 +28,7 @@
     let sideNav;
 
     beforeEach(async () => {
-      sideNav = fixtureSync('<vaadin-side-nav></vaadin-side-nav>');
+      sideNav = fixtureSync('<vaadin24-side-nav></vaadin24-side-nav>');
       await nextRender();
     });
 
@@ -91,10 +91,10 @@
       wrapper = fixtureSync(`
         <div>
           <input />
-          <vaadin-side-nav collapsible>
-            <vaadin-side-nav-item path="/foo">Foo</vaadin-side-nav-item>
-            <vaadin-side-nav-item path="/bar">Bar</vaadin-side-nav-item>
-          </vaadin-side-nav>
+          <vaadin24-side-nav collapsible>
+            <vaadin24-side-nav-item path="/foo">Foo</vaadin24-side-nav-item>
+            <vaadin24-side-nav-item path="/bar">Bar</vaadin24-side-nav-item>
+          </vaadin24-side-nav>
         </div>
       `);
       await nextRender();
@@ -130,17 +130,17 @@
 
     beforeEach(async () => {
       sideNav = fixtureSync(`
-        <vaadin-side-nav>
-          <vaadin-side-nav-item path="/foo">Foo</vaadin-side-nav-item>
-          <vaadin-side-nav-item path="/bar">
+        <vaadin24-side-nav>
+          <vaadin24-side-nav-item path="/foo">Foo</vaadin24-side-nav-item>
+          <vaadin24-side-nav-item path="/bar">
             Bar
-            <vaadin-side-nav-item path="/bar/baz" slot="children">Baz</vaadin-side-nav-item>
-            <vaadin-side-nav-item path="/bar/qux" slot="children">Qux</vaadin-side-nav-item>
-          </vaadin-side-nav-item>
-        </vaadin-side-nav>
+            <vaadin24-side-nav-item path="/bar/baz" slot="children">Baz</vaadin24-side-nav-item>
+            <vaadin24-side-nav-item path="/bar/qux" slot="children">Qux</vaadin24-side-nav-item>
+          </vaadin24-side-nav-item>
+        </vaadin24-side-nav>
       `);
       await nextRender();
-      items = [...sideNav.querySelectorAll('vaadin-side-nav-item')];
+      items = [...sideNav.querySelectorAll('vaadin24-side-nav-item')];
     });
 
     it('should set i18n on child items of the side-nav item', async () => {
@@ -166,17 +166,17 @@
 
     beforeEach(async () => {
       sideNav = fixtureSync(`
-        <vaadin-side-nav>
-          <vaadin-side-nav-item path="/foo">Foo</vaadin-side-nav-item>
-          <vaadin-side-nav-item path="/bar">
+        <vaadin24-side-nav>
+          <vaadin24-side-nav-item path="/foo">Foo</vaadin24-side-nav-item>
+          <vaadin24-side-nav-item path="/bar">
             Bar
-            <vaadin-side-nav-item path="/bar/baz" slot="children">Baz</vaadin-side-nav-item>
-            <vaadin-side-nav-item path="/bar/qux" slot="children">Qux</vaadin-side-nav-item>
-          </vaadin-side-nav-item>
-        </vaadin-side-nav>
+            <vaadin24-side-nav-item path="/bar/baz" slot="children">Baz</vaadin24-side-nav-item>
+            <vaadin24-side-nav-item path="/bar/qux" slot="children">Qux</vaadin24-side-nav-item>
+          </vaadin24-side-nav-item>
+        </vaadin24-side-nav>
       `);
       await nextRender();
-      items = [...sideNav.querySelectorAll('vaadin-side-nav-item')];
+      items = [...sideNav.querySelectorAll('vaadin24-side-nav-item')];
     });
 
     it('should propagate disabled to child items of the side-nav item', async () => {
