Index: test/navigation.test.js
===================================================================
--- test/navigation.test.js
+++ test/navigation.test.js
@@ -18,16 +18,16 @@
 
   beforeEach(async () => {
     sideNav = fixtureSync(`
-      <vaadin-side-nav>
-        <vaadin-side-nav-item path="/1">1</vaadin-side-nav-item>
-        <vaadin-side-nav-item path="/2">
+      <vaadin24-side-nav>
+        <vaadin24-side-nav-item path="/1">1</vaadin24-side-nav-item>
+        <vaadin24-side-nav-item path="/2">
           2
-          <vaadin-side-nav-item slot="children" path="/21">21</vaadin-side-nav-item>
-          <vaadin-side-nav-item slot="children" path="/22">22</vaadin-side-nav-item>
-        </vaadin-side-nav-item>
-      </vaadin-side-nav>
+          <vaadin24-side-nav-item slot="children" path="/21">21</vaadin24-side-nav-item>
+          <vaadin24-side-nav-item slot="children" path="/22">22</vaadin24-side-nav-item>
+        </vaadin24-side-nav-item>
+      </vaadin24-side-nav>
     `);
-    items = sideNav.querySelectorAll('vaadin-side-nav-item');
+    items = sideNav.querySelectorAll('vaadin24-side-nav-item');
     await nextRender();
   });
 
