Index: test/navigation-callback.test.js
===================================================================
--- test/navigation-callback.test.js
+++ test/navigation-callback.test.js
@@ -16,14 +16,14 @@
 
   beforeEach(async () => {
     sideNav = fixtureSync(`
-      <vaadin-side-nav>
+      <vaadin24-side-nav>
         <a slot="label" href="/home">Home</a>
 
-        <vaadin-side-nav-item path="/foo">
+        <vaadin24-side-nav-item path="/foo">
           foo
-          <vaadin-side-nav-item slot="children" path="/bar">bar</vaadin-side-nav-item>
-        </vaadin-side-nav-item>
-      </vaadin-side-nav>
+          <vaadin24-side-nav-item slot="children" path="/bar">bar</vaadin24-side-nav-item>
+        </vaadin24-side-nav-item>
+      </vaadin24-side-nav>
     `);
 
     sideNav.addEventListener('click', (e) => {
@@ -37,7 +37,7 @@
     sideNav.onNavigate = sinon.spy();
 
     await nextRender();
-    sideNavItem = sideNav.querySelector('vaadin-side-nav-item');
+    sideNavItem = sideNav.querySelector('vaadin24-side-nav-item');
   });
 
   it('should cancel the click event', () => {
