Index: test/confirm-dialog.test.js
===================================================================
--- test/confirm-dialog.test.js
+++ test/confirm-dialog.test.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import {
   aTimeout,
@@ -12,23 +13,23 @@
 import sinon from 'sinon';
 import './not-animated-styles.js';
 import '../src/vaadin-confirm-dialog.js';
-import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
+import { getDeepActiveElement } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 
-describe('vaadin-confirm-dialog', () => {
+describe('vaadin24-confirm-dialog', () => {
   describe('custom element definition', () => {
     let confirm, tagName;
 
     beforeEach(() => {
-      confirm = fixtureSync('<vaadin-confirm-dialog></vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog></vaadin24-confirm-dialog>');
       tagName = confirm.tagName.toLowerCase();
     });
 
     it('should be defined in custom element registry', () => {
-      expect(customElements.get(tagName)).to.be.ok;
+      expect(internalCustomElements.get(tagName)).to.be.ok;
     });
 
     it('should have a valid static "is" getter', () => {
-      expect(customElements.get(tagName).is).to.equal(tagName);
+      expect(internalCustomElements.get(tagName).is).to.equal(tagName);
     });
   });
 
@@ -36,7 +37,7 @@
     let confirm;
 
     beforeEach(() => {
-      confirm = fixtureSync('<vaadin-confirm-dialog></vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog></vaadin24-confirm-dialog>');
     });
 
     it('should enforce display: none to hide the host element', () => {
@@ -49,7 +50,7 @@
     let confirm, dialog, overlay;
 
     beforeEach(async () => {
-      confirm = fixtureSync('<vaadin-confirm-dialog>Confirmation message</vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog>Confirmation message</vaadin24-confirm-dialog>');
       await nextRender();
       dialog = confirm.$.dialog;
       overlay = dialog.$.overlay;
@@ -116,7 +117,7 @@
 
     describe('default', () => {
       beforeEach(async () => {
-        confirm = fixtureSync('<vaadin-confirm-dialog></vaadin-confirm-dialog>');
+        confirm = fixtureSync('<vaadin24-confirm-dialog></vaadin24-confirm-dialog>');
         await nextRender();
       });
 
@@ -133,7 +134,7 @@
 
     describe('property', () => {
       beforeEach(async () => {
-        confirm = fixtureSync('<vaadin-confirm-dialog opened header="Property header"></vaadin-confirm-dialog>');
+        confirm = fixtureSync('<vaadin24-confirm-dialog opened header="Property header"></vaadin24-confirm-dialog>');
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
         headerSlot = overlay.shadowRoot.querySelector('slot[name="header"]');
@@ -155,9 +156,9 @@
     describe('slot', () => {
       beforeEach(async () => {
         confirm = fixtureSync(`
-          <vaadin-confirm-dialog opened header="Property header">
+          <vaadin24-confirm-dialog opened header="Property header">
             <h3 slot="header">Slotted header</h3>
-          </vaadin-confirm-dialog>
+          </vaadin24-confirm-dialog>
         `);
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
@@ -188,7 +189,7 @@
 
     describe('property', () => {
       beforeEach(async () => {
-        confirm = fixtureSync('<vaadin-confirm-dialog opened message="Confirmation message"></vaadin-confirm-dialog>');
+        confirm = fixtureSync('<vaadin24-confirm-dialog opened message="Confirmation message"></vaadin24-confirm-dialog>');
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
         messageSlot = overlay.shadowRoot.querySelector('[part="message"] > slot');
@@ -219,10 +220,10 @@
     describe('slot', () => {
       beforeEach(async () => {
         confirm = fixtureSync(`
-          <vaadin-confirm-dialog opened>
+          <vaadin24-confirm-dialog opened>
             Confirmation message
             <div>Additional content</dib>
-          </vaadin-confirm-dialog>
+          </vaadin24-confirm-dialog>
         `);
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
@@ -249,10 +250,10 @@
 
       beforeEach(async () => {
         confirm = fixtureSync(`
-          <vaadin-confirm-dialog opened>
+          <vaadin24-confirm-dialog opened>
             ${firstChild}
             ${secondChild}
-          </vaadin-confirm-dialog>
+          </vaadin24-confirm-dialog>
         `);
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
@@ -292,7 +293,7 @@
     let confirm, overlay;
 
     beforeEach(async () => {
-      confirm = fixtureSync('<vaadin-confirm-dialog opened>Confirmation message</vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog opened>Confirmation message</vaadin24-confirm-dialog>');
       await nextRender();
       overlay = confirm.$.dialog.$.overlay;
     });
@@ -442,11 +443,11 @@
 
     beforeEach(async () => {
       confirm = fixtureSync(`
-        <vaadin-confirm-dialog>
+        <vaadin24-confirm-dialog>
           <button slot="confirm-button" theme="custom-confirm-theme">Custom Confirm</button>
           <button slot="cancel-button" theme="custom-cancel-theme">Custom Cancel</button>
           <button slot="reject-button" theme="custom-reject-theme">Custom Reject</button>
-        </vaadin-confirm-dialog>
+        </vaadin24-confirm-dialog>
       `);
       await nextRender();
       overlay = confirm.$.dialog.$.overlay;
@@ -502,7 +503,7 @@
     let confirm, overlay;
 
     beforeEach(async () => {
-      confirm = fixtureSync('<vaadin-confirm-dialog opened>Confirmation message</vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog opened>Confirmation message</vaadin24-confirm-dialog>');
       await nextRender();
       overlay = confirm.$.dialog.$.overlay;
     });
@@ -558,7 +559,7 @@
     let confirm, spy;
 
     beforeEach(async () => {
-      confirm = fixtureSync('<vaadin-confirm-dialog opened>Confirmation message</vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog opened>Confirmation message</vaadin24-confirm-dialog>');
       await nextRender();
       spy = sinon.spy();
       confirm.addEventListener('cancel', spy);
@@ -583,7 +584,7 @@
     let confirm;
 
     beforeEach(async () => {
-      confirm = fixtureSync('<vaadin-confirm-dialog theme="foo">Confirmation message</vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog theme="foo">Confirmation message</vaadin24-confirm-dialog>');
       await nextRender();
     });
 
@@ -610,7 +611,7 @@
 
     describe('default', () => {
       beforeEach(async () => {
-        confirm = fixtureSync('<vaadin-confirm-dialog opened>Confirmation message</vaadin-confirm-dialog>');
+        confirm = fixtureSync('<vaadin24-confirm-dialog opened>Confirmation message</vaadin24-confirm-dialog>');
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
       });
@@ -666,7 +667,7 @@
 
     describe('before attach', () => {
       beforeEach(() => {
-        confirm = document.createElement('vaadin-confirm-dialog');
+        confirm = document.createElement('vaadin24-confirm-dialog');
         confirm.message = 'Message';
       });
 
@@ -712,7 +713,7 @@
     beforeEach(async () => {
       const wrapper = fixtureSync(`
         <div>
-          <vaadin-confirm-dialog></vaadin-confirm-dialog>
+          <vaadin24-confirm-dialog></vaadin24-confirm-dialog>
           <button></button>
         </div>
       `);
