Index: test/confirm-dialog.common.js
===================================================================
--- test/confirm-dialog.common.js
+++ test/confirm-dialog.common.js
@@ -1,23 +1,24 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { aTimeout, esc, fixtureSync, nextFrame, nextRender, nextUpdate, oneEvent } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
+import { getDeepActiveElement } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 
-describe('vaadin-confirm-dialog', () => {
+describe('vaadin24-confirm-dialog', () => {
   describe('custom element definition', () => {
     let confirm, tagName;
 
     beforeEach(() => {
-      confirm = fixtureSync('<vaadin-confirm-dialog></vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog></vaadin24-confirm-dialog>');
       tagName = confirm.tagName.toLowerCase();
     });
 
     it('should be defined in custom element registry', () => {
-      expect(customElements.get(tagName)).to.be.ok;
+      expect(internalCustomElements.get(tagName)).to.be.ok;
     });
 
     it('should have a valid static "is" getter', () => {
-      expect(customElements.get(tagName).is).to.equal(tagName);
+      expect(internalCustomElements.get(tagName).is).to.equal(tagName);
     });
   });
 
@@ -25,7 +26,7 @@
     let confirm;
 
     beforeEach(() => {
-      confirm = fixtureSync('<vaadin-confirm-dialog></vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog></vaadin24-confirm-dialog>');
     });
 
     it('should enforce display: none to hide the host element', () => {
@@ -38,7 +39,7 @@
     let confirm, dialog, overlay;
 
     beforeEach(async () => {
-      confirm = fixtureSync('<vaadin-confirm-dialog>Confirmation message</vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog>Confirmation message</vaadin24-confirm-dialog>');
       await nextRender();
       dialog = confirm.$.dialog;
       overlay = dialog.$.overlay;
@@ -105,7 +106,7 @@
 
     describe('default', () => {
       beforeEach(async () => {
-        confirm = fixtureSync('<vaadin-confirm-dialog></vaadin-confirm-dialog>');
+        confirm = fixtureSync('<vaadin24-confirm-dialog></vaadin24-confirm-dialog>');
         await nextRender();
       });
 
@@ -122,7 +123,7 @@
 
     describe('property', () => {
       beforeEach(async () => {
-        confirm = fixtureSync('<vaadin-confirm-dialog opened header="Property header"></vaadin-confirm-dialog>');
+        confirm = fixtureSync('<vaadin24-confirm-dialog opened header="Property header"></vaadin24-confirm-dialog>');
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
         headerSlot = overlay.shadowRoot.querySelector('slot[name="header"]');
@@ -144,9 +145,9 @@
     describe('slot', () => {
       beforeEach(async () => {
         confirm = fixtureSync(`
-          <vaadin-confirm-dialog opened header="Property header">
+          <vaadin24-confirm-dialog opened header="Property header">
             <h3 slot="header">Slotted header</h3>
-          </vaadin-confirm-dialog>
+          </vaadin24-confirm-dialog>
         `);
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
@@ -177,7 +178,7 @@
 
     describe('property', () => {
       beforeEach(async () => {
-        confirm = fixtureSync('<vaadin-confirm-dialog opened message="Confirmation message"></vaadin-confirm-dialog>');
+        confirm = fixtureSync('<vaadin24-confirm-dialog opened message="Confirmation message"></vaadin24-confirm-dialog>');
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
         messageSlot = overlay.shadowRoot.querySelector('[part="message"] > slot');
@@ -208,10 +209,10 @@
     describe('slot', () => {
       beforeEach(async () => {
         confirm = fixtureSync(`
-          <vaadin-confirm-dialog opened>
+          <vaadin24-confirm-dialog opened>
             Confirmation message
             <div>Additional content</dib>
-          </vaadin-confirm-dialog>
+          </vaadin24-confirm-dialog>
         `);
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
@@ -238,10 +239,10 @@
 
       beforeEach(async () => {
         confirm = fixtureSync(`
-          <vaadin-confirm-dialog opened>
+          <vaadin24-confirm-dialog opened>
             ${firstChild}
             ${secondChild}
-          </vaadin-confirm-dialog>
+          </vaadin24-confirm-dialog>
         `);
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
@@ -281,7 +282,7 @@
     let confirm, overlay;
 
     beforeEach(async () => {
-      confirm = fixtureSync('<vaadin-confirm-dialog opened>Confirmation message</vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog opened>Confirmation message</vaadin24-confirm-dialog>');
       await nextRender();
       overlay = confirm.$.dialog.$.overlay;
     });
@@ -431,11 +432,11 @@
 
     beforeEach(async () => {
       confirm = fixtureSync(`
-        <vaadin-confirm-dialog>
+        <vaadin24-confirm-dialog>
           <button slot="confirm-button" theme="custom-confirm-theme">Custom Confirm</button>
           <button slot="cancel-button" theme="custom-cancel-theme">Custom Cancel</button>
           <button slot="reject-button" theme="custom-reject-theme">Custom Reject</button>
-        </vaadin-confirm-dialog>
+        </vaadin24-confirm-dialog>
       `);
       await nextRender();
       overlay = confirm.$.dialog.$.overlay;
@@ -491,7 +492,7 @@
     let confirm, overlay;
 
     beforeEach(async () => {
-      confirm = fixtureSync('<vaadin-confirm-dialog opened>Confirmation message</vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog opened>Confirmation message</vaadin24-confirm-dialog>');
       await nextRender();
       overlay = confirm.$.dialog.$.overlay;
     });
@@ -524,7 +525,7 @@
     let confirm, spy;
 
     beforeEach(async () => {
-      confirm = fixtureSync('<vaadin-confirm-dialog opened>Confirmation message</vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog opened>Confirmation message</vaadin24-confirm-dialog>');
       await nextRender();
       spy = sinon.spy();
       confirm.addEventListener('cancel', spy);
@@ -549,7 +550,7 @@
     let confirm;
 
     beforeEach(async () => {
-      confirm = fixtureSync('<vaadin-confirm-dialog theme="foo">Confirmation message</vaadin-confirm-dialog>');
+      confirm = fixtureSync('<vaadin24-confirm-dialog theme="foo">Confirmation message</vaadin24-confirm-dialog>');
       await nextRender();
     });
 
@@ -576,7 +577,7 @@
 
     describe('default', () => {
       beforeEach(async () => {
-        confirm = fixtureSync('<vaadin-confirm-dialog opened>Confirmation message</vaadin-confirm-dialog>');
+        confirm = fixtureSync('<vaadin24-confirm-dialog opened>Confirmation message</vaadin24-confirm-dialog>');
         await nextRender();
         overlay = confirm.$.dialog.$.overlay;
       });
@@ -624,7 +625,7 @@
 
     describe('before attach', () => {
       beforeEach(() => {
-        confirm = document.createElement('vaadin-confirm-dialog');
+        confirm = document.createElement('vaadin24-confirm-dialog');
         confirm.message = 'Message';
       });
 
@@ -670,7 +671,7 @@
     beforeEach(async () => {
       const wrapper = fixtureSync(`
         <div>
-          <vaadin-confirm-dialog></vaadin-confirm-dialog>
+          <vaadin24-confirm-dialog></vaadin24-confirm-dialog>
           <button></button>
         </div>
       `);
