Index: test/visual/material/checkbox-group.test.js
===================================================================
--- test/visual/material/checkbox-group.test.js
+++ test/visual/material/checkbox-group.test.js
@@ -13,11 +13,11 @@
 
     element = fixtureSync(
       `
-        <vaadin-checkbox-group>
-          <vaadin-checkbox value="a" label="A"></vaadin-checkbox>
-          <vaadin-checkbox value="b" label="B"></vaadin-checkbox>
-          <vaadin-checkbox value="c" label="C"></vaadin-checkbox>
-        </vaadin-checkbox-group>
+        <vaadin23-checkbox-group>
+          <vaadin23-checkbox value="a" label="A"></vaadin23-checkbox>
+          <vaadin23-checkbox value="b" label="B"></vaadin23-checkbox>
+          <vaadin23-checkbox value="c" label="C"></vaadin23-checkbox>
+        </vaadin23-checkbox-group>
       `,
       div,
     );
@@ -88,7 +88,7 @@
       div.style.width = '300px';
       div.style.padding = '0';
       element.style.width = '100%';
-      element.querySelectorAll('vaadin-checkbox').forEach((checkbox) => {
+      element.querySelectorAll('vaadin23-checkbox').forEach((checkbox) => {
         checkbox.style.width = '33%';
         // Compensate inline-block whitespace
         checkbox.style.marginRight = '-3px';
