Index: test/visual/lumo/radio-button.test.js
===================================================================
--- test/visual/lumo/radio-button.test.js
+++ test/visual/lumo/radio-button.test.js
@@ -10,7 +10,7 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-radio-button label="Radio button"></vaadin-radio-button>', div);
+    element = fixtureSync('<vaadin24-radio-button label="Radio button"></vaadin24-radio-button>', div);
   });
 
   it('basic', async () => {
