Index: src/vaadin-lit-login-form-wrapper.js
===================================================================
--- src/vaadin-lit-login-form-wrapper.js
+++ src/vaadin-lit-login-form-wrapper.js
@@ -4,13 +4,13 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { loginFormWrapperStyles } from './vaadin-login-form-wrapper-styles.js';
 
 /**
- * An element used internally by `<vaadin-login-form>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-login-form>`. Not intended to be used separately.
  *
  * @extends HTMLElement
  * @mixes ThemableMixin
@@ -18,7 +18,7 @@
  */
 class LoginFormWrapper extends ThemableMixin(PolylitMixin(LitElement)) {
   static get is() {
-    return 'vaadin-login-form-wrapper';
+    return 'vaadin24-login-form-wrapper';
   }
 
   static get styles() {
