Index: src/vaadin-lit-login-overlay.js
===================================================================
--- src/vaadin-lit-login-overlay.js
+++ src/vaadin-lit-login-overlay.js
@@ -7,14 +7,14 @@
 import './vaadin-lit-login-overlay-wrapper.js';
 import { html, LitElement } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { LoginOverlayMixin } from './vaadin-login-overlay-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-login-overlay>` web component.
+ * LitElement based version of `<vaadin24-login-overlay>` web component.
  *
  * ## Disclaimer
  *
@@ -24,13 +24,13 @@
  */
 class LoginOverlay extends LoginOverlayMixin(ElementMixin(ThemableMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-login-overlay';
+    return 'vaadin24-login-overlay';
   }
 
   /** @protected */
   render() {
     return html`
-      <vaadin-login-overlay-wrapper
+      <vaadin24-login-overlay-wrapper
         id="vaadinLoginOverlayWrapper"
         .opened="${this.opened}"
         .title="${this.title}"
@@ -42,7 +42,7 @@
         @vaadin-overlay-outside-click="${this._preventClosingLogin}"
         @opened-changed="${this._onOpenedChanged}"
       >
-        <vaadin-login-form
+        <vaadin24-login-form
           theme="with-overlay"
           id="vaadinLoginForm"
           .action="${this.action}"
@@ -54,8 +54,8 @@
           @login="${this._retargetEvent}"
           @forgot-password="${this._retargetEvent}"
           @disabled-changed="${this._onDisabledChanged}"
-        ></vaadin-login-form>
-      </vaadin-login-overlay-wrapper>
+        ></vaadin24-login-form>
+      </vaadin24-login-overlay-wrapper>
 
       <div hidden>
         <slot name="custom-form-area"></slot>
