Index: login/src/vaadin-login-overlay-mixin.d.ts
===================================================================
--- login/src/vaadin-login-overlay-mixin.d.ts	null
+++ login/src/vaadin-login-overlay-mixin.d.ts	null
@@ -4,7 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { OverlayClassMixinClass } from '@vaadin/component-base/src/overlay-class-mixin.js';
+import type { OverlayClassMixinClass } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
 import type { LoginMixinClass } from './vaadin-login-mixin.js';
 
 export declare function LoginOverlayMixin<T extends Constructor<HTMLElement>>(
