Index: src/vaadin-login-overlay-wrapper.js
===================================================================
--- src/vaadin-login-overlay-wrapper.js
+++ src/vaadin-login-overlay-wrapper.js
@@ -1,10 +1,11 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2018 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { DomModule } from '@polymer/polymer/lib/elements/dom-module.js';
-import { Overlay } from '@vaadin/overlay/src/vaadin-overlay.js';
+import { Overlay } from '@scoped-vaadin/overlay/src/vaadin-overlay.js';
 
 const template = document.createElement('template');
 
@@ -57,14 +58,14 @@
 let memoizedTemplate;
 
 /**
- * An element used internally by `<vaadin-login-overlay>`. Not intended to be used separately.
+ * An element used internally by `<vaadin23-login-overlay>`. Not intended to be used separately.
  *
  * @extends Overlay
  * @private
  */
 class LoginOverlayWrapper extends Overlay {
   static get is() {
-    return 'vaadin-login-overlay-wrapper';
+    return 'vaadin23-login-overlay-wrapper';
   }
 
   static get properties() {
@@ -105,4 +106,4 @@
   }
 }
 
-customElements.define(LoginOverlayWrapper.is, LoginOverlayWrapper);
+internalCustomElements.define(LoginOverlayWrapper.is, LoginOverlayWrapper);
