Index: src/vaadin-login-overlay-wrapper.js
===================================================================
--- src/vaadin-login-overlay-wrapper.js
+++ src/vaadin-login-overlay-wrapper.js
@@ -4,18 +4,18 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { overlayStyles } from '@vaadin/overlay/src/vaadin-overlay-styles.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { overlayStyles } from '@scoped-vaadin/overlay/src/vaadin-overlay-styles.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { LoginOverlayWrapperMixin } from './vaadin-login-overlay-wrapper-mixin.js';
 import { loginOverlayWrapperStyles } from './vaadin-login-overlay-wrapper-styles.js';
 
-registerStyles('vaadin-login-overlay-wrapper', [overlayStyles, loginOverlayWrapperStyles], {
+registerStyles('vaadin24-login-overlay-wrapper', [overlayStyles, loginOverlayWrapperStyles], {
   moduleId: 'vaadin-login-overlay-wrapper-styles',
 });
 
 /**
- * An element used internally by `<vaadin-login-overlay>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-login-overlay>`. Not intended to be used separately.
  *
  * @extends HTMLElement
  * @mixes LoginOverlayWrapperMixin
@@ -24,7 +24,7 @@
  */
 class LoginOverlayWrapper extends LoginOverlayWrapperMixin(ThemableMixin(PolymerElement)) {
   static get is() {
-    return 'vaadin-login-overlay-wrapper';
+    return 'vaadin24-login-overlay-wrapper';
   }
 
   static get template() {
