Index: src/vaadin-login-overlay.js
===================================================================
--- src/vaadin-login-overlay.js
+++ src/vaadin-login-overlay.js
@@ -6,37 +6,37 @@
 import './vaadin-login-form.js';
 import './vaadin-login-overlay-wrapper.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { LoginOverlayMixin } from './vaadin-login-overlay-mixin.js';
 
 /**
- * `<vaadin-login-overlay>` is a wrapper of the `<vaadin-login-form>` which opens a login form in an overlay and
- * having an additional `brand` part for application title and description. Using `<vaadin-login-overlay>` allows
+ * `<vaadin24-login-overlay>` is a wrapper of the `<vaadin24-login-form>` which opens a login form in an overlay and
+ * having an additional `brand` part for application title and description. Using `<vaadin24-login-overlay>` allows
  * password managers to work with login form.
  *
  * ```
- * <vaadin-login-overlay opened></vaadin-login-overlay>
+ * <vaadin24-login-overlay opened></vaadin24-login-overlay>
  * ```
  *
  * ### Styling
  *
  * The component doesn't have a shadowRoot, so the `<form>` and input fields can be styled from a global scope.
- * Use `<vaadin-login-overlay-wrapper>` and `<vaadin-login-form-wrapper>` to apply styles.
+ * Use `<vaadin24-login-overlay-wrapper>` and `<vaadin24-login-form-wrapper>` to apply styles.
  *
- * The following shadow DOM parts of the `<vaadin-login-overlay-wrapper>` are available for styling:
+ * The following shadow DOM parts of the `<vaadin24-login-overlay-wrapper>` are available for styling:
  *
  * Part name       | Description
  * ----------------|---------------------------------------------------------|
  * `card`          | Container for the entire component's content
  * `brand`         | Container for application title and description
- * `form`          | Container for the `<vaadin-login-form>` component
+ * `form`          | Container for the `<vaadin24-login-form>` component
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
- * See [`<vaadin-login-form>`](#/elements/vaadin-login-form)
- * documentation for  `<vaadin-login-form-wrapper>` stylable parts.
+ * See [`<vaadin24-login-form>`](#/elements/vaadin-login-form)
+ * documentation for  `<vaadin24-login-form-wrapper>` stylable parts.
  *
  * @fires {CustomEvent} description-changed - Fired when the `description` property changes.
  * @fires {CustomEvent} disabled-changed - Fired when the `disabled` property changes.
@@ -53,7 +53,7 @@
 class LoginOverlay extends LoginOverlayMixin(ElementMixin(ThemableMixin(PolymerElement))) {
   static get template() {
     return html`
-      <vaadin-login-overlay-wrapper
+      <vaadin24-login-overlay-wrapper
         id="vaadinLoginOverlayWrapper"
         opened="{{opened}}"
         role="dialog"
@@ -66,7 +66,7 @@
         on-vaadin-overlay-escape-press="_preventClosingLogin"
         on-vaadin-overlay-outside-click="_preventClosingLogin"
       >
-        <vaadin-login-form
+        <vaadin24-login-form
           theme="with-overlay"
           id="vaadinLoginForm"
           action="[[action]]"
@@ -78,8 +78,8 @@
           i18n="{{__effectiveI18n}}"
           on-login="_retargetEvent"
           on-forgot-password="_retargetEvent"
-        ></vaadin-login-form>
-      </vaadin-login-overlay-wrapper>
+        ></vaadin24-login-form>
+      </vaadin24-login-overlay-wrapper>
 
       <div hidden>
         <slot name="custom-form-area"></slot>
@@ -89,7 +89,7 @@
   }
 
   static get is() {
-    return 'vaadin-login-overlay';
+    return 'vaadin24-login-overlay';
   }
 }
 
