Index: login/src/vaadin-login-mixin.js
===================================================================
--- login/src/vaadin-login-mixin.js
+++ login/src/vaadin-login-mixin.js
@@ -85,7 +85,7 @@
          *
          * The object has the following JSON structure (by default it doesn't include `additionalInformation`
          * and `header` sections, `header` can be added to override `title` and `description` properties
-         * in `vaadin-login-overlay`):
+         * in `vaadin24-login-overlay`):
          *
          * ```
          * {
