Index: theme/material/vaadin-dialog-styles.d.ts
===================================================================
--- theme/material/vaadin-dialog-styles.d.ts
+++ theme/material/vaadin-dialog-styles.d.ts
@@ -1,4 +1,4 @@
-import '@vaadin/vaadin-material-styles/shadow.js';
-import '@vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/shadow.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
 declare const dialogOverlay: import("lit").CSSResult;
 export { dialogOverlay };
