Index: menu-bar/theme/material/vaadin-menu-bar-button-styles.js
===================================================================
--- menu-bar/theme/material/vaadin-menu-bar-button-styles.js
+++ menu-bar/theme/material/vaadin-menu-bar-button-styles.js
@@ -1,5 +1,5 @@
-import { button } from '@vaadin/button/theme/material/vaadin-button-styles';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { button } from '@scoped-vaadin/button/theme/material/vaadin-button-styles';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const menuBarButton = css`
   [part='label'] {
@@ -106,6 +106,6 @@
   }
 `;
 
-registerStyles('vaadin-menu-bar-button', [button, menuBarButton], {
+registerStyles('vaadin24-menu-bar-button', [button, menuBarButton], {
   moduleId: 'material-menu-bar-button',
 });
