Index: theme/material/vaadin-menu-bar-button-styles.js
===================================================================
--- theme/material/vaadin-menu-bar-button-styles.js
+++ theme/material/vaadin-menu-bar-button-styles.js
@@ -1,12 +1,12 @@
-import { button } from '@vaadin/button/theme/material/vaadin-button-styles';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { button } from '@scoped-vaadin/button/theme/material/vaadin-button-styles';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const menuBarButton = css`
   [part='label'] {
     width: 100%;
   }
 
-  [part='label'] ::slotted(vaadin-menu-bar-item) {
+  [part='label'] ::slotted(vaadin24-menu-bar-item) {
     line-height: 20px;
     background-color: transparent;
     margin: -8px;
@@ -19,8 +19,8 @@
     border-radius: 0;
   }
 
-  :host([theme~='contained']) ::slotted(vaadin-menu-bar-item),
-  :host([theme~='outlined']) ::slotted(vaadin-menu-bar-item) {
+  :host([theme~='contained']) ::slotted(vaadin24-menu-bar-item),
+  :host([theme~='outlined']) ::slotted(vaadin24-menu-bar-item) {
     margin: -8px -16px;
     padding: 8px 16px;
   }
@@ -106,6 +106,6 @@
   }
 `;
 
-registerStyles('vaadin-menu-bar-button', [button, menuBarButton], {
+registerStyles('vaadin24-menu-bar-button', [button, menuBarButton], {
   moduleId: 'material-menu-bar-button',
 });
