Index: test/dom/__snapshots__/checkbox.test.snap.js
===================================================================
--- test/dom/__snapshots__/checkbox.test.snap.js
+++ test/dom/__snapshots__/checkbox.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-checkbox host default"] = 
-`<vaadin-checkbox has-value="">
+`<vaadin24-checkbox has-value="">
   <label
     for="input-vaadin-checkbox-1"
     id="label-vaadin-checkbox-0"
@@ -16,12 +16,12 @@
     type="checkbox"
     value="on"
   >
-</vaadin-checkbox>
+</vaadin24-checkbox>
 `;
 /* end snapshot vaadin-checkbox host default */
 
 snapshots["vaadin-checkbox host disabled"] = 
-`<vaadin-checkbox
+`<vaadin24-checkbox
   aria-disabled="true"
   disabled=""
   has-value=""
@@ -40,7 +40,7 @@
     type="checkbox"
     value="on"
   >
-</vaadin-checkbox>
+</vaadin24-checkbox>
 `;
 /* end snapshot vaadin-checkbox host disabled */
 
@@ -62,7 +62,7 @@
 /* end snapshot vaadin-checkbox shadow default */
 
 snapshots["vaadin-checkbox host name"] = 
-`<vaadin-checkbox has-value="">
+`<vaadin24-checkbox has-value="">
   <label
     for="input-vaadin-checkbox-1"
     id="label-vaadin-checkbox-0"
@@ -77,12 +77,12 @@
     type="checkbox"
     value="on"
   >
-</vaadin-checkbox>
+</vaadin24-checkbox>
 `;
 /* end snapshot vaadin-checkbox host name */
 
 snapshots["vaadin-checkbox host label"] = 
-`<vaadin-checkbox
+`<vaadin24-checkbox
   has-label=""
   has-value=""
 >
@@ -100,7 +100,7 @@
     type="checkbox"
     value="on"
   >
-</vaadin-checkbox>
+</vaadin24-checkbox>
 `;
 /* end snapshot vaadin-checkbox host label */
 
