Index: test/keyboard.common.js
===================================================================
--- test/keyboard.common.js
+++ test/keyboard.common.js
@@ -7,7 +7,7 @@
   let select, overlay, menu, valueButton;
 
   beforeEach(async () => {
-    select = fixtureSync('<vaadin-select></vaadin-select>');
+    select = fixtureSync('<vaadin24-select></vaadin24-select>');
     select.items = [
       { label: 'Option 1', value: 'option-1' },
       { label: 'Option 2', value: 'option-2' },
