Index: test/pre-opened-polymer.test.js
===================================================================
--- test/pre-opened-polymer.test.js
+++ test/pre-opened-polymer.test.js
@@ -9,7 +9,7 @@
 describe('pre-opened', () => {
   it('should not throw error when adding a pre-opened select', () => {
     expect(() => {
-      fixtureSync('<vaadin-select opened></vaadin-select>');
+      fixtureSync('<vaadin24-select opened></vaadin24-select>');
     }).to.not.throw(Error);
   });
 });
