Index: test/renderer.test.js
===================================================================
--- test/renderer.test.js
+++ test/renderer.test.js
@@ -1,8 +1,8 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import '@vaadin/item/vaadin-item.js';
-import '@vaadin/list-box/vaadin-list-box.js';
+import '@scoped-vaadin/item/vaadin-item.js';
+import '@scoped-vaadin/list-box/vaadin-list-box.js';
 import './not-animated-styles.js';
 import '../vaadin-select.js';
 
@@ -21,9 +21,9 @@
         return;
       }
 
-      const listBox = document.createElement('vaadin-list-box');
+      const listBox = document.createElement('vaadin23-list-box');
       items.forEach((text) => {
-        const item = document.createElement('vaadin-item');
+        const item = document.createElement('vaadin23-item');
         item.textContent = text + (select.__testVar || '');
         item.value = text;
         listBox.appendChild(item);
@@ -33,10 +33,10 @@
   }
 
   beforeEach(() => {
-    select = fixtureSync(`<vaadin-select></vaadin-select>`);
-    overlay = select.shadowRoot.querySelector('vaadin-select-overlay');
-    rendererContent = document.createElement('vaadin-list-box');
-    const rendererItem = document.createElement('vaadin-item');
+    select = fixtureSync(`<vaadin23-select></vaadin23-select>`);
+    overlay = select.shadowRoot.querySelector('vaadin23-select-overlay');
+    rendererContent = document.createElement('vaadin23-list-box');
+    const rendererItem = document.createElement('vaadin23-item');
     rendererItem.textContent = 'renderer item';
     rendererContent.appendChild(rendererItem);
   });
@@ -75,7 +75,7 @@
   });
 
   it('should not throw when requesting content update before attaching to the DOM', () => {
-    const select = document.createElement('vaadin-select');
+    const select = document.createElement('vaadin23-select');
     expect(() => select.requestContentUpdate()).not.to.throw(Error);
   });
 
@@ -129,7 +129,7 @@
 
   it('should not throw when setting renderer before overlay is ready', () => {
     expect(() => {
-      const select = document.createElement('vaadin-select');
+      const select = document.createElement('vaadin23-select');
       select.renderer = () => {};
       document.body.appendChild(select);
       document.body.removeChild(select);
@@ -146,7 +146,7 @@
 
     it('should work with list-box connected before renderer is set', () => {
       select.renderer = (root) => {
-        const listBox = Array.from(select.children).find((el) => el.tagName.toLowerCase() === 'vaadin-list-box');
+        const listBox = Array.from(select.children).find((el) => el.tagName.toLowerCase() === 'vaadin23-list-box');
         if (listBox) {
           if (root.firstChild) {
             root.removeChild(root.firstChild);
