Index: test/renderer.test.js
===================================================================
--- test/renderer.test.js
+++ test/renderer.test.js
@@ -3,8 +3,8 @@
 import sinon from 'sinon';
 import './not-animated-styles.js';
 import '../src/vaadin-select.js';
-import '@vaadin/item/src/vaadin-item.js';
-import '@vaadin/list-box/src/vaadin-list-box.js';
+import '@scoped-vaadin/item/src/vaadin-item.js';
+import '@scoped-vaadin/list-box/src/vaadin-list-box.js';
 
 describe('renderer', () => {
   let select;
@@ -20,9 +20,9 @@
         return;
       }
 
-      const listBox = document.createElement('vaadin-list-box');
+      const listBox = document.createElement('vaadin24-list-box');
       items.forEach((text) => {
-        const item = document.createElement('vaadin-item');
+        const item = document.createElement('vaadin24-item');
         item.textContent = text + (select.__testVar || '');
         item.value = text;
         listBox.appendChild(item);
@@ -32,11 +32,11 @@
   }
 
   beforeEach(async () => {
-    select = fixtureSync(`<vaadin-select></vaadin-select>`);
+    select = fixtureSync(`<vaadin24-select></vaadin24-select>`);
     await nextRender();
-    overlay = select.shadowRoot.querySelector('vaadin-select-overlay');
-    rendererContent = document.createElement('vaadin-list-box');
-    const rendererItem = document.createElement('vaadin-item');
+    overlay = select.shadowRoot.querySelector('vaadin24-select-overlay');
+    rendererContent = document.createElement('vaadin24-list-box');
+    const rendererItem = document.createElement('vaadin24-item');
     rendererItem.textContent = 'renderer item';
     rendererContent.appendChild(rendererItem);
   });
@@ -80,7 +80,7 @@
   });
 
   it('should not throw when requesting content update before attaching to the DOM', () => {
-    const select = document.createElement('vaadin-select');
+    const select = document.createElement('vaadin24-select');
     expect(() => select.requestContentUpdate()).not.to.throw(Error);
   });
 
@@ -121,7 +121,7 @@
     select.requestContentUpdate();
     await nextUpdate(select);
     expect(select._menuElement.selected).to.be.equal(1);
-    const valueButton = select.querySelector('vaadin-select-value-button');
+    const valueButton = select.querySelector('vaadin24-select-value-button');
     expect(valueButton.textContent.trim()).to.be.equal('barbaz');
   });
 
@@ -138,7 +138,7 @@
 
   it('should not throw when setting renderer before overlay is ready', () => {
     expect(() => {
-      const select = document.createElement('vaadin-select');
+      const select = document.createElement('vaadin24-select');
       select.renderer = () => {};
       document.body.appendChild(select);
       document.body.removeChild(select);
@@ -155,7 +155,7 @@
 
     it('should work with list-box connected before renderer is set', async () => {
       select.renderer = (root) => {
-        const listBox = Array.from(select.children).find((el) => el.tagName.toLowerCase() === 'vaadin-list-box');
+        const listBox = Array.from(select.children).find((el) => el.tagName.toLowerCase() === 'vaadin24-list-box');
         if (listBox) {
           if (root.firstChild) {
             root.removeChild(root.firstChild);
