Index: test/select.test.js
===================================================================
--- test/select.test.js
+++ test/select.test.js
@@ -17,18 +17,18 @@
 import './not-animated-styles.js';
 import './select-test-styles.test.js';
 import '../src/vaadin-select.js';
-import '@vaadin/item/src/vaadin-item.js';
-import '@vaadin/list-box/src/vaadin-list-box.js';
+import '@scoped-vaadin/item/src/vaadin-item.js';
+import '@scoped-vaadin/list-box/src/vaadin-list-box.js';
 import { html, render } from 'lit';
 
-describe('vaadin-select', () => {
+describe('vaadin24-select', () => {
   let select, valueButton;
 
   describe('empty', () => {
     let select;
 
     beforeEach(async () => {
-      select = fixtureSync(`<vaadin-select></vaadin-select>`);
+      select = fixtureSync(`<vaadin24-select></vaadin24-select>`);
       await nextRender();
     });
 
@@ -59,8 +59,8 @@
         if (root.firstElementChild) {
           return;
         }
-        const menu = document.createElement('vaadin-list-box');
-        const item = document.createElement('vaadin-item');
+        const menu = document.createElement('vaadin24-list-box');
+        const item = document.createElement('vaadin24-item');
         menu.appendChild(item);
         root.appendChild(menu);
       };
@@ -72,9 +72,9 @@
     it('should assign menu element set using innerHTML after opening', async () => {
       const renderer = (root) => {
         root.innerHTML = `
-          <vaadin-list-box>
-            <vaadin-item>Test<vaadin-item>
-          </vaadin-list-box>
+          <vaadin24-list-box>
+            <vaadin24-item>Test<vaadin24-item>
+          </vaadin24-list-box>
         `;
       };
       select.renderer = renderer;
@@ -92,28 +92,28 @@
 
   describe('with items', () => {
     beforeEach(async () => {
-      select = fixtureSync('<vaadin-select></vaadin-select>');
+      select = fixtureSync('<vaadin24-select></vaadin24-select>');
       await nextRender();
       select.renderer = (root) => {
         render(
           html`
-            <vaadin-list-box>
-              <vaadin-item>Option 1</vaadin-item>
-              <vaadin-item value="v2" label="o2"><span>Option 2</span></vaadin-item>
-              <vaadin-item value="">Option 3</vaadin-item>
-              <vaadin-item></vaadin-item>
-              <vaadin-item label="">Empty</vaadin-item>
-              <vaadin-item value="v4" disabled>Disabled</vaadin-item>
-              <vaadin-item value="5">A number</vaadin-item>
-              <vaadin-item value="false">A boolean</vaadin-item>
-              <vaadin-item label="foo"></vaadin-item>
-              <vaadin-item><img src="data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=" alt="" /></vaadin-item>
-            </vaadin-list-box>
+            <vaadin24-list-box>
+              <vaadin24-item>Option 1</vaadin24-item>
+              <vaadin24-item value="v2" label="o2"><span>Option 2</span></vaadin24-item>
+              <vaadin24-item value="">Option 3</vaadin24-item>
+              <vaadin24-item></vaadin24-item>
+              <vaadin24-item label="">Empty</vaadin24-item>
+              <vaadin24-item value="v4" disabled>Disabled</vaadin24-item>
+              <vaadin24-item value="5">A number</vaadin24-item>
+              <vaadin24-item value="false">A boolean</vaadin24-item>
+              <vaadin24-item label="foo"></vaadin24-item>
+              <vaadin24-item><img src="data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=" alt="" /></vaadin24-item>
+            </vaadin24-list-box>
           `,
           root,
         );
       };
-      valueButton = select.querySelector('vaadin-select-value-button');
+      valueButton = select.querySelector('vaadin24-select-value-button');
       await nextUpdate(select);
     });
 
@@ -187,7 +187,7 @@
         menu.selected = 1;
         await nextUpdate(select);
         const item = valueButton.firstElementChild;
-        expect(item.localName).to.equal('vaadin-select-item');
+        expect(item.localName).to.equal('vaadin24-select-item');
         expect(item.textContent).to.equal('o2');
         expect(item.selected).to.be.true;
         expect(item.getAttribute('tabindex')).to.be.null;
@@ -448,7 +448,7 @@
 
       // IOS Safari has incorrect viewport height when navigation bar is
       // visible in landscape orientation. This is workarounded by exposing
-      // --vaadin-overlay-viewport-bottom in <vaadin-overlay>.
+      // --vaadin-overlay-viewport-bottom in <vaadin24-overlay>.
       it('should support --vaadin-overlay-viewport-bottom CSS property', () => {
         overlay.setAttribute('phone', '');
         overlay.style.setProperty('--vaadin-overlay-viewport-bottom', '50px');
@@ -753,20 +753,20 @@
     let menu, valueButton;
 
     beforeEach(async () => {
-      select = fixtureSync(`<vaadin-select value="v2"></vaadin-select>`);
+      select = fixtureSync(`<vaadin24-select value="v2"></vaadin24-select>`);
       await nextRender();
       select.renderer = (root) => {
         render(
           html`
-            <vaadin-list-box>
-              <vaadin-item value="v1">t1</vaadin-item>
-              <vaadin-item value="v2" label="o2">t2</vaadin-item>
-            </vaadin-list-box>
+            <vaadin24-list-box>
+              <vaadin24-item value="v1">t1</vaadin24-item>
+              <vaadin24-item value="v2" label="o2">t2</vaadin24-item>
+            </vaadin24-list-box>
           `,
           root,
         );
       };
-      valueButton = select.querySelector('vaadin-select-value-button');
+      valueButton = select.querySelector('vaadin24-select-value-button');
       await nextUpdate(select);
       menu = select._menuElement;
     });
@@ -783,14 +783,14 @@
     beforeEach(async () => {
       container = fixtureSync(`
         <div style="display: flex; flex-direction: column; width: 500px;">
-          <vaadin-select></vaadin-select>
+          <vaadin24-select></vaadin24-select>
         </div>
       `);
       await nextRender();
     });
 
     it('should stretch inside a column flex container', () => {
-      const select = container.querySelector('vaadin-select');
+      const select = container.querySelector('vaadin24-select');
       expect(window.getComputedStyle(container).width).to.eql('500px');
       expect(parseFloat(window.getComputedStyle(select).width)).to.eql(500);
     });
@@ -798,7 +798,7 @@
 
   describe('pre-opened', () => {
     beforeEach(() => {
-      select = document.createElement('vaadin-select');
+      select = document.createElement('vaadin24-select');
       select.items = [{ label: 'Option 1', value: 'value-1' }];
     });
 
