Index: test/select-test-styles.test.js
===================================================================
--- test/select-test-styles.test.js
+++ test/select-test-styles.test.js
@@ -1,9 +1,9 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 // TODO: subset of Lumo needed for unit tests to pass.
 // These should be eventually covered by base styles.
 registerStyles(
-  'vaadin-item',
+  'vaadin24-item',
   css`
     :host {
       display: flex;
