Index: test/template.test.js
===================================================================
--- test/template.test.js
+++ test/template.test.js
@@ -1,8 +1,8 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync } from '@vaadin/testing-helpers';
-import '@vaadin/item';
-import '@vaadin/list-box';
-import '@vaadin/polymer-legacy-adapter/template-renderer.js';
+import '@scoped-vaadin/item';
+import '@scoped-vaadin/list-box';
+import '@scoped-vaadin/polymer-legacy-adapter/template-renderer.js';
 import './not-animated-styles.js';
 import '../vaadin-select.js';
 
@@ -11,17 +11,17 @@
 
   beforeEach(() => {
     select = fixtureSync(`
-      <vaadin-select>
+      <vaadin24-select>
         <template>
-          <vaadin-list-box>
-            <vaadin-item>item</vaadin-item>
-          </vaadin-list-box>
+          <vaadin24-list-box>
+            <vaadin24-item>item</vaadin24-item>
+          </vaadin24-list-box>
         </template>
-      </vaadin-select>
+      </vaadin24-select>
     `);
 
-    overlay = select.shadowRoot.querySelector('vaadin-select-overlay');
-    listBox = overlay.querySelector('vaadin-list-box');
+    overlay = select.shadowRoot.querySelector('vaadin24-select-overlay');
+    listBox = overlay.querySelector('vaadin24-list-box');
 
     select.opened = true;
   });
