Index: test/validation.common.js
===================================================================
--- test/validation.common.js
+++ test/validation.common.js
@@ -2,8 +2,8 @@
 import { fixtureSync, nextRender, nextUpdate, outsideClick } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
-import '@vaadin/item/vaadin-item.js';
-import '@vaadin/list-box/vaadin-list-box.js';
+import '@scoped-vaadin/item/vaadin-item.js';
+import '@scoped-vaadin/list-box/vaadin-list-box.js';
 
 describe('validation', () => {
   let select, validateSpy, valueChangedSpy, changeSpy;
@@ -12,7 +12,7 @@
     beforeEach(async () => {
       select = fixtureSync(
         `<div>
-          <vaadin-select></vaadin-select>
+          <vaadin24-select></vaadin24-select>
           <input id="last-global-focusable" />
         </div>`,
       ).firstElementChild;
@@ -144,7 +144,7 @@
 
   describe('required', () => {
     beforeEach(async () => {
-      select = fixtureSync('<vaadin-select required></vaadin-select>');
+      select = fixtureSync('<vaadin24-select required></vaadin24-select>');
       select.items = [
         { label: '', value: '' },
         { label: 'Option 1', value: 'option-1' },
@@ -195,7 +195,7 @@
     let validateSpy;
 
     beforeEach(() => {
-      select = document.createElement('vaadin-select');
+      select = document.createElement('vaadin24-select');
       validateSpy = sinon.spy(select, 'validate');
     });
 
