Index: test/validation.common.js
===================================================================
--- test/validation.common.js
+++ test/validation.common.js
@@ -2,15 +2,15 @@
 import { fixtureSync, nextRender, nextUpdate, outsideClick } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
-import '@vaadin/item/vaadin-item.js';
-import '@vaadin/list-box/vaadin-list-box.js';
+import '@scoped-vaadin/item/vaadin-item.js';
+import '@scoped-vaadin/list-box/vaadin-list-box.js';
 
 describe('validation', () => {
   let select, validateSpy, valueChangedSpy, changeSpy;
 
   describe('basic', () => {
     beforeEach(async () => {
-      select = fixtureSync('<vaadin-select></vaadin-select>');
+      select = fixtureSync('<vaadin24-select></vaadin24-select>');
       select.items = [
         { label: 'Option 1', value: 'option-1' },
         { label: 'Option 2', value: 'option-2' },
@@ -139,7 +139,7 @@
 
   describe('required', () => {
     beforeEach(async () => {
-      select = fixtureSync('<vaadin-select required></vaadin-select>');
+      select = fixtureSync('<vaadin24-select required></vaadin24-select>');
       select.items = [
         { label: '', value: '' },
         { label: 'Option 1', value: 'option-1' },
@@ -190,7 +190,7 @@
     let validateSpy;
 
     beforeEach(() => {
-      select = document.createElement('vaadin-select');
+      select = document.createElement('vaadin24-select');
       validateSpy = sinon.spy(select, 'validate');
     });
 
