Index: test/animation.test.js
===================================================================
--- test/animation.test.js
+++ test/animation.test.js
@@ -10,8 +10,8 @@
   beforeEach(async () => {
     wrapper = fixtureSync(`
       <div>
-        <vaadin-notification position="bottom-center"></vaadin-notification>
-        <vaadin-notification position="middle"></vaadin-notification>
+        <vaadin24-notification position="bottom-center"></vaadin24-notification>
+        <vaadin24-notification position="middle"></vaadin24-notification>
       </div>
     `);
     notifications = Array.from(wrapper.children);
