Index: test/lit-renderer-directives.test.js
===================================================================
--- test/lit-renderer-directives.test.js
+++ test/lit-renderer-directives.test.js
@@ -7,13 +7,13 @@
 
 async function renderNotification(container, { content }) {
   render(
-    html`<vaadin-notification
+    html`<vaadin23-notification
       ${content ? notificationRenderer(() => html`${content}`, content) : null}
-    ></vaadin-notification>`,
+    ></vaadin23-notification>`,
     container,
   );
   await nextFrame();
-  return container.querySelector('vaadin-notification');
+  return container.querySelector('vaadin23-notification');
 }
 
 describe('lit renderer directives', () => {
@@ -27,7 +27,7 @@
     describe('basic', () => {
       beforeEach(async () => {
         notification = await renderNotification(container, { content: 'Content' });
-        card = notification.shadowRoot.querySelector('vaadin-notification-card');
+        card = notification.shadowRoot.querySelector('vaadin23-notification-card');
         notification.open();
       });
 
@@ -56,11 +56,11 @@
       beforeEach(async () => {
         rendererSpy = sinon.spy();
         render(
-          html`<vaadin-notification opened ${notificationRenderer(rendererSpy)}></vaadin-notification>`,
+          html`<vaadin23-notification opened ${notificationRenderer(rendererSpy)}></vaadin23-notification>`,
           container,
         );
         await nextFrame();
-        notification = container.querySelector('vaadin-notification');
+        notification = container.querySelector('vaadin23-notification');
       });
 
       it('should pass the notification instance to the renderer', () => {
