Index: test/lit.test.js
===================================================================
--- test/lit.test.js
+++ test/lit.test.js
@@ -8,7 +8,7 @@
     let notification;
 
     beforeEach(() => {
-      notification = fixtureSync(`<vaadin-notification></vaadin-notification>`);
+      notification = fixtureSync(`<vaadin23-notification></vaadin23-notification>`);
       notification.open();
       notification.renderer = (root) => {
         render(html`Initial Content`, root);
