Index: test/not-animated-styles.js
===================================================================
--- test/not-animated-styles.js
+++ test/not-animated-styles.js
@@ -1,7 +1,7 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-notification-card',
+  'vaadin24-notification-card',
   css`
     :host([opening]),
     :host([closing]) {
