Index: test/renderer.test.js
===================================================================
--- test/renderer.test.js
+++ test/renderer.test.js
@@ -12,7 +12,7 @@
       rendererContent = document.createElement('p');
       rendererContent.textContent = 'renderer-content';
 
-      notification = fixtureSync('<vaadin-notification></vaadin-notification>');
+      notification = fixtureSync('<vaadin24-notification></vaadin24-notification>');
       await nextFrame();
 
       // Force sync card attaching and removal instead of waiting for the animation
@@ -106,7 +106,7 @@
     let notification;
 
     beforeEach(() => {
-      notification = document.createElement('vaadin-notification');
+      notification = document.createElement('vaadin24-notification');
     });
 
     afterEach(() => {
