Index: theme/material/vaadin-select-styles.d.ts
===================================================================
--- theme/material/vaadin-select-styles.d.ts
+++ theme/material/vaadin-select-styles.d.ts
@@ -3,5 +3,5 @@
  * Copyright (c) 2017 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/theme/material/vaadin-input-container-styles.js';
-import '@vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/input-container/theme/material/vaadin-input-container-styles.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
