Index: theme/material/vaadin-select-styles.js
===================================================================
--- theme/material/vaadin-select-styles.js
+++ theme/material/vaadin-select-styles.js
@@ -3,17 +3,17 @@
  * Copyright (c) 2017 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/theme/material/vaadin-input-container-styles.js';
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import { item } from '@vaadin/item/theme/material/vaadin-item-styles.js';
-import { listBox } from '@vaadin/list-box/theme/material/vaadin-list-box-styles.js';
-import { inputFieldShared } from '@vaadin/vaadin-material-styles/mixins/input-field-shared.js';
-import { menuOverlay } from '@vaadin/vaadin-material-styles/mixins/menu-overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/input-container/theme/material/vaadin-input-container-styles.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import { item } from '@scoped-vaadin/item/theme/material/vaadin-item-styles.js';
+import { listBox } from '@scoped-vaadin/list-box/theme/material/vaadin-list-box-styles.js';
+import { inputFieldShared } from '@scoped-vaadin/vaadin-material-styles/mixins/input-field-shared.js';
+import { menuOverlay } from '@scoped-vaadin/vaadin-material-styles/mixins/menu-overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
-registerStyles('vaadin-select-item', item, { moduleId: 'material-select-item' });
+registerStyles('vaadin24-select-item', item, { moduleId: 'material-select-item' });
 
-registerStyles('vaadin-select-list-box', listBox, { moduleId: 'material-select-list-box' });
+registerStyles('vaadin24-select-list-box', listBox, { moduleId: 'material-select-list-box' });
 
 const select = css`
   :host {
@@ -52,10 +52,10 @@
   }
 `;
 
-registerStyles('vaadin-select', [inputFieldShared, select], { moduleId: 'material-select' });
+registerStyles('vaadin24-select', [inputFieldShared, select], { moduleId: 'material-select' });
 
 registerStyles(
-  'vaadin-select-value-button',
+  'vaadin24-select-value-button',
   css`
     :host {
       font: inherit;
@@ -81,4 +81,4 @@
   }
 `;
 
-registerStyles('vaadin-select-overlay', [menuOverlay, selectOverlay], { moduleId: 'material-select-overlay' });
+registerStyles('vaadin24-select-overlay', [menuOverlay, selectOverlay], { moduleId: 'material-select-overlay' });
