Index: test/validation.common.js
===================================================================
--- test/validation.common.js
+++ test/validation.common.js
@@ -8,7 +8,7 @@
 
   describe('initial', () => {
     beforeEach(() => {
-      group = document.createElement('vaadin-checkbox-group');
+      group = document.createElement('vaadin24-checkbox-group');
       validateSpy = sinon.spy(group, 'validate');
     });
 
@@ -41,11 +41,11 @@
   describe('basic', () => {
     beforeEach(async () => {
       group = fixtureSync(`
-        <vaadin-checkbox-group>
-          <vaadin-checkbox name="language" value="en" label="English"></vaadin-checkbox>
-          <vaadin-checkbox name="language" value="fr" label="Français"></vaadin-checkbox>
-          <vaadin-checkbox name="language" value="de" label="Deutsch"></vaadin-checkbox>
-        </vaadin-checkbox-group>
+        <vaadin24-checkbox-group>
+          <vaadin24-checkbox name="language" value="en" label="English"></vaadin24-checkbox>
+          <vaadin24-checkbox name="language" value="fr" label="Français"></vaadin24-checkbox>
+          <vaadin24-checkbox name="language" value="de" label="Deutsch"></vaadin24-checkbox>
+        </vaadin24-checkbox-group>
       `);
       await nextRender();
       validateSpy = sinon.spy(group, 'validate');
@@ -140,14 +140,14 @@
 
     beforeEach(async () => {
       group = fixtureSync(`
-        <vaadin-checkbox-group required>
-          <vaadin-checkbox name="language" value="en" label="English"></vaadin-checkbox>
-          <vaadin-checkbox name="language" value="fr" label="Français"></vaadin-checkbox>
-          <vaadin-checkbox name="language" value="de" label="Deutsch"></vaadin-checkbox>
-        </vaadin-checkbox-group>
+        <vaadin24-checkbox-group required>
+          <vaadin24-checkbox name="language" value="en" label="English"></vaadin24-checkbox>
+          <vaadin24-checkbox name="language" value="fr" label="Français"></vaadin24-checkbox>
+          <vaadin24-checkbox name="language" value="de" label="Deutsch"></vaadin24-checkbox>
+        </vaadin24-checkbox-group>
       `);
       await nextFrame();
-      checkboxes = [...group.querySelectorAll('vaadin-checkbox')];
+      checkboxes = [...group.querySelectorAll('vaadin24-checkbox')];
       validateSpy = sinon.spy(group, 'validate');
     });
 
