Index: src/lit/column-renderer-directives.js
===================================================================
--- src/lit/column-renderer-directives.js
+++ src/lit/column-renderer-directives.js
@@ -5,9 +5,9 @@
  */
 /* eslint-disable max-classes-per-file */
 import { directive } from 'lit/directive.js';
-import { microTask } from '@vaadin/component-base/src/async.js';
-import { Debouncer } from '@vaadin/component-base/src/debounce.js';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import { microTask } from '@scoped-vaadin/component-base/src/async.js';
+import { Debouncer } from '@scoped-vaadin/component-base/src/debounce.js';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 import { CONTENT_UPDATE_DEBOUNCER } from './renderer-directives.js';
 
 class AbstractGridColumnRendererDirective extends LitRendererDirective {
@@ -89,9 +89,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-grid-column
+ * `<vaadin23-grid-column
  *   ${columnBodyRenderer((item, model, column) => html`...`)}
- * ></vaadin-grid-column>`
+ * ></vaadin23-grid-column>`
  * ```
  *
  * @param renderer the renderer callback.
@@ -117,9 +117,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-grid-column
+ * `<vaadin23-grid-column
  *   ${columnHeaderRenderer((column) => html`...`)}
- * ></vaadin-grid-column>`
+ * ></vaadin23-grid-column>`
  * ```
  *
  * @param renderer the renderer callback.
@@ -145,9 +145,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-grid-column
+ * `<vaadin23-grid-column
  *   ${columnFooterRenderer((column) => html`...`)}
- * ></vaadin-grid-column>`
+ * ></vaadin23-grid-column>`
  * ```
  *
  * @param renderer the renderer callback.
