Index: test/typings/upload-file.types.ts
===================================================================
--- test/typings/upload-file.types.ts
+++ test/typings/upload-file.types.ts
@@ -8,7 +8,7 @@
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const uploadFile = document.createElement('vaadin-upload-file');
+const uploadFile = document.createElement('vaadin24-upload-file');
 
 // Properties
 assertType<boolean>(uploadFile.complete);
