Index: test/avatar.common.js
===================================================================
--- test/avatar.common.js
+++ test/avatar.common.js
@@ -1,14 +1,15 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, focusin, focusout, mousedown, nextUpdate, oneEvent, tabKeyDown } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import '@vaadin/tooltip/vaadin-tooltip.js';
-import { Tooltip } from '@vaadin/tooltip';
+import '@scoped-vaadin/tooltip/vaadin-tooltip.js';
+import { Tooltip } from '@scoped-vaadin/tooltip';
 
-describe('vaadin-avatar', () => {
+describe('vaadin24-avatar', () => {
   let avatar, imgElement, iconElement, abbrElement;
 
   beforeEach(async () => {
-    avatar = fixtureSync('<vaadin-avatar></vaadin-avatar>');
+    avatar = fixtureSync('<vaadin24-avatar></vaadin24-avatar>');
     await nextUpdate(avatar);
   });
 
@@ -20,11 +21,11 @@
     });
 
     it('should be defined in custom element registry', () => {
-      expect(customElements.get(tagName)).to.be.ok;
+      expect(internalCustomElements.get(tagName)).to.be.ok;
     });
 
     it('should have a valid static "is" getter', () => {
-      expect(customElements.get(tagName).is).to.equal(tagName);
+      expect(internalCustomElements.get(tagName).is).to.equal(tagName);
     });
   });
 
@@ -278,7 +279,7 @@
         container.setAttribute('style', 'width: 100px; height: 100px; overflow: auto;');
         container.appendChild(avatar);
 
-        const overlay = tooltip.shadowRoot.querySelector('vaadin-tooltip-overlay');
+        const overlay = tooltip.shadowRoot.querySelector('vaadin24-tooltip-overlay');
         tabKeyDown(avatar);
         avatar.focus();
         expect(overlay.opened).to.be.true;
@@ -386,7 +387,7 @@
     });
 
     it('should not override custom role set on the avatar', () => {
-      const custom = fixtureSync('<vaadin-avatar role="button"></vaadin-avatar>');
+      const custom = fixtureSync('<vaadin24-avatar role="button"></vaadin24-avatar>');
       expect(custom.getAttribute('role')).to.equal('button');
     });
 
