Index: test/login-submit.test.js
===================================================================
--- test/login-submit.test.js
+++ test/login-submit.test.js
@@ -49,7 +49,7 @@
 
   describe('default', () => {
     beforeEach(() => {
-      login = fixtureSync('<vaadin-login-form action="login-action"></vaadin-login-form>');
+      login = fixtureSync('<vaadin23-login-form action="login-action"></vaadin23-login-form>');
     });
 
     it('should submit form values from login element', (done) => {
@@ -75,7 +75,7 @@
     let overlay;
 
     beforeEach(() => {
-      overlay = fixtureSync('<vaadin-login-overlay action="login-action" opened></vaadin-login-overlay>');
+      overlay = fixtureSync('<vaadin23-login-overlay action="login-action" opened></vaadin23-login-overlay>');
       login = overlay.$.vaadinLoginForm;
     });
 
