Index: test/login-form-wrapper-styles.js
===================================================================
--- test/login-form-wrapper-styles.js
+++ test/login-form-wrapper-styles.js
@@ -1,7 +1,7 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-login-form-wrapper',
+  'vaadin24-login-form-wrapper',
   css`
     :host([theme='green']) [part='form-title'],
     :host([theme='green']) [part='error-message-title'],
