Index: test/login-submit.common.js
===================================================================
--- test/login-submit.common.js
+++ test/login-submit.common.js
@@ -29,7 +29,7 @@
       done();
     };
 
-    login.querySelector('vaadin-button').click();
+    login.querySelector('vaadin24-button').click();
     expect(submitSpy.called).to.equal(expectation);
 
     if (preventDefault) {
@@ -55,7 +55,7 @@
 
   describe('default', () => {
     beforeEach(async () => {
-      login = fixtureSync('<vaadin-login-form action="login-action"></vaadin-login-form>');
+      login = fixtureSync('<vaadin24-login-form action="login-action"></vaadin24-login-form>');
       await nextRender();
     });
 
@@ -82,7 +82,7 @@
     let overlay;
 
     beforeEach(async () => {
-      overlay = fixtureSync('<vaadin-login-overlay action="login-action" opened></vaadin-login-overlay>');
+      overlay = fixtureSync('<vaadin24-login-overlay action="login-action" opened></vaadin24-login-overlay>');
       await nextRender();
       login = overlay.$.vaadinLoginForm;
     });
@@ -97,10 +97,10 @@
 
     beforeEach(async () => {
       overlay = fixtureSync(`
-        <vaadin-login-overlay opened>
+        <vaadin24-login-overlay opened>
           <input name="foo" value="bar" slot="custom-form-area">
-          <vaadin-text-field name="code" value="1234" slot="custom-form-area"></vaadin-text-field>
-        </vaadin-login-overlay>
+          <vaadin24-text-field name="code" value="1234" slot="custom-form-area"></vaadin24-text-field>
+        </vaadin24-login-overlay>
       `);
       await nextRender();
       login = overlay.$.vaadinLoginForm;
