Index: test/login-overlay.common.js
===================================================================
--- test/login-overlay.common.js
+++ test/login-overlay.common.js
@@ -7,7 +7,7 @@
   let overlay;
 
   beforeEach(async () => {
-    overlay = fixtureSync('<vaadin-login-overlay></vaadin-login-overlay>');
+    overlay = fixtureSync('<vaadin24-login-overlay></vaadin24-login-overlay>');
     await nextRender();
   });
 
@@ -18,7 +18,7 @@
   it('should render form wrapper when opened', async () => {
     overlay.opened = true;
     await nextRender();
-    expect(document.querySelector('vaadin-login-form-wrapper')).to.be.ok;
+    expect(document.querySelector('vaadin24-login-form-wrapper')).to.be.ok;
   });
 });
 
@@ -34,7 +34,7 @@
   });
 
   beforeEach(async () => {
-    overlay = fixtureSync('<vaadin-login-overlay opened theme="some-theme"></vaadin-login-overlay>');
+    overlay = fixtureSync('<vaadin24-login-overlay opened theme="some-theme"></vaadin24-login-overlay>');
     await nextRender();
     await nextUpdate(overlay.$.vaadinLoginForm);
   });
@@ -46,13 +46,13 @@
 
   it('should set opened using attribute', () => {
     expect(overlay.opened).to.be.true;
-    expect(document.querySelector('vaadin-login-form-wrapper')).to.exist;
+    expect(document.querySelector('vaadin24-login-form-wrapper')).to.exist;
   });
 
   it('should remove form wrapper when closed', async () => {
     overlay.opened = false;
     await nextUpdate(overlay);
-    expect(document.querySelector('vaadin-login-form-wrapper')).not.to.exist;
+    expect(document.querySelector('vaadin24-login-form-wrapper')).not.to.exist;
   });
 
   it('should not remove form wrapper when moved within DOM', async () => {
@@ -61,11 +61,11 @@
     newParent.appendChild(overlay);
     await nextUpdate(overlay);
 
-    expect(document.querySelector('vaadin-login-form-wrapper')).to.exist;
+    expect(document.querySelector('vaadin24-login-form-wrapper')).to.exist;
   });
 
   it('should propagate theme to a wrapper', () => {
-    const wrapper = document.querySelector('vaadin-login-overlay-wrapper');
+    const wrapper = document.querySelector('vaadin24-login-overlay-wrapper');
     expect(wrapper.getAttribute('theme')).to.be.equal('some-theme');
   });
 
@@ -128,7 +128,7 @@
   let overlay;
 
   beforeEach(() => {
-    overlay = fixtureSync('<vaadin-login-overlay no-autofocus></vaadin-login-overlay>');
+    overlay = fixtureSync('<vaadin24-login-overlay no-autofocus></vaadin24-login-overlay>');
   });
 
   it('should not focus the username field', async () => {
@@ -144,7 +144,7 @@
 
   beforeEach(async () => {
     overlay = fixtureSync(`
-      <vaadin-login-overlay title="New title" description="New description" opened></vaadin-login-overlay>
+      <vaadin24-login-overlay title="New title" description="New description" opened></vaadin24-login-overlay>
     `);
     await nextRender();
     headerElement = overlay.$.vaadinLoginOverlayWrapper.shadowRoot.querySelector('[part="brand"] h1');
@@ -197,9 +197,9 @@
 
   beforeEach(async () => {
     overlay = fixtureSync(`
-      <vaadin-login-overlay description="New description" opened>
+      <vaadin24-login-overlay description="New description" opened>
         <div slot="title">Teleported title</div>
-      </vaadin-login-overlay>
+      </vaadin24-login-overlay>
     `);
     await nextRender();
     overlayWrapper = overlay.$.vaadinLoginOverlayWrapper;
@@ -270,10 +270,10 @@
 
   beforeEach(async () => {
     overlay = fixtureSync(`
-      <vaadin-login-overlay>
+      <vaadin24-login-overlay>
         <input id="one" slot="custom-form-area" />
         <input id="two" slot="custom-form-area" />
-      </vaadin-login-overlay>
+      </vaadin24-login-overlay>
     `);
     await nextRender();
     form = overlay.$.vaadinLoginForm;
@@ -284,11 +284,11 @@
     overlay.opened = true;
     await nextRender();
 
-    const wrapper = form.querySelector('vaadin-login-form-wrapper');
+    const wrapper = form.querySelector('vaadin24-login-form-wrapper');
     expect(inputs[0].parentElement).to.equal(wrapper);
     expect(inputs[1].parentElement).to.equal(wrapper);
 
-    const button = wrapper.querySelector('vaadin-button');
+    const button = wrapper.querySelector('vaadin24-button');
     expect(inputs[0].nextElementSibling).to.equal(inputs[1]);
     expect(inputs[1].nextElementSibling).to.equal(button);
 
@@ -305,10 +305,10 @@
 
   beforeEach(async () => {
     overlay = fixtureSync(`
-      <vaadin-login-overlay>
+      <vaadin24-login-overlay>
         <div id="foo" slot="footer">Foo</div>
         <div id="bar" slot="footer">Bar</div>
-      </vaadin-login-overlay>
+      </vaadin24-login-overlay>
     `);
     await nextRender();
     form = overlay.$.vaadinLoginForm;
@@ -319,7 +319,7 @@
     overlay.opened = true;
     await nextRender();
 
-    const wrapper = form.querySelector('vaadin-login-form-wrapper');
+    const wrapper = form.querySelector('vaadin24-login-form-wrapper');
     expect(divs[0].parentElement).to.equal(wrapper);
     expect(divs[1].parentElement).to.equal(wrapper);
 
