Index: grid/theme/material/vaadin-grid-styles.js
===================================================================
--- grid/theme/material/vaadin-grid-styles.js	null
+++ grid/theme/material/vaadin-grid-styles.js	null
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-grid',
+  'vaadin24-grid',
   css`
     :host {
       background-color: var(--material-background-color);
