Index: grid/theme/material/vaadin-grid-tree-toggle-styles.js
===================================================================
--- grid/theme/material/vaadin-grid-tree-toggle-styles.js
+++ grid/theme/material/vaadin-grid-tree-toggle-styles.js
@@ -1,10 +1,10 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-grid-tree-toggle',
+  'vaadin24-grid-tree-toggle',
   css`
     :host {
       --_material-grid-tree-toggle-collapsed-icon-transform: rotate(0);
