Index: test/checked-mixin.test.js
===================================================================
--- test/checked-mixin.test.js
+++ test/checked-mixin.test.js
@@ -1,13 +1,14 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { fire, fixtureSync } from '@vaadin/testing-helpers';
 import { resetMouse, sendMouse } from '@web/test-runner-commands';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
 import { CheckedMixin } from '../src/checked-mixin.js';
 import { DelegateFocusMixin } from '../src/delegate-focus-mixin.js';
 import { InputController } from '../src/input-controller.js';
 
-customElements.define(
+internalCustomElements.define(
   'checked-mixin-element',
   class extends CheckedMixin(DelegateFocusMixin(ControllerMixin(PolymerElement))) {
     static get template() {
