Index: test/clear-button-mixin.test.js
===================================================================
--- test/clear-button-mixin.test.js
+++ test/clear-button-mixin.test.js
@@ -10,9 +10,9 @@
   nextUpdate,
 } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { isTouch } from '@scoped-vaadin/component-base/src/browser-utils.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
 import { ClearButtonMixin } from '../src/clear-button-mixin.js';
 import { InputController } from '../src/input-controller.js';
 
