Index: test/field-aria-controller.test.js
===================================================================
--- test/field-aria-controller.test.js
+++ test/field-aria-controller.test.js
@@ -1,10 +1,11 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync } from '@vaadin/testing-helpers';
 import { PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
 import { FieldAriaController } from '../src/field-aria-controller.js';
 
-customElements.define('field-element', class extends ControllerMixin(PolymerElement) {});
+internalCustomElements.define('field-element', class extends ControllerMixin(PolymerElement) {});
 
 describe('field-aria-controller', () => {
   let element, input, controller;
