Index: test/input-control-mixin.test.js
===================================================================
--- test/input-control-mixin.test.js
+++ test/input-control-mixin.test.js
@@ -1,12 +1,13 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { escKeyDown, fixtureSync, keyboardEventFor, keyDownOn, mousedown } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
+import { isTouch } from '@scoped-vaadin/component-base/src/browser-utils.js';
 import { InputControlMixin } from '../src/input-control-mixin.js';
 import { InputController } from '../src/input-controller.js';
 
-customElements.define(
+internalCustomElements.define(
   'input-control-mixin-element',
   class extends InputControlMixin(PolymerElement) {
     static get template() {
