Index: test/shadow-focus-mixin.test.js
===================================================================
--- test/shadow-focus-mixin.test.js
+++ test/shadow-focus-mixin.test.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { aTimeout, fixtureSync, focusin, focusout, keyDownOn } from '@vaadin/testing-helpers';
 import { resetMouse, sendKeys, sendMouse } from '@web/test-runner-commands';
@@ -5,7 +6,7 @@
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
 import { ShadowFocusMixin } from '../src/shadow-focus-mixin.js';
 
-customElements.define(
+internalCustomElements.define(
   'shadow-focus-element',
   class extends ShadowFocusMixin(PolymerElement) {
     static get template() {
@@ -23,7 +24,7 @@
   },
 );
 
-customElements.define(
+internalCustomElements.define(
   'shadow-focus-wrapper',
   class extends ShadowFocusMixin(PolymerElement) {
     static get template() {
