Index: test/slot-styles-mixin.test.js
===================================================================
--- test/slot-styles-mixin.test.js
+++ test/slot-styles-mixin.test.js
@@ -1,7 +1,8 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync } from '@vaadin/testing-helpers';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { SlotMixin } from '@vaadin/component-base/src/slot-mixin.js';
+import { SlotMixin } from '@scoped-vaadin/component-base/src/slot-mixin.js';
 import { SlotStylesMixin } from '../src/slot-styles-mixin.js';
 
 describe('slot-styles-mixin', () => {
@@ -9,7 +10,7 @@
   let wrapper, element, button;
 
   before(() => {
-    customElements.define(
+    internalCustomElements.define(
       'slot-styles-mixin-element',
       class extends SlotStylesMixin(SlotMixin(PolymerElement)) {
         static get template() {
