Index: test/text-area-controller.test.js
===================================================================
--- test/text-area-controller.test.js
+++ test/text-area-controller.test.js
@@ -1,11 +1,12 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync } from '@vaadin/testing-helpers';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
 import { InputMixin } from '../src/input-mixin.js';
 import { TextAreaController } from '../src/text-area-controller.js';
 
-customElements.define(
+internalCustomElements.define(
   'textarea-controller-element',
   class extends InputMixin(ControllerMixin(PolymerElement)) {
     static get template() {
