Index: test/visual/material/login-overlay.test.js
===================================================================
--- test/visual/material/login-overlay.test.js
+++ test/visual/material/login-overlay.test.js
@@ -9,7 +9,7 @@
   beforeEach(() => {
     div = document.createElement('div');
     div.style.height = '100%';
-    element = fixtureSync('<vaadin-login-overlay opened></vaadin-login-overlay>', div);
+    element = fixtureSync('<vaadin23-login-overlay opened></vaadin23-login-overlay>', div);
   });
 
   it('basic', async () => {
