Index: notification/theme/material/vaadin-notification-styles.js
===================================================================
--- notification/theme/material/vaadin-notification-styles.js	null
+++ notification/theme/material/vaadin-notification-styles.js	null
@@ -1,10 +1,10 @@
-import '@vaadin/vaadin-material-styles/typography.js';
-import '@vaadin/vaadin-material-styles/shadow.js';
-import { colorDark } from '@vaadin/vaadin-material-styles/color.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import '@scoped-vaadin/vaadin-material-styles/shadow.js';
+import { colorDark } from '@scoped-vaadin/vaadin-material-styles/color.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-notification-container',
+  'vaadin24-notification-container',
   css`
     :host {
       inset: 0 4px;
@@ -145,6 +145,6 @@
   }
 `;
 
-registerStyles('vaadin-notification-card', [colorDark, notificationCard], {
+registerStyles('vaadin24-notification-card', [colorDark, notificationCard], {
   moduleId: 'material-notification-card',
 });
