Index: theme/lumo/vaadin-custom-field-styles.d.ts
===================================================================
--- theme/lumo/vaadin-custom-field-styles.d.ts
+++ theme/lumo/vaadin-custom-field-styles.d.ts
@@ -3,9 +3,9 @@
  * Copyright (c) 2019 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
 declare const customField: import("lit").CSSResult;
 export { customField };
