Index: test/dom/__snapshots__/date-picker.test.snap.js
===================================================================
--- test/dom/__snapshots__/date-picker.test.snap.js
+++ test/dom/__snapshots__/date-picker.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-date-picker host default"] = 
-`<vaadin-date-picker>
+`<vaadin24-date-picker>
   <label
     for="input-vaadin-date-picker-3"
     id="label-vaadin-date-picker-0"
@@ -23,12 +23,12 @@
     role="combobox"
     slot="input"
   >
-</vaadin-date-picker>
+</vaadin24-date-picker>
 `;
 /* end snapshot vaadin-date-picker host default */
 
 snapshots["vaadin-date-picker host disabled"] = 
-`<vaadin-date-picker
+`<vaadin24-date-picker
   aria-disabled="true"
   disabled=""
 >
@@ -54,12 +54,12 @@
     slot="input"
     tabindex="-1"
   >
-</vaadin-date-picker>
+</vaadin24-date-picker>
 `;
 /* end snapshot vaadin-date-picker host disabled */
 
 snapshots["vaadin-date-picker host placeholder"] = 
-`<vaadin-date-picker placeholder="Placeholder">
+`<vaadin24-date-picker placeholder="Placeholder">
   <label
     for="input-vaadin-date-picker-3"
     id="label-vaadin-date-picker-0"
@@ -81,12 +81,12 @@
     role="combobox"
     slot="input"
   >
-</vaadin-date-picker>
+</vaadin24-date-picker>
 `;
 /* end snapshot vaadin-date-picker host placeholder */
 
 snapshots["vaadin-date-picker host readonly"] = 
-`<vaadin-date-picker readonly="">
+`<vaadin24-date-picker readonly="">
   <label
     for="input-vaadin-date-picker-3"
     id="label-vaadin-date-picker-0"
@@ -108,12 +108,12 @@
     role="combobox"
     slot="input"
   >
-</vaadin-date-picker>
+</vaadin24-date-picker>
 `;
 /* end snapshot vaadin-date-picker host readonly */
 
 snapshots["vaadin-date-picker host required"] = 
-`<vaadin-date-picker required="">
+`<vaadin24-date-picker required="">
   <label
     for="input-vaadin-date-picker-3"
     id="label-vaadin-date-picker-0"
@@ -135,12 +135,12 @@
     role="combobox"
     slot="input"
   >
-</vaadin-date-picker>
+</vaadin24-date-picker>
 `;
 /* end snapshot vaadin-date-picker host required */
 
 snapshots["vaadin-date-picker host label"] = 
-`<vaadin-date-picker has-label="">
+`<vaadin24-date-picker has-label="">
   <label
     for="input-vaadin-date-picker-3"
     id="label-vaadin-date-picker-0"
@@ -163,12 +163,12 @@
     role="combobox"
     slot="input"
   >
-</vaadin-date-picker>
+</vaadin24-date-picker>
 `;
 /* end snapshot vaadin-date-picker host label */
 
 snapshots["vaadin-date-picker host helper"] = 
-`<vaadin-date-picker has-helper="">
+`<vaadin24-date-picker has-helper="">
   <label
     for="input-vaadin-date-picker-3"
     id="label-vaadin-date-picker-0"
@@ -196,12 +196,12 @@
   >
     Helper
   </div>
-</vaadin-date-picker>
+</vaadin24-date-picker>
 `;
 /* end snapshot vaadin-date-picker host helper */
 
 snapshots["vaadin-date-picker host error"] = 
-`<vaadin-date-picker
+`<vaadin24-date-picker
   has-error-message=""
   invalid=""
 >
@@ -229,12 +229,12 @@
     role="combobox"
     slot="input"
   >
-</vaadin-date-picker>
+</vaadin24-date-picker>
 `;
 /* end snapshot vaadin-date-picker host error */
 
 snapshots["vaadin-date-picker host name"] = 
-`<vaadin-date-picker name="Field Name">
+`<vaadin24-date-picker name="Field Name">
   <label
     for="input-vaadin-date-picker-3"
     id="label-vaadin-date-picker-0"
@@ -256,12 +256,12 @@
     role="combobox"
     slot="input"
   >
-</vaadin-date-picker>
+</vaadin24-date-picker>
 `;
 /* end snapshot vaadin-date-picker host name */
 
 snapshots["vaadin-date-picker host value"] = 
-`<vaadin-date-picker has-value="">
+`<vaadin24-date-picker has-value="">
   <label
     for="input-vaadin-date-picker-3"
     id="label-vaadin-date-picker-0"
@@ -282,12 +282,12 @@
     role="combobox"
     slot="input"
   >
-</vaadin-date-picker>
+</vaadin24-date-picker>
 `;
 /* end snapshot vaadin-date-picker host value */
 
 snapshots["vaadin-date-picker host opened overlay"] = 
-`<vaadin-date-picker-overlay
+`<vaadin24-date-picker-overlay
   dir="ltr"
   id="overlay"
   opened=""
@@ -295,161 +295,161 @@
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-date-picker-overlay-content
+  <vaadin24-date-picker-overlay-content
     class="animate"
     desktop=""
     role="dialog"
   >
-    <vaadin-button
+    <vaadin24-button
       role="button"
       slot="today-button"
       tabindex="0"
       theme="tertiary"
     >
       Today
-    </vaadin-button>
-    <vaadin-button
+    </vaadin24-button>
+    <vaadin24-button
       role="button"
       slot="cancel-button"
       tabindex="0"
       theme="tertiary"
     >
       Cancel
-    </vaadin-button>
-    <vaadin-date-picker-month-scroller slot="months">
+    </vaadin24-button>
+    <vaadin24-date-picker-month-scroller slot="months">
       <div slot="vaadin-infinite-scroller-item-content-4">
-        <vaadin-month-calendar aria-hidden="true">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar aria-hidden="true">
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-5">
-        <vaadin-month-calendar aria-hidden="true">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar aria-hidden="true">
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-6">
-        <vaadin-month-calendar aria-hidden="true">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar aria-hidden="true">
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-7">
-        <vaadin-month-calendar>
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar>
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-8">
-        <vaadin-month-calendar aria-hidden="true">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar aria-hidden="true">
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-9">
-        <vaadin-month-calendar aria-hidden="true">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar aria-hidden="true">
+        </vaadin24-month-calendar>
       </div>
-    </vaadin-date-picker-month-scroller>
-    <vaadin-date-picker-year-scroller
+    </vaadin24-date-picker-month-scroller>
+    <vaadin24-date-picker-year-scroller
       aria-hidden="true"
       slot="years"
     >
       <div slot="vaadin-infinite-scroller-item-content-10">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-11">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-12">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-13">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-14">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-15">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-16">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-17">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-18">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-19">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-20">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-21">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-22">
-        <vaadin-date-picker-year current="">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year current="">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-23">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-24">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-25">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-26">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-27">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-28">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-29">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-30">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-31">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-32">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-33">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
-    </vaadin-date-picker-year-scroller>
-  </vaadin-date-picker-overlay-content>
-</vaadin-date-picker-overlay>
+    </vaadin24-date-picker-year-scroller>
+  </vaadin24-date-picker-overlay-content>
+</vaadin24-date-picker-overlay>
 `;
 /* end snapshot vaadin-date-picker host opened overlay */
 
 snapshots["vaadin-date-picker host opened overlay class"] = 
-`<vaadin-date-picker-overlay
+`<vaadin24-date-picker-overlay
   class="custom date-picker-overlay"
   dir="ltr"
   id="overlay"
@@ -458,161 +458,161 @@
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-date-picker-overlay-content
+  <vaadin24-date-picker-overlay-content
     class="animate"
     desktop=""
     role="dialog"
   >
-    <vaadin-button
+    <vaadin24-button
       role="button"
       slot="today-button"
       tabindex="0"
       theme="tertiary"
     >
       Today
-    </vaadin-button>
-    <vaadin-button
+    </vaadin24-button>
+    <vaadin24-button
       role="button"
       slot="cancel-button"
       tabindex="0"
       theme="tertiary"
     >
       Cancel
-    </vaadin-button>
-    <vaadin-date-picker-month-scroller slot="months">
+    </vaadin24-button>
+    <vaadin24-date-picker-month-scroller slot="months">
       <div slot="vaadin-infinite-scroller-item-content-4">
-        <vaadin-month-calendar aria-hidden="true">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar aria-hidden="true">
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-5">
-        <vaadin-month-calendar aria-hidden="true">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar aria-hidden="true">
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-6">
-        <vaadin-month-calendar aria-hidden="true">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar aria-hidden="true">
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-7">
-        <vaadin-month-calendar>
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar>
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-8">
-        <vaadin-month-calendar aria-hidden="true">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar aria-hidden="true">
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-9">
-        <vaadin-month-calendar aria-hidden="true">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar aria-hidden="true">
+        </vaadin24-month-calendar>
       </div>
-    </vaadin-date-picker-month-scroller>
-    <vaadin-date-picker-year-scroller
+    </vaadin24-date-picker-month-scroller>
+    <vaadin24-date-picker-year-scroller
       aria-hidden="true"
       slot="years"
     >
       <div slot="vaadin-infinite-scroller-item-content-10">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-11">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-12">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-13">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-14">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-15">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-16">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-17">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-18">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-19">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-20">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-21">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-22">
-        <vaadin-date-picker-year current="">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year current="">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-23">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-24">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-25">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-26">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-27">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-28">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-29">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-30">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-31">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-32">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-33">
-        <vaadin-date-picker-year>
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
-    </vaadin-date-picker-year-scroller>
-  </vaadin-date-picker-overlay-content>
-</vaadin-date-picker-overlay>
+    </vaadin24-date-picker-year-scroller>
+  </vaadin24-date-picker-overlay-content>
+</vaadin24-date-picker-overlay>
 `;
 /* end snapshot vaadin-date-picker host opened overlay class */
 
 snapshots["vaadin-date-picker host opened overlay theme"] = 
-`<vaadin-date-picker-overlay
+`<vaadin24-date-picker-overlay
   dir="ltr"
   id="overlay"
   opened=""
@@ -621,175 +621,175 @@
   theme="custom"
   top-aligned=""
 >
-  <vaadin-date-picker-overlay-content
+  <vaadin24-date-picker-overlay-content
     class="animate"
     desktop=""
     role="dialog"
     theme="custom"
   >
-    <vaadin-button
+    <vaadin24-button
       role="button"
       slot="today-button"
       tabindex="0"
       theme="tertiary"
     >
       Today
-    </vaadin-button>
-    <vaadin-button
+    </vaadin24-button>
+    <vaadin24-button
       role="button"
       slot="cancel-button"
       tabindex="0"
       theme="tertiary"
     >
       Cancel
-    </vaadin-button>
-    <vaadin-date-picker-month-scroller slot="months">
+    </vaadin24-button>
+    <vaadin24-date-picker-month-scroller slot="months">
       <div slot="vaadin-infinite-scroller-item-content-4">
-        <vaadin-month-calendar
+        <vaadin24-month-calendar
           aria-hidden="true"
           theme="custom"
         >
-        </vaadin-month-calendar>
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-5">
-        <vaadin-month-calendar
+        <vaadin24-month-calendar
           aria-hidden="true"
           theme="custom"
         >
-        </vaadin-month-calendar>
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-6">
-        <vaadin-month-calendar
+        <vaadin24-month-calendar
           aria-hidden="true"
           theme="custom"
         >
-        </vaadin-month-calendar>
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-7">
-        <vaadin-month-calendar theme="custom">
-        </vaadin-month-calendar>
+        <vaadin24-month-calendar theme="custom">
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-8">
-        <vaadin-month-calendar
+        <vaadin24-month-calendar
           aria-hidden="true"
           theme="custom"
         >
-        </vaadin-month-calendar>
+        </vaadin24-month-calendar>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-9">
-        <vaadin-month-calendar
+        <vaadin24-month-calendar
           aria-hidden="true"
           theme="custom"
         >
-        </vaadin-month-calendar>
+        </vaadin24-month-calendar>
       </div>
-    </vaadin-date-picker-month-scroller>
-    <vaadin-date-picker-year-scroller
+    </vaadin24-date-picker-month-scroller>
+    <vaadin24-date-picker-year-scroller
       aria-hidden="true"
       slot="years"
     >
       <div slot="vaadin-infinite-scroller-item-content-10">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-11">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-12">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-13">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-14">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-15">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-16">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-17">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-18">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-19">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-20">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-21">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-22">
-        <vaadin-date-picker-year
+        <vaadin24-date-picker-year
           current=""
           theme="custom"
         >
-        </vaadin-date-picker-year>
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-23">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-24">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-25">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-26">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-27">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-28">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-29">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-30">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-31">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-32">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
       <div slot="vaadin-infinite-scroller-item-content-33">
-        <vaadin-date-picker-year theme="custom">
-        </vaadin-date-picker-year>
+        <vaadin24-date-picker-year theme="custom">
+        </vaadin24-date-picker-year>
       </div>
-    </vaadin-date-picker-year-scroller>
-  </vaadin-date-picker-overlay-content>
-</vaadin-date-picker-overlay>
+    </vaadin24-date-picker-year-scroller>
+  </vaadin24-date-picker-overlay-content>
+</vaadin24-date-picker-overlay>
 `;
 /* end snapshot vaadin-date-picker host opened overlay theme */
 
@@ -804,7 +804,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container part="input-field">
+  <vaadin24-input-container part="input-field">
     <slot
       name="prefix"
       slot="prefix"
@@ -825,7 +825,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -835,11 +835,11 @@
     </slot>
   </div>
 </div>
-<vaadin-date-picker-overlay
+<vaadin24-date-picker-overlay
   id="overlay"
   restore-focus-on-close=""
 >
-</vaadin-date-picker-overlay>
+</vaadin24-date-picker-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -856,7 +856,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     disabled=""
     part="input-field"
   >
@@ -880,7 +880,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -890,11 +890,11 @@
     </slot>
   </div>
 </div>
-<vaadin-date-picker-overlay
+<vaadin24-date-picker-overlay
   id="overlay"
   restore-focus-on-close=""
 >
-</vaadin-date-picker-overlay>
+</vaadin24-date-picker-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -911,7 +911,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     part="input-field"
     readonly=""
   >
@@ -935,7 +935,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -945,11 +945,11 @@
     </slot>
   </div>
 </div>
-<vaadin-date-picker-overlay
+<vaadin24-date-picker-overlay
   id="overlay"
   restore-focus-on-close=""
 >
-</vaadin-date-picker-overlay>
+</vaadin24-date-picker-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -966,7 +966,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     invalid=""
     part="input-field"
   >
@@ -990,7 +990,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -1000,11 +1000,11 @@
     </slot>
   </div>
 </div>
-<vaadin-date-picker-overlay
+<vaadin24-date-picker-overlay
   id="overlay"
   restore-focus-on-close=""
 >
-</vaadin-date-picker-overlay>
+</vaadin24-date-picker-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -1021,7 +1021,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     part="input-field"
     theme="align-right"
   >
@@ -1045,7 +1045,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -1055,12 +1055,12 @@
     </slot>
   </div>
 </div>
-<vaadin-date-picker-overlay
+<vaadin24-date-picker-overlay
   id="overlay"
   restore-focus-on-close=""
   theme="align-right"
 >
-</vaadin-date-picker-overlay>
+</vaadin24-date-picker-overlay>
 <slot name="tooltip">
 </slot>
 `;
