Index: theme/lumo/vaadin-text-area-styles.js
===================================================================
--- theme/lumo/vaadin-text-area-styles.js
+++ theme/lumo/vaadin-text-area-styles.js
@@ -3,12 +3,12 @@
  * Copyright (c) 2017 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/theme/lumo/vaadin-input-container-styles.js';
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { inputFieldShared } from '@vaadin/vaadin-lumo-styles/mixins/input-field-shared.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/input-container/theme/lumo/vaadin-input-container-styles.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { inputFieldShared } from '@scoped-vaadin/vaadin-lumo-styles/mixins/input-field-shared.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const textArea = css`
   [part='input-field'],
@@ -65,13 +65,13 @@
   }
 
   /* Vertically align icon prefix/suffix/clear button with the first line of text */
-  [part='input-field'] ::slotted(vaadin-icon[slot$='fix']),
+  [part='input-field'] ::slotted(vaadin24-icon[slot$='fix']),
   [part='clear-button'] {
     top: calc((var(--lumo-icon-size-m) - 1em * var(--lumo-line-height-s)) / -2);
     margin-top: calc((var(--lumo-icon-size-m) - 1em * var(--lumo-line-height-s)) / -2);
   }
 `;
 
-registerStyles('vaadin-text-area', [inputFieldShared, textArea], {
+registerStyles('vaadin24-text-area', [inputFieldShared, textArea], {
   moduleId: 'lumo-text-area',
 });
