Index: src/define.js
===================================================================
--- src/define.js
+++ src/define.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
@@ -5,7 +6,7 @@
  */
 
 export function defineCustomElement(CustomElement) {
-  const defined = customElements.get(CustomElement.is);
+  const defined = internalCustomElements.get(CustomElement.is);
   if (!defined) {
     Object.defineProperty(CustomElement, 'version', {
       get() {
@@ -13,7 +14,7 @@
       },
     });
 
-    customElements.define(CustomElement.is, CustomElement);
+    internalCustomElements.define(CustomElement.is, CustomElement);
   } else {
     const definedVersion = defined.version;
     if (definedVersion && CustomElement.version && definedVersion === CustomElement.version) {
