Index: src/polylit-mixin.js
===================================================================
--- src/polylit-mixin.js
+++ src/polylit-mixin.js
@@ -224,7 +224,7 @@
     connectedCallback() {
       super.connectedCallback();
 
-      // Components like `vaadin-overlay` are teleported to the body element when opened.
+      // Components like `vaadin24-overlay` are teleported to the body element when opened.
       // If their opened state is set as an attribute, the teleportation happens immediately
       // after they are connected to the DOM. This means they will be outside the scope of
       // querySelectorAll in the parent component's `firstUpdated()`. To ensure their reference
