Index: theme/material/vaadin-upload-styles.js
===================================================================
--- theme/material/vaadin-upload-styles.js
+++ theme/material/vaadin-upload-styles.js
@@ -1,12 +1,12 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import '@vaadin/button/theme/material/vaadin-button-styles.js';
-import '@vaadin/progress-bar/theme/material/vaadin-progress-bar-styles.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import '@scoped-vaadin/button/theme/material/vaadin-button-styles.js';
+import '@scoped-vaadin/progress-bar/theme/material/vaadin-progress-bar-styles.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-upload',
+  'vaadin24-upload',
   css`
     :host(:not([nodrop])) {
       overflow: hidden;
@@ -81,7 +81,7 @@
 );
 
 registerStyles(
-  'vaadin-upload-icon',
+  'vaadin24-upload-icon',
   css`
     :host {
       margin-right: 8px;
@@ -98,7 +98,7 @@
 );
 
 registerStyles(
-  'vaadin-upload-file',
+  'vaadin24-upload-file',
   css`
     :host {
       outline: none;
