Index: theme/lumo/vaadin-avatar-group-styles.js
===================================================================
--- theme/lumo/vaadin-avatar-group-styles.js
+++ theme/lumo/vaadin-avatar-group-styles.js
@@ -1,14 +1,14 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { item } from '@vaadin/item/theme/lumo/vaadin-item-styles.js';
-import { listBox } from '@vaadin/list-box/theme/lumo/vaadin-list-box-styles.js';
-import { menuOverlayCore } from '@vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
-import { overlay } from '@vaadin/vaadin-lumo-styles/mixins/overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { item } from '@scoped-vaadin/item/theme/lumo/vaadin-item-styles.js';
+import { listBox } from '@scoped-vaadin/list-box/theme/lumo/vaadin-list-box-styles.js';
+import { menuOverlayCore } from '@scoped-vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
+import { overlay } from '@scoped-vaadin/vaadin-lumo-styles/mixins/overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-avatar-group',
+  'vaadin24-avatar-group',
   css`
     :host {
       --vaadin-avatar-size: var(--lumo-size-m);
@@ -52,14 +52,14 @@
   }
 `;
 
-registerStyles('vaadin-avatar-group-overlay', [overlay, menuOverlayCore, avatarGroupOverlay], {
+registerStyles('vaadin24-avatar-group-overlay', [overlay, menuOverlayCore, avatarGroupOverlay], {
   moduleId: 'lumo-avatar-group-overlay',
 });
 
-registerStyles('vaadin-avatar-group-menu', listBox, { moduleId: 'lumo-avatar-group-menu' });
+registerStyles('vaadin24-avatar-group-menu', listBox, { moduleId: 'lumo-avatar-group-menu' });
 
 registerStyles(
-  'vaadin-avatar-group-menu-item',
+  'vaadin24-avatar-group-menu-item',
   [
     item,
     css`
@@ -73,7 +73,7 @@
         align-items: center;
       }
 
-      [part='content'] ::slotted(vaadin-avatar) {
+      [part='content'] ::slotted(vaadin24-avatar) {
         width: var(--lumo-size-xs);
         height: var(--lumo-size-xs);
         margin-inline-end: var(--lumo-space-s);
