Index: test/dom/__snapshots__/context-menu.test.snap.js
===================================================================
--- test/dom/__snapshots__/context-menu.test.snap.js
+++ test/dom/__snapshots__/context-menu.test.snap.js
@@ -2,15 +2,15 @@
 export const snapshots = {};
 
 snapshots["context-menu items"] = 
-`<vaadin-context-menu-overlay
+`<vaadin24-context-menu-overlay
   dir="ltr"
   opened=""
 >
-  <vaadin-context-menu-list-box
+  <vaadin24-context-menu-list-box
     aria-orientation="vertical"
     role="menu"
   >
-    <vaadin-context-menu-item
+    <vaadin24-context-menu-item
       aria-haspopup="false"
       aria-selected="false"
       class="first"
@@ -20,9 +20,9 @@
       tabindex="0"
     >
       Menu Item 1
-    </vaadin-context-menu-item>
+    </vaadin24-context-menu-item>
     <hr role="separator">
-    <vaadin-context-menu-item
+    <vaadin24-context-menu-item
       aria-expanded="false"
       aria-haspopup="true"
       aria-selected="false"
@@ -30,8 +30,8 @@
       tabindex="-1"
     >
       Menu Item 2
-    </vaadin-context-menu-item>
-    <vaadin-context-menu-item
+    </vaadin24-context-menu-item>
+    <vaadin24-context-menu-item
       aria-disabled="true"
       aria-haspopup="false"
       aria-selected="false"
@@ -40,7 +40,7 @@
       tabindex="-1"
     >
       Menu Item 3
-    </vaadin-context-menu-item>
+    </vaadin24-context-menu-item>
     <div
       aria-haspopup="false"
       class="custom"
@@ -53,15 +53,15 @@
     >
       Menu Item 5
     </div>
-  </vaadin-context-menu-list-box>
-  <vaadin-context-menu hidden="">
-  </vaadin-context-menu>
-</vaadin-context-menu-overlay>
+  </vaadin24-context-menu-list-box>
+  <vaadin24-context-menu hidden="">
+  </vaadin24-context-menu>
+</vaadin24-context-menu-overlay>
 `;
 /* end snapshot context-menu items */
 
 snapshots["context-menu items nested"] = 
-`<vaadin-context-menu-overlay
+`<vaadin24-context-menu-overlay
   dir="ltr"
   modeless=""
   opened=""
@@ -69,11 +69,11 @@
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-context-menu-list-box
+  <vaadin24-context-menu-list-box
     aria-orientation="vertical"
     role="menu"
   >
-    <vaadin-context-menu-item
+    <vaadin24-context-menu-item
       aria-haspopup="false"
       aria-selected="false"
       class="first"
@@ -81,8 +81,8 @@
       tabindex="0"
     >
       Menu Item 2-1
-    </vaadin-context-menu-item>
-    <vaadin-context-menu-item
+    </vaadin24-context-menu-item>
+    <vaadin24-context-menu-item
       aria-expanded="true"
       aria-haspopup="true"
       aria-selected="false"
@@ -92,25 +92,25 @@
       tabindex="-1"
     >
       Menu Item 2-2
-    </vaadin-context-menu-item>
-  </vaadin-context-menu-list-box>
-  <vaadin-context-menu hidden="">
-  </vaadin-context-menu>
-</vaadin-context-menu-overlay>
+    </vaadin24-context-menu-item>
+  </vaadin24-context-menu-list-box>
+  <vaadin24-context-menu hidden="">
+  </vaadin24-context-menu>
+</vaadin24-context-menu-overlay>
 `;
 /* end snapshot context-menu items nested */
 
 snapshots["context-menu items overlay class"] = 
-`<vaadin-context-menu-overlay
+`<vaadin24-context-menu-overlay
   class="context-menu-overlay custom"
   dir="ltr"
   opened=""
 >
-  <vaadin-context-menu-list-box
+  <vaadin24-context-menu-list-box
     aria-orientation="vertical"
     role="menu"
   >
-    <vaadin-context-menu-item
+    <vaadin24-context-menu-item
       aria-haspopup="false"
       aria-selected="false"
       class="first"
@@ -120,9 +120,9 @@
       tabindex="0"
     >
       Menu Item 1
-    </vaadin-context-menu-item>
+    </vaadin24-context-menu-item>
     <hr role="separator">
-    <vaadin-context-menu-item
+    <vaadin24-context-menu-item
       aria-expanded="false"
       aria-haspopup="true"
       aria-selected="false"
@@ -130,8 +130,8 @@
       tabindex="-1"
     >
       Menu Item 2
-    </vaadin-context-menu-item>
-    <vaadin-context-menu-item
+    </vaadin24-context-menu-item>
+    <vaadin24-context-menu-item
       aria-disabled="true"
       aria-haspopup="false"
       aria-selected="false"
@@ -140,7 +140,7 @@
       tabindex="-1"
     >
       Menu Item 3
-    </vaadin-context-menu-item>
+    </vaadin24-context-menu-item>
     <div
       aria-haspopup="false"
       class="custom"
@@ -153,15 +153,15 @@
     >
       Menu Item 5
     </div>
-  </vaadin-context-menu-list-box>
-  <vaadin-context-menu hidden="">
-  </vaadin-context-menu>
-</vaadin-context-menu-overlay>
+  </vaadin24-context-menu-list-box>
+  <vaadin24-context-menu hidden="">
+  </vaadin24-context-menu>
+</vaadin24-context-menu-overlay>
 `;
 /* end snapshot context-menu items overlay class */
 
 snapshots["context-menu items overlay class nested"] = 
-`<vaadin-context-menu-overlay
+`<vaadin24-context-menu-overlay
   class="context-menu-overlay custom"
   dir="ltr"
   modeless=""
@@ -170,11 +170,11 @@
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-context-menu-list-box
+  <vaadin24-context-menu-list-box
     aria-orientation="vertical"
     role="menu"
   >
-    <vaadin-context-menu-item
+    <vaadin24-context-menu-item
       aria-haspopup="false"
       aria-selected="false"
       class="first"
@@ -182,8 +182,8 @@
       tabindex="0"
     >
       Menu Item 2-1
-    </vaadin-context-menu-item>
-    <vaadin-context-menu-item
+    </vaadin24-context-menu-item>
+    <vaadin24-context-menu-item
       aria-expanded="true"
       aria-haspopup="true"
       aria-selected="false"
@@ -193,11 +193,11 @@
       tabindex="-1"
     >
       Menu Item 2-2
-    </vaadin-context-menu-item>
-  </vaadin-context-menu-list-box>
-  <vaadin-context-menu hidden="">
-  </vaadin-context-menu>
-</vaadin-context-menu-overlay>
+    </vaadin24-context-menu-item>
+  </vaadin24-context-menu-list-box>
+  <vaadin24-context-menu hidden="">
+  </vaadin24-context-menu>
+</vaadin24-context-menu-overlay>
 `;
 /* end snapshot context-menu items overlay class nested */
 
