Index: theme/lumo/vaadin-confirm-dialog.d.ts
===================================================================
--- theme/lumo/vaadin-confirm-dialog.d.ts
+++ theme/lumo/vaadin-confirm-dialog.d.ts
@@ -1,3 +1,3 @@
-import '@vaadin/button/theme/lumo/vaadin-button.js';
+import '@scoped-vaadin/button/theme/lumo/vaadin-button.js';
 import './vaadin-confirm-dialog-styles.js';
 import '../../src/vaadin-confirm-dialog.js';
