Index: theme/lumo/vaadin-confirm-dialog-styles.js
===================================================================
--- theme/lumo/vaadin-confirm-dialog-styles.js
+++ theme/lumo/vaadin-confirm-dialog-styles.js
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-confirm-dialog-overlay',
+  'vaadin23-confirm-dialog-overlay',
   css`
     [part='header'] ::slotted(h3) {
       margin-top: 0 !important;
