Index: confirm-dialog/theme/lumo/vaadin-confirm-dialog-styles.js
===================================================================
--- confirm-dialog/theme/lumo/vaadin-confirm-dialog-styles.js	null
+++ confirm-dialog/theme/lumo/vaadin-confirm-dialog-styles.js	null
@@ -1,12 +1,12 @@
-import '@vaadin/button/theme/lumo/vaadin-button-styles.js';
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { dialogOverlay } from '@vaadin/dialog/theme/lumo/vaadin-dialog-styles.js';
-import { overlay } from '@vaadin/vaadin-lumo-styles/mixins/overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/button/theme/lumo/vaadin-button-styles.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { dialogOverlay } from '@scoped-vaadin/dialog/theme/lumo/vaadin-dialog-styles.js';
+import { overlay } from '@scoped-vaadin/vaadin-lumo-styles/mixins/overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-confirm-dialog-overlay',
+  'vaadin24-confirm-dialog-overlay',
   [
     overlay,
     dialogOverlay,
