Index: test/visual/lumo/field-highlighter.test.js
===================================================================
--- test/visual/lumo/field-highlighter.test.js
+++ test/visual/lumo/field-highlighter.test.js
@@ -1,14 +1,14 @@
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/checkbox/theme/lumo/vaadin-checkbox.js';
-import '@vaadin/checkbox-group/theme/lumo/vaadin-checkbox-group.js';
-import '@vaadin/date-time-picker/theme/lumo/vaadin-date-time-picker.js';
-import '@vaadin/item/theme/lumo/vaadin-item.js';
-import '@vaadin/list-box/theme/lumo/vaadin-list-box.js';
-import '@vaadin/radio-group/theme/lumo/vaadin-radio-group.js';
-import '@vaadin/text-area/theme/lumo/vaadin-text-area.js';
-import '@vaadin/text-field/theme/lumo/vaadin-text-field.js';
+import '@scoped-vaadin/checkbox/theme/lumo/vaadin-checkbox.js';
+import '@scoped-vaadin/checkbox-group/theme/lumo/vaadin-checkbox-group.js';
+import '@scoped-vaadin/date-time-picker/theme/lumo/vaadin-date-time-picker.js';
+import '@scoped-vaadin/item/theme/lumo/vaadin-item.js';
+import '@scoped-vaadin/list-box/theme/lumo/vaadin-list-box.js';
+import '@scoped-vaadin/radio-group/theme/lumo/vaadin-radio-group.js';
+import '@scoped-vaadin/text-area/theme/lumo/vaadin-text-area.js';
+import '@scoped-vaadin/text-field/theme/lumo/vaadin-text-field.js';
 import '../../../theme/lumo/vaadin-field-highlighter.js';
 import '../common.js';
 import { setUsers } from '../helpers.js';
@@ -25,7 +25,7 @@
 
   describe('checkbox', () => {
     beforeEach(async () => {
-      element = fixtureSync(`<vaadin-checkbox label="I agree with terms"></vaadin-checkbox>`, div);
+      element = fixtureSync(`<vaadin24-checkbox label="I agree with terms"></vaadin24-checkbox>`, div);
       setUsers(element);
       await nextFrame();
     });
@@ -45,12 +45,12 @@
     beforeEach(async () => {
       element = fixtureSync(
         `
-          <vaadin-checkbox-group>
-            <vaadin-checkbox name="user" value="1" label="Option 1"></vaadin-checkbox>
-            <vaadin-checkbox name="user" value="2" label="Option 2"></vaadin-checkbox>
-            <vaadin-checkbox name="user" value="3" label="Option 3"></vaadin-checkbox>
-            <vaadin-checkbox name="user" value="4" label="Option 4"></vaadin-checkbox>
-          </vaadin-checkbox-group>
+          <vaadin24-checkbox-group>
+            <vaadin24-checkbox name="user" value="1" label="Option 1"></vaadin24-checkbox>
+            <vaadin24-checkbox name="user" value="2" label="Option 2"></vaadin24-checkbox>
+            <vaadin24-checkbox name="user" value="3" label="Option 3"></vaadin24-checkbox>
+            <vaadin24-checkbox name="user" value="4" label="Option 4"></vaadin24-checkbox>
+          </vaadin24-checkbox-group>
         `,
         div,
       );
@@ -71,7 +71,7 @@
 
   describe('date-time-picker', () => {
     beforeEach(async () => {
-      element = fixtureSync(`<vaadin-date-time-picker></vaadin-date-time-picker>`, div);
+      element = fixtureSync(`<vaadin24-date-time-picker></vaadin24-date-time-picker>`, div);
       setUsers(element);
       await nextFrame();
     });
@@ -92,12 +92,12 @@
       div.style.height = '250px';
       element = fixtureSync(
         `
-          <vaadin-list-box>
-            <vaadin-item>Option one</vaadin-item>
-            <vaadin-item>Option two</vaadin-item>
-            <vaadin-item>Option three</vaadin-item>
-            <vaadin-item>Option four</vaadin-item>
-          </vaadin-list-box>
+          <vaadin24-list-box>
+            <vaadin24-item>Option one</vaadin24-item>
+            <vaadin24-item>Option two</vaadin24-item>
+            <vaadin24-item>Option three</vaadin24-item>
+            <vaadin24-item>Option four</vaadin24-item>
+          </vaadin24-list-box>
         `,
         div,
       );
@@ -120,7 +120,7 @@
 
   describe('radio-button', () => {
     beforeEach(async () => {
-      element = fixtureSync(`<vaadin-radio-button label="I agree with terms"></vaadin-radio-button>`, div);
+      element = fixtureSync(`<vaadin24-radio-button label="I agree with terms"></vaadin24-radio-button>`, div);
       setUsers(element);
       await nextFrame();
     });
@@ -140,12 +140,12 @@
     beforeEach(async () => {
       element = fixtureSync(
         `
-          <vaadin-radio-group>
-            <vaadin-radio-button name="user" value="1" label="Option 1"></vaadin-radio-button>
-            <vaadin-radio-button name="user" value="2" label="Option 2"></vaadin-radio-button>
-            <vaadin-radio-button name="user" value="3" label="Option 3"></vaadin-radio-button>
-            <vaadin-radio-button name="user" value="4" label="Option 4"></vaadin-radio-button>
-          </vaadin-radio-group>
+          <vaadin24-radio-group>
+            <vaadin24-radio-button name="user" value="1" label="Option 1"></vaadin24-radio-button>
+            <vaadin24-radio-button name="user" value="2" label="Option 2"></vaadin24-radio-button>
+            <vaadin24-radio-button name="user" value="3" label="Option 3"></vaadin24-radio-button>
+            <vaadin24-radio-button name="user" value="4" label="Option 4"></vaadin24-radio-button>
+          </vaadin24-radio-group>
         `,
         div,
       );
@@ -169,10 +169,10 @@
       div.style.height = '250px';
       element = fixtureSync(
         `
-          <vaadin-text-area
+          <vaadin24-text-area
             label="Text area"
             helper-text="Helper"
-          ></vaadin-text-area>
+          ></vaadin24-text-area>
         `,
         div,
       );
@@ -193,7 +193,7 @@
 
   describe('text-field', () => {
     beforeEach(async () => {
-      element = fixtureSync(`<vaadin-text-field></vaadin-text-field>`, div);
+      element = fixtureSync(`<vaadin24-text-field></vaadin24-text-field>`, div);
       setUsers(element);
       await nextFrame();
     });
