Index: test/typings/number-field.types.ts
===================================================================
--- test/typings/number-field.types.ts
+++ test/typings/number-field.types.ts
@@ -1,9 +1,9 @@
 import '../../vaadin-number-field.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { InputFieldMixinClass } from '@vaadin/field-base/src/input-field-mixin.js';
-import type { SlotStylesMixinClass } from '@vaadin/field-base/src/slot-styles-mixin.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { InputFieldMixinClass } from '@scoped-vaadin/field-base/src/input-field-mixin.js';
+import type { SlotStylesMixinClass } from '@scoped-vaadin/field-base/src/slot-styles-mixin.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import type {
   NumberField,
   NumberFieldChangeEvent,
@@ -14,7 +14,7 @@
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const field = document.createElement('vaadin-number-field');
+const field = document.createElement('vaadin23-number-field');
 
 // Mixins
 assertType<ControllerMixinClass>(field);
