Index: test/typings/confirm-dialog.types.ts
===================================================================
--- test/typings/confirm-dialog.types.ts
+++ test/typings/confirm-dialog.types.ts
@@ -1,11 +1,11 @@
 import '../../vaadin-confirm-dialog.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { ThemePropertyMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { ThemePropertyMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 import type { ConfirmDialogClosedEvent, ConfirmDialogOpenedChangedEvent } from '../../vaadin-confirm-dialog.js';
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const dialog = document.createElement('vaadin-confirm-dialog');
+const dialog = document.createElement('vaadin24-confirm-dialog');
 
 // Mixins
 assertType<ElementMixinClass>(dialog);
