Index: test/basic.test.js
===================================================================
--- test/basic.test.js
+++ test/basic.test.js
@@ -4,12 +4,12 @@
 import '../src/vaadin-overlay.js';
 import { createOverlay } from './helpers.js';
 
-describe('vaadin-overlay', () => {
+describe('vaadin24-overlay', () => {
   describe('vaadin-overlay-open event', () => {
     let overlay, spy;
 
     beforeEach(() => {
-      overlay = fixtureSync('<vaadin-overlay></vaadin-overlay>');
+      overlay = fixtureSync('<vaadin24-overlay></vaadin24-overlay>');
       overlay.renderer = (root) => {
         root.textContent = 'overlay content';
       };
@@ -42,7 +42,7 @@
 
     beforeEach(async () => {
       parent = document.createElement('div');
-      overlay = fixtureSync('<vaadin-overlay></vaadin-overlay>', parent);
+      overlay = fixtureSync('<vaadin24-overlay></vaadin24-overlay>', parent);
       overlay.renderer = (root) => {
         root.textContent = 'overlay content';
       };
