Index: test/basic.common.js
===================================================================
--- test/basic.common.js
+++ test/basic.common.js
@@ -3,13 +3,13 @@
 import sinon from 'sinon';
 import { createOverlay } from './helpers.js';
 
-describe('vaadin-overlay', () => {
+describe('vaadin24-overlay', () => {
   describe('moving overlay', () => {
     let parent, overlay;
 
     beforeEach(async () => {
       parent = document.createElement('div');
-      overlay = fixtureSync('<vaadin-overlay></vaadin-overlay>', parent);
+      overlay = fixtureSync('<vaadin24-overlay></vaadin24-overlay>', parent);
       overlay.renderer = (root) => {
         root.textContent = 'overlay content';
       };
