Index: test/focus-trap.common.js
===================================================================
--- test/focus-trap.common.js
+++ test/focus-trap.common.js
@@ -1,6 +1,6 @@
 import { expect } from '@esm-bundle/chai';
 import { aTimeout, fixtureSync, nextRender, oneEvent, tabKeyDown } from '@vaadin/testing-helpers';
-import { getFocusableElements, isElementFocused } from '@vaadin/a11y-base/src/focus-utils.js';
+import { getFocusableElements, isElementFocused } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 
 describe('focus-trap', () => {
   let overlay, overlayPart, focusableElements;
@@ -11,7 +11,7 @@
 
   describe('focusable elements', () => {
     beforeEach(async () => {
-      overlay = fixtureSync('<vaadin-overlay focus-trap></vaadin-overlay>');
+      overlay = fixtureSync('<vaadin24-overlay focus-trap></vaadin24-overlay>');
       overlay.renderer = (root) => {
         if (!root.firstChild) {
           root.innerHTML = `
@@ -79,7 +79,7 @@
 
   describe('empty', () => {
     beforeEach(async () => {
-      overlay = fixtureSync('<vaadin-overlay></vaadin-overlay>');
+      overlay = fixtureSync('<vaadin24-overlay></vaadin24-overlay>');
       await nextRender();
       overlayPart = overlay.$.overlay;
     });
@@ -106,7 +106,7 @@
     let nested;
 
     beforeEach(async () => {
-      overlay = fixtureSync('<vaadin-overlay focus-trap></vaadin-overlay>');
+      overlay = fixtureSync('<vaadin24-overlay focus-trap></vaadin24-overlay>');
       await nextRender();
       overlay.renderer = (root) => {
         if (!root.firstChild) {
@@ -114,7 +114,7 @@
           button.textContent = 'Button';
           root.appendChild(button);
 
-          const nested = document.createElement('vaadin-overlay');
+          const nested = document.createElement('vaadin24-overlay');
           nested.renderer = (root) => {
             root.textContent = 'Inner content';
           };
@@ -125,7 +125,7 @@
       await oneEvent(overlay, 'vaadin-overlay-open');
       focusableElements = getFocusableElements(overlay.$.overlay);
       await nextRender();
-      nested = overlay.querySelector('vaadin-overlay');
+      nested = overlay.querySelector('vaadin24-overlay');
     });
 
     afterEach(() => {
@@ -157,14 +157,14 @@
         `
         <div>
           <button>Foo</button>
-          <vaadin-overlay focus-trap></vaadin-overlay>
+          <vaadin24-overlay focus-trap></vaadin24-overlay>
           <button>Bar</button>
         </div>
       `,
         outer,
       );
 
-      overlay = inner.querySelector('vaadin-overlay');
+      overlay = inner.querySelector('vaadin24-overlay');
       await nextRender();
       overlay.renderer = (root) => {
         root.innerHTML = '<input placeholder="Input">';
