Index: test/interactions.common.js
===================================================================
--- test/interactions.common.js
+++ test/interactions.common.js
@@ -66,7 +66,7 @@
 
     beforeEach(async () => {
       parent = document.createElement('div');
-      overlay = fixtureSync('<vaadin-overlay></vaadin-overlay>', parent);
+      overlay = fixtureSync('<vaadin24-overlay></vaadin24-overlay>', parent);
       overlay.renderer = (root) => {
         if (!root.firstChild) {
           const div = document.createElement('div');
