Index: test/lit.test.js
===================================================================
--- test/lit.test.js
+++ test/lit.test.js
@@ -8,7 +8,7 @@
     let overlay;
 
     beforeEach(() => {
-      overlay = fixtureSync(`<vaadin-overlay></vaadin-overlay>`);
+      overlay = fixtureSync(`<vaadin24-overlay></vaadin24-overlay>`);
       overlay.opened = true;
       overlay.renderer = (root) => {
         render(html`Initial Content`, root);
