Index: test/multiple.common.js
===================================================================
--- test/multiple.common.js
+++ test/multiple.common.js
@@ -194,7 +194,7 @@
 
     const getFrontmostOverlayFromScreenCenter = () => {
       let elementFromPoint = document.elementFromPoint(window.innerWidth / 2, window.innerHeight / 2);
-      while (elementFromPoint && elementFromPoint.localName !== 'vaadin-overlay') {
+      while (elementFromPoint && elementFromPoint.localName !== 'vaadin24-overlay') {
         elementFromPoint = elementFromPoint.host || elementFromPoint.parentNode;
       }
       return elementFromPoint;
@@ -203,8 +203,8 @@
     beforeEach(async () => {
       parent = fixtureSync(`
         <div id="parent">
-          <vaadin-overlay modeless></vaadin-overlay>
-          <vaadin-overlay modeless></vaadin-overlay>
+          <vaadin24-overlay modeless></vaadin24-overlay>
+          <vaadin24-overlay modeless></vaadin24-overlay>
         </div>
       `);
       modeless1 = parent.children[0];
