Index: test/multiple.test.js
===================================================================
--- test/multiple.test.js
+++ test/multiple.test.js
@@ -10,21 +10,21 @@
     beforeEach(() => {
       parent = fixtureSync(`
         <div id="parent">
-          <vaadin-overlay>
+          <vaadin24-overlay>
             <template>
               overlay 1
             </template>
-          </vaadin-overlay>
-          <vaadin-overlay>
+          </vaadin24-overlay>
+          <vaadin24-overlay>
             <template>
               overlay 2
             </template>
-          </vaadin-overlay>
-          <vaadin-overlay>
+          </vaadin24-overlay>
+          <vaadin24-overlay>
             <template>
               overlay 3
             </template>
-          </vaadin-overlay>
+          </vaadin24-overlay>
         </div>
       `);
       overlay1 = parent.children[0];
@@ -153,7 +153,7 @@
 
     const getFrontmostOverlayFromScreenCenter = () => {
       let elementFromPoint = document.elementFromPoint(window.innerWidth / 2, window.innerHeight / 2);
-      while (elementFromPoint && elementFromPoint.localName !== 'vaadin-overlay') {
+      while (elementFromPoint && elementFromPoint.localName !== 'vaadin24-overlay') {
         elementFromPoint = elementFromPoint.host || elementFromPoint.parentNode;
       }
       return elementFromPoint;
@@ -162,18 +162,18 @@
     beforeEach(() => {
       parent = fixtureSync(`
         <div id="parent">
-          <vaadin-overlay modeless>
+          <vaadin24-overlay modeless>
             <template>
               overlay 1
               <input />
             </template>
-          </vaadin-overlay>
-          <vaadin-overlay modeless>
+          </vaadin24-overlay>
+          <vaadin24-overlay modeless>
             <template>
               overlay 2
               <input />
             </template>
-          </vaadin-overlay>
+          </vaadin24-overlay>
         </div>
       `);
       modeless1 = parent.children[0];
