Index: test/position-mixin-polymer.js
===================================================================
--- test/position-mixin-polymer.js
+++ test/position-mixin-polymer.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { Overlay } from '../src/vaadin-overlay.js';
 import { PositionMixin } from '../src/vaadin-overlay-position-mixin.js';
 
@@ -7,4 +8,4 @@
   }
 }
 
-customElements.define(PositionedOverlay.is, PositionedOverlay);
+internalCustomElements.define(PositionedOverlay.is, PositionedOverlay);
