Index: test/position-mixin-listeners.test.js
===================================================================
--- test/position-mixin-listeners.test.js
+++ test/position-mixin-listeners.test.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { aTimeout, fire, fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
@@ -10,7 +11,7 @@
   }
 }
 
-customElements.define(PositionedOverlay.is, PositionedOverlay);
+internalCustomElements.define(PositionedOverlay.is, PositionedOverlay);
 
 class ScrollableWrapper extends HTMLElement {
   constructor() {
@@ -25,7 +26,7 @@
     `;
   }
 }
-customElements.define('scrollable-wrapper', ScrollableWrapper);
+internalCustomElements.define('scrollable-wrapper', ScrollableWrapper);
 
 describe('position mixin listeners', () => {
   let wrapper, target, overlay, updatePositionSpy;
