Index: test/restore-focus.common.js
===================================================================
--- test/restore-focus.common.js
+++ test/restore-focus.common.js
@@ -1,15 +1,16 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { escKeyDown, fixtureSync, mousedown, nextRender, oneEvent } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
+import { getDeepActiveElement } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 
-customElements.define(
+internalCustomElements.define(
   'overlay-field-wrapper',
   class extends PolymerElement {
     static get template() {
       return html`
-        <vaadin-overlay id="overlay" renderer="[[renderer]]" focus-trap></vaadin-overlay>
+        <vaadin24-overlay id="overlay" renderer="[[renderer]]" focus-trap></vaadin24-overlay>
         <input id="focusable" />
       `;
     }
@@ -31,7 +32,7 @@
   },
 );
 
-customElements.define(
+internalCustomElements.define(
   'focus-input-wrapper',
   class extends HTMLElement {
     constructor() {
