Index: test/typings/master-detail-layout.types.ts
===================================================================
--- test/typings/master-detail-layout.types.ts
+++ test/typings/master-detail-layout.types.ts
@@ -1,10 +1,10 @@
 import '../../vaadin-master-detail-layout.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const assertType = <TExpected>(value: TExpected) => value;
 
-const layout = document.createElement('vaadin-master-detail-layout');
+const layout = document.createElement('vaadin24-master-detail-layout');
 
 // Mixins
 assertType<ElementMixinClass>(layout);
