Index: theme/material/vaadin-radio-group-styles.js
===================================================================
--- theme/material/vaadin-radio-group-styles.js
+++ theme/material/vaadin-radio-group-styles.js
@@ -1,7 +1,7 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import { helper } from '@vaadin/vaadin-material-styles/mixins/helper.js';
-import { requiredField } from '@vaadin/vaadin-material-styles/mixins/required-field.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import { helper } from '@scoped-vaadin/vaadin-material-styles/mixins/helper.js';
+import { requiredField } from '@scoped-vaadin/vaadin-material-styles/mixins/required-field.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const radioGroup = css`
   :host {
@@ -40,4 +40,4 @@
   }
 `;
 
-registerStyles('vaadin-radio-group', [requiredField, helper, radioGroup], { moduleId: 'material-radio-group' });
+registerStyles('vaadin24-radio-group', [requiredField, helper, radioGroup], { moduleId: 'material-radio-group' });
