Index: test/dom/__snapshots__/select.test.snap.js
===================================================================
--- test/dom/__snapshots__/select.test.snap.js
+++ test/dom/__snapshots__/select.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-select host default"] = 
-`<vaadin-select>
+`<vaadin24-select>
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -14,25 +14,25 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin24-select-value-button
     aria-expanded="false"
     aria-haspopup="listbox"
     role="button"
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
+  </vaadin24-select-value-button>
   <label
     id="label-vaadin-select-4"
     slot="sr-label"
   >
   </label>
-</vaadin-select>
+</vaadin24-select>
 `;
 /* end snapshot vaadin-select host default */
 
 snapshots["vaadin-select host label"] = 
-`<vaadin-select has-label="">
+`<vaadin24-select has-label="">
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -45,7 +45,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin24-select-value-button
     aria-expanded="false"
     aria-haspopup="listbox"
     aria-labelledby="label-vaadin-select-0"
@@ -53,18 +53,18 @@
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
+  </vaadin24-select-value-button>
   <label
     id="label-vaadin-select-4"
     slot="sr-label"
   >
   </label>
-</vaadin-select>
+</vaadin24-select>
 `;
 /* end snapshot vaadin-select host label */
 
 snapshots["vaadin-select host placeholder"] = 
-`<vaadin-select>
+`<vaadin24-select>
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -76,7 +76,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin24-select-value-button
     aria-expanded="false"
     aria-haspopup="listbox"
     aria-labelledby="value-vaadin-select-3"
@@ -85,21 +85,21 @@
     slot="value"
     tabindex="0"
   >
-    <vaadin-select-item id="value-vaadin-select-3">
+    <vaadin24-select-item id="value-vaadin-select-3">
       Placeholder
-    </vaadin-select-item>
-  </vaadin-select-value-button>
+    </vaadin24-select-item>
+  </vaadin24-select-value-button>
   <label
     id="label-vaadin-select-4"
     slot="sr-label"
   >
   </label>
-</vaadin-select>
+</vaadin24-select>
 `;
 /* end snapshot vaadin-select host placeholder */
 
 snapshots["vaadin-select host disabled"] = 
-`<vaadin-select
+`<vaadin24-select
   aria-disabled="true"
   disabled=""
 >
@@ -114,7 +114,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin24-select-value-button
     aria-disabled="true"
     aria-expanded="false"
     aria-haspopup="listbox"
@@ -123,18 +123,18 @@
     slot="value"
     tabindex="-1"
   >
-  </vaadin-select-value-button>
+  </vaadin24-select-value-button>
   <label
     id="label-vaadin-select-4"
     slot="sr-label"
   >
   </label>
-</vaadin-select>
+</vaadin24-select>
 `;
 /* end snapshot vaadin-select host disabled */
 
 snapshots["vaadin-select host required"] = 
-`<vaadin-select required="">
+`<vaadin24-select required="">
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -146,7 +146,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin24-select-value-button
     aria-expanded="false"
     aria-haspopup="listbox"
     aria-required="true"
@@ -154,18 +154,18 @@
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
+  </vaadin24-select-value-button>
   <label
     id="label-vaadin-select-4"
     slot="sr-label"
   >
   </label>
-</vaadin-select>
+</vaadin24-select>
 `;
 /* end snapshot vaadin-select host required */
 
 snapshots["vaadin-select host value"] = 
-`<vaadin-select has-value="">
+`<vaadin24-select has-value="">
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -177,7 +177,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin24-select-value-button
     aria-expanded="false"
     aria-haspopup="listbox"
     aria-labelledby="value-vaadin-select-3"
@@ -185,24 +185,24 @@
     slot="value"
     tabindex="0"
   >
-    <vaadin-select-item
+    <vaadin24-select-item
       id="value-vaadin-select-3"
       selected=""
     >
       Option 1
-    </vaadin-select-item>
-  </vaadin-select-value-button>
+    </vaadin24-select-item>
+  </vaadin24-select-value-button>
   <label
     id="label-vaadin-select-4"
     slot="sr-label"
   >
   </label>
-</vaadin-select>
+</vaadin24-select>
 `;
 /* end snapshot vaadin-select host value */
 
 snapshots["vaadin-select host helper"] = 
-`<vaadin-select has-helper="">
+`<vaadin24-select has-helper="">
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -214,7 +214,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin24-select-value-button
     aria-describedby="helper-vaadin-select-1"
     aria-expanded="false"
     aria-haspopup="listbox"
@@ -222,7 +222,7 @@
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
+  </vaadin24-select-value-button>
   <label
     id="label-vaadin-select-4"
     slot="sr-label"
@@ -234,12 +234,12 @@
   >
     Helper
   </div>
-</vaadin-select>
+</vaadin24-select>
 `;
 /* end snapshot vaadin-select host helper */
 
 snapshots["vaadin-select host error"] = 
-`<vaadin-select
+`<vaadin24-select
   has-error-message=""
   invalid=""
 >
@@ -255,7 +255,7 @@
   >
     Error
   </div>
-  <vaadin-select-value-button
+  <vaadin24-select-value-button
     aria-describedby="error-message-vaadin-select-2"
     aria-expanded="false"
     aria-haspopup="listbox"
@@ -265,18 +265,18 @@
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
+  </vaadin24-select-value-button>
   <label
     id="label-vaadin-select-4"
     slot="sr-label"
   >
   </label>
-</vaadin-select>
+</vaadin24-select>
 `;
 /* end snapshot vaadin-select host error */
 
 snapshots["vaadin-select host opened default"] = 
-`<vaadin-select opened="">
+`<vaadin24-select opened="">
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -288,35 +288,35 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin24-select-value-button
     aria-expanded="true"
     aria-haspopup="listbox"
     role="button"
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
+  </vaadin24-select-value-button>
   <label
     id="label-vaadin-select-4"
     slot="sr-label"
   >
   </label>
-</vaadin-select>
+</vaadin24-select>
 `;
 /* end snapshot vaadin-select host opened default */
 
 snapshots["vaadin-select host opened overlay"] = 
-`<vaadin-select-overlay
+`<vaadin24-select-overlay
   dir="ltr"
   opened=""
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-select-list-box
+  <vaadin24-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin24-select-item
       aria-selected="false"
       focus-ring=""
       focused=""
@@ -324,32 +324,32 @@
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin24-select-item>
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin24-select-item>
+  </vaadin24-select-list-box>
+</vaadin24-select-overlay>
 `;
 /* end snapshot vaadin-select host opened overlay */
 
 snapshots["vaadin-select host opened overlay class"] = 
-`<vaadin-select-overlay
+`<vaadin24-select-overlay
   class="custom select-overlay"
   dir="ltr"
   opened=""
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-select-list-box
+  <vaadin24-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin24-select-item
       aria-selected="false"
       focus-ring=""
       focused=""
@@ -357,16 +357,16 @@
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin24-select-item>
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin24-select-item>
+  </vaadin24-select-list-box>
+</vaadin24-select-overlay>
 `;
 /* end snapshot vaadin-select host opened overlay class */
 
@@ -381,7 +381,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container part="input-field">
+  <vaadin24-input-container part="input-field">
     <slot
       name="prefix"
       slot="prefix"
@@ -395,7 +395,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -405,27 +405,27 @@
     </slot>
   </div>
 </div>
-<vaadin-select-overlay>
-  <vaadin-select-list-box
+<vaadin24-select-overlay>
+  <vaadin24-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin24-select-item>
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin24-select-item>
+  </vaadin24-select-list-box>
+</vaadin24-select-overlay>
 <slot name="tooltip">
 </slot>
 <div class="sr-only">
@@ -446,7 +446,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     disabled=""
     part="input-field"
   >
@@ -463,7 +463,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -473,27 +473,27 @@
     </slot>
   </div>
 </div>
-<vaadin-select-overlay>
-  <vaadin-select-list-box
+<vaadin24-select-overlay>
+  <vaadin24-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin24-select-item>
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin24-select-item>
+  </vaadin24-select-list-box>
+</vaadin24-select-overlay>
 <slot name="tooltip">
 </slot>
 <div class="sr-only">
@@ -514,7 +514,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     part="input-field"
     readonly=""
   >
@@ -531,7 +531,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -541,27 +541,27 @@
     </slot>
   </div>
 </div>
-<vaadin-select-overlay>
-  <vaadin-select-list-box
+<vaadin24-select-overlay>
+  <vaadin24-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin24-select-item>
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin24-select-item>
+  </vaadin24-select-list-box>
+</vaadin24-select-overlay>
 <slot name="tooltip">
 </slot>
 <div class="sr-only">
@@ -582,7 +582,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     invalid=""
     part="input-field"
   >
@@ -599,7 +599,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -609,27 +609,27 @@
     </slot>
   </div>
 </div>
-<vaadin-select-overlay>
-  <vaadin-select-list-box
+<vaadin24-select-overlay>
+  <vaadin24-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin24-select-item>
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin24-select-item>
+  </vaadin24-select-list-box>
+</vaadin24-select-overlay>
 <slot name="tooltip">
 </slot>
 <div class="sr-only">
@@ -650,7 +650,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin24-input-container
     part="input-field"
     theme="align-right"
   >
@@ -667,7 +667,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin24-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -677,27 +677,27 @@
     </slot>
   </div>
 </div>
-<vaadin-select-overlay theme="align-right">
-  <vaadin-select-list-box
+<vaadin24-select-overlay theme="align-right">
+  <vaadin24-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin24-select-item>
+    <vaadin24-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin24-select-item>
+  </vaadin24-select-list-box>
+</vaadin24-select-overlay>
 <slot name="tooltip">
 </slot>
 <div class="sr-only">
