Index: test/dom/__snapshots__/select.test.snap.js
===================================================================
--- test/dom/__snapshots__/select.test.snap.js
+++ test/dom/__snapshots__/select.test.snap.js
@@ -2,7 +2,7 @@
 export const snapshots = {};
 
 snapshots["vaadin-select host default"] = 
-`<vaadin-select>
+`<vaadin23-select>
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -14,7 +14,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin23-select-value-button
     aria-expanded="false"
     aria-haspopup="listbox"
     aria-labelledby="vaadin-select-3"
@@ -22,13 +22,13 @@
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
-</vaadin-select>
+  </vaadin23-select-value-button>
+</vaadin23-select>
 `;
 /* end snapshot vaadin-select host default */
 
 snapshots["vaadin-select host label"] = 
-`<vaadin-select has-label="">
+`<vaadin23-select has-label="">
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -41,7 +41,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin23-select-value-button
     aria-expanded="false"
     aria-haspopup="listbox"
     aria-labelledby="vaadin-select-3 label-vaadin-select-0"
@@ -49,13 +49,13 @@
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
-</vaadin-select>
+  </vaadin23-select-value-button>
+</vaadin23-select>
 `;
 /* end snapshot vaadin-select host label */
 
 snapshots["vaadin-select host disabled"] = 
-`<vaadin-select
+`<vaadin23-select
   aria-disabled="true"
   disabled=""
 >
@@ -70,7 +70,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin23-select-value-button
     aria-disabled="true"
     aria-expanded="false"
     aria-haspopup="listbox"
@@ -80,13 +80,13 @@
     slot="value"
     tabindex="-1"
   >
-  </vaadin-select-value-button>
-</vaadin-select>
+  </vaadin23-select-value-button>
+</vaadin23-select>
 `;
 /* end snapshot vaadin-select host disabled */
 
 snapshots["vaadin-select host required"] = 
-`<vaadin-select required="">
+`<vaadin23-select required="">
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -98,7 +98,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin23-select-value-button
     aria-expanded="false"
     aria-haspopup="listbox"
     aria-labelledby="vaadin-select-3"
@@ -107,13 +107,13 @@
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
-</vaadin-select>
+  </vaadin23-select-value-button>
+</vaadin23-select>
 `;
 /* end snapshot vaadin-select host required */
 
 snapshots["vaadin-select host value"] = 
-`<vaadin-select has-value="">
+`<vaadin23-select has-value="">
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -125,7 +125,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin23-select-value-button
     aria-expanded="false"
     aria-haspopup="listbox"
     aria-labelledby="vaadin-select-3"
@@ -133,19 +133,19 @@
     slot="value"
     tabindex="0"
   >
-    <vaadin-select-item
+    <vaadin23-select-item
       id="vaadin-select-3"
       selected=""
     >
       Option 1
-    </vaadin-select-item>
-  </vaadin-select-value-button>
-</vaadin-select>
+    </vaadin23-select-item>
+  </vaadin23-select-value-button>
+</vaadin23-select>
 `;
 /* end snapshot vaadin-select host value */
 
 snapshots["vaadin-select host helper"] = 
-`<vaadin-select has-helper="">
+`<vaadin23-select has-helper="">
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -157,7 +157,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin23-select-value-button
     aria-describedby="helper-vaadin-select-1"
     aria-expanded="false"
     aria-haspopup="listbox"
@@ -166,19 +166,19 @@
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
+  </vaadin23-select-value-button>
   <div
     id="helper-vaadin-select-1"
     slot="helper"
   >
     Helper
   </div>
-</vaadin-select>
+</vaadin23-select>
 `;
 /* end snapshot vaadin-select host helper */
 
 snapshots["vaadin-select host error"] = 
-`<vaadin-select
+`<vaadin23-select
   has-error-message=""
   invalid=""
 >
@@ -194,7 +194,7 @@
   >
     Error
   </div>
-  <vaadin-select-value-button
+  <vaadin23-select-value-button
     aria-describedby="error-message-vaadin-select-2"
     aria-expanded="false"
     aria-haspopup="listbox"
@@ -205,13 +205,13 @@
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
-</vaadin-select>
+  </vaadin23-select-value-button>
+</vaadin23-select>
 `;
 /* end snapshot vaadin-select host error */
 
 snapshots["vaadin-select host opened default"] = 
-`<vaadin-select opened="">
+`<vaadin23-select opened="">
   <label
     id="label-vaadin-select-0"
     slot="label"
@@ -223,7 +223,7 @@
     slot="error-message"
   >
   </div>
-  <vaadin-select-value-button
+  <vaadin23-select-value-button
     aria-expanded="true"
     aria-haspopup="listbox"
     aria-labelledby="vaadin-select-3"
@@ -231,23 +231,23 @@
     slot="value"
     tabindex="0"
   >
-  </vaadin-select-value-button>
-</vaadin-select>
+  </vaadin23-select-value-button>
+</vaadin23-select>
 `;
 /* end snapshot vaadin-select host opened default */
 
 snapshots["vaadin-select host opened overlay"] = 
-`<vaadin-select-overlay
+`<vaadin23-select-overlay
   dir="ltr"
   opened=""
   start-aligned=""
   top-aligned=""
 >
-  <vaadin-select-list-box
+  <vaadin23-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin23-select-item
       aria-selected="false"
       focus-ring=""
       focused=""
@@ -255,16 +255,16 @@
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin23-select-item>
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin23-select-item>
+  </vaadin23-select-list-box>
+</vaadin23-select-overlay>
 `;
 /* end snapshot vaadin-select host opened overlay */
 
@@ -279,7 +279,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container part="input-field">
+  <vaadin23-input-container part="input-field">
     <slot
       name="prefix"
       slot="prefix"
@@ -293,7 +293,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin23-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -303,27 +303,27 @@
     </slot>
   </div>
 </div>
-<vaadin-select-overlay>
-  <vaadin-select-list-box
+<vaadin23-select-overlay>
+  <vaadin23-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin23-select-item>
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin23-select-item>
+  </vaadin23-select-list-box>
+</vaadin23-select-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -340,7 +340,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin23-input-container
     disabled=""
     part="input-field"
   >
@@ -357,7 +357,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin23-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -367,27 +367,27 @@
     </slot>
   </div>
 </div>
-<vaadin-select-overlay>
-  <vaadin-select-list-box
+<vaadin23-select-overlay>
+  <vaadin23-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin23-select-item>
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin23-select-item>
+  </vaadin23-select-list-box>
+</vaadin23-select-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -404,7 +404,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin23-input-container
     part="input-field"
     readonly=""
   >
@@ -421,7 +421,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin23-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -431,27 +431,27 @@
     </slot>
   </div>
 </div>
-<vaadin-select-overlay>
-  <vaadin-select-list-box
+<vaadin23-select-overlay>
+  <vaadin23-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin23-select-item>
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin23-select-item>
+  </vaadin23-select-list-box>
+</vaadin23-select-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -468,7 +468,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin23-input-container
     invalid=""
     part="input-field"
   >
@@ -485,7 +485,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin23-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -495,27 +495,27 @@
     </slot>
   </div>
 </div>
-<vaadin-select-overlay>
-  <vaadin-select-list-box
+<vaadin23-select-overlay>
+  <vaadin23-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin23-select-item>
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin23-select-item>
+  </vaadin23-select-list-box>
+</vaadin23-select-overlay>
 <slot name="tooltip">
 </slot>
 `;
@@ -532,7 +532,7 @@
     >
     </span>
   </div>
-  <vaadin-input-container
+  <vaadin23-input-container
     part="input-field"
     theme="align-right"
   >
@@ -549,7 +549,7 @@
       slot="suffix"
     >
     </div>
-  </vaadin-input-container>
+  </vaadin23-input-container>
   <div part="helper-text">
     <slot name="helper">
     </slot>
@@ -559,27 +559,27 @@
     </slot>
   </div>
 </div>
-<vaadin-select-overlay theme="align-right">
-  <vaadin-select-list-box
+<vaadin23-select-overlay theme="align-right">
+  <vaadin23-select-list-box
     aria-orientation="vertical"
     role="listbox"
   >
-    <vaadin-select-item
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="0"
     >
       Option 1
-    </vaadin-select-item>
-    <vaadin-select-item
+    </vaadin23-select-item>
+    <vaadin23-select-item
       aria-selected="false"
       role="option"
       tabindex="-1"
     >
       Option 2
-    </vaadin-select-item>
-  </vaadin-select-list-box>
-</vaadin-select-overlay>
+    </vaadin23-select-item>
+  </vaadin23-select-list-box>
+</vaadin23-select-overlay>
 <slot name="tooltip">
 </slot>
 `;
