Index: test/chips.common.js
===================================================================
--- test/chips.common.js
+++ test/chips.common.js
@@ -5,12 +5,12 @@
 describe('chips', () => {
   let comboBox, inputElement;
 
-  const getChips = (combo) => combo.querySelectorAll('vaadin-multi-select-combo-box-chip');
+  const getChips = (combo) => combo.querySelectorAll('vaadin24-multi-select-combo-box-chip');
 
   const getChipContent = (chip) => chip.shadowRoot.querySelector('[part="label"]').textContent;
 
   beforeEach(async () => {
-    comboBox = fixtureSync(`<vaadin-multi-select-combo-box></vaadin-multi-select-combo-box>`);
+    comboBox = fixtureSync(`<vaadin24-multi-select-combo-box></vaadin24-multi-select-combo-box>`);
     comboBox.items = ['apple', 'banana', 'lemon', 'orange'];
     comboBox.selectedItems = ['orange'];
     await nextRender();
@@ -543,7 +543,7 @@
     it('should adapt overlay width to the input field width while opened', async () => {
       comboBox.opened = true;
 
-      const overlay = document.querySelector('vaadin-multi-select-combo-box-overlay');
+      const overlay = document.querySelector('vaadin24-multi-select-combo-box-overlay');
       const overlayPart = overlay.$.overlay;
       const width = overlayPart.clientWidth;
       expect(width).to.equal(comboBox.clientWidth);
