Index: test/lit-renderer-directives.test.js
===================================================================
--- test/lit-renderer-directives.test.js
+++ test/lit-renderer-directives.test.js
@@ -9,15 +9,15 @@
 async function renderComboBox(container, { items }) {
   render(
     html`
-      <vaadin-multi-select-combo-box
+      <vaadin23-multi-select-combo-box
         .items="${items}"
         ${items ? multiSelectComboBoxRenderer((item) => html`${item}`, items) : null}
-      ></vaadin-multi-select-combo-box>
+      ></vaadin23-multi-select-combo-box>
     `,
     container,
   );
   await nextFrame();
-  return container.querySelector('vaadin-multi-select-combo-box');
+  return container.querySelector('vaadin23-multi-select-combo-box');
 }
 
 describe('lit renderer directives', () => {
@@ -44,14 +44,14 @@
 
       it('should render items with the renderer when the combo-box is opened', () => {
         comboBox.opened = true;
-        const items = document.querySelectorAll('vaadin-multi-select-combo-box-item');
+        const items = document.querySelectorAll('vaadin23-multi-select-combo-box-item');
         expect(items[0].textContent).to.equal('Item');
       });
 
       it('should re-render items when the combo-box is opened and a renderer dependency changes', async () => {
         comboBox.opened = true;
         await renderComboBox(container, { items: ['New Item'] });
-        const items = document.querySelectorAll('vaadin-multi-select-combo-box-item');
+        const items = document.querySelectorAll('vaadin23-multi-select-combo-box-item');
         expect(items[0].textContent).to.equal('New Item');
       });
     });
@@ -63,16 +63,16 @@
         rendererSpy = sinon.spy();
         render(
           html`
-            <vaadin-multi-select-combo-box
+            <vaadin23-multi-select-combo-box
               opened
               .items="${['Item']}"
               ${multiSelectComboBoxRenderer(rendererSpy)}
-            ></vaadin-multi-select-combo-box>
+            ></vaadin23-multi-select-combo-box>
           `,
           container,
         );
         await nextFrame();
-        comboBox = container.querySelector('vaadin-multi-select-combo-box');
+        comboBox = container.querySelector('vaadin23-multi-select-combo-box');
       });
 
       it('should pass the item to the renderer', () => {
