Index: test/validation.common.js
===================================================================
--- test/validation.common.js
+++ test/validation.common.js
@@ -8,7 +8,7 @@
 
   describe('initial', () => {
     beforeEach(() => {
-      comboBox = document.createElement('vaadin-multi-select-combo-box');
+      comboBox = document.createElement('vaadin24-multi-select-combo-box');
       comboBox.items = ['apple', 'banana'];
       validateSpy = sinon.spy(comboBox, 'validate');
     });
@@ -41,7 +41,7 @@
 
   describe('basic', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync(`<vaadin-multi-select-combo-box></vaadin-multi-select-combo-box>`);
+      comboBox = fixtureSync(`<vaadin24-multi-select-combo-box></vaadin24-multi-select-combo-box>`);
       comboBox.items = ['apple', 'banana'];
       await nextRender();
       input = comboBox.inputElement;
@@ -92,7 +92,7 @@
 
   describe('required', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync(`<vaadin-multi-select-combo-box required></vaadin-multi-select-combo-box>`);
+      comboBox = fixtureSync(`<vaadin24-multi-select-combo-box required></vaadin24-multi-select-combo-box>`);
       comboBox.items = ['apple', 'banana'];
       await nextRender();
     });
