Index: test/typings/tabsheet.types.ts
===================================================================
--- test/typings/tabsheet.types.ts
+++ test/typings/tabsheet.types.ts
@@ -1,9 +1,9 @@
 import '../../vaadin-tabsheet.js';
-import type { Tab } from '@vaadin/tabs/src/vaadin-tab.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin';
+import type { Tab } from '@scoped-vaadin/tabs/src/vaadin-tab.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin';
 import type { TabSheetItemsChangedEvent, TabSheetSelectedChangedEvent } from '../../vaadin-tabsheet.js';
 
-const tabsheet = document.createElement('vaadin-tabsheet');
+const tabsheet = document.createElement('vaadin24-tabsheet');
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
