Index: test/iconset-lazy-upgrade.test.js
===================================================================
--- test/iconset-lazy-upgrade.test.js
+++ test/iconset-lazy-upgrade.test.js
@@ -7,7 +7,7 @@
   const ICON = '<path d="M3 4h10l-5 7z"></path>';
 
   before(() => {
-    iconset = document.createElement('vaadin-iconset');
+    iconset = document.createElement('vaadin24-iconset');
     iconset.name = 'vaadin';
     iconset.size = 16;
     iconset.innerHTML = `
@@ -25,7 +25,7 @@
   });
 
   it('should use correct name and size set as attributes before upgrade', async () => {
-    const icon = fixtureSync('<vaadin-icon icon="vaadin:caret-down"></vaadin-icon>');
+    const icon = fixtureSync('<vaadin24-icon icon="vaadin:caret-down"></vaadin24-icon>');
 
     // Import vaadin-icon, which also imports vaadin-iconset internally.
     await import('../src/vaadin-icon.js');
