Index: theme/material/vaadin-popover-styles.js
===================================================================
--- theme/material/vaadin-popover-styles.js
+++ theme/material/vaadin-popover-styles.js
@@ -1,6 +1,6 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import { overlay } from '@vaadin/vaadin-material-styles/mixins/overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import { overlay } from '@scoped-vaadin/vaadin-material-styles/mixins/overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const popoverOverlay = css`
   :host {
@@ -105,4 +105,4 @@
   }
 `;
 
-registerStyles('vaadin-popover-overlay', [overlay, popoverOverlay], { moduleId: 'material-popover-overlay' });
+registerStyles('vaadin24-popover-overlay', [overlay, popoverOverlay], { moduleId: 'material-popover-overlay' });
