Index: test/dom/drawer-toggle.test.js
===================================================================
--- test/dom/drawer-toggle.test.js
+++ test/dom/drawer-toggle.test.js
@@ -2,11 +2,11 @@
 import { fixtureSync } from '@vaadin/testing-helpers';
 import '../../src/vaadin-drawer-toggle.js';
 
-describe('vaadin-app-layout', () => {
+describe('vaadin24-app-layout', () => {
   let toggle;
 
   beforeEach(() => {
-    toggle = fixtureSync('<vaadin-drawer-toggle></vaadin-drawer-toggle>');
+    toggle = fixtureSync('<vaadin24-drawer-toggle></vaadin24-drawer-toggle>');
   });
 
   it('default', async () => {
