Index: test/active-mixin.test.js
===================================================================
--- test/active-mixin.test.js
+++ test/active-mixin.test.js
@@ -1,10 +1,11 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, isIOS, mousedown, mouseup, touchend, touchstart } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
 import { ActiveMixin } from '../src/active-mixin.js';
 
-customElements.define(
+internalCustomElements.define(
   'active-mixin-element',
   class extends ActiveMixin(PolymerElement) {
     static get template() {
