Index: test/active-mixin.test.js
===================================================================
--- test/active-mixin.test.js
+++ test/active-mixin.test.js
@@ -9,8 +9,8 @@
   touchend,
   touchstart,
 } from '@vaadin/testing-helpers';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
 import { ActiveMixin } from '../src/active-mixin.js';
 
 const runTests = (defineHelper, baseMixin) => {
