Index: test/delegate-focus-mixin.test.js
===================================================================
--- test/delegate-focus-mixin.test.js
+++ test/delegate-focus-mixin.test.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, focusin, focusout, nextFrame } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
@@ -7,7 +8,7 @@
 describe('delegate-focus-mixin', () => {
   let element, input, setFocusedSpy;
 
-  customElements.define(
+  internalCustomElements.define(
     'delegate-focus-mixin-element',
     class extends DelegateFocusMixin(PolymerElement) {
       static get template() {
