Index: test/field-aria-controller.test.js
===================================================================
--- test/field-aria-controller.test.js
+++ test/field-aria-controller.test.js
@@ -1,7 +1,7 @@
 import { expect } from '@vaadin/chai-plugins';
 import { defineLit, definePolymer, fixtureSync, nextRender } from '@vaadin/testing-helpers';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
 import { FieldAriaController } from '../src/field-aria-controller.js';
 
 const runTests = (defineHelper, baseMixin) => {
