Index: test/focus-trap-controller.test.js
===================================================================
--- test/focus-trap-controller.test.js
+++ test/focus-trap-controller.test.js
@@ -2,8 +2,8 @@
 import { sendKeys } from '@vaadin/test-runner-commands';
 import { defineLit, definePolymer, fixtureSync } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
 import { FocusTrapController } from '../src/focus-trap-controller.js';
 
 async function tab() {
