Index: test/keyboard-direction-mixin.test.js
===================================================================
--- test/keyboard-direction-mixin.test.js
+++ test/keyboard-direction-mixin.test.js
@@ -12,8 +12,8 @@
   tabKeyDown,
 } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
 import { KeyboardDirectionMixin } from '../src/keyboard-direction-mixin.js';
 
 const runTests = (defineHelper, baseMixin) => {
