Index: test/keyboard-mixin.test.js
===================================================================
--- test/keyboard-mixin.test.js
+++ test/keyboard-mixin.test.js
@@ -2,8 +2,8 @@
 import { sendKeys } from '@vaadin/test-runner-commands';
 import { defineLit, definePolymer, fixtureSync } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
 import { KeyboardMixin } from '../src/keyboard-mixin.js';
 
 const runTests = (defineHelper, baseMixin) => {
