Index: test/typings/list-box.types.ts
===================================================================
--- test/typings/list-box.types.ts
+++ test/typings/list-box.types.ts
@@ -1,5 +1,5 @@
 import '../../vaadin-list-box.js';
-import type { ListMixinClass } from '@vaadin/a11y-base/src/list-mixin.js';
+import type { ListMixinClass } from '@scoped-vaadin/a11y-base/src/list-mixin.js';
 import type { MultiSelectListMixinClass } from '../../src/vaadin-multi-select-list-mixin.js';
 import type {
   ListBoxItemsChangedEvent,
@@ -9,7 +9,7 @@
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const listBox = document.createElement('vaadin-list-box');
+const listBox = document.createElement('vaadin24-list-box');
 
 listBox.addEventListener('items-changed', (event) => {
   assertType<ListBoxItemsChangedEvent>(event);
