Index: test/typings/list-box.types.ts
===================================================================
--- test/typings/list-box.types.ts
+++ test/typings/list-box.types.ts
@@ -9,7 +9,7 @@
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const listBox = document.createElement('vaadin-list-box');
+const listBox = document.createElement('vaadin24-list-box');
 
 listBox.addEventListener('items-changed', (event) => {
   assertType<ListBoxItemsChangedEvent>(event);
