Index: test/dom/login-overlay.test.js
===================================================================
--- test/dom/login-overlay.test.js
+++ test/dom/login-overlay.test.js
@@ -1,9 +1,9 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import '../../vaadin-login-overlay.js';
-import { resetUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
+import { resetUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
 
-describe('vaadin-login-overlay', () => {
+describe('vaadin23-login-overlay', () => {
   let overlay, wrapper;
 
   const I18N_FINNISH = {
@@ -27,9 +27,9 @@
 
   beforeEach(async () => {
     resetUniqueId();
-    overlay = fixtureSync('<vaadin-login-overlay opened></vaadin-login-overlay>');
+    overlay = fixtureSync('<vaadin23-login-overlay opened></vaadin23-login-overlay>');
     await nextFrame();
-    wrapper = document.querySelector('vaadin-login-overlay-wrapper');
+    wrapper = document.querySelector('vaadin23-login-overlay-wrapper');
   });
 
   describe('host', () => {
