Index: src/vaadin-avatar-mixin.js
===================================================================
--- src/vaadin-avatar-mixin.js
+++ src/vaadin-avatar-mixin.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2017 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { I18nMixin } from '@vaadin/component-base/src/i18n-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { I18nMixin } from '@scoped-vaadin/component-base/src/i18n-mixin.js';
 
 const DEFAULT_I18N = {
   anonymous: 'anonymous',
@@ -200,7 +200,7 @@
     __withTooltipChanged(withTooltip, oldWithTooltip) {
       if (withTooltip) {
         // Create and attach tooltip
-        const tooltipNode = document.createElement('vaadin-tooltip');
+        const tooltipNode = document.createElement('vaadin24-tooltip');
         tooltipNode.setAttribute('slot', 'tooltip');
         this.appendChild(tooltipNode);
         this.__tooltipNode = tooltipNode;
@@ -241,7 +241,7 @@
     /** @protected */
     __onImageLoadError() {
       if (this.img) {
-        console.warn(`<vaadin-avatar> The specified image could not be loaded: ${this.img}`);
+        console.warn(`<vaadin24-avatar> The specified image could not be loaded: ${this.img}`);
         this.__imgFailedToLoad = true;
         this.__updateVisibility();
       }
