Index: test/visual/material/select.test.js
===================================================================
--- test/visual/material/select.test.js
+++ test/visual/material/select.test.js
@@ -1,8 +1,8 @@
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/item/theme/material/vaadin-item.js';
-import '@vaadin/list-box/theme/material/vaadin-list-box.js';
+import '@scoped-vaadin/item/theme/material/vaadin-item.js';
+import '@scoped-vaadin/list-box/theme/material/vaadin-list-box.js';
 import '../../not-animated-styles.js';
 import '../../../theme/material/vaadin-select.js';
 
@@ -13,7 +13,7 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-select></vaadin-select>', div);
+    element = fixtureSync('<vaadin24-select></vaadin24-select>', div);
     element.items = [
       { label: 'item 1', value: 'value-1' },
       { label: 'item 2', value: 'value-2' },
@@ -79,9 +79,9 @@
   it('value-overflow', async () => {
     element.renderer = (root) => {
       root.innerHTML = `
-        <vaadin-list-box>
-          <vaadin-item value="long">Very long item text content</vaadin-item>
-        </vaadin-list-box>
+        <vaadin24-list-box>
+          <vaadin24-item value="long">Very long item text content</vaadin24-item>
+        </vaadin24-list-box>
       `;
     };
     element.value = 'long';
@@ -145,9 +145,9 @@
     element.placeholder = 'Placeholder';
     element.renderer = (root) => {
       root.innerHTML = `
-        <vaadin-list-box>
-          <vaadin-item value="">Empty</vaadin-item>
-        </vaadin-list-box>
+        <vaadin24-list-box>
+          <vaadin24-item value="">Empty</vaadin24-item>
+        </vaadin24-list-box>
         `;
     };
     await visualDiff(div, 'empty-value');
