Index: test/visual/material/select.test.js
===================================================================
--- test/visual/material/select.test.js
+++ test/visual/material/select.test.js
@@ -1,8 +1,8 @@
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/item/theme/material/vaadin-item.js';
-import '@vaadin/list-box/theme/material/vaadin-list-box.js';
+import '@scoped-vaadin/item/theme/material/vaadin-item.js';
+import '@scoped-vaadin/list-box/theme/material/vaadin-list-box.js';
 import '../../not-animated-styles.js';
 import '../../../theme/material/vaadin-select.js';
 
@@ -13,14 +13,14 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-select></vaadin-select>', div);
+    element = fixtureSync('<vaadin23-select></vaadin23-select>', div);
     element.renderer = (root) => {
       root.innerHTML = `
-        <vaadin-list-box>
-          <vaadin-item>item 1</vaadin-item>
-          <vaadin-item>item 2</vaadin-item>
-          <vaadin-item>item 3</vaadin-item>
-        </vaadin-list-box>
+        <vaadin23-list-box>
+          <vaadin23-item>item 1</vaadin23-item>
+          <vaadin23-item>item 2</vaadin23-item>
+          <vaadin23-item>item 3</vaadin23-item>
+        </vaadin23-list-box>
       `;
     };
     await nextFrame();
@@ -83,9 +83,9 @@
   it('value-overflow', async () => {
     element.renderer = (root) => {
       root.innerHTML = `
-        <vaadin-list-box>
-          <vaadin-item value="long">Very long item text content</vaadin-item>
-        </vaadin-list-box>
+        <vaadin23-list-box>
+          <vaadin23-item value="long">Very long item text content</vaadin23-item>
+        </vaadin23-list-box>
       `;
     };
     element.value = 'long';
@@ -149,9 +149,9 @@
     element.placeholder = 'Placeholder';
     element.renderer = (root) => {
       root.innerHTML = `
-        <vaadin-list-box>
-          <vaadin-item value="">Empty</vaadin-item>
-        </vaadin-list-box>
+        <vaadin23-list-box>
+          <vaadin23-item value="">Empty</vaadin23-item>
+        </vaadin23-list-box>
         `;
     };
     await visualDiff(div, 'empty-value');
