Index: test/visual/lumo/accordion-panel.test.js
===================================================================
--- test/visual/lumo/accordion-panel.test.js
+++ test/visual/lumo/accordion-panel.test.js
@@ -11,9 +11,9 @@
 
     element = fixtureSync(
       `
-        <vaadin-accordion-panel summary="Panel">
+        <vaadin24-accordion-panel summary="Panel">
           <div>Content</div>
-        </vaadin-accordion-panel>
+        </vaadin24-accordion-panel>
       `,
       div,
     );
