Index: theme/material/vaadin-horizontal-layout-styles.js
===================================================================
--- theme/material/vaadin-horizontal-layout-styles.js
+++ theme/material/vaadin-horizontal-layout-styles.js
@@ -1,4 +1,4 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const horizontalLayout = css`
   [theme~='margin'] {
@@ -30,4 +30,4 @@
   }
 `;
 
-registerStyles('vaadin-horizontal-layout', horizontalLayout, { moduleId: 'material-horizontal-layout' });
+registerStyles('vaadin23-horizontal-layout', horizontalLayout, { moduleId: 'material-horizontal-layout' });
