Index: theme/lumo/vaadin-tabs-styles.js
===================================================================
--- theme/lumo/vaadin-tabs-styles.js
+++ theme/lumo/vaadin-tabs-styles.js
@@ -1,13 +1,13 @@
-import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-tabs',
+  'vaadin24-tabs',
   css`
     :host {
       -webkit-tap-highlight-color: transparent;
@@ -19,7 +19,7 @@
       min-height: var(--lumo-size-l);
     }
 
-    :host([orientation='horizontal']) [part='tabs'] ::slotted(vaadin-tab:not([theme~='icon-on-top'])) {
+    :host([orientation='horizontal']) [part='tabs'] ::slotted(vaadin24-tab:not([theme~='icon-on-top'])) {
       justify-content: center;
     }
 
@@ -123,11 +123,11 @@
 
     /* Centered */
 
-    :host([theme~='centered'][orientation='horizontal']) ::slotted(vaadin-tab:first-of-type) {
+    :host([theme~='centered'][orientation='horizontal']) ::slotted(vaadin24-tab:first-of-type) {
       margin-inline-start: auto;
     }
 
-    :host([theme~='centered'][orientation='horizontal']) ::slotted(vaadin-tab:last-of-type) {
+    :host([theme~='centered'][orientation='horizontal']) ::slotted(vaadin24-tab:last-of-type) {
       margin-inline-end: auto;
     }
 
@@ -138,7 +138,7 @@
       min-height: var(--lumo-size-m);
     }
 
-    :host([theme~='small']) [part='tabs'] ::slotted(vaadin-tab) {
+    :host([theme~='small']) [part='tabs'] ::slotted(vaadin24-tab) {
       font-size: var(--lumo-font-size-s);
     }
 
@@ -180,7 +180,7 @@
       flex: auto;
     }
 
-    :host([theme~='equal-width-tabs']) [part='tabs'] ::slotted(vaadin-tab) {
+    :host([theme~='equal-width-tabs']) [part='tabs'] ::slotted(vaadin24-tab) {
       flex: 1 0 0%;
     }
 
