Index: test/visual/material/menu-bar.test.js
===================================================================
--- test/visual/material/menu-bar.test.js
+++ test/visual/material/menu-bar.test.js
@@ -1,7 +1,7 @@
 import { arrowDown, fixtureSync, nextRender, oneEvent } from '@vaadin/testing-helpers';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/icon/theme/material/vaadin-icon.js';
-import '@vaadin/vaadin-lumo-styles/vaadin-iconset.js';
+import '@scoped-vaadin/icon/theme/material/vaadin-icon.js';
+import '@scoped-vaadin/vaadin-lumo-styles/vaadin-iconset.js';
 import '../../../theme/material/vaadin-menu-bar.js';
 import '../../not-animated-styles.js';
 
@@ -23,7 +23,7 @@
           div = document.createElement('div');
           div.style.padding = '10px';
 
-          element = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>', div);
+          element = fixtureSync('<vaadin23-menu-bar></vaadin23-menu-bar>', div);
           element.items = [
             { text: 'Home' },
             {
@@ -62,7 +62,7 @@
           div = document.createElement('div');
           div.style.padding = '10px';
 
-          element = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>', div);
+          element = fixtureSync('<vaadin23-menu-bar></vaadin23-menu-bar>', div);
           element.items = [{ text: 'Actions' }];
           element.setAttribute('theme', 'outlined');
         });
@@ -80,8 +80,8 @@
 
       describe('theme', () => {
         function makeIcon(img) {
-          const item = document.createElement('vaadin-context-menu-item');
-          const icon = document.createElement('vaadin-icon');
+          const item = document.createElement('vaadin23-context-menu-item');
+          const icon = document.createElement('vaadin23-icon');
           icon.setAttribute('icon', img);
           item.appendChild(icon);
           item.setAttribute('theme', 'menu-bar-item');
@@ -92,7 +92,7 @@
           div = document.createElement('div');
           div.style.padding = '10px';
 
-          element = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>', div);
+          element = fixtureSync('<vaadin23-menu-bar></vaadin23-menu-bar>', div);
           element.items = [
             { component: 'u', text: 'Home' },
             {
