Index: test/typings/virtual-list.types.ts
===================================================================
--- test/typings/virtual-list.types.ts
+++ test/typings/virtual-list.types.ts
@@ -1,11 +1,11 @@
 import '../../vaadin-virtual-list.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin';
 import type { VirtualList, VirtualListItemModel, VirtualListRenderer } from '../../vaadin-virtual-list.js';
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const genericVirtualList = document.createElement('vaadin-virtual-list');
+const genericVirtualList = document.createElement('vaadin24-virtual-list');
 
 assertType<VirtualList>(genericVirtualList);
 
