Index: test/visual/material/button.test.js
===================================================================
--- test/visual/material/button.test.js
+++ test/visual/material/button.test.js
@@ -1,8 +1,8 @@
 import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
 import { sendKeys } from '@web/test-runner-commands';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/icon/theme/material/vaadin-icon.js';
-import '@vaadin/vaadin-lumo-styles/vaadin-iconset.js';
+import '@scoped-vaadin/icon/theme/material/vaadin-icon.js';
+import '@scoped-vaadin/vaadin-lumo-styles/vaadin-iconset.js';
 import '../../../theme/material/vaadin-button.js';
 import { hover, resetHover } from '../helpers.js';
 
@@ -13,7 +13,7 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-button>Button</vaadin-button>', div);
+    element = fixtureSync('<vaadin23-button>Button</vaadin23-button>', div);
   });
 
   afterEach(async () => {
@@ -66,7 +66,7 @@
     let icon;
 
     beforeEach(() => {
-      icon = document.createElement('vaadin-icon');
+      icon = document.createElement('vaadin23-icon');
       icon.setAttribute('icon', 'lumo:plus');
     });
 
