Index: src/vaadin-checkbox.d.ts
===================================================================
--- src/vaadin-checkbox.d.ts
+++ src/vaadin-checkbox.d.ts
@@ -3,13 +3,13 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ActiveMixin } from '@vaadin/component-base/src/active-mixin.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { CheckedMixin } from '@vaadin/field-base/src/checked-mixin.js';
-import { DelegateFocusMixin } from '@vaadin/field-base/src/delegate-focus-mixin.js';
-import { LabelMixin } from '@vaadin/field-base/src/label-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ActiveMixin } from '@scoped-vaadin/component-base/src/active-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { CheckedMixin } from '@scoped-vaadin/field-base/src/checked-mixin.js';
+import { DelegateFocusMixin } from '@scoped-vaadin/field-base/src/delegate-focus-mixin.js';
+import { LabelMixin } from '@scoped-vaadin/field-base/src/label-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
  * Fired when the `checked` property changes.
@@ -30,10 +30,10 @@
 export interface CheckboxEventMap extends HTMLElementEventMap, CheckboxCustomEventMap {}
 
 /**
- * `<vaadin-checkbox>` is an input field representing a binary choice.
+ * `<vaadin23-checkbox>` is an input field representing a binary choice.
  *
  * ```html
- * <vaadin-checkbox label="I accept the terms and conditions"></vaadin-checkbox>
+ * <vaadin23-checkbox label="I accept the terms and conditions"></vaadin23-checkbox>
  * ```
  *
  * ### Styling
@@ -93,7 +93,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-checkbox': Checkbox;
+    'vaadin23-checkbox': Checkbox;
   }
 }
 
