Index: checkbox/src/vaadin-checkbox-mixin.d.ts
===================================================================
--- checkbox/src/vaadin-checkbox-mixin.d.ts	null
+++ checkbox/src/vaadin-checkbox-mixin.d.ts	null
@@ -4,16 +4,16 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { ActiveMixinClass } from '@vaadin/a11y-base/src/active-mixin.js';
-import type { DelegateFocusMixinClass } from '@vaadin/a11y-base/src/delegate-focus-mixin.js';
-import type { DisabledMixinClass } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin.js';
-import type { KeyboardMixinClass } from '@vaadin/a11y-base/src/keyboard-mixin.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { DelegateStateMixinClass } from '@vaadin/component-base/src/delegate-state-mixin.js';
-import type { CheckedMixinClass } from '@vaadin/field-base/src/checked-mixin.js';
-import type { InputMixinClass } from '@vaadin/field-base/src/input-mixin.js';
-import type { LabelMixinClass } from '@vaadin/field-base/src/label-mixin.js';
+import type { ActiveMixinClass } from '@scoped-vaadin/a11y-base/src/active-mixin.js';
+import type { DelegateFocusMixinClass } from '@scoped-vaadin/a11y-base/src/delegate-focus-mixin.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import type { KeyboardMixinClass } from '@scoped-vaadin/a11y-base/src/keyboard-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { DelegateStateMixinClass } from '@scoped-vaadin/component-base/src/delegate-state-mixin.js';
+import type { CheckedMixinClass } from '@scoped-vaadin/field-base/src/checked-mixin.js';
+import type { InputMixinClass } from '@scoped-vaadin/field-base/src/input-mixin.js';
+import type { LabelMixinClass } from '@scoped-vaadin/field-base/src/label-mixin.js';
 
 /**
  * A mixin providing common checkbox functionality.
