Index: src/vaadin-upload-file.d.ts
===================================================================
--- src/vaadin-upload-file.d.ts
+++ src/vaadin-upload-file.d.ts
@@ -3,8 +3,8 @@
  * Copyright (c) 2016 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { UploadFileMixin } from './vaadin-upload-file-mixin.js';
 
 export type { UploadFileI18n } from './vaadin-upload-file-mixin.js';
@@ -35,7 +35,7 @@
 export interface UploadFileEventMap extends HTMLElementEventMap, UploadFileCustomEventMap {}
 
 /**
- * `<vaadin-upload-file>` element represents a file in the file list of `<vaadin-upload>`.
+ * `<vaadin24-upload-file>` element represents a file in the file list of `<vaadin24-upload>`.
  *
  * ### Styling
  *
@@ -86,7 +86,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-upload-file': UploadFile;
+    'vaadin24-upload-file': UploadFile;
   }
 }
 
