Index: src/vaadin-lit-upload-file-list.js
===================================================================
--- src/vaadin-lit-upload-file-list.js
+++ src/vaadin-lit-upload-file-list.js
@@ -5,13 +5,13 @@
  */
 import './vaadin-lit-upload-file.js';
 import { css, html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { UploadFileListMixin } from './vaadin-upload-file-list-mixin.js';
 
 /**
- * An element used internally by `<vaadin-upload>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-upload>`. Not intended to be used separately.
  *
  * @extends HTMLElement
  * @mixes ThemableMixin
@@ -20,7 +20,7 @@
  */
 class UploadFileList extends UploadFileListMixin(ThemableMixin(PolylitMixin(LitElement))) {
   static get is() {
-    return 'vaadin-upload-file-list';
+    return 'vaadin24-upload-file-list';
   }
 
   static get styles() {
