Index: src/vaadin-upload-file-list-mixin.js
===================================================================
--- src/vaadin-upload-file-list-mixin.js
+++ src/vaadin-upload-file-list-mixin.js
@@ -49,7 +49,7 @@
     }
 
     /**
-     * Requests an update for the `vaadin-upload-file` elements.
+     * Requests an update for the `vaadin24-upload-file` elements.
      *
      * It is not guaranteed that the update happens immediately (synchronously) after it is requested.
      */
@@ -61,7 +61,7 @@
           ${items.map(
             (file) => html`
               <li>
-                <vaadin-upload-file
+                <vaadin24-upload-file
                   .disabled="${disabled}"
                   .file="${file}"
                   .complete="${file.complete}"
@@ -73,7 +73,7 @@
                   .status="${file.status}"
                   .uploading="${file.uploading}"
                   .i18n="${i18n}"
-                ></vaadin-upload-file>
+                ></vaadin24-upload-file>
               </li>
             `,
           )}
