Index: upload/src/vaadin-upload-file-mixin.js
===================================================================
--- upload/src/vaadin-upload-file-mixin.js
+++ upload/src/vaadin-upload-file-mixin.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { SlotController } from '@scoped-vaadin/component-base/src/slot-controller.js';
 
 /**
  * @polymerMixin
@@ -119,7 +119,7 @@
       super.ready();
 
       this.addController(
-        new SlotController(this, 'progress', 'vaadin-progress-bar', {
+        new SlotController(this, 'progress', 'vaadin24-progress-bar', {
           initializer: (progress) => {
             this._progress = progress;
           },
