Index: upload/src/vaadin-upload.js
===================================================================
--- upload/src/vaadin-upload.js
+++ upload/src/vaadin-upload.js
@@ -3,24 +3,24 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/button/src/vaadin-button.js';
+import '@scoped-vaadin/button/src/vaadin-button.js';
 import './vaadin-upload-icon.js';
 import './vaadin-upload-icons.js';
 import './vaadin-upload-file-list.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { UploadMixin } from './vaadin-upload-mixin.js';
 
 /**
- * `<vaadin-upload>` is a Web Component for uploading multiple files with drag and drop support.
+ * `<vaadin24-upload>` is a Web Component for uploading multiple files with drag and drop support.
  *
  * Example:
  *
  * ```
- * <vaadin-upload></vaadin-upload>
+ * <vaadin24-upload></vaadin24-upload>
  * ```
  *
  * ### Styling
@@ -104,7 +104,7 @@
   }
 
   static get is() {
-    return 'vaadin-upload';
+    return 'vaadin24-upload';
   }
 
   /**
@@ -131,7 +131,7 @@
   /**
    * Fired when the XHR has been opened but not sent yet. Useful for appending
    * data keys to the FormData object, for changing some parameters like
-   * headers, etc. If the event is defaultPrevented, `vaadin-upload` will not
+   * headers, etc. If the event is defaultPrevented, `vaadin24-upload` will not
    * send the request allowing the user to do something on his own.
    *
    * @event upload-request
@@ -165,7 +165,7 @@
    * on the server response. If the event is defaultPrevented the vaadin-upload
    * will return allowing the user to do something on his own like retry the
    * upload, etc. since he has full access to the `xhr` and `file` objects.
-   * Otherwise, if the event is not prevented default `vaadin-upload` continues
+   * Otherwise, if the event is not prevented default `vaadin24-upload` continues
    * with the normal workflow checking the `xhr.status` and `file.error`
    * which also might be modified by the user to force a customized response.
    *
