Index: src/vaadin-lit-upload-file.js
===================================================================
--- src/vaadin-lit-upload-file.js
+++ src/vaadin-lit-upload-file.js
@@ -3,17 +3,17 @@
  * Copyright (c) 2016 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/progress-bar/src/vaadin-progress-bar.js';
+import '@scoped-vaadin/progress-bar/src/vaadin-progress-bar.js';
 import './vaadin-upload-icons.js';
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { UploadFileMixin } from './vaadin-upload-file-mixin.js';
 import { uploadFileStyles } from './vaadin-upload-file-styles.js';
 
 /**
- * LitElement based version of `<vaadin-upload-file>` web component.
+ * LitElement based version of `<vaadin24-upload-file>` web component.
  *
  * ## Disclaimer
  *
@@ -23,7 +23,7 @@
  */
 class UploadFile extends UploadFileMixin(ThemableMixin(PolylitMixin(LitElement))) {
   static get is() {
-    return 'vaadin-upload-file';
+    return 'vaadin24-upload-file';
   }
 
   static get styles() {
