Index: upload/src/vaadin-upload-file.js
===================================================================
--- upload/src/vaadin-upload-file.js	null
+++ upload/src/vaadin-upload-file.js	null
@@ -3,19 +3,19 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/progress-bar/src/vaadin-progress-bar.js';
+import '@scoped-vaadin/progress-bar/src/vaadin-progress-bar.js';
 import './vaadin-upload-icons.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { UploadFileMixin } from './vaadin-upload-file-mixin.js';
 import { uploadFileStyles } from './vaadin-upload-file-styles.js';
 
-registerStyles('vaadin-upload-file', uploadFileStyles, { moduleId: 'vaadin-upload-file-styles' });
+registerStyles('vaadin24-upload-file', uploadFileStyles, { moduleId: 'vaadin-upload-file-styles' });
 
 /**
- * `<vaadin-upload-file>` element represents a file in the file list of `<vaadin-upload>`.
+ * `<vaadin24-upload-file>` element represents a file in the file list of `<vaadin24-upload>`.
  *
  * ### Styling
  *
@@ -104,11 +104,11 @@
   }
 
   static get is() {
-    return 'vaadin-upload-file';
+    return 'vaadin24-upload-file';
   }
 
   /**
-   * Fired when the retry button is pressed. It is listened by `vaadin-upload`
+   * Fired when the retry button is pressed. It is listened by `vaadin24-upload`
    * which will start a new upload process of this file.
    *
    * @event file-retry
@@ -117,7 +117,7 @@
    */
 
   /**
-   * Fired when the start button is pressed. It is listened by `vaadin-upload`
+   * Fired when the start button is pressed. It is listened by `vaadin24-upload`
    * which will start a new upload process of this file.
    *
    * @event file-start
@@ -126,7 +126,7 @@
    */
 
   /**
-   * Fired when abort button is pressed. It is listened by `vaadin-upload` which
+   * Fired when abort button is pressed. It is listened by `vaadin24-upload` which
    * will abort the upload in progress, and then remove the file from the list.
    *
    * @event file-abort
