Index: src/vaadin-lit-accordion.js
===================================================================
--- src/vaadin-lit-accordion.js
+++ src/vaadin-lit-accordion.js
@@ -5,14 +5,14 @@
  */
 import './vaadin-lit-accordion-panel.js';
 import { css, html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { AccordionMixin } from './vaadin-accordion-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-accordion>` web component.
+ * LitElement based version of `<vaadin24-accordion>` web component.
  *
  * ## Disclaimer
  *
@@ -22,7 +22,7 @@
  */
 class Accordion extends AccordionMixin(ThemableMixin(ElementMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-accordion';
+    return 'vaadin24-accordion';
   }
 
   static get styles() {
