Index: src/vaadin-accordion-panel-mixin.d.ts
===================================================================
--- src/vaadin-accordion-panel-mixin.d.ts
+++ src/vaadin-accordion-panel-mixin.d.ts
@@ -4,12 +4,12 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { DelegateFocusMixinClass } from '@vaadin/a11y-base/src/delegate-focus-mixin.js';
-import type { DisabledMixinClass } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin.js';
-import type { TabindexMixinClass } from '@vaadin/a11y-base/src/tabindex-mixin.js';
-import type { DelegateStateMixinClass } from '@vaadin/component-base/src/delegate-state-mixin.js';
-import type { CollapsibleMixinClass } from '@vaadin/details/src/collapsible-mixin.js';
+import type { DelegateFocusMixinClass } from '@scoped-vaadin/a11y-base/src/delegate-focus-mixin.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import type { TabindexMixinClass } from '@scoped-vaadin/a11y-base/src/tabindex-mixin.js';
+import type { DelegateStateMixinClass } from '@scoped-vaadin/component-base/src/delegate-state-mixin.js';
+import type { CollapsibleMixinClass } from '@scoped-vaadin/details/src/collapsible-mixin.js';
 
 /**
  * A mixin providing common accordion panel functionality.
