Index: src/vaadin-lit-accordion-heading.js
===================================================================
--- src/vaadin-lit-accordion-heading.js
+++ src/vaadin-lit-accordion-heading.js
@@ -4,15 +4,15 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { ActiveMixin } from '@vaadin/a11y-base/src/active-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ActiveMixin } from '@scoped-vaadin/a11y-base/src/active-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { accordionHeading } from './vaadin-accordion-heading-styles.js';
 
 /**
- * LitElement based version of `<vaadin-accordion-heading>` web component.
+ * LitElement based version of `<vaadin24-accordion-heading>` web component.
  *
  * ## Disclaimer
  *
@@ -22,7 +22,7 @@
  */
 class AccordionHeading extends ActiveMixin(DirMixin(ThemableMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-accordion-heading';
+    return 'vaadin24-accordion-heading';
   }
 
   static get shadowRootOptions() {
