Index: accordion/src/vaadin-accordion-heading.d.ts
===================================================================
--- accordion/src/vaadin-accordion-heading.d.ts
+++ accordion/src/vaadin-accordion-heading.d.ts
@@ -3,14 +3,14 @@
  * Copyright (c) 2019 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ActiveMixin } from '@vaadin/a11y-base/src/active-mixin.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ActiveMixin } from '@scoped-vaadin/a11y-base/src/active-mixin.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
  * The accordion heading element.
  *
- * `vaadin-accordion-heading` is the element for the headings in the accordion.
+ * `vaadin24-accordion-heading` is the element for the headings in the accordion.
  * As recommended by the WAI ARIA Best Practices, each heading needs to wrap a
  * `<button>`. This element puts that button in the Shadow DOM, as it is more
  * convenient to use and doesn't make styling of the heading more problematic.
@@ -49,7 +49,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-accordion-heading': AccordionHeading;
+    'vaadin24-accordion-heading': AccordionHeading;
   }
 }
 
