Index: src/vaadin-accordion-panel.js
===================================================================
--- src/vaadin-accordion-panel.js
+++ src/vaadin-accordion-panel.js
@@ -5,13 +5,13 @@
  */
 import './vaadin-accordion-heading.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { accordionPanel } from './vaadin-accordion-panel-core-styles.js';
 import { AccordionPanelMixin } from './vaadin-accordion-panel-mixin.js';
 
-registerStyles('vaadin-accordion-panel', accordionPanel, { moduleId: 'vaadin-accordion-panel-styles' });
+registerStyles('vaadin24-accordion-panel', accordionPanel, { moduleId: 'vaadin-accordion-panel-styles' });
 
 /**
  * The accordion panel element.
@@ -45,7 +45,7 @@
  */
 class AccordionPanel extends AccordionPanelMixin(ThemableMixin(ControllerMixin(PolymerElement))) {
   static get is() {
-    return 'vaadin-accordion-panel';
+    return 'vaadin24-accordion-panel';
   }
 
   static get template() {
