Index: src/vaadin-accordion.js
===================================================================
--- src/vaadin-accordion.js
+++ src/vaadin-accordion.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2019 - 2022 Vaadin Ltd.
@@ -5,15 +6,15 @@
  */
 import { FlattenedNodesObserver } from '@polymer/polymer/lib/utils/flattened-nodes-observer.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { KeyboardDirectionMixin } from '@vaadin/component-base/src/keyboard-direction-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { KeyboardDirectionMixin } from '@scoped-vaadin/component-base/src/keyboard-direction-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { AccordionPanel } from './vaadin-accordion-panel.js';
 
 /**
- * `<vaadin-accordion>` is a Web Component implementing accordion widget —
+ * `<vaadin23-accordion>` is a Web Component implementing accordion widget —
  * a vertically stacked set of expandable panels. The component should be
- * used as a wrapper for two or more `<vaadin-accordion-panel>` components.
+ * used as a wrapper for two or more `<vaadin23-accordion-panel>` components.
  *
  * Panel headings function as controls that enable users to open (expand)
  * or hide (collapse) their associated sections of content. The user can
@@ -23,28 +24,28 @@
  * previous panel to close and hide its content.
  *
  * ```
- * <vaadin-accordion>
- *   <vaadin-accordion-panel>
+ * <vaadin23-accordion>
+ *   <vaadin23-accordion-panel>
  *     <div slot="summary">Panel 1</div>
  *     This panel is opened, so the text is visible by default.
- *   </vaadin-accordion-panel>
- *   <vaadin-accordion-panel>
+ *   </vaadin23-accordion-panel>
+ *   <vaadin23-accordion-panel>
  *     <div slot="summary">Panel 2</div>
  *     After opening this panel, the first one becomes closed.
- *   </vaadin-accordion-panel>
- * </vaadin-accordion>
+ *   </vaadin23-accordion-panel>
+ * </vaadin23-accordion>
  * ```
  *
  * ### Styling
  *
- * See the [`<vaadin-accordion-panel>`](#/elements/vaadin-accordion-panel)
+ * See the [`<vaadin23-accordion-panel>`](#/elements/vaadin-accordion-panel)
  * documentation for the available state attributes and stylable shadow parts.
  *
- * **Note:** You can apply the theme to `<vaadin-accordion>` component itself,
+ * **Note:** You can apply the theme to `<vaadin23-accordion>` component itself,
  * especially by using the following CSS selector:
  *
  * ```
- * :host ::slotted(vaadin-accordion-panel) {
+ * :host ::slotted(vaadin23-accordion-panel) {
  *   margin-bottom: 5px;
  * }
  * ```
@@ -76,7 +77,7 @@
   }
 
   static get is() {
-    return 'vaadin-accordion';
+    return 'vaadin23-accordion';
   }
 
   static get properties() {
@@ -95,7 +96,7 @@
       },
 
       /**
-       * The list of `<vaadin-accordion-panel>` child elements.
+       * The list of `<vaadin23-accordion-panel>` child elements.
        * It is populated from the elements passed to the light DOM,
        * and updated dynamically when adding or removing panels.
        * @type {!Array<!AccordionPanel>}
@@ -206,6 +207,6 @@
   }
 }
 
-customElements.define(Accordion.is, Accordion);
+internalCustomElements.define(Accordion.is, Accordion);
 
 export { Accordion };
