Index: test/drag-and-drop.test.js
===================================================================
--- test/drag-and-drop.test.js
+++ test/drag-and-drop.test.js
@@ -10,10 +10,10 @@
   beforeEach(async () => {
     container = fixtureSync(`
       <div style="width: 300px; height: 300px;">
-        <vaadin-virtual-list draggable="true"></vaadin-virtual-list>
+        <vaadin24-virtual-list draggable="true"></vaadin24-virtual-list>
       </div>
     `);
-    virtualList = container.querySelector('vaadin-virtual-list');
+    virtualList = container.querySelector('vaadin24-virtual-list');
     virtualList.renderer = (root, _, { item }) => {
       root.innerHTML = `<div>${item.label}</div>`;
     };
