Index: test/drag-and-drop.common.js
===================================================================
--- test/drag-and-drop.common.js
+++ test/drag-and-drop.common.js
@@ -1,7 +1,7 @@
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import { resetMouse, sendMouse } from '@web/test-runner-commands';
-import { hover } from '@vaadin/button/test/visual/helpers.js';
+import { hover } from '@scoped-vaadin/button/test/visual/helpers.js';
 
 describe('drag and drop', () => {
   let virtualList;
@@ -10,10 +10,10 @@
   beforeEach(async () => {
     container = fixtureSync(`
       <div style="width: 300px; height: 300px;">
-        <vaadin-virtual-list draggable="true"></vaadin-virtual-list>
+        <vaadin24-virtual-list draggable="true"></vaadin24-virtual-list>
       </div>
     `);
-    virtualList = container.querySelector('vaadin-virtual-list');
+    virtualList = container.querySelector('vaadin24-virtual-list');
     virtualList.renderer = (root, _, { item }) => {
       root.innerHTML = `<div>${item.label}</div>`;
     };
