Index: test/lit.common.js
===================================================================
--- test/lit.common.js
+++ test/lit.common.js
@@ -7,7 +7,7 @@
     let list;
 
     beforeEach(async () => {
-      list = fixtureSync(`<vaadin-virtual-list></vaadin-virtual-list>`);
+      list = fixtureSync(`<vaadin24-virtual-list></vaadin24-virtual-list>`);
       await nextFrame();
 
       const size = 100;
