Index: test/template.test.js
===================================================================
--- test/template.test.js
+++ test/template.test.js
@@ -1,6 +1,6 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync } from '@vaadin/testing-helpers';
-import '@vaadin/polymer-legacy-adapter/template-renderer.js';
+import '@scoped-vaadin/polymer-legacy-adapter/template-renderer.js';
 import '../vaadin-virtual-list.js';
 
 describe('template', () => {
@@ -8,11 +8,11 @@
 
   beforeEach(() => {
     list = fixtureSync(`
-      <vaadin-virtual-list>
+      <vaadin24-virtual-list>
         <template>
           [[index]]
         </template>
-      </vaadin-virtual-list>
+      </vaadin24-virtual-list>
     `);
 
     list.items = [0, 1, 2];
