Index: test/virtual-list.common.js
===================================================================
--- test/virtual-list.common.js
+++ test/virtual-list.common.js
@@ -5,7 +5,7 @@
   let list;
 
   beforeEach(() => {
-    list = fixtureSync(`<vaadin-virtual-list></vaadin-virtual-list>`);
+    list = fixtureSync(`<vaadin24-virtual-list></vaadin24-virtual-list>`);
   });
 
   it('should have a default height', () => {
@@ -30,7 +30,7 @@
   it('should not collapse inside a flexbox', () => {
     const flexBox = fixtureSync(`
       <div style="display:flex">
-        <vaadin-virtual-list></vaadin-virtual-list>
+        <vaadin24-virtual-list></vaadin24-virtual-list>
       </div>`);
 
     expect(flexBox.firstElementChild.offsetWidth).to.equal(flexBox.offsetWidth);
