Index: test/field-components.test.js
===================================================================
--- test/field-components.test.js
+++ test/field-components.test.js
@@ -3,16 +3,16 @@
 import { arrowDown, fixtureSync, nextFrame, nextRender, oneEvent, outsideClick } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import './test-styles.test.js';
-import '@vaadin/checkbox-group/src/vaadin-checkbox-group.js';
-import '@vaadin/date-picker/src/vaadin-date-picker.js';
-import '@vaadin/date-time-picker/src/vaadin-date-time-picker.js';
-import '@vaadin/item/src/vaadin-item.js';
-import '@vaadin/list-box/src/vaadin-list-box.js';
-import '@vaadin/radio-group/src/vaadin-radio-group.js';
-import '@vaadin/select/src/vaadin-select.js';
-import '@vaadin/text-field/src/vaadin-text-field.js';
+import '@scoped-vaadin/checkbox-group/src/vaadin-checkbox-group.js';
+import '@scoped-vaadin/date-picker/src/vaadin-date-picker.js';
+import '@scoped-vaadin/date-time-picker/src/vaadin-date-time-picker.js';
+import '@scoped-vaadin/item/src/vaadin-item.js';
+import '@scoped-vaadin/list-box/src/vaadin-list-box.js';
+import '@scoped-vaadin/radio-group/src/vaadin-radio-group.js';
+import '@scoped-vaadin/select/src/vaadin-select.js';
+import '@scoped-vaadin/text-field/src/vaadin-text-field.js';
 import { html, render } from 'lit';
-import { untilOverlayRendered } from '@vaadin/date-picker/test/helpers.js';
+import { untilOverlayRendered } from '@scoped-vaadin/date-picker/test/helpers.js';
 import { FieldHighlighter } from '../src/vaadin-field-highlighter.js';
 
 async function waitForIntersectionObserver() {
@@ -38,7 +38,7 @@
 
   describe('text field', () => {
     beforeEach(async () => {
-      field = fixtureSync(`<vaadin-text-field></vaadin-text-field>`);
+      field = fixtureSync(`<vaadin24-text-field></vaadin24-text-field>`);
       FieldHighlighter.init(field);
       showSpy = sinon.spy();
       hideSpy = sinon.spy();
@@ -61,7 +61,7 @@
 
   describe('date picker', () => {
     beforeEach(async () => {
-      field = fixtureSync(`<vaadin-date-picker></vaadin-date-picker>`);
+      field = fixtureSync(`<vaadin24-date-picker></vaadin24-date-picker>`);
       FieldHighlighter.init(field);
       overlay = field.$.overlay;
       showSpy = sinon.spy();
@@ -173,7 +173,7 @@
 
   describe('select', () => {
     beforeEach(async () => {
-      field = fixtureSync(`<vaadin-select></vaadin-select>`);
+      field = fixtureSync(`<vaadin24-select></vaadin24-select>`);
       field.renderer = (root) => {
         if (root.firstChild) {
           return;
@@ -181,11 +181,11 @@
 
         render(
           html`
-            <vaadin-list-box>
-              <vaadin-item>Foo</vaadin-item>
-              <vaadin-item>Bar</vaadin-item>
-              <vaadin-item>Baz</vaadin-item>
-            </vaadin-list-box>
+            <vaadin24-list-box>
+              <vaadin24-item>Foo</vaadin24-item>
+              <vaadin24-item>Bar</vaadin24-item>
+              <vaadin24-item>Baz</vaadin24-item>
+            </vaadin24-list-box>
           `,
           root,
         );
@@ -221,7 +221,7 @@
         field.focus();
         await open(field);
 
-        overlay.querySelector('vaadin-item').click();
+        overlay.querySelector('vaadin24-item').click();
         await nextRender();
 
         expect(hideSpy.callCount).to.equal(0);
@@ -266,7 +266,7 @@
         field.focus();
         await open(field);
 
-        overlay.querySelector('vaadin-item').click();
+        overlay.querySelector('vaadin24-item').click();
         await nextRender();
 
         expect(hideSpy.callCount).to.equal(1);
@@ -279,11 +279,11 @@
 
     beforeEach(async () => {
       field = fixtureSync(`
-        <vaadin-checkbox-group>
-          <vaadin-checkbox value="1" label="Checkbox 1"></vaadin-checkbox>
-          <vaadin-checkbox value="2" label="Checkbox 2"></vaadin-checkbox>
-          <vaadin-checkbox value="3" label="Checkbox 3"></vaadin-checkbox>
-        </vaadin-checkbox-group>
+        <vaadin24-checkbox-group>
+          <vaadin24-checkbox value="1" label="Checkbox 1"></vaadin24-checkbox>
+          <vaadin24-checkbox value="2" label="Checkbox 2"></vaadin24-checkbox>
+          <vaadin24-checkbox value="3" label="Checkbox 3"></vaadin24-checkbox>
+        </vaadin24-checkbox-group>
       `);
       FieldHighlighter.init(field);
       showSpy = sinon.spy();
@@ -336,11 +336,11 @@
 
     beforeEach(async () => {
       field = fixtureSync(`
-        <vaadin-radio-group>
-          <vaadin-radio-button value="1" label="Radio 1"></vaadin-radio-button>
-          <vaadin-radio-button value="2" label="Radio 2"></vaadin-radio-button>
-          <vaadin-radio-button value="3" label="Radio 3"></vaadin-radio-button>
-        </vaadin-radio-group>
+        <vaadin24-radio-group>
+          <vaadin24-radio-button value="1" label="Radio 1"></vaadin24-radio-button>
+          <vaadin24-radio-button value="2" label="Radio 2"></vaadin24-radio-button>
+          <vaadin24-radio-button value="3" label="Radio 3"></vaadin24-radio-button>
+        </vaadin24-radio-group>
       `);
       FieldHighlighter.init(field);
       showSpy = sinon.spy();
@@ -393,10 +393,10 @@
     let time;
 
     beforeEach(async () => {
-      field = fixtureSync(`<vaadin-date-time-picker></vaadin-date-time-picker>`);
+      field = fixtureSync(`<vaadin24-date-time-picker></vaadin24-date-time-picker>`);
       FieldHighlighter.init(field);
-      date = field.querySelector('vaadin-date-picker');
-      time = field.querySelector('vaadin-time-picker');
+      date = field.querySelector('vaadin24-date-picker');
+      time = field.querySelector('vaadin24-time-picker');
       overlay = field.$.overlay;
       showSpy = sinon.spy();
       hideSpy = sinon.spy();
@@ -489,11 +489,11 @@
 
     beforeEach(async () => {
       field = fixtureSync(`
-        <vaadin-list-box>
-          <vaadin-item>Option 1</vaadin-item>
-          <vaadin-item>Option 2</vaadin-item>
-          <vaadin-item>Option 3</vaadin-item>
-        </vaadin-list-box>
+        <vaadin24-list-box>
+          <vaadin24-item>Option 1</vaadin24-item>
+          <vaadin24-item>Option 2</vaadin24-item>
+          <vaadin24-item>Option 3</vaadin24-item>
+        </vaadin24-list-box>
       `);
       await nextFrame();
       FieldHighlighter.init(field);
