Index: test/test-styles.test.js
===================================================================
--- test/test-styles.test.js
+++ test/test-styles.test.js
@@ -1,9 +1,9 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 // TODO: subset of Lumo needed for unit tests to pass.
 // These should be eventually covered by base styles.
 registerStyles(
-  'vaadin-text-field',
+  'vaadin24-text-field',
   css`
     /* For user-tags scrollable container tests */
     ::slotted([slot='input']) {
@@ -13,7 +13,7 @@
 );
 
 registerStyles(
-  'vaadin-select-overlay',
+  'vaadin24-select-overlay',
   css`
     /* Dummy animation for outside click test that covers focusin logic
     in SelectObserver relying on "closing" attribute being present */
