Index: theme/lumo/vaadin-context-menu-item-styles.js
===================================================================
--- theme/lumo/vaadin-context-menu-item-styles.js
+++ theme/lumo/vaadin-context-menu-item-styles.js
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { item } from '@vaadin/item/theme/lumo/vaadin-item-styles.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { item } from '@scoped-vaadin/item/theme/lumo/vaadin-item-styles.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const contextMenuItem = css`
   /* :hover needed to workaround https://github.com/vaadin/web-components/issues/3133 */
@@ -40,6 +40,6 @@
   }
 `;
 
-registerStyles('vaadin-context-menu-item', [item, contextMenuItem], { moduleId: 'lumo-context-menu-item' });
+registerStyles('vaadin24-context-menu-item', [item, contextMenuItem], { moduleId: 'lumo-context-menu-item' });
 
 export { contextMenuItem };
