Index: theme/lumo/vaadin-context-menu-overlay-styles.js
===================================================================
--- theme/lumo/vaadin-context-menu-overlay-styles.js
+++ theme/lumo/vaadin-context-menu-overlay-styles.js
@@ -1,7 +1,7 @@
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import { menuOverlay } from '@vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import { menuOverlay } from '@scoped-vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const contextMenuOverlay = css`
   :host([phone]) {
@@ -28,7 +28,7 @@
   }
 `;
 
-registerStyles('vaadin-context-menu-overlay', [menuOverlay, contextMenuOverlay], {
+registerStyles('vaadin24-context-menu-overlay', [menuOverlay, contextMenuOverlay], {
   moduleId: 'lumo-context-menu-overlay',
 });
 
