Index: context-menu/theme/lumo/vaadin-context-menu-list-box-styles.js
===================================================================
--- context-menu/theme/lumo/vaadin-context-menu-list-box-styles.js
+++ context-menu/theme/lumo/vaadin-context-menu-list-box-styles.js
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import { listBox } from '@vaadin/list-box/theme/lumo/vaadin-list-box-styles.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import { listBox } from '@scoped-vaadin/list-box/theme/lumo/vaadin-list-box-styles.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const contextMenuListBox = css`
   :host {
@@ -40,7 +40,7 @@
   }
 `;
 
-registerStyles('vaadin-context-menu-list-box', [listBox, contextMenuListBox], {
+registerStyles('vaadin24-context-menu-list-box', [listBox, contextMenuListBox], {
   moduleId: 'lumo-context-menu-list-box',
 });
 
