Index: theme/lumo/vaadin-context-menu.js
===================================================================
--- theme/lumo/vaadin-context-menu.js
+++ theme/lumo/vaadin-context-menu.js
@@ -1,5 +1,5 @@
 import './vaadin-context-menu-styles.js';
-import '@vaadin/item/theme/lumo/vaadin-item.js';
-import '@vaadin/list-box/theme/lumo/vaadin-list-box.js';
-import '@vaadin/overlay/theme/lumo/vaadin-overlay.js';
+import '@scoped-vaadin/item/theme/lumo/vaadin-item.js';
+import '@scoped-vaadin/list-box/theme/lumo/vaadin-list-box.js';
+import '@scoped-vaadin/overlay/theme/lumo/vaadin-overlay.js';
 import '../../src/vaadin-context-menu.js';
