Index: theme/lumo/vaadin-item-styles.js
===================================================================
--- theme/lumo/vaadin-item-styles.js
+++ theme/lumo/vaadin-item-styles.js
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const item = css`
   :host {
@@ -85,12 +85,12 @@
   }
 
   /* Slotted icons */
-  :host ::slotted(vaadin-icon) {
+  :host ::slotted(vaadin24-icon) {
     width: var(--lumo-icon-size-m);
     height: var(--lumo-icon-size-m);
   }
 `;
 
-registerStyles('vaadin-item', item, { moduleId: 'lumo-item' });
+registerStyles('vaadin24-item', item, { moduleId: 'lumo-item' });
 
 export { item };
