Index: src/vaadin-select-overlay.js
===================================================================
--- src/vaadin-select-overlay.js
+++ src/vaadin-select-overlay.js
@@ -1,14 +1,15 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { Overlay } from '@vaadin/overlay/src/vaadin-overlay.js';
-import { PositionMixin } from '@vaadin/overlay/src/vaadin-overlay-position-mixin.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { Overlay } from '@scoped-vaadin/overlay/src/vaadin-overlay.js';
+import { PositionMixin } from '@scoped-vaadin/overlay/src/vaadin-overlay-position-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-select-overlay',
+  'vaadin23-select-overlay',
   css`
     :host {
       align-items: flex-start;
@@ -19,14 +20,14 @@
 );
 
 /**
- * An element used internally by `<vaadin-select>`. Not intended to be used separately.
+ * An element used internally by `<vaadin23-select>`. Not intended to be used separately.
  *
  * @extends Overlay
  * @protected
  */
 class SelectOverlay extends PositionMixin(Overlay) {
   static get is() {
-    return 'vaadin-select-overlay';
+    return 'vaadin23-select-overlay';
   }
 
   requestContentUpdate() {
@@ -45,4 +46,4 @@
   }
 }
 
-customElements.define(SelectOverlay.is, SelectOverlay);
+internalCustomElements.define(SelectOverlay.is, SelectOverlay);
