Index: select/src/vaadin-select-base-mixin.d.ts
===================================================================
--- select/src/vaadin-select-base-mixin.d.ts	null
+++ select/src/vaadin-select-base-mixin.d.ts	null
@@ -4,16 +4,16 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { Constructor } from '@open-wc/dedupe-mixin';
-import type { DelegateFocusMixinClass } from '@vaadin/a11y-base/src/delegate-focus-mixin.js';
-import type { DisabledMixinClass } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin.js';
-import type { KeyboardMixinClass } from '@vaadin/a11y-base/src/keyboard-mixin.js';
-import type { TabindexMixinClass } from '@vaadin/a11y-base/src/tabindex-mixin.js';
-import type { DelegateStateMixinClass } from '@vaadin/component-base/src/delegate-state-mixin.js';
-import type { OverlayClassMixinClass } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import type { FieldMixinClass } from '@vaadin/field-base/src/field-mixin.js';
-import type { LabelMixinClass } from '@vaadin/field-base/src/label-mixin.js';
-import type { ValidateMixinClass } from '@vaadin/field-base/src/validate-mixin.js';
+import type { DelegateFocusMixinClass } from '@scoped-vaadin/a11y-base/src/delegate-focus-mixin.js';
+import type { DisabledMixinClass } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import type { FocusMixinClass } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import type { KeyboardMixinClass } from '@scoped-vaadin/a11y-base/src/keyboard-mixin.js';
+import type { TabindexMixinClass } from '@scoped-vaadin/a11y-base/src/tabindex-mixin.js';
+import type { DelegateStateMixinClass } from '@scoped-vaadin/component-base/src/delegate-state-mixin.js';
+import type { OverlayClassMixinClass } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import type { FieldMixinClass } from '@scoped-vaadin/field-base/src/field-mixin.js';
+import type { LabelMixinClass } from '@scoped-vaadin/field-base/src/label-mixin.js';
+import type { ValidateMixinClass } from '@scoped-vaadin/field-base/src/validate-mixin.js';
 import type { SelectItem, SelectRenderer } from './vaadin-select.js';
 
 export declare function SelectBaseMixin<T extends Constructor<HTMLElement>>(
@@ -48,7 +48,7 @@
    * ];
    * ```
    *
-   * Note: each item is rendered by default as the internal `<vaadin-select-item>` that is an extension of `<vaadin-item>`.
+   * Note: each item is rendered by default as the internal `<vaadin24-select-item>` that is an extension of `<vaadin24-item>`.
    * To render the item with a custom component, provide a tag name by the `component` property.
    *
    * @type {!Array<!SelectItem>}
@@ -61,12 +61,12 @@
   opened: boolean;
 
   /**
-   * Custom function for rendering the content of the `<vaadin-select>`.
+   * Custom function for rendering the content of the `<vaadin24-select>`.
    * Receives two arguments:
    *
-   * - `root` The `<vaadin-select-overlay>` internal container
+   * - `root` The `<vaadin24-select-overlay>` internal container
    *   DOM element. Append your content to it.
-   * - `select` The reference to the `<vaadin-select>` element.
+   * - `select` The reference to the `<vaadin24-select>` element.
    */
   renderer: SelectRenderer | undefined;
 
