Index: src/vaadin-select-value-button.js
===================================================================
--- src/vaadin-select-value-button.js
+++ src/vaadin-select-value-button.js
@@ -1,14 +1,15 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ButtonMixin } from '@vaadin/button/src/vaadin-button-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ButtonMixin } from '@scoped-vaadin/button/src/vaadin-button-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * An element used internally by `<vaadin-select>`. Not intended to be used separately.
+ * An element used internally by `<vaadin23-select>`. Not intended to be used separately.
  *
  * @extends HTMLElement
  * @mixes ButtonMixin
@@ -17,7 +18,7 @@
  */
 class SelectValueButton extends ButtonMixin(ThemableMixin(PolymerElement)) {
   static get is() {
-    return 'vaadin-select-value-button';
+    return 'vaadin23-select-value-button';
   }
 
   static get template() {
@@ -74,4 +75,4 @@
   }
 }
 
-customElements.define(SelectValueButton.is, SelectValueButton);
+internalCustomElements.define(SelectValueButton.is, SelectValueButton);
