Index: theme/lumo/vaadin-combo-box.js
===================================================================
--- theme/lumo/vaadin-combo-box.js
+++ theme/lumo/vaadin-combo-box.js
@@ -1,4 +1,4 @@
-import '@vaadin/input-container/theme/lumo/vaadin-input-container.js';
+import '@scoped-vaadin/input-container/theme/lumo/vaadin-input-container.js';
 import './vaadin-combo-box-dropdown-styles.js';
 import './vaadin-combo-box-item-styles.js';
 import './vaadin-combo-box-styles.js';
