Index: test/validation.common.js
===================================================================
--- test/validation.common.js
+++ test/validation.common.js
@@ -10,10 +10,10 @@
 
     beforeEach(async () => {
       customField = fixtureSync(`
-        <vaadin-custom-field>
+        <vaadin24-custom-field>
           <input type="text" />
           <input type="number" />
-        </vaadin-custom-field>
+        </vaadin24-custom-field>
       `);
       await nextRender();
       validateSpy = sinon.spy(customField, 'validate');
@@ -65,7 +65,7 @@
 
     it('should not throw when checkValidity() called without inputs', () => {
       expect(() => {
-        document.createElement('vaadin-custom-field').checkValidity();
+        document.createElement('vaadin24-custom-field').checkValidity();
       }).to.not.throw(Error);
     });
   });
@@ -73,10 +73,10 @@
   describe('required', () => {
     beforeEach(async () => {
       customField = fixtureSync(`
-        <vaadin-custom-field required>
+        <vaadin24-custom-field required>
           <input type="text" />
           <input type="number" />
-        </vaadin-custom-field>
+        </vaadin24-custom-field>
       `);
       await nextRender();
     });
