Index: test/validation.test.js
===================================================================
--- test/validation.test.js
+++ test/validation.test.js
@@ -11,10 +11,10 @@
 
     beforeEach(async () => {
       customField = fixtureSync(`
-        <vaadin-custom-field>
+        <vaadin24-custom-field>
           <input type="text" />
           <input type="number" />
-        </vaadin-custom-field>
+        </vaadin24-custom-field>
       `);
       await nextRender();
       validateSpy = sinon.spy(customField, 'validate');
@@ -67,10 +67,10 @@
   describe('required', () => {
     beforeEach(async () => {
       customField = fixtureSync(`
-        <vaadin-custom-field required>
+        <vaadin24-custom-field required>
           <input type="text" />
           <input type="number" />
-        </vaadin-custom-field>
+        </vaadin24-custom-field>
       `);
       await nextRender();
     });
