Index: test/aria.test.js
===================================================================
--- test/aria.test.js
+++ test/aria.test.js
@@ -6,7 +6,7 @@
   let timePicker, comboBox, input;
 
   beforeEach(async () => {
-    timePicker = fixtureSync(`<vaadin-time-picker></vaadin-time-picker>`);
+    timePicker = fixtureSync(`<vaadin24-time-picker></vaadin24-time-picker>`);
     await nextRender();
     comboBox = timePicker.$.comboBox;
     input = timePicker.inputElement;
@@ -31,7 +31,7 @@
 
     beforeEach(async () => {
       arrowDownKeyDown(input);
-      items = comboBox._scroller.querySelectorAll('vaadin-time-picker-item');
+      items = comboBox._scroller.querySelectorAll('vaadin24-time-picker-item');
       await nextFrame();
     });
 
