Index: test/events.test.js
===================================================================
--- test/events.test.js
+++ test/events.test.js
@@ -14,7 +14,7 @@
     beforeEach(() => {
       hasInputValueChangedSpy = sinon.spy();
       valueChangedSpy = sinon.spy();
-      timePicker = fixtureSync('<vaadin-time-picker clear-button-visible></vaadin-time-picker>');
+      timePicker = fixtureSync('<vaadin24-time-picker clear-button-visible></vaadin24-time-picker>');
       clearButton = timePicker.shadowRoot.querySelector('[part=clear-button]');
       timePicker.addEventListener('has-input-value-changed', hasInputValueChangedSpy);
       timePicker.addEventListener('value-changed', valueChangedSpy);
