Index: test/helpers.js
===================================================================
--- test/helpers.js
+++ test/helpers.js
@@ -16,7 +16,7 @@
  */
 export const getAllItems = (timePicker) => {
   const { comboBox } = timePicker.$;
-  return Array.from(comboBox._scroller.querySelectorAll('vaadin-time-picker-item'))
+  return Array.from(comboBox._scroller.querySelectorAll('vaadin24-time-picker-item'))
     .filter((item) => !item.hidden)
     .sort((a, b) => a.index - b.index);
 };
