Index: test/time-picker.test.js
===================================================================
--- test/time-picker.test.js
+++ test/time-picker.test.js
@@ -1,17 +1,18 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { arrowDown, arrowUp, enter, esc, fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
 import './not-animated-styles.js';
 import '../vaadin-time-picker.js';
-import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
+import { isTouch } from '@scoped-vaadin/component-base/src/browser-utils.js';
 import { outsideClick, setInputValue } from './helpers.js';
 
 describe('time-picker', () => {
   let timePicker, comboBox, inputElement;
 
   beforeEach(() => {
-    timePicker = fixtureSync(`<vaadin-time-picker></vaadin-time-picker>`);
+    timePicker = fixtureSync(`<vaadin23-time-picker></vaadin23-time-picker>`);
     comboBox = timePicker.$.comboBox;
     inputElement = timePicker.inputElement;
   });
@@ -24,11 +25,11 @@
     });
 
     it('should be defined in custom element registry', () => {
-      expect(customElements.get(tagName)).to.be.ok;
+      expect(internalCustomElements.get(tagName)).to.be.ok;
     });
 
     it('should have a valid static "is" getter', () => {
-      expect(customElements.get(tagName).is).to.equal(tagName);
+      expect(internalCustomElements.get(tagName).is).to.equal(tagName);
     });
   });
 
@@ -185,7 +186,7 @@
     let overlay;
 
     beforeEach(() => {
-      overlay = comboBox.shadowRoot.querySelector('vaadin-time-picker-overlay');
+      overlay = comboBox.shadowRoot.querySelector('vaadin23-time-picker-overlay');
     });
 
     it('should open overlay using open() call', () => {
