Index: test/value-commit.test.js
===================================================================
--- test/value-commit.test.js
+++ test/value-commit.test.js
@@ -42,7 +42,7 @@
   }
 
   beforeEach(async () => {
-    timePicker = fixtureSync(`<vaadin-time-picker></vaadin-time-picker>`);
+    timePicker = fixtureSync(`<vaadin24-time-picker></vaadin24-time-picker>`);
     await nextRender();
     validateSpy = sinon.spy(timePicker, 'validate').named('validateSpy');
 
