Index: scroller/theme/material/vaadin-scroller-styles.js
===================================================================
--- scroller/theme/material/vaadin-scroller-styles.js
+++ scroller/theme/material/vaadin-scroller-styles.js
@@ -1,5 +1,5 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const scroller = css`
   :host {
@@ -11,6 +11,6 @@
   }
 `;
 
-registerStyles('vaadin-scroller', scroller, { moduleId: 'material-scroller' });
+registerStyles('vaadin24-scroller', scroller, { moduleId: 'material-scroller' });
 
 export { scroller };
