Index: src/vaadin-message-list.js
===================================================================
--- src/vaadin-message-list.js
+++ src/vaadin-message-list.js
@@ -6,27 +6,27 @@
 import { html as legacyHtml, PolymerElement } from '@polymer/polymer/polymer-element.js';
 import { html, render } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { KeyboardDirectionMixin } from '@vaadin/a11y-base/src/keyboard-direction-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { KeyboardDirectionMixin } from '@scoped-vaadin/a11y-base/src/keyboard-direction-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { Message } from './vaadin-message.js';
 
 /**
- * `<vaadin-message-list>` is a Web Component for showing an ordered list of messages. The messages are rendered as <vaadin-message>
+ * `<vaadin24-message-list>` is a Web Component for showing an ordered list of messages. The messages are rendered as <vaadin24-message>
  *
  * ### Example
  *
  * To create a new message list, add the component to the page:
  *
  * ```html
- * <vaadin-message-list></vaadin-message-list>
+ * <vaadin24-message-list></vaadin24-message-list>
  * ```
  *
  * Provide the messages to the message list with the [`items`](#/elements/vaadin-message-list#property-items) property.
  *
  * ```js
- * document.querySelector('vaadin-message-list').items = [
+ * document.querySelector('vaadin24-message-list').items = [
  *   { text: 'Hello list', time: 'yesterday', userName: 'Matt Mambo', userAbbr: 'MM', userColorIndex: 1 },
  *   { text: 'Another message', time: 'right now', userName: 'Linsey Listy', userAbbr: 'LL', userColorIndex: 2, userImg: '/static/img/avatar.jpg' }
  * ];
@@ -40,7 +40,7 @@
  * ----------|----------------
  * `list`    | The container wrapping messages.
  *
- * See the [`<vaadin-message>`](#/elements/vaadin-message) documentation for the available
+ * See the [`<vaadin24-message>`](#/elements/vaadin-message) documentation for the available
  * state attributes and stylable shadow parts of message elements.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
@@ -53,7 +53,7 @@
  */
 class MessageList extends KeyboardDirectionMixin(ElementMixin(ThemableMixin(PolymerElement))) {
   static get is() {
-    return 'vaadin-message-list';
+    return 'vaadin24-message-list';
   }
 
   static get properties() {
@@ -102,7 +102,7 @@
 
   /** @protected */
   get _messages() {
-    return [...this.querySelectorAll('vaadin-message')];
+    return [...this.querySelectorAll('vaadin24-message')];
   }
 
   /** @protected */
@@ -153,7 +153,7 @@
         ${items.map(
           (item) =>
             html`
-              <vaadin-message
+              <vaadin24-message
                 role="listitem"
                 .time="${item.time}"
                 .userAbbr="${item.userAbbr}"
@@ -163,8 +163,8 @@
                 theme="${ifDefined(item.theme)}"
                 class="${ifDefined(item.className)}"
                 @focusin="${this._onMessageFocusIn}"
-                >${item.text}<vaadin-avatar slot="avatar"></vaadin-avatar
-              ></vaadin-message>
+                >${item.text}<vaadin24-avatar slot="avatar"></vaadin24-avatar
+              ></vaadin24-message>
             `,
         )}
       `,
