Index: src/vaadin-message-avatar.js
===================================================================
--- src/vaadin-message-avatar.js
+++ src/vaadin-message-avatar.js
@@ -1,13 +1,14 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { Avatar } from '@vaadin/avatar/src/vaadin-avatar.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { Avatar } from '@scoped-vaadin/avatar/src/vaadin-avatar.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-message-avatar',
+  'vaadin23-message-avatar',
   css`
     :host {
       --vaadin-avatar-outline-width: 0px; /* stylelint-disable-line length-zero-no-unit */
@@ -18,15 +19,15 @@
 );
 
 /**
- * An element used internally by `<vaadin-message>`. Not intended to be used separately.
+ * An element used internally by `<vaadin23-message>`. Not intended to be used separately.
  *
  * @extends Avatar
  * @private
  */
 class MessageAvatar extends Avatar {
   static get is() {
-    return 'vaadin-message-avatar';
+    return 'vaadin23-message-avatar';
   }
 }
 
-customElements.define(MessageAvatar.is, MessageAvatar);
+internalCustomElements.define(MessageAvatar.is, MessageAvatar);
