Index: src/vaadin-message-mixin.js
===================================================================
--- src/vaadin-message-mixin.js
+++ src/vaadin-message-mixin.js
@@ -3,9 +3,9 @@
  * Copyright (c) 2021 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { SlotController } from '@scoped-vaadin/component-base/src/slot-controller.js';
 
 /**
  * @polymerMixin
@@ -95,7 +95,7 @@
     ready() {
       super.ready();
 
-      this._avatarController = new SlotController(this, 'avatar', 'vaadin-avatar', {
+      this._avatarController = new SlotController(this, 'avatar', 'vaadin24-avatar', {
         initializer: (avatar) => {
           avatar.setAttribute('tabindex', '-1');
           avatar.setAttribute('aria-hidden', 'true');
