Index: src/vaadin-lit-message-list.js
===================================================================
--- src/vaadin-lit-message-list.js
+++ src/vaadin-lit-message-list.js
@@ -5,14 +5,14 @@
  */
 import './vaadin-lit-message.js';
 import { css, html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { MessageListMixin } from './vaadin-message-list-mixin.js';
 
 /**
- * LitElement based version of `<vaadin-message-list>` web component.
+ * LitElement based version of `<vaadin24-message-list>` web component.
  *
  * ## Disclaimer
  *
@@ -22,7 +22,7 @@
  */
 class MessageList extends MessageListMixin(ElementMixin(ThemableMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-message-list';
+    return 'vaadin24-message-list';
   }
 
   static get styles() {
