Index: src/vaadin-message.d.ts
===================================================================
--- src/vaadin-message.d.ts
+++ src/vaadin-message.d.ts
@@ -3,18 +3,18 @@
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { FocusMixin } from '@vaadin/component-base/src/focus-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/component-base/src/focus-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * `<vaadin-message>` is a Web Component for showing a single message with an author, message and time.
+ * `<vaadin23-message>` is a Web Component for showing a single message with an author, message and time.
  *
  * ```html
- * <vaadin-message time="2021-01-28 10:43"
+ * <vaadin23-message time="2021-01-28 10:43"
  *     user-name = "Bob Ross"
  *     user-abbr = "BR"
- *     user-img = "/static/img/avatar.jpg">There is no real ending. It's just the place where you stop the story.</vaadin-message>
+ *     user-img = "/static/img/avatar.jpg">There is no real ending. It's just the place where you stop the story.</vaadin23-message>
  * ```
  *
  * ### Styling
@@ -39,10 +39,10 @@
  *
  * ### Internal components
  *
- * In addition to `<vaadin-message>` itself, the following internal
+ * In addition to `<vaadin23-message>` itself, the following internal
  * components are themable:
  *
- * - `<vaadin-message-avatar>` - has the same API as [`<vaadin-avatar>`](#/elements/vaadin-avatar).
+ * - `<vaadin23-message-avatar>` - has the same API as [`<vaadin23-avatar>`](#/elements/vaadin-avatar).
  */
 declare class Message extends FocusMixin(ThemableMixin(ElementMixin(HTMLElement))) {
   /**
@@ -102,7 +102,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-message': Message;
+    'vaadin23-message': Message;
   }
 }
 
