Index: theme/lumo/vaadin-menu-bar.js
===================================================================
--- theme/lumo/vaadin-menu-bar.js
+++ theme/lumo/vaadin-menu-bar.js
@@ -2,5 +2,5 @@
 import './vaadin-menu-bar-item-styles.js';
 import './vaadin-menu-bar-overlay-styles.js';
 import './vaadin-menu-bar-styles.js';
-import '@vaadin/context-menu/theme/lumo/vaadin-context-menu.js';
+import '@scoped-vaadin/context-menu/theme/lumo/vaadin-context-menu.js';
 import '../../src/vaadin-menu-bar.js';
