Index: theme/lumo/vaadin-menu-bar-item-styles.js
===================================================================
--- theme/lumo/vaadin-menu-bar-item-styles.js
+++ theme/lumo/vaadin-menu-bar-item-styles.js
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { contextMenuItem } from '@vaadin/context-menu/theme/lumo/vaadin-context-menu-item-styles.js';
-import { item } from '@vaadin/item/theme/lumo/vaadin-item-styles.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { contextMenuItem } from '@scoped-vaadin/context-menu/theme/lumo/vaadin-context-menu-item-styles.js';
+import { item } from '@scoped-vaadin/item/theme/lumo/vaadin-item-styles.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const menuBarItem = css`
   [part='content'] {
@@ -12,16 +12,16 @@
     justify-content: inherit;
   }
 
-  [part='content'] ::slotted(vaadin-icon) {
+  [part='content'] ::slotted(vaadin24-icon) {
     display: inline-block;
     width: var(--lumo-icon-size-m);
     height: var(--lumo-icon-size-m);
   }
 
-  [part='content'] ::slotted(vaadin-icon[icon^='vaadin:']) {
+  [part='content'] ::slotted(vaadin24-icon[icon^='vaadin:']) {
     padding: var(--lumo-space-xs);
     box-sizing: border-box !important;
   }
 `;
 
-registerStyles('vaadin-menu-bar-item', [item, contextMenuItem, menuBarItem], { moduleId: 'lumo-menu-bar-item' });
+registerStyles('vaadin24-menu-bar-item', [item, contextMenuItem, menuBarItem], { moduleId: 'lumo-menu-bar-item' });
