Index: theme/lumo/vaadin-menu-bar-item-styles.js
===================================================================
--- theme/lumo/vaadin-menu-bar-item-styles.js
+++ theme/lumo/vaadin-menu-bar-item-styles.js
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-context-menu-item',
+  'vaadin23-context-menu-item',
   css`
     :host([theme='menu-bar-item']) [part='content'] {
       display: flex;
@@ -12,14 +12,14 @@
       justify-content: inherit;
     }
 
-    :host([theme='menu-bar-item']) [part='content'] ::slotted(vaadin-icon),
+    :host([theme='menu-bar-item']) [part='content'] ::slotted(vaadin23-icon),
     :host([theme='menu-bar-item']) [part='content'] ::slotted(iron-icon) {
       display: inline-block;
       width: var(--lumo-icon-size-m);
       height: var(--lumo-icon-size-m);
     }
 
-    :host([theme='menu-bar-item']) [part='content'] ::slotted(vaadin-icon[icon^='vaadin:']),
+    :host([theme='menu-bar-item']) [part='content'] ::slotted(vaadin23-icon[icon^='vaadin:']),
     :host([theme='menu-bar-item']) [part='content'] ::slotted(iron-icon[icon^='vaadin:']) {
       padding: var(--lumo-space-xs);
       box-sizing: border-box !important;
