Index: menu-bar/theme/lumo/vaadin-menu-bar-overlay-styles.js
===================================================================
--- menu-bar/theme/lumo/vaadin-menu-bar-overlay-styles.js	null
+++ menu-bar/theme/lumo/vaadin-menu-bar-overlay-styles.js	null
@@ -1,6 +1,6 @@
-import { contextMenuOverlay } from '@vaadin/context-menu/theme/lumo/vaadin-context-menu-overlay-styles.js';
-import { menuOverlay } from '@vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { contextMenuOverlay } from '@scoped-vaadin/context-menu/theme/lumo/vaadin-context-menu-overlay-styles.js';
+import { menuOverlay } from '@scoped-vaadin/vaadin-lumo-styles/mixins/menu-overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const menuBarOverlay = css`
   :host(:first-of-type) {
@@ -8,6 +8,6 @@
   }
 `;
 
-registerStyles('vaadin-menu-bar-overlay', [menuOverlay, contextMenuOverlay, menuBarOverlay], {
+registerStyles('vaadin24-menu-bar-overlay', [menuOverlay, contextMenuOverlay, menuBarOverlay], {
   moduleId: 'lumo-menu-bar-overlay',
 });
