Index: src/vaadin-side-nav-children-mixin.js
===================================================================
--- src/vaadin-side-nav-children-mixin.js
+++ src/vaadin-side-nav-children-mixin.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2023 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { I18nMixin } from '@vaadin/component-base/src/i18n-mixin.js';
-import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
+import { I18nMixin } from '@scoped-vaadin/component-base/src/i18n-mixin.js';
+import { SlotController } from '@scoped-vaadin/component-base/src/slot-controller.js';
 
 const DEFAULT_I18N = {
   toggle: 'Toggle child items',
