Index: src/vaadin-side-nav-item.js
===================================================================
--- src/vaadin-side-nav-item.js
+++ src/vaadin-side-nav-item.js
@@ -5,30 +5,30 @@
  */
 import { html, LitElement } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { DisabledMixin } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { matchPaths } from '@vaadin/component-base/src/url-utils.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DisabledMixin } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { matchPaths } from '@scoped-vaadin/component-base/src/url-utils.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { location } from './location.js';
 import { sideNavItemBaseStyles } from './vaadin-side-nav-base-styles.js';
 import { SideNavChildrenMixin } from './vaadin-side-nav-children-mixin.js';
 
 /**
- * A navigation item to be used within `<vaadin-side-nav>`. Represents a navigation target.
+ * A navigation item to be used within `<vaadin24-side-nav>`. Represents a navigation target.
  * Not intended to be used separately.
  *
  * ```html
- * <vaadin-side-nav-item>
+ * <vaadin24-side-nav-item>
  *   Item 1
- *   <vaadin-side-nav-item path="/path1" slot="children">
+ *   <vaadin24-side-nav-item path="/path1" slot="children">
  *     Child item 1
- *   </vaadin-side-nav-item>
- *   <vaadin-side-nav-item path="/path2" slot="children">
+ *   </vaadin24-side-nav-item>
+ *   <vaadin24-side-nav-item path="/path2" slot="children">
  *     Child item 2
- *   </vaadin-side-nav-item>
- * </vaadin-side-nav-item>
+ *   </vaadin24-side-nav-item>
+ * </vaadin24-side-nav-item>
  * ```
  *
  * ### Customization
@@ -43,11 +43,11 @@
  * #### Example
  *
  * ```html
- * <vaadin-side-nav-item>
- *   <vaadin-icon icon="vaadin:chart" slot="prefix"></vaadin-icon>
+ * <vaadin24-side-nav-item>
+ *   <vaadin24-icon icon="vaadin:chart" slot="prefix"></vaadin24-icon>
  *   Item
  *   <span theme="badge primary" slot="suffix">Suffix</span>
- * </vaadin-side-nav-item>
+ * </vaadin24-side-nav-item>
  * ```
  *
  * ### Styling
@@ -82,7 +82,7 @@
  */
 class SideNavItem extends SideNavChildrenMixin(DisabledMixin(ElementMixin(ThemableMixin(PolylitMixin(LitElement))))) {
   static get is() {
-    return 'vaadin-side-nav-item';
+    return 'vaadin24-side-nav-item';
   }
 
   static get properties() {
