Index: src/vaadin-side-nav.js
===================================================================
--- src/vaadin-side-nav.js
+++ src/vaadin-side-nav.js
@@ -4,25 +4,25 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { generateUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { generateUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { sideNavBaseStyles } from './vaadin-side-nav-base-styles.js';
 import { SideNavChildrenMixin } from './vaadin-side-nav-children-mixin.js';
 
 /**
- * `<vaadin-side-nav>` is a Web Component for navigation menus.
+ * `<vaadin24-side-nav>` is a Web Component for navigation menus.
  *
  * ```html
- * <vaadin-side-nav>
- *   <vaadin-side-nav-item>Item 1</vaadin-side-nav-item>
- *   <vaadin-side-nav-item>Item 2</vaadin-side-nav-item>
- *   <vaadin-side-nav-item>Item 3</vaadin-side-nav-item>
- *   <vaadin-side-nav-item>Item 4</vaadin-side-nav-item>
- * </vaadin-side-nav>
+ * <vaadin24-side-nav>
+ *   <vaadin24-side-nav-item>Item 1</vaadin24-side-nav-item>
+ *   <vaadin24-side-nav-item>Item 2</vaadin24-side-nav-item>
+ *   <vaadin24-side-nav-item>Item 3</vaadin24-side-nav-item>
+ *   <vaadin24-side-nav-item>Item 4</vaadin24-side-nav-item>
+ * </vaadin24-side-nav>
  * ```
  *
  * ### Customization
@@ -36,10 +36,10 @@
  * #### Example
  *
  * ```html
- * <vaadin-side-nav>
+ * <vaadin24-side-nav>
  *   <span slot="label">Main menu</span>
- *   <vaadin-side-nav-item>Item</vaadin-side-nav-item>
- * </vaadin-side-nav>
+ *   <vaadin24-side-nav-item>Item</vaadin24-side-nav-item>
+ * </vaadin24-side-nav>
  * ```
  *
  * ### Styling
@@ -72,7 +72,7 @@
  */
 class SideNav extends SideNavChildrenMixin(FocusMixin(ElementMixin(ThemableMixin(PolylitMixin(LitElement))))) {
   static get is() {
-    return 'vaadin-side-nav';
+    return 'vaadin24-side-nav';
   }
 
   static get shadowRootOptions() {
