Index: test/visual/material/email-field.test.js
===================================================================
--- test/visual/material/email-field.test.js
+++ test/visual/material/email-field.test.js
@@ -9,7 +9,7 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-email-field></vaadin-email-field>', div);
+    element = fixtureSync('<vaadin23-email-field></vaadin23-email-field>', div);
   });
 
   it('placeholder', async () => {
