Index: notification/src/lit/renderer-directives.d.ts
===================================================================
--- notification/src/lit/renderer-directives.d.ts	null
+++ notification/src/lit/renderer-directives.d.ts	null
@@ -4,8 +4,8 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import type { DirectiveResult } from 'lit/directive.js';
-import type { LitRendererResult } from '@vaadin/lit-renderer';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import type { LitRendererResult } from '@scoped-vaadin/lit-renderer';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 import type { Notification } from '../vaadin-notification.js';
 
 export type NotificationLitRenderer = (notification: Notification) => LitRendererResult;
@@ -44,9 +44,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-notification
+ * `<vaadin24-notification
  *   ${notificationRenderer((notification) => html`...`)}
- * ></vaadin-notification>`
+ * ></vaadin24-notification>`
  * ```
  *
  * @param renderer the renderer callback that returns a Lit template.
