Index: date-picker/src/vaadin-infinite-scroller.js
===================================================================
--- date-picker/src/vaadin-infinite-scroller.js
+++ date-picker/src/vaadin-infinite-scroller.js
@@ -3,10 +3,10 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { timeOut } from '@vaadin/component-base/src/async.js';
-import { isFirefox } from '@vaadin/component-base/src/browser-utils.js';
-import { Debouncer } from '@vaadin/component-base/src/debounce.js';
-import { generateUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
+import { timeOut } from '@scoped-vaadin/component-base/src/async.js';
+import { isFirefox } from '@scoped-vaadin/component-base/src/browser-utils.js';
+import { Debouncer } from '@scoped-vaadin/component-base/src/debounce.js';
+import { generateUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
 
 const template = document.createElement('template');
 template.innerHTML = `
