Index: src/vaadin-month-calendar.js
===================================================================
--- src/vaadin-month-calendar.js
+++ src/vaadin-month-calendar.js
@@ -5,12 +5,12 @@
  */
 import '@polymer/polymer/lib/elements/dom-repeat.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { MonthCalendarMixin } from './vaadin-month-calendar-mixin.js';
 import { monthCalendarStyles } from './vaadin-month-calendar-styles.js';
 
-registerStyles('vaadin-month-calendar', monthCalendarStyles, {
+registerStyles('vaadin24-month-calendar', monthCalendarStyles, {
   moduleId: 'vaadin-month-calendar-styles',
 });
 
@@ -71,7 +71,7 @@
   }
 
   static get is() {
-    return 'vaadin-month-calendar';
+    return 'vaadin24-month-calendar';
   }
 }
 
