Index: src/vaadin-date-picker.js
===================================================================
--- src/vaadin-date-picker.js
+++ src/vaadin-date-picker.js
@@ -3,28 +3,28 @@
  * Copyright (c) 2016 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/src/vaadin-input-container.js';
+import '@scoped-vaadin/input-container/src/vaadin-input-container.js';
 import './vaadin-date-picker-overlay.js';
 import './vaadin-date-picker-overlay-content.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
-import { InputControlMixin } from '@vaadin/field-base/src/input-control-mixin.js';
-import { InputController } from '@vaadin/field-base/src/input-controller.js';
-import { LabelledInputController } from '@vaadin/field-base/src/labelled-input-controller.js';
-import { inputFieldShared } from '@vaadin/field-base/src/styles/input-field-shared-styles.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { TooltipController } from '@scoped-vaadin/component-base/src/tooltip-controller.js';
+import { InputControlMixin } from '@scoped-vaadin/field-base/src/input-control-mixin.js';
+import { InputController } from '@scoped-vaadin/field-base/src/input-controller.js';
+import { LabelledInputController } from '@scoped-vaadin/field-base/src/labelled-input-controller.js';
+import { inputFieldShared } from '@scoped-vaadin/field-base/src/styles/input-field-shared-styles.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { DatePickerMixin } from './vaadin-date-picker-mixin.js';
 import { datePickerStyles } from './vaadin-date-picker-styles.js';
 
-registerStyles('vaadin-date-picker', [inputFieldShared, datePickerStyles], { moduleId: 'vaadin-date-picker-styles' });
+registerStyles('vaadin24-date-picker', [inputFieldShared, datePickerStyles], { moduleId: 'vaadin-date-picker-styles' });
 
 /**
- * `<vaadin-date-picker>` is an input field that allows to enter a date by typing or by selecting from a calendar overlay.
+ * `<vaadin24-date-picker>` is an input field that allows to enter a date by typing or by selecting from a calendar overlay.
  *
  * ```html
- * <vaadin-date-picker label="Birthday"></vaadin-date-picker>
+ * <vaadin24-date-picker label="Birthday"></vaadin24-date-picker>
  * ```
  *
  * ```js
@@ -41,38 +41,38 @@
  * -------------------------------|----------------------------|---------
  * `--vaadin-field-default-width` | Default width of the field | `12em`
  *
- * `<vaadin-date-picker>` provides the same set of shadow DOM parts and state attributes as `<vaadin-text-field>`.
- * See [`<vaadin-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
+ * `<vaadin24-date-picker>` provides the same set of shadow DOM parts and state attributes as `<vaadin24-text-field>`.
+ * See [`<vaadin24-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
  *
- * In addition to `<vaadin-text-field>` parts, the following parts are available for theming:
+ * In addition to `<vaadin24-text-field>` parts, the following parts are available for theming:
  *
  * Part name             | Description
  * ----------------------|--------------------
  * `toggle-button`       | Toggle button
  *
- * In addition to `<vaadin-text-field>` state attributes, the following state attributes are available for theming:
+ * In addition to `<vaadin24-text-field>` state attributes, the following state attributes are available for theming:
  *
  * Attribute  | Description                                      | Part name
  * -----------|--------------------------------------------------|-----------
  * `opened`   | Set when the date selector overlay is opened     | :host
  *
  * If you want to replace the default `<input>` and its container with a custom implementation to get full control
- * over the input field, consider using the [`<vaadin-date-picker-light>`](#/elements/vaadin-date-picker-light) element.
+ * over the input field, consider using the [`<vaadin24-date-picker-light>`](#/elements/vaadin-date-picker-light) element.
  *
  * ### Internal components
  *
- * In addition to `<vaadin-date-picker>` itself, the following internal
+ * In addition to `<vaadin24-date-picker>` itself, the following internal
  * components are themable:
  *
- * - `<vaadin-date-picker-overlay>` - has the same API as [`<vaadin-overlay>`](#/elements/vaadin-overlay).
- * - `<vaadin-date-picker-overlay-content>`
- * - `<vaadin-date-picker-month-scroller>`
- * - `<vaadin-date-picker-year-scroller>`
- * - `<vaadin-date-picker-year>`
- * - `<vaadin-month-calendar>`
- * - [`<vaadin-input-container>`](#/elements/vaadin-input-container) - an internal element wrapping the input.
+ * - `<vaadin24-date-picker-overlay>` - has the same API as [`<vaadin24-overlay>`](#/elements/vaadin-overlay).
+ * - `<vaadin24-date-picker-overlay-content>`
+ * - `<vaadin24-date-picker-month-scroller>`
+ * - `<vaadin24-date-picker-year-scroller>`
+ * - `<vaadin24-date-picker-year>`
+ * - `<vaadin24-month-calendar>`
+ * - [`<vaadin24-input-container>`](#/elements/vaadin-input-container) - an internal element wrapping the input.
  *
- * In order to style the overlay content, use `<vaadin-date-picker-overlay-content>` shadow DOM parts:
+ * In order to style the overlay content, use `<vaadin24-date-picker-overlay-content>` shadow DOM parts:
  *
  * Part name             | Description
  * ----------------------|--------------------
@@ -83,7 +83,7 @@
  * `years-toggle-button` | Fullscreen mode years scroller toggle
  * `toolbar`             | Footer bar with slotted buttons
  *
- * The following state attributes are available on the `<vaadin-date-picker-overlay-content>` element:
+ * The following state attributes are available on the `<vaadin24-date-picker-overlay-content>` element:
  *
  * Attribute       | Description
  * ----------------|-------------------------------------------------
@@ -91,7 +91,7 @@
  * `fullscreen`    | Set when the overlay content is in fullscreen mode
  * `years-visible` | Set when the year scroller is visible in fullscreen mode
  *
- * In order to style the month calendar, use `<vaadin-month-calendar>` shadow DOM parts:
+ * In order to style the month calendar, use `<vaadin24-month-calendar>` shadow DOM parts:
  *
  * Part name             | Description
  * ----------------------|--------------------
@@ -106,14 +106,14 @@
  * `selected`            | Selected date element
  * `today`               | Date element corresponding to the current day
  *
- * In order to style year scroller elements, use `<vaadin-date-picker-year>` shadow DOM parts:
+ * In order to style year scroller elements, use `<vaadin24-date-picker-year>` shadow DOM parts:
  *
  * Part name             | Description
  * ----------------------|--------------------
  * `year-number`         | Year number
  * `year-separator`      | Year separator
  *
- * Note: the `theme` attribute value set on `<vaadin-date-picker>` is
+ * Note: the `theme` attribute value set on `<vaadin24-date-picker>` is
  * propagated to the internal components listed above.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
@@ -150,7 +150,7 @@
  */
 class DatePicker extends DatePickerMixin(InputControlMixin(ThemableMixin(ElementMixin(PolymerElement)))) {
   static get is() {
-    return 'vaadin-date-picker';
+    return 'vaadin24-date-picker';
   }
 
   static get template() {
@@ -161,7 +161,7 @@
           <span part="required-indicator" aria-hidden="true" on-click="focus"></span>
         </div>
 
-        <vaadin-input-container
+        <vaadin24-input-container
           part="input-field"
           readonly="[[readonly]]"
           disabled="[[disabled]]"
@@ -172,7 +172,7 @@
           <slot name="input"></slot>
           <div id="clearButton" part="clear-button" slot="suffix" aria-hidden="true"></div>
           <div part="toggle-button" slot="suffix" aria-hidden="true" on-click="_toggle"></div>
-        </vaadin-input-container>
+        </vaadin24-input-container>
 
         <div part="helper-text">
           <slot name="helper"></slot>
@@ -183,7 +183,7 @@
         </div>
       </div>
 
-      <vaadin-date-picker-overlay
+      <vaadin24-date-picker-overlay
         id="overlay"
         fullscreen$="[[_fullscreen]]"
         theme$="[[_theme]]"
@@ -193,7 +193,7 @@
         on-vaadin-overlay-closing="_onOverlayClosed"
         restore-focus-on-close
         restore-focus-node="[[inputElement]]"
-      ></vaadin-date-picker-overlay>
+      ></vaadin24-date-picker-overlay>
 
       <slot name="tooltip"></slot>
     `;
