Index: src/vaadin-infinite-scroller.js
===================================================================
--- src/vaadin-infinite-scroller.js
+++ src/vaadin-infinite-scroller.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2016 - 2022 Vaadin Ltd.
@@ -6,9 +7,9 @@
 import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
 import { templatize } from '@polymer/polymer/lib/utils/templatize.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { timeOut } from '@vaadin/component-base/src/async.js';
-import { isFirefox } from '@vaadin/component-base/src/browser-utils.js';
-import { Debouncer } from '@vaadin/component-base/src/debounce.js';
+import { timeOut } from '@scoped-vaadin/component-base/src/async.js';
+import { isFirefox } from '@scoped-vaadin/component-base/src/browser-utils.js';
+import { Debouncer } from '@scoped-vaadin/component-base/src/debounce.js';
 
 /**
  * @extends HTMLElement
@@ -70,7 +71,7 @@
   }
 
   static get is() {
-    return 'vaadin-infinite-scroller';
+    return 'vaadin23-infinite-scroller';
   }
 
   static get properties() {
@@ -396,4 +397,4 @@
   }
 }
 
-customElements.define(InfiniteScroller.is, InfiniteScroller);
+internalCustomElements.define(InfiniteScroller.is, InfiniteScroller);
