Index: date-picker/src/vaadin-date-picker-overlay-content.js
===================================================================
--- date-picker/src/vaadin-date-picker-overlay-content.js	null
+++ date-picker/src/vaadin-date-picker-overlay-content.js	null
@@ -3,20 +3,20 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/button/src/vaadin-button.js';
+import '@scoped-vaadin/button/src/vaadin-button.js';
 import './vaadin-date-picker-month-scroller.js';
 import './vaadin-date-picker-year-scroller.js';
 import './vaadin-date-picker-year.js';
 import './vaadin-month-calendar.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { DatePickerOverlayContentMixin } from './vaadin-date-picker-overlay-content-mixin.js';
 import { overlayContentStyles } from './vaadin-date-picker-overlay-content-styles.js';
 
-registerStyles('vaadin-date-picker-overlay-content', overlayContentStyles, {
+registerStyles('vaadin24-date-picker-overlay-content', overlayContentStyles, {
   moduleId: 'vaadin-date-picker-overlay-content-styles',
 });
 
@@ -53,7 +53,7 @@
   }
 
   static get is() {
-    return 'vaadin-date-picker-overlay-content';
+    return 'vaadin24-date-picker-overlay-content';
   }
 
   /** @protected */
