Index: src/vaadin-date-picker-overlay.js
===================================================================
--- src/vaadin-date-picker-overlay.js
+++ src/vaadin-date-picker-overlay.js
@@ -1,29 +1,30 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 /**
  * @license
  * Copyright (c) 2016 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { DisableUpgradeMixin } from '@polymer/polymer/lib/mixins/disable-upgrade-mixin.js';
-import { Overlay } from '@vaadin/overlay/src/vaadin-overlay.js';
-import { PositionMixin } from '@vaadin/overlay/src/vaadin-overlay-position-mixin.js';
-import { registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { Overlay } from '@scoped-vaadin/overlay/src/vaadin-overlay.js';
+import { PositionMixin } from '@scoped-vaadin/overlay/src/vaadin-overlay-position-mixin.js';
+import { registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { datePickerOverlayStyles } from './vaadin-date-picker-styles.js';
 
-registerStyles('vaadin-date-picker-overlay', datePickerOverlayStyles, {
+registerStyles('vaadin23-date-picker-overlay', datePickerOverlayStyles, {
   moduleId: 'vaadin-date-picker-overlay-styles',
 });
 
 let memoizedTemplate;
 
 /**
- * An element used internally by `<vaadin-date-picker>`. Not intended to be used separately.
+ * An element used internally by `<vaadin23-date-picker>`. Not intended to be used separately.
  *
  * @extends Overlay
  * @private
  */
 class DatePickerOverlay extends DisableUpgradeMixin(PositionMixin(Overlay)) {
   static get is() {
-    return 'vaadin-date-picker-overlay';
+    return 'vaadin23-date-picker-overlay';
   }
 
   static get template() {
@@ -36,4 +37,4 @@
   }
 }
 
-customElements.define(DatePickerOverlay.is, DatePickerOverlay);
+internalCustomElements.define(DatePickerOverlay.is, DatePickerOverlay);
