Index: src/vaadin-month-calendar-mixin.js
===================================================================
--- src/vaadin-month-calendar-mixin.js
+++ src/vaadin-month-calendar-mixin.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2016 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
-import { addListener } from '@vaadin/component-base/src/gestures.js';
+import { FocusMixin } from '@scoped-vaadin/a11y-base/src/focus-mixin.js';
+import { addListener } from '@scoped-vaadin/component-base/src/gestures.js';
 import { dateAllowed, dateEquals, getISOWeekNumber } from './vaadin-date-picker-helper.js';
 
 /**
