Index: src/vaadin-date-picker-light.d.ts
===================================================================
--- src/vaadin-date-picker-light.d.ts
+++ src/vaadin-date-picker-light.d.ts
@@ -3,8 +3,8 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ValidateMixin } from '@vaadin/field-base/src/validate-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ValidateMixin } from '@scoped-vaadin/field-base/src/validate-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { DatePickerMixin } from './vaadin-date-picker-mixin.js';
 export { DatePickerDate, DatePickerI18n } from './vaadin-date-picker-mixin.js';
 
@@ -50,7 +50,7 @@
 }
 
 /**
- * `<vaadin-date-picker-light>` is a customizable version of the `<vaadin-date-picker>` providing
+ * `<vaadin24-date-picker-light>` is a customizable version of the `<vaadin24-date-picker>` providing
  * only the scrollable month calendar view and leaving the input field definition to the user.
  *
  * To create a custom input field, you need to add a child element which has a two-way
@@ -59,23 +59,23 @@
  * using an `<input>` element.
  *
  * ```html
- * <vaadin-date-picker-light attr-for-value="value">
+ * <vaadin24-date-picker-light attr-for-value="value">
  *   <input class="input">
- * </vaadin-date-picker-light>
+ * </vaadin24-date-picker-light>
  * ```
  *
  * ### Styling
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
- * In addition to `<vaadin-date-picker-light>` itself, the following
+ * In addition to `<vaadin24-date-picker-light>` itself, the following
  * internal components are themable:
  *
- * - `<vaadin-date-picker-overlay>`
- * - `<vaadin-date-picker-overlay-content>`
- * - `<vaadin-month-calendar>`
+ * - `<vaadin24-date-picker-overlay>`
+ * - `<vaadin24-date-picker-overlay-content>`
+ * - `<vaadin24-month-calendar>`
  *
- * Note: the `theme` attribute value set on `<vaadin-date-picker-light>`
+ * Note: the `theme` attribute value set on `<vaadin24-date-picker-light>`
  * is propagated to the internal themable components listed above.
  *
  * @fires {Event} change - Fired when the user commits a value change.
@@ -105,7 +105,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-date-picker-light': DatePickerLight;
+    'vaadin24-date-picker-light': DatePickerLight;
   }
 }
 
