Index: test/visual/material/checkbox.test.js
===================================================================
--- test/visual/material/checkbox.test.js
+++ test/visual/material/checkbox.test.js
@@ -10,7 +10,7 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-checkbox label="Checkbox"></vaadin-checkbox>', div);
+    element = fixtureSync('<vaadin24-checkbox label="Checkbox"></vaadin24-checkbox>', div);
   });
 
   it('basic', async () => {
