Index: src/vaadin-text-field.d.ts
===================================================================
--- src/vaadin-text-field.d.ts
+++ src/vaadin-text-field.d.ts
@@ -3,8 +3,8 @@
  * Copyright (c) 2017 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { TextFieldMixin } from './vaadin-text-field-mixin.js';
 
 /**
@@ -42,24 +42,24 @@
 }
 
 /**
- * `<vaadin-text-field>` is a web component that allows the user to input and edit text.
+ * `<vaadin24-text-field>` is a web component that allows the user to input and edit text.
  *
  * ```html
- * <vaadin-text-field label="First Name"></vaadin-text-field>
+ * <vaadin24-text-field label="First Name"></vaadin24-text-field>
  * ```
  *
  * ### Prefixes and suffixes
  *
- * These are child elements of a `<vaadin-text-field>` that are displayed
+ * These are child elements of a `<vaadin24-text-field>` that are displayed
  * inline with the input, before or after.
  * In order for an element to be considered as a prefix, it must have the slot
  * attribute set to `prefix` (and similarly for `suffix`).
  *
  * ```html
- * <vaadin-text-field label="Email address">
+ * <vaadin24-text-field label="Email address">
  *   <div slot="prefix">Sent to:</div>
  *   <div slot="suffix">@vaadin.com</div>
- * </vaadin-text-field>
+ * </vaadin24-text-field>
  * ```
  *
  * ### Styling
@@ -120,7 +120,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-text-field': TextField;
+    'vaadin24-text-field': TextField;
   }
 }
 
