Index: src/vaadin-text-field-mixin.js
===================================================================
--- src/vaadin-text-field-mixin.js
+++ src/vaadin-text-field-mixin.js
@@ -3,9 +3,9 @@
  * Copyright (c) 2021 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { InputController } from '@vaadin/field-base/src/input-controller.js';
-import { InputFieldMixin } from '@vaadin/field-base/src/input-field-mixin.js';
-import { LabelledInputController } from '@vaadin/field-base/src/labelled-input-controller.js';
+import { InputController } from '@scoped-vaadin/field-base/src/input-controller.js';
+import { InputFieldMixin } from '@scoped-vaadin/field-base/src/input-field-mixin.js';
+import { LabelledInputController } from '@scoped-vaadin/field-base/src/labelled-input-controller.js';
 
 /**
  * A mixin providing common text field functionality.
