Index: src/vaadin-text-field.js
===================================================================
--- src/vaadin-text-field.js
+++ src/vaadin-text-field.js
@@ -3,36 +3,36 @@
  * Copyright (c) 2017 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/src/vaadin-input-container.js';
+import '@scoped-vaadin/input-container/src/vaadin-input-container.js';
 import { html, PolymerElement } from '@polymer/polymer';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
-import { inputFieldShared } from '@vaadin/field-base/src/styles/input-field-shared-styles.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { TooltipController } from '@scoped-vaadin/component-base/src/tooltip-controller.js';
+import { inputFieldShared } from '@scoped-vaadin/field-base/src/styles/input-field-shared-styles.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { TextFieldMixin } from './vaadin-text-field-mixin.js';
 
-registerStyles('vaadin-text-field', inputFieldShared, { moduleId: 'vaadin-text-field-styles' });
+registerStyles('vaadin24-text-field', inputFieldShared, { moduleId: 'vaadin-text-field-styles' });
 
 /**
- * `<vaadin-text-field>` is a web component that allows the user to input and edit text.
+ * `<vaadin24-text-field>` is a web component that allows the user to input and edit text.
  *
  * ```html
- * <vaadin-text-field label="First Name"></vaadin-text-field>
+ * <vaadin24-text-field label="First Name"></vaadin24-text-field>
  * ```
  *
  * ### Prefixes and suffixes
  *
- * These are child elements of a `<vaadin-text-field>` that are displayed
+ * These are child elements of a `<vaadin24-text-field>` that are displayed
  * inline with the input, before or after.
  * In order for an element to be considered as a prefix, it must have the slot
  * attribute set to `prefix` (and similarly for `suffix`).
  *
  * ```html
- * <vaadin-text-field label="Email address">
+ * <vaadin24-text-field label="Email address">
  *   <div slot="prefix">Sent to:</div>
  *   <div slot="suffix">@vaadin.com</div>
- * </vaadin-text-field>
+ * </vaadin24-text-field>
  * ```
  *
  * ### Styling
@@ -85,7 +85,7 @@
  */
 export class TextField extends TextFieldMixin(ThemableMixin(ElementMixin(PolymerElement))) {
   static get is() {
-    return 'vaadin-text-field';
+    return 'vaadin24-text-field';
   }
 
   static get template() {
@@ -102,7 +102,7 @@
           <span part="required-indicator" aria-hidden="true" on-click="focus"></span>
         </div>
 
-        <vaadin-input-container
+        <vaadin24-input-container
           part="input-field"
           readonly="[[readonly]]"
           disabled="[[disabled]]"
@@ -113,7 +113,7 @@
           <slot name="input"></slot>
           <slot name="suffix" slot="suffix"></slot>
           <div id="clearButton" part="clear-button" slot="suffix" aria-hidden="true"></div>
-        </vaadin-input-container>
+        </vaadin24-input-container>
 
         <div part="helper-text">
           <slot name="helper"></slot>
