Index: src/vaadin-text-field.d.ts
===================================================================
--- src/vaadin-text-field.d.ts
+++ src/vaadin-text-field.d.ts
@@ -3,10 +3,10 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { InputFieldMixin } from '@vaadin/field-base/src/input-field-mixin.js';
-import { PatternMixin } from '@vaadin/field-base/src/pattern-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { InputFieldMixin } from '@scoped-vaadin/field-base/src/input-field-mixin.js';
+import { PatternMixin } from '@scoped-vaadin/field-base/src/pattern-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
  * Fired when the user commits a value change.
@@ -43,24 +43,24 @@
 }
 
 /**
- * `<vaadin-text-field>` is a web component that allows the user to input and edit text.
+ * `<vaadin23-text-field>` is a web component that allows the user to input and edit text.
  *
  * ```html
- * <vaadin-text-field label="First Name"></vaadin-text-field>
+ * <vaadin23-text-field label="First Name"></vaadin23-text-field>
  * ```
  *
  * ### Prefixes and suffixes
  *
- * These are child elements of a `<vaadin-text-field>` that are displayed
+ * These are child elements of a `<vaadin23-text-field>` that are displayed
  * inline with the input, before or after.
  * In order for an element to be considered as a prefix, it must have the slot
  * attribute set to `prefix` (and similarly for `suffix`).
  *
  * ```html
- * <vaadin-text-field label="Email address">
+ * <vaadin23-text-field label="Email address">
  *   <div slot="prefix">Sent to:</div>
  *   <div slot="suffix">@vaadin.com</div>
- * </vaadin-text-field>
+ * </vaadin23-text-field>
  * ```
  *
  * ### Styling
@@ -131,7 +131,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-text-field': TextField;
+    'vaadin23-text-field': TextField;
   }
 }
 
