Index: form-layout/theme/material/vaadin-form-item-styles.js
===================================================================
--- form-layout/theme/material/vaadin-form-item-styles.js	null
+++ form-layout/theme/material/vaadin-form-item-styles.js	null
@@ -1,9 +1,9 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-form-item',
+  'vaadin24-form-item',
   css`
     [part='label'] {
       font-family: var(--material-font-family);
