Index: theme/lumo/vaadin-upload-styles.d.ts
===================================================================
--- theme/lumo/vaadin-upload-styles.d.ts
+++ theme/lumo/vaadin-upload-styles.d.ts
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
-import '@vaadin/button/theme/lumo/vaadin-button-styles.js';
-import '@vaadin/progress-bar/theme/lumo/vaadin-progress-bar-styles.js';
+import '@scoped-vaadin/vaadin-lumo-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-lumo-styles/color.js';
+import '@scoped-vaadin/vaadin-lumo-styles/sizing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/spacing.js';
+import '@scoped-vaadin/vaadin-lumo-styles/style.js';
+import '@scoped-vaadin/vaadin-lumo-styles/typography.js';
+import '@scoped-vaadin/button/theme/lumo/vaadin-button-styles.js';
+import '@scoped-vaadin/progress-bar/theme/lumo/vaadin-progress-bar-styles.js';
