Index: test/list-box.test.js
===================================================================
--- test/list-box.test.js
+++ test/list-box.test.js
@@ -1,17 +1,17 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync } from '@vaadin/testing-helpers';
-import '@vaadin/item/vaadin-item.js';
+import '@scoped-vaadin/item/vaadin-item.js';
 import '../vaadin-list-box.js';
 
-describe('vaadin-list-box', () => {
+describe('vaadin24-list-box', () => {
   let listBox;
 
   beforeEach(() => {
     listBox = fixtureSync(`
-      <vaadin-list-box>
-        <vaadin-item>Foo</vaadin-item>
-        <vaadin-item>Bar</vaadin-item>
-      </vaadin-list-box>
+      <vaadin24-list-box>
+        <vaadin24-item>Foo</vaadin24-item>
+        <vaadin24-item>Bar</vaadin24-item>
+      </vaadin24-list-box>
     `);
   });
 
