Index: test/missing-import-polymer.test.js
===================================================================
--- test/missing-import-polymer.test.js
+++ test/missing-import-polymer.test.js
@@ -8,10 +8,10 @@
 
   beforeEach(() => {
     listBox = fixtureSync(`
-      <vaadin-list-box>
-        <vaadin-item>Foo</vaadin-item>
-        <vaadin-item>Bar</vaadin-item>
-      </vaadin-list-box>
+      <vaadin24-list-box>
+        <vaadin24-item>Foo</vaadin24-item>
+        <vaadin24-item>Bar</vaadin24-item>
+      </vaadin24-list-box>
     `);
 
     sinon.stub(console, 'warn');
@@ -28,7 +28,7 @@
   });
 
   it('should not warn after vaadin-item is imported', async () => {
-    await import('@vaadin/item/vaadin-item.js');
+    await import('@scoped-vaadin/item/vaadin-item.js');
     listBox._checkImport();
     expect(console.warn.called).to.be.false;
   });
