Index: theme/lumo/vaadin-email-field-styles.js
===================================================================
--- theme/lumo/vaadin-email-field-styles.js
+++ theme/lumo/vaadin-email-field-styles.js
@@ -3,8 +3,8 @@
  * Copyright (c) 2021 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { inputFieldShared } from '@vaadin/vaadin-lumo-styles/mixins/input-field-shared.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { inputFieldShared } from '@scoped-vaadin/vaadin-lumo-styles/mixins/input-field-shared.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const emailField = css`
   :host([dir='rtl']) [part='input-field'] ::slotted(input) {
@@ -16,6 +16,6 @@
   }
 `;
 
-registerStyles('vaadin-email-field', [inputFieldShared, emailField], {
+registerStyles('vaadin24-email-field', [inputFieldShared, emailField], {
   moduleId: 'lumo-email-field',
 });
