Index: src/lit/renderer-directives.d.ts
===================================================================
--- src/lit/renderer-directives.d.ts
+++ src/lit/renderer-directives.d.ts
@@ -5,8 +5,8 @@
  */
 /* eslint-disable max-classes-per-file */
 import type { DirectiveResult } from 'lit/directive.js';
-import type { LitRendererResult } from '@vaadin/lit-renderer';
-import { LitRendererDirective } from '@vaadin/lit-renderer';
+import type { LitRendererResult } from '@scoped-vaadin/lit-renderer';
+import { LitRendererDirective } from '@scoped-vaadin/lit-renderer';
 import type { Dialog } from '../vaadin-dialog.js';
 
 export type DialogLitRenderer = (dialog: Dialog) => LitRendererResult;
@@ -62,9 +62,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-dialog
+ * `<vaadin24-dialog
  *   ${dialogRenderer((dialog) => html`...`)}
- * ></vaadin-dialog>`
+ * ></vaadin24-dialog>`
  * ```
  *
  * @param renderer the renderer callback that returns a Lit template.
@@ -93,9 +93,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-dialog
+ * `<vaadin24-dialog
  *   ${dialogHeaderRenderer((dialog) => html`...`)}
- * ></vaadin-dialog>`
+ * ></vaadin24-dialog>`
  * ```
  *
  * @param renderer the renderer callback.
@@ -124,9 +124,9 @@
  *
  * Example of usage:
  * ```js
- * `<vaadin-dialog
+ * `<vaadin24-dialog
  *   ${dialogFooterRenderer((dialog) => html`...`)}
- * ></vaadin-dialog>`
+ * ></vaadin24-dialog>`
  * ```
  *
  * @param renderer the renderer callback.
