Index: src/vaadin-tab-mixin.js
===================================================================
--- src/vaadin-tab-mixin.js
+++ src/vaadin-tab-mixin.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ItemMixin } from '@vaadin/item/src/vaadin-item-mixin.js';
+import { ItemMixin } from '@scoped-vaadin/item/src/vaadin-item-mixin.js';
 
 /**
  * @polymerMixin
