Index: src/vaadin-tabs-mixin.js
===================================================================
--- src/vaadin-tabs-mixin.js
+++ src/vaadin-tabs-mixin.js
@@ -3,9 +3,9 @@
  * Copyright (c) 2017 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ListMixin } from '@vaadin/a11y-base/src/list-mixin.js';
-import { getNormalizedScrollLeft } from '@vaadin/component-base/src/dir-utils.js';
-import { ResizeMixin } from '@vaadin/component-base/src/resize-mixin.js';
+import { ListMixin } from '@scoped-vaadin/a11y-base/src/list-mixin.js';
+import { getNormalizedScrollLeft } from '@scoped-vaadin/component-base/src/dir-utils.js';
+import { ResizeMixin } from '@scoped-vaadin/component-base/src/resize-mixin.js';
 
 /**
  * @polymerMixin
