Index: src/vaadin-tabs.d.ts
===================================================================
--- src/vaadin-tabs.d.ts
+++ src/vaadin-tabs.d.ts
@@ -3,8 +3,8 @@
  * Copyright (c) 2017 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { TabsMixin, TabsOrientation } from './vaadin-tabs-mixin.js';
 
 export { TabsOrientation };
@@ -28,15 +28,15 @@
 export interface TabsEventMap extends HTMLElementEventMap, TabsCustomEventMap {}
 
 /**
- * `<vaadin-tabs>` is a Web Component for organizing and grouping content into sections.
+ * `<vaadin24-tabs>` is a Web Component for organizing and grouping content into sections.
  *
  * ```
- *   <vaadin-tabs selected="4">
- *     <vaadin-tab>Page 1</vaadin-tab>
- *     <vaadin-tab>Page 2</vaadin-tab>
- *     <vaadin-tab>Page 3</vaadin-tab>
- *     <vaadin-tab>Page 4</vaadin-tab>
- *   </vaadin-tabs>
+ *   <vaadin24-tabs selected="4">
+ *     <vaadin24-tab>Page 1</vaadin24-tab>
+ *     <vaadin24-tab>Page 2</vaadin24-tab>
+ *     <vaadin24-tab>Page 3</vaadin24-tab>
+ *     <vaadin24-tab>Page 4</vaadin24-tab>
+ *   </vaadin24-tabs>
  * ```
  *
  * ### Styling
@@ -77,7 +77,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-tabs': Tabs;
+    'vaadin24-tabs': Tabs;
   }
 }
 
