Index: src/vaadin-lit-tabs.js
===================================================================
--- src/vaadin-lit-tabs.js
+++ src/vaadin-lit-tabs.js
@@ -5,15 +5,15 @@
  */
 import './vaadin-lit-tab.js';
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { TabsMixin } from './vaadin-tabs-mixin.js';
 import { tabsStyles } from './vaadin-tabs-styles.js';
 
 /**
- * LitElement based version of `<vaadin-tabs>` web component.
+ * LitElement based version of `<vaadin24-tabs>` web component.
  *
  * ## Disclaimer
  *
@@ -28,7 +28,7 @@
  */
 class Tabs extends TabsMixin(ElementMixin(ThemableMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-tabs';
+    return 'vaadin24-tabs';
   }
 
   static get styles() {
