Index: test/visual/lumo/tooltip.test.js
===================================================================
--- test/visual/lumo/tooltip.test.js
+++ test/visual/lumo/tooltip.test.js
@@ -1,6 +1,6 @@
 import { fire, fixtureSync } from '@vaadin/testing-helpers';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/vaadin-lumo-styles/test/autoload.js';
+import '@scoped-vaadin/vaadin-lumo-styles/test/autoload.js';
 import '../../not-animated-styles.js';
 import '../../../theme/lumo/vaadin-tooltip.js';
 import { Tooltip } from '../../../src/vaadin-tooltip.js';
@@ -15,7 +15,7 @@
   });
 
   beforeEach(() => {
-    element = fixtureSync('<vaadin-tooltip text="tooltip"></vaadin-tooltip>');
+    element = fixtureSync('<vaadin24-tooltip text="tooltip"></vaadin24-tooltip>');
     div = fixtureSync(`
       <div style="display: flex; width: 300px; height: 300px; justify-content: center; align-items: center">
         <div style="width: 100px; height: 100px; outline: 1px solid red;"></div>
@@ -70,7 +70,7 @@
   it('white-space-pre', async () => {
     element.text = 'Line 1\n\nLine 2';
     fire(target, 'mouseenter');
-    const overlay = document.querySelector('vaadin-tooltip-overlay');
+    const overlay = document.querySelector('vaadin24-tooltip-overlay');
     const content = overlay.shadowRoot.querySelector('[part="content"]');
     content.style.whiteSpace = 'pre';
     await visualDiff(div, 'white-space-pre');
