Index: test/visual/lumo/text-field.test.js
===================================================================
--- test/visual/lumo/text-field.test.js
+++ test/visual/lumo/text-field.test.js
@@ -1,7 +1,7 @@
 import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
 import { sendKeys } from '@web/test-runner-commands';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/vaadin-lumo-styles/test/autoload.js';
+import '@scoped-vaadin/vaadin-lumo-styles/test/autoload.js';
 import '../common.js';
 import '../../../theme/lumo/vaadin-text-field.js';
 
@@ -12,7 +12,7 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-text-field></vaadin-text-field>', div);
+    element = fixtureSync('<vaadin24-text-field></vaadin24-text-field>', div);
   });
 
   it('basic', async () => {
@@ -105,7 +105,7 @@
     let field;
 
     beforeEach(() => {
-      field = document.createElement('vaadin-text-field');
+      field = document.createElement('vaadin24-text-field');
       field.label = 'Label';
       field.style.marginLeft = '10px';
       element.parentNode.appendChild(field);
