Index: test/horizontal-layout.test.js
===================================================================
--- test/horizontal-layout.test.js
+++ test/horizontal-layout.test.js
@@ -1,22 +1,23 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import '../src/vaadin-horizontal-layout.js';
 
-describe('vaadin-horizontal-layout', () => {
+describe('vaadin24-horizontal-layout', () => {
   describe('basic features', () => {
     let layout, tagName;
 
     beforeEach(() => {
-      layout = fixtureSync('<vaadin-horizontal-layout></vaadin-horizontal-layout>');
+      layout = fixtureSync('<vaadin24-horizontal-layout></vaadin24-horizontal-layout>');
       tagName = layout.tagName.toLowerCase();
     });
 
     it('should be defined in custom element registry', () => {
-      expect(customElements.get(tagName)).to.be.ok;
+      expect(internalCustomElements.get(tagName)).to.be.ok;
     });
 
     it('should have a valid static "is" getter', () => {
-      expect(customElements.get(tagName).is).to.equal(tagName);
+      expect(internalCustomElements.get(tagName).is).to.equal(tagName);
     });
 
     it('should have a box-sizing set to border-box', () => {
@@ -30,9 +31,9 @@
     beforeEach(() => {
       wrapper = fixtureSync(`
         <div>
-          <vaadin-horizontal-layout>
+          <vaadin24-horizontal-layout>
             <div>Content</div>
-          </vaadin-horizontal-layout>
+          </vaadin24-horizontal-layout>
         </div>
       `);
       layout = wrapper.firstElementChild;
@@ -52,10 +53,10 @@
     describe('flex', () => {
       beforeEach(async () => {
         layout = fixtureSync(`
-          <vaadin-horizontal-layout>
+          <vaadin24-horizontal-layout>
             <div></div>
             <div data-width-full></div>
-          </vaadin-horizontal-layout>
+          </vaadin24-horizontal-layout>
         `);
         children = Array.from(layout.children);
         await nextFrame();
@@ -71,16 +72,16 @@
     describe('min-width', () => {
       beforeEach(async () => {
         layout = fixtureSync(`
-          <vaadin-horizontal-layout>
+          <vaadin24-horizontal-layout>
             <div></div>
             <div data-width-full></div>
-            <vaadin-button></vaadin-button>
-            <vaadin-button data-width-full></vaadin-button>
-            <vaadin-horizontal-layout></vaadin-horizontal-layout>
-            <vaadin-horizontal-layout data-width-full></vaadin-horizontal-layout>
-            <vaadin-vertical-layout></vaadin-vertical-layout>
-            <vaadin-vertical-layout data-width-full></vaadin-vertical-layout>
-          </vaadin-horizontal-layout>
+            <vaadin24-button></vaadin24-button>
+            <vaadin24-button data-width-full></vaadin24-button>
+            <vaadin24-horizontal-layout></vaadin24-horizontal-layout>
+            <vaadin24-horizontal-layout data-width-full></vaadin24-horizontal-layout>
+            <vaadin24-vertical-layout></vaadin24-vertical-layout>
+            <vaadin24-vertical-layout data-width-full></vaadin24-vertical-layout>
+          </vaadin24-horizontal-layout>
         `);
         children = Array.from(layout.children);
         await nextFrame();
@@ -98,7 +99,7 @@
     let layout;
 
     beforeEach(() => {
-      layout = fixtureSync('<vaadin-horizontal-layout></vaadin-horizontal-layout>');
+      layout = fixtureSync('<vaadin24-horizontal-layout></vaadin24-horizontal-layout>');
     });
 
     describe('start', () => {
