Index: theme/material/vaadin-email-field-styles.js
===================================================================
--- theme/material/vaadin-email-field-styles.js
+++ theme/material/vaadin-email-field-styles.js
@@ -3,9 +3,9 @@
  * Copyright (c) 2021 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { inputFieldShared } from '@vaadin/vaadin-material-styles/mixins/input-field-shared.js';
-import { registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { inputFieldShared } from '@scoped-vaadin/vaadin-material-styles/mixins/input-field-shared.js';
+import { registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
-registerStyles('vaadin-email-field', inputFieldShared, {
+registerStyles('vaadin23-email-field', inputFieldShared, {
   moduleId: 'material-email-field',
 });
