Index: theme/material/vaadin-combo-box-overlay-styles.d.ts
===================================================================
--- theme/material/vaadin-combo-box-overlay-styles.d.ts
+++ theme/material/vaadin-combo-box-overlay-styles.d.ts
@@ -1,4 +1,4 @@
-import '@vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
 declare const comboBoxOverlay: import("lit").CSSResult;
 declare const comboBoxLoader: import("lit").CSSResult;
 export { comboBoxLoader, comboBoxOverlay };
