Index: test/typings/popover.types.ts
===================================================================
--- test/typings/popover.types.ts
+++ test/typings/popover.types.ts
@@ -1,6 +1,6 @@
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { OverlayClassMixinClass } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import type { ThemePropertyMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { OverlayClassMixinClass } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import type { ThemePropertyMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 import type { PopoverPositionMixinClass } from '../../src/vaadin-popover-position-mixin.js';
 import type { PopoverTargetMixinClass } from '../../src/vaadin-popover-target-mixin.js';
 import type {
@@ -13,7 +13,7 @@
 
 const assertType = <TExpected>(actual: TExpected) => actual;
 
-const popover = document.createElement('vaadin-popover');
+const popover = document.createElement('vaadin24-popover');
 
 // Mixins
 assertType<ElementMixinClass>(popover);
