Index: theme/material/vaadin-lit-confirm-dialog.js
===================================================================
--- theme/material/vaadin-lit-confirm-dialog.js
+++ theme/material/vaadin-lit-confirm-dialog.js
@@ -1,4 +1,4 @@
-import '@vaadin/dialog/theme/material/vaadin-dialog-styles.js';
-import '@vaadin/button/theme/material/vaadin-button-styles.js';
+import '@scoped-vaadin/dialog/theme/material/vaadin-dialog-styles.js';
+import '@scoped-vaadin/button/theme/material/vaadin-button-styles.js';
 import './vaadin-confirm-dialog-styles.js';
 import '../../src/vaadin-lit-confirm-dialog.js';
