Index: theme/material/vaadin-lit-confirm-dialog.d.ts
===================================================================
--- theme/material/vaadin-lit-confirm-dialog.d.ts
+++ theme/material/vaadin-lit-confirm-dialog.d.ts
@@ -1,3 +1,3 @@
-import '@vaadin/button/theme/material/vaadin-lit-button.js';
+import '@scoped-vaadin/button/theme/material/vaadin-lit-button.js';
 import './vaadin-confirm-dialog-styles.js';
 import '../../src/vaadin-lit-confirm-dialog.js';
