Index: theme/material/vaadin-lit-date-time-picker.js
===================================================================
--- theme/material/vaadin-lit-date-time-picker.js
+++ theme/material/vaadin-lit-date-time-picker.js
@@ -1,4 +1,4 @@
-import '@vaadin/date-picker/theme/material/vaadin-lit-date-picker.js';
-import '@vaadin/time-picker/theme/material/vaadin-lit-time-picker.js';
+import '@scoped-vaadin/date-picker/theme/material/vaadin-lit-date-picker.js';
+import '@scoped-vaadin/time-picker/theme/material/vaadin-lit-time-picker.js';
 import './vaadin-date-time-picker-styles.js';
 import '../../src/vaadin-lit-date-time-picker.js';
