Index: test/visual/lumo/password-field.test.js
===================================================================
--- test/visual/lumo/password-field.test.js
+++ test/visual/lumo/password-field.test.js
@@ -10,7 +10,7 @@
     div = document.createElement('div');
     div.style.display = 'inline-block';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-password-field></vaadin-password-field>', div);
+    element = fixtureSync('<vaadin23-password-field></vaadin23-password-field>', div);
   });
 
   ['ltr', 'rtl'].forEach((dir) => {
