Index: virtual-list/src/vaadin-virtual-list-mixin.js
===================================================================
--- virtual-list/src/vaadin-virtual-list-mixin.js
+++ virtual-list/src/vaadin-virtual-list-mixin.js
@@ -3,10 +3,10 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
-import { OverflowController } from '@vaadin/component-base/src/overflow-controller.js';
-import { processTemplates } from '@vaadin/component-base/src/templates.js';
-import { Virtualizer } from '@vaadin/component-base/src/virtualizer.js';
+import { ControllerMixin } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import { OverflowController } from '@scoped-vaadin/component-base/src/overflow-controller.js';
+import { processTemplates } from '@scoped-vaadin/component-base/src/templates.js';
+import { Virtualizer } from '@scoped-vaadin/component-base/src/virtualizer.js';
 
 /**
  * @polymerMixin
@@ -27,7 +27,7 @@
          * Receives three arguments:
          *
          * - `root` The render target element representing one item at a time.
-         * - `virtualList` The reference to the `<vaadin-virtual-list>` element.
+         * - `virtualList` The reference to the `<vaadin24-virtual-list>` element.
          * - `model` The object with the properties related with the rendered
          *   item, contains:
          *   - `model.index` The index of the rendered item.
