Index: src/vaadin-lit-virtual-list.js
===================================================================
--- src/vaadin-lit-virtual-list.js
+++ src/vaadin-lit-virtual-list.js
@@ -4,15 +4,15 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { VirtualListMixin } from './vaadin-virtual-list-mixin.js';
 import { virtualListStyles } from './vaadin-virtual-list-styles.js';
 
 /**
- * LitElement based version of `<vaadin-virtual-list>` web component.
+ * LitElement based version of `<vaadin24-virtual-list>` web component.
  *
  * ## Disclaimer
  *
@@ -22,7 +22,7 @@
  */
 class VirtualList extends VirtualListMixin(ThemableMixin(ElementMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-virtual-list';
+    return 'vaadin24-virtual-list';
   }
 
   static get styles() {
