Index: src/vaadin-avatar-group-mixin.js
===================================================================
--- src/vaadin-avatar-group-mixin.js
+++ src/vaadin-avatar-group-mixin.js
@@ -5,11 +5,11 @@
  */
 import { html, render } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { announce } from '@vaadin/a11y-base/src/announce.js';
-import { I18nMixin } from '@vaadin/component-base/src/i18n-mixin.js';
-import { OverlayClassMixin } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import { ResizeMixin } from '@vaadin/component-base/src/resize-mixin.js';
-import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
+import { announce } from '@scoped-vaadin/a11y-base/src/announce.js';
+import { I18nMixin } from '@scoped-vaadin/component-base/src/i18n-mixin.js';
+import { OverlayClassMixin } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import { ResizeMixin } from '@scoped-vaadin/component-base/src/resize-mixin.js';
+import { SlotController } from '@scoped-vaadin/component-base/src/slot-controller.js';
 
 const MINIMUM_DISPLAYED_AVATARS = 2;
 
@@ -171,7 +171,7 @@
     ready() {
       super.ready();
 
-      this._overflowController = new SlotController(this, 'overflow', 'vaadin-avatar', {
+      this._overflowController = new SlotController(this, 'overflow', 'vaadin24-avatar', {
         initializer: (overflow) => {
           overflow.setAttribute('role', 'button');
           overflow.setAttribute('aria-haspopup', 'menu');
@@ -179,7 +179,7 @@
           overflow.addEventListener('click', (e) => this._onOverflowClick(e));
           overflow.addEventListener('keydown', (e) => this._onOverflowKeyDown(e));
 
-          const tooltip = document.createElement('vaadin-tooltip');
+          const tooltip = document.createElement('vaadin24-tooltip');
           tooltip.setAttribute('slot', 'tooltip');
           overflow.appendChild(tooltip);
 
@@ -215,7 +215,7 @@
     __overlayRenderer(root) {
       let menu = root.firstElementChild;
       if (!menu) {
-        menu = document.createElement('vaadin-avatar-group-menu');
+        menu = document.createElement('vaadin24-avatar-group-menu');
         menu.addEventListener('keydown', (event) => this._onListKeyDown(event));
         root.appendChild(menu);
       }
@@ -233,9 +233,9 @@
 
     /** @private */
     __createItemElement(item) {
-      const itemElement = document.createElement('vaadin-avatar-group-menu-item');
+      const itemElement = document.createElement('vaadin24-avatar-group-menu-item');
 
-      const avatar = document.createElement('vaadin-avatar');
+      const avatar = document.createElement('vaadin24-avatar');
       itemElement.appendChild(avatar);
 
       avatar.setAttribute('aria-hidden', 'true');
@@ -317,7 +317,7 @@
         html`
           ${items.map(
             (item) => html`
-              <vaadin-avatar
+              <vaadin24-avatar
                 .name="${item.name}"
                 .abbr="${item.abbr}"
                 .img="${item.img}"
@@ -325,7 +325,7 @@
                 .i18n="${this.__effectiveI18n}"
                 class="${ifDefined(item.className)}"
                 with-tooltip
-              ></vaadin-avatar>
+              ></vaadin24-avatar>
             `,
           )}
         `,
@@ -344,7 +344,7 @@
 
       this.__renderAvatars(limit ? items.slice(0, limit) : items);
 
-      this._avatars = [...this.querySelectorAll('vaadin-avatar')];
+      this._avatars = [...this.querySelectorAll('vaadin24-avatar')];
     }
 
     /** @private */
@@ -481,7 +481,7 @@
 
       if (opened) {
         if (!this._menuElement) {
-          this._menuElement = this.$.overlay.querySelector('vaadin-avatar-group-menu');
+          this._menuElement = this.$.overlay.querySelector('vaadin24-avatar-group-menu');
         }
 
         this._openedWithFocusRing = overflow.hasAttribute('focus-ring');
