Index: split-layout/theme/material/vaadin-split-layout-styles.js
===================================================================
--- split-layout/theme/material/vaadin-split-layout-styles.js
+++ split-layout/theme/material/vaadin-split-layout-styles.js
@@ -1,8 +1,8 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-split-layout',
+  'vaadin24-split-layout',
   css`
     [part='splitter'] {
       min-width: 8px;
