Index: theme/material/vaadin-message-styles.js
===================================================================
--- theme/material/vaadin-message-styles.js
+++ theme/material/vaadin-message-styles.js
@@ -1,10 +1,10 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/typography.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
 import './vaadin-message-avatar-styles.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
-  'vaadin-message',
+  'vaadin23-message',
   css`
     :host {
       color: var(--material-body-text-color);
