Index: test/visual/lumo/context-menu.test.js
===================================================================
--- test/visual/lumo/context-menu.test.js
+++ test/visual/lumo/context-menu.test.js
@@ -1,7 +1,7 @@
 import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/item/theme/lumo/vaadin-item.js';
-import '@vaadin/list-box/theme/lumo/vaadin-list-box.js';
+import '@scoped-vaadin/item/theme/lumo/vaadin-item.js';
+import '@scoped-vaadin/list-box/theme/lumo/vaadin-list-box.js';
 import '../../../theme/lumo/vaadin-context-menu.js';
 import '../../not-animated-styles.js';
 import { openSubMenus } from '../../helpers.js';
@@ -35,16 +35,16 @@
       describe('basic', () => {
         beforeEach(() => {
           element = fixtureSync(`
-            <vaadin-context-menu>
+            <vaadin24-context-menu>
               <div style="padding: 10px">Target</div>
-            </vaadin-context-menu>
+            </vaadin24-context-menu>
           `);
           element.renderer = (root) => {
             root.innerHTML = `
-              <vaadin-list-box>
-                <vaadin-item>Item 1</vaadin-item>
-                <vaadin-item>Item 2</vaadin-item>
-              </vaadin-list-box>
+              <vaadin24-list-box>
+                <vaadin24-item>Item 1</vaadin24-item>
+                <vaadin24-item>Item 2</vaadin24-item>
+              </vaadin24-list-box>
             `;
           };
         });
@@ -58,18 +58,18 @@
       describe('long', () => {
         beforeEach(() => {
           element = fixtureSync(`
-            <vaadin-context-menu>
+            <vaadin24-context-menu>
               <div style="padding: 10px">Target</div>
-            </vaadin-context-menu>
+            </vaadin24-context-menu>
           `);
           element.renderer = (root) => {
             root.innerHTML = `
-              <vaadin-list-box>
+              <vaadin24-list-box>
                 ${new Array(30)
                   .fill(0)
-                  .map((_, idx) => `<vaadin-item>Item ${idx}</vaadin-item>`)
+                  .map((_, idx) => `<vaadin24-item>Item ${idx}</vaadin24-item>`)
                   .join('')}
-              </vaadin-list-box>
+              </vaadin24-list-box>
             `;
           };
         });
@@ -91,9 +91,9 @@
       describe('items', () => {
         beforeEach(() => {
           element = fixtureSync(`
-            <vaadin-context-menu>
+            <vaadin24-context-menu>
               <div style="padding: 10px">Target</div>
-            </vaadin-context-menu>
+            </vaadin24-context-menu>
           `);
         });
 
