Index: test/typings/avatar-group.types.ts
===================================================================
--- test/typings/avatar-group.types.ts
+++ test/typings/avatar-group.types.ts
@@ -1,14 +1,14 @@
 import '../../vaadin-avatar-group.js';
-import type { ControllerMixinClass } from '@vaadin/component-base/src/controller-mixin.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { OverlayClassMixinClass } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import type { ResizeMixinClass } from '@vaadin/component-base/src/resize-mixin.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import type { ControllerMixinClass } from '@scoped-vaadin/component-base/src/controller-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { OverlayClassMixinClass } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import type { ResizeMixinClass } from '@scoped-vaadin/component-base/src/resize-mixin.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import type { AvatarGroupI18n, AvatarGroupItem } from '../../vaadin-avatar-group.js';
 
 const assertType = <TExpected>(value: TExpected) => value;
 
-const group = document.createElement('vaadin-avatar-group');
+const group = document.createElement('vaadin24-avatar-group');
 
 // Properties
 assertType<AvatarGroupItem[] | undefined>(group.items);
