Index: src/vaadin-app-layout-mixin.js
===================================================================
--- src/vaadin-app-layout-mixin.js
+++ src/vaadin-app-layout-mixin.js
@@ -3,11 +3,11 @@
  * Copyright (c) 2018 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { AriaModalController } from '@vaadin/a11y-base/src/aria-modal-controller.js';
-import { FocusTrapController } from '@vaadin/a11y-base/src/focus-trap-controller.js';
-import { animationFrame } from '@vaadin/component-base/src/async.js';
-import { Debouncer } from '@vaadin/component-base/src/debounce.js';
-import { I18nMixin } from '@vaadin/component-base/src/i18n-mixin.js';
+import { AriaModalController } from '@scoped-vaadin/a11y-base/src/aria-modal-controller.js';
+import { FocusTrapController } from '@scoped-vaadin/a11y-base/src/focus-trap-controller.js';
+import { animationFrame } from '@scoped-vaadin/component-base/src/async.js';
+import { Debouncer } from '@scoped-vaadin/component-base/src/debounce.js';
+import { I18nMixin } from '@scoped-vaadin/component-base/src/i18n-mixin.js';
 
 /**
  * @typedef {import('./vaadin-app-layout.js').AppLayoutI18n} AppLayoutI18n
@@ -132,7 +132,7 @@
 
       // Hide all the elements except the drawer toggle and drawer content
       this.__ariaModalController = new AriaModalController(this, () => [
-        ...this.querySelectorAll('vaadin-drawer-toggle, [slot="drawer"]'),
+        ...this.querySelectorAll('vaadin24-drawer-toggle, [slot="drawer"]'),
       ]);
       this.__focusTrapController = new FocusTrapController(this);
     }
@@ -291,7 +291,7 @@
 
     /** @private */
     __setAriaExpanded() {
-      const toggle = this.querySelector('vaadin-drawer-toggle');
+      const toggle = this.querySelector('vaadin24-drawer-toggle');
       if (toggle) {
         toggle.setAttribute('aria-expanded', this.drawerOpened);
       }
@@ -429,7 +429,7 @@
       this.__restoreFocus();
 
       // Move focus to the drawer toggle when closing the drawer.
-      const toggle = this.querySelector('vaadin-drawer-toggle');
+      const toggle = this.querySelector('vaadin24-drawer-toggle');
       if (toggle) {
         toggle.focus();
         toggle.setAttribute('focus-ring', 'focus');
