Index: app-layout/src/vaadin-drawer-toggle.d.ts
===================================================================
--- app-layout/src/vaadin-drawer-toggle.d.ts
+++ app-layout/src/vaadin-drawer-toggle.d.ts
@@ -3,15 +3,15 @@
  * Copyright (c) 2018 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { Button } from '@vaadin/button/src/vaadin-button.js';
+import { Button } from '@scoped-vaadin/button/src/vaadin-button.js';
 
 /**
  * The Drawer Toggle component controls the drawer in App Layout component.
  *
  * ```
- * <vaadin-app-layout>
- *   <vaadin-drawer-toggle slot="navbar">Toggle drawer</vaadin-drawer-toggle>
- * </vaadin-app-layout>
+ * <vaadin24-app-layout>
+ *   <vaadin24-drawer-toggle slot="navbar">Toggle drawer</vaadin24-drawer-toggle>
+ * </vaadin24-app-layout>
  * ```
  */
 declare class DrawerToggle extends Button {
@@ -20,7 +20,7 @@
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-drawer-toggle': DrawerToggle;
+    'vaadin24-drawer-toggle': DrawerToggle;
   }
 }
 
