Index: src/vaadin-drawer-toggle.js
===================================================================
--- src/vaadin-drawer-toggle.js
+++ src/vaadin-drawer-toggle.js
@@ -4,23 +4,23 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { buttonStyles } from '@vaadin/button/src/vaadin-button-core-styles.js';
-import { ButtonMixin } from '@vaadin/button/src/vaadin-button-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { isEmptyTextNode } from '@vaadin/component-base/src/dom-utils.js';
-import { registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { buttonStyles } from '@scoped-vaadin/button/src/vaadin-button-core-styles.js';
+import { ButtonMixin } from '@scoped-vaadin/button/src/vaadin-button-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { isEmptyTextNode } from '@scoped-vaadin/component-base/src/dom-utils.js';
+import { registerStyles, ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { drawerToggle } from './vaadin-drawer-toggle-styles.js';
 
-registerStyles('vaadin-drawer-toggle', [buttonStyles, drawerToggle], { moduleId: 'vaadin-drawer-toggle-styles' });
+registerStyles('vaadin24-drawer-toggle', [buttonStyles, drawerToggle], { moduleId: 'vaadin-drawer-toggle-styles' });
 
 /**
  * The Drawer Toggle component controls the drawer in App Layout component.
  *
  * ```
- * <vaadin-app-layout>
- *   <vaadin-drawer-toggle slot="navbar">Toggle drawer</vaadin-drawer-toggle>
- * </vaadin-app-layout>
+ * <vaadin24-app-layout>
+ *   <vaadin24-drawer-toggle slot="navbar">Toggle drawer</vaadin24-drawer-toggle>
+ * </vaadin24-app-layout>
  * ```
  *
  * @customElement
@@ -40,7 +40,7 @@
   }
 
   static get is() {
-    return 'vaadin-drawer-toggle';
+    return 'vaadin24-drawer-toggle';
   }
 
   static get properties() {
