Index: src/vaadin-lit-drawer-toggle.js
===================================================================
--- src/vaadin-lit-drawer-toggle.js
+++ src/vaadin-lit-drawer-toggle.js
@@ -4,17 +4,17 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { html, LitElement } from 'lit';
-import { buttonStyles } from '@vaadin/button/src/vaadin-button-core-styles.js';
-import { ButtonMixin } from '@vaadin/button/src/vaadin-button-mixin.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { isEmptyTextNode } from '@vaadin/component-base/src/dom-utils.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { buttonStyles } from '@scoped-vaadin/button/src/vaadin-button-core-styles.js';
+import { ButtonMixin } from '@scoped-vaadin/button/src/vaadin-button-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { isEmptyTextNode } from '@scoped-vaadin/component-base/src/dom-utils.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { drawerToggle } from './vaadin-drawer-toggle-styles.js';
 
 /**
- * LitElement based version of `<vaadin-drawer-toggle>` web component.
+ * LitElement based version of `<vaadin24-drawer-toggle>` web component.
  *
  * ## Disclaimer
  *
@@ -24,7 +24,7 @@
  */
 class DrawerToggle extends ButtonMixin(DirMixin(ThemableMixin(PolylitMixin(LitElement)))) {
   static get is() {
-    return 'vaadin-drawer-toggle';
+    return 'vaadin24-drawer-toggle';
   }
 
   static get styles() {
