Index: test/validation.common.js
===================================================================
--- test/validation.common.js
+++ test/validation.common.js
@@ -8,7 +8,7 @@
 
   describe('initial', () => {
     beforeEach(() => {
-      checkbox = document.createElement('vaadin-checkbox');
+      checkbox = document.createElement('vaadin24-checkbox');
       validateSpy = sinon.spy(checkbox, 'validate');
     });
 
@@ -42,7 +42,7 @@
     beforeEach(async () => {
       checkbox = fixtureSync(
         `<div>
-          <vaadin-checkbox label="Checkbox"></vaadin-checkbox>
+          <vaadin24-checkbox label="Checkbox"></vaadin24-checkbox>
           <input id="last-global-focusable" />
         </div>`,
       ).firstElementChild;
@@ -119,7 +119,7 @@
 
   describe('required', () => {
     beforeEach(async () => {
-      checkbox = fixtureSync('<vaadin-checkbox label="Checkbox" required></vaadin-checkbox>');
+      checkbox = fixtureSync('<vaadin24-checkbox label="Checkbox" required></vaadin24-checkbox>');
       await nextFrame();
     });
 
