Index: test/typings/lit-grid.types.ts
===================================================================
--- test/typings/lit-grid.types.ts
+++ test/typings/lit-grid.types.ts
@@ -1,6 +1,6 @@
-import type { DisabledMixinClass } from '@vaadin/a11y-base/src/disabled-mixin.js';
-import type { ElementMixinClass } from '@vaadin/component-base/src/element-mixin.js';
-import type { ThemableMixinClass } from '@vaadin/vaadin-themable-mixin';
+import type { DisabledMixinClass } from '@scoped-vaadin/a11y-base/src/disabled-mixin.js';
+import type { ElementMixinClass } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import type { ThemableMixinClass } from '@scoped-vaadin/vaadin-themable-mixin';
 import type {
   GridColumnGroup,
   GridFilter,
@@ -66,7 +66,7 @@
 const assertType = <TExpected>(actual: TExpected) => actual;
 
 /* Grid */
-const genericGrid = document.createElement('vaadin-grid');
+const genericGrid = document.createElement('vaadin24-grid');
 assertType<Grid>(genericGrid);
 
 const narrowedGrid = genericGrid as Grid<TestGridItem>;
@@ -213,7 +213,7 @@
 assertType<() => void>(narrowedGrid.requestContentUpdate);
 
 /* GridColumn */
-const genericColumn = document.createElement('vaadin-grid-column');
+const genericColumn = document.createElement('vaadin24-grid-column');
 assertType<GridColumn>(genericColumn);
 
 const bodyRenderer: GridBodyRenderer<TestGridItem> = (root, column, model) => {
@@ -252,7 +252,7 @@
 assertType<string | null | undefined>(narrowedColumn.footerPartName);
 
 /* GridColumnGroup */
-const genericColumnGroup = document.createElement('vaadin-grid-column-group');
+const genericColumnGroup = document.createElement('vaadin24-grid-column-group');
 assertType<GridColumnGroup>(genericColumnGroup);
 
 const narrowedColumnGroup = genericColumnGroup as GridColumnGroup<TestGridItem>;
@@ -267,7 +267,7 @@
 };
 
 /* GridFilter */
-const filter = document.createElement('vaadin-grid-filter');
+const filter = document.createElement('vaadin24-grid-filter');
 assertType<GridFilter>(filter);
 assertType<HTMLElement>(filter);
 assertType<string | null | undefined>(filter.path);
@@ -279,7 +279,7 @@
 });
 
 /* GridFilterColumn */
-const genericFilterColumn = document.createElement('vaadin-grid-filter-column');
+const genericFilterColumn = document.createElement('vaadin24-grid-filter-column');
 assertType<GridFilterColumn>(genericFilterColumn);
 
 const narrowedFilterColumn = genericFilterColumn as GridFilterColumn<TestGridItem>;
@@ -288,7 +288,7 @@
 assertType<string | null | undefined>(narrowedFilterColumn.path);
 
 /* GridSelectionColumn */
-const genericSelectionColumn = document.createElement('vaadin-grid-selection-column');
+const genericSelectionColumn = document.createElement('vaadin24-grid-selection-column');
 assertType<GridSelectionColumn>(genericSelectionColumn);
 
 const narrowedSelectionColumn = genericSelectionColumn as GridSelectionColumn<TestGridItem>;
@@ -306,7 +306,7 @@
 assertType<boolean>(narrowedSelectionColumn.autoSelect);
 
 /* GridSortColumn */
-const genericSortColumn = document.createElement('vaadin-grid-sort-column');
+const genericSortColumn = document.createElement('vaadin24-grid-sort-column');
 assertType<GridSortColumn>(genericSortColumn);
 
 const narrowedSortColumn = genericSortColumn as GridSortColumn<TestGridItem>;
@@ -320,7 +320,7 @@
 });
 
 /* GridSorter */
-const sorter = document.createElement('vaadin-grid-sorter');
+const sorter = document.createElement('vaadin24-grid-sorter');
 assertType<GridSorter>(sorter);
 
 sorter.addEventListener('sorter-changed', (event) => {
@@ -338,7 +338,7 @@
 assertType<GridSorterDirection | null | undefined>(sorter.direction);
 
 /* GridTreeColumn */
-const genericTreeColumn = document.createElement('vaadin-grid-tree-column');
+const genericTreeColumn = document.createElement('vaadin24-grid-tree-column');
 assertType<GridTreeColumn>(genericTreeColumn);
 
 const narrowedTreeColumn = genericTreeColumn as GridTreeColumn<TestGridItem>;
@@ -346,7 +346,7 @@
 assertType<string | null | undefined>(narrowedTreeColumn.path);
 
 /* GridTreeToggle */
-const treeToggle = document.createElement('vaadin-grid-tree-toggle');
+const treeToggle = document.createElement('vaadin24-grid-tree-toggle');
 assertType<GridTreeToggle>(treeToggle);
 assertType<ThemableMixinClass>(treeToggle);
 assertType<number>(treeToggle.level);
