Index: test/items.test.js
===================================================================
--- test/items.test.js
+++ test/items.test.js
@@ -15,25 +15,25 @@
 } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import '../src/vaadin-context-menu.js';
-import '@vaadin/item/src/vaadin-item.js';
-import '@vaadin/list-box/src/vaadin-list-box.js';
-import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
+import '@scoped-vaadin/item/src/vaadin-item.js';
+import '@scoped-vaadin/list-box/src/vaadin-list-box.js';
+import { isTouch } from '@scoped-vaadin/component-base/src/browser-utils.js';
 import { activateItem, getMenuItems, getSubMenu, openMenu } from './helpers.js';
 
 describe('items', () => {
   let rootMenu, subMenu, target, rootOverlay, subOverlay1;
 
   const createComponent = (text) => {
-    const item = document.createElement('vaadin-context-menu-item');
+    const item = document.createElement('vaadin24-context-menu-item');
     item.textContent = text;
     return item;
   };
 
   beforeEach(async () => {
     rootMenu = fixtureSync(`
-      <vaadin-context-menu>
+      <vaadin24-context-menu>
         <button id="target"></button>
-      </vaadin-context-menu>
+      </vaadin24-context-menu>
     `);
     rootMenu.openOn = isTouch ? 'click' : 'mouseover';
     target = rootMenu.firstElementChild;
@@ -189,7 +189,7 @@
   });
 
   it('should have default item type', () => {
-    expect(getMenuItems(rootMenu)[0].localName).to.equal('vaadin-context-menu-item');
+    expect(getMenuItems(rootMenu)[0].localName).to.equal('vaadin24-context-menu-item');
   });
 
   it('should accept component items', async () => {
