Index: test/items-theme.test.js
===================================================================
--- test/items-theme.test.js
+++ test/items-theme.test.js
@@ -1,9 +1,9 @@
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import '../src/vaadin-context-menu.js';
-import '@vaadin/item/src/vaadin-item.js';
-import '@vaadin/list-box/src/vaadin-list-box.js';
-import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
+import '@scoped-vaadin/item/src/vaadin-item.js';
+import '@scoped-vaadin/list-box/src/vaadin-list-box.js';
+import { isTouch } from '@scoped-vaadin/component-base/src/browser-utils.js';
 import { getMenuItems, getSubMenu, openMenu } from './helpers.js';
 
 describe('items theme', () => {
@@ -17,9 +17,9 @@
 
   beforeEach(async () => {
     rootMenu = fixtureSync(`
-      <vaadin-context-menu theme="foo">
+      <vaadin24-context-menu theme="foo">
         <button id="target"></button>
-      </vaadin-context-menu>
+      </vaadin24-context-menu>
     `);
     rootMenu.openOn = isTouch ? 'click' : 'mouseover';
     target = rootMenu.firstElementChild;
@@ -57,8 +57,8 @@
   it('should propagate host theme attribute to the nested elements', () => {
     [rootMenu, subMenu, subMenu2].forEach((subMenu) => {
       const overlay = subMenu._overlayElement;
-      const listBox = overlay.querySelector('vaadin-context-menu-list-box');
-      const items = Array.from(listBox.querySelectorAll('vaadin-context-menu-item'));
+      const listBox = overlay.querySelector('vaadin24-context-menu-list-box');
+      const items = Array.from(listBox.querySelectorAll('vaadin24-context-menu-item'));
 
       expect(overlay.getAttribute('theme')).to.equal('foo');
       expect(listBox.getAttribute('theme')).to.equal('foo');
@@ -87,8 +87,8 @@
 
     [rootMenu, subMenu, subMenu2].forEach((subMenu) => {
       const overlay = subMenu._overlayElement;
-      const listBox = overlay.querySelector('vaadin-context-menu-list-box');
-      const items = Array.from(listBox.querySelectorAll('vaadin-context-menu-item'));
+      const listBox = overlay.querySelector('vaadin24-context-menu-list-box');
+      const items = Array.from(listBox.querySelectorAll('vaadin24-context-menu-item'));
 
       expect(overlay.hasAttribute('theme')).to.be.false;
       expect(listBox.hasAttribute('theme')).to.be.false;
