Index: test/overlay.common.js
===================================================================
--- test/overlay.common.js
+++ test/overlay.common.js
@@ -7,9 +7,9 @@
 
   beforeEach(async () => {
     menu = fixtureSync(`
-      <vaadin-context-menu>
+      <vaadin24-context-menu>
         <div id="target" style="width: 100px; outline: 1px dashed #000;">FOOOO</div>
-      </vaadin-context-menu>
+      </vaadin24-context-menu>
     `);
     menu.renderer = (root) => {
       root.textContent = 'OVERLAY CONTENT';
@@ -308,7 +308,7 @@
       });
     });
 
-    (isIOS ? describe : describe.skip)('<vaadin-overlay> iOS viewport workaround (phone mode)', () => {
+    (isIOS ? describe : describe.skip)('<vaadin24-overlay> iOS viewport workaround (phone mode)', () => {
       it('should have zero bottom by default', async () => {
         contextmenu();
         await oneEvent(overlay, 'vaadin-overlay-open');
