Index: test/properties.test.js
===================================================================
--- test/properties.test.js
+++ test/properties.test.js
@@ -15,11 +15,11 @@
 
     beforeEach(() => {
       menu = fixtureSync(`
-        <vaadin-context-menu>
+        <vaadin23-context-menu>
           <section>
             <div id="target"></div>
           </section>
-        </vaadin-context-menu>
+        </vaadin23-context-menu>
       `);
       target = menu.querySelector('#target');
     });
@@ -41,7 +41,7 @@
 
   describe('openOn', () => {
     beforeEach(() => {
-      menu = fixtureSync('<vaadin-context-menu></vaadin-context-menu>');
+      menu = fixtureSync('<vaadin23-context-menu></vaadin23-context-menu>');
     });
 
     it('should open on custom event', () => {
@@ -71,7 +71,7 @@
 
   describe('opened', () => {
     beforeEach(() => {
-      menu = fixtureSync('<vaadin-context-menu></vaadin-context-menu>');
+      menu = fixtureSync('<vaadin23-context-menu></vaadin23-context-menu>');
     });
 
     it('should be read-only', () => {
@@ -91,7 +91,7 @@
 
   describe('closeOn', () => {
     beforeEach(() => {
-      menu = fixtureSync('<vaadin-context-menu></vaadin-context-menu>');
+      menu = fixtureSync('<vaadin23-context-menu></vaadin23-context-menu>');
       menu._setOpened(true);
     });
 
@@ -118,7 +118,7 @@
     beforeEach(() => {
       wrapper = fixtureSync(`
         <div>
-          <vaadin-context-menu></vaadin-context-menu>
+          <vaadin23-context-menu></vaadin23-context-menu>
           <section>
             <div id="target"></div>
           </section>
@@ -161,7 +161,7 @@
 
   describe('theme attribute', () => {
     beforeEach(() => {
-      menu = fixtureSync('<vaadin-context-menu theme="foo"></vaadin-context-menu>');
+      menu = fixtureSync('<vaadin23-context-menu theme="foo"></vaadin23-context-menu>');
     });
 
     it('should propagate theme attribute to overlay', () => {
