Index: test/renderer.test.js
===================================================================
--- test/renderer.test.js
+++ test/renderer.test.js
@@ -9,9 +9,9 @@
 
   beforeEach(() => {
     menu = fixtureSync(`
-      <vaadin-context-menu>
+      <vaadin23-context-menu>
         <div id="target"></div>
-      </vaadin-context-menu>
+      </vaadin23-context-menu>
     `);
     menu.renderer = sinon.spy((root, _, context) => {
       if (root.firstChild) {
@@ -110,7 +110,7 @@
   });
 
   it('should not throw if requestContentUpdate() called before adding to DOM', () => {
-    const contextMenu = document.createElement('vaadin-context-menu');
+    const contextMenu = document.createElement('vaadin23-context-menu');
     expect(() => contextMenu.requestContentUpdate()).not.to.throw(Error);
   });
 });
