Index: test/renderer.common.js
===================================================================
--- test/renderer.common.js
+++ test/renderer.common.js
@@ -7,9 +7,9 @@
 
   beforeEach(async () => {
     menu = fixtureSync(`
-      <vaadin-context-menu>
+      <vaadin24-context-menu>
         <div id="target"></div>
-      </vaadin-context-menu>
+      </vaadin24-context-menu>
     `);
     menu.renderer = sinon.spy((root, _, context) => {
       if (root.firstChild) {
@@ -130,7 +130,7 @@
   });
 
   it('should not throw if requestContentUpdate() called before adding to DOM', () => {
-    const contextMenu = document.createElement('vaadin-context-menu');
+    const contextMenu = document.createElement('vaadin24-context-menu');
     expect(() => contextMenu.requestContentUpdate()).not.to.throw(Error);
   });
 });
