Index: test/touch.test.js
===================================================================
--- test/touch.test.js
+++ test/touch.test.js
@@ -13,16 +13,16 @@
 import sinon from 'sinon';
 import './context-menu-test-styles.js';
 import '../src/vaadin-context-menu.js';
-import { gestures } from '@vaadin/component-base/src/gestures.js';
+import { gestures } from '@scoped-vaadin/component-base/src/gestures.js';
 
 describe('mobile support', () => {
   let menu, target;
 
   beforeEach(async () => {
     menu = fixtureSync(`
-      <vaadin-context-menu selector="#target">
+      <vaadin24-context-menu selector="#target">
         <div id="target">Target</div>
-      </vaadin-context-menu>
+      </vaadin24-context-menu>
     `);
     await nextRender();
     target = menu.querySelector('#target');
