Index: test/touch.test.js
===================================================================
--- test/touch.test.js
+++ test/touch.test.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import {
   fire,
@@ -13,14 +14,14 @@
 import './not-animated-styles.js';
 import '../vaadin-context-menu.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { gestures } from '@vaadin/component-base/src/gestures.js';
+import { gestures } from '@scoped-vaadin/component-base/src/gestures.js';
 
 class MenuTouchWrapper extends PolymerElement {
   static get template() {
     return html`
-      <vaadin-context-menu selector="#target" id="menu" renderer="[[_renderer]]">
+      <vaadin23-context-menu selector="#target" id="menu" renderer="[[_renderer]]">
         <div id="target">Target</div>
-      </vaadin-context-menu>
+      </vaadin23-context-menu>
     `;
   }
 
@@ -33,7 +34,7 @@
   }
 }
 
-customElements.define('menu-touch-wrapper', MenuTouchWrapper);
+internalCustomElements.define('menu-touch-wrapper', MenuTouchWrapper);
 
 describe('mobile support', () => {
   let menu, target;
