Index: test/button.common.ts
===================================================================
--- test/button.common.ts
+++ test/button.common.ts
@@ -1,33 +1,34 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextRender, nextUpdate } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
 import type { Button } from '../vaadin-button.js';
 
-describe('vaadin-button', () => {
+describe('vaadin24-button', () => {
   let button: Button;
 
   describe('custom element definition', () => {
     let tagName: string;
 
     beforeEach(() => {
-      button = fixtureSync('<vaadin-button></vaadin-button>');
+      button = fixtureSync('<vaadin24-button></vaadin24-button>');
       tagName = button.tagName.toLowerCase();
     });
 
     it('should be defined in custom element registry', () => {
-      expect(customElements.get(tagName)).to.be.ok;
+      expect(internalCustomElements.get(tagName)).to.be.ok;
     });
 
     it('should have a valid static "is" getter', () => {
-      expect((customElements.get(tagName) as any).is).to.equal(tagName);
+      expect((internalCustomElements.get(tagName) as any).is).to.equal(tagName);
     });
   });
 
   describe('role', () => {
     describe('default', () => {
       beforeEach(async () => {
-        button = fixtureSync('<vaadin-button>Press me</vaadin-button>');
+        button = fixtureSync('<vaadin24-button>Press me</vaadin24-button>');
         await nextRender(button);
       });
 
@@ -38,7 +39,7 @@
 
     describe('custom', () => {
       beforeEach(async () => {
-        button = fixtureSync('<vaadin-button role="menuitem">Press me</vaadin-button>');
+        button = fixtureSync('<vaadin24-button role="menuitem">Press me</vaadin24-button>');
         await nextRender(button);
       });
 
@@ -50,7 +51,7 @@
 
   describe('keyboard', () => {
     beforeEach(async () => {
-      button = fixtureSync('<vaadin-button>Press me</vaadin-button>');
+      button = fixtureSync('<vaadin24-button>Press me</vaadin24-button>');
       await nextRender(button);
       button.focus();
     });
