Index: details/theme/material/vaadin-details-styles.js
===================================================================
--- details/theme/material/vaadin-details-styles.js	null
+++ details/theme/material/vaadin-details-styles.js	null
@@ -1,7 +1,7 @@
-import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/vaadin-material-styles/shadow.js';
-import '@vaadin/vaadin-material-styles/typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import '@scoped-vaadin/vaadin-material-styles/shadow.js';
+import '@scoped-vaadin/vaadin-material-styles/typography.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const details = css`
   :host {
@@ -19,6 +19,6 @@
   }
 `;
 
-registerStyles('vaadin-details', details, { moduleId: 'material-details' });
+registerStyles('vaadin24-details', details, { moduleId: 'material-details' });
 
 export { details };
