Index: src/vaadin-dialog-overlay.d.ts
===================================================================
--- src/vaadin-dialog-overlay.d.ts
+++ src/vaadin-dialog-overlay.d.ts
@@ -3,19 +3,19 @@
  * Copyright (c) 2017 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DirMixin } from '@scoped-vaadin/component-base/src/dir-mixin.js';
+import { ThemableMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { DialogOverlayMixin } from './vaadin-dialog-overlay-mixin.js';
 
 export { DialogOverlayBounds, DialogOverlayBoundsParam } from './vaadin-dialog-overlay-mixin.js';
 
 /**
- * An element used internally by `<vaadin-dialog>`. Not intended to be used separately.
+ * An element used internally by `<vaadin24-dialog>`. Not intended to be used separately.
  */
 export class DialogOverlay extends DialogOverlayMixin(DirMixin(ThemableMixin(HTMLElement))) {}
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-dialog-overlay': DialogOverlay;
+    'vaadin24-dialog-overlay': DialogOverlay;
   }
 }
