Index: src/vaadin-dialog-resizable-mixin.js
===================================================================
--- src/vaadin-dialog-resizable-mixin.js
+++ src/vaadin-dialog-resizable-mixin.js
@@ -3,11 +3,11 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { eventInWindow, getMouseOrFirstTouchEvent } from './vaadin-dialog-utils.js';
 
 registerStyles(
-  'vaadin-dialog-overlay',
+  'vaadin23-dialog-overlay',
   css`
     [part='overlay'] {
       position: relative;
