Index: src/vaadin-dialog-styles.js
===================================================================
--- src/vaadin-dialog-styles.js
+++ src/vaadin-dialog-styles.js
@@ -68,10 +68,10 @@
 
   /*
       NOTE(platosha): Make some min-width to prevent collapsing of the content
-      taking the parent width, e. g., <vaadin-grid> and such.
+      taking the parent width, e. g., <vaadin24-grid> and such.
     */
   [part='content'] {
-    min-width: 12em; /* matches the default <vaadin-text-field> width */
+    min-width: 12em; /* matches the default <vaadin24-text-field> width */
   }
 
   :host([has-bounds-set]) [part='overlay'] {
