Index: src/vaadin-lit-dialog.js
===================================================================
--- src/vaadin-lit-dialog.js
+++ src/vaadin-lit-dialog.js
@@ -6,11 +6,11 @@
 import './vaadin-lit-dialog-overlay.js';
 import { css, html, LitElement } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { OverlayClassMixin } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { ThemePropertyMixin } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { OverlayClassMixin } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import { PolylitMixin } from '@scoped-vaadin/component-base/src/polylit-mixin.js';
+import { ThemePropertyMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 import { DialogBaseMixin } from './vaadin-dialog-base-mixin.js';
 import { DialogDraggableMixin } from './vaadin-dialog-draggable-mixin.js';
 import { DialogRendererMixin } from './vaadin-dialog-renderer-mixin.js';
@@ -19,7 +19,7 @@
 export { DialogOverlay } from './vaadin-lit-dialog-overlay.js';
 
 /**
- * LitElement based version of `<vaadin-dialog>` web component.
+ * LitElement based version of `<vaadin24-dialog>` web component.
  *
  * ## Disclaimer
  *
@@ -42,7 +42,7 @@
   ),
 ) {
   static get is() {
-    return 'vaadin-dialog';
+    return 'vaadin24-dialog';
   }
 
   static get styles() {
@@ -70,7 +70,7 @@
   /** @protected */
   render() {
     return html`
-      <vaadin-dialog-overlay
+      <vaadin24-dialog-overlay
         id="overlay"
         role="dialog"
         .owner="${this}"
@@ -89,7 +89,7 @@
         ?resizable="${this.resizable}"
         restore-focus-on-close
         focus-trap
-      ></vaadin-dialog-overlay>
+      ></vaadin24-dialog-overlay>
     `;
   }
 }
