Index: dialog/src/vaadin-dialog-renderer-mixin.d.ts
===================================================================
--- dialog/src/vaadin-dialog-renderer-mixin.d.ts
+++ dialog/src/vaadin-dialog-renderer-mixin.d.ts
@@ -16,7 +16,7 @@
    * Receives two arguments:
    *
    * - `root` The root container DOM element. Append your content to it.
-   * - `dialog` The reference to the `<vaadin-dialog>` element.
+   * - `dialog` The reference to the `<vaadin24-dialog>` element.
    */
   renderer: DialogRenderer | null | undefined;
 
@@ -37,7 +37,7 @@
    * Receives two arguments:
    *
    * - `root` The root container DOM element. Append your content to it.
-   * - `dialog` The reference to the `<vaadin-dialog>` element.
+   * - `dialog` The reference to the `<vaadin24-dialog>` element.
    *
    * If both `headerTitle` and `headerRenderer` are defined, the title
    * and the elements created by the renderer will be placed next to
@@ -52,7 +52,7 @@
    * Receives two arguments:
    *
    * - `root` The root container DOM element. Append your content to it.
-   * - `dialog` The reference to the `<vaadin-dialog>` element.
+   * - `dialog` The reference to the `<vaadin24-dialog>` element.
    *
    * When `footerRenderer` is set, the attribute `has-footer` is added to the overlay element.
    */
