Index: dialog/src/vaadin-dialog.js
===================================================================
--- dialog/src/vaadin-dialog.js
+++ dialog/src/vaadin-dialog.js
@@ -5,11 +5,11 @@
  */
 import './vaadin-dialog-overlay.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
-import { OverlayClassMixin } from '@vaadin/component-base/src/overlay-class-mixin.js';
-import { processTemplates } from '@vaadin/component-base/src/templates.js';
-import { ThemePropertyMixin } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import { defineCustomElement } from '@scoped-vaadin/component-base/src/define.js';
+import { ElementMixin } from '@scoped-vaadin/component-base/src/element-mixin.js';
+import { OverlayClassMixin } from '@scoped-vaadin/component-base/src/overlay-class-mixin.js';
+import { processTemplates } from '@scoped-vaadin/component-base/src/templates.js';
+import { ThemePropertyMixin } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
 import { DialogBaseMixin } from './vaadin-dialog-base-mixin.js';
 import { DialogDraggableMixin } from './vaadin-dialog-draggable-mixin.js';
 import { DialogRendererMixin } from './vaadin-dialog-renderer-mixin.js';
@@ -18,7 +18,7 @@
 export { DialogOverlay } from './vaadin-dialog-overlay.js';
 
 /**
- * `<vaadin-dialog>` is a Web Component for creating customized modal dialogs.
+ * `<vaadin24-dialog>` is a Web Component for creating customized modal dialogs.
  *
  * ### Rendering
  *
@@ -30,7 +30,7 @@
  * users are able to check if there is already content in `root` for reusing it.
  *
  * ```html
- * <vaadin-dialog id="dialog"></vaadin-dialog>
+ * <vaadin24-dialog id="dialog"></vaadin24-dialog>
  * ```
  * ```js
  * const dialog = document.querySelector('#dialog');
@@ -46,13 +46,13 @@
  *
  * ### Styling
  *
- * `<vaadin-dialog>` uses `<vaadin-dialog-overlay>` internal
+ * `<vaadin24-dialog>` uses `<vaadin24-dialog-overlay>` internal
  * themable component as the actual visible dialog overlay.
  *
- * See [`<vaadin-overlay>`](#/elements/vaadin-overlay) documentation.
- * for `<vaadin-dialog-overlay>` parts.
+ * See [`<vaadin24-overlay>`](#/elements/vaadin-overlay) documentation.
+ * for `<vaadin24-dialog-overlay>` parts.
  *
- * In addition to `<vaadin-overlay>` parts, the following parts are available for styling:
+ * In addition to `<vaadin24-overlay>` parts, the following parts are available for styling:
  *
  * Part name        | Description
  * -----------------|-------------------------------------------
@@ -70,8 +70,8 @@
  * `has-footer`     | Set when the element has footer renderer
  * `overflow`       | Set to `top`, `bottom`, none or both
  *
- * Note: the `theme` attribute value set on `<vaadin-dialog>` is
- * propagated to the internal `<vaadin-dialog-overlay>` component.
+ * Note: the `theme` attribute value set on `<vaadin24-dialog>` is
+ * propagated to the internal `<vaadin24-dialog-overlay>` component.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
@@ -101,7 +101,7 @@
         }
       </style>
 
-      <vaadin-dialog-overlay
+      <vaadin24-dialog-overlay
         id="overlay"
         header-title="[[headerTitle]]"
         on-opened-changed="_onOverlayOpened"
@@ -113,12 +113,12 @@
         resizable$="[[resizable]]"
         restore-focus-on-close
         focus-trap
-      ></vaadin-dialog-overlay>
+      ></vaadin24-dialog-overlay>
     `;
   }
 
   static get is() {
-    return 'vaadin-dialog';
+    return 'vaadin24-dialog';
   }
 
   static get properties() {
