Index: list-box/theme/material/vaadin-list-box-styles.js
===================================================================
--- list-box/theme/material/vaadin-list-box-styles.js
+++ list-box/theme/material/vaadin-list-box-styles.js
@@ -1,6 +1,6 @@
-import '@vaadin/vaadin-material-styles/font-icons.js';
-import '@vaadin/vaadin-material-styles/color.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-material-styles/font-icons.js';
+import '@scoped-vaadin/vaadin-material-styles/color.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const listBox = css`
   :host {
@@ -22,6 +22,6 @@
   }
 `;
 
-registerStyles('vaadin-list-box', listBox, { moduleId: 'material-list-box' });
+registerStyles('vaadin24-list-box', listBox, { moduleId: 'material-list-box' });
 
 export { listBox };
