Index: test/basic.test.js
===================================================================
--- test/basic.test.js
+++ test/basic.test.js
@@ -8,7 +8,7 @@
   let comboBox, overlay, input;
 
   beforeEach(async () => {
-    comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+    comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
     await nextRender();
     overlay = comboBox.$.overlay;
     input = comboBox.inputElement;
@@ -206,14 +206,14 @@
   beforeEach(async () => {
     container = fixtureSync(`
       <div style="display: flex; flex-direction: column; width:500px;">
-        <vaadin-combo-box></vaadin-combo-box>
+        <vaadin24-combo-box></vaadin24-combo-box>
       </div>
     `);
     await nextRender();
   });
 
   it('combo-box should stretch to fit the flex container', () => {
-    const combobox = container.querySelector('vaadin-combo-box');
+    const combobox = container.querySelector('vaadin24-combo-box');
     expect(window.getComputedStyle(container).width).to.eql('500px');
     expect(window.getComputedStyle(combobox).width).to.eql('500px');
   });
@@ -223,7 +223,7 @@
   let comboBox;
 
   beforeEach(() => {
-    comboBox = document.createElement('vaadin-combo-box');
+    comboBox = document.createElement('vaadin24-combo-box');
   });
 
   afterEach(() => {
@@ -241,15 +241,15 @@
 
 describe('pre-opened', () => {
   it('should not throw error when adding a pre-opened combo-box', () => {
-    expect(() => fixtureSync(`<vaadin-combo-box opened></vaadin-combo-box>`)).to.not.throw(Error);
+    expect(() => fixtureSync(`<vaadin24-combo-box opened></vaadin24-combo-box>`)).to.not.throw(Error);
   });
 
   it('should not throw error when adding a pre-opened combo-box with items', () => {
-    expect(() => fixtureSync(`<vaadin-combo-box opened items="[0]"></vaadin-combo-box>`)).to.not.throw(Error);
+    expect(() => fixtureSync(`<vaadin24-combo-box opened items="[0]"></vaadin24-combo-box>`)).to.not.throw(Error);
   });
 
   it('should have overlay with correct width', async () => {
-    const comboBox = fixtureSync(`<vaadin-combo-box opened items="[0]"></vaadin-combo-box>`);
+    const comboBox = fixtureSync(`<vaadin24-combo-box opened items="[0]"></vaadin24-combo-box>`);
     await nextRender();
     const expectedOverlayWidth = comboBox.clientWidth;
     const actualOverlayWidth = comboBox.$.overlay.$.content.clientWidth;
@@ -258,13 +258,13 @@
 
   it('should have scroller with correct max height', async () => {
     const comboBox = fixtureSync(`
-      <vaadin-combo-box
+      <vaadin24-combo-box
         opened
         items="[0]"
         style="--vaadin-combo-box-overlay-max-height: 200px"
-      ></vaadin-combo-box>`);
+      ></vaadin24-combo-box>`);
     await nextRender();
-    const scroller = comboBox.$.overlay.querySelector('vaadin-combo-box-scroller');
+    const scroller = comboBox.$.overlay.querySelector('vaadin24-combo-box-scroller');
     expect(scroller.style.maxHeight).to.equal('200px');
   });
 });
