Index: test/combo-box-light.common.js
===================================================================
--- test/combo-box-light.common.js
+++ test/combo-box-light.common.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import {
   arrowDownKeyDown,
@@ -14,9 +15,9 @@
 } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
-import '@vaadin/text-field/vaadin-text-field.js';
+import '@scoped-vaadin/text-field/vaadin-text-field.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
+import { isTouch } from '@scoped-vaadin/component-base/src/browser-utils.js';
 import { getFirstItem } from './helpers.js';
 
 class MyInput extends PolymerElement {
@@ -27,7 +28,7 @@
           display: inline-block;
         }
       </style>
-      <vaadin-text-field id="input" value="{{customValue}}"></vaadin-text-field>
+      <vaadin24-text-field id="input" value="{{customValue}}"></vaadin24-text-field>
     `;
   }
 
@@ -41,21 +42,21 @@
   }
 }
 
-customElements.define('my-input', MyInput);
+internalCustomElements.define('my-input', MyInput);
 
-describe('vaadin-combo-box-light', () => {
+describe('vaadin24-combo-box-light', () => {
   let comboBox, overlay, inputElement;
 
   beforeEach(async () => {
     comboBox = fixtureSync(`
-      <vaadin-combo-box-light>
-        <vaadin-text-field></vaadin-text-field>
-      </vaadin-combo-box-light>
+      <vaadin24-combo-box-light>
+        <vaadin24-text-field></vaadin24-text-field>
+      </vaadin24-combo-box-light>
     `);
     await nextRender();
     comboBox.items = ['foo', 'bar', 'baz'];
     overlay = comboBox.$.overlay;
-    inputElement = comboBox.querySelector('vaadin-text-field');
+    inputElement = comboBox.querySelector('vaadin24-text-field');
   });
 
   it('should find the input element correctly', () => {
@@ -166,9 +167,9 @@
 
   beforeEach(async () => {
     comboBox = fixtureSync(`
-      <vaadin-combo-box-light attr-for-value="custom-value">
+      <vaadin24-combo-box-light attr-for-value="custom-value">
         <my-input class="input"></my-input>
-      </vaadin-combo-box-light>
+      </vaadin24-combo-box-light>
     `);
     await nextRender();
     comboBox.items = ['foo', 'bar', 'baz'];
@@ -211,9 +212,9 @@
   describe('input in light DOM', () => {
     beforeEach(async () => {
       comboBox = fixtureSync(`
-        <vaadin-combo-box-light>
-          <vaadin-text-field></vaadin-text-field>
-        </vaadin-combo-box-light>
+        <vaadin24-combo-box-light>
+          <vaadin24-text-field></vaadin24-text-field>
+        </vaadin24-combo-box-light>
       `);
       await nextRender();
       comboBox.items = ['foo', 'bar', 'baz'];
@@ -236,9 +237,9 @@
   describe('input in Shadow DOM', () => {
     beforeEach(async () => {
       comboBox = fixtureSync(`
-        <vaadin-combo-box-light attr-for-value="custom-value">
+        <vaadin24-combo-box-light attr-for-value="custom-value">
           <my-input class="input"></my-input>
-        </vaadin-combo-box-light>
+        </vaadin24-combo-box-light>
       `);
       await nextRender();
       comboBox.items = ['foo', 'bar', 'baz'];
@@ -264,12 +265,12 @@
 
   beforeEach(async () => {
     comboBox = fixtureSync(`
-      <vaadin-combo-box-light>
-        <vaadin-text-field>
+      <vaadin24-combo-box-light>
+        <vaadin24-text-field>
           <button slot="suffix" class="clear-button">Clear</button>
           <button slot="suffix" class="toggle-button">Toggle</button>
-        </vaadin-text-field>
-      </vaadin-combo-box-light>
+        </vaadin24-text-field>
+      </vaadin24-combo-box-light>
     `);
     await nextRender();
     comboBox.items = ['foo', 'bar', 'baz'];
@@ -449,9 +450,9 @@
 
   beforeEach(async () => {
     comboBox = fixtureSync(`
-      <vaadin-combo-box-light theme="foo">
-        <vaadin-text-field></vaadin-text-field>
-      </vaadin-combo-box-light>
+      <vaadin24-combo-box-light theme="foo">
+        <vaadin24-text-field></vaadin24-text-field>
+      </vaadin24-combo-box-light>
     `);
     await nextRender();
   });
