Index: test/combo-box-light-validation.test.js
===================================================================
--- test/combo-box-light-validation.test.js
+++ test/combo-box-light-validation.test.js
@@ -12,9 +12,9 @@
 
     beforeEach(async () => {
       comboBox = fixtureSync(`
-        <vaadin-combo-box-light>
+        <vaadin24-combo-box-light>
           <input class="input" />
-        </vaadin-combo-box-light>
+        </vaadin24-combo-box-light>
       `);
       comboBox.items = ['foo', 'bar', 'baz'];
       await nextRender();
@@ -65,9 +65,9 @@
   describe('required', () => {
     beforeEach(async () => {
       comboBox = fixtureSync(`
-        <vaadin-combo-box-light>
+        <vaadin24-combo-box-light>
           <input class="input" />
-        </vaadin-combo-box-light>
+        </vaadin24-combo-box-light>
       `);
       comboBox.items = ['foo', 'bar', 'baz'];
       comboBox.required = true;
