Index: test/data-provider.common.js
===================================================================
--- test/data-provider.common.js
+++ test/data-provider.common.js
@@ -17,18 +17,18 @@
 } from './helpers.js';
 
 const TEMPLATES = {
-  'vaadin-combo-box': `
-    <vaadin-combo-box></vaadin-combo-box>
+  'vaadin24-combo-box': `
+    <vaadin24-combo-box></vaadin24-combo-box>
   `,
-  'vaadin-combo-box-light': `
-    <vaadin-combo-box-light>
-      <vaadin-text-field></vaadin-text-field>
-    </vaadin-combo-box-light>
+  'vaadin24-combo-box-light': `
+    <vaadin24-combo-box-light>
+      <vaadin24-text-field></vaadin24-text-field>
+    </vaadin24-combo-box-light>
   `,
 };
 
-['vaadin-combo-box', 'vaadin-combo-box-light'].forEach((tag) => {
-  const isComboBoxLight = tag === 'vaadin-combo-box-light';
+['vaadin24-combo-box', 'vaadin24-combo-box-light'].forEach((tag) => {
+  const isComboBoxLight = tag === 'vaadin24-combo-box-light';
 
   describe(`data provider - ${tag}`, () => {
     let comboBox;
