Index: test/data-provider-lit.test.js
===================================================================
--- test/data-provider-lit.test.js
+++ test/data-provider-lit.test.js
@@ -1,6 +1,6 @@
 import './not-animated-styles.js';
 import './data-provider-styles.js';
-import '@vaadin/text-field/vaadin-lit-text-field.js';
+import '@scoped-vaadin/text-field/vaadin-lit-text-field.js';
 import '../theme/lumo/vaadin-lit-combo-box-light.js';
 import '../theme/lumo/vaadin-lit-combo-box.js';
 import './data-provider.common.js';
