Index: test/data-provider-filtering-lit.test.js
===================================================================
--- test/data-provider-filtering-lit.test.js
+++ test/data-provider-filtering-lit.test.js
@@ -1,5 +1,5 @@
 import './not-animated-styles.js';
-import '@vaadin/text-field/vaadin-lit-text-field.js';
+import '@scoped-vaadin/text-field/vaadin-lit-text-field.js';
 import '../src/vaadin-lit-combo-box-light.js';
 import '../src/vaadin-lit-combo-box.js';
 import './data-provider-filtering.common.js';
