Index: test/data-provider-dynamic-size-lit.test.js
===================================================================
--- test/data-provider-dynamic-size-lit.test.js
+++ test/data-provider-dynamic-size-lit.test.js
@@ -1,4 +1,4 @@
-import '@vaadin/text-field/vaadin-lit-text-field.js';
+import '@scoped-vaadin/text-field/vaadin-lit-text-field.js';
 import '../src/vaadin-lit-combo-box.js';
 import '../src/vaadin-lit-combo-box-light.js';
 import './data-provider-dynamic-size.common.js';
