Index: test/dropdown.test.js
===================================================================
--- test/dropdown.test.js
+++ test/dropdown.test.js
@@ -6,7 +6,7 @@
   let overlay;
 
   beforeEach(() => {
-    overlay = fixtureSync(`<vaadin-combo-box-overlay></vaadin-combo-box-overlay>`);
+    overlay = fixtureSync(`<vaadin23-combo-box-overlay></vaadin23-combo-box-overlay>`);
   });
 
   it('should not show the overlay when closed', () => {
