Index: test/dynamic-size.common.js
===================================================================
--- test/dynamic-size.common.js
+++ test/dynamic-size.common.js
@@ -26,7 +26,7 @@
     }
 
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.dataProvider = dataProvider;
       await nextRender();
     });
