Index: test/events.test.js
===================================================================
--- test/events.test.js
+++ test/events.test.js
@@ -10,7 +10,7 @@
 
   describe('custom-value-set event', () => {
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.allowCustomValue = true;
       comboBox.items = ['a', 'b'];
       comboBox.inputElement.focus();
