Index: test/external-filtering.test.js
===================================================================
--- test/external-filtering.test.js
+++ test/external-filtering.test.js
@@ -10,7 +10,7 @@
 
   describe('basic', () => {
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin23-combo-box></vaadin23-combo-box>');
       comboBox.filteredItems = ['foo', 'bar', 'baz'];
       overlay = comboBox.$.overlay;
     });
@@ -105,7 +105,7 @@
 
   describe('filtered items attribute', () => {
     beforeEach(() => {
-      comboBox = fixtureSync(`<vaadin-combo-box filtered-items='["a", "b", "c"]' value='b'></vaadin-combo-box>`);
+      comboBox = fixtureSync(`<vaadin23-combo-box filtered-items='["a", "b", "c"]' value='b'></vaadin23-combo-box>`);
     });
 
     it('should not throw when passing filteredItems and value as attributes', () => {
@@ -116,7 +116,7 @@
 
   describe('value is set after', () => {
     beforeEach(() => {
-      comboBox = fixtureSync(`<vaadin-combo-box></vaadin-combo-box>`);
+      comboBox = fixtureSync(`<vaadin23-combo-box></vaadin23-combo-box>`);
       comboBox.filteredItems = ['foo', 'bar'];
       comboBox.value = 'foo';
     });
@@ -153,7 +153,7 @@
 
   describe('value is set before', () => {
     beforeEach(() => {
-      comboBox = fixtureSync(`<vaadin-combo-box value="foo"></vaadin-combo-box>`);
+      comboBox = fixtureSync(`<vaadin23-combo-box value="foo"></vaadin23-combo-box>`);
       comboBox.filteredItems = ['foo', 'bar'];
     });
 
@@ -181,7 +181,7 @@
 
   describe('value is set before + autoOpenDisabled', () => {
     beforeEach(() => {
-      comboBox = fixtureSync(`<vaadin-combo-box auto-open-disabled value="bar"></vaadin-combo-box>`);
+      comboBox = fixtureSync(`<vaadin23-combo-box auto-open-disabled value="bar"></vaadin23-combo-box>`);
       comboBox.filteredItems = ['foo', 'bar'];
     });
 
@@ -238,7 +238,7 @@
     ];
 
     beforeEach(() => {
-      comboBox = fixtureSync(`<vaadin-combo-box></vaadin-combo-box>`);
+      comboBox = fixtureSync(`<vaadin23-combo-box></vaadin23-combo-box>`);
       comboBox.selectedItem = items[0];
       comboBox.filteredItems = items;
     });
