Index: test/internal-filtering.common.js
===================================================================
--- test/internal-filtering.common.js
+++ test/internal-filtering.common.js
@@ -9,7 +9,7 @@
 
   describe('value is set before', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync(`<vaadin-combo-box value="foo"></vaadin-combo-box>`);
+      comboBox = fixtureSync(`<vaadin24-combo-box value="foo"></vaadin24-combo-box>`);
       await nextRender();
       comboBox.items = ['foo', 'bar'];
     });
@@ -25,7 +25,7 @@
 
   describe('setting the input field value', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       await nextRender();
       comboBox.items = ['foo', 'bar', 'baz'];
     });
@@ -84,7 +84,7 @@
 
   describe('focusing items while filtering', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       await nextRender();
       comboBox.items = ['foo', 'bar', 'baz'];
     });
@@ -135,7 +135,7 @@
     let overlay;
 
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       await nextRender();
       comboBox.items = ['foo', 'bar', 'baz'];
       overlay = comboBox.$.overlay;
@@ -244,7 +244,7 @@
 
   describe('filtering with many items', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = makeItems(1000);
       await nextRender();
     });
@@ -259,7 +259,7 @@
 
   describe('setting items when opened', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       await nextRender();
       comboBox.items = [];
     });
@@ -273,7 +273,7 @@
 
   describe('setting placeholder items', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       await nextRender();
       comboBox.items = [new ComboBoxPlaceholder(), new ComboBoxPlaceholder()];
     });
