Index: test/interactions.common.js
===================================================================
--- test/interactions.common.js
+++ test/interactions.common.js
@@ -15,7 +15,7 @@
 } from '@vaadin/testing-helpers';
 import { resetMouse, sendKeys, sendMouse } from '@web/test-runner-commands';
 import sinon from 'sinon';
-import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
+import { isTouch } from '@scoped-vaadin/component-base/src/browser-utils.js';
 import { getFirstItem, setInputValue } from './helpers.js';
 
 describe('interactions', () => {
@@ -24,7 +24,7 @@
   beforeEach(async () => {
     comboBox = fixtureSync(
       `<div>
-        <vaadin-combo-box label="Label"></vaadin-combo-box>
+        <vaadin24-combo-box label="Label"></vaadin24-combo-box>
         <input id="last-global-focusable" />
       </div>`,
     ).firstElementChild;
