Index: test/internal-filtering.test.js
===================================================================
--- test/internal-filtering.test.js
+++ test/internal-filtering.test.js
@@ -11,7 +11,7 @@
 
   describe('value is set before', () => {
     beforeEach(() => {
-      comboBox = fixtureSync(`<vaadin-combo-box value="foo"></vaadin-combo-box>`);
+      comboBox = fixtureSync(`<vaadin24-combo-box value="foo"></vaadin24-combo-box>`);
       comboBox.items = ['foo', 'bar'];
     });
 
@@ -26,7 +26,7 @@
 
   describe('setting the input field value', () => {
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = ['foo', 'bar', 'baz'];
     });
 
@@ -124,7 +124,7 @@
 
   describe('focusing items while filtering', () => {
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = ['foo', 'bar', 'baz'];
     });
 
@@ -174,7 +174,7 @@
     let overlay;
 
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = ['foo', 'bar', 'baz'];
       overlay = comboBox.$.overlay;
     });
@@ -282,7 +282,7 @@
 
   describe('setting items when opened', () => {
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = [];
     });
 
@@ -295,7 +295,7 @@
 
   describe('setting placeholder items', () => {
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = [new ComboBoxPlaceholder(), new ComboBoxPlaceholder()];
     });
 
