Index: test/interactions.test.js
===================================================================
--- test/interactions.test.js
+++ test/interactions.test.js
@@ -16,7 +16,7 @@
 } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import '../src/vaadin-combo-box.js';
-import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
+import { isTouch } from '@scoped-vaadin/component-base/src/browser-utils.js';
 import { getFirstItem, setInputValue } from './helpers.js';
 
 describe('interactions', () => {
@@ -25,7 +25,7 @@
   beforeEach(async () => {
     comboBox = fixtureSync(
       `<div>
-        <vaadin-combo-box label="Label"></vaadin-combo-box>
+        <vaadin24-combo-box label="Label"></vaadin24-combo-box>
         <input id="last-global-focusable" />
       </div>`,
     ).firstElementChild;
