Index: test/item-template.test.js
===================================================================
--- test/item-template.test.js
+++ test/item-template.test.js
@@ -1,7 +1,7 @@
 import { expect } from '@esm-bundle/chai';
 import { arrowDownKeyDown, fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import '@vaadin/polymer-legacy-adapter/template-renderer.js';
+import '@scoped-vaadin/polymer-legacy-adapter/template-renderer.js';
 import './not-animated-styles.js';
 import './fixtures/mock-combo-box-template-wrapper.js';
 import './fixtures/mock-combo-box-light-template-wrapper.js';
