Index: test/item-class-name-generator.common.js
===================================================================
--- test/item-class-name-generator.common.js
+++ test/item-class-name-generator.common.js
@@ -6,7 +6,7 @@
   let comboBox;
 
   beforeEach(async () => {
-    comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+    comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
     await nextRender();
     comboBox.items = ['foo', 'bar', 'baz'];
   });
