Index: test/keyboard.common.js
===================================================================
--- test/keyboard.common.js
+++ test/keyboard.common.js
@@ -24,7 +24,7 @@
   beforeEach(async () => {
     [comboBox, lastGlobalFocusable] = fixtureSync(
       `<div>
-        <vaadin-combo-box></vaadin-combo-box>
+        <vaadin24-combo-box></vaadin24-combo-box>
         <input id="last-global-focusable" />
       </div>`,
     ).children;
