Index: test/lazy-loading.common.js
===================================================================
--- test/lazy-loading.common.js
+++ test/lazy-loading.common.js
@@ -1306,7 +1306,7 @@
 
   describe('combo-box', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       await nextRender();
     });
 
@@ -1316,9 +1316,9 @@
   describe('combo-box-light', () => {
     beforeEach(async () => {
       comboBox = fixtureSync(`
-        <vaadin-combo-box-light>
-          <vaadin-text-field></vaadin-text-field>
-        </vaadin-combo-box-light>
+        <vaadin24-combo-box-light>
+          <vaadin24-text-field></vaadin24-text-field>
+        </vaadin24-combo-box-light>
       `);
       await nextRender();
     });
