Index: test/lazy-loading.test.js
===================================================================
--- test/lazy-loading.test.js
+++ test/lazy-loading.test.js
@@ -1,11 +1,11 @@
 import { expect } from '@esm-bundle/chai';
 import { arrowDownKeyDown, aTimeout, enterKeyDown, fixtureSync, nextFrame, nextRender } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import '@vaadin/text-field/vaadin-text-field.js';
+import '@scoped-vaadin/text-field/vaadin-text-field.js';
 import './not-animated-styles.js';
 import '../vaadin-combo-box.js';
 import '../vaadin-combo-box-light.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ComboBoxPlaceholder } from '../src/vaadin-combo-box-placeholder.js';
 import {
   clickItem,
@@ -1289,7 +1289,7 @@
 
   describe('combo-box', () => {
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
     });
 
     describeLazyLoading();
@@ -1298,9 +1298,9 @@
   describe('combo-box-light', () => {
     beforeEach(async () => {
       comboBox = fixtureSync(`
-        <vaadin-combo-box-light>
-          <vaadin-text-field></vaadin-text-field>
-        </vaadin-combo-box-light>
+        <vaadin24-combo-box-light>
+          <vaadin24-text-field></vaadin24-text-field>
+        </vaadin24-combo-box-light>
       `);
       await nextRender();
     });
