Index: test/lazy-loading-polymer.test.js
===================================================================
--- test/lazy-loading-polymer.test.js
+++ test/lazy-loading-polymer.test.js
@@ -1,6 +1,6 @@
 import './not-animated-styles.js';
 import './lazy-loading-styles.js';
-import '@vaadin/text-field/vaadin-text-field.js';
+import '@scoped-vaadin/text-field/vaadin-text-field.js';
 import '../vaadin-combo-box-light.js';
 import '../vaadin-combo-box.js';
 import './lazy-loading.common.js';
