Index: test/lazy-loading-styles.js
===================================================================
--- test/lazy-loading-styles.js
+++ test/lazy-loading-styles.js
@@ -1,4 +1,4 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
   'vaadin-combo-box*',
