Index: test/lit-renderer-directives.test.js
===================================================================
--- test/lit-renderer-directives.test.js
+++ test/lit-renderer-directives.test.js
@@ -8,14 +8,14 @@
 
 async function renderComboBox(container, { items }) {
   render(
-    html`<vaadin-combo-box
+    html`<vaadin23-combo-box
       .items="${items}"
       ${items ? comboBoxRenderer((item) => html`${item}`, items) : null}
-    ></vaadin-combo-box>`,
+    ></vaadin23-combo-box>`,
     container,
   );
   await nextFrame();
-  return container.querySelector('vaadin-combo-box');
+  return container.querySelector('vaadin23-combo-box');
 }
 
 describe('lit renderer directives', () => {
@@ -60,11 +60,11 @@
       beforeEach(async () => {
         rendererSpy = sinon.spy();
         render(
-          html`<vaadin-combo-box opened .items="${['Item']}" ${comboBoxRenderer(rendererSpy)}></vaadin-combo-box>`,
+          html`<vaadin23-combo-box opened .items="${['Item']}" ${comboBoxRenderer(rendererSpy)}></vaadin23-combo-box>`,
           container,
         );
         await nextFrame();
-        comboBox = container.querySelector('vaadin-combo-box');
+        comboBox = container.querySelector('vaadin23-combo-box');
       });
 
       it('should pass the item to the renderer', () => {
