Index: test/object-values.test.js
===================================================================
--- test/object-values.test.js
+++ test/object-values.test.js
@@ -10,7 +10,7 @@
 
   describe('label and value paths', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       input = comboBox.inputElement;
 
       comboBox.items = [
@@ -293,7 +293,7 @@
     });
 
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       input = comboBox.inputElement;
       comboBox.itemValuePath = undefined;
       comboBox.itemLabelPath = undefined;
