Index: test/overlay-opening.common.js
===================================================================
--- test/overlay-opening.common.js
+++ test/overlay-opening.common.js
@@ -6,7 +6,7 @@
   let comboBox, overlay, input;
 
   beforeEach(async () => {
-    comboBox = fixtureSync('<vaadin-combo-box label="Label"></vaadin-combo-box>');
+    comboBox = fixtureSync('<vaadin24-combo-box label="Label"></vaadin24-combo-box>');
     await nextRender();
     comboBox.items = ['foo', 'bar', 'baz'];
     input = comboBox.inputElement;
