Index: test/selecting-items.test.js
===================================================================
--- test/selecting-items.test.js
+++ test/selecting-items.test.js
@@ -10,7 +10,7 @@
   let valueChangedSpy, selectedItemChangedSpy, selectionChangedSpy, changeSpy;
 
   beforeEach(() => {
-    comboBox = fixtureSync('<vaadin-combo-box style="width: 320px"></vaadin-combo-box>');
+    comboBox = fixtureSync('<vaadin23-combo-box style="width: 320px"></vaadin23-combo-box>');
     comboBox.items = ['foo', 'bar'];
 
     valueChangedSpy = sinon.spy();
@@ -249,7 +249,7 @@
   let comboBox, clearIcon;
 
   beforeEach(() => {
-    comboBox = fixtureSync('<vaadin-combo-box style="width: 320px" clear-button-visible></vaadin-combo-box>');
+    comboBox = fixtureSync('<vaadin23-combo-box style="width: 320px" clear-button-visible></vaadin23-combo-box>');
     comboBox.items = ['foo', 'bar'];
     comboBox.value = 'foo';
     clearIcon = comboBox.$.clearButton;
@@ -283,7 +283,7 @@
   it('should de-select dropdown item after clearing a selection', () => {
     comboBox.open();
 
-    const item = document.querySelector('vaadin-combo-box-item');
+    const item = document.querySelector('vaadin23-combo-box-item');
     expect(item.hasAttribute('selected')).to.be.true;
 
     clearIcon.click();
@@ -310,7 +310,7 @@
   let comboBox;
 
   beforeEach(async () => {
-    comboBox = fixtureSync('<vaadin-combo-box style="width: 320px" allow-custom-value></vaadin-combo-box>');
+    comboBox = fixtureSync('<vaadin23-combo-box style="width: 320px" allow-custom-value></vaadin23-combo-box>');
     comboBox.items = ['foo', 'bar', 'barbar'];
     comboBox.open();
     await aTimeout(0);
