Index: test/validation.test.js
===================================================================
--- test/validation.test.js
+++ test/validation.test.js
@@ -12,7 +12,7 @@
     let validateSpy, changeSpy;
 
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = ['foo', 'bar', 'baz'];
       input = comboBox.inputElement;
       validateSpy = sinon.spy(comboBox, 'validate');
@@ -117,7 +117,7 @@
 
   describe('required', () => {
     beforeEach(() => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = ['foo', 'bar', 'baz'];
       comboBox.required = true;
       input = comboBox.inputElement;
@@ -149,7 +149,7 @@
     let validateSpy;
 
     beforeEach(() => {
-      comboBox = document.createElement('vaadin-combo-box');
+      comboBox = document.createElement('vaadin24-combo-box');
       comboBox.allowCustomValue = true;
       validateSpy = sinon.spy(comboBox, 'validate');
     });
