Index: test/validation.test.js
===================================================================
--- test/validation.test.js
+++ test/validation.test.js
@@ -11,7 +11,7 @@
     let validateSpy, changeSpy;
 
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = ['foo', 'bar', 'baz'];
       await nextRender();
       input = comboBox.inputElement;
@@ -118,7 +118,7 @@
 
   describe('required', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = ['foo', 'bar', 'baz'];
       comboBox.required = true;
       await nextRender();
@@ -151,7 +151,7 @@
     let validateSpy;
 
     beforeEach(() => {
-      comboBox = document.createElement('vaadin-combo-box');
+      comboBox = document.createElement('vaadin24-combo-box');
       comboBox.allowCustomValue = true;
       validateSpy = sinon.spy(comboBox, 'validate');
     });
