Index: test/validation.common.js
===================================================================
--- test/validation.common.js
+++ test/validation.common.js
@@ -10,7 +10,7 @@
     let validateSpy, changeSpy;
 
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = ['foo', 'bar', 'baz'];
       await nextRender();
       input = comboBox.inputElement;
@@ -117,7 +117,7 @@
 
   describe('required', () => {
     beforeEach(async () => {
-      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+      comboBox = fixtureSync('<vaadin24-combo-box></vaadin24-combo-box>');
       comboBox.items = ['foo', 'bar', 'baz'];
       comboBox.required = true;
       await nextRender();
@@ -150,7 +150,7 @@
     let validateSpy;
 
     beforeEach(() => {
-      comboBox = document.createElement('vaadin-combo-box');
+      comboBox = document.createElement('vaadin24-combo-box');
       comboBox.allowCustomValue = true;
       validateSpy = sinon.spy(comboBox, 'validate');
     });
