Index: test/visual/lumo/input-container.test.js
===================================================================
--- test/visual/lumo/input-container.test.js
+++ test/visual/lumo/input-container.test.js
@@ -1,8 +1,8 @@
 import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
 import { visualDiff } from '@web/test-runner-visual-regression';
-import '@vaadin/icon/vaadin-icon.js';
-import '@vaadin/vaadin-lumo-styles/test/autoload.js';
-import '@vaadin/vaadin-lumo-styles/vaadin-iconset.js';
+import '@scoped-vaadin/icon/vaadin-icon.js';
+import '@scoped-vaadin/vaadin-lumo-styles/test/autoload.js';
+import '@scoped-vaadin/vaadin-lumo-styles/vaadin-iconset.js';
 import '../../../theme/lumo/vaadin-input-container.js';
 
 describe('input-container', () => {
@@ -12,7 +12,7 @@
     div = document.createElement('div');
     div.style.width = 'fit-content';
     div.style.padding = '10px';
-    element = fixtureSync('<vaadin-input-container><input></vaadin-input-container>', div);
+    element = fixtureSync('<vaadin24-input-container><input></vaadin24-input-container>', div);
     input = element.querySelector('input');
   });
 
@@ -36,7 +36,7 @@
   });
 
   it('prefix icon', async () => {
-    const icon = document.createElement('vaadin-icon');
+    const icon = document.createElement('vaadin24-icon');
     icon.setAttribute('slot', 'prefix');
     icon.icon = 'lumo:user';
     element.appendChild(icon);
@@ -44,7 +44,7 @@
   });
 
   it('suffix icon', async () => {
-    const icon = document.createElement('vaadin-icon');
+    const icon = document.createElement('vaadin24-icon');
     icon.setAttribute('slot', 'suffix');
     icon.icon = 'lumo:user';
     element.appendChild(icon);
