Index: test/dom/__snapshots__/message.test.snap.js
===================================================================
--- test/dom/__snapshots__/message.test.snap.js
+++ test/dom/__snapshots__/message.test.snap.js
@@ -2,13 +2,13 @@
 export const snapshots = {};
 
 snapshots["vaadin-message default"] = 
-`<vaadin-message-avatar
+`<vaadin23-message-avatar
   aria-hidden="true"
   part="avatar"
   role="button"
   tabindex="-1"
 >
-</vaadin-message-avatar>
+</vaadin23-message-avatar>
 <div part="content">
   <div part="header">
     <span part="name">
@@ -25,7 +25,7 @@
 /* end snapshot vaadin-message default */
 
 snapshots["vaadin-message userName"] = 
-`<vaadin-message-avatar
+`<vaadin23-message-avatar
   abbr="JD"
   aria-hidden="true"
   name="Joan Doe"
@@ -33,7 +33,7 @@
   role="button"
   tabindex="-1"
 >
-</vaadin-message-avatar>
+</vaadin23-message-avatar>
 <div part="content">
   <div part="header">
     <span part="name">
@@ -51,14 +51,14 @@
 /* end snapshot vaadin-message userName */
 
 snapshots["vaadin-message userAbbr"] = 
-`<vaadin-message-avatar
+`<vaadin23-message-avatar
   abbr="JD"
   aria-hidden="true"
   part="avatar"
   role="button"
   tabindex="-1"
 >
-</vaadin-message-avatar>
+</vaadin23-message-avatar>
 <div part="content">
   <div part="header">
     <span part="name">
@@ -75,14 +75,14 @@
 /* end snapshot vaadin-message userAbbr */
 
 snapshots["vaadin-message userImg"] = 
-`<vaadin-message-avatar
+`<vaadin23-message-avatar
   aria-hidden="true"
   img="data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs="
   part="avatar"
   role="button"
   tabindex="-1"
 >
-</vaadin-message-avatar>
+</vaadin23-message-avatar>
 <div part="content">
   <div part="header">
     <span part="name">
@@ -99,14 +99,14 @@
 /* end snapshot vaadin-message userImg */
 
 snapshots["vaadin-message userColorIndex"] = 
-`<vaadin-message-avatar
+`<vaadin23-message-avatar
   aria-hidden="true"
   has-color-index=""
   part="avatar"
   role="button"
   tabindex="-1"
 >
-</vaadin-message-avatar>
+</vaadin23-message-avatar>
 <div part="content">
   <div part="header">
     <span part="name">
@@ -123,13 +123,13 @@
 /* end snapshot vaadin-message userColorIndex */
 
 snapshots["vaadin-message time"] = 
-`<vaadin-message-avatar
+`<vaadin23-message-avatar
   aria-hidden="true"
   part="avatar"
   role="button"
   tabindex="-1"
 >
-</vaadin-message-avatar>
+</vaadin23-message-avatar>
 <div part="content">
   <div part="header">
     <span part="name">
