Index: test/dom/__snapshots__/message.test.snap.js
===================================================================
--- test/dom/__snapshots__/message.test.snap.js
+++ test/dom/__snapshots__/message.test.snap.js
@@ -58,8 +58,8 @@
 /* end snapshot vaadin-message time */
 
 snapshots["vaadin-message avatar username"] = 
-`<vaadin-message>
-  <vaadin-avatar
+`<vaadin24-message>
+  <vaadin24-avatar
     abbr="JD"
     aria-hidden="true"
     name="Joan Doe"
@@ -67,42 +67,42 @@
     slot="avatar"
     tabindex="-1"
   >
-  </vaadin-avatar>
-</vaadin-message>
+  </vaadin24-avatar>
+</vaadin24-message>
 `;
 /* end snapshot vaadin-message avatar username */
 
 snapshots["vaadin-message avatar abbr"] = 
-`<vaadin-message>
-  <vaadin-avatar
+`<vaadin24-message>
+  <vaadin24-avatar
     abbr="JD"
     aria-hidden="true"
     role="button"
     slot="avatar"
     tabindex="-1"
   >
-  </vaadin-avatar>
-</vaadin-message>
+  </vaadin24-avatar>
+</vaadin24-message>
 `;
 /* end snapshot vaadin-message avatar abbr */
 
 snapshots["vaadin-message avatar img"] = 
-`<vaadin-message>
-  <vaadin-avatar
+`<vaadin24-message>
+  <vaadin24-avatar
     aria-hidden="true"
     img="data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs="
     role="button"
     slot="avatar"
     tabindex="-1"
   >
-  </vaadin-avatar>
-</vaadin-message>
+  </vaadin24-avatar>
+</vaadin24-message>
 `;
 /* end snapshot vaadin-message avatar img */
 
 snapshots["vaadin-message avatar userColorIndex"] = 
-`<vaadin-message>
-  <vaadin-avatar
+`<vaadin24-message>
+  <vaadin24-avatar
     aria-hidden="true"
     has-color-index=""
     role="button"
@@ -110,8 +110,8 @@
     style="--vaadin-avatar-user-color: var(--vaadin-user-color-2);"
     tabindex="-1"
   >
-  </vaadin-avatar>
-</vaadin-message>
+  </vaadin24-avatar>
+</vaadin24-message>
 `;
 /* end snapshot vaadin-message avatar userColorIndex */
 
