Index: test/draggable-resizable.common.js
===================================================================
--- test/draggable-resizable.common.js
+++ test/draggable-resizable.common.js
@@ -1,9 +1,10 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextFrame, nextRender, nextUpdate } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import '@vaadin/text-area/vaadin-text-area.js';
+import '@scoped-vaadin/text-area/vaadin-text-area.js';
 
-customElements.define(
+internalCustomElements.define(
   'internally-draggable',
   class extends HTMLElement {
     constructor() {
@@ -51,8 +52,8 @@
   beforeEach(async () => {
     wrapper = fixtureSync(`
       <div>
-        <vaadin-dialog modeless draggable opened></vaadin-dialog>
-        <vaadin-dialog modeless draggable opened></vaadin-dialog>
+        <vaadin24-dialog modeless draggable opened></vaadin24-dialog>
+        <vaadin24-dialog modeless draggable opened></vaadin24-dialog>
       </div>
     `);
     await nextRender();
@@ -116,7 +117,7 @@
 
   beforeEach(async () => {
     dialog = fixtureSync(`
-      <vaadin-dialog resizable modeless></vaadin-dialog>
+      <vaadin24-dialog resizable modeless></vaadin24-dialog>
     `);
     await nextRender();
     dialog.renderer = (root) => {
@@ -387,7 +388,7 @@
 
   beforeEach(async () => {
     dialog = fixtureSync(`
-      <vaadin-dialog draggable modeless></vaadin-dialog>
+      <vaadin24-dialog draggable modeless></vaadin24-dialog>
     `);
     await nextRender();
     dialog.renderer = (root) => {
@@ -667,7 +668,7 @@
 
   function getFrontmostOverlayFromScreenCenter() {
     let elementFromPoint = document.elementFromPoint(window.innerWidth / 2, window.innerHeight / 2);
-    while (elementFromPoint && elementFromPoint.localName !== 'vaadin-dialog-overlay') {
+    while (elementFromPoint && elementFromPoint.localName !== 'vaadin24-dialog-overlay') {
       elementFromPoint = elementFromPoint.host || elementFromPoint.parentNode;
     }
     return elementFromPoint;
@@ -682,12 +683,12 @@
     draggableContainer;
 
   beforeEach(async () => {
-    resizable = fixtureSync('<vaadin-dialog resizable modeless></vaadin-dialog>');
+    resizable = fixtureSync('<vaadin24-dialog resizable modeless></vaadin24-dialog>');
     resizable.renderer = (root) => {
       root.innerHTML = `<div>Resizable dialog</div>`;
     };
 
-    draggable = fixtureSync('<vaadin-dialog draggable modeless></vaadin-dialog>');
+    draggable = fixtureSync('<vaadin24-dialog draggable modeless></vaadin24-dialog>');
     draggable.renderer = (root) => {
       root.innerHTML = `
         <div>Draggable dialog</div>
@@ -793,8 +794,8 @@
   beforeEach(async () => {
     wrapper = fixtureSync(`
       <div>
-        <vaadin-dialog id="modalDialog"></vaadin-dialog>
-        <vaadin-dialog id="modelessDialog" modeless></vaadin-dialog>
+        <vaadin24-dialog id="modalDialog"></vaadin24-dialog>
+        <vaadin24-dialog id="modelessDialog" modeless></vaadin24-dialog>
       </div>
     `);
     await nextRender();
@@ -837,7 +838,7 @@
 
   beforeEach(async () => {
     dialog = fixtureSync(`
-      <vaadin-dialog resizable opened modeless></vaadin-dialog>
+      <vaadin24-dialog resizable opened modeless></vaadin24-dialog>
     `);
     await nextRender();
 
@@ -858,7 +859,7 @@
   });
 
   it('should not overflow when using vaadin-textarea in the content', async () => {
-    const textarea = document.createElement('vaadin-text-area');
+    const textarea = document.createElement('vaadin24-text-area');
     textarea.value = Array(20).join('Lorem ipsum dolor sit amet');
     overlay.appendChild(textarea);
     overlay.$.content.style.padding = '20px';
