Index: test/renderer.test.js
===================================================================
--- test/renderer.test.js
+++ test/renderer.test.js
@@ -7,7 +7,7 @@
   let dialog, overlay;
 
   beforeEach(async () => {
-    dialog = fixtureSync('<vaadin-dialog></vaadin-dialog>');
+    dialog = fixtureSync('<vaadin24-dialog></vaadin24-dialog>');
     await nextRender();
     overlay = dialog.$.overlay;
   });
@@ -37,7 +37,7 @@
   });
 
   it('should not throw when requesting content update for an unupgraded dialog', () => {
-    const dialog = document.createElement('vaadin-dialog');
+    const dialog = document.createElement('vaadin24-dialog');
 
     expect(() => dialog.requestContentUpdate()).not.to.throw();
   });
