Index: test/message-list-markdown-import.test.js
===================================================================
--- test/message-list-markdown-import.test.js
+++ test/message-list-markdown-import.test.js
@@ -23,7 +23,7 @@
   ];
 
   beforeEach(() => {
-    messageList = fixtureSync('<vaadin-message-list></vaadin-message-list>');
+    messageList = fixtureSync('<vaadin24-message-list></vaadin24-message-list>');
   });
 
   it('should not render incorrect content during import', async () => {
