Index: app-layout/theme/lumo/vaadin-drawer-toggle-styles.js
===================================================================
--- app-layout/theme/lumo/vaadin-drawer-toggle-styles.js
+++ app-layout/theme/lumo/vaadin-drawer-toggle-styles.js
@@ -1,6 +1,6 @@
-import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import { button } from '@vaadin/button/theme/lumo/vaadin-button-styles.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import '@scoped-vaadin/vaadin-lumo-styles/font-icons.js';
+import { button } from '@scoped-vaadin/button/theme/lumo/vaadin-button-styles.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const drawerToggle = css`
   :host {
@@ -33,4 +33,4 @@
   }
 `;
 
-registerStyles('vaadin-drawer-toggle', [button, drawerToggle], { moduleId: 'lumo-drawer-toggle' });
+registerStyles('vaadin24-drawer-toggle', [button, drawerToggle], { moduleId: 'lumo-drawer-toggle' });
