Index: test/a11y.test.js
===================================================================
--- test/a11y.test.js
+++ test/a11y.test.js
@@ -12,7 +12,7 @@
 } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import './not-animated-styles.js';
-import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
+import { getDeepActiveElement } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
 import { Popover } from '../src/vaadin-popover.js';
 import { mouseenter, mouseleave } from './helpers.js';
 
@@ -26,7 +26,7 @@
   });
 
   beforeEach(async () => {
-    popover = fixtureSync('<vaadin-popover></vaadin-popover>');
+    popover = fixtureSync('<vaadin24-popover></vaadin24-popover>');
     target = fixtureSync('<button>Target</button>');
     popover.target = target;
     popover.renderer = (root) => {
@@ -35,7 +35,7 @@
       }
     };
     await nextRender();
-    overlay = popover.shadowRoot.querySelector('vaadin-popover-overlay');
+    overlay = popover.shadowRoot.querySelector('vaadin24-popover-overlay');
   });
 
   describe('ARIA attributes', () => {
