Index: test/lit-renderer-directives.test.js
===================================================================
--- test/lit-renderer-directives.test.js
+++ test/lit-renderer-directives.test.js
@@ -9,11 +9,11 @@
 async function renderOpenedPopover(container, { content }) {
   render(
     html`
-      <vaadin-popover opened ${content ? popoverRenderer(() => html`${content}`, content) : nothing}></vaadin-popover>
+      <vaadin24-popover opened ${content ? popoverRenderer(() => html`${content}`, content) : nothing}></vaadin24-popover>
     `,
     container,
   );
-  const popover = container.querySelector('vaadin-popover');
+  const popover = container.querySelector('vaadin24-popover');
   await nextUpdate(popover);
   return popover;
 }
@@ -56,9 +56,9 @@
 
       beforeEach(async () => {
         rendererSpy = sinon.spy();
-        render(html`<vaadin-popover opened ${popoverRenderer(rendererSpy)}></vaadin-popover>`, container);
+        render(html`<vaadin24-popover opened ${popoverRenderer(rendererSpy)}></vaadin24-popover>`, container);
         await nextFrame();
-        popover = container.querySelector('vaadin-popover');
+        popover = container.querySelector('vaadin24-popover');
       });
 
       it('should pass the popover instance to the renderer', () => {
