Index: test/nested.test.js
===================================================================
--- test/nested.test.js
+++ test/nested.test.js
@@ -14,7 +14,7 @@
   });
 
   beforeEach(async () => {
-    popover = fixtureSync('<vaadin-popover></vaadin-popover>');
+    popover = fixtureSync('<vaadin24-popover></vaadin24-popover>');
     target = fixtureSync('<button>Target</button>');
     popover.target = target;
     popover.renderer = (root) => {
@@ -23,7 +23,7 @@
       }
       root.innerHTML = `
         <button id="second-target">Second target</button>
-        <vaadin-popover for="second-target"></vaadin-popover>
+        <vaadin24-popover for="second-target"></vaadin24-popover>
       `;
       [secondTarget, secondPopover] = root.children;
       secondPopover.renderer = (root2) => {
