Index: test/position.test.js
===================================================================
--- test/position.test.js
+++ test/position.test.js
@@ -7,14 +7,14 @@
   let popover, target, overlay;
 
   beforeEach(async () => {
-    popover = fixtureSync('<vaadin-popover></vaadin-popover>');
+    popover = fixtureSync('<vaadin24-popover></vaadin24-popover>');
     popover.renderer = (root) => {
       root.textContent = 'Content';
     };
     target = fixtureSync('<div style="width: 100px; height: 100px; margin: 100px; outline: 1px solid red;"></div>');
     popover.target = target;
     await nextRender();
-    overlay = popover.shadowRoot.querySelector('vaadin-popover-overlay');
+    overlay = popover.shadowRoot.querySelector('vaadin24-popover-overlay');
   });
 
   // Overlay above the target (position="top-*")
