Index: test/accessibility.test.js
===================================================================
--- test/accessibility.test.js
+++ test/accessibility.test.js
@@ -7,13 +7,13 @@
   let grid;
 
   function initGridRenderer(grouped) {
-    grid = fixtureSync('<vaadin-grid></vaadin-grid>');
-    const col1 = document.createElement('vaadin-grid-column');
-    const col2 = document.createElement('vaadin-grid-column');
-    const col3 = document.createElement('vaadin-grid-column');
+    grid = fixtureSync('<vaadin24-grid></vaadin24-grid>');
+    const col1 = document.createElement('vaadin24-grid-column');
+    const col2 = document.createElement('vaadin24-grid-column');
+    const col3 = document.createElement('vaadin24-grid-column');
     col3.setAttribute('row-header', '');
     if (grouped) {
-      const grp = document.createElement('vaadin-grid-column-group');
+      const grp = document.createElement('vaadin24-grid-column-group');
       grp.headerRenderer = (root) => {
         root.textContent = 'Group header';
       };
@@ -103,7 +103,7 @@
       });
 
       it('should change role from rowheader to gridcell when `rowHeader` is set to false', () => {
-        const columns = grid.querySelectorAll('vaadin-grid-column');
+        const columns = grid.querySelectorAll('vaadin24-grid-column');
         columns[2].rowHeader = false;
         expect(grid.$.items.children[0].children[2].getAttribute('role')).to.equal('gridcell');
         expect(grid.$.items.children[1].children[2].getAttribute('role')).to.equal('gridcell');
@@ -168,9 +168,9 @@
 
     beforeEach(() => {
       grid = fixtureSync(`
-        <vaadin-grid item-id-path="name">
-          <vaadin-grid-column path="name" width="200px" flex-shrink="0"></vaadin-grid-column>
-        </vaadin-grid>
+        <vaadin24-grid item-id-path="name">
+          <vaadin24-grid-column path="name" width="200px" flex-shrink="0"></vaadin24-grid-column>
+        </vaadin24-grid>
       `);
       grid.dataProvider = hierarchicalDataProvider;
       flushGrid(grid);
@@ -231,7 +231,7 @@
       });
 
       it('should update aria-colcount when column is added', async () => {
-        const column = document.createElement('vaadin-grid-column');
+        const column = document.createElement('vaadin24-grid-column');
         column.renderer = (root, _, model) => {
           root.textContent = model.item;
         };
@@ -244,7 +244,7 @@
         const emptyStateText = document.createElement('span');
         emptyStateText.setAttribute('slot', 'empty-state');
         grid.appendChild(emptyStateText);
-        grid.querySelectorAll('vaadin-grid-column').forEach((col) => {
+        grid.querySelectorAll('vaadin24-grid-column').forEach((col) => {
           col.headerRenderer = null;
           col.footerRenderer = null;
         });
@@ -273,8 +273,8 @@
     let col;
 
     beforeEach(() => {
-      grid = fixtureSync('<vaadin-grid></vaadin-grid>');
-      col = document.createElement('vaadin-grid-column');
+      grid = fixtureSync('<vaadin24-grid></vaadin24-grid>');
+      col = document.createElement('vaadin24-grid-column');
       grid.appendChild(col);
       grid.items = [{ value: 'foo' }, { value: 'bar' }];
     });
@@ -329,7 +329,7 @@
 
     it('should count the footer rows correctly as part of the aria-rowcount', async () => {
       // Remove the header renderers
-      grid.querySelectorAll('vaadin-grid-column').forEach((col) => {
+      grid.querySelectorAll('vaadin24-grid-column').forEach((col) => {
         col.headerRenderer = null;
       });
 
@@ -339,7 +339,7 @@
 
     it('should count the header rows correctly as part of the aria-rowcount', async () => {
       // Remove the footer renderers
-      grid.querySelectorAll('vaadin-grid-column').forEach((col) => {
+      grid.querySelectorAll('vaadin24-grid-column').forEach((col) => {
         col.footerRenderer = null;
       });
 
