Index: test/array-data-provider.common.js
===================================================================
--- test/array-data-provider.common.js
+++ test/array-data-provider.common.js
@@ -9,10 +9,10 @@
   describe('with items', () => {
     beforeEach(async () => {
       grid = fixtureSync(`
-        <vaadin-grid>
-          <vaadin-grid-column path="name.first" header="First"></vaadin-grid-column>
-          <vaadin-grid-column path="name.last" header="Last"></vaadin-grid-column>
-        </vaadin-grid>
+        <vaadin24-grid>
+          <vaadin24-grid-column path="name.first" header="First"></vaadin24-grid-column>
+          <vaadin24-grid-column path="name.last" header="Last"></vaadin24-grid-column>
+        </vaadin24-grid>
       `);
       grid.items = [
         {
@@ -85,10 +85,10 @@
   describe('without items', () => {
     beforeEach(async () => {
       grid = fixtureSync(`
-        <vaadin-grid>
-          <vaadin-grid-column path="name.first" header="First"></vaadin-grid-column>
-          <vaadin-grid-column path="name.last" header="Last"></vaadin-grid-column>
-        </vaadin-grid>
+        <vaadin24-grid>
+          <vaadin24-grid-column path="name.first" header="First"></vaadin24-grid-column>
+          <vaadin24-grid-column path="name.last" header="Last"></vaadin24-grid-column>
+        </vaadin24-grid>
       `);
       flushGrid(grid);
       await nextFrame();
@@ -108,10 +108,10 @@
 
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid>
-        <vaadin-grid-column path="name.first"></vaadin-grid-column>
-        <vaadin-grid-column path="name.last"></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid>
+        <vaadin24-grid-column path="name.first"></vaadin24-grid-column>
+        <vaadin24-grid-column path="name.last"></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
 
     grid.items = [
@@ -122,19 +122,19 @@
         },
       },
     ];
-    const columns = grid.querySelectorAll('vaadin-grid-column');
+    const columns = grid.querySelectorAll('vaadin24-grid-column');
     columns[0].headerRenderer = (root) => {
       if (!root.firstChild) {
         root.innerHTML = `
-          <vaadin-grid-sorter>
+          <vaadin24-grid-sorter>
             Sorter with invalid path
-          </vaadin-grid-sorter>
+          </vaadin24-grid-sorter>
         `;
       }
     };
     columns[1].headerRenderer = (root) => {
       if (!root.firstChild) {
-        root.innerHTML = '<vaadin-grid-filter value="foo"></vaadin-grid-filter>';
+        root.innerHTML = '<vaadin24-grid-filter value="foo"></vaadin24-grid-filter>';
       }
     };
     flushGrid(grid);
@@ -153,7 +153,7 @@
     let sorter;
 
     beforeEach(() => {
-      sorter = grid.querySelector('vaadin-grid-sorter');
+      sorter = grid.querySelector('vaadin24-grid-sorter');
     });
 
     it('should warn about invalid path with undefined parent property', () => {
@@ -199,7 +199,7 @@
     let filter;
 
     beforeEach(() => {
-      filter = grid.querySelector('vaadin-grid-filter');
+      filter = grid.querySelector('vaadin24-grid-filter');
       filter.path = '';
       filter._debouncerFilterChanged.flush();
     });
@@ -231,9 +231,9 @@
 
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid>
-        <vaadin-grid-column path="value"></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid>
+        <vaadin24-grid-column path="value"></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
     await nextFrame();
   });
