Index: test/column-groups.test.js
===================================================================
--- test/column-groups.test.js
+++ test/column-groups.test.js
@@ -1,7 +1,8 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextFrame, nextRender } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import '@vaadin/polymer-legacy-adapter/template-renderer.js';
+import '@scoped-vaadin/polymer-legacy-adapter/template-renderer.js';
 import '../vaadin-grid.js';
 import '../vaadin-grid-column-group.js';
 import {
@@ -16,120 +17,120 @@
 } from './helpers.js';
 
 const createColumn = () => {
-  const column = document.createElement('vaadin-grid-column');
-  column.innerHTML = '<template class="footer">Foobar</template></vaadin-grid-column>';
+  const column = document.createElement('vaadin24-grid-column');
+  column.innerHTML = '<template class="footer">Foobar</template></vaadin24-grid-column>';
   return column;
 };
 
 const fixtures = {
   'no-headers': `
-    <vaadin-grid size="10">
-      <vaadin-grid-column>
+    <vaadin24-grid size="10">
+      <vaadin24-grid-column>
         <template>body-0</template>
-      </vaadin-grid-column>
-      <vaadin-grid-column-group>
-        <vaadin-grid-column>
+      </vaadin24-grid-column>
+      <vaadin24-grid-column-group>
+        <vaadin24-grid-column>
           <template>body-1</template>
-        </vaadin-grid-column>
-        <vaadin-grid-column>
+        </vaadin24-grid-column>
+        <vaadin24-grid-column>
           <template>body-2</template>
-        </vaadin-grid-column>
-      </vaadin-grid-column-group>
-    </vaadin-grid>
+        </vaadin24-grid-column>
+      </vaadin24-grid-column-group>
+    </vaadin24-grid>
   `,
   '1-column-1-group': `
-    <vaadin-grid size="10">
-      <vaadin-grid-column frozen>
+    <vaadin24-grid size="10">
+      <vaadin24-grid-column frozen>
         <template class="header">header-0</template>
         <template>body-0</template>
-      </vaadin-grid-column>
+      </vaadin24-grid-column>
 
-      <vaadin-grid-column-group frozen>
+      <vaadin24-grid-column-group frozen>
         <template class="header">group-0</template>
 
-        <vaadin-grid-column>
+        <vaadin24-grid-column>
           <template class="header">header-1</template>
           <template>body-1</template>
-        </vaadin-grid-column>
-        <vaadin-grid-column>
+        </vaadin24-grid-column>
+        <vaadin24-grid-column>
           <template class="header">header-2</template>
           <template>body-2</template>
-        </vaadin-grid-column>
-      </vaadin-grid-column-group>
-    </vaadin-grid>
+        </vaadin24-grid-column>
+      </vaadin24-grid-column-group>
+    </vaadin24-grid>
   `,
   nested: `
-    <vaadin-grid size="100">
-      <vaadin-grid-column-group>
+    <vaadin24-grid size="100">
+      <vaadin24-grid-column-group>
         <template class="header">group-0</template>
         <template class="footer">group-0</template>
-        <vaadin-grid-column>
+        <vaadin24-grid-column>
           <template>body-0</template>
-        </vaadin-grid-column>
-        <vaadin-grid-column>
+        </vaadin24-grid-column>
+        <vaadin24-grid-column>
           <template>body-1</template>
-        </vaadin-grid-column>
-      </vaadin-grid-column-group>
+        </vaadin24-grid-column>
+      </vaadin24-grid-column-group>
 
-      <vaadin-grid-column-group>
+      <vaadin24-grid-column-group>
         <template class="header">group-1</template>
         <template class="footer">group-1</template>
-        <vaadin-grid-column-group>
+        <vaadin24-grid-column-group>
           <template class="header">group-2</template>
           <template class="footer">group-2</template>
-          <vaadin-grid-column>
+          <vaadin24-grid-column>
             <template>body-2</template>
-          </vaadin-grid-column>
-          <vaadin-grid-column>
+          </vaadin24-grid-column>
+          <vaadin24-grid-column>
             <template>body-3</template>
-          </vaadin-grid-column>
-        </vaadin-grid-column-group>
+          </vaadin24-grid-column>
+        </vaadin24-grid-column-group>
 
-        <vaadin-grid-column-group frozen>
+        <vaadin24-grid-column-group frozen>
           <template class="header">group-3</template>
           <template class="footer">group-3</template>
-          <vaadin-grid-column>
+          <vaadin24-grid-column>
             <template>body-4</template>
             <template class="header">header-4</template>
             <template class="footer">footer-4</template>
-          </vaadin-grid-column>
-          <vaadin-grid-column>
+          </vaadin24-grid-column>
+          <vaadin24-grid-column>
             <template>body-5</template>
             <template class="header">header-5</template>
             <template class="footer">footer-5</template>
-          </vaadin-grid-column>
-        </vaadin-grid-column-group>
-      </vaadin-grid-column-group>
-      <vaadin-grid-column>
+          </vaadin24-grid-column>
+        </vaadin24-grid-column-group>
+      </vaadin24-grid-column-group>
+      <vaadin24-grid-column>
         <template>body-6</template>
         <template class="header">header-6</template>
         <template class="footer">footer-6</template>
-      </vaadin-grid-column>
-    </vaadin-grid>
+      </vaadin24-grid-column>
+    </vaadin24-grid>
   `,
   'hidden-group': `
-    <vaadin-grid>
-      <vaadin-grid-column-group hidden>
-        <vaadin-grid-column>
+    <vaadin24-grid>
+      <vaadin24-grid-column-group hidden>
+        <vaadin24-grid-column>
           <template></template>
-        </vaadin-grid-column>
-        <vaadin-grid-column>
+        </vaadin24-grid-column>
+        <vaadin24-grid-column>
           <template></template>
-        </vaadin-grid-column>
-      </vaadin-grid-column-group>
-    </vaadin-grid>
+        </vaadin24-grid-column>
+      </vaadin24-grid-column-group>
+    </vaadin24-grid>
   `,
   'hidden-column': `
-    <vaadin-grid>
-      <vaadin-grid-column-group>
-        <vaadin-grid-column hidden>
+    <vaadin24-grid>
+      <vaadin24-grid-column-group>
+        <vaadin24-grid-column hidden>
           <template></template>
-        </vaadin-grid-column>
-        <vaadin-grid-column>
+        </vaadin24-grid-column>
+        <vaadin24-grid-column>
           <template></template>
-        </vaadin-grid-column>
-      </vaadin-grid-column-group>
-    </vaadin-grid>
+        </vaadin24-grid-column>
+      </vaadin24-grid-column-group>
+    </vaadin24-grid>
   `,
 };
 
@@ -208,7 +209,7 @@
       let emptyGroup;
 
       beforeEach(() => {
-        emptyGroup = grid.querySelector('vaadin-grid-column-group');
+        emptyGroup = grid.querySelector('vaadin24-grid-column-group');
         emptyGroup.header = 'Header';
       });
 
@@ -239,7 +240,7 @@
       let emptyGroup;
 
       beforeEach(() => {
-        emptyGroup = grid.querySelector('vaadin-grid-column-group');
+        emptyGroup = grid.querySelector('vaadin24-grid-column-group');
         emptyGroup.headerRenderer = (root) => {
           root.textContent = 'header';
         };
@@ -334,7 +335,7 @@
     });
 
     it('should have a column frozen to end', () => {
-      const col = grid.querySelector('vaadin-grid-column');
+      const col = grid.querySelector('vaadin24-grid-column');
       col.frozen = false;
       col.frozenToEnd = true;
       expect(getHeaderCell(0, 0).hasAttribute('frozen-to-end')).to.be.true;
@@ -342,7 +343,7 @@
     });
 
     it('should have a group frozen to end', () => {
-      const group = grid.querySelector('vaadin-grid-column-group');
+      const group = grid.querySelector('vaadin24-grid-column-group');
       group.frozen = false;
       group.frozenToEnd = true;
       expect(getHeaderCell(0, 1).hasAttribute('frozen-to-end')).to.be.true;
@@ -500,15 +501,15 @@
     it('should hide children', async () => {
       init('hidden-group');
       await nextFrame();
-      expect(grid.querySelector('vaadin-grid-column').hidden).to.be.true;
-      expect(grid.querySelector('vaadin-grid-column-group').hidden).to.be.true;
+      expect(grid.querySelector('vaadin24-grid-column').hidden).to.be.true;
+      expect(grid.querySelector('vaadin24-grid-column-group').hidden).to.be.true;
     });
 
     it('should hide self', async () => {
       init('hidden-column');
       await nextFrame();
-      expect(grid.querySelector('vaadin-grid-column-group').hidden).not.to.be.true;
-      const columns = grid.querySelectorAll('vaadin-grid-column');
+      expect(grid.querySelector('vaadin24-grid-column-group').hidden).not.to.be.true;
+      const columns = grid.querySelectorAll('vaadin24-grid-column');
       expect(columns[0].hidden).to.be.true;
       expect(columns[1].hidden).not.to.be.true;
     });
@@ -519,22 +520,22 @@
 
     beforeEach(() => {
       grid = fixtureSync(`
-        <vaadin-grid>
-          <vaadin-grid-column-group header="groupA">
+        <vaadin24-grid>
+          <vaadin24-grid-column-group header="groupA">
             ${Array.from(Array(13).keys())
               .slice(1)
               .map((group) => {
                 const col = (group - 1) * 2;
                 return `
-                  <vaadin-grid-column-group header="group${group - 1}">
-                    <vaadin-grid-column id="col${col}" header="Col ${col}"></vaadin-grid-column>
-                    <vaadin-grid-column id="col${col + 1}" header="Col ${col + 1}"></vaadin-grid-column>
-                  </vaadin-grid-column-group>
+                  <vaadin24-grid-column-group header="group${group - 1}">
+                    <vaadin24-grid-column id="col${col}" header="Col ${col}"></vaadin24-grid-column>
+                    <vaadin24-grid-column id="col${col + 1}" header="Col ${col + 1}"></vaadin24-grid-column>
+                  </vaadin24-grid-column-group>
                 `;
               })
               .join('')}
-          </vaadin-grid-column-group>
-        </vaadin-grid>
+          </vaadin24-grid-column-group>
+        </vaadin24-grid>
       `);
     });
 
@@ -551,17 +552,17 @@
 
     beforeEach(() => {
       grid = fixtureSync(`
-        <vaadin-grid>
+        <vaadin24-grid>
           ${Array.from(Array(101).keys())
             .map(
               (col) => `
-              <vaadin-grid-column-group>
-                <vaadin-grid-column id="col${col}" header="Col ${col}"></vaadin-grid-column>
-              </vaadin-grid-column-group>
+              <vaadin24-grid-column-group>
+                <vaadin24-grid-column id="col${col}" header="Col ${col}"></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
               `,
             )
             .join('')}
-        </vaadin-grid>
+        </vaadin24-grid>
       `);
     });
 
@@ -578,14 +579,14 @@
       describe(frozenAttr, () => {
         it('should not warn when grouping frozen only columns', async () => {
           fixtureSync(`
-            <vaadin-grid>
-              <vaadin-grid-column-group>
-                <vaadin-grid-column ${frozenAttr}></vaadin-grid-column>
-                <vaadin-grid-column ${frozenAttr}></vaadin-grid-column>
-              </vaadin-grid-column-group>
+            <vaadin24-grid>
+              <vaadin24-grid-column-group>
+                <vaadin24-grid-column ${frozenAttr}></vaadin24-grid-column>
+                <vaadin24-grid-column ${frozenAttr}></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
     
-              <vaadin-grid-column></vaadin-grid-column>
-            </vaadin-grid>
+              <vaadin24-grid-column></vaadin24-grid-column>
+            </vaadin24-grid>
           `);
 
           await nextFrame();
@@ -594,14 +595,14 @@
 
         it('should not warn when freezing a root level group', async () => {
           fixtureSync(`
-            <vaadin-grid>
-              <vaadin-grid-column-group ${frozenAttr}>
-                <vaadin-grid-column></vaadin-grid-column>
-                <vaadin-grid-column></vaadin-grid-column>
-              </vaadin-grid-column-group>
+            <vaadin24-grid>
+              <vaadin24-grid-column-group ${frozenAttr}>
+                <vaadin24-grid-column></vaadin24-grid-column>
+                <vaadin24-grid-column></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
     
-              <vaadin-grid-column></vaadin-grid-column>
-            </vaadin-grid>
+              <vaadin24-grid-column></vaadin24-grid-column>
+            </vaadin24-grid>
           `);
 
           await nextFrame();
@@ -610,14 +611,14 @@
 
         it('should not warn when freezing a root level column', async () => {
           fixtureSync(`
-            <vaadin-grid>
-              <vaadin-grid-column-group>
-                <vaadin-grid-column></vaadin-grid-column>
-                <vaadin-grid-column></vaadin-grid-column>
-              </vaadin-grid-column-group>
+            <vaadin24-grid>
+              <vaadin24-grid-column-group>
+                <vaadin24-grid-column></vaadin24-grid-column>
+                <vaadin24-grid-column></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
     
-              <vaadin-grid-column ${frozenAttr}></vaadin-grid-column>
-            </vaadin-grid>
+              <vaadin24-grid-column ${frozenAttr}></vaadin24-grid-column>
+            </vaadin24-grid>
           `);
 
           await nextFrame();
@@ -625,17 +626,17 @@
         });
 
         it('should not warn when grouping a frozen column with unimported frozen column', async () => {
-          expect(customElements.get('vaadin-grid-selection-column')).to.be.undefined;
+          expect(internalCustomElements.get('vaadin24-grid-selection-column')).to.be.undefined;
 
           fixtureSync(`
-            <vaadin-grid>
-              <vaadin-grid-column-group>
-                <vaadin-grid-selection-column ${frozenAttr}></vaadin-grid-selection-column>
-                <vaadin-grid-column ${frozenAttr}></vaadin-grid-column>
-              </vaadin-grid-column-group>
+            <vaadin24-grid>
+              <vaadin24-grid-column-group>
+                <vaadin24-grid-selection-column ${frozenAttr}></vaadin24-grid-selection-column>
+                <vaadin24-grid-column ${frozenAttr}></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
     
-              <vaadin-grid-column></vaadin-grid-column>
-            </vaadin-grid>
+              <vaadin24-grid-column></vaadin24-grid-column>
+            </vaadin24-grid>
           `);
 
           await nextFrame();
@@ -644,14 +645,14 @@
 
         it('should warn when grouping frozen and non-frozen columns', async () => {
           fixtureSync(`
-            <vaadin-grid>
-              <vaadin-grid-column-group>
-                <vaadin-grid-column ${frozenAttr}></vaadin-grid-column>
-                <vaadin-grid-column></vaadin-grid-column>
-              </vaadin-grid-column-group>
+            <vaadin24-grid>
+              <vaadin24-grid-column-group>
+                <vaadin24-grid-column ${frozenAttr}></vaadin24-grid-column>
+                <vaadin24-grid-column></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
 
-              <vaadin-grid-column></vaadin-grid-column>
-            </vaadin-grid>
+              <vaadin24-grid-column></vaadin24-grid-column>
+            </vaadin24-grid>
           `);
 
           await nextFrame();
@@ -659,20 +660,20 @@
         });
 
         it('should warn when grouping a frozen column with an unimported column', async () => {
-          expect(customElements.get('vaadin-grid-selection-column')).to.be.undefined;
+          expect(internalCustomElements.get('vaadin24-grid-selection-column')).to.be.undefined;
 
           // Testing the other way around (frozen unimported column with non-frozen column)
           // is not feasible because an unimported column never notifies its parent group
           // of its frozen state. A warning about the missing import is logged instead.
           fixtureSync(`
-            <vaadin-grid>
-              <vaadin-grid-column-group>
-                <vaadin-grid-selection-column></vaadin-grid-selection-column>
-                <vaadin-grid-column ${frozenAttr}></vaadin-grid-column>
-              </vaadin-grid-column-group>
+            <vaadin24-grid>
+              <vaadin24-grid-column-group>
+                <vaadin24-grid-selection-column></vaadin24-grid-selection-column>
+                <vaadin24-grid-column ${frozenAttr}></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
     
-              <vaadin-grid-column></vaadin-grid-column>
-            </vaadin-grid>
+              <vaadin24-grid-column></vaadin24-grid-column>
+            </vaadin24-grid>
           `);
 
           await nextFrame();
@@ -681,16 +682,16 @@
 
         it('should warn when grouping frozen group and non-frozen column', async () => {
           fixtureSync(`
-            <vaadin-grid>
-              <vaadin-grid-column-group>
-                <vaadin-grid-column-group ${frozenAttr}>
-                  <vaadin-grid-column></vaadin-grid-column>
-                  <vaadin-grid-column></vaadin-grid-column>
-                </vaadin-grid-column-group>
+            <vaadin24-grid>
+              <vaadin24-grid-column-group>
+                <vaadin24-grid-column-group ${frozenAttr}>
+                  <vaadin24-grid-column></vaadin24-grid-column>
+                  <vaadin24-grid-column></vaadin24-grid-column>
+                </vaadin24-grid-column-group>
 
-                <vaadin-grid-column></vaadin-grid-column>
-              </vaadin-grid-column-group>
-            </vaadin-grid>
+                <vaadin24-grid-column></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
+            </vaadin24-grid>
           `);
 
           await nextFrame();
@@ -699,16 +700,16 @@
 
         it('should not warn when grouping frozen group and frozen column', async () => {
           fixtureSync(`
-            <vaadin-grid>
-              <vaadin-grid-column-group>
-                <vaadin-grid-column-group ${frozenAttr}>
-                  <vaadin-grid-column></vaadin-grid-column>
-                  <vaadin-grid-column></vaadin-grid-column>
-                </vaadin-grid-column-group>
+            <vaadin24-grid>
+              <vaadin24-grid-column-group>
+                <vaadin24-grid-column-group ${frozenAttr}>
+                  <vaadin24-grid-column></vaadin24-grid-column>
+                  <vaadin24-grid-column></vaadin24-grid-column>
+                </vaadin24-grid-column-group>
 
-                <vaadin-grid-column ${frozenAttr}></vaadin-grid-column>
-              </vaadin-grid-column-group>
-            </vaadin-grid>
+                <vaadin24-grid-column ${frozenAttr}></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
+            </vaadin24-grid>
           `);
 
           await nextFrame();
@@ -717,17 +718,17 @@
 
         it('should only warn once', async () => {
           fixtureSync(`
-            <vaadin-grid>
-              <vaadin-grid-column-group>
-                <vaadin-grid-column ${frozenAttr}></vaadin-grid-column>
-                <vaadin-grid-column></vaadin-grid-column>
-              </vaadin-grid-column-group>
+            <vaadin24-grid>
+              <vaadin24-grid-column-group>
+                <vaadin24-grid-column ${frozenAttr}></vaadin24-grid-column>
+                <vaadin24-grid-column></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
 
-              <vaadin-grid-column-group>
-                <vaadin-grid-column ${frozenAttr}></vaadin-grid-column>
-                <vaadin-grid-column></vaadin-grid-column>
-              </vaadin-grid-column-group>
-            </vaadin-grid>
+              <vaadin24-grid-column-group>
+                <vaadin24-grid-column ${frozenAttr}></vaadin24-grid-column>
+                <vaadin24-grid-column></vaadin24-grid-column>
+              </vaadin24-grid-column-group>
+            </vaadin24-grid>
           `);
 
           await nextFrame();
