Index: test/column-observer.test.js
===================================================================
--- test/column-observer.test.js
+++ test/column-observer.test.js
@@ -1,10 +1,11 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@esm-bundle/chai';
 import { aTimeout, fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import { ColumnObserver } from '../src/vaadin-grid-helpers.js';
 
 function createColumn() {
-  return document.createElement('vaadin-grid-column');
+  return document.createElement('vaadin24-grid-column');
 }
 
 class Wrapper extends HTMLElement {
@@ -16,7 +17,7 @@
   }
 }
 
-customElements.define('wrapper-component', Wrapper);
+internalCustomElements.define('wrapper-component', Wrapper);
 
 describe('column observer', () => {
   let wrapper;
