Index: test/deprecated-api.common.js
===================================================================
--- test/deprecated-api.common.js
+++ test/deprecated-api.common.js
@@ -10,9 +10,9 @@
     sinon.stub(console, 'warn');
 
     grid = fixtureSync(`
-      <vaadin-grid size="2">
-        <vaadin-grid-column></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid size="2">
+        <vaadin24-grid-column></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
     grid.dataProvider = ({ parentItem }, callback) => {
       if (parentItem) {
@@ -41,7 +41,7 @@
       grid._loadPage(0, grid._dataProviderController.rootCache);
       expect(console.warn).to.be.calledOnce;
       expect(console.warn).to.be.calledWith(
-        '<vaadin-grid> The `_loadPage` method is deprecated and will be removed in Vaadin 25.',
+        '<vaadin24-grid> The `_loadPage` method is deprecated and will be removed in Vaadin 25.',
       );
     });
   });
@@ -55,7 +55,7 @@
       const result = grid._effectiveSize; // eslint-disable-line @typescript-eslint/no-unused-vars
       expect(console.warn).to.be.calledOnce;
       expect(console.warn).to.be.calledWith(
-        '<vaadin-grid> The `_effectiveSize` property is deprecated and will be removed in Vaadin 25.',
+        '<vaadin24-grid> The `_effectiveSize` property is deprecated and will be removed in Vaadin 25.',
       );
     });
   });
@@ -69,7 +69,7 @@
       const result = grid._cache; // eslint-disable-line @typescript-eslint/no-unused-vars
       expect(console.warn).to.be.calledOnce;
       expect(console.warn).to.be.calledWith(
-        '<vaadin-grid> The `_cache` property is deprecated and will be removed in Vaadin 25.',
+        '<vaadin24-grid> The `_cache` property is deprecated and will be removed in Vaadin 25.',
       );
     });
   });
@@ -83,7 +83,7 @@
       const result = grid._cache.effectiveSize; // eslint-disable-line @typescript-eslint/no-unused-vars
       expect(console.warn).to.be.calledTwice;
       expect(console.warn.lastCall).to.be.calledWith(
-        '<vaadin-grid> The `effectiveSize` property of ItemCache is deprecated and will be removed in Vaadin 25.',
+        '<vaadin24-grid> The `effectiveSize` property of ItemCache is deprecated and will be removed in Vaadin 25.',
       );
     });
   });
@@ -97,7 +97,7 @@
       grid._cache.getItemForIndex(1);
       expect(console.warn).to.be.calledTwice;
       expect(console.warn.lastCall).to.be.calledWith(
-        '<vaadin-grid> The `getItemForIndex` method of ItemCache is deprecated and will be removed in Vaadin 25.',
+        '<vaadin24-grid> The `getItemForIndex` method of ItemCache is deprecated and will be removed in Vaadin 25.',
       );
     });
   });
@@ -113,7 +113,7 @@
       grid._cache.updateSize();
       expect(console.warn).to.be.calledTwice;
       expect(console.warn.lastCall).to.be.calledWith(
-        '<vaadin-grid> The `updateSize` method of ItemCache is deprecated and will be removed in Vaadin 25.',
+        '<vaadin24-grid> The `updateSize` method of ItemCache is deprecated and will be removed in Vaadin 25.',
       );
     });
   });
@@ -129,7 +129,7 @@
       grid._cache.getCacheAndIndex(1);
       expect(console.warn).to.be.calledTwice;
       expect(console.warn.lastCall).to.be.calledWith(
-        '<vaadin-grid> The `getCacheAndIndex` method of ItemCache is deprecated and will be removed in Vaadin 25.',
+        '<vaadin24-grid> The `getCacheAndIndex` method of ItemCache is deprecated and will be removed in Vaadin 25.',
       );
     });
   });
@@ -146,7 +146,7 @@
       grid._cache.ensureSubCacheForScaledIndex(1);
       expect(console.warn).to.be.calledTwice;
       expect(console.warn.lastCall).to.be.calledWith(
-        '<vaadin-grid> The `ensureSubCacheForScaledIndex` method of ItemCache is deprecated and will be removed in Vaadin 25.',
+        '<vaadin24-grid> The `ensureSubCacheForScaledIndex` method of ItemCache is deprecated and will be removed in Vaadin 25.',
       );
     });
   });
@@ -160,7 +160,7 @@
       const result = grid._cache.grid; // eslint-disable-line @typescript-eslint/no-unused-vars
       expect(console.warn).to.be.calledTwice;
       expect(console.warn.lastCall).to.be.calledWith(
-        '<vaadin-grid> The `grid` property of ItemCache is deprecated and will be removed in Vaadin 25.',
+        '<vaadin24-grid> The `grid` property of ItemCache is deprecated and will be removed in Vaadin 25.',
       );
     });
   });
@@ -175,7 +175,7 @@
       const result = grid._cache.itemCaches; // eslint-disable-line @typescript-eslint/no-unused-vars
       expect(console.warn).to.be.calledTwice;
       expect(console.warn.lastCall).to.be.calledWith(
-        '<vaadin-grid> The `itemCaches` property of ItemCache is deprecated and will be removed in Vaadin 25.',
+        '<vaadin24-grid> The `itemCaches` property of ItemCache is deprecated and will be removed in Vaadin 25.',
       );
     });
   });
