Index: test/disabled.test.js
===================================================================
--- test/disabled.test.js
+++ test/disabled.test.js
@@ -11,10 +11,10 @@
 
   beforeEach(() => {
     grid = fixtureSync(`
-      <vaadin-grid>
-        <vaadin-grid-filter-column path="foo"></vaadin-grid-filter-column>
-        <vaadin-grid-sort-column path="bar"></vaadin-grid-sort-column>
-      </vaadin-grid>
+      <vaadin23-grid>
+        <vaadin23-grid-filter-column path="foo"></vaadin23-grid-filter-column>
+        <vaadin23-grid-sort-column path="bar"></vaadin23-grid-sort-column>
+      </vaadin23-grid>
     `);
 
     grid.items = [
