Index: test/drag-and-drop.test.js
===================================================================
--- test/drag-and-drop.test.js
+++ test/drag-and-drop.test.js
@@ -107,10 +107,10 @@
 
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid hidden>
-        <vaadin-grid-column path="first" header="First name"></vaadin-grid-column>
-        <vaadin-grid-column path="last" header="Last name"></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin23-grid hidden>
+        <vaadin23-grid-column path="first" header="First name"></vaadin23-grid-column>
+        <vaadin23-grid-column path="last" header="Last name"></vaadin23-grid-column>
+      </vaadin23-grid>
     `);
     await nextFrame();
     grid.hidden = false;
@@ -315,7 +315,7 @@
 
       it('should auto generate data transfer text data in order', () => {
         grid.selectedItems = grid.items;
-        const columns = grid.querySelectorAll('vaadin-grid-column');
+        const columns = grid.querySelectorAll('vaadin23-grid-column');
         grid._swapColumnOrders(columns[0], columns[1]);
         fireDragStart();
         fireDrop();
