Index: test/event-context.common.js
===================================================================
--- test/event-context.common.js
+++ test/event-context.common.js
@@ -7,17 +7,17 @@
 
   beforeEach(() => {
     grid = fixtureSync(`
-      <vaadin-grid>
-        <vaadin-grid-column-group header="column group header">
-          <vaadin-grid-column path="foo" header="column header"></vaadin-grid-column>
-        </vaadin-grid-column-group>
-      </vaadin-grid>
+      <vaadin24-grid>
+        <vaadin24-grid-column-group header="column group header">
+          <vaadin24-grid-column path="foo" header="column header"></vaadin24-grid-column>
+        </vaadin24-grid-column-group>
+      </vaadin24-grid>
     `);
-    column = grid.querySelector('vaadin-grid-column');
+    column = grid.querySelector('vaadin24-grid-column');
     column.footerRenderer = (root) => {
       root.textContent = 'column footer';
     };
-    columnGroup = grid.querySelector('vaadin-grid-column-group');
+    columnGroup = grid.querySelector('vaadin24-grid-column-group');
     columnGroup.footerRenderer = (root) => {
       root.textContent = 'column group footer';
     };
