Index: test/frozen-columns.common.js
===================================================================
--- test/frozen-columns.common.js
+++ test/frozen-columns.common.js
@@ -2,8 +2,8 @@
 import { fixtureSync, listenOnce, nextRender, nextResize } from '@vaadin/testing-helpers';
 import { resetMouse, sendMouse } from '@web/test-runner-commands';
 import sinon from 'sinon';
-import { isElementFocused } from '@vaadin/a11y-base/src/focus-utils.js';
-import { setNormalizedScrollLeft } from '@vaadin/component-base/src/dir-utils.js';
+import { isElementFocused } from '@scoped-vaadin/a11y-base/src/focus-utils.js';
+import { setNormalizedScrollLeft } from '@scoped-vaadin/component-base/src/dir-utils.js';
 import { flushGrid, getRowCells, getRows, infiniteDataProvider, isWithinParentConstraints } from './helpers.js';
 
 // Returns true if the element's computed transform style matches with the
@@ -20,15 +20,15 @@
 
 const frozenGridFixture = (frozen, frozenToEnd) => {
   const grid = fixtureSync(`
-    <vaadin-grid style="width: 200px; height: 400px;" size="10">
-      <vaadin-grid-column ${frozen ? 'frozen ' : ''} header="foo"></vaadin-grid-column>
-      <vaadin-grid-column header="bar"></vaadin-grid-column>
-      <vaadin-grid-column ${frozenToEnd ? 'frozen-to-end' : ''} header="baz"></vaadin-grid-column>
-    </vaadin-grid>
+    <vaadin24-grid style="width: 200px; height: 400px;" size="10">
+      <vaadin24-grid-column ${frozen ? 'frozen ' : ''} header="foo"></vaadin24-grid-column>
+      <vaadin24-grid-column header="bar"></vaadin24-grid-column>
+      <vaadin24-grid-column ${frozenToEnd ? 'frozen-to-end' : ''} header="baz"></vaadin24-grid-column>
+    </vaadin24-grid>
   `);
 
   const values = ['foo', 'bar', 'baz'];
-  const columns = grid.querySelectorAll('vaadin-grid-column');
+  const columns = grid.querySelectorAll('vaadin24-grid-column');
   columns.forEach((col, idx) => {
     col.renderer = (root) => {
       root.textContent = values[idx];
