Index: test/grid-test-styles.js
===================================================================
--- test/grid-test-styles.js
+++ test/grid-test-styles.js
@@ -1,9 +1,9 @@
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 // TODO: subset of Lumo needed for unit tests to pass.
 // These should be eventually covered by base styles.
 registerStyles(
-  'vaadin-grid',
+  'vaadin24-grid',
   css`
     :host {
       --_lumo-grid-border-width: 0px;
