Index: test/helpers.js
===================================================================
--- test/helpers.js
+++ test/helpers.js
@@ -194,7 +194,7 @@
 export const dragStart = (source) => {
   let grid = source.parentElement;
   while (grid) {
-    if (grid.localName === 'vaadin-grid') {
+    if (grid.localName === 'vaadin24-grid') {
       grid._touchDevice = false;
     }
     grid = grid.parentNode || grid.host;
