Index: test/keyboard-navigation-cell-button.test.js
===================================================================
--- test/keyboard-navigation-cell-button.test.js
+++ test/keyboard-navigation-cell-button.test.js
@@ -19,12 +19,12 @@
 
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid>
-        <vaadin-grid-column path="name"></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid>
+        <vaadin24-grid-column path="name"></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
 
-    const column = document.createElement('vaadin-grid-column');
+    const column = document.createElement('vaadin24-grid-column');
     column.path = 'name';
     column._focusButtonMode = true;
     grid.insertBefore(column, grid.firstElementChild);
