Index: test/keyboard-navigation-row-focus.common.js
===================================================================
--- test/keyboard-navigation-row-focus.common.js
+++ test/keyboard-navigation-row-focus.common.js
@@ -115,10 +115,10 @@
 describe('keyboard navigation - row focus', () => {
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid>
-        <vaadin-grid-tree-column path="name" header="" width="100px" flex-shrink="0"></vaadin-grid-tree-column>
-        <vaadin-grid-column path="name" width="200px" flex-shrink="0"></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid>
+        <vaadin24-grid-tree-column path="name" header="" width="100px" flex-shrink="0"></vaadin24-grid-tree-column>
+        <vaadin24-grid-column path="name" width="200px" flex-shrink="0"></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
 
     grid.dataProvider = hierarchicalDataProvider;
@@ -416,16 +416,16 @@
 describe('keyboard navigation on column groups - row focus', () => {
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid>
-        <vaadin-grid-column-group header="main group header">
-          <vaadin-grid-column-group header="sub group header">
-            <vaadin-grid-column header="column header"></vaadin-grid-column>
-          </vaadin-grid-column-group>
-        </vaadin-grid-column-group>
-      </vaadin-grid>
+      <vaadin24-grid>
+        <vaadin24-grid-column-group header="main group header">
+          <vaadin24-grid-column-group header="sub group header">
+            <vaadin24-grid-column header="column header"></vaadin24-grid-column>
+          </vaadin24-grid-column-group>
+        </vaadin24-grid-column-group>
+      </vaadin24-grid>
     `);
     grid.items = ['foo', 'bar'];
-    grid.querySelector('vaadin-grid-column').renderer = (root, _, model) => {
+    grid.querySelector('vaadin24-grid-column').renderer = (root, _, model) => {
       root.textContent = model.item;
     };
     flushGrid(grid);
@@ -448,9 +448,9 @@
 
       beforeEach(() => {
         header = grid.$.header;
-        mainGroup = grid.querySelector('vaadin-grid-column-group');
-        subGroup = mainGroup.querySelector('vaadin-grid-column-group');
-        column = subGroup.querySelector('vaadin-grid-column');
+        mainGroup = grid.querySelector('vaadin24-grid-column-group');
+        subGroup = mainGroup.querySelector('vaadin24-grid-column-group');
+        column = subGroup.querySelector('vaadin24-grid-column');
       });
 
       it('should update tabbable header row on header row hide', async () => {
@@ -540,9 +540,9 @@
 ['header', 'footer', 'body'].forEach((section) => {
   describe(`empty grid - row focus - ${section}`, () => {
     beforeEach(() => {
-      grid = fixtureSync(`<vaadin-grid>
-        <vaadin-grid-column></vaadin-grid-column>
-      </vaadin-grid>`);
+      grid = fixtureSync(`<vaadin24-grid>
+        <vaadin24-grid-column></vaadin24-grid-column>
+      </vaadin24-grid>`);
       const column = grid.firstElementChild;
 
       if (section === 'header') {
