Index: test/lit-renderers.test.js
===================================================================
--- test/lit-renderers.test.js
+++ test/lit-renderers.test.js
@@ -14,9 +14,9 @@
 
   beforeEach(() => {
     grid = fixtureSync(`
-      <vaadin-grid>
-        <vaadin-grid-column></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin23-grid>
+        <vaadin23-grid-column></vaadin23-grid-column>
+      </vaadin23-grid>
     `);
     grid.items = ['item1'];
     column = grid.firstElementChild;
