Index: test/lit.test.js
===================================================================
--- test/lit.test.js
+++ test/lit.test.js
@@ -12,11 +12,11 @@
     function renderGrid(columnPaths, items) {
       render(
         html`
-          <vaadin-grid .items=${items}>
+          <vaadin24-grid .items=${items}>
             ${columnPaths.map((columnPath) => {
-              return html`<vaadin-grid-column path=${columnPath}></vaadin-grid-column>`;
+              return html`<vaadin24-grid-column path=${columnPath}></vaadin24-grid-column>`;
             })}
-          </vaadin-grid>
+          </vaadin24-grid>
         `,
         wrapper,
       );
@@ -40,11 +40,11 @@
     function renderGrid(columnPaths, items) {
       render(
         html`
-          <vaadin-grid .items=${items}>
+          <vaadin24-grid .items=${items}>
             ${columnPaths.map((columnPath) => {
-              return html`<vaadin-grid-sort-column path="${columnPath}"></vaadin-grid-sort-column>`;
+              return html`<vaadin24-grid-sort-column path="${columnPath}"></vaadin24-grid-sort-column>`;
             })}
-          </vaadin-grid>
+          </vaadin24-grid>
         `,
         wrapper,
       );
@@ -55,7 +55,7 @@
 
     await aTimeout(0);
 
-    document.querySelector('vaadin-grid-sorter').click();
+    document.querySelector('vaadin24-grid-sorter').click();
 
     // Then render a grid with one column
     renderGrid(['c1'], [{ c1: '1-1', c2: '1-2' }]);
