Index: test/lit.test.js
===================================================================
--- test/lit.test.js
+++ test/lit.test.js
@@ -11,11 +11,11 @@
     function renderGrid(columnPaths, items) {
       render(
         html`
-          <vaadin-grid .items=${items}>
+          <vaadin23-grid .items=${items}>
             ${columnPaths.map((columnPath) => {
-              return html`<vaadin-grid-column path=${columnPath}></vaadin-grid-column>`;
+              return html`<vaadin23-grid-column path=${columnPath}></vaadin23-grid-column>`;
             })}
-          </vaadin-grid>
+          </vaadin23-grid>
         `,
         wrapper,
       );
