Index: test/min-height.test.js
===================================================================
--- test/min-height.test.js
+++ test/min-height.test.js
@@ -32,9 +32,9 @@
 
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid style="height: 0;">
-        <vaadin-grid-column></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid style="height: 0;">
+        <vaadin24-grid-column></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
     flushGrid(grid);
     await nextResize(grid);
@@ -48,7 +48,7 @@
 
   describe('with header', () => {
     beforeEach(async () => {
-      grid.querySelector('vaadin-grid-column').header = 'Header';
+      grid.querySelector('vaadin24-grid-column').header = 'Header';
       flushGrid(grid);
       await nextResize(grid);
     });
@@ -60,7 +60,7 @@
 
   describe('with footer', () => {
     beforeEach(async () => {
-      grid.querySelector('vaadin-grid-column').footerRenderer = (root) => {
+      grid.querySelector('vaadin24-grid-column').footerRenderer = (root) => {
         root.textContent = 'Footer';
       };
       flushGrid(grid);
@@ -74,8 +74,8 @@
 
   describe('with header and footer', () => {
     beforeEach(async () => {
-      grid.querySelector('vaadin-grid-column').header = 'Header';
-      grid.querySelector('vaadin-grid-column').footerRenderer = (root) => {
+      grid.querySelector('vaadin24-grid-column').header = 'Header';
+      grid.querySelector('vaadin24-grid-column').footerRenderer = (root) => {
         root.textContent = 'Footer';
       };
       flushGrid(grid);
@@ -89,8 +89,8 @@
 
   describe('with data', () => {
     beforeEach(async () => {
-      grid.querySelector('vaadin-grid-column').path = 'value';
-      grid.querySelector('vaadin-grid-column').header = null;
+      grid.querySelector('vaadin24-grid-column').path = 'value';
+      grid.querySelector('vaadin24-grid-column').header = null;
       grid.dataProvider = infiniteDataProvider;
       flushGrid(grid);
       await nextResize(grid);
