Index: test/resizing-material.test.js
===================================================================
--- test/resizing-material.test.js
+++ test/resizing-material.test.js
@@ -8,11 +8,11 @@
 
   beforeEach(() => {
     grid = fixtureSync(`
-      <vaadin-grid size="10">
-        <vaadin-grid-column header="Name"></vaadin-grid-column>
-        <vaadin-grid-column header="Surname"></vaadin-grid-column>
-        <vaadin-grid-column header="Effort" resizable></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid size="10">
+        <vaadin24-grid-column header="Name"></vaadin24-grid-column>
+        <vaadin24-grid-column header="Surname"></vaadin24-grid-column>
+        <vaadin24-grid-column header="Effort" resizable></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
     grid.dataProvider = infiniteDataProvider;
     flushGrid(grid);
