Index: test/row-height.common.js
===================================================================
--- test/row-height.common.js
+++ test/row-height.common.js
@@ -5,23 +5,23 @@
 const fixtures = {
   defaultContent: () => {
     const grid = fixtureSync(`
-      <vaadin-grid style="width: 200px; height: 200px;" size="10000" hidden>
-        <vaadin-grid-column header="foo"></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid style="width: 200px; height: 200px;" size="10000" hidden>
+        <vaadin24-grid-column header="foo"></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
-    grid.querySelector('vaadin-grid-column').renderer = (root) => {
+    grid.querySelector('vaadin24-grid-column').renderer = (root) => {
       root.innerHTML = '<div>foobar</div>';
     };
     return grid;
   },
   twoColumns: () => {
     const grid = fixtureSync(`
-      <vaadin-grid style="width: 200px; height: 200px;" size="10000" hidden>
-        <vaadin-grid-column header="foo"></vaadin-grid-column>
-        <vaadin-grid-column></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid style="width: 200px; height: 200px;" size="10000" hidden>
+        <vaadin24-grid-column header="foo"></vaadin24-grid-column>
+        <vaadin24-grid-column></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
-    grid.querySelectorAll('vaadin-grid-column').forEach((col) => {
+    grid.querySelectorAll('vaadin24-grid-column').forEach((col) => {
       col.renderer = (root) => {
         root.innerHTML = '<div>foo</div>';
       };
@@ -30,12 +30,12 @@
   },
   unevenContent: () => {
     const grid = fixtureSync(`
-      <vaadin-grid style="width: 200px; height: 200px;" size="10000" hidden>
-        <vaadin-grid-column header="foo"></vaadin-grid-column>
-        <vaadin-grid-column header="foo"></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid style="width: 200px; height: 200px;" size="10000" hidden>
+        <vaadin24-grid-column header="foo"></vaadin24-grid-column>
+        <vaadin24-grid-column header="foo"></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
-    const columns = grid.querySelectorAll('vaadin-grid-column');
+    const columns = grid.querySelectorAll('vaadin24-grid-column');
     columns[0].renderer = (root) => {
       root.innerHTML = '<div style="line-height: 2em;">foobar</div>';
     };
@@ -108,7 +108,7 @@
 
   it('should update height when hiding columns', async () => {
     await init(fixtures.unevenContent);
-    grid.querySelectorAll('vaadin-grid-column')[0].hidden = true;
+    grid.querySelectorAll('vaadin24-grid-column')[0].hidden = true;
     // Flush row height and position calculations
     flushGrid(grid);
     expect(rows[0].getBoundingClientRect().bottom).to.eql(rows[1].getBoundingClientRect().top);
