Index: test/scrolling-mode.common.js
===================================================================
--- test/scrolling-mode.common.js
+++ test/scrolling-mode.common.js
@@ -7,11 +7,11 @@
 
   beforeEach(() => {
     grid = fixtureSync(`
-      <vaadin-grid style="width: 50px; height: 400px;" size="1000">
-        <vaadin-grid-column></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid style="width: 50px; height: 400px;" size="1000">
+        <vaadin24-grid-column></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
-    grid.querySelector('vaadin-grid-column').renderer = (root, _, model) => {
+    grid.querySelector('vaadin24-grid-column').renderer = (root, _, model) => {
       root.textContent = model.index;
     };
     grid.dataProvider = infiniteDataProvider;
@@ -151,11 +151,11 @@
 
   beforeEach(() => {
     grid = fixtureSync(`
-      <vaadin-grid style="width: 50px; height: 400px;" size="1000">
-        <vaadin-grid-column></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid style="width: 50px; height: 400px;" size="1000">
+        <vaadin24-grid-column></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
-    grid.querySelector('vaadin-grid-column').renderer = (root, _, model) => {
+    grid.querySelector('vaadin24-grid-column').renderer = (root, _, model) => {
       root.textContent = model.index;
     };
     flushGrid(grid);
@@ -173,11 +173,11 @@
     document.documentElement.scrollTop = 10000;
     const scrollTop = document.documentElement.scrollTop;
     const grid = fixtureSync(`
-      <vaadin-grid style="width: 50px; height: 400px;" size="1000">
-        <vaadin-grid-column></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid style="width: 50px; height: 400px;" size="1000">
+        <vaadin24-grid-column></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
-    grid.querySelector('vaadin-grid-column').renderer = (root, _, model) => {
+    grid.querySelector('vaadin24-grid-column').renderer = (root, _, model) => {
       root.textContent = model.index;
     };
     const newScrollTop = document.documentElement.scrollTop;
