Index: test/scroll-to-index.test.js
===================================================================
--- test/scroll-to-index.test.js
+++ test/scroll-to-index.test.js
@@ -13,9 +13,9 @@
 
 const createGrid = (height, size) => {
   const grid = fixtureSync(`
-    <vaadin-grid style="width: 200px; height: ${height}px;" size="${size}">
-      <vaadin-grid-column header="foo"></vaadin-grid-column>
-    </vaadin-grid>
+    <vaadin24-grid style="width: 200px; height: ${height}px;" size="${size}">
+      <vaadin24-grid-column header="foo"></vaadin24-grid-column>
+    </vaadin24-grid>
   `);
   grid.firstElementChild.renderer = (root, _, model) => {
     root.textContent = model.index;
@@ -224,9 +224,9 @@
 
     beforeEach(() => {
       grid = fixtureSync(`
-        <vaadin-grid style="width: 200px; height: 500px;" item-has-children-path="hasChildren">
-          <vaadin-grid-tree-column path="name" header="foo"></vaadin-grid-tree-column>
-        </vaadin-grid>
+        <vaadin24-grid style="width: 200px; height: 500px;" item-has-children-path="hasChildren">
+          <vaadin24-grid-tree-column path="name" header="foo"></vaadin24-grid-tree-column>
+        </vaadin24-grid>
       `);
     });
 
@@ -563,9 +563,9 @@
 
     beforeEach(async () => {
       grid = fixtureSync(`
-        <vaadin-grid>
-          <vaadin-grid-column></vaadin-grid-column>
-        </vaadin-grid>
+        <vaadin24-grid>
+          <vaadin24-grid-column></vaadin24-grid-column>
+        </vaadin24-grid>
       `);
       grid.items = Array.from({ length: 100 }, (_, index) => `Item ${index}`);
       grid.scrollToIndex(50);
@@ -583,8 +583,8 @@
 
     beforeEach(async () => {
       const container = fixtureSync('<div></div>');
-      grid = document.createElement('vaadin-grid');
-      const column = document.createElement('vaadin-grid-column');
+      grid = document.createElement('vaadin24-grid');
+      const column = document.createElement('vaadin24-grid-column');
       grid.appendChild(column);
       grid.items = Array.from({ length: 100 }, (_, index) => `Item ${index}`);
       grid.scrollToIndex(50);
