Index: test/scroll-restoration.common.js
===================================================================
--- test/scroll-restoration.common.js
+++ test/scroll-restoration.common.js
@@ -14,17 +14,17 @@
     beforeEach(() => {
       const fixture = fixtureSync(`
       <div>
-        <vaadin-grid style="height: 200px; width: 200px;" size="100">
-          <vaadin-grid-column header="foo"></vaadin-grid-column>
-        </vaadin-grid>
-        <vaadin-grid style="height: 200px; width: 200px;" hidden size="100">
-          <vaadin-grid-column header="bar"></vaadin-grid-column>
-        </vaadin-grid>
+        <vaadin24-grid style="height: 200px; width: 200px;" size="100">
+          <vaadin24-grid-column header="foo"></vaadin24-grid-column>
+        </vaadin24-grid>
+        <vaadin24-grid style="height: 200px; width: 200px;" hidden size="100">
+          <vaadin24-grid-column header="bar"></vaadin24-grid-column>
+        </vaadin24-grid>
       </div>
     `);
-      const grids = fixture.querySelectorAll('vaadin-grid');
+      const grids = fixture.querySelectorAll('vaadin24-grid');
       grids.forEach((grid) => {
-        grid.querySelector('vaadin-grid-column').renderer = (root, _, model) => {
+        grid.querySelector('vaadin24-grid-column').renderer = (root, _, model) => {
           root.textContent = model.index;
         };
         grid.dataProvider = infiniteDataProvider;
