Index: test/scroll-into-view.test.js
===================================================================
--- test/scroll-into-view.test.js
+++ test/scroll-into-view.test.js
@@ -25,11 +25,11 @@
 
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid style="height: 150px">
-        <vaadin-grid-column></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid style="height: 150px">
+        <vaadin24-grid-column></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
-    column = grid.querySelector('vaadin-grid-column');
+    column = grid.querySelector('vaadin24-grid-column');
     column.renderer = (root, _, model) => {
       root.innerHTML = `<div style="height: 100px">${model.item} <textarea style="height: 80px"></div>`;
     };
