Index: test/scroll-to-index.test.js
===================================================================
--- test/scroll-to-index.test.js
+++ test/scroll-to-index.test.js
@@ -1,6 +1,6 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, listenOnce, nextFrame } from '@vaadin/testing-helpers';
-import '@vaadin/polymer-legacy-adapter/template-renderer.js';
+import '@scoped-vaadin/polymer-legacy-adapter/template-renderer.js';
 import '../vaadin-grid.js';
 import '../vaadin-grid-tree-column.js';
 import {
@@ -14,25 +14,25 @@
 
 const fixtures = {
   small: `
-    <vaadin-grid style="width: 200px; height: 200px;" size="100">
-      <vaadin-grid-column>
+    <vaadin23-grid style="width: 200px; height: 200px;" size="100">
+      <vaadin23-grid-column>
         <template class="header">foo</template>
         <template>[[index]]</template>
-      </vaadin-grid-column>
-    </vaadin-grid>
+      </vaadin23-grid-column>
+    </vaadin23-grid>
   `,
   large: `
-    <vaadin-grid style="width: 200px; height: 500px;" size="500000000">
-      <vaadin-grid-column>
+    <vaadin23-grid style="width: 200px; height: 500px;" size="500000000">
+      <vaadin23-grid-column>
         <template class="header">foo</template>
         <template>[[index]]</template>
-      </vaadin-grid-column>
-    </vaadin-grid>
+      </vaadin23-grid-column>
+    </vaadin23-grid>
   `,
   treeGrid: `
-    <vaadin-grid style="width: 200px; height: 500px;" item-has-children-path="hasChildren">
-      <vaadin-grid-tree-column path="name" header="foo"></vaadin-grid-tree-column>
-    </vaadin-grid>
+    <vaadin23-grid style="width: 200px; height: 500px;" item-has-children-path="hasChildren">
+      <vaadin23-grid-tree-column path="name" header="foo"></vaadin23-grid-tree-column>
+    </vaadin23-grid>
   `,
 };
 
