Index: test/selection.test.js
===================================================================
--- test/selection.test.js
+++ test/selection.test.js
@@ -1,7 +1,7 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, listenOnce } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import '@vaadin/polymer-legacy-adapter/template-renderer.js';
+import '@scoped-vaadin/polymer-legacy-adapter/template-renderer.js';
 import '../vaadin-grid.js';
 import '../vaadin-grid-selection-column.js';
 import '../vaadin-grid-filter.js';
@@ -17,20 +17,20 @@
 
 const fixtures = {
   renderer: `
-    <vaadin-grid style="width: 200px; height: 200px;" size="10">
-      <vaadin-grid-column></vaadin-grid-column>
-      <vaadin-grid-column></vaadin-grid-column>
-    </vaadin-grid>
+    <vaadin23-grid style="width: 200px; height: 200px;" size="10">
+      <vaadin23-grid-column></vaadin23-grid-column>
+      <vaadin23-grid-column></vaadin23-grid-column>
+    </vaadin23-grid>
   `,
   template: `
-    <vaadin-grid style="width: 200px; height: 200px;" size="10">
-      <vaadin-grid-column>
+    <vaadin23-grid style="width: 200px; height: 200px;" size="10">
+      <vaadin23-grid-column>
         <template>foo</template>
-      </vaadin-grid-column>
-      <vaadin-grid-column>
+      </vaadin23-grid-column>
+      <vaadin23-grid-column>
         <template>bar</template>
-      </vaadin-grid-column>
-    </vaadin-grid>
+      </vaadin23-grid-column>
+    </vaadin23-grid>
   `,
 };
 
@@ -177,24 +177,24 @@
 
   beforeEach(() => {
     grid = fixtureSync(`
-      <vaadin-grid style="width: 200px; height: 200px;">
-        <vaadin-grid-selection-column auto-select></vaadin-grid-selection-column>
-        <vaadin-grid-selection-column>
+      <vaadin23-grid style="width: 200px; height: 200px;">
+        <vaadin23-grid-selection-column auto-select></vaadin23-grid-selection-column>
+        <vaadin23-grid-selection-column>
           <template class="header">header</template>
           <template>[[item]]</template>
-        </vaadin-grid-selection-column>
-        <vaadin-grid-column-group>
-          <vaadin-grid-selection-column></vaadin-grid-selection-column>
-        </vaadin-grid-column-group>
-        <vaadin-grid-column>
+        </vaadin23-grid-selection-column>
+        <vaadin23-grid-column-group>
+          <vaadin23-grid-selection-column></vaadin23-grid-selection-column>
+        </vaadin23-grid-column-group>
+        <vaadin23-grid-column>
           <template class="header">
-            <vaadin-grid-filter path="value" value="[[filter]]">
+            <vaadin23-grid-filter path="value" value="[[filter]]">
               <input id="filter" slot="filter" value="{{filter::input}}">
-            </vaadin-grid-filter>
+            </vaadin23-grid-filter>
           </template>
           <template></template>
-        </vaadin-grid-column>
-      </vaadin-grid>
+        </vaadin23-grid-column>
+      </vaadin23-grid>
     `);
 
     grid.items = ['foo', 'bar', 'baz'];
@@ -221,7 +221,7 @@
   });
 
   it('should have a checkbox in the body cell', () => {
-    expect(firstBodyCheckbox.localName).to.eql('vaadin-checkbox');
+    expect(firstBodyCheckbox.localName).to.eql('vaadin23-checkbox');
   });
 
   it('should select item when checkbox is checked', () => {
@@ -268,7 +268,7 @@
   });
 
   it('should have a select all checkbox in the header', () => {
-    expect(selectAllCheckbox.localName).to.eql('vaadin-checkbox');
+    expect(selectAllCheckbox.localName).to.eql('vaadin23-checkbox');
   });
 
   it('should have class name for the select all checkbox', () => {
@@ -411,10 +411,10 @@
 
   it('should not clear pre-selected items', () => {
     const grid = fixtureSync(`
-      <vaadin-grid selected-items='[{"value": "foo"}]'>
-        <vaadin-grid-selection-column></vaadin-grid-selection-column>
-        <vaadin-grid-column path="value"></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin23-grid selected-items='[{"value": "foo"}]'>
+        <vaadin23-grid-selection-column></vaadin23-grid-selection-column>
+        <vaadin23-grid-column path="value"></vaadin23-grid-column>
+      </vaadin23-grid>
     `);
     expect(grid.selectedItems).to.have.length(1);
   });
