Index: test/selection.common.js
===================================================================
--- test/selection.common.js
+++ test/selection.common.js
@@ -31,10 +31,10 @@
   describe('with renderer', () => {
     beforeEach(async () => {
       grid = fixtureSync(`
-        <vaadin-grid style="width: 200px; height: 200px;" size="10">
-          <vaadin-grid-column></vaadin-grid-column>
-          <vaadin-grid-column></vaadin-grid-column>
-        </vaadin-grid>
+        <vaadin24-grid style="width: 200px; height: 200px;" size="10">
+          <vaadin24-grid-column></vaadin24-grid-column>
+          <vaadin24-grid-column></vaadin24-grid-column>
+        </vaadin24-grid>
       `);
 
       const cols = grid.children;
@@ -117,10 +117,10 @@
   describe('renderer cells', () => {
     beforeEach(async () => {
       grid = fixtureSync(`
-        <vaadin-grid style="width: 200px; height: 200px;" size="10">
-          <vaadin-grid-column></vaadin-grid-column>
-          <vaadin-grid-column></vaadin-grid-column>
-        </vaadin-grid>
+        <vaadin24-grid style="width: 200px; height: 200px;" size="10">
+          <vaadin24-grid-column></vaadin24-grid-column>
+          <vaadin24-grid-column></vaadin24-grid-column>
+        </vaadin24-grid>
       `);
 
       const cols = grid.children;
@@ -172,16 +172,16 @@
 
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid style="width: 200px; height: 200px;">
-        <vaadin-grid-selection-column auto-select></vaadin-grid-selection-column>
-        <vaadin-grid-selection-column header="header"></vaadin-grid-selection-column>
+      <vaadin24-grid style="width: 200px; height: 200px;">
+        <vaadin24-grid-selection-column auto-select></vaadin24-grid-selection-column>
+        <vaadin24-grid-selection-column header="header"></vaadin24-grid-selection-column>
 
-        <vaadin-grid-column-group>
-          <vaadin-grid-selection-column></vaadin-grid-selection-column>
-        </vaadin-grid-column-group>
+        <vaadin24-grid-column-group>
+          <vaadin24-grid-selection-column></vaadin24-grid-selection-column>
+        </vaadin24-grid-column-group>
 
-        <vaadin-grid-filter-column path="value"></vaadin-grid-filter-column>
-      </vaadin-grid>
+        <vaadin24-grid-filter-column path="value"></vaadin24-grid-filter-column>
+      </vaadin24-grid>
     `);
 
     grid.querySelector('[header="header"]').renderer = (root, _, { item }) => {
@@ -214,7 +214,7 @@
   });
 
   it('should have a checkbox in the body cell', () => {
-    expect(firstBodyCheckbox.localName).to.eql('vaadin-checkbox');
+    expect(firstBodyCheckbox.localName).to.eql('vaadin24-checkbox');
   });
 
   it('should have the checkbox unselected by default', () => {
@@ -334,7 +334,7 @@
   });
 
   it('should have a select all checkbox in the header', () => {
-    expect(selectAllCheckbox.localName).to.eql('vaadin-checkbox');
+    expect(selectAllCheckbox.localName).to.eql('vaadin24-checkbox');
   });
 
   it('should have class name for the select all checkbox', () => {
@@ -516,10 +516,10 @@
 
   it('should not clear pre-selected items', () => {
     const grid = fixtureSync(`
-      <vaadin-grid selected-items='[{"value": "foo"}]'>
-        <vaadin-grid-selection-column></vaadin-grid-selection-column>
-        <vaadin-grid-column path="value"></vaadin-grid-column>
-      </vaadin-grid>
+      <vaadin24-grid selected-items='[{"value": "foo"}]'>
+        <vaadin24-grid-selection-column></vaadin24-grid-selection-column>
+        <vaadin24-grid-column path="value"></vaadin24-grid-column>
+      </vaadin24-grid>
     `);
     expect(grid.selectedItems).to.have.length(1);
   });
@@ -570,7 +570,7 @@
     firstBodyCheckbox.click();
 
     await nextFrame();
-    const filter = grid.querySelector('vaadin-grid-filter');
+    const filter = grid.querySelector('vaadin24-grid-filter');
     filter.value = 'foo';
     filter.dispatchEvent(new CustomEvent('input', { bubbles: true, composed: true }));
 
@@ -648,10 +648,10 @@
 
     beforeEach(async () => {
       grid = fixtureSync(`
-        <vaadin-grid style="width: 200px; height: 450px;">
-          <vaadin-grid-selection-column drag-select></vaadin-grid-selection-column>
-          <vaadin-grid-column></vaadin-grid-column>
-        </vaadin-grid>
+        <vaadin24-grid style="width: 200px; height: 450px;">
+          <vaadin24-grid-selection-column drag-select></vaadin24-grid-selection-column>
+          <vaadin24-grid-column></vaadin24-grid-column>
+        </vaadin24-grid>
       `);
       await nextFrame();
       grid.items = [...new Array(100)].map((_, i) => `${i}`);
@@ -737,7 +737,7 @@
 
     it('should not toggle checkbox after dragging on a single checkbox', () => {
       const cell = getBodyCellContent(grid, 1, 0);
-      const checkBox = cell.querySelector('vaadin-checkbox');
+      const checkBox = cell.querySelector('vaadin24-checkbox');
       const input = checkBox.querySelector('input');
 
       fireTrackEvent(input, input, 'start');
