Index: test/tree-toggle.test.js
===================================================================
--- test/tree-toggle.test.js
+++ test/tree-toggle.test.js
@@ -1,7 +1,7 @@
 import { expect } from '@esm-bundle/chai';
 import { click, fixtureSync } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
-import '@vaadin/polymer-legacy-adapter/template-renderer.js';
+import '@scoped-vaadin/polymer-legacy-adapter/template-renderer.js';
 import '../vaadin-grid.js';
 import '../vaadin-grid-tree-toggle.js';
 import '../vaadin-grid-tree-column.js';
@@ -12,7 +12,7 @@
 
   describe('default', () => {
     beforeEach(() => {
-      toggle = fixtureSync('<vaadin-grid-tree-toggle></vaadin-grid-tree-toggle>');
+      toggle = fixtureSync('<vaadin24-grid-tree-toggle></vaadin24-grid-tree-toggle>');
     });
 
     describe('properties', () => {
@@ -91,11 +91,11 @@
   describe('with content', () => {
     beforeEach(() => {
       toggle = fixtureSync(`
-        <vaadin-grid-tree-toggle>
+        <vaadin24-grid-tree-toggle>
           <label for="foo-input">foo label</label>
           <input id="foo-input">
           <div>foo</div>
-        </vaadin-grid-tree-toggle>
+        </vaadin24-grid-tree-toggle>
       `);
     });
 
@@ -123,11 +123,11 @@
 
     beforeEach(() => {
       grid = fixtureSync(`
-        <vaadin-grid>
-          <vaadin-grid-tree-column></vaadin-grid-tree-column>
-        </vaadin-grid>
+        <vaadin24-grid>
+          <vaadin24-grid-tree-column></vaadin24-grid-tree-column>
+        </vaadin24-grid>
       `);
-      column = grid.querySelector('vaadin-grid-tree-column');
+      column = grid.querySelector('vaadin24-grid-tree-column');
       grid.dataProvider = (params, cb) => {
         cb([{ name: 'foo', hasChildren: true }], 1);
       };
