Index: mixins/helper.js
===================================================================
--- mixins/helper.js
+++ mixins/helper.js
@@ -7,7 +7,7 @@
 import '../sizing.js';
 import '../style.js';
 import '../typography.js';
-import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 export const helper = css`
   :host([has-helper]) [part='helper-text']::before {
