Index: mixins/loader.js
===================================================================
--- mixins/loader.js
+++ mixins/loader.js
@@ -5,7 +5,7 @@
  */
 import '../color.js';
 import '../sizing.js';
-import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const loader = css`
   [part~='loader'] {
