Index: vaadin-lumo-styles/mixins/menu-overlay.js
===================================================================
--- vaadin-lumo-styles/mixins/menu-overlay.js	null
+++ vaadin-lumo-styles/mixins/menu-overlay.js	null
@@ -5,7 +5,7 @@
  */
 import '../spacing.js';
 import '../style.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { overlay } from './overlay.js';
 
 const menuOverlayCore = css`
