Index: vaadin-lumo-styles/mixins/required-field.js
===================================================================
--- vaadin-lumo-styles/mixins/required-field.js
+++ vaadin-lumo-styles/mixins/required-field.js
@@ -7,7 +7,7 @@
 import '../spacing.js';
 import '../style.js';
 import '../typography.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const requiredField = css`
   [part='label'] {
