Index: test/validation.common.js
===================================================================
--- test/validation.common.js
+++ test/validation.common.js
@@ -37,7 +37,7 @@
 
   describe('basic', () => {
     beforeEach(async () => {
-      emailField = fixtureSync('<vaadin-email-field></vaadin-email-field>');
+      emailField = fixtureSync('<vaadin24-email-field></vaadin24-email-field>');
       await nextRender();
     });
 
@@ -88,7 +88,7 @@
 
   describe('custom pattern', () => {
     beforeEach(async () => {
-      emailField = fixtureSync('<vaadin-email-field pattern=".+@example.com"></vaadin-email-field>');
+      emailField = fixtureSync('<vaadin24-email-field pattern=".+@example.com"></vaadin24-email-field>');
       await nextRender();
     });
 
@@ -101,7 +101,7 @@
     let validateSpy;
 
     beforeEach(() => {
-      emailField = document.createElement('vaadin-email-field');
+      emailField = document.createElement('vaadin24-email-field');
       validateSpy = sinon.spy(emailField, 'validate');
     });
 
