Index: test/fixtures/mock-grid-pro-host.js
===================================================================
--- test/fixtures/mock-grid-pro-host.js
+++ test/fixtures/mock-grid-pro-host.js
@@ -1,18 +1,19 @@
-import '@vaadin/grid-pro';
-import '@vaadin/grid-pro/vaadin-grid-pro-edit-column';
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
+import '@scoped-vaadin/grid-pro';
+import '@scoped-vaadin/grid-pro/vaadin-grid-pro-edit-column';
 import { html, PolymerElement } from '@polymer/polymer';
 
 export class MockGridProHost extends PolymerElement {
   static get template() {
     return html`
-      <vaadin-grid-pro id="grid" items="[[items]]">
-        <vaadin-grid-pro-edit-column path="title">
+      <vaadin24-grid-pro id="grid" items="[[items]]">
+        <vaadin24-grid-pro-edit-column path="title">
           <template class="body">[[item.title]]</template>
           <template class="editor">
             <input value="{{item.title::input}}" />
           </template>
-        </vaadin-grid-pro-edit-column>
-      </vaadin-grid-pro>
+        </vaadin24-grid-pro-edit-column>
+      </vaadin24-grid-pro>
     `;
   }
 
@@ -30,4 +31,4 @@
   onClick() {}
 }
 
-customElements.define('mock-grid-pro-host', MockGridProHost);
+internalCustomElements.define('mock-grid-pro-host', MockGridProHost);
