Index: test/visual/lumo/item.test.js
===================================================================
--- test/visual/lumo/item.test.js
+++ test/visual/lumo/item.test.js
@@ -12,7 +12,7 @@
   beforeEach(() => {
     div = document.createElement('div');
     div.style.display = 'inline-block';
-    element = fixtureSync('<vaadin-item>Basic item</vaadin-item>', div);
+    element = fixtureSync('<vaadin23-item>Basic item</vaadin23-item>', div);
   });
 
   ['ltr', 'rtl'].forEach((dir) => {
