Index: test/progress-bar.common.js
===================================================================
--- test/progress-bar.common.js
+++ test/progress-bar.common.js
@@ -1,3 +1,4 @@
+import { internalCustomElements } from '@scoped-vaadin/internal-custom-elements-registry';
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextRender, nextUpdate } from '@vaadin/testing-helpers';
 
@@ -8,16 +9,16 @@
     let tagName;
 
     beforeEach(() => {
-      progress = fixtureSync('<vaadin-progress-bar></vaadin-progress-bar>');
+      progress = fixtureSync('<vaadin24-progress-bar></vaadin24-progress-bar>');
       tagName = progress.tagName.toLowerCase();
     });
 
     it('should be defined in custom element registry', () => {
-      expect(customElements.get(tagName)).to.be.ok;
+      expect(internalCustomElements.get(tagName)).to.be.ok;
     });
 
     it('should have a valid static "is" getter', () => {
-      expect(customElements.get(tagName).is).to.equal(tagName);
+      expect(internalCustomElements.get(tagName).is).to.equal(tagName);
     });
   });
 
@@ -25,7 +26,7 @@
     let value;
 
     beforeEach(async () => {
-      progress = fixtureSync('<vaadin-progress-bar></vaadin-progress-bar>');
+      progress = fixtureSync('<vaadin24-progress-bar></vaadin24-progress-bar>');
       await nextRender();
       value = progress.shadowRoot.querySelector('[part="value"]');
     });
@@ -124,7 +125,7 @@
     beforeEach(async () => {
       layout = fixtureSync(`
         <div style="display: flex; flex-direction: column; align-items: flex-start;">
-          <vaadin-progress-bar></vaadin-progress-bar>
+          <vaadin24-progress-bar></vaadin24-progress-bar>
         </div>
       `);
       await nextRender();
