Index: test/validation.test.js
===================================================================
--- test/validation.test.js
+++ test/validation.test.js
@@ -8,7 +8,7 @@
 
   describe('initial', () => {
     beforeEach(() => {
-      field = document.createElement('vaadin-text-field');
+      field = document.createElement('vaadin24-text-field');
       validateSpy = sinon.spy(field, 'validate');
     });
 
@@ -60,7 +60,7 @@
     let validatedSpy;
 
     beforeEach(async () => {
-      field = fixtureSync('<vaadin-text-field></vaadin-text-field>');
+      field = fixtureSync('<vaadin24-text-field></vaadin24-text-field>');
       await nextRender();
       validatedSpy = sinon.spy();
       field.addEventListener('validated', validatedSpy);
@@ -88,7 +88,7 @@
 
   describe('required', () => {
     beforeEach(async () => {
-      field = fixtureSync('<vaadin-text-field></vaadin-text-field>');
+      field = fixtureSync('<vaadin24-text-field></vaadin24-text-field>');
       await nextRender();
     });
 
@@ -106,7 +106,7 @@
 
   describe('minlength', () => {
     beforeEach(async () => {
-      field = fixtureSync('<vaadin-text-field></vaadin-text-field>');
+      field = fixtureSync('<vaadin24-text-field></vaadin24-text-field>');
       await nextRender();
     });
 
@@ -152,7 +152,7 @@
 
   describe('maxlength', () => {
     beforeEach(async () => {
-      field = fixtureSync('<vaadin-text-field></vaadin-text-field>');
+      field = fixtureSync('<vaadin24-text-field></vaadin24-text-field>');
       await nextRender();
     });
 
@@ -197,7 +197,7 @@
 
   describe('pattern', () => {
     beforeEach(async () => {
-      field = fixtureSync('<vaadin-text-field></vaadin-text-field>');
+      field = fixtureSync('<vaadin24-text-field></vaadin24-text-field>');
       await nextRender();
     });
 
