Index: test/dom/avatar.test.js
===================================================================
--- test/dom/avatar.test.js
+++ test/dom/avatar.test.js
@@ -2,11 +2,11 @@
 import { fixtureSync } from '@vaadin/testing-helpers';
 import '../../src/vaadin-avatar.js';
 
-describe('vaadin-avatar', () => {
+describe('vaadin24-avatar', () => {
   let avatar;
 
   beforeEach(() => {
-    avatar = fixtureSync('<vaadin-avatar></vaadin-avatar>');
+    avatar = fixtureSync('<vaadin24-avatar></vaadin24-avatar>');
   });
 
   it('default', async () => {
