Index: date-picker/theme/material/vaadin-date-picker-overlay-styles.js
===================================================================
--- date-picker/theme/material/vaadin-date-picker-overlay-styles.js
+++ date-picker/theme/material/vaadin-date-picker-overlay-styles.js
@@ -1,5 +1,5 @@
-import { overlay } from '@vaadin/vaadin-material-styles/mixins/overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { overlay } from '@scoped-vaadin/vaadin-material-styles/mixins/overlay.js';
+import { css, registerStyles } from '@scoped-vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const datePickerOverlay = css`
   :host([fullscreen]) {
@@ -42,6 +42,6 @@
   }
 `;
 
-registerStyles('vaadin-date-picker-overlay', [overlay, datePickerOverlay], {
+registerStyles('vaadin24-date-picker-overlay', [overlay, datePickerOverlay], {
   moduleId: 'material-date-picker-overlay',
 });
