Index: test/dom/__snapshots__/vaadin-upload.test.snap.js
===================================================================
--- test/dom/__snapshots__/vaadin-upload.test.snap.js
+++ test/dom/__snapshots__/vaadin-upload.test.snap.js
@@ -5,14 +5,14 @@
 `<div part="primary-buttons">
   <div id="addFiles">
     <slot name="add-button">
-      <vaadin-button
+      <vaadin24-button
         id="addButton"
         part="upload-button"
         role="button"
         tabindex="0"
       >
         Upload Files...
-      </vaadin-button>
+      </vaadin24-button>
     </slot>
   </div>
   <div
@@ -38,22 +38,22 @@
     part="file-list"
   >
     <li>
-      <vaadin-upload-file
+      <vaadin24-upload-file
         complete=""
         tabindex="0"
       >
-      </vaadin-upload-file>
+      </vaadin24-upload-file>
     </li>
     <li>
-      <vaadin-upload-file tabindex="0">
-      </vaadin-upload-file>
+      <vaadin24-upload-file tabindex="0">
+      </vaadin24-upload-file>
     </li>
     <li>
-      <vaadin-upload-file
+      <vaadin24-upload-file
         error=""
         tabindex="0"
       >
-      </vaadin-upload-file>
+      </vaadin24-upload-file>
     </li>
     <dom-repeat
       as="file"
