Index: test/password-field.test.js
===================================================================
--- test/password-field.test.js
+++ test/password-field.test.js
@@ -8,7 +8,7 @@
   let passwordField, input, revealButton;
 
   beforeEach(() => {
-    passwordField = fixtureSync('<vaadin-password-field></vaadin-password-field>');
+    passwordField = fixtureSync('<vaadin23-password-field></vaadin23-password-field>');
     input = passwordField.inputElement;
     revealButton = passwordField.querySelector('[slot=reveal]');
   });
@@ -184,7 +184,7 @@
     let validateSpy;
 
     beforeEach(() => {
-      passwordField = document.createElement('vaadin-password-field');
+      passwordField = document.createElement('vaadin23-password-field');
       validateSpy = sinon.spy(passwordField, 'validate');
     });
 
@@ -242,7 +242,7 @@
   let field;
 
   beforeEach(() => {
-    field = fixtureSync('<vaadin-password-field invalid></vaadin-password-field>');
+    field = fixtureSync('<vaadin23-password-field invalid></vaadin23-password-field>');
   });
 
   it('should not remove "invalid" state when ready', () => {
@@ -254,7 +254,7 @@
   let field;
 
   beforeEach(() => {
-    field = fixtureSync('<vaadin-password-field invalid value="123456"></vaadin-password-field>');
+    field = fixtureSync('<vaadin23-password-field invalid value="123456"></vaadin23-password-field>');
   });
 
   it('should not remove "invalid" state when ready', () => {
@@ -267,7 +267,7 @@
   let revealButton;
 
   beforeEach(() => {
-    passwordField = fixtureSync('<vaadin-password-field disabled></vaadin-password-field>');
+    passwordField = fixtureSync('<vaadin23-password-field disabled></vaadin23-password-field>');
     revealButton = passwordField.querySelector('[slot=reveal]');
   });
 
@@ -323,7 +323,7 @@
 
   describe('default', () => {
     beforeEach(() => {
-      passwordField = fixtureSync('<vaadin-password-field></vaadin-password-field>');
+      passwordField = fixtureSync('<vaadin23-password-field></vaadin23-password-field>');
       revealButton = passwordField.querySelector('[slot=reveal]');
     });
 
@@ -339,7 +339,7 @@
 
   describe('set before attach', () => {
     beforeEach(() => {
-      passwordField = document.createElement('vaadin-password-field');
+      passwordField = document.createElement('vaadin23-password-field');
     });
 
     afterEach(() => {
