Index: test/password-field.test.js
===================================================================
--- test/password-field.test.js
+++ test/password-field.test.js
@@ -8,7 +8,7 @@
   let passwordField, input, revealButton;
 
   beforeEach(async () => {
-    passwordField = fixtureSync('<vaadin-password-field></vaadin-password-field>');
+    passwordField = fixtureSync('<vaadin24-password-field></vaadin24-password-field>');
     await nextRender();
     input = passwordField.inputElement;
     revealButton = passwordField.querySelector('[slot=reveal]');
@@ -226,7 +226,7 @@
   let revealButton;
 
   beforeEach(async () => {
-    passwordField = fixtureSync('<vaadin-password-field disabled></vaadin-password-field>');
+    passwordField = fixtureSync('<vaadin24-password-field disabled></vaadin24-password-field>');
     await nextRender();
     revealButton = passwordField.querySelector('[slot=reveal]');
   });
@@ -288,7 +288,7 @@
 
   describe('default', () => {
     beforeEach(async () => {
-      passwordField = fixtureSync('<vaadin-password-field></vaadin-password-field>');
+      passwordField = fixtureSync('<vaadin24-password-field></vaadin24-password-field>');
       await nextRender();
       revealButton = passwordField.querySelector('[slot=reveal]');
     });
@@ -306,7 +306,7 @@
 
   describe('set before attach', () => {
     beforeEach(() => {
-      passwordField = document.createElement('vaadin-password-field');
+      passwordField = document.createElement('vaadin24-password-field');
     });
 
     afterEach(() => {
