Index: test/password-field.test.js
===================================================================
--- test/password-field.test.js
+++ test/password-field.test.js
@@ -8,7 +8,7 @@
   let passwordField, input, revealButton;
 
   beforeEach(() => {
-    passwordField = fixtureSync('<vaadin-password-field></vaadin-password-field>');
+    passwordField = fixtureSync('<vaadin24-password-field></vaadin24-password-field>');
     input = passwordField.inputElement;
     revealButton = passwordField.querySelector('[slot=reveal]');
   });
@@ -215,7 +215,7 @@
   let revealButton;
 
   beforeEach(() => {
-    passwordField = fixtureSync('<vaadin-password-field disabled></vaadin-password-field>');
+    passwordField = fixtureSync('<vaadin24-password-field disabled></vaadin24-password-field>');
     revealButton = passwordField.querySelector('[slot=reveal]');
   });
 
@@ -271,7 +271,7 @@
 
   describe('default', () => {
     beforeEach(() => {
-      passwordField = fixtureSync('<vaadin-password-field></vaadin-password-field>');
+      passwordField = fixtureSync('<vaadin24-password-field></vaadin24-password-field>');
       revealButton = passwordField.querySelector('[slot=reveal]');
     });
 
@@ -287,7 +287,7 @@
 
   describe('set before attach', () => {
     beforeEach(() => {
-      passwordField = document.createElement('vaadin-password-field');
+      passwordField = document.createElement('vaadin24-password-field');
     });
 
     afterEach(() => {
