Index: test/validation.test.js
===================================================================
--- test/validation.test.js
+++ test/validation.test.js
@@ -10,7 +10,7 @@
     let validateSpy;
 
     beforeEach(() => {
-      passwordField = document.createElement('vaadin-password-field');
+      passwordField = document.createElement('vaadin24-password-field');
       validateSpy = sinon.spy(passwordField, 'validate');
     });
 
@@ -42,7 +42,7 @@
 
   describe('basic', () => {
     beforeEach(async () => {
-      passwordField = fixtureSync('<vaadin-password-field></vaadin-password-field>');
+      passwordField = fixtureSync('<vaadin24-password-field></vaadin24-password-field>');
       await nextRender();
     });
 
@@ -71,7 +71,7 @@
 
   describe('invalid', () => {
     beforeEach(async () => {
-      passwordField = fixtureSync('<vaadin-password-field invalid></vaadin-password-field>');
+      passwordField = fixtureSync('<vaadin24-password-field invalid></vaadin24-password-field>');
       await nextRender();
     });
 
@@ -82,7 +82,7 @@
 
   describe('invalid with value', () => {
     beforeEach(async () => {
-      passwordField = fixtureSync('<vaadin-password-field invalid value="123456"></vaadin-password-field>');
+      passwordField = fixtureSync('<vaadin24-password-field invalid value="123456"></vaadin24-password-field>');
       await nextRender();
     });
 
