Index: test/validation.common.js
===================================================================
--- test/validation.common.js
+++ test/validation.common.js
@@ -9,7 +9,7 @@
     let validateSpy;
 
     beforeEach(() => {
-      passwordField = document.createElement('vaadin-password-field');
+      passwordField = document.createElement('vaadin24-password-field');
       validateSpy = sinon.spy(passwordField, 'validate');
     });
 
@@ -41,7 +41,7 @@
 
   describe('basic', () => {
     beforeEach(async () => {
-      passwordField = fixtureSync('<vaadin-password-field></vaadin-password-field>');
+      passwordField = fixtureSync('<vaadin24-password-field></vaadin24-password-field>');
       await nextRender();
     });
 
@@ -70,7 +70,7 @@
 
   describe('invalid', () => {
     beforeEach(async () => {
-      passwordField = fixtureSync('<vaadin-password-field invalid></vaadin-password-field>');
+      passwordField = fixtureSync('<vaadin24-password-field invalid></vaadin24-password-field>');
       await nextRender();
     });
 
@@ -81,7 +81,7 @@
 
   describe('invalid with value', () => {
     beforeEach(async () => {
-      passwordField = fixtureSync('<vaadin-password-field invalid value="123456"></vaadin-password-field>');
+      passwordField = fixtureSync('<vaadin24-password-field invalid value="123456"></vaadin24-password-field>');
       await nextRender();
     });
 
