Index: test/dom/details.test.js
===================================================================
--- test/dom/details.test.js
+++ test/dom/details.test.js
@@ -1,18 +1,18 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync } from '@vaadin/testing-helpers';
 import '../../vaadin-details.js';
-import { resetUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
+import { resetUniqueId } from '@scoped-vaadin/component-base/src/unique-id-utils.js';
 
-describe('vaadin-details', () => {
+describe('vaadin24-details', () => {
   let details;
 
   beforeEach(() => {
     resetUniqueId();
     details = fixtureSync(`
-      <vaadin-details>
-        <vaadin-details-summary slot="summary">Summary</vaadin-details-summary>
+      <vaadin24-details>
+        <vaadin24-details-summary slot="summary">Summary</vaadin24-details-summary>
         <div>Content</div>
-      </vaadin-details>
+      </vaadin24-details>
     `);
   });
 
