Index: test/dom/details.test.js
===================================================================
--- test/dom/details.test.js
+++ test/dom/details.test.js
@@ -7,10 +7,10 @@
 
   beforeEach(() => {
     details = fixtureSync(`
-      <vaadin-details>
+      <vaadin23-details>
         <div slot="summary">Summary</div>
         <input>
-      </vaadin-details>
+      </vaadin23-details>
     `);
   });
 
